---
title: fhR-4.4.2-foss-2024a
date: 2025-03-11
---

### Known Issues
 * None

### Package List
  * [ALDEx2-1.38.0](https://www.bioconductor.org/packages/release/bioc/html/ALDEx2.html) Analysis Of Differential Abundance Taking Sample and Scale Variation Into Account
  * [ALL-1.46.0](https://www.bioconductor.org/packages/release/bioc/html/ALL.html) A data package
  * [AgiMicroRna-2.56.0](https://www.bioconductor.org/packages/release/bioc/html/AgiMicroRna.html) Processing and Differential Expression Analysis of Agilent microRNA chips
  * [AnnotationDbi-1.68.0](https://www.bioconductor.org/packages/release/bioc/html/AnnotationDbi.html) Manipulation of SQLite-based annotations in Bioconductor
  * [AnnotationFilter-1.30.0](https://www.bioconductor.org/packages/release/bioc/html/AnnotationFilter.html) Facilities for Filtering Bioconductor Annotation Resources
  * [AnnotationForge-1.48.0](https://www.bioconductor.org/packages/release/bioc/html/AnnotationForge.html) Tools for building SQLite-based annotation data packages
  * [AnnotationHub-3.14.0](https://www.bioconductor.org/packages/release/bioc/html/AnnotationHub.html) Client to access AnnotationHub resources
  * ArchR 1.0.2 [CRAN]&emsp;
  * [BSgenome-1.74.0](https://www.bioconductor.org/packages/release/bioc/html/BSgenome.html) Software infrastructure for efficient representation of full genomes and their SNPs
  * [BSgenome.Hsapiens.UCSC.hg19-1.4.3](https://www.bioconductor.org/packages/release/bioc/html/BSgenome.Hsapiens.UCSC.hg19.html) Full genome sequences for Homo sapiens (UCSC version hg19, based on GRCh37.p13)
  * [Bhat-0.9-12](https://cran.r-project.org/web/packages/Bhat/index.html) General Likelihood Exploration
  * [Biobase-2.66.0](https://www.bioconductor.org/packages/release/bioc/html/Biobase.html) Biobase: Base functions for Bioconductor
  * [BiocBaseUtils-1.8.0](https://www.bioconductor.org/packages/release/bioc/html/BiocBaseUtils.html) General utility functions for developing Bioconductor packages
  * [BiocFileCache-2.14.0](https://www.bioconductor.org/packages/release/bioc/html/BiocFileCache.html) Manage Files Across Sessions
  * [BiocGenerics-0.52.0](https://www.bioconductor.org/packages/release/bioc/html/BiocGenerics.html) S4 generic functions used in Bioconductor
  * [BiocIO-1.16.0](https://www.bioconductor.org/packages/release/bioc/html/BiocIO.html) Standard Input and Output for Bioconductor Packages
  * [BiocManager-1.30.25](https://cran.r-project.org/web/packages/BiocManager/index.html) Access the Bioconductor Project Package Repository
  * [BiocNeighbors-2.0.1](https://www.bioconductor.org/packages/release/bioc/html/BiocNeighbors.html) Nearest Neighbor Detection for Bioconductor Packages
  * [BiocParallel-1.40.0](https://www.bioconductor.org/packages/release/bioc/html/BiocParallel.html) Bioconductor facilities for parallel evaluation
  * [BiocSingular-1.22.0](https://www.bioconductor.org/packages/release/bioc/html/BiocSingular.html) Singular Value Decomposition for Bioconductor Packages
  * [BiocStyle-2.34.0](https://www.bioconductor.org/packages/release/bioc/html/BiocStyle.html) Standard styles for vignettes and other Bioconductor documents
  * [BiocVersion-3.20.0](https://www.bioconductor.org/packages/release/bioc/html/BiocVersion.html) Set the appropriate version of Bioconductor packages
  * [Biostrings-2.74.1](https://www.bioconductor.org/packages/release/bioc/html/Biostrings.html) Efficient manipulation of biological strings
  * [BivarP-1.0](https://cran.r-project.org/web/packages/BivarP/index.html) Estimating the Parameters of Some Bivariate Distributions
  * [Bolstad-0.2.42](https://cran.r-project.org/web/packages/Bolstad/index.html) Functions for Elementary Bayesian Inference
  * [Bolstad2-1.0-29](https://cran.r-project.org/web/packages/Bolstad2/index.html) Bolstad Functions
  * [BradleyTerry2-1.1-2](https://cran.r-project.org/web/packages/BradleyTerry2/index.html) Bradley-Terry Models
  * [CATALYST-1.30.2](https://www.bioconductor.org/packages/release/bioc/html/CATALYST.html) Cytometry dATa anALYSis Tools
  * [CGHbase-1.66.0](https://www.bioconductor.org/packages/release/bioc/html/CGHbase.html) CGHbase: Base functions and classes for arrayCGH data analysis.
  * [CLONETv2-2.2.1](https://cran.r-project.org/web/packages/CLONETv2/index.html) Clonality Estimates in Tumor
  * [CMA-1.64.0](https://www.bioconductor.org/packages/release/bioc/html/CMA.html) Synthesis of microarray-based classification
  * [CNEr-1.42.0](https://www.bioconductor.org/packages/release/bioc/html/CNEr.html) CNE Detection and Visualization
  * [COMPASS-1.44.0](https://www.bioconductor.org/packages/release/bioc/html/COMPASS.html) Combinatorial Polyfunctionality Analysis of Single Cells
  * CaSpER v2.0 [CRAN]&emsp;
  * [Category-2.72.0](https://www.bioconductor.org/packages/release/bioc/html/Category.html) Category Analysis
  * [ChIPQC-1.42.0](https://www.bioconductor.org/packages/release/bioc/html/ChIPQC.html) Quality metrics for ChIPseq data
  * [ChIPpeakAnno-3.40.0](https://www.bioconductor.org/packages/release/bioc/html/ChIPpeakAnno.html) Batch annotation of the peaks identified from either ChIP-seq, ChIP-chip experiments, or any experiments that result in large number of genomic interval data
  * [ChIPseeker-1.42.1](https://www.bioconductor.org/packages/release/bioc/html/ChIPseeker.html) ChIPseeker for ChIP peak Annotation, Comparison, and Visualization
  * [ComplexHeatmap-2.22.0](https://www.bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html) Make Complex Heatmaps
  * [ConsensusClusterPlus-1.70.0](https://www.bioconductor.org/packages/release/bioc/html/ConsensusClusterPlus.html) ConsensusClusterPlus
  * [CytoML-2.18.2](https://www.bioconductor.org/packages/release/bioc/html/CytoML.html) A GatingML Interface for Cross Platform Cytometry Data Sharing
  * [DAAG-1.25.6](https://cran.r-project.org/web/packages/DAAG/index.html) Data Analysis and Graphics Data and Functions
  * [DECIPHER-3.2.0](https://www.bioconductor.org/packages/release/bioc/html/DECIPHER.html) Tools for curating, analyzing, and manipulating biological sequences
  * [DEGseq-1.60.0](https://www.bioconductor.org/packages/release/bioc/html/DEGseq.html) Identify Differentially Expressed Genes from RNA-seq data
  * [DESeq2-1.46.0](https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html) Differential gene expression analysis based on the negative binomial distribution
  * [DNAcopy-1.80.0](https://www.bioconductor.org/packages/release/bioc/html/DNAcopy.html) DNA Copy Number Data Analysis
  * [DO.db-2.9](https://www.bioconductor.org/packages/release/bioc/html/DO.db.html) A set of annotation maps describing the entire Disease Ontology
  * [DOSE-4.0.0](https://www.bioconductor.org/packages/release/bioc/html/DOSE.html) Disease Ontology Semantic and Enrichment analysis
  * [DOT-0.1](https://cran.r-project.org/web/packages/DOT/index.html) Render and Export DOT Graphs in R
  * [DelayedArray-0.32.0](https://www.bioconductor.org/packages/release/bioc/html/DelayedArray.html) A unified framework for working transparently with on-disk and in-memory array-like datasets
  * [DelayedMatrixStats-1.28.1](https://www.bioconductor.org/packages/release/bioc/html/DelayedMatrixStats.html) Functions that Apply to Rows and Columns of 'DelayedMatrix' Objects
  * [DiceDesign-1.10](https://cran.r-project.org/web/packages/DiceDesign/index.html) Designs of Computer Experiments
  * [DiffBind-3.16.0](https://www.bioconductor.org/packages/release/bioc/html/DiffBind.html) Differential Binding Analysis of ChIP-Seq Peak Data
  * [DirichletMultinomial-1.48.0](https://www.bioconductor.org/packages/release/bioc/html/DirichletMultinomial.html) Dirichlet-Multinomial Mixture Model Machine Learning for Microbiome Data
  * [DynDoc-1.84.0](https://www.bioconductor.org/packages/release/bioc/html/DynDoc.html) Dynamic document tools
  * [EBImage-4.48.0](https://www.bioconductor.org/packages/release/bioc/html/EBImage.html) Image processing and analysis toolbox for R
  * [EpiModel-2.5.0](https://cran.r-project.org/web/packages/EpiModel/index.html) Mathematical Modeling of Infectious Disease Dynamics
  * [ExperimentHub-2.14.0](https://www.bioconductor.org/packages/release/bioc/html/ExperimentHub.html) Client to access ExperimentHub resources
  * [FDb.InfiniumMethylation.hg19-2.2.0](https://www.bioconductor.org/packages/release/bioc/html/FDb.InfiniumMethylation.hg19.html) Annotation package for Illumina Infinium DNA methylation probes
  * [FlowSOM-2.14.0](https://www.bioconductor.org/packages/release/bioc/html/FlowSOM.html) Using self-organizing maps for visualization and interpretation of cytometry data
  * [GEOquery-2.74.0](https://www.bioconductor.org/packages/release/bioc/html/GEOquery.html) Get data from NCBI Gene Expression Omnibus (GEO)
  * [GO.db-3.19.1](https://www.bioconductor.org/packages/release/bioc/html/GO.db.html) A set of annotation maps describing the entire Gene Ontology
  * [GOSemSim-2.32.0](https://www.bioconductor.org/packages/release/bioc/html/GOSemSim.html) GO-terms Semantic Similarity Measures
  * [GOstats-2.72.0](https://www.bioconductor.org/packages/release/bioc/html/GOstats.html) Tools for manipulating GO and microarrays
  * [GPfit-1.0-8](https://cran.r-project.org/web/packages/GPfit/index.html) Gaussian Processes Modeling
  * [GSEABase-1.68.0](https://www.bioconductor.org/packages/release/bioc/html/GSEABase.html) Gene set enrichment data structures and methods
  * [GSVA-2.0.5](https://www.bioconductor.org/packages/release/bioc/html/GSVA.html) Gene Set Variation Analysis for Microarray and RNA-Seq Data
  * [GWASExactHW-1.2](https://cran.r-project.org/web/packages/GWASExactHW/index.html) Exact Hardy-Weinburg Testing for Genome Wide Association Studies
  * [GWASTools-1.52.0](https://www.bioconductor.org/packages/release/bioc/html/GWASTools.html) Tools for Genome Wide Association Studies
  * [GenomeInfoDb-1.42.3](https://www.bioconductor.org/packages/release/bioc/html/GenomeInfoDb.html) Utilities for manipulating chromosome names, including modifying them to follow a particular naming style
  * [GenomeInfoDbData-1.2.12](https://www.bioconductor.org/packages/release/bioc/html/GenomeInfoDbData.html) Species and taxonomy ID look up tables used by GenomeInfoDb
  * [GenomicAlignments-1.42.0](https://www.bioconductor.org/packages/release/bioc/html/GenomicAlignments.html) Representation and manipulation of short genomic alignments
  * [GenomicFeatures-1.58.0](https://www.bioconductor.org/packages/release/bioc/html/GenomicFeatures.html) Query the gene models of a given organism/assembly
  * [GenomicFiles-1.42.0](https://www.bioconductor.org/packages/release/bioc/html/GenomicFiles.html) Distributed computing by file or by range
  * [GenomicRanges-1.58.0](https://www.bioconductor.org/packages/release/bioc/html/GenomicRanges.html) Representation and manipulation of genomic intervals
  * [Glimma-2.16.0](https://www.bioconductor.org/packages/release/bioc/html/Glimma.html) Interactive visualizations for gene expression analysis
  * [Gmisc-3.0.3](https://cran.r-project.org/web/packages/Gmisc/index.html) Descriptive Statistics, Transition Plots, and More
  * [GreyListChIP-1.38.0](https://www.bioconductor.org/packages/release/bioc/html/GreyListChIP.html) Grey Lists -- Mask Artefact Regions Based on ChIP Inputs
  * [Gviz-1.50.0](https://www.bioconductor.org/packages/release/bioc/html/Gviz.html) Plotting data and annotation information along genomic coordinates
  * [HDF5Array-1.34.0](https://www.bioconductor.org/packages/release/bioc/html/HDF5Array.html) HDF5 datasets as array-like objects in R
  * [HDO.db-0.99.1](https://www.bioconductor.org/packages/release/bioc/html/HDO.db.html) A set of annotation maps describing the entire Human Disease Ontology
  * [HMMcopy-1.48.0](https://www.bioconductor.org/packages/release/bioc/html/HMMcopy.html) Copy number prediction with correction for GC and mappability bias for HTS data
  * [HSMMSingleCell-1.24.0](https://www.bioconductor.org/packages/release/bioc/html/HSMMSingleCell.html) Single-cell RNA-Seq for differentiating human skeletal muscle myoblasts (HSMM)
  * [HaploSim-1.8.4.2](https://cran.r-project.org/web/packages/HaploSim/index.html) Functions to Simulate Haplotypes
  * [Homo.sapiens-1.3.1](https://www.bioconductor.org/packages/release/bioc/html/Homo.sapiens.html) Annotation package for the Homo.sapiens object
  * [IHW-1.34.0](https://www.bioconductor.org/packages/release/bioc/html/IHW.html) Independent Hypothesis Weighting
  * [IRanges-2.40.1](https://www.bioconductor.org/packages/release/bioc/html/IRanges.html) Foundation of integer range manipulation in Bioconductor
  * [IRdisplay-1.1](https://cran.r-project.org/web/packages/IRdisplay/index.html) 'Jupyter' Display Machinery
  * [IlluminaHumanMethylation450kmanifest-0.4.0](https://www.bioconductor.org/packages/release/bioc/html/IlluminaHumanMethylation450kmanifest.html) Annotation for Illumina's 450k methylation arrays
  * [InteractionSet-1.34.0](https://www.bioconductor.org/packages/release/bioc/html/InteractionSet.html) Base Classes for Storing Genomic Interaction Data
  * [KEGGREST-1.46.0](https://www.bioconductor.org/packages/release/bioc/html/KEGGREST.html) Client-side REST access to the Kyoto Encyclopedia of Genes and Genomes (KEGG)
  * [KEGGgraph-1.66.0](https://www.bioconductor.org/packages/release/bioc/html/KEGGgraph.html) KEGGgraph: A graph approach to KEGG PATHWAY in R and Bioconductor
  * [LogicReg-1.6.6](https://cran.r-project.org/web/packages/LogicReg/index.html) Logic Regression
  * [LymphoSeq-1.34.0](https://www.bioconductor.org/packages/release/bioc/html/LymphoSeq.html) Analyze high-throughput sequencing of T and B cell receptors
  * [LymphoSeqDB-0.99.2](https://www.bioconductor.org/packages/release/bioc/html/LymphoSeqDB.html) LymphoSeq annotation databases
  * [MAGeCKFlute-2.9.0](https://www.bioconductor.org/packages/release/bioc/html/MAGeCKFlute.html) Integrative Analysis Pipeline for Pooled CRISPR Functional Genetic Screens
  * [MAST-1.32.0](https://www.bioconductor.org/packages/release/bioc/html/MAST.html) Model-based Analysis of Single Cell Transcriptomics
  * [MCMCglmm-2.36](https://cran.r-project.org/web/packages/MCMCglmm/index.html) MCMC Generalised Linear Mixed Models
  * [MPV-1.64](https://cran.r-project.org/web/packages/MPV/index.html) Data Sets from Montgomery, Peck and Vining
  * [MatrixEQTL-2.3](https://cran.r-project.org/web/packages/MatrixEQTL/index.html) Matrix eQTL: Ultra Fast eQTL Analysis via Large Matrix
Operations
  * [MatrixExtra-0.1.15](https://cran.r-project.org/web/packages/MatrixExtra/index.html) Extra Methods for Sparse Matrices
  * [MatrixGenerics-1.18.1](https://www.bioconductor.org/packages/release/bioc/html/MatrixGenerics.html) S4 Generic Summary Statistic Functions that Operate on Matrix-Like Objects
  * [Mfuzz-2.66.0](https://www.bioconductor.org/packages/release/bioc/html/Mfuzz.html) Soft clustering of omics time series data
  * [NADA-1.6-1.1](https://cran.r-project.org/web/packages/NADA/index.html) Nondetects and Data Analysis for Environmental Data
  * [NOISeq-2.50.0](https://www.bioconductor.org/packages/release/bioc/html/NOISeq.html) Exploratory analysis and differential expression for RNA-seq data
  * [Nozzle.R1-1.1-1.1](https://cran.r-project.org/web/packages/Nozzle.R1/index.html) Nozzle Reports
  * [OrganismDbi-1.48.0](https://www.bioconductor.org/packages/release/bioc/html/OrganismDbi.html) Software to enable the smooth interfacing of different database packages
  * [PBSddesolve-1.13.4](https://cran.r-project.org/web/packages/PBSddesolve/index.html) Solver for Delay Differential Equations
  * [PFAM.db-3.19.1](https://www.bioconductor.org/packages/release/bioc/html/PFAM.db.html) A set of protein ID mappings for PFAM
  * [PICS-2.50.0](https://www.bioconductor.org/packages/release/bioc/html/PICS.html) Probabilistic inference of ChIP-seq
  * [PolynomF-2.0-8](https://cran.r-project.org/web/packages/PolynomF/index.html) Polynomials in R
  * [ProtGenerics-1.38.0](https://www.bioconductor.org/packages/release/bioc/html/ProtGenerics.html) Generic infrastructure for Bioconductor mass spectrometry packages
  * [QUIC-1.1.1](https://cran.r-project.org/web/packages/QUIC/index.html) Regularized Sparse Inverse Covariance Matrix Estimation
  * [R2jags-0.8-9](https://cran.r-project.org/web/packages/R2jags/index.html) Using R to Run 'JAGS'
  * [RBGL-1.82.0](https://cran.r-project.org/web/packages/RBGL/index.html) An interface to the BOOST graph library
  * [REDCapR-1.4.0](https://cran.r-project.org/web/packages/REDCapR/index.html) Interaction Between R and REDCap
  * [RLRsim-3.1-8](https://cran.r-project.org/web/packages/RLRsim/index.html) Exact (Restricted) Likelihood Ratio Tests for Mixed and Additive
Models
  * [RNASeqPower-1.46.0](https://www.bioconductor.org/packages/release/bioc/html/RNASeqPower.html) Sample size for RNAseq studies
  * [RPostgres-1.4.8](https://cran.r-project.org/web/packages/RPostgres/index.html) C++ Interface to PostgreSQL
  * [RPresto-1.4.7](https://cran.r-project.org/web/packages/RPresto/index.html) DBI Connector to Presto
  * [RProtoBuf-0.4.23](https://cran.r-project.org/web/packages/RProtoBuf/index.html) R Interface to the 'Protocol Buffers' 'API' (Version 2 or 3)
  * [RProtoBufLib-2.18.0](https://www.bioconductor.org/packages/release/bioc/html/RProtoBufLib.html) C++ headers and static libraries of Protocol buffers
  * [RSclient-0.7-10](https://cran.r-project.org/web/packages/RSclient/index.html) Client for Rserve
  * [RVenn-1.1.0](https://cran.r-project.org/web/packages/RVenn/index.html) Set Operations for Many Sets
  * [RaggedExperiment-1.30.0](https://www.bioconductor.org/packages/release/bioc/html/RaggedExperiment.html) Representation of Sparse Experiments and Assays Across Samples
  * [RcppAnnoy-0.0.22](https://cran.r-project.org/web/packages/RcppAnnoy/index.html) 'Rcpp' Bindings for 'Annoy', a Library for Approximate Nearest
Neighbors
  * [RcppCCTZ-0.2.13](https://cran.r-project.org/web/packages/RcppCCTZ/index.html) 'Rcpp' Bindings for the 'CCTZ' Library
  * [RcppDate-0.0.5](https://cran.r-project.org/web/packages/RcppDate/index.html) 'date' C++ Header Library for Date and Time Functionality
  * [RcppHNSW-0.6.0](https://cran.r-project.org/web/packages/RcppHNSW/index.html) 'Rcpp' Bindings for 'hnswlib', a Library for Approximate Nearest
Neighbors
  * [RcppML-0.3.7](https://cran.r-project.org/web/packages/RcppML/index.html) Rcpp Machine Learning Library
  * [RcppNumerical-0.6-0](https://cran.r-project.org/web/packages/RcppNumerical/index.html) 'Rcpp' Integration for Numerical Computing Libraries
  * RcppPlanc 1.0.0 [CRAN]&emsp;
  * [RcppZiggurat-0.1.6](https://cran.r-project.org/web/packages/RcppZiggurat/index.html) 'Rcpp' Integration of Different "Ziggurat" Normal RNG
Implementations
  * [Rdsdp-1.0.5.2.1](https://cran.r-project.org/web/packages/Rdsdp/index.html) R Interface to DSDP Semidefinite Programming Library
  * [ReportingTools-2.46.0](https://www.bioconductor.org/packages/release/bioc/html/ReportingTools.html) Tools for making reports in various formats
  * [Rfast-2.1.5](https://cran.r-project.org/web/packages/Rfast/index.html) A Collection of Efficient and Extremely Fast R Functions
  * [Rgraphviz-2.50.0](https://cran.r-project.org/web/packages/Rgraphviz/index.html) Provides plotting capabilities for R graph objects
  * [Rhdf5lib-1.28.0](https://www.bioconductor.org/packages/release/bioc/html/Rhdf5lib.html) hdf5 library as an R package
  * [Rhpc-0.21-247](https://cran.r-project.org/web/packages/Rhpc/index.html) Permits *apply() Style Dispatch for `HPC`
  * [Rhtslib-3.2.0](https://www.bioconductor.org/packages/release/bioc/html/Rhtslib.html) HTSlib high-throughput sequencing library as an R package
  * [Rlabkey-3.4.1](https://cran.r-project.org/web/packages/Rlabkey/index.html) Data Exchange Between R and 'LabKey' Server
  * [Rmpi-0.7-3.3](https://cran.r-project.org/web/packages/Rmpi/index.html) Interface (Wrapper) to MPI (Message-Passing Interface)
  * [Rsamtools-2.22.0](https://www.bioconductor.org/packages/release/bioc/html/Rsamtools.html) Binary alignment (BAM), FASTA, variant call (BCF), and tabix file import
  * [Rsubread-2.20.0](https://www.bioconductor.org/packages/release/bioc/html/Rsubread.html) Mapping, quantification and variant analysis of sequencing data
  * [S4Arrays-1.6.0](https://www.bioconductor.org/packages/release/bioc/html/S4Arrays.html) Foundation of array-like containers in Bioconductor
  * [S4Vectors-0.44.0](https://www.bioconductor.org/packages/release/bioc/html/S4Vectors.html) Foundation of vector-like and list-like containers in Bioconductor
  * [SNPRelate-1.40.0](https://cran.r-project.org/web/packages/SNPRelate/index.html) Parallel Computing Toolset for Genome-Wide Association Studies
(GWAS)
  * [SNPassoc-2.1-2](https://cran.r-project.org/web/packages/SNPassoc/index.html) SNPs-Based Whole Genome Association Studies
  * [SPIA-2.58.0](https://www.bioconductor.org/packages/release/bioc/html/SPIA.html) Signaling Pathway Impact Analysis (SPIA) using combined evidence of pathway over-representation and unusual signaling perturbations
  * [ScaledMatrix-1.14.0](https://www.bioconductor.org/packages/release/bioc/html/ScaledMatrix.html) Creating a DelayedMatrix of Scaled and Centered Values
  * [SemiPar-1.0-4.2](https://cran.r-project.org/web/packages/SemiPar/index.html) Semiparametic Regression
  * [ShortRead-1.64.0](https://www.bioconductor.org/packages/release/bioc/html/ShortRead.html) FASTQ input and manipulation
  * [SingleCellExperiment-1.28.1](https://www.bioconductor.org/packages/release/bioc/html/SingleCellExperiment.html) S4 Classes for Single Cell Data
  * [SparseArray-1.6.2](https://www.bioconductor.org/packages/release/bioc/html/SparseArray.html) High-performance sparse data representation and manipulation in R
  * [SpatialExperiment-1.16.0](https://www.bioconductor.org/packages/release/bioc/html/SpatialExperiment.html) S4 Class for Spatially Resolved -omics Data
  * [SummarizedExperiment-1.36.0](https://www.bioconductor.org/packages/release/bioc/html/SummarizedExperiment.html) A container (S4 class) for matrix-like assays
  * [TFBSTools-1.44.0](https://www.bioconductor.org/packages/release/bioc/html/TFBSTools.html) Software Package for Transcription Factor Binding Site (TFBS) Analysis
  * [TFMPvalue-0.0.9](https://cran.r-project.org/web/packages/TFMPvalue/index.html) Efficient and Accurate P-Value Computation for Position Weight
Matrices
  * [TitanCNA-1.44.0](https://www.bioconductor.org/packages/release/bioc/html/TitanCNA.html) Subclonal copy number and LOH prediction from whole genome sequencing of tumours
  * [TxDb.Celegans.UCSC.ce6.ensGene-3.2.2](https://www.bioconductor.org/packages/release/bioc/html/TxDb.Celegans.UCSC.ce6.ensGene.html) Annotation package for TxDb object(s)
  * [TxDb.Dmelanogaster.UCSC.dm3.ensGene-3.2.2](https://www.bioconductor.org/packages/release/bioc/html/TxDb.Dmelanogaster.UCSC.dm3.ensGene.html) Annotation package for TxDb object(s)
  * [TxDb.Hsapiens.UCSC.hg18.knownGene-3.2.2](https://www.bioconductor.org/packages/release/bioc/html/TxDb.Hsapiens.UCSC.hg18.knownGene.html) Annotation package for TxDb object(s)
  * [TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2](https://www.bioconductor.org/packages/release/bioc/html/TxDb.Hsapiens.UCSC.hg19.knownGene.html) Annotation package for TxDb object(s)
  * [TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0](https://www.bioconductor.org/packages/release/bioc/html/TxDb.Mmusculus.UCSC.mm10.knownGene.html) Annotation package for TxDb object(s)
  * [TxDb.Mmusculus.UCSC.mm9.knownGene-3.2.2](https://www.bioconductor.org/packages/release/bioc/html/TxDb.Mmusculus.UCSC.mm9.knownGene.html) Annotation package for TxDb object(s)
  * [TxDb.Rnorvegicus.UCSC.rn4.ensGene-3.2.2](https://www.bioconductor.org/packages/release/bioc/html/TxDb.Rnorvegicus.UCSC.rn4.ensGene.html) Annotation package for TxDb object(s)
  * [UCSC.utils-1.2.0](https://www.bioconductor.org/packages/release/bioc/html/UCSC.utils.html) Low-level utilities to retrieve data from the UCSC Genome Browser
  * [VariantAnnotation-1.52.0](https://www.bioconductor.org/packages/release/bioc/html/VariantAnnotation.html) Annotation of Genetic Variants
  * [Wrench-1.24.0](https://www.bioconductor.org/packages/release/bioc/html/Wrench.html) Wrench normalization for sparse count data
  * [XVector-0.46.0](https://www.bioconductor.org/packages/release/bioc/html/XVector.html) Foundation of external vector representation and manipulation in Bioconductor
  * [affxparser-1.78.0](https://www.bioconductor.org/packages/release/bioc/html/affxparser.html) Affymetrix File Parsing SDK
  * [affy-1.84.0](https://www.bioconductor.org/packages/release/bioc/html/affy.html) Methods for Affymetrix Oligonucleotide Arrays
  * [affycoretools-1.78.0](https://www.bioconductor.org/packages/release/bioc/html/affycoretools.html) Functions useful for those doing repetitive analyses with Affymetrix GeneChips
  * [affyio-1.76.0](https://www.bioconductor.org/packages/release/bioc/html/affyio.html) Tools for parsing Affymetrix data files
  * [alr3-2.0.8](https://cran.r-project.org/web/packages/alr3/index.html) Data to Accompany Applied Linear Regression 3rd Edition
  * [amap-0.8-20](https://cran.r-project.org/web/packages/amap/index.html) Another Multidimensional Analysis Package
  * [annaffy-1.78.0](https://www.bioconductor.org/packages/release/bioc/html/annaffy.html) Annotation tools for Affymetrix biological metadata
  * [annotate-1.84.0](https://www.bioconductor.org/packages/release/bioc/html/annotate.html) Annotation for microarrays
  * [apeglm-1.28.0](https://www.bioconductor.org/packages/release/bioc/html/apeglm.html) Approximate posterior estimation for GLM coefficients
  * [argparser-0.7.2](https://cran.r-project.org/web/packages/argparser/index.html) Command-Line Argument Parser
  * [arsenal-3.6.3](https://cran.r-project.org/web/packages/arsenal/index.html) An Arsenal of 'R' Functions for Large-Scale Statistical
Summaries
  * [arules-1.7-9](https://cran.r-project.org/web/packages/arules/index.html) Mining Association Rules and Frequent Itemsets
  * [ash-1.0-15](https://cran.r-project.org/web/packages/ash/index.html) David Scott's ASH Routines
  * [ashr-2.2-63](https://cran.r-project.org/web/packages/ashr/index.html) Methods for Adaptive Shrinkage, using Empirical Bayes
  * [assorthead-1.0.1](https://www.bioconductor.org/packages/release/bioc/html/assorthead.html) Assorted Header-Only C++ Libraries
  * [aws.ec2metadata-0.2.0](https://cran.r-project.org/web/packages/aws.ec2metadata/index.html) Get EC2 Instance Metadata
  * [aws.s3-0.3.21](https://cran.r-project.org/web/packages/aws.s3/index.html) 'AWS S3' Client Package
  * [aws.signature-0.6.0](https://cran.r-project.org/web/packages/aws.signature/index.html) Amazon Web Services Request Signatures
  * [babelgene-22.9](https://cran.r-project.org/web/packages/babelgene/index.html) Gene Orthologs for Model Organisms in a Tidy Data Format
  * [bamsignals-1.38.0](https://www.bioconductor.org/packages/release/bioc/html/bamsignals.html) Extract read count signals from bam files
  * [base64url-1.4](https://cran.r-project.org/web/packages/base64url/index.html) Fast and URL-Safe Base64 Encoder and Decoder
  * [basilisk-1.18.0](https://www.bioconductor.org/packages/release/bioc/html/basilisk.html) Freezing Python Dependencies Inside Bioconductor Packages
  * [basilisk.utils-1.18.0](https://www.bioconductor.org/packages/release/bioc/html/basilisk.utils.html) Basilisk Installation Utilities
  * [batchtools-0.9.17](https://cran.r-project.org/web/packages/batchtools/index.html) Tools for Computation on Batch Systems
  * [beachmat-2.22.0](https://www.bioconductor.org/packages/release/bioc/html/beachmat.html) Compiling Bioconductor to Handle Each Matrix Type
  * [benchmark-0.3-6](https://cran.r-project.org/web/packages/benchmark/index.html) Benchmark Experiments Toolbox
  * [bezier-1.1.2](https://cran.r-project.org/web/packages/bezier/index.html) Toolkit for Bezier Curves and Splines
  * [biglm-0.9-3](https://cran.r-project.org/web/packages/biglm/index.html) Bounded Memory Linear and Generalized Linear Models
  * [biomaRt-2.62.1](https://www.bioconductor.org/packages/release/bioc/html/biomaRt.html) Interface to BioMart databases (i.e. Ensembl)
  * [biomformat-1.34.0](https://www.bioconductor.org/packages/release/bioc/html/biomformat.html) An interface package for the BIOM file format
  * [biovizBase-1.54.0](https://www.bioconductor.org/packages/release/bioc/html/biovizBase.html) Basic graphic utilities for visualization of genomic data.
  * [bookdown-0.42](https://cran.r-project.org/web/packages/bookdown/index.html) Authoring Books and Technical Documents with R Markdown
  * [breakaway-4.8.4](https://cran.r-project.org/web/packages/breakaway/index.html) Species Richness Estimation and Modeling
  * [bumphunter-1.48.0](https://www.bioconductor.org/packages/release/bioc/html/bumphunter.html) Bump Hunter
  * [catdata-1.2.4](https://cran.r-project.org/web/packages/catdata/index.html) Categorical Data
  * [chipseq-1.56.0](https://www.bioconductor.org/packages/release/bioc/html/chipseq.html) chipseq: A package for analyzing chipseq data
  * [chromVAR-1.28.0](https://www.bioconductor.org/packages/release/bioc/html/chromVAR.html) Chromatin Variation Across Regions
  * [clusterProfiler-4.14.6](https://www.bioconductor.org/packages/release/bioc/html/clusterProfiler.html) A universal enrichment tool for interpreting omics data
  * [collections-0.3.7](https://cran.r-project.org/web/packages/collections/index.html) High Performance Container Data Types
  * [colorRamps-2.3.4](https://cran.r-project.org/web/packages/colorRamps/index.html) Builds Color Tables
  * [compare-0.2-6](https://cran.r-project.org/web/packages/compare/index.html) Comparing Objects for Differences
  * [config-0.3.2](https://cran.r-project.org/web/packages/config/index.html) Manage Environment Specific Configuration Values
  * [cummeRbund-2.48.0](https://www.bioconductor.org/packages/release/bioc/html/cummeRbund.html) Analysis, exploration, manipulation, and visualization of Cufflinks high-throughput sequencing data.
  * [curry-0.1.1](https://cran.r-project.org/web/packages/curry/index.html) Partial Function Application with %<%, %-<%, and %><%
  * [cyclocomp-1.1.1](https://cran.r-project.org/web/packages/cyclocomp/index.html) Cyclomatic Complexity of R Code
  * [cytolib-2.16.0](https://www.bioconductor.org/packages/release/bioc/html/cytolib.html) C++ infrastructure for representing and interacting with the gated cytometry data
  * [dada2-1.34.0](https://www.bioconductor.org/packages/release/bioc/html/dada2.html) Accurate, high-resolution sample inference from amplicon sequencing data
  * [degreenet-1.3-6](https://cran.r-project.org/web/packages/degreenet/index.html) Models for Skewed Count Distributions Relevant to Networks
  * [densvis-1.16.0](https://www.bioconductor.org/packages/release/bioc/html/densvis.html) Density-Preserving Data Visualization via Non-Linear Dimensionality Reduction
  * [depmap-1.18.0](https://www.bioconductor.org/packages/release/bioc/html/depmap.html) Cancer Dependency Map Data Package
  * [derfinder-1.40.0](https://www.bioconductor.org/packages/release/bioc/html/derfinder.html) Annotation-agnostic differential expression analysis of RNA-seq data at base-pair resolution via the DER Finder approach
  * [derfinderHelper-1.40.0](https://www.bioconductor.org/packages/release/bioc/html/derfinderHelper.html) derfinder helper package
  * [descr-1.1.8](https://cran.r-project.org/web/packages/descr/index.html) Descriptive Statistics
  * [dfoptim-2023.1.0](https://cran.r-project.org/web/packages/dfoptim/index.html) Derivative-Free Optimization
  * [dials-1.4.0](https://cran.r-project.org/web/packages/dials/index.html) Tools for Creating Tuning Parameter Values
  * [diamonds-1.0-5](https://cran.r-project.org/web/packages/diamonds/index.html) Analysis and sampling grids from diamond partitions
  * [diffusionMap-1.2.0](https://cran.r-project.org/web/packages/diffusionMap/index.html) Diffusion Map
  * [dir.expiry-1.14.0](https://www.bioconductor.org/packages/release/bioc/html/dir.expiry.html) Managing Expiration for Cache Directories
  * [directlabels-2024.1.21](https://cran.r-project.org/web/packages/directlabels/index.html) Direct Labels for Multicolor Plots
  * [doFuture-1.0.1](https://cran.r-project.org/web/packages/doFuture/index.html) Use Foreach to Parallelize via the Future Framework
  * [doMPI-0.2.2](https://cran.r-project.org/web/packages/doMPI/index.html) Foreach Parallel Adaptor for the Rmpi Package
  * [dqrng-0.4.1](https://cran.r-project.org/web/packages/dqrng/index.html) Fast Pseudo Random Number Generators
  * [drc-3.0-1](https://cran.r-project.org/web/packages/drc/index.html) Analysis of Dose-Response Curves
  * [edgeR-4.4.2](https://www.bioconductor.org/packages/release/bioc/html/edgeR.html) Empirical Analysis of Digital Gene Expression Data in R
  * [egor-1.24.2](https://cran.r-project.org/web/packages/egor/index.html) Import and Analyse Ego-Centered Network Data
  * [elasticnet-1.3](https://cran.r-project.org/web/packages/elasticnet/index.html) Elastic-Net for Sparse Estimation and Sparse PCA
  * [enrichplot-1.26.6](https://www.bioconductor.org/packages/release/bioc/html/enrichplot.html) Visualization of Functional Enrichment Result
  * [ensembldb-2.30.0](https://www.bioconductor.org/packages/release/bioc/html/ensembldb.html) Utilities to create and use Ensembl-based annotation databases
  * [ergm.ego-1.1.0](https://cran.r-project.org/web/packages/ergm.ego/index.html) Fit, Simulate and Diagnose Exponential-Family Random Graph
Models to Egocentrically Sampled Network Data
  * [etrunct-0.1](https://cran.r-project.org/web/packages/etrunct/index.html) Computes Moments of Univariate Truncated t Distribution
  * [europepmc-0.4.3](https://cran.r-project.org/web/packages/europepmc/index.html) R Interface to the Europe PubMed Central RESTful Web Service
  * [fANCOVA-0.6-1](https://cran.r-project.org/web/packages/fANCOVA/index.html) Nonparametric Analysis of Covariance
  * [faraway-1.0.9](https://cran.r-project.org/web/packages/faraway/index.html) Datasets and Functions for Books by Julian Faraway
  * [fastseg-1.52.0](https://www.bioconductor.org/packages/release/bioc/html/fastseg.html) fastseg - a fast segmentation algorithm
  * [fasttime-1.1-0](https://cran.r-project.org/web/packages/fasttime/index.html) Fast Utility Function for Time Parsing and Conversion
  * [fda-6.2.0](https://cran.r-project.org/web/packages/fda/index.html) Functional Data Analysis
  * [fds-1.8](https://cran.r-project.org/web/packages/fds/index.html) Functional Data Sets
  * [fgsea-1.32.2](https://www.bioconductor.org/packages/release/bioc/html/fgsea.html) Fast Gene Set Enrichment Analysis
  * [filelock-1.0.3](https://cran.r-project.org/web/packages/filelock/index.html) Portable File Locking
  * [fit.models-0.64](https://cran.r-project.org/web/packages/fit.models/index.html) Compare Fitted Models
  * [float-0.3-2](https://cran.r-project.org/web/packages/float/index.html) 32-Bit Floats
  * [flowClust-3.44.0](https://www.bioconductor.org/packages/release/bioc/html/flowClust.html) Clustering for Flow Cytometry
  * [flowCore-2.18.0](https://www.bioconductor.org/packages/release/bioc/html/flowCore.html) flowCore: Basic structures for flow cytometry data
  * [flowStats-4.18.0](https://www.bioconductor.org/packages/release/bioc/html/flowStats.html) Statistical methods for the analysis of flow cytometry data
  * [flowViz-1.70.0](https://www.bioconductor.org/packages/release/bioc/html/flowViz.html) Visualization for flow cytometry
  * [flowWorkspace-4.18.0](https://www.bioconductor.org/packages/release/bioc/html/flowWorkspace.html) Infrastructure for representing and interacting with gated and ungated cytometry data sets.
  * [forestplot-3.1.6](https://cran.r-project.org/web/packages/forestplot/index.html) Advanced Forest Plot Using 'grid' Graphics
  * [forge-0.2.0](https://cran.r-project.org/web/packages/forge/index.html) Casting Values into Shape
  * [fts-0.9.9.2](https://cran.r-project.org/web/packages/fts/index.html) R Interface to 'tslib' (a Time Series Library in C++)
  * [gage-2.56.0](https://www.bioconductor.org/packages/release/bioc/html/gage.html) Generally Applicable Gene-set Enrichment for Pathway Analysis
  * [gcrma-2.78.0](https://www.bioconductor.org/packages/release/bioc/html/gcrma.html) Background Adjustment Using Sequence Information
  * [gdsfmt-1.42.1](https://cran.r-project.org/web/packages/gdsfmt/index.html) 
  * [geneLenDataBase-1.39.0](https://www.bioconductor.org/packages/release/bioc/html/geneLenDataBase.html) Lengths of mRNA transcripts for a number of genomes
  * [genefilter-1.88.0](https://www.bioconductor.org/packages/release/bioc/html/genefilter.html) genefilter: methods for filtering genes from high-throughput experiments
  * [geneplotter-1.84.0](https://www.bioconductor.org/packages/release/bioc/html/geneplotter.html) Graphics related functions for Bioconductor
  * [ggVennDiagram-1.5.2](https://cran.r-project.org/web/packages/ggVennDiagram/index.html) A 'ggplot2' Implement of Venn Diagram
  * [ggbio-1.54.0](https://www.bioconductor.org/packages/release/bioc/html/ggbio.html) Visualization tools for genomic data
  * [ggcyto-1.34.0](https://www.bioconductor.org/packages/release/bioc/html/ggcyto.html) Visualize Cytometry data with ggplot
  * [ggdendro-0.2.0](https://cran.r-project.org/web/packages/ggdendro/index.html) Create Dendrograms and Tree Diagrams Using 'ggplot2'
  * [ggnewscale-0.5.1](https://cran.r-project.org/web/packages/ggnewscale/index.html) Multiple Fill and Colour Scales in 'ggplot2'
  * [ggpointdensity-0.1.0](https://cran.r-project.org/web/packages/ggpointdensity/index.html) A Cross Between a 2D Density Plot and a Scatter Plot
  * [ggrastr-1.0.2](https://cran.r-project.org/web/packages/ggrastr/index.html) Rasterize Layers for 'ggplot2'
  * [ggtangle-0.0.6](https://cran.r-project.org/web/packages/ggtangle/index.html) Draw Network with Data
  * [ggthemes-5.1.0](https://cran.r-project.org/web/packages/ggthemes/index.html) Extra Themes, Scales and Geoms for 'ggplot2'
  * [ggtree-3.14.0](https://www.bioconductor.org/packages/release/bioc/html/ggtree.html) an R package for visualization of tree and annotation data
  * [glmpath-0.98](https://cran.r-project.org/web/packages/glmpath/index.html) L1 Regularization Path for Generalized Linear Models and Cox
Proportional Hazards Model
  * [graph-1.84.1](https://cran.r-project.org/web/packages/graph/index.html) graph: A package to handle graph data structures
  * [gsignal-0.3-7](https://cran.r-project.org/web/packages/gsignal/index.html) Signal Processing
  * [gson-0.1.0](https://cran.r-project.org/web/packages/gson/index.html) Base Class and Methods for 'gson' Format
  * [gsubfn-0.7](https://cran.r-project.org/web/packages/gsubfn/index.html) Utilities for Strings and Function Arguments
  * [hapassoc-1.2-9](https://cran.r-project.org/web/packages/hapassoc/index.html) Inference of Trait Associations with SNP Haplotypes and Other
Attributes using the EM Algorithm
  * [haplo.stats-1.9.7](https://cran.r-project.org/web/packages/haplo.stats/index.html) Statistical Analysis of Haplotypes with Traits and Covariates
when Linkage Phase is Ambiguous
  * [hardhat-1.4.1](https://cran.r-project.org/web/packages/hardhat/index.html) Construct Modeling Packages
  * [hdrcde-3.4](https://cran.r-project.org/web/packages/hdrcde/index.html) Highest Density Regions and Conditional Density Estimation
  * [hgu133plus2.db-3.13.0](https://www.bioconductor.org/packages/release/bioc/html/hgu133plus2.db.html) Affymetrix Affymetrix HG-U133_Plus_2 Array annotation data (chip hgu133plus2)
  * [hopach-2.66.0](https://cran.r-project.org/web/packages/hopach/index.html) Hierarchical Ordered Partitioning and Collapsing Hybrid (HOPACH)
  * [hypergate-0.8.5](https://cran.r-project.org/web/packages/hypergate/index.html) Machine Learning of Hyperrectangular Gating Strategies for
High-Dimensional Cytometry
  * [igraphdata-1.0.1](https://cran.r-project.org/web/packages/igraphdata/index.html) A Collection of Network Data Sets for the 'igraph' Package
  * [illuminaio-0.48.0](https://www.bioconductor.org/packages/release/bioc/html/illuminaio.html) Parsing Illumina Microarray Output Files
  * [impute-1.80.0](https://cran.r-project.org/web/packages/impute/index.html) impute: Imputation for microarray data
  * [inferference-1.0.2](https://cran.r-project.org/web/packages/inferference/index.html) Methods for Causal Inference with Interference
  * [interactiveDisplayBase-1.44.0](https://www.bioconductor.org/packages/release/bioc/html/interactiveDisplayBase.html) Base package for enabling powerful shiny web displays of Bioconductor objects
  * [invgamma-1.1](https://cran.r-project.org/web/packages/invgamma/index.html) The Inverse Gamma Distribution
  * [iotools-0.3-5](https://cran.r-project.org/web/packages/iotools/index.html) I/O Tools for Streaming
  * [its-1.1.8](https://cran.r-project.org/web/packages/its/index.html) Irregular Time Series
  * [karyoploteR-1.32.0](https://www.bioconductor.org/packages/release/bioc/html/karyoploteR.html) Plot customizable linear genomes displaying arbitrary data
  * [keyring-1.3.2](https://cran.r-project.org/web/packages/keyring/index.html) Access the System Credential Store from R
  * [kin.cohort-0.7](https://cran.r-project.org/web/packages/kin.cohort/index.html) Analysis of Kin-Cohort Studies
  * [knockoff-0.3.6](https://cran.r-project.org/web/packages/knockoff/index.html) The Knockoff Filter for Controlled Variable Selection
  * [kyotil-2024.11-01](https://cran.r-project.org/web/packages/kyotil/index.html) Utility Functions for Statistical Analysis Report Generation and
Monte Carlo Studies
  * [latentnet-2.11.0](https://cran.r-project.org/web/packages/latentnet/index.html) Latent Position and Cluster Models for Statistical Networks
  * [lgr-0.4.4](https://cran.r-project.org/web/packages/lgr/index.html) A Fully Featured Logging Framework
  * [limma-3.62.2](https://cran.r-project.org/web/packages/limma/index.html) Linear Models for Microarray Data
  * [lintr-3.2.0](https://cran.r-project.org/web/packages/lintr/index.html) A 'Linter' for R Code
  * [lokern-1.1-12](https://cran.r-project.org/web/packages/lokern/index.html) Kernel Regression Smoothing with Local or Global Plug-in
Bandwidth
  * [lpridge-1.1-0](https://cran.r-project.org/web/packages/lpridge/index.html) Local Polynomial (Ridge) Regression
  * [lpsymphony-1.34.0](https://www.bioconductor.org/packages/release/bioc/html/lpsymphony.html) Symphony integer linear programming solver in R
  * [lumi-2.58.0](https://www.bioconductor.org/packages/release/bioc/html/lumi.html) BeadArray Specific Methods for Illumina Methylation and Expression Microarrays
  * [maSigPro-1.78.0](https://www.bioconductor.org/packages/release/bioc/html/maSigPro.html) Significant Gene Expression Profile Differences in Time Course Gene Expression Data
  * [mapdata-2.3.1](https://cran.r-project.org/web/packages/mapdata/index.html) Extra Map Databases
  * [marray-1.84.0](https://www.bioconductor.org/packages/release/bioc/html/marray.html) Exploratory analysis for two-color spotted microarray data
  * [metagenomeSeq-1.48.1](https://www.bioconductor.org/packages/release/bioc/html/metagenomeSeq.html) Statistical analysis for sparse high-throughput sequencing
  * [methylKit-1.32.1](https://www.bioconductor.org/packages/release/bioc/html/methylKit.html) DNA methylation analysis from high-throughput bisulfite sequencing results
  * [methylumi-2.52.0](https://www.bioconductor.org/packages/release/bioc/html/methylumi.html) Handle Illumina methylation data
  * [minfi-1.52.1](https://www.bioconductor.org/packages/release/bioc/html/minfi.html) Analyze Illumina Infinium DNA methylation arrays
  * [mixOmics-6.30.0](https://cran.r-project.org/web/packages/mixOmics/index.html) Omics Data Integration Project
  * [mixsqp-0.3-54](https://cran.r-project.org/web/packages/mixsqp/index.html) Sequential Quadratic Programming for Fast Maximum-Likelihood
Estimation of Mixture Proportions
  * [mlapi-0.1.1](https://cran.r-project.org/web/packages/mlapi/index.html) Abstract Classes for Building 'scikit-learn' Like API
  * [modelenv-0.2.0](https://cran.r-project.org/web/packages/modelenv/index.html) Provide Tools to Register Models for Use in 'tidymodels'
  * [mondate-1.0](https://cran.r-project.org/web/packages/mondate/index.html) Keep Track of Dates in Terms of Months
  * [motifmatchr-1.28.0](https://www.bioconductor.org/packages/release/bioc/html/motifmatchr.html) Fast Motif Matching in R
  * [msa-1.38.0](https://www.bioconductor.org/packages/release/bioc/html/msa.html) Multiple Sequence Alignment
  * [msigdbr-7.5.1](https://cran.r-project.org/web/packages/msigdbr/index.html) MSigDB Gene Sets for Multiple Organisms in a Tidy Data Format
  * [multtest-2.62.0](https://cran.r-project.org/web/packages/multtest/index.html) Resampling-based multiple hypothesis testing
  * [nanotime-0.3.11](https://cran.r-project.org/web/packages/nanotime/index.html) Nanosecond-Resolution Time Support for R
  * [ncdfFlow-2.52.1](https://www.bioconductor.org/packages/release/bioc/html/ncdfFlow.html) ncdfFlow: A package that provides HDF5 based storage for flow cytometry data.
  * [networksis-2.1-3](https://cran.r-project.org/web/packages/networksis/index.html) Simulate Bipartite Graphs with Fixed Marginals Through
Sequential Importance Sampling
  * [numbers-0.8-5](https://cran.r-project.org/web/packages/numbers/index.html) Number-Theoretic Functions
  * [nws-1.7.0.1](https://cran.r-project.org/web/packages/nws/index.html) R functions for NetWorkSpaces and Sleigh
  * [odbc-1.6.0](https://cran.r-project.org/web/packages/odbc/index.html) Connect to ODBC Compatible Databases (using the DBI Interface)
  * [oligo-1.70.1](https://www.bioconductor.org/packages/release/bioc/html/oligo.html) Preprocessing tools for oligonucleotide arrays
  * [oligoClasses-1.68.0](https://www.bioconductor.org/packages/release/bioc/html/oligoClasses.html) Classes for high-throughput arrays supported by oligo and crlmm
  * [openCyto-2.18.0](https://www.bioconductor.org/packages/release/bioc/html/openCyto.html) Hierarchical Gating Pipeline for flow cytometry data
  * [org.Hs.eg.db-3.19.1](https://www.bioconductor.org/packages/release/bioc/html/org.Hs.eg.db.html) Genome wide annotation for Human
  * [org.Mm.eg.db-3.19.1](https://www.bioconductor.org/packages/release/bioc/html/org.Mm.eg.db.html) Genome wide annotation for Mouse
  * [pack-0.1-1](https://cran.r-project.org/web/packages/pack/index.html) Convert values to/from raw vectors
  * [parsnip-1.3.0](https://cran.r-project.org/web/packages/parsnip/index.html) A Common API to Modeling and Analysis Functions
  * [partools-1.1.6](https://cran.r-project.org/web/packages/partools/index.html) Tools for the 'Parallel' Package
  * [pathview-1.46.0](https://www.bioconductor.org/packages/release/bioc/html/pathview.html) a tool set for pathway based data integration and visualization
  * [paws-0.8.0](https://cran.r-project.org/web/packages/paws/index.html) Amazon Web Services Software Development Kit
  * [paws.analytics-0.8.0](https://cran.r-project.org/web/packages/paws.analytics/index.html) 'Amazon Web Services' Analytics Services
  * [paws.application.integration-0.8.0](https://cran.r-project.org/web/packages/paws.application.integration/index.html) 'Amazon Web Services' Application Integration Services
  * [paws.common-0.8.2](https://cran.r-project.org/web/packages/paws.common/index.html) Paws Low-Level Amazon Web Services API
  * [paws.compute-0.8.0](https://cran.r-project.org/web/packages/paws.compute/index.html) 'Amazon Web Services' Compute Services
  * [paws.cost.management-0.8.0](https://cran.r-project.org/web/packages/paws.cost.management/index.html) 'Amazon Web Services' Cost Management Services
  * [paws.customer.engagement-0.8.0](https://cran.r-project.org/web/packages/paws.customer.engagement/index.html) 'Amazon Web Services' Customer Engagement Services
  * [paws.database-0.8.0](https://cran.r-project.org/web/packages/paws.database/index.html) 'Amazon Web Services' Database Services
  * [paws.developer.tools-0.8.0](https://cran.r-project.org/web/packages/paws.developer.tools/index.html) 'Amazon Web Services' Developer Tools Services
  * [paws.end.user.computing-0.8.0](https://cran.r-project.org/web/packages/paws.end.user.computing/index.html) 'Amazon Web Services' End User Computing Services
  * [paws.machine.learning-0.8.0](https://cran.r-project.org/web/packages/paws.machine.learning/index.html) 'Amazon Web Services' Machine Learning Services
  * [paws.management-0.8.0](https://cran.r-project.org/web/packages/paws.management/index.html) 'Amazon Web Services' Management & Governance Services
  * [paws.networking-0.8.0](https://cran.r-project.org/web/packages/paws.networking/index.html) 'Amazon Web Services' Networking & Content Delivery Services
  * [paws.security.identity-0.8.0](https://cran.r-project.org/web/packages/paws.security.identity/index.html) 'Amazon Web Services' Security, Identity, & Compliance Services
  * [paws.storage-0.8.0](https://cran.r-project.org/web/packages/paws.storage/index.html) 'Amazon Web Services' Storage Services
  * [pbs-1.1](https://cran.r-project.org/web/packages/pbs/index.html) Periodic B Splines
  * [pcaMethods-1.98.0](https://www.bioconductor.org/packages/release/bioc/html/pcaMethods.html) A collection of PCA methods
  * [pdist-1.2.1](https://cran.r-project.org/web/packages/pdist/index.html) Partitioned Distance Function
  * [pedigree-1.4.2](https://cran.r-project.org/web/packages/pedigree/index.html) Pedigree Functions
  * [pedigreemm-0.3-5](https://cran.r-project.org/web/packages/pedigreemm/index.html) Pedigree-Based Mixed-Effects Models
  * [perm-1.0-0.4](https://cran.r-project.org/web/packages/perm/index.html) Exact or Asymptotic Permutation Tests
  * [phyloseq-1.50.0](https://www.bioconductor.org/packages/release/bioc/html/phyloseq.html) Handling and analysis of high-throughput microbiome census data
  * [poisbinom-1.0.1](https://cran.r-project.org/web/packages/poisbinom/index.html) A Faster Implementation of the Poisson-Binomial Distribution
  * [polyester-1.39.0](https://www.bioconductor.org/packages/release/bioc/html/polyester.html) Simulate RNA-seq reads
  * [polyreg-0.8.0](https://cran.r-project.org/web/packages/polyreg/index.html) Polynomial Regression
  * [poweRlaw-1.0.0](https://cran.r-project.org/web/packages/poweRlaw/index.html) Analysis of Heavy Tailed Distributions
  * premessa 0.3.4 [CRAN]&emsp; premessa is an R package for pre-processing of flow and mass cytometry data, that includes panel editing/renaming for FCS files, bead-based normalization and debarcoding.
  * [preprocessCore-1.68.0](https://www.bioconductor.org/packages/release/bioc/html/preprocessCore.html) A collection of pre-processing functions
  * [psychotools-0.7-4](https://cran.r-project.org/web/packages/psychotools/index.html) Psychometric Modeling Infrastructure
  * [pwalign-1.2.0](https://www.bioconductor.org/packages/release/bioc/html/pwalign.html) Perform pairwise sequence alignments
  * [qdapRegex-0.7.8](https://cran.r-project.org/web/packages/qdapRegex/index.html) Regular Expression Removal, Extraction, and Replacement Tools
  * [qpdf-1.3.4](https://cran.r-project.org/web/packages/qpdf/index.html) Split, Combine and Compress PDF Files
  * [qqconf-1.3.2](https://cran.r-project.org/web/packages/qqconf/index.html) Creates Simultaneous Testing Bands for QQ-Plots
  * [qtl-1.70](https://cran.r-project.org/web/packages/qtl/index.html) Tools for Analyzing QTL Experiments
  * [quantoptr-0.1.3](https://cran.r-project.org/web/packages/quantoptr/index.html) Algorithms for Quantile- And Mean-Optimal Treatment Regimes
  * [quantsmooth-1.72.0](https://www.bioconductor.org/packages/release/bioc/html/quantsmooth.html) Quantile smoothing and genomic visualization of array data
  * [qvalue-2.38.0](https://cran.r-project.org/web/packages/qvalue/index.html) Q-value estimation for false discovery rate control
  * [qvcalc-1.0.4](https://cran.r-project.org/web/packages/qvcalc/index.html) Quasi Variances for Factor Effects in Statistical Models
  * [r2d3-0.2.6](https://cran.r-project.org/web/packages/r2d3/index.html) Interface to 'D3' Visualizations
  * [rGADEM-2.54.0](https://www.bioconductor.org/packages/release/bioc/html/rGADEM.html) de novo motif discovery
  * [rainbow-3.8](https://cran.r-project.org/web/packages/rainbow/index.html) Bagplots, Boxplots and Rainbow Plots for Functional Data
  * [rbenchmark-1.0.0](https://cran.r-project.org/web/packages/rbenchmark/index.html) Benchmarking routine for R
  * [refGenome-1.7.7](https://cran.r-project.org/web/packages/refGenome/index.html) Gene and Splice Site Annotation Using Annotation Data from
'Ensembl' and 'UCSC' Genome Browsers
  * [refund-0.1-37](https://cran.r-project.org/web/packages/refund/index.html) Regression with Functional Data
  * [regioneR-1.38.0](https://www.bioconductor.org/packages/release/bioc/html/regioneR.html) Association analysis of genomic regions based on permutation tests
  * [regress-1.3-21](https://cran.r-project.org/web/packages/regress/index.html) Gaussian Linear Models with Linear Covariance Structure
  * [regtools-1.7.0](https://cran.r-project.org/web/packages/regtools/index.html) Regression and Classification Tools
  * [relations-0.6-15](https://cran.r-project.org/web/packages/relations/index.html) Data Structures and Algorithms for Relations
  * [relevent-1.2-1](https://cran.r-project.org/web/packages/relevent/index.html) Relational Event Models
  * [repr-1.1.7](https://cran.r-project.org/web/packages/repr/index.html) Serializable Representations
  * [restfulr-0.0.15](https://cran.r-project.org/web/packages/restfulr/index.html) R Interface to RESTful Web Services
  * [rgenoud-5.9-0.11](https://cran.r-project.org/web/packages/rgenoud/index.html) R Version of GENetic Optimization Using Derivatives
  * [rhdf5-2.50.2](https://www.bioconductor.org/packages/release/bioc/html/rhdf5.html) R Interface to HDF5
  * [rhdf5filters-1.18.1](https://www.bioconductor.org/packages/release/bioc/html/rhdf5filters.html) HDF5 Compression Filters
  * [rjags-4-16](https://cran.r-project.org/web/packages/rjags/index.html) Bayesian Graphical Models using MCMC
  * [rje-1.12.1](https://cran.r-project.org/web/packages/rje/index.html) Miscellaneous Useful Functions for Statistics
  * [rslurm-0.6.2](https://cran.r-project.org/web/packages/rslurm/index.html) Submit R Calculations to a 'Slurm' Cluster
  * [rsparse-0.5.3](https://cran.r-project.org/web/packages/rsparse/index.html) Statistical Learning on Sparse Matrices
  * [rstanarm-2.32.1](https://cran.r-project.org/web/packages/rstanarm/index.html) Bayesian Applied Regression Modeling via Stan
  * [rsvd-1.0.5](https://cran.r-project.org/web/packages/rsvd/index.html) Randomized Singular Value Decomposition
  * [rtracklayer-1.66.0](https://www.bioconductor.org/packages/release/bioc/html/rtracklayer.html) R interface to genome annotation files and the UCSC genome browser
  * [rvcheck-0.2.1](https://cran.r-project.org/web/packages/rvcheck/index.html) R/Package Version Check
  * [rzmq-0.9.15](https://cran.r-project.org/web/packages/rzmq/index.html) R Bindings for 'ZeroMQ'
  * [samr-3.0](https://cran.r-project.org/web/packages/samr/index.html) SAM: Significance Analysis of Microarrays
  * [scater-1.34.1](https://www.bioconductor.org/packages/release/bioc/html/scater.html) Single-Cell Analysis Toolkit for Gene Expression Data in R
  * [scattermore-1.2](https://cran.r-project.org/web/packages/scattermore/index.html) Scatterplots with More Points
  * [schoolmath-0.4.2](https://cran.r-project.org/web/packages/schoolmath/index.html) Functions and Datasets for Math Used in School
  * [scidb-3.0.0](https://cran.r-project.org/web/packages/scidb/index.html) An R Interface to SciDB
  * [scrime-1.3.5](https://cran.r-project.org/web/packages/scrime/index.html) Analysis of High-Dimensional Categorical Data Such as SNP Data
  * [scuttle-1.16.0](https://www.bioconductor.org/packages/release/bioc/html/scuttle.html) Single-Cell RNA-Seq Analysis Utilities
  * [seqLogo-1.72.0](https://www.bioconductor.org/packages/release/bioc/html/seqLogo.html) Sequence logos for DNA sequence alignments
  * [seqminer-9.7](https://cran.r-project.org/web/packages/seqminer/index.html) Efficiently Read Sequence Data (VCF Format, BCF Format, METAL
Format and BGEN Format) into R
  * [sets-1.0-25](https://cran.r-project.org/web/packages/sets/index.html) Sets, Generalized Sets, Customizable Sets and Intervals
  * [sfd-0.1.0](https://cran.r-project.org/web/packages/sfd/index.html) Space-Filling Design Library
  * [shinyBS-0.61.1](https://cran.r-project.org/web/packages/shinyBS/index.html) Twitter Bootstrap Components for Shiny
  * [shinyFiles-0.9.3](https://cran.r-project.org/web/packages/shinyFiles/index.html) A Server-Side File System Viewer for Shiny
  * [shinyjqui-0.4.1](https://cran.r-project.org/web/packages/shinyjqui/index.html) 'jQuery UI' Interactions and Effects for Shiny
  * [siggenes-1.80.0](https://www.bioconductor.org/packages/release/bioc/html/siggenes.html) Multiple Testing using SAM and Efron's Empirical Bayes Approaches
  * [sitmo-2.0.2](https://cran.r-project.org/web/packages/sitmo/index.html) Parallel Pseudo Random Number Generator (PPRNG) 'sitmo' Header
Files
  * [snpStats-1.56.0](https://www.bioconductor.org/packages/release/bioc/html/snpStats.html) SnpMatrix and XSnpMatrix classes and methods
  * [sodium-1.4.0](https://cran.r-project.org/web/packages/sodium/index.html) A Modern and Easy-to-Use Crypto Library
  * [sparklyr-1.8.6](https://cran.r-project.org/web/packages/sparklyr/index.html) R Interface to Apache Spark
  * [sparseMatrixStats-1.18.0](https://www.bioconductor.org/packages/release/bioc/html/sparseMatrixStats.html) Summary Statistics for Rows and Columns of Sparse Matrices
  * [sparsevctrs-0.3.0](https://cran.r-project.org/web/packages/sparsevctrs/index.html) Sparse Vectors for Use in Data Frames
  * [sqldf-0.4-11](https://cran.r-project.org/web/packages/sqldf/index.html) Manipulate R Data Frames Using SQL
  * [squash-1.0.9](https://cran.r-project.org/web/packages/squash/index.html) Color-Based Plots for Multivariate Visualization
  * [srvyr-1.3.0](https://cran.r-project.org/web/packages/srvyr/index.html) 'dplyr'-Like Syntax for Summary Statistics of Survey Data
  * [ssgraph-1.15](https://cran.r-project.org/web/packages/ssgraph/index.html) Bayesian Graph Structure Learning using Spike-and-Slab Priors
  * [sva-3.54.0](https://www.bioconductor.org/packages/release/bioc/html/sva.html) Surrogate Variable Analysis
  * [systemPipeR-2.12.0](https://www.bioconductor.org/packages/release/bioc/html/systemPipeR.html) systemPipeR: Workflow Environment for Data Analysis and Report Generation
  * [tables-0.9.31](https://cran.r-project.org/web/packages/tables/index.html) Formula-Driven Table Generation
  * [tergmLite-2.6.1](https://cran.r-project.org/web/packages/tergmLite/index.html) Fast Simulation of Simple Temporal Exponential Random Graph
Models
  * [text2vec-0.6.4](https://cran.r-project.org/web/packages/text2vec/index.html) Modern Text Mining Framework for R
  * [tidyposterior-1.0.1](https://cran.r-project.org/web/packages/tidyposterior/index.html) Bayesian Analysis to Compare Models using Resampling Statistics
  * [tilingArray-1.84.0](https://www.bioconductor.org/packages/release/bioc/html/tilingArray.html) Transcript mapping with high-density oligonucleotide tiling arrays
  * [timeSeries-4041.111](https://cran.r-project.org/web/packages/timeSeries/index.html) Financial Time Series Objects (Rmetrics)
  * [tkWidgets-1.84.0](https://www.bioconductor.org/packages/release/bioc/html/tkWidgets.html) R based tk widgets
  * [treeio-1.30.0](https://www.bioconductor.org/packages/release/bioc/html/treeio.html) Base Classes and Functions for Phylogenetic Tree Input and Output
  * [tune-1.3.0](https://cran.r-project.org/web/packages/tune/index.html) Tidy Tuning Tools
  * [txdbmaker-1.2.1](https://www.bioconductor.org/packages/release/bioc/html/txdbmaker.html) Tools for making TxDb objects from genomic annotations
  * [udunits2-0.13.2.1](https://cran.r-project.org/web/packages/udunits2/index.html) Udunits-2 Bindings for R
  * [universalmotif-1.24.2](https://www.bioconductor.org/packages/release/bioc/html/universalmotif.html) Import, Modify, and Export Motifs with R
  * [uwot-0.2.3](https://cran.r-project.org/web/packages/uwot/index.html) The Uniform Manifold Approximation and Projection (UMAP) Method
for Dimensionality Reduction
  * [venn-1.12](https://cran.r-project.org/web/packages/venn/index.html) Draw Venn Diagrams
  * [vimp-2.3.3](https://cran.r-project.org/web/packages/vimp/index.html) Perform Inference on Algorithm-Agnostic Variable Importance
  * [vsn-3.74.0](https://www.bioconductor.org/packages/release/bioc/html/vsn.html) Variance stabilization and calibration for microarray data
  * [wavethresh-4.7.3](https://cran.r-project.org/web/packages/wavethresh/index.html) Wavelets Statistics and Transforms
  * [webp-1.3.0](https://cran.r-project.org/web/packages/webp/index.html) A New Format for Lossless and Lossy Image Compression
  * [widgetTools-1.84.0](https://www.bioconductor.org/packages/release/bioc/html/widgetTools.html) Creates an interactive tcltk widget
  * [workflows-1.2.0](https://cran.r-project.org/web/packages/workflows/index.html) Modeling Workflows
  * [workflowsets-1.1.0](https://cran.r-project.org/web/packages/workflowsets/index.html) Create a Collection of 'tidymodels' Workflows
  * xCell 1.1.0 [CRAN]&emsp;
  * [xmlparsedata-1.0.5](https://cran.r-project.org/web/packages/xmlparsedata/index.html) Parse Data of 'R' Code as an 'XML' Tree
  * [yardstick-1.3.2](https://cran.r-project.org/web/packages/yardstick/index.html) Tidy Characterizations of Model Performance
  * [yulab.utils-0.2.0](https://cran.r-project.org/web/packages/yulab.utils/index.html) Supporting Functions for Packages Maintained by 'YuLab-SMU'
  * [zCompositions-1.5.0-4](https://cran.r-project.org/web/packages/zCompositions/index.html) Treatment of Zeros, Left-Censored and Missing Values in
Compositional Data Sets
  * [zigg-0.0.2](https://cran.r-project.org/web/packages/zigg/index.html) Lightweight Interfaces to the 'Ziggurat' Pseudo Random Number
Generator
  * [zlibbioc-1.52.0](https://www.bioconductor.org/packages/release/bioc/html/zlibbioc.html) An R packaged zlib-1.2.5
