diff -ruN rstudio-1.4.1106.orig/src/gwt/build.xml rstudio-1.4.1106/src/gwt/build.xml
--- rstudio-1.4.1106.orig/src/gwt/build.xml	2021-02-10 15:26:02.000000000 -0800
+++ rstudio-1.4.1106/src/gwt/build.xml	2021-03-25 15:00:30.843147000 -0700
@@ -84,7 +84,7 @@
    <!-- panmirror typescript library -->
    <property name="node.version" value="10.19.0"/>
    <property name="node.dir" value="../../dependencies/common/node/${node.version}"/>
-   <condition property="node.bin" value="../../../${node.dir}/bin/node">
+   <condition property="node.bin" value="${node.dir}/bin/node">
      <not>
         <os family="windows" />
      </not>
diff -ruN rstudio-1.4.1106.orig/src/gwt/CMakeLists.txt rstudio-1.4.1106/src/gwt/CMakeLists.txt
--- rstudio-1.4.1106.orig/src/gwt/CMakeLists.txt	2021-02-10 15:26:02.000000000 -0800
+++ rstudio-1.4.1106/src/gwt/CMakeLists.txt	2021-03-25 15:00:16.998088000 -0700
@@ -50,15 +50,28 @@
 
    # invoke ant to build
    add_custom_target(gwt_build ALL)
-   add_custom_command(
-      TARGET gwt_build
-      WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
-      COMMAND ant -Dbuild.dir="${GWT_BIN_DIR}"
-                  -Dwww.dir="${GWT_WWW_DIR}"
-                  -Dextras.dir="${GWT_EXTRAS_DIR}"
-                  -Dlib.dir="${GWT_LIB_DIR}"
-                  -Dgwt.main.module="org.rstudio.studio.${GWT_MAIN_MODULE}")
+   if(NODEJS_HOME)
+      add_custom_command(
+         TARGET gwt_build
+         WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
+         COMMAND ant -Dbuild.dir="${GWT_BIN_DIR}"
+                     -Dwww.dir="${GWT_WWW_DIR}"
+                     -Dextras.dir="${GWT_EXTRAS_DIR}"
+                     -Dlib.dir="${GWT_LIB_DIR}"
+                     -Dgwt.main.module="org.rstudio.studio.${GWT_MAIN_MODULE}"
+		     -Dnode.dir="${NODEJS_HOME}")
+   else()
+      add_custom_command(
+         TARGET gwt_build
+         WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
+         COMMAND ant -Dbuild.dir="${GWT_BIN_DIR}"
+                     -Dwww.dir="${GWT_WWW_DIR}"
+                     -Dextras.dir="${GWT_EXTRAS_DIR}"
+                     -Dlib.dir="${GWT_LIB_DIR}"
+                     -Dgwt.main.module="org.rstudio.studio.${GWT_MAIN_MODULE}")
+   endif()
 endif()
+
 if(NOT DEFINED GWT_COPY)
    set(GWT_COPY Yes)
 endif()
