diff -ruN llvmlite-0.25.0.orig/ffi/Makefile.linux llvmlite-0.25.0/ffi/Makefile.linux
--- llvmlite-0.25.0.orig/ffi/Makefile.linux	2018-09-21 12:31:30.000000000 -0700
+++ llvmlite-0.25.0/ffi/Makefile.linux	2018-12-23 18:47:16.312756000 -0800
@@ -3,11 +3,11 @@
 
 # -flto and --exclude-libs allow us to remove those parts of LLVM we don't use
 CXX_FLTO_FLAGS ?= -flto
-LD_FLTO_FLAGS ?= -flto -Wl,--exclude-libs=ALL
+LD_FLTO_FLAGS ?=  -Wl,--exclude-libs=ALL
 
-CXXFLAGS = $(LLVM_CXXFLAGS) $(CXX_FLTO_FLAGS)
+CXXFLAGS := $(CXXFLAGS) $(LLVM_CXXFLAGS) 
 LDFLAGS := $(LDFLAGS) $(LLVM_LDFLAGS) $(LD_FLTO_FLAGS)
-LIBS = $(LLVM_LIBS)
+LIBS := $(LIBS) $(LLVM_LIBS)
 INCLUDE = core.h
 SRC = assembly.cpp bitcode.cpp core.cpp initfini.cpp module.cpp value.cpp \
 	  executionengine.cpp transforms.cpp passmanagers.cpp targets.cpp dylib.cpp \
