--- setup.py	2024-07-18 13:42:45
+++ setup.py	2024-07-18 15:22:10
@@ -18,14 +18,14 @@
         "Operating System :: OS Independent",
     ),
     install_requires=[
-        "python-levenshtein>=0.12.0",
+        "Levenshtein>=0.25.1",
         "scipy>=1.1.0",
         "multiprocess>=0.70.6.1",
-        "umi_tools==1.0.0",
-        "pytest==4.1.0",
-        "pytest-dependency==0.4.0",
+        "umi_tools>=1.0.0",
+        "pytest>=4.1.0",
+        "pytest-dependency>=0.4.0",
         "pandas>=0.23.4",
-        "pybktree==1.1",
+        "pybktree>=1.1",
     ],
     python_requires=">=3.6",
 )

