--- run_gistic_example.orig	2017-03-27 13:43:43.000000000 -0700
+++ run_gistic_example	2017-06-08 11:46:47.756847492 -0700
@@ -8,11 +8,11 @@
 
 echo --- running GISTIC ---
 ## input file definitions
-segfile=`pwd`/examplefiles/segmentationfile.txt
-markersfile=`pwd`/examplefiles/markersfile.txt
-refgenefile=`pwd`/refgenefiles/hg16.mat
-alf=`pwd`/examplefiles/arraylistfile.txt
-cnvfile=`pwd`/examplefiles/cnvfile.txt
+segfile=$EBROOTGISTIC/examplefiles/segmentationfile.txt
+markersfile=$EBROOTGISTIC/examplefiles/markersfile.txt
+refgenefile=$EBROOTGISTIC/refgenefiles/hg16.mat
+alf=$EBROOTGISTIC/examplefiles/arraylistfile.txt
+cnvfile=$EBROOTGISTIC/examplefiles/cnvfile.txt
 ## call script that sets MCR environment and calls GISTIC executable 
-./gistic2 -b $basedir -seg $segfile -mk $markersfile -refgene $refgenefile -alf $alf -cnv $cnvfile -genegistic 1 -smallmem 1 -broad 1 -brlen 0.5 -conf 0.90 -armpeel 1 -savegene 1 -gcm extreme
+$EBROOTGISTIC/gistic2 -b $basedir -seg $segfile -mk $markersfile -refgene $refgenefile -alf $alf -cnv $cnvfile -genegistic 1 -smallmem 1 -broad 1 -brlen 0.5 -conf 0.90 -armpeel 1 -savegene 1 -gcm extreme
