---
title: canto Bio Modules 24.04
layout: single
permalink: /sw_inventory/canto-bio-modules-24.04/
created: 2026-02-11
toc: true
toc_label: "On This Page"
sidebar:
  nav: "docs"
---

 - [AGAT/1.4.0-GCC-12.3.0](https://agat.readthedocs.io/en/latest/)
AGAT: Another GTF/GFF Analysis Toolkit. Suite of tools to handle gene annotations  in any GTF/GFF format.
 - [ANTs/2.6.0-foss-2023b](https://stnava.github.io/ANTs/)
ANTs extracts information from complex datasets that include  imaging. ANTs is useful for managing, interpreting and visualizing  multidimensional data.
 - [AlphaPulldown/2.0.3-foss-2023a](https://github.com/KosinskiLab/AlphaPulldown)
AlphaPulldown is a Python package that streamlines protein-protein interaction screens and high-throughput modelling of higher-order oligomers using AlphaFold-Multimer
 - [Arriba/2.4.0-GCC-12.2.0](https://github.com/suhrig/arriba)
Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data.  It was developed for the use in a clinical research setting. Therefore, short runtimes and high  sensitivity were important design criteria.
 - [Arrow/17.0.0-gfbf-2024a](https://arrow.apache.org)
Apache Arrow (incl. PyArrow Python bindings), a cross-language development platform  for in-memory data.
 - [ArviZ/0.16.1-foss-2023a](https://github.com/arviz-devs/arviz)
Exploratory analysis of Bayesian models with Python
 - [BAli-Phy/4.0-beta8-gfbf-2022b](http://www.bali-phy.org/)
BAli-Phy estimates multiple sequence alignments and evolutionary trees from DNA, amino acid,  or codon sequences.
 - [BCFtools/1.19-GCC-13.2.0](https://www.htslib.org/)
Samtools is a suite of programs for interacting with high-throughput sequencing data.  BCFtools
 - [BEDTools/2.31.1-GCC-13.3.0](https://bedtools.readthedocs.io/)
BEDTools: a powerful toolset for genome arithmetic. The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM.
 - [BLAST+/2.16.0-gompi-2024a](https://blast.ncbi.nlm.nih.gov/)
Basic Local Alignment Search Tool, or BLAST, is an algorithm  for comparing primary biological sequence information, such as the amino-acid  sequences of different proteins or the nucleotides of DNA sequences.
 - [BPCells/0.3.0-foss-2024a](https://cole-trapnell-lab.github.io/monocle3)
Single-cell transcriptome sequencing (sc-RNA-seq) experiments  allow us to discover new cell types and help us understand how they arise in  development. The Monocle 3 package provides a toolkit for analyzing  single-cell gene expression experiments.
 - [BWA/0.7.18-GCCcore-13.3.0](https://bio-bwa.sourceforge.net/)
Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively  short nucleotide sequences against a long reference sequence such as the human  genome. 
 - [BamTools/2.5.2-GCC-13.3.0](https://github.com/pezmaster31/bamtools)
BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.
 - [Beast2/2.7.7-GCC-12.3.0-beagle-lib-4.0.1-CUDA-12.1.1](https://beast2.org)
BEAST is a cross-platform program for Bayesian MCMC analysis of molecular   sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using   strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies   but is also a framework for testing evolutionary hypotheses without conditioning on a single   tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted   proportional to its posterior probability. 
 - [BeautifulSoup/4.12.3-GCCcore-13.3.0](https://www.crummy.com/software/BeautifulSoup)
Beautiful Soup is a Python library designed for quick turnaround projects like screen-scraping.
 - [BindCraft/20250520-foss-2023a-CUDA-12.1.1](https://github.com/martinpacesa/BindCraft)
Simple binder design pipeline using AlphaFold2 backpropagation, MPNN, and PyRosetta.  Select your target and let the script do the rest of the work and finish once you have enough designs to order!
 - [Bio-DB-HTS/3.01-GCC-13.3.0](https://metacpan.org/release/Bio-DB-HTS)
Read files using HTSlib including BAM/CRAM, Tabix and BCF database files
 - [BioPerl/1.7.8-GCCcore-12.2.0](http://www.bioperl.org/)
Bioperl is the product of a community effort to produce Perl code which is useful in biology.  Examples include Sequence objects, Alignment objects and database searching objects.
 - [Biopython/1.84-foss-2024a](https://www.biopython.org)
Biopython is a set of freely available tools for biological  computation written in Python by an international team of developers. It is  a distributed collaborative effort to develop Python libraries and  applications which address the needs of current and future work in  bioinformatics. 
 - [Bismark/0.24.1-GCC-12.2.0](https://www.bioinformatics.babraham.ac.uk/projects/bismark/)
A tool to map bisulfite converted sequence reads and determine cytosine methylation states
 - [Bowtie2/2.5.4-GCC-13.2.0](https://github.com/BenLangmead/bowtie2)
Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads  to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s  of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.  Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,  its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.
 - [CITE-seq-Count/1.4.4-foss-2023b-Python-3.11.5](https://github.com/Hoohm/CITE-seq-Count)
A python package that allows to count antibody TAGS from a CITE-seq and/or cell hashing experiment.
 - [CRISPResso2/2.3.1-foss-2023b](https://github.com/pinellolab/CRISPResso2/)
CRISPResso2 is a software pipeline designed to enable rapid and intuitive interpretation of genome editing experiments.
 - [Cassiopeia/2.0.0-foss-2023a](https://github.com/YosefLab/Cassiopeia)
A Package for Cas9-Enabled Single Cell Lineage Tracing Tree Reconstruction.
 - [Cbc/2.10.12-foss-2024a](https://github.com/coin-or/Cbc)
Cbc (Coin-or branch and cut) is an open-source mixed integer linear programming solver written in C++. It can be used as a callable library or using a stand-alone executable.
 - [CellBender/0.3.0-foss-2023a](http://github.com/broadinstitute/CellBender)
CellBender is a software package for eliminating technical artifacts from high-throughput single-cell RNA sequencing (scRNA-seq) data. 
 - [CellRanger/9.0.0](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger)
Cell Ranger is a set of analysis pipelines that process Chromium  single-cell RNA-seq output to align reads, generate gene-cell matrices and perform  clustering and gene expression analysis.
 - [CellRanger-ATAC/2.1.0](https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac)
Cell Ranger ATAC is a set of analysis pipelines that process Chromium Single Cell ATAC data.
 - [CellRank/2.0.2-foss-2023a-CUDA-12.1.1](https://cellrank.readthedocs.io/en/stable/)
CellRank is a toolkit to uncover cellular dynamics based on  Markov state modeling of single-cell data.  It contains two main modules: kernels compute cell-cell transition probabilities and estimators generate hypothesis based on these. 
 - [CellTypist/1.6.2-foss-2023a](https://www.celltypist.org/)
A tool for semi-automatic cell type annotation
 - [Cgl/0.60.8-foss-2024a](https://github.com/coin-or/Cgl)
The COIN-OR Cut Generation Library (Cgl) is a collection of cut generators that can be used with other COIN-OR packages that make use of cuts, such as, among others, the linear solver Clp or the mixed integer linear programming solvers Cbc or BCP. Cgl uses the abstract class OsiSolverInterface (see Osi) to use or communicate with a solver. It does not directly call a solver.
 - [CheckM2/1.1.0-foss-2024a](https://github.com/chklovski/CheckM2/)
Assessing the quality of metagenome-derived genome bins using machine learning
 - [Clp/1.17.10-foss-2024a](https://github.com/coin-or/Clp)
Clp (Coin-or linear programming) is an open-source linear programming solver. It is primarily meant to be used as a callable library, but a basic, stand-alone executable version is also available.
 - [CogentAP/3.0.19-foss-2024a](https://www.takarabio.com/products/next-generation-sequencing/bioinformatics-tools/cogent-ngs-analysis-pipeline)
Cogent NGS Analysis Pipeline (also referred to as CogentAP) is  bioinformatic software for analyzing single-cell RNA-seq, bulk RNA-seq, and single-cell  DNA-seq data.
 - [CoinUtils/2.11.12-GCC-13.3.0](https://github.com/coin-or/CoinUtils)
CoinUtils (Coin-OR Utilities) is an open-source collection of classes and functions that are generally useful to more than one COIN-OR project.
 - [CrossMap/0.7.3-foss-2023b](https://crossmap.readthedocs.io/)
CrossMap is a program for genome coordinates conversion  between different assemblies (such as hg18 (NCBI36) <=> hg19 (GRCh37)).  It supports commonly used file formats including BAM, CRAM, SAM, Wiggle,  BigWig, BED, GFF, GTF and VCF.
 - [DBD-mysql/4.051-GCC-13.3.0](https://metacpan.org/pod/distribution/DBD-mysql/lib/DBD/mysql.pm)
Perl binding for MySQL
 - [DB_File/1.859-GCCcore-12.3.0](https://perldoc.perl.org/DB_File.html)
Perl5 access to Berkeley DB version 1.x.
 - [DIAMOND/2.1.11-GCC-13.3.0](https://github.com/bbuchfink/diamond)
Accelerated BLAST compatible local sequence aligner
 - [DeltaLake/0.15.1-gfbf-2023a](https://delta-io.github.io/delta-rs/)
Native Delta Lake Python binding based on delta-rs with Pandas integration. The Delta Lake project aims to unlock the power of the Deltalake for as many users and projects as possible by providing native low-level APIs aimed at developers and integrators, as well as a high-level operations API that lets you query, inspect, and operate your Delta Lake with ease.
 - [DendroPy/4.6.1-GCCcore-12.3.0](https://dendropy.org/)
A Python library for phylogenetics and phylogenetic computing: reading, writing, simulation, processing and manipulation of phylogenetic trees (phylogenies) and characters.
 - [ESM-2/2.0.0-foss-2023a-CUDA-12.1.1](https://github.com/facebookresearch/esm)
ESM-2 outperforms all tested single-sequence protein language models  across a range of structure prediction tasks. ESMFold harnesses the ESM-2 language model to generate  accurate structure predictions end to end directly from the sequence of a protein.
 - [Eigen/3.4.0-GCCcore-14.2.0](https://eigen.tuxfamily.org)
Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers,  and related algorithms.
 - [FASTA/36.3.8i-GCC-12.2.0](https://fasta.bioch.virginia.edu/fasta_www2/fasta_list2.shtml)
The FASTA programs find regions of local or global (new) similarity between protein or DNA sequences, either by searching Protein or DNA databases, or by identifying local duplications within a sequence.
 - [FLAIR/2.0-foss-2023a](https://github.com/BrooksLabUCSC/flair)
FLAIR (Full-Length Alternative Isoform analysis of RNA) for the correction, isoform definition, and alternative splicing analysis of noisy reads. FLAIR has primarily been used for nanopore cDNA, native RNA, and PacBio sequencing reads.
 - [FastQC/0.12.1-Java-11](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
FastQC is a quality control application for high throughput sequence data. It reads in sequence data in a variety of formats and can either provide an interactive application to review the results of several different QC checks, or create an HTML based report which can be integrated into a pipeline.
 - [FastTree/2.1.11-GCCcore-12.3.0](http://www.microbesonline.org/fasttree/)
FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide  or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of  time and memory. 
 - [Fiona/1.9.5-foss-2023a](https://github.com/Toblerity/Fiona)
Fiona is designed to be simple and dependable. It focuses on reading and writing data in standard Python IO style and relies upon familiar Python types and protocols such as files, dictionaries, mappings, and iterators instead of classes specific to OGR. Fiona can read and write real-world data using multi-layered GIS formats and zipped virtual file systems and integrates readily with other Python GIS packages such as pyproj, Rtree, and Shapely.
 - [Flax/0.8.4-gfbf-2023a-CUDA-12.1.1](https://flax.readthedocs.io)
Flax is a high-performance neural network library and ecosystem for JAX that is designed for flexibility: Try new forms of training by forking an example and by modifying the training loop, not by adding features to a framework.
 - [GATK/4.4.0.0-GCCcore-12.2.0-Java-17](https://www.broadinstitute.org/gatk/)
The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute  to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,  with a primary focus on variant discovery and genotyping as well as strong emphasis on  data quality assurance. Its robust architecture, powerful processing engine and  high-performance computing features make it capable of taking on projects of any size.
 - [GDAL/3.10.0-foss-2024a](https://www.gdal.org)
GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style  Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model  to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for  data translation and processing.
 - [GEOS/3.12.2-GCC-13.3.0](https://trac.osgeo.org/geos)
GEOS (Geometry Engine
 - [GMP/6.3.0-GCCcore-13.3.0](https://gmplib.org/)
GMP is a free library for arbitrary precision arithmetic, operating on signed  integers, rational numbers, and floating point numbers. 
 - [GROMACS/2024.4-foss-2023b-CUDA-12.4.0](https://www.gromacs.org)
GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.  This is a GPU enabled build, containing both MPI and threadMPI binaries.  It also contains the gmxapi extension for the single precision MPI build. 
 - [HDF/4.3.0-GCCcore-13.3.0](https://support.hdfgroup.org/products/hdf4/)
HDF (also known as HDF4) is a library and multi-object file format for  storing and managing data between machines. 
 - [HDF5/1.14.5-gompi-2024a](https://portal.hdfgroup.org/display/support)
HDF5 is a data model, library, and file format for storing and managing data.  It supports an unlimited variety of datatypes, and is designed for flexible  and efficient I/O and for high volume and complex data.
 - [HH-suite/3.3.0-gompi-2023a](https://github.com/soedinglab/hh-suite)
The HH-suite is an open-source software package  for sensitive protein sequence searching based on the pairwise  alignment of hidden Markov models (HMMs).
 - [HMMER/3.4-gompi-2024a](http://hmmer.org/)
HMMER is used for searching sequence databases for homologs  of protein sequences, and for making protein sequence alignments. It  implements methods using probabilistic models called profile hidden Markov  models (profile HMMs).  Compared to BLAST, FASTA, and other sequence  alignment and database search tools based on older scoring methodology,  HMMER aims to be significantly more accurate and more able to detect remote  homologs because of the strength of its underlying mathematical models. In the  past, this strength came at significant computational expense, but in the new  HMMER3 project, HMMER is now essentially as fast as BLAST.
 - [HTSlib/1.21-GCC-13.3.0](https://www.htslib.org/)
A C library for reading/writing high-throughput sequencing data.  This package includes the utilities bgzip and tabix
 - [IgBLAST/1.22.0-x64-linux](https://ncbi.github.io/igblast)
IgBLAST faclilitates the analysis of immunoglobulin and T cell receptor variable domain sequences.
 - [Infernal/1.1.4-foss-2022b](http://eddylab.org/infernal/)
Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases  for RNA structure and sequence similarities.
 - [JAGS/4.3.2-foss-2023b](http://mcmc-jags.sourceforge.net/)
JAGS is Just Another Gibbs Sampler.  It is a program for analysis  of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  
 - [Kalign/3.4.0-GCCcore-12.3.0](https://github.com/TimoLassmann/kalign)
Kalign is a fast multiple sequence alignment program for biological sequences.
 - [Kent_tools/468-GCC-12.3.0](https://genome.cse.ucsc.edu/)
Kent utilities: collection of tools used by the UCSC genome browser.
 - [Kraken2/2.1.3-gompi-2022b](https://github.com/DerrickWood/kraken2/wiki)
Kraken is a system for assigning taxonomic labels to short DNA sequences,  usually obtained through metagenomic studies. Previous attempts by other  bioinformatics software to accomplish this task have often used sequence  alignment or machine learning techniques that were quite slow, leading to  the development of less sensitive but much faster abundance estimation  programs. Kraken aims to achieve high sensitivity and high speed by  utilizing exact alignments of k-mers and a novel classification algorithm.
 - [LAME/3.100-GCCcore-13.2.0](http://lame.sourceforge.net/)
LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL.
 - [Levenshtein/0.25.1-GCCcore-13.2.0](https://pypi.org/project/python-Levenshtein/)
Python extension for computing string edit distances and similarities.
 - [LightGBM/4.6.0-foss-2024a](https://lightgbm.readthedocs.io)
A fast, distributed, high performance gradient boosting (GBT, GBDT, GBRT, GBM or MART) framework based on decision tree algorithms, used for ranking, classification and many other machine learning tasks.
 - [Longshot/1.0.0-GCCcore-13.2.0](https://github.com/pjedge/longshot)
Longshot is a variant calling tool for diploid genomes using long error prone reads such as Pacific  Biosciences (PacBio) SMRT and Oxford Nanopore Technologies (ONT). It takes as input an aligned BAM file and outputs  a phased VCF file with variants and haplotype information. It can also output haplotype-separated BAM files that can  be used for downstream analysis. Currently, it only calls single nucleotide variants (SNVs).
 - [MACS2/2.2.9.1-foss-2022b](https://github.com/taoliu/MACS)
Model Based Analysis for ChIP-Seq data
 - [MACS3/3.0.1-gfbf-2023a](https://macs3-project.github.io/MACS/)
Model Based Analysis for ChIP-Seq data
 - [MAFFT/7.520-GCC-12.3.0-with-extensions](https://mafft.cbrc.jp/alignment/software/source.html)
MAFFT is a multiple sequence alignment program for unix-like operating systems. It offers a range of multiple alignment methods, L-INS-i (accurate; for alignment of <∼200 sequences), FFT-NS-2 (fast; for alignment of <∼30,000 sequences), etc.
 - [MCR/R2023b.9](https://www.mathworks.com/products/compiler/mcr/)
The MATLAB Runtime is a standalone set of shared libraries  that enables the execution of compiled MATLAB applications  or components on computers that do not have MATLAB installed.
 - [METIS/5.1.0-GCCcore-13.3.0](https://karypis.github.io/glaros/projects/gp.html)
METIS is a set of serial programs for partitioning graphs, partitioning  finite element meshes, and producing fill reducing orderings for sparse  matrices. The algorithms implemented in METIS are based on the multilevel  recursive-bisection, multilevel k-way, and multi-constraint partitioning  schemes. 
 - [MPC/1.3.1-GCCcore-12.3.0](http://www.multiprecision.org/)
Gnu Mpc is a C library for the arithmetic of  complex numbers with arbitrarily high precision and correct  rounding of the result. It extends the principles of the IEEE-754  standard for fixed precision real floating point numbers to  complex numbers, providing well-defined semantics for every  operation. At the same time, speed of operation at high precision  is a major design goal.
 - [MPFR/4.2.1-GCCcore-13.2.0](https://www.mpfr.org)
The MPFR library is a C library for multiple-precision floating-point  computations with correct rounding. 
 - [MRtrix/3.0.5-foss-2023b](http://www.brain.org.au/software/index.html#mrtrix)
MRtrix provides a set of tools to perform diffusion-weighted MR  white-matter tractography in a manner robust to crossing fibres, using  constrained spherical deconvolution (CSD) and probabilistic streamlines.
 - [MUMPS/5.7.2-foss-2024a-metis](https://mumps-solver.org)
A parallel sparse direct solver
 - [MUMmer/4.0.0rc1-GCCcore-12.3.0](https://mummer.sourceforge.net/)
MUMmer is a system for rapidly aligning entire genomes,  whether in complete or draft form. AMOS makes use of it. 
 - [MUSCLE/5.1.0-GCCcore-12.3.0](https://drive5.com/muscle/)
MUSCLE is one of the best-performing multiple alignment programs  according to published benchmark tests, with accuracy and speed that are consistently  better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users  learn everything they need to know about MUSCLE in a few minutes-only a handful of  command-line options are needed to perform common alignment tasks.
 - [MariaDB/11.8.3-GCC-14.2.0](https://mariadb.org/)
MariaDB is an enhanced, drop-in replacement for MySQL. Included engines: myISAM, Aria, InnoDB, RocksDB, TokuDB, OQGraph, Mroonga.
 - [Monocle3/1.3.1-foss-2023a-R-4.3.2](https://cole-trapnell-lab.github.io/monocle3/)
An analysis toolkit for single-cell RNA-seq. 
 - [NextPolish/1.4.1-GCC-13.3.02025-01-08](https://github.com/Nextomics/NextPolish)
NextDenovo is a string graph-based de novo assembler for long reads.
 - [OpenMM/8.0.0-foss-2023a](https://openmm.org)
OpenMM is a toolkit for molecular simulation.
 - [Osi/0.108.11-GCC-13.3.0](https://github.com/coin-or/Osi)
Osi (Open Solver Interface) provides an abstract base class to a generic linear programming (LP) solver, along with derived classes for specific solvers. Many applications may be able to use the Osi to insulate themselves from a specific LP solver. That is, programs written to the OSI standard may be linked to any solver with an OSI interface and should produce correct results. The OSI has been significantly extended compared to its first incarnation. Currently, the OSI supports linear programming solvers and has rudimentary support for integer programming.
 - [ParMETIS/4.0.3-gompi-2023a](https://karypis.github.io/glaros/projects/gp.html)
ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning  unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the  functionality provided by METIS and includes routines that are especially suited for parallel AMR computations and  large scale numerical simulations. The algorithms implemented in ParMETIS are based on the parallel multilevel k-way  graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning schemes.
 - [Porechop/0.2.4-GCCcore-12.3.0](https://github.com/rrwick/Porechop)
Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.  Adapters on the ends of reads are trimmed off, and when a read has an adapter in its middle,  it is treated as chimeric and chopped into separate reads. Porechop performs thorough alignments  to effectively find adapters, even at low sequence identity
 - [PostgreSQL/17.5-GCCcore-14.2.0](https://www.postgresql.org/)
PostgreSQL is a powerful, open source object-relational database system.  It is fully ACID compliant, has full support for foreign keys,  joins, views, triggers, and stored procedures (in multiple languages).  It includes most SQL:2008 data types, including INTEGER,  NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP.  It also supports storage of binary large objects, including pictures,  sounds, or video. It has native programming interfaces for C/C++, Java,  .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation.
 - [PyRosetta/4.387-gompi-2023a](https://www.pyrosetta.org/)
PyRosetta is an interactive Python-based interface to the powerful Rosetta molecular modeling suite. It enables users to design their own custom molecular modeling algorithms using Rosetta sampling methods and energy functions. 
 - [PyTables/3.8.0-foss-2023a](https://www.pytables.org)
PyTables is a package for managing hierarchical datasets and designed to efficiently and easily cope  with extremely large amounts of data. PyTables is built on top of the HDF5 library, using the Python language and the  NumPy package. It features an object-oriented interface that, combined with C extensions for the performance-critical  parts of the code (generated using Cython), makes it a fast, yet extremely easy to use tool for interactively browsing,  processing and searching very large amounts of data. One important feature of PyTables is that it optimizes memory and  disk resources so that data takes much less space (specially if on-flight compression is used) than other solutions  such as relational or object oriented databases.
 - [PyTorch/2.3.0-foss-2024a](https://pytorch.org/)
Tensors and Dynamic neural networks in Python with strong GPU acceleration. PyTorch is a deep learning framework that puts Python first.
 - [PyTorch-bundle/2.1.2-foss-2023a-CUDA-12.1.1](https://pytorch.org/)
PyTorch with compatible versions of official Torch extensions.
 - [Pyomo/6.7.3-foss-2023a](https://www.pyomo.org/)
Pyomo is a Python-based open-source software package that supports a diverse set of optimization  capabilities for formulating and analyzing optimization models. 
 - [Pysam/0.22.1-GCC-13.3.0](https://github.com/pysam-developers/pysam)
Pysam is a python module for reading and manipulating Samfiles.  It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.
 - [Qhull/2020.2-GCCcore-12.2.0](http://www.qhull.org)
Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram,  halfspace intersection about a point, furthest-site Delaunay triangulation,  and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and  higher dimensions. Qhull implements the Quickhull algorithm for computing the  convex hull. 
 - [R-Torch/2.7.1-foss-2024a-R-4.4.2](https://cran.r-project.org/web/packages/torch/index.html)
Provides functionality to define and train neural networks similar to 'PyTorch'.  Includes Bioconductor 3.20.
 - [R-bundle-Bioconductor/3.20-foss-2024a-R-4.4.2](https://bioconductor.org)
Bioconductor provides tools for the analysis and coprehension  of high-throughput genomic data.
 - [RAPIDS/24.4-foss-2023a-CUDA-12.1.1](https://rapids.ai/)
RAPIDS provides unmatched speed with familiar APIs that match the most popular PyData libraries. Built on state-of-the-art foundations like NVIDIA CUDA and Apache Arrow, it unlocks the speed of GPUs with code you already know.
 - [RDKit/2025.03.3-foss-2024a](https://www.rdkit.org)
RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python.
 - [Redis/7.2.3-GCCcore-12.3.0](https://redis.io)
Redis is an open source (BSD licensed), in-memory data structure store, used as a database, cache, and message broker. Redis provides data structures such as strings, hashes, lists, sets, sorted sets with range queries, bitmaps, hyperloglogs, geospatial indexes, and streams. Redis has built-in replication, Lua scripting, LRU eviction, transactions, and different levels of on-disk persistence, and provides high availability via Redis Sentinel and automatic partitioning with Redis Cluster.
 - [RocksDB/10.10.1-GCCcore-13.3.0](https://www.rocksdb.org)
RocksDB is an embeddabel persistent key-value storage for fast storage. 
 - [SAMtools/1.21-GCC-13.3.0](https://www.htslib.org/)
SAM Tools provide various utilities for manipulating alignments in the SAM format,  including sorting, merging, indexing and generating alignments in a per-position format.
 - [SCOTCH/7.0.6-gompi-2024a](https://www.labri.fr/perso/pelegrin/scotch/)
Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning.
 - [SCimilarity/0.3.0-foss-2023a](https://github.com/genentech/scimilarity)
SCimilarity is a unifying representation of single cell expression profiles that  quantifies similarity between expression states and generalizes to represent new studies without additional  training.
 - [SRA-Toolkit/3.2.0-gompi-2024a](https://github.com/ncbi/sra-tools)
The SRA Toolkit, and the source-code SRA System Development  Kit (SDK), will allow you to programmatically access data housed within SRA  and convert it from the SRA format
 - [STAR/2.7.11b-GCC-13.3.0](https://github.com/alexdobin/STAR)
STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.
 - [STAR-Fusion/1.15.1-foss-2024a](https://github.com/STAR-Fusion/STAR-Fusion)
STAR-Fusion uses the STAR aligner to identify candidate fusion transcripts  supported by Illumina reads. STAR-Fusion further processes the output generated by the STAR aligner  to map junction reads and spanning reads to a reference annotation set.
 - [SVIM/2.0.0-foss-2024a](https://github.com/eldariont/svim)
SVIM (pronounced swim) is a structural variant caller for third-generation sequencing reads.  It is able to detect and classify the following six classes of structural variation: deletions, insertions,  inversions, tandem duplications, interspersed duplications and translocations.
 - [SVclone/1.1.2-foss-2022b](https://github.com/mcmero/SVclone)
Cluster structural variants of similar cancer cell fraction (CCF).
 - [SYMPHONY/5.7.2-foss-2023b](https://github.com/coin-or/SYMPHONY/wiki)
SYMPHONY is an open-source solver for mixed-integer linear programs (MILPs) written in C.
 - [Salmon/1.10.3-GCC-13.3.0](https://github.com/COMBINE-lab/salmon)
Salmon is a wicked-fast program to produce a highly-accurate,  transcript-level quantification estimate from RNA-seq data.
 - [Sambamba/1.0.1-GCC-13.2.0](https://lomereiter.github.io/sambamba/)
Sambamba is a high performance modern robust and fast tool  (and library), written in the D programming language, for working with SAM  and BAM files. Current functionality is an important subset of samtools  functionality, including view, index, sort, markdup, and depth.
 - [Seaborn/0.13.2-gfbf-2023a](https://seaborn.pydata.org/)
Seaborn is a Python visualization library based on matplotlib.  It provides a high-level interface for drawing attractive statistical graphics. 
 - [SeqKit/2.10.1](https://bioinf.shenwei.me/seqkit/)
SeqKit
 - [Seurat/5.1.0-foss-2023a-R-4.3.2](https://satijalab.org/seurat)
Seurat is an R package designed for QC, analysis, and exploration of single cell RNA-seq data.
 - [ShapeMapper2/2.3-GCC-13.3.0](https://github.com/Weeks-UNC/shapemapper2)
ShapeMapper automates the calculation of RNA chemical probing reactivities  from mutational profiling (MaP) experiments, in which chemical adducts on RNA  are detected as internal mutations in cDNA through reverse transcription and  read out by massively parallel sequencing. 
 - [Shapely/2.0.1-gfbf-2023a](https://github.com/Toblerity/Shapely)
Shapely is a BSD-licensed Python package for manipulation and analysis of planar geometric objects. It is based on the widely deployed GEOS (the engine of PostGIS) and JTS (from which GEOS is ported) libraries.
 - [Sniffles/2.6.2-gfbf-2024a](https://github.com/fritzsedlazeck/Sniffles)
A fast structural variant caller for long-read sequencing,  Sniffles2 accurately detect SVs on germline, somatic and population-level for PacBio and Oxford Nanopore read data.
 - [SoX/14.4.2-GCCcore-12.2.0](http://sox.sourceforge.net/)
Sound eXchange, the Swiss Army knife of audio manipulation
 - [SortMeRNA/4.3.6-GCC-13.3.0](https://bioinfo.lifl.fr/RNA/sortmerna/)
SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking  NGS reads.
 - [Squidpy/1.4.1-foss-2023a](https://squidpy.readthedocs.io)
Squidpy is a tool for the analysis and visualization of spatial molecular data.
 - [TGS-GapCloser/1.2.1-GCCcore-13.3.0](https://github.com/BGI-Qingdao/TGS-GapCloser)
A gap-closing software tool that uses error-prone long reads generated by  third-generation-sequence techniques (Pacbio, Oxford Nanopore, etc.) or preassembled contigs  to fill N-gap in the genome assembly.
 - [TRUST4/1.0.7-GCC-13.3.0](https://github.com/liulab-dfci/TRUST4)
Tcr Receptor Utilities for Solid Tissue (TRUST) is a computational tool to analyze TCR and BCR sequences using unselected RNA sequencing data, profiled from solid tissues, including tumors. TRUST4 performs de novo assembly on V, J, C genes including the hypervariable complementarity-determining region 3 (CDR3) and reports consensus of BCR/TCR sequences. TRUST4 then realigns the contigs to IMGT reference gene sequences to report the corresponding information. TRUST4 supports both single-end and paired-end sequencing data with any read length.
 - [Telescope/1.0.3-20230222-gfbf-2022b](https://github.com/mlbendall/telescope)
Single locus resolution of Transposable ELEment expression using next-generation sequencing.
 - [UMI-tools/1.1.4-foss-2023b](https://umi-tools.readthedocs.io)
Tools for handling Unique Molecular Identifiers in NGS data sets
 - [Uni-Core/0.0.3-foss-2023a](https://github.com/dptech-corp/Uni-Core)
An efficient distributed PyTorch framework
 - [UniFrac/1.4-foss-2023a](https://github.com/biocore/unifrac-binaries)
UniFrac is the de facto repository for high-performance phylogenetic diversity calculations. The methods in this repository are based on an implementation of the Strided State UniFrac algorithm which is faster, and uses less memory than Fast UniFrac. Strided State UniFrac supports Unweighted UniFrac, Weighted UniFrac, Generalized UniFrac, Variance Adjusted UniFrac and meta UniFrac, in both double and single precision (fp32). This repository also includes Stacked Faith (manuscript in preparation), a method for calculating Faith's PD that is faster and uses less memory than the Fast UniFrac-based reference implementation.
 - [VEP/113.3-GCC-13.3.0](https://www.ensembl.org/info/docs/tools/vep)
Variant Effect Predictor (VEP) determines the effect of your  variants (SNPs, insertions, deletions, CNVs or structural variants) on genes,  transcripts, and protein sequence, as well as regulatory regions.  Includes EnsEMBL-XS, which provides pre-compiled replacements for frequently  used routines in VEP.
 - [VSEARCH/2.25.0-GCC-12.3.0](https://github.com/torognes/vsearch)
VSEARCH supports de novo and reference based chimera detection,  clustering, full-length and prefix dereplication, rereplication,  reverse complementation, masking, all-vs-all pairwise global alignment,  exact and global alignment searching, shuffling, subsampling and sorting.  It also supports FASTQ file analysis, filtering,  conversion and merging of paired-end reads.
 - [WhatsHap/2.2-foss-2023a](https://whatshap.readthedocs.io)
WhatsHap is a software for phasing genomic variants using DNA sequencing reads, also called read-based phasing or haplotype assembly. It is especially suitable for long reads, but works also well with short reads.
 - [XML-LibXML/2.0210-GCCcore-13.3.0](https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod)
Perl binding for libxml2
 - [anndata/0.11.3-foss-2024a](https://github.com/scverse/anndata)
anndata is a Python package for handling annotated data matrices in memory and on disk,  positioned between pandas and xarray
 - [bam-readcount/1.0.1-GCC-12.2.0](https://github.com/genome/bam-readcount)
Count DNA sequence reads in BAM files
 - [biom-format/2.1.15-foss-2023a](https://biom-format.org)
The BIOM file format (canonically pronounced biome) is designed to be  a general-use format for representing biological sample by observation  contingency tables. BIOM is a recognized standard for the Earth Microbiome  Project and is a Genomics Standards Consortium supported project. 
 - [bx-python/0.13.0-foss-2023b](https://github.com/bxlab/bx-python)
The bx-python project is a Python library and associated set of scripts to allow for rapid  implementation of genome scale analyses.
 - [castor/1.8.2-foss-2023a](https://CRAN.R-project.org/package=castor)
Efficient phylogenetic analyses on massive phylogenies comprising up to millions of tips. Functions include pruning, rerooting, calculation of most-recent common ancestors, calculating distances from the tree root and calculating pairwise distances. Calculation of phylogenetic signal and mean trait depth (trait conservatism), ancestral state reconstruction and hidden character prediction of discrete characters, simulating and fitting models of trait evolution, fitting and simulating diversification models, dating trees, comparing trees, and reading/writing trees in Newick format. 
 - [cellsnp-lite/1.2.3-GCC-13.2.0](https://cellsnp-lite.readthedocs.io/en/latest)
Cellsnp-lite is a C/C++ tool for efficient genotyping bi-allelic SNPs on single cells. You can use cellsnp-lite after read alignment to obtain the snp x cell pileup UMI or read count matrices for each allele of given or detected SNPs.
 - [cooler/0.10.2-foss-2023a](https://open2c.github.io/cooler)
Cooler is a support library for a storage format, also called cooler, used to store  genomic interaction data of any size, such as Hi-C contact matrices.
 - [cooltools/0.7.1-foss-2024a](https://cooltools.readthedocs.io/en/latest/index.html)
cooltools provides a suite of computational tools with a paired python API and  command line access, which facilitates workflows either on high-performance  computing clusters or via custom analysis notebooks. As part of the Open2C  ecosystem, cooltools also provides detailed introductions to key concepts in  Hi-C-data analysis with interactive notebook documentation.
 - [cutadapt/5.1-GCCcore-13.3.0](https://opensource.scilifelab.se/projects/cutadapt/)
Cutadapt finds and removes adapter sequences, primers, poly-A tails and  other types of unwanted sequence from your high-throughput sequencing reads.
 - [cuteSV/2.1.2-foss-2024a](https://github.com/tjiangHIT/cuteSV)
cuteSV uses tailored methods to collect the signatures of various types of SVs  and employs a clustering-and-refinement method to analyze the signatures to implement sensitive SV detection.
 - [dask/2024.9.1-gfbf-2024a](https://dask.org/)
Dask natively scales Python. Dask provides advanced parallelism for analytics, enabling performance at scale for the tools you love.
 - [dcm2niix/1.0.20241211-GCCcore-13.2.0](https://github.com/rordenlab/dcm2niix)
dcm2niix is designed to convert neuroimaging data from the DICOM format to the NIfTI format.
 - [dill/0.3.9-GCCcore-13.3.0](https://pypi.org/project/dill/)
dill extends python's pickle module for serializing and de-serializing python objects to the majority  of the built-in python types. Serialization is the process of converting an object to a byte stream, and the inverse  of which is converting a byte stream back to on python object hierarchy.
 - [dorado/0.9.1-foss-2023a-CUDA-12.1.1](https://github.com/nanoporetech/dorado)
Dorado is a high-performance, easy-to-use, open source basecaller for Oxford Nanopore reads.
 - [dssp/4.4.10-GCC-13.3.0](https://github.com/PDB-REDO/dssp)
This is a rewrite of DSSP, now offering full mmCIF support. The difference with previous releases of DSSP is that it now writes out an annotated mmCIF file by default, storing the secondary structure information in the _struct_conf category.  Another new feature in this version of DSSP is that it now defines Poly-Proline helices as well.  The DSSP program was designed by Wolfgang Kabsch and Chris Sander to standardize secondary structure assignment. DSSP is a database of secondary structure assignments (and much more) for all protein entries in the Protein Data Bank (PDB). DSSP is also the program that calculates DSSP entries from PDB entries.  DSSP does not predict secondary structure.
 - [e3nn/0.5.0-foss-2022b-PyTorch-1.13.1-CUDA-11.7.0](https://e3nn.org/)
Euclidean neural networks (e3nn) is a python library based on pytorch to create equivariant neural networks for the group O(3). 
 - [easel/0.48-GCC-12.2.0](https://bioeasel.org/)
Easel supports computational analysis of biological sequences using probabilistic models.
 - [edlib/1.3.9.post1-GCC-13.3.0](https://martinsos.github.io/edlib)
Lightweight, super fast library for sequence alignment using edit (Levenshtein) distance.
 - [einops/0.7.0-GCCcore-12.3.0](https://einops.rocks/)
Flexible and powerful tensor operations for readable and reliable code. Supports numpy, pytorch, tensorflow, jax, and others.
 - [epiScanpy/0.4.0-foss-2023a](https://github.com/colomemaria/episcanpy)
EpiScanpy is a toolkit to analyse single-cell open chromatin (scATAC-seq) and single-cell DNA methylation (for example scBS-seq) data. EpiScanpy is the epigenomic extension of the very popular scRNA-seq analysis tool Scanpy (Genome Biology, 2018) [Wolf18].
 - [fastp/0.23.4-GCC-13.2.0](https://github.com/OpenGene/fastp)
A tool designed to provide fast all-in-one preprocessing for FastQ files.  This tool is developed in C++ with multithreading supported to afford high performance.
 - [fhR/4.4.2-foss-2024a-R-4.4.2](http://www.r-project.org/)
R is a free software environment for statistical computing and graphics.
 - [gffread/0.12.7-GCCcore-12.3.0](https://ccb.jhu.edu/software/stringtie/gff.shtml#gffread)
GFF/GTF parsing utility providing format conversions, region filtering, FASTA sequence extraction and more.
 - [gmpy2/2.2.0-GCCcore-13.3.0](https://github.com/aleaxit/gmpy)
GMP/MPIR, MPFR, and MPC interface to Python 2.6+ and 3.x
 - [h5netcdf/1.2.0-foss-2023a](https://h5netcdf.org/)
A Python interface for the netCDF4 file-format that reads and writes local or remote HDF5 files directly via h5py or h5pyd, without relying on the Unidata netCDF library.
 - [h5py/3.12.1-foss-2024a](https://www.h5py.org/)
HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,  version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous  amounts of data.
 - [hermes/0.1.0-foss-2022b-PyTorch-1.13.1-CUDA-11.7.0](https://github.com/StatPhysBio/hermes)
HERMES: Holographic Equivariant neuRal network model for  Mutational Effect and Stability prediction
 - [infercnvpy/0.4.3-foss-2023a](https://github.com/icbi-lab/infercnvpy)
Infer copy number variation (CNV) from scRNA-seq data. Plays nicely with Scanpy.
 - [jax/0.4.25-gfbf-2023a](https://jax.readthedocs.io/)
Composable transformations of Python+NumPy programs: differentiate, vectorize, JIT to GPU/TPU, and more
 - [kallisto/0.50.1-foss-2022b](https://pachterlab.github.io/kallisto/)
kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally  of target sequences using high-throughput sequencing reads.
 - [libcerf/2.3-GCCcore-12.3.0](https://jugit.fz-juelich.de/mlz/libcerf)
libcerf is a self-contained numeric library that provides an efficient and  accurate implementation of complex error functions, along with Dawson,  Faddeeva, and Voigt functions. 
 - [libcifpp/7.0.8-GCC-13.3.0](https://github.com/PDB-REDO/libcifpp)
This library contains code to work with mmCIF and PDB files
 - [libmcfp/1.3.4-GCCcore-12.2.0](https://github.com/mhekkel/libmcfp)
A library for parsing command line arguments and configuration files and making them available throughout a program.
 - [loompy/3.0.8-foss-2024a](https://loompy.org/)
Python implementation of the Loom file format, an efficient file format for large omics datasets
 - [magma/2.7.2-foss-2023a-CUDA-12.1.1](https://icl.cs.utk.edu/magma/)
The MAGMA project aims to develop a dense linear algebra library similar to  LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems.
 - [minimap2/2.29-GCCcore-13.3.0](https://github.com/lh3/minimap2)
Minimap2 is a fast sequence mapping and alignment program that can find overlaps between long noisy reads, or map long reads or their assemblies to a reference genome optionally with detailed alignment (i.e. CIGAR). At present, it works efficiently with query sequences from a few kilobases to ~100 megabases in length at an error rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited test data sets, minimap2 is over 20 times faster than most other long-read aligners. It will replace BWA-MEM for long reads and contig alignment.
 - [mpath/1.1.3-GCCcore-12.2.0](https://pypi.org/project/mpath/)
For now it's quit simple and get_path_info() method returns information about given path. It can be either a directory or a file path.
 - [mpmath/1.3.0-GCCcore-12.3.0](https://mpmath.org/)
mpmath can be used as an arbitrary-precision substitute for Python's float/complex  types and math/cmath modules, but also does much more advanced mathematics. Almost any calculation  can be performed just as well at 10-digit or 1000-digit precision, with either real or complex  numbers, and in many cases mpmath implements efficient algorithms that scale well for extremely  high precision work.
 - [mrcfile/1.4.3-gfbf-2022b](https://github.com/ccpem/mrcfile)
mrcfile is a Python implementation of the MRC2014 file format, which is used in structural biology to store image and volume data.  It allows MRC files to be created and opened easily using a very simple API, which exposes the file’s header and data as numpy arrays. The code runs in Python 2 and 3 and is fully unit-tested.  This library aims to allow users and developers to read and write standard- compliant MRC files in Python as easily as possible, and with no dependencies on any compiled libraries except numpy. You can use it interactively to inspect files, correct headers and so on, or in scripts and larger software packages to provide basic MRC file I/O functions. 
 - [ncbi-vdb/3.2.0-gompi-2024a](https://github.com/ncbi/ncbi-vdb)
The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for  using data in the INSDC Sequence Read Archives.
 - [netCDF/4.9.2-gompi-2023b](https://www.unidata.ucar.edu/software/netcdf/)
NetCDF (network Common Data Form) is a set of software libraries  and machine-independent data formats that support the creation, access, and sharing of array-oriented  scientific data.
 - [netcdf4-python/1.6.4-foss-2023a](https://unidata.github.io/netcdf4-python/)
Python/numpy interface to netCDF.
 - [nf-core/3.3.1-foss-2024a](https://github.com/nf-core/tools)
Python package with helper tools for the nf-core community.
 - [numexpr/2.9.0-foss-2023a](https://numexpr.readthedocs.io/en/latest/)
The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.  It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into  code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a  compiler at runtime.
 - [openpyxl/3.1.2-GCCcore-12.3.0](https://openpyxl.readthedocs.io)
A Python library to read/write Excel 2010 xlsx/xlsm files
 - [parasail/2.6.2-GCC-12.3.0](https://github.com/jeffdaily/parasail)
parasail is a SIMD C (C99) library containing implementations  of the Smith-Waterman (local), Needleman-Wunsch (global), and semi-global  pairwise sequence alignment algorithms. 
 - [polars/0.20.2-gfbf-2023a](https://pola.rs/)
Polars is a blazingly fast DataFrame library for manipulating structured data. The core is written in Rust and this module provides its interface for Python.
 - [prodigal/2.6.3-GCCcore-13.3.0](https://github.com/hyattpd/Prodigal/)
Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)    is a microbial (bacterial and archaeal) gene finding program developed    at Oak Ridge National Laboratory and the University of Tennessee.
 - [pyBigWig/0.3.23-gfbf-2023b](https://github.com/deeptools/pyBigWig)
A python extension, written in C, for quick access to bigBed files and access to and creation of bigWig files.
 - [pyEGA3/5.0.2-GCCcore-12.3.0](https://github.com/EGA-archive/ega-download-client)
A basic Python-based EGA download client 
 - [pyGAM/0.9.1-gfbf-2023a](https://pygam.readthedocs.io)
pyGAM is a package for building Generalized Additive Models in Python, with an emphasis on modularity and performance. The API will be immediately familiar to anyone with experience of scikit-learn or scipy. 
 - [pySCENIC/0.12.1-20240311-foss-2023a](https://github.com/aertslab/pySCENIC)
pySCENIC is a lightning-fast python implementation of the SCENIC pipeline (Single-Cell rEgulatory Network Inference and Clustering) which enables biologists to infer transcription factors, gene regulatory networks and cell types from single-cell RNA-seq data.
 - [pybedtools/0.9.1-foss-2023a](https://daler.github.io/pybedtools)
pybedtools wraps and extends BEDTools and offers feature-level manipulations from within Python.
 - [pyfaidx/0.8.1.2-GCCcore-13.3.0](https://pypi.python.org/pypi/pyfaidx)
pyfaidx: efficient pythonic random access to fasta subsequences
 - [pyproj/3.6.0-GCCcore-12.3.0](https://pyproj4.github.io/pyproj)
Python interface to PROJ4 library for cartographic transformations
 - [python-parasail/1.3.4-foss-2023a](https://github.com/jeffdaily/parasail-python)
Python Bindings for the Parasail C Library
 - [redis-py/5.0.1-GCCcore-12.3.0](https://github.com/redis/redis-py)
The Python interface to the Redis key-value store.
 - [samblaster/0.1.26-GCC-13.2.0](https://github.com/GregoryFaust/samblaster)
samblaster is a fast and flexible program for marking  duplicates in read-id grouped1 paired-end SAM files.
 - [scArches/0.6.1-foss-2023a](https://github.com/theislab/scarches)
Single-cell architecture surgery (scArches) is a package for reference-based analysis of single-cell  data.
 - [scCODA/0.1.9-foss-2023a](https://github.com/theislab/scCODA)
scCODA allows for identification of compositional changes in high-throughput sequencing count data, especially cell compositions from scRNA-seq.
 - [scGPT/0.2.1-foss-2023a-CUDA-12.1.1](https://github.com/bowang-lab/scGPT/)
scGPT: Towards Building a Foundation Model for Single-Cell Multi-omics Using Generative AI.
 - [scVelo/0.3.1-foss-2023a](https://scvelo.org)
scVelo is a scalable toolkit for estimating and analyzing RNA velocities in single cells using  dynamical modeling.
 - [scanpy/1.10.4-foss-2024a](https://scanpy.readthedocs.io/en/stable/)
Scanpy is a scalable toolkit for analyzing single-cell gene expression data built  jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference  and differential expression testing. The Python-based implementation efficiently deals with  datasets of more than one million cells. 
 - [scib/1.1.4-foss-2023a](https://github.com/theislab/scib)
Benchmarking atlas-level data integration in single-cell genomics.
 - [scib-metrics/0.5.1-foss-2023a](https://scib-metrics.readthedocs.io)
Accelerated and Python-only metrics for benchmarking single-cell integration outputs
 - [scikit-bio/0.6.0-foss-2023a](http://scikit-bio.org)
scikit-bio is an open-source, BSD-licensed Python 3 package providing data structures, algorithms and educational resources for bioinformatics.
 - [scikit-learn/1.6.1-gfbf-2024a](https://scikit-learn.org/stable/index.html)
Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world, building upon numpy, scipy, and matplotlib. As a machine-learning module, it provides versatile tools for data mining and analysis in any field of science and engineering. It strives to be simple and efficient, accessible to everybody, and reusable in various contexts.
 - [scrublet/0.2.3-foss-2023a](https://github.com/swolock/scrublet)
Single-Cell Remover of Doublets
 - [scvi-tools/1.1.2-foss-2023a](https://github.com/scverse/scvi-tools)
scvi-tools (single-cell variational inference tools) is a package for probabilistic modeling and analysis of single-cell omics data, built on top of PyTorch and AnnData.
 - [skani/0.2.2-GCCcore-12.3.0](https://github.com/bluenote-1577/skani)
skani
 - [spams/2.6.5.4-foss-2023b](https://thoth.inrialpes.fr/people/mairal/spams/)
SPAMS (SPArse Modeling Software) is an optimization toolbox for solving various  sparse estimation problems.
 - [spektral/1.3.1-foss-2023a-CUDA-12.1.1](https://graphneural.network)
Spektral is a Python library for graph deep learning, based on the Keras API and TensorFlow 2
 - [spoa/4.1.0-GCC-13.3.0](https://github.com/rvaser/spoa)
Spoa (SIMD POA) is a c++ implementation of the partial order alignment (POA) algorithm  which is used to generate consensus sequences
 - [starcode/1.4-GCC-13.2.0](https://github.com/gui11aume/starcode)
Starcode is a DNA sequence clustering software. Starcode clustering is based on all pairs search within a specified Levenshtein distance (allowing insertions and deletions), followed by a clustering algorithm: Message Passing, Spheres or Connected Components.
 - [statsmodels/0.14.4-gfbf-2024a-numpy-2](https://www.statsmodels.org/)
Statsmodels is a Python module that allows users to explore data, estimate statistical models, and perform statistical tests.
 - [sympy/1.13.3-gfbf-2024a](https://sympy.org/)
SymPy is a Python library for symbolic mathematics. It aims to  become a full-featured computer algebra system (CAS) while keeping the code as  simple as possible in order to be comprehensible and easily extensible. SymPy  is written entirely in Python and does not require any external libraries.
 - [unixODBC/2.3.12-GCC-13.3.0](https://www.unixodbc.org)
unixODBC provides a uniform interface between application and database driver
 - [wandb/0.16.1-GCC-12.3.0](https://wandb.ai)
CLI and Python API for Weights and Biases (wandb), a tool for visualizing and tracking your machine learning experiments.
 - [xarray/2023.9.0-gfbf-2023a](https://github.com/pydata/xarray)
xarray (formerly xray) is an open source project and Python package that aims to bring  the labeled data power of pandas to the physical sciences, by providing N-dimensional variants of the  core pandas data structures.
 - [zarr/3.0.10-foss-2025a](https://zarr.readthedocs.io/en/stable/)
Zarr is a Python package providing an implementation of compressed, chunked, N-dimensional arrays, designed for use in parallel computing.
