---
title: chorus Bio Modules 24.04
layout: single
permalink: /sw_inventory/chorus-all-modules-24.04/
created: 2025-11-12
toc: true
toc_label: "On This Page"
sidebar:
  nav: "docs"
---

 - [AGAT/1.4.0-GCC-12.3.0](https://agat.readthedocs.io/en/latest/)
AGAT: Another GTF/GFF Analysis Toolkit. Suite of tools to handle gene annotations
 in any GTF/GFF format.
 - [ANTs/2.6.0-foss-2023b](https://stnava.github.io/ANTs/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/a/ANTs/ANTs-2.6.0-foss-2023b.eb)
ANTs extracts information from complex datasets that include
 imaging. ANTs is useful for managing, interpreting and visualizing
 multidimensional data.
 - [AOCC/4.0.0-GCCcore-12.3.0](https://developer.amd.com/amd-aocc/)
AMD Optimized C/C++ & Fortran compilers (AOCC) based on LLVM 13.0
 - [AOCL-BLAS/5.0-GCC-14.2.0](https://github.com/amd/blis)
AOCL-BLAS is AMD's optimized version of
                 BLAS targeted for AMD EPYC and Ryzen CPUs.
 - [ATK/2.38.0-GCCcore-13.2.0](https://developer.gnome.org/atk/)
ATK provides the set of accessibility interfaces that are implemented by other
 toolkits and applications. Using the ATK interfaces, accessibility tools have
 full access to view and control running applications.

 - [Abseil/20240722.0-GCCcore-13.3.0](https://abseil.io/)
Abseil is an open-source collection of C++ library code designed to augment the
C++ standard library. The Abseil library code is collected from Google's own
C++ code base, has been extensively tested and used in production, and is the
same code we depend on in our daily coding lives.
 - [AlphaFold/2.3.2-foss-2023a-CUDA-12.1.1](https://deepmind.com/research/case-studies/alphafold)
AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known. AlphaFold 2 is deprecated. Please use AllphaFold 3.
 - [AlphaFold3/3.0.0](https://deepmind.com/research/case-studies/alphafold)
AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known. AlphaFold 3 is run via Apptainer on Chorus cluster. AlphaFold 2 id deprecated.
 - [AlphaPulldown/2.0.3-foss-2023a](https://github.com/KosinskiLab/AlphaPulldown)
AlphaPulldown is a Python package that streamlines protein-protein
interaction screens and high-throughput modelling of higher-order oligomers using AlphaFold-Multimer
 - [Archive-Zip/1.68-GCCcore-13.3.0](https://metacpan.org/pod/Archive::Zip)
Provide an interface to ZIP archive files.
 - [Armadillo/14.0.3-foss-2024a](https://arma.sourceforge.net/)
Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards
 a good balance between speed and ease of use. Integer, floating point and complex numbers are supported,
 as well as a subset of trigonometric and statistics functions.
 - [Arriba/2.4.0-GCC-12.2.0](https://github.com/suhrig/arriba)
Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data.
 It was developed for the use in a clinical research setting. Therefore, short runtimes and high
 sensitivity were important design criteria.
 - [Arrow/17.0.0-gfbf-2024a](https://arrow.apache.org)
Apache Arrow (incl. PyArrow Python bindings), a cross-language development platform
 for in-memory data.
 - [ArviZ/0.16.1-foss-2023a](https://github.com/arviz-devs/arviz)
Exploratory analysis of Bayesian models with Python
 - [Autoconf/2.72-GCCcore-13.3.0](https://www.gnu.org/software/autoconf/)
Autoconf is an extensible package of M4 macros that produce shell scripts
 to automatically configure software source code packages. These scripts can
 adapt the packages to many kinds of UNIX-like systems without manual user
 intervention. Autoconf creates a configuration script for a package from a
 template file that lists the operating system features that the package can
 use, in the form of M4 macro calls.

 - [Automake/1.17-GCCcore-14.2.0](https://www.gnu.org/software/automake/automake.html)
Automake: GNU Standards-compliant Makefile generator
 - [Autotools/20240712-GCCcore-14.2.0](https://autotools.io)
This bundle collect the standard GNU build tools: Autoconf, Automake
 and libtool

 - [BAli-Phy/4.0-beta8-gfbf-2022b](http://www.bali-phy.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/BAli-Phy/BAli-Phy-4.0-beta8-gfbf-2022b.eb)
BAli-Phy estimates multiple sequence alignments and evolutionary trees from DNA, amino acid, 
or codon sequences.
 - [BCFtools/1.19-GCC-13.2.0](https://www.htslib.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/BCFtools/BCFtools-1.19-GCC-13.2.0.eb)
Samtools is a suite of programs for interacting with high-throughput sequencing data.
 BCFtools
 - [BEDTools/2.31.1-GCC-13.3.0](https://bedtools.readthedocs.io/)
BEDTools: a powerful toolset for genome arithmetic.
The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and
computing coverage.
The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM.
 - [BLAST+/2.16.0-gompi-2024a](https://blast.ncbi.nlm.nih.gov/)
Basic Local Alignment Search Tool, or BLAST, is an algorithm
 for comparing primary biological sequence information, such as the amino-acid
 sequences of different proteins or the nucleotides of DNA sequences.
 - [BLIS/1.1-GCC-14.2.0](https://github.com/flame/blis/)
BLIS is a portable software framework for instantiating high-performance
BLAS-like dense linear algebra libraries.
 - [BPCells/0.3.0-foss-2024a](https://cole-trapnell-lab.github.io/monocle3)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/BPCells/BPCells-0.3.0-foss-2024a.eb)
Single-cell transcriptome sequencing (sc-RNA-seq) experiments
 allow us to discover new cell types and help us understand how they arise in
 development. The Monocle 3 package provides a toolkit for analyzing
 single-cell gene expression experiments.
 - [BWA/0.7.18-GCCcore-13.3.0](https://bio-bwa.sourceforge.net/)
Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively
 short nucleotide sequences against a long reference sequence such as the human
 genome.

 - [BamTools/2.5.2-GCC-12.3.0](https://github.com/pezmaster31/bamtools)
BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.
 - [Bazel/7.3.2-GCCcore-13.3.0](https://bazel.io/)
Bazel is a build tool that builds code quickly and reliably.
It is used to build the majority of Google's software.
 - [Beast/10.5.0-beta3-GCC-12.3.0-beagle-lib-4.0.1-CUDA-12.1.1](https://beast.community)
BEAST is a cross-platform program for Bayesian analysis of molecular sequences
 using MCMC. It is entirely orientated towards rooted, time-measured phylogenies inferred using
 strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies
 but is also a framework for testing evolutionary hypotheses without conditioning on a single
 tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted
 proportional to its posterior probability. 
 - [Beast2/2.7.7-GCC-12.3.0-beagle-lib-4.0.1-CUDA-12.1.1](https://beast2.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/Beast2/Beast2-2.7.7-GCC-12.3.0-beagle-lib-4.0.1-CUDA-12.1.1.eb)
BEAST is a cross-platform program for Bayesian MCMC analysis of molecular 
 sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using 
 strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies 
 but is also a framework for testing evolutionary hypotheses without conditioning on a single 
 tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted 
 proportional to its posterior probability. 
 - [BeautifulSoup/4.12.3-GCCcore-13.3.0](https://www.crummy.com/software/BeautifulSoup)
Beautiful Soup is a Python library designed for quick turnaround projects like screen-scraping.
 - [BindCraft/20250520-foss-2023a-CUDA-12.1.1](https://github.com/martinpacesa/BindCraft)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/BindCraft/BindCraft-20250520-foss-2023a-CUDA-12.1.1.eb)
Simple binder design pipeline using AlphaFold2 backpropagation, MPNN, and PyRosetta.
 Select your target and let the script do the rest of the work and finish once you have enough designs to order!
 - [Bio-DB-HTS/3.01-GCC-13.3.0](https://metacpan.org/release/Bio-DB-HTS)
Read files using HTSlib including BAM/CRAM, Tabix and BCF database files
 - [BioPerl/1.7.8-GCCcore-12.3.0](https://bioperl.org/)
Bioperl is the product of a community effort to produce Perl code which is useful in biology.
 Examples include Sequence objects, Alignment objects and database searching objects.
 - [Biopython/1.84-foss-2024a](https://www.biopython.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/Biopython/Biopython-1.84-foss-2024a.eb)
Biopython is a set of freely available tools for biological
 computation written in Python by an international team of developers. It is
 a distributed collaborative effort to develop Python libraries and
 applications which address the needs of current and future work in
 bioinformatics. 
 - [Bismark/0.24.1-GCC-12.2.0](https://www.bioinformatics.babraham.ac.uk/projects/bismark/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/Bismark/Bismark-0.24.1-GCC-12.2.0.eb)
A tool to map bisulfite converted sequence reads and determine cytosine methylation states
 - [Bison/3.8.2-GCCcore-12.3.0](https://www.gnu.org/software/bison)
Bison is a general-purpose parser generator that converts an annotated context-free grammar
 into a deterministic LR or generalized LR (GLR) parser employing LALR(1) parser tables.
 - [Blosc/1.21.6-GCCcore-13.3.0](https://www.blosc.org/)
Blosc, an extremely fast, multi-threaded, meta-compressor library
 - [Blosc2/2.17.0-GCCcore-13.3.0](https://www.blosc.org/)
Blosc, an extremely fast, multi-threaded, meta-compressor library
 - [Boost/1.88.0-GCC-14.2.0](https://www.boost.org/)
Boost provides free peer-reviewed portable C++ source libraries.
 - [Boost.Python/1.85.0-GCC-13.3.0](https://boostorg.github.io/python)
Boost.Python is a C++ library which enables seamless interoperability between C++
 and the Python programming language.
 - [Boost.Python-NumPy/1.85.0-gfbf-2024a](https://boostorg.github.io/python)
Boost.Python is a C++ library which enables seamless interoperability between C++
 and the Python programming language.
 - [Bowtie2/2.5.4-GCC-13.2.0](https://github.com/BenLangmead/bowtie2)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/Bowtie2/Bowtie2-2.5.4-GCC-13.2.0.eb)
Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads
 to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s
 of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.
 Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,
 its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.
 - [Brotli/1.1.0-GCCcore-13.2.0](https://github.com/google/brotli)
Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination
 of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio
 comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate
 but offers more dense compression.
The specification of the Brotli Compressed Data Format is defined in RFC 7932.
 - [Brotli-python/1.0.9-GCCcore-12.3.0](https://github.com/google/brotli)
Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination
 of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio
 comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate
 but offers more dense compression.
The specification of the Brotli Compressed Data Format is defined in RFC 7932.
 - [Brunsli/0.1-GCCcore-13.3.0](https://github.com/google/brunsli/)
Brunsli is a lossless JPEG repacking library.
 - [CFITSIO/4.4.1-GCCcore-13.3.0](https://heasarc.gsfc.nasa.gov/fitsio/)
CFITSIO is a library of C and Fortran subroutines for reading and writing data files in
FITS (Flexible Image Transport System) data format.
 - [CGAL/5.6-GCCcore-12.3.0](https://www.cgal.org/)
The goal of the CGAL Open Source Project is to provide easy access to efficient
 and reliable geometric algorithms in the form of a C++ library.
 - [CITE-seq-Count/1.4.4-foss-2023b-Python-3.11.5](https://github.com/Hoohm/CITE-seq-Count)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/CITE-seq-Count/CITE-seq-Count-1.4.4-foss-2023b-Python-3.11.5.eb)
A python package that allows to count antibody TAGS from a CITE-seq and/or cell hashing experiment.
 - [CMake/3.31.3-GCCcore-14.2.0](https://www.cmake.org)
CMake, the cross-platform, open-source build system.  CMake is a family of
 tools designed to build, test and package software.

 - [CRISPResso2/2.3.1-foss-2023b](https://github.com/pinellolab/CRISPResso2/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/CRISPResso2/CRISPResso2-2.3.1-foss-2023b.eb)
CRISPResso2 is a software pipeline designed to enable rapid and
intuitive interpretation of genome editing experiments.
 - [CUDA/12.8.0](https://developer.nvidia.com/cuda-toolkit)
CUDA (formerly Compute Unified Device Architecture) is a parallel
 computing platform and programming model created by NVIDIA and implemented by the
 graphics processing units (GPUs) that they produce. CUDA gives developers access
 to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.
 - [CUDA-Python/12.1.0-gfbf-2023a-CUDA-12.1.1](https://nvidia.github.io/cuda-python/)
Python bindings for CUDA
 - [CUTLASS/3.4.0-foss-2023a-CUDA-12.1.1](https://github.com/NVIDIA/cutlass)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/CUTLASS/CUTLASS-3.4.0-foss-2023a-CUDA-12.1.1.eb)
CUTLASS is a collection of CUDA C++ template
abstractions for implementing high-performance matrix-matrix
multiplication (GEMM) and related computations at all levels and scales
within CUDA. It incorporates strategies for hierarchical decomposition
and data movement similar to those used to implement cuBLAS and cuDNN.
CUTLASS decomposes these "moving parts" into reusable, modular software
components abstracted by C++ template classes. Primitives for different
levels of a conceptual parallelization hierarchy can be specialized and
tuned via custom tiling sizes, data types, and other algorithmic policy.
The resulting flexibility simplifies their use as building blocks within
custom kernels and applications.
 - [Cassiopeia/2.0.0-foss-2023a](https://github.com/YosefLab/Cassiopeia)
A Package for Cas9-Enabled Single Cell Lineage Tracing Tree Reconstruction.
 - [Catch2/3.8.0-GCCcore-12.2.0](https://github.com/catchorg/Catch2)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/Catch2/Catch2-3.8.0-GCCcore-12.2.0.eb)
A modern, C++-native, header-only,
 test framework for unit-tests, TDD and BDD

 - [Cbc/2.10.12-foss-2024a](https://github.com/coin-or/Cbc)
Cbc (Coin-or branch and cut) is an open-source mixed integer linear programming
solver written in C++. It can be used as a callable library or using a
stand-alone executable.
 - [CellBender/0.3.0-foss-2023a-CUDA-12.1.1](http://github.com/broadinstitute/CellBender)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/CellBender/CellBender-0.3.0-foss-2023a-CUDA-12.1.1.eb)
CellBender is a software package for eliminating technical artifacts from
high-throughput single-cell RNA sequencing (scRNA-seq) data.

 - [CellRanger/8.0.0](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/CellRanger/CellRanger-8.0.0.eb)
Cell Ranger is a set of analysis pipelines that process Chromium
 single-cell RNA-seq output to align reads, generate gene-cell matrices and perform
 clustering and gene expression analysis.
 - [CellRanger-ATAC/2.1.0](https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac)
Cell Ranger ATAC is a set of analysis pipelines that process 
Chromium Single Cell ATAC data.
 - [CellRank/2.0.2-foss-2023a-CUDA-12.1.1](https://cellrank.readthedocs.io/en/stable/)
CellRank is a toolkit to uncover cellular dynamics based on
 Markov state modeling of single-cell data.  It contains two main modules:
kernels compute cell-cell transition probabilities and estimators generate
hypothesis based on these. 
 - [CellTypist/1.6.2-foss-2023a](https://www.celltypist.org/)
A tool for semi-automatic cell type annotation
 - [Cgl/0.60.8-foss-2023a](https://github.com/coin-or/Cgl)
The COIN-OR Cut Generation Library (Cgl) is a collection of cut generators that
can be used with other COIN-OR packages that make use of cuts, such as, among
others, the linear solver Clp or the mixed integer linear programming solvers
Cbc or BCP. Cgl uses the abstract class OsiSolverInterface (see Osi) to use or
communicate with a solver. It does not directly call a solver.
 - [CharLS/2.4.2-GCCcore-13.2.0](https://github.com/team-charls/charls)
CharLS is a C++ implementation of the JPEG-LS standard for lossless and near-lossless image
compression and decompression. JPEG-LS is a low-complexity image compression standard that matches JPEG 2000
compression ratios.
 - [CheckM2/1.1.0-foss-2024a](https://github.com/chklovski/CheckM2/)
Assessing the quality of metagenome-derived genome bins using machine learning
 - [Clang/18.1.8-GCCcore-13.3.0](https://clang.llvm.org/)
C, C++, Objective-C compiler, based on LLVM.  Does not
 include C++ standard library -- use libstdc++ from GCC.
 - [Clp/1.17.10-foss-2024a](https://github.com/coin-or/Clp)
Clp (Coin-or linear programming) is an open-source linear programming solver.
It is primarily meant to be used as a callable library, but a basic,
stand-alone executable version is also available.
 - [Cogent_NGS_Immune_Profiler/v2.0-foss-2024a](https://www.takarabio.com/products/next-generation-sequencing/bioinformatics-tools/cogent-ngs-immune-profiler)
Cogent NGS Immune Profiler (CogentIP) is software designed to analyze sequence
 data stored in FASTQ files generated by Illumina sequencers from libraries prepared using certain
 Takara Bio immune profiling kits. 
 - [CoinUtils/2.11.12-GCC-13.3.0](https://github.com/coin-or/CoinUtils)
CoinUtils (Coin-OR Utilities) is an open-source collection of classes and
functions that are generally useful to more than one COIN-OR project.
 - [Compress-Raw-Zlib/2.213-GCCcore-13.3.0](https://metacpan.org/pod/Compress::Raw::Zlib)
Low-Level Interface to zlib or zlib-ng compression library
 - [CrossMap/0.7.3-foss-2023b](https://crossmap.readthedocs.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/CrossMap/CrossMap-0.7.3-foss-2023b.eb)
CrossMap is a program for genome coordinates conversion
 between different assemblies (such as hg18 (NCBI36) <=> hg19 (GRCh37)).
 It supports commonly used file formats including BAM, CRAM, SAM, Wiggle,
 BigWig, BED, GFF, GTF and VCF.
 - [Cython/3.1.1-GCCcore-14.2.0](https://cython.org/)
Cython is an optimising static compiler for both the Python programming
language and the extended Cython programming language (based on Pyrex).

 - [DB/18.1.40-GCCcore-13.3.0](https://www.oracle.com/technetwork/products/berkeleydb)
Berkeley DB enables the development of custom data management
 solutions, without the overhead traditionally associated with such custom
 projects.
 - [DBD-mysql/4.051-GCC-13.3.0](https://metacpan.org/pod/distribution/DBD-mysql/lib/DBD/mysql.pm)
Perl binding for MySQL
 - [DB_File/1.859-GCCcore-12.3.0](https://perldoc.perl.org/DB_File.html)
Perl5 access to Berkeley DB version 1.x.
 - [DBus/1.15.8-GCCcore-13.2.0](https://dbus.freedesktop.org/)
D-Bus is a message bus system, a simple way for applications to talk
 to one another.  In addition to interprocess communication, D-Bus helps
 coordinate process lifecycle; it makes it simple and reliable to code
 a "single instance" application or daemon, and to launch applications
 and daemons on demand when their services are needed.

 - [DIAMOND/2.1.11-GCC-13.3.0](https://github.com/bbuchfink/diamond)
Accelerated BLAST compatible local sequence aligner
 - [DeltaLake/0.15.1-gfbf-2023a](https://delta-io.github.io/delta-rs/)
Native Delta Lake Python binding based on delta-rs with Pandas integration.
The Delta Lake project aims to unlock the power of the Deltalake for as many
users and projects as possible by providing native low-level APIs aimed at
developers and integrators, as well as a high-level operations API that lets
you query, inspect, and operate your Delta Lake with ease.
 - [DendroPy/4.6.1-GCCcore-12.3.0](https://dendropy.org/)
A Python library for phylogenetics and phylogenetic computing: 
reading, writing, simulation, processing and manipulation of phylogenetic trees 
(phylogenies) and characters.
 - [Deprecated/1.2.18-gfbf-2024a](https://github.com/tantale/deprecated)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/d/Deprecated/Deprecated-1.2.18-gfbf-2024a.eb)
If you need to mark a function or a method as deprecated, you can use the @deprecated decorator.
 - [Doxygen/1.14.0-GCCcore-14.2.0](https://www.doxygen.nl)
Doxygen is a documentation system for C++, C, Java, Objective-C, Python,
 IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some
 extent D.

 - [EPA-ng/0.3.8-GCC-12.3.0](https://github.com/pierrebarbera/epa-ng)
EPA-ng
 - [ESM-2/2.0.0-foss-2023a-CUDA-12.1.1](https://github.com/facebookresearch/esm)
ESM-2 outperforms all tested single-sequence protein language models
 across a range of structure prediction tasks. ESMFold harnesses the ESM-2 language model to generate
 accurate structure predictions end to end directly from the sequence of a protein.
 - [ETE/3.1.3-foss-2023a](http://etetoolkit.org)
A Python framework for the analysis and visualization of trees
 - [EasyBuild/5.1.1](https://easybuilders.github.io/easybuild)
EasyBuild is a software build and installation framework
 written in Python that allows you to install software in a structured,
 repeatable and robust way.
 - [Eigen/3.4.0-GCCcore-14.2.0](https://eigen.tuxfamily.org)
Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers,
 and related algorithms.
 - [FASTA/36.3.8i-GCC-12.2.0](https://fasta.bioch.virginia.edu/fasta_www2/fasta_list2.shtml)
The FASTA programs find regions of local or global (new) similarity between 
protein or DNA sequences, either by searching Protein or DNA databases, or by identifying 
local duplications within a sequence.
 - [FFTW/3.3.10-GCC-12.3.0](https://www.fftw.org)
FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)
in one or more dimensions, of arbitrary input size, and of both real and complex data.
 - [FFTW.MPI/3.3.10-gompi-2023a](https://www.fftw.org)
FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)
in one or more dimensions, of arbitrary input size, and of both real and complex data.
 - [FFmpeg/7.0.2-GCCcore-13.3.0](https://www.ffmpeg.org/)
A complete, cross-platform solution to record, convert and stream audio and video.
 - [FLAC/1.4.3-GCCcore-13.3.0](https://xiph.org/flac/)
FLAC stands for Free Lossless Audio Codec, an audio format similar to MP3, but lossless, meaning
that audio is compressed in FLAC without any loss in quality.
 - [FLAIR/2.0-foss-2023a](https://github.com/BrooksLabUCSC/flair)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/f/FLAIR/FLAIR-2.0-foss-2023a.eb)
FLAIR (Full-Length Alternative Isoform analysis of RNA)
for the correction, isoform definition, and alternative splicing analysis of noisy reads.
FLAIR has primarily been used for nanopore cDNA, native RNA, and PacBio sequencing reads.
 - [FSL/6.0.7.17](https://www.fmrib.ox.ac.uk/fsl/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/f/FSL/FSL-6.0.7.17.eb)
FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data.
 - [FastQC/0.12.1-Java-11](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
FastQC is a quality control application for high throughput
sequence data. It reads in sequence data in a variety of formats and can either
provide an interactive application to review the results of several different
QC checks, or create an HTML based report which can be integrated into a
pipeline.
 - [FastTree/2.1.11-GCCcore-12.3.0](http://www.microbesonline.org/fasttree/)
FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide
 or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of
 time and memory. 
 - [Fiona/1.9.5-foss-2023a](https://github.com/Toblerity/Fiona)
Fiona is designed to be simple and dependable. It focuses on reading and writing data
in standard Python IO style and relies upon familiar Python types and protocols such as files, dictionaries,
mappings, and iterators instead of classes specific to OGR. Fiona can read and write real-world data using
multi-layered GIS formats and zipped virtual file systems and integrates readily with other Python GIS
packages such as pyproj, Rtree, and Shapely.
 - [Flax/0.8.4-gfbf-2023a-CUDA-12.1.1](https://flax.readthedocs.io)
Flax is a high-performance neural network library and ecosystem for JAX that is
designed for flexibility: Try new forms of training by forking an example and
by modifying the training loop, not by adding features to a framework.
 - [FlexiBLAS/3.4.5-GCC-14.2.0](https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release)
FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation
used by a program without recompiling or relinking it.
 - [FriBidi/1.0.16-GCCcore-14.2.0](https://github.com/fribidi/fribidi)
The Free Implementation of the Unicode Bidirectional Algorithm.

 - [GATK/4.4.0.0-GCCcore-12.2.0-Java-17](https://www.broadinstitute.org/gatk/)
The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute
 to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,
 with a primary focus on variant discovery and genotyping as well as strong emphasis on
 data quality assurance. Its robust architecture, powerful processing engine and
 high-performance computing features make it capable of taking on projects of any size.
 - [GCC/14.2.0](https://gcc.gnu.org/)
The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
 as well as libraries for these languages (libstdc++, libgcj,...).
 - [GCCcore/14.2.0](https://gcc.gnu.org/)
The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
 as well as libraries for these languages (libstdc++, libgcj,...).
 - [GDAL/3.10.0-foss-2024a](https://www.gdal.org)
GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
 Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
 to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
 data translation and processing.
 - [GDRCopy/2.4.1-GCCcore-13.3.0](https://github.com/NVIDIA/gdrcopy)
A low-latency GPU memory copy library based on NVIDIA GPUDirect RDMA technology.
 - [GEOS/3.12.2-GCC-13.3.0](https://trac.osgeo.org/geos)
GEOS (Geometry Engine
 - [GLPK/5.0-GCCcore-12.3.0](https://www.gnu.org/software/glpk/)
The GLPK (GNU Linear Programming Kit) package is intended for
 solving large-scale linear programming (LP),
 mixed integer programming (MIP), and other related problems.
 It is a set of routines written in ANSI C
 and organized in the form of a callable library.
 - [GLib/2.85.1-GCCcore-14.2.0](https://www.gtk.org/)
GLib is one of the base libraries of the GTK+ project
 - [GMP/6.3.0-GCCcore-13.2.0](https://gmplib.org/)
GMP is a free library for arbitrary precision arithmetic, operating on signed
 integers, rational numbers, and floating point numbers.

 - [GObject-Introspection/1.84.0-GCCcore-14.2.0](https://gi.readthedocs.io/en/latest/)
GObject introspection is a middleware layer between C libraries
 (using GObject) and language bindings. The C library can be scanned at
 compile time and generate a metadata file, in addition to the actual
 native C library. Then at runtime, language bindings can read this
 metadata and automatically provide bindings to call into the C library.
 - [GROMACS/2024.4-foss-2023b](https://www.gromacs.org)
GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the
Newtonian equations of motion for systems with hundreds to millions of
particles.

This is a CPU only build, containing both MPI and threadMPI binaries
for both single and double precision.

It also contains the gmxapi extension for the single precision MPI build.

 - [GSL/2.8-GCC-13.3.0](https://www.gnu.org/software/gsl/)
The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.
 The library provides a wide range of mathematical routines such as random number generators, special functions
 and least-squares fitting.
 - [GST-plugins-bad/1.22.5-GCC-12.3.0](https://gstreamer.freedesktop.org/)
GStreamer is a library for constructing graphs of media-handling
 components. The applications it supports range from simple
 Ogg/Vorbis playback, audio/video streaming to complex audio
 (mixing) and video (non-linear editing) processing.
 - [GST-plugins-base/1.22.5-GCC-12.3.0](https://gstreamer.freedesktop.org/)
GStreamer is a library for constructing graphs of media-handling
 components. The applications it supports range from simple
 Ogg/Vorbis playback, audio/video streaming to complex audio
 (mixing) and video (non-linear editing) processing.
 - [GStreamer/1.24.8-GCC-13.2.0](https://gstreamer.freedesktop.org/)
GStreamer is a library for constructing graphs of media-handling
 components. The applications it supports range from simple
 Ogg/Vorbis playback, audio/video streaming to complex audio
 (mixing) and video (non-linear editing) processing.
 - [GTK3/3.24.39-GCCcore-13.2.0](https://developer.gnome.org/gtk3/stable/)
GTK+ is the primary library used to construct user interfaces in GNOME. It
 provides all the user interface controls, or widgets, used in a common
 graphical application. Its object-oriented API allows you to construct
 user interfaces without dealing with the low-level details of drawing and
 device interaction.

 - [GTK4/4.13.1-GCC-12.3.0](https://docs.gtk.org/gtk4/)
GTK+ is the primary library used to construct user interfaces in GNOME. It
 provides all the user interface controls, or widgets, used in a common
 graphical application. Its object-oriented API allows you to construct
 user interfaces without dealing with the low-level details of drawing and
 device interaction.

 - [GTS/0.7.6-GCCcore-12.3.0](http://gts.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/g/GTS/GTS-0.7.6-GCCcore-12.3.0.eb)
GTS stands for the GNU Triangulated Surface Library.
It is an Open Source Free Software Library intended to provide a set of useful
functions to deal with 3D surfaces meshed with interconnected triangles.
 - [Gdk-Pixbuf/2.42.11-GCCcore-13.3.0](https://docs.gtk.org/gdk-pixbuf/)
The Gdk Pixbuf is a toolkit for image loading and pixel buffer manipulation.
 It is used by GTK+ 2 and GTK+ 3 to load and manipulate images. In the past it
 was distributed as part of GTK+ 2 but it was split off into a separate package
 in preparation for the change to GTK+ 3.

 - [Ghostscript/10.03.1-GCCcore-13.3.0](https://ghostscript.com)
Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
 different targets. It used to be part of the cups printing stack, but is no longer used for that.
 - [GitPython/3.1.43-GCCcore-13.3.0](https://gitpython.readthedocs.org)
GitPython is a python library used to interact with Git repositories 
 - [Go/1.23.6](https://www.golang.org)
Go is an open source programming language that makes it easy to build
 simple, reliable, and efficient software.
 - [Graphene/1.10.8-GCCcore-13.2.0](https://ebassi.github.io/graphene/)
Graphene is a thin layer of types for graphic libraries
 - [Graphviz/12.2.0-GCCcore-13.3.0](https://www.graphviz.org/)
Graphviz is open source graph visualization software. Graph visualization
 is a way of representing structural information as diagrams of
 abstract graphs and networks. It has important applications in networking,
 bioinformatics,  software engineering, database and web design, machine learning,
 and in visual interfaces for other technical domains.
 - [Greenlet/3.1.1-GCCcore-13.3.0](https://github.com/python-greenlet/greenlet)
The greenlet package is a spin-off of Stackless, a version of CPython that
supports micro-threads called "tasklets". Tasklets run pseudo-concurrently (typically in a single
or a few OS-level threads) and are synchronized with data exchanges on "channels".
A "greenlet", on the other hand, is a still more primitive notion of micro-thread with no implicit
scheduling; coroutines, in other words. This is useful when you want to control exactly when your code runs.

 - [HDF/4.3.0-GCCcore-13.3.0](https://support.hdfgroup.org/products/hdf4/)
HDF (also known as HDF4) is a library and multi-object file format for
 storing and managing data between machines.

 - [HDF5/1.14.5-gompi-2024a](https://portal.hdfgroup.org/display/support)
HDF5 is a data model, library, and file format for storing and managing data.
 It supports an unlimited variety of datatypes, and is designed for flexible
 and efficient I/O and for high volume and complex data.
 - [HH-suite/3.3.0-gompi-2023a](https://github.com/soedinglab/hh-suite)
The HH-suite is an open-source software package
 for sensitive protein sequence searching based on the pairwise
 alignment of hidden Markov models (HMMs).
 - [HMMER/3.4-gompi-2023a](http://hmmer.org/)
HMMER is used for searching sequence databases for homologs
 of protein sequences, and for making protein sequence alignments. It
 implements methods using probabilistic models called profile hidden Markov
 models (profile HMMs).  Compared to BLAST, FASTA, and other sequence
 alignment and database search tools based on older scoring methodology,
 HMMER aims to be significantly more accurate and more able to detect remote
 homologs because of the strength of its underlying mathematical models. In the
 past, this strength came at significant computational expense, but in the new
 HMMER3 project, HMMER is now essentially as fast as BLAST.
 - [HTSlib/1.21-GCC-13.3.0](https://www.htslib.org/)
A C library for reading/writing high-throughput sequencing data.
 This package includes the utilities bgzip and tabix
 - [HarfBuzz/11.2.1-GCCcore-14.2.0](https://www.freedesktop.org/wiki/Software/HarfBuzz)
HarfBuzz is an OpenType text shaping engine.
 - [HiGHS/1.7.0-gfbf-2023b](https://ergo-code.github.io/HiGHS)
Open source serial and parallel solvers for large-scale sparse linear programming (LP),
mixed-integer programming (MIP), and quadratic programming (QP) models.
 - [Highway/1.0.7-GCCcore-13.2.0](https://github.com/google/highway)
Highway is a C++ library for SIMD (Single Instruction, Multiple Data), i.e. applying the same
operation to 'lanes'.
 - [Hypre/2.29.0-foss-2023a](https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods)
Hypre is a library for solving large, sparse linear systems of equations on massively
 parallel computers. The problems of interest arise in the simulation codes being developed at LLNL
 and elsewhere to study physical phenomena in the defense, environmental, energy, and biological sciences.
 - [ICU/76.1-GCCcore-14.2.0](https://icu.unicode.org)
ICU is a mature, widely used set of C/C++ and Java libraries providing Unicode and Globalization
 support for software applications.
 - [IPython/8.28.0-GCCcore-13.3.0](https://ipython.org/index.html)
IPython provides a rich architecture for interactive computing with:
 Powerful interactive shells (terminal and Qt-based).
 A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media.
 Support for interactive data visualization and use of GUI toolkits.
 Flexible, embeddable interpreters to load into your own projects.
 Easy to use, high performance tools for parallel computing.
 - [ISA-L/2.31.0-GCCcore-13.3.0](https://github.com/intel/isa-l)
Intelligent Storage Acceleration Library
 - [IgBLAST/1.22.0-x64-linux](https://ncbi.github.io/igblast)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/i/IgBLAST/IgBLAST-1.22.0-x64-linux.eb)
IgBLAST faclilitates the analysis of immunoglobulin and T cell receptor variable domain sequences.
 - [ImageMagick/7.1.1-34-GCCcore-13.2.0](https://www.imagemagick.org/)
ImageMagick is a software suite to create, edit, compose, or convert bitmap images
 - [Imath/3.1.11-GCCcore-13.3.0](https://imath.readthedocs.io/en/latest/)
Imath is a C++ and python library of 2D and 3D vector, matrix, and math operations for computer graphics

 - [InChI/1.07.1-GCC-13.3.0](https://www.inchi-trust.org/)
The IUPAC International Chemical Identifier (InChI TM) is a non-proprietary
identifier for chemical substances that can be used in printed and electronic
data sources thus enabling easier linking of diverse data compilations.
 - [Infernal/1.1.4-foss-2022b](http://eddylab.org/infernal/)
Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases
 for RNA structure and sequence similarities.
 - [JAGS/4.3.2-foss-2024a](http://mcmc-jags.sourceforge.net/)
JAGS is Just Another Gibbs Sampler.  It is a program for analysis
 of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  
 - [JasPer/4.2.4-GCCcore-13.3.0](https://www.ece.uvic.ca/~frodo/jasper/)
The JasPer Project is an open-source initiative to provide a free
 software-based reference implementation of the codec specified in
 the JPEG-2000 Part-1 standard.

 - [Java/23.0.2](https://openjdk.org)
Java Platform, Standard Edition (Java SE) lets you develop and deploy
Java applications on desktops and servers.
 - [JsonCpp/1.9.5-GCCcore-13.3.0](https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html)
JsonCpp is a C++ library that allows manipulating JSON values,
 including serialization and deserialization to and from strings. It can also preserve existing comment in
 unserialization/serialization steps, making it a convenient format to store user input files. 
 - [Judy/1.0.5-GCCcore-12.3.0](http://judy.sourceforge.net/)
A C library that implements a dynamic array.
 - [Jupyter-bundle/20230823-GCCcore-12.3.0](https://jupyter.org/)
This bundle collects a range of Jupyter interfaces (Lab, Notebook and nbclassic),
  extensions (Jupyter Server Proxy, Jupyter Resource Monitor, Jupyter Lmod) and
  the JupyterHub.

 - [JupyterHub/4.0.2-GCCcore-12.3.0](https://jupyter.org)
JupyterHub is a multiuser version of the Jupyter (IPython) notebook designed
 for centralized deployments in companies, university classrooms and research labs.
 - [JupyterLab/4.2.5-GCCcore-13.3.0](https://jupyter.org/)
JupyterLab is the next-generation user interface for Project Jupyter offering all the familiar
 building blocks of the classic Jupyter Notebook (notebook, terminal, text editor, file browser, rich outputs,
 etc.) in a flexible and powerful user interface. JupyterLab will eventually replace the classic Jupyter
 Notebook.
 - [JupyterNotebook/7.0.2-GCCcore-12.3.0](https://jupyter.org/)
The Jupyter Notebook is the original web application for creating and
 sharing computational documents. It offers a simple, streamlined, document-centric experience.
 - [Kaleido/0.2.1-GCCcore-12.3.0](https://github.com/plotly/Kaleido)
Fast static image export for web-based visualization libraries with zero dependencies
 - [Kalign/3.4.0-GCCcore-12.3.0](https://github.com/TimoLassmann/kalign)
Kalign is a fast multiple sequence alignment program for biological sequences.
 - [Kent_tools/479-GCC-13.3.0](https://genome.cse.ucsc.edu/)
Kent utilities: collection of tools used by the UCSC genome browser.
 - [Kraken2/2.1.3-gompi-2022b](https://github.com/DerrickWood/kraken2/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/k/Kraken2/Kraken2-2.1.3-gompi-2022b.eb)
Kraken is a system for assigning taxonomic labels to short DNA sequences,
 usually obtained through metagenomic studies. Previous attempts by other
 bioinformatics software to accomplish this task have often used sequence
 alignment or machine learning techniques that were quite slow, leading to
 the development of less sensitive but much faster abundance estimation
 programs. Kraken aims to achieve high sensitivity and high speed by
 utilizing exact alignments of k-mers and a novel classification algorithm.
 - [LAME/3.100-GCCcore-12.3.0](http://lame.sourceforge.net/)
LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL.
 - [LAPACK/3.12.0-GCC-13.2.0](https://www.netlib.org/lapack/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/l/LAPACK/LAPACK-3.12.0-GCC-13.2.0.eb)
LAPACK is written in Fortran90 and provides routines for solving systems of
 simultaneous linear equations, least-squares solutions of linear systems of equations, eigenvalue
 problems, and singular value problems.
 - [LDC/1.39.0-GCCcore-13.2.0](https://wiki.dlang.org/LDC)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/l/LDC/LDC-1.39.0-GCCcore-13.2.0.eb)
The LLVM-based D Compiler
 - [LERC/4.0.0-GCCcore-13.2.0](https://github.com/Esri/lerc)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/l/LERC/LERC-4.0.0-GCCcore-13.2.0.eb)
LERC is an open-source image or raster format which supports rapid encoding and decoding
for any pixel type (not just RGB or Byte). Users set the maximum compression error per pixel while encoding,
so the precision of the original input image is preserved (within user defined error bounds).
 - [LLVM/20.1.7-GCCcore-14.2.0](https://llvm.org/)
The LLVM Core libraries provide a modern source- and target-independent
 optimizer, along with code generation support for many popular CPUs
 (as well as some less common ones!) These libraries are built around a well
 specified code representation known as the LLVM intermediate representation
 ("LLVM IR"). The LLVM Core libraries are well documented, and it is
 particularly easy to invent your own language (or port an existing compiler)
 to use LLVM as an optimizer and code generator.
 - [LMDB/0.9.31-GCCcore-13.3.0](https://symas.com/lmdb)
LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance
 of a pure in-memory database while retaining the persistence of standard disk-based databases.
 - [LZO/2.10-GCCcore-13.2.0](https://www.oberhumer.com/opensource/lzo/)
Portable lossless data compression library
 - [Levenshtein/0.25.1-GCCcore-13.2.0](https://pypi.org/project/python-Levenshtein/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/l/Levenshtein/Levenshtein-0.25.1-GCCcore-13.2.0.eb)
Python extension for computing string edit distances and similarities.
 - [LibLZF/3.6-GCCcore-12.3.0](http://oldhome.schmorp.de/marc/liblzf.html)
LibLZF is a very small data compression library. It consists of only two .c and two .h files
and is very easy to incorporate into your own programs. The compression algorithm is very, very fast, yet still
written in portable C.
 - [LibTIFF/4.7.0-GCCcore-14.2.0](https://libtiff.gitlab.io/libtiff/)
tiff: Library and tools for reading and writing TIFF data files
 - [LightGBM/4.6.0-foss-2024a](https://lightgbm.readthedocs.io)
A fast, distributed, high performance gradient boosting (GBT, GBDT, GBRT, GBM
or MART) framework based on decision tree algorithms, used for ranking,
classification and many other machine learning tasks.
 - [LittleCMS/2.16-GCCcore-13.3.0](https://www.littlecms.com/)
Little CMS intends to be an OPEN SOURCE small-footprint color management engine,
 with special focus on accuracy and performance. 
 - [Longshot/1.0.0-GCCcore-13.2.0](https://github.com/pjedge/longshot)
Longshot is a variant calling tool for diploid genomes using long error prone reads such as Pacific
 Biosciences (PacBio) SMRT and Oxford Nanopore Technologies (ONT). It takes as input an aligned BAM file and outputs
 a phased VCF file with variants and haplotype information. It can also output haplotype-separated BAM files that can
 be used for downstream analysis. Currently, it only calls single nucleotide variants (SNVs).
 - [Lua/5.4.6-GCCcore-12.3.0](https://www.lua.org/)
Lua is a powerful, fast, lightweight, embeddable scripting language.
 Lua combines simple procedural syntax with powerful data description constructs based
 on associative arrays and extensible semantics. Lua is dynamically typed,
 runs by interpreting bytecode for a register-based virtual machine,
 and has automatic memory management with incremental garbage collection,
 making it ideal for configuration, scripting, and rapid prototyping.
 - [M4/1.4.19-GCCcore-14.2.0](https://www.gnu.org/software/m4/m4.html)
GNU M4 is an implementation of the traditional Unix macro processor. It is mostly SVR4 compatible
  although it has some extensions (for example, handling more than 9 positional parameters to macros).
 GNU M4 also has built-in functions for including files, running shell commands, doing arithmetic, etc.
 - [MACS2/2.2.9.1-foss-2022b](https://github.com/taoliu/MACS)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/MACS2/MACS2-2.2.9.1-foss-2022b.eb)
Model Based Analysis for ChIP-Seq data
 - [MACS3/3.0.1-gfbf-2023a](https://macs3-project.github.io/MACS/)
Model Based Analysis for ChIP-Seq data
 - [MAFFT/7.520-GCC-12.3.0-with-extensions](https://mafft.cbrc.jp/alignment/software/source.html)
MAFFT is a multiple sequence alignment program for unix-like operating systems.
It offers a range of multiple alignment methods, L-INS-i (accurate; for alignment
of <∼200 sequences), FFT-NS-2 (fast; for alignment of <∼30,000 sequences), etc.
 - [MATIO/1.5.26-GCCcore-13.2.0](https://sourceforge.net/projects/matio/)
matio is an C library for reading and writing Matlab MAT files.
 - [MBROLA/3.3-GCCcore-12.3.0-voices-20200330](['https://github.com/numediart/MBROLA', 'https://github.com/numediart/MBROLA-voices'])
MBROLA is a speech synthesizer based on the concatenation of diphones.
It takes a list of phonemes as input, together with prosodic information
(duration of phonemes and a piecewise linear description of pitch),
and produces speech samples on 16 bits (linear),
at the sampling frequency of the diphone database.

MBROLA voices project provides list of MBROLA speech synthesizer voices.
It is intended to provide easier collaboration and
automatic updates for individual users and packagers.

 - [MCR/R2023b.9](https://www.mathworks.com/products/compiler/mcr/)
The MATLAB Runtime is a standalone set of shared libraries
 that enables the execution of compiled MATLAB applications
 or components on computers that do not have MATLAB installed.
 - [METIS/5.1.0-GCCcore-13.2.0](https://karypis.github.io/glaros/projects/gp.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/METIS/METIS-5.1.0-GCCcore-13.2.0.eb)
METIS is a set of serial programs for partitioning graphs, partitioning
 finite element meshes, and producing fill reducing orderings for sparse
 matrices. The algorithms implemented in METIS are based on the multilevel
 recursive-bisection, multilevel k-way, and multi-constraint partitioning
 schemes.

 - [MPC/1.3.1-GCCcore-13.2.0](http://www.multiprecision.org/)
Gnu Mpc is a C library for the arithmetic of
 complex numbers with arbitrarily high precision and correct
 rounding of the result. It extends the principles of the IEEE-754
 standard for fixed precision real floating point numbers to
 complex numbers, providing well-defined semantics for every
 operation. At the same time, speed of operation at high precision
 is a major design goal.
 - [MPFR/4.2.1-GCCcore-13.3.0](https://www.mpfr.org)
The MPFR library is a C library for multiple-precision floating-point
 computations with correct rounding.

 - [MRtrix/3.0.5-foss-2023b](http://www.brain.org.au/software/index.html#mrtrix)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/MRtrix/MRtrix-3.0.5-foss-2023b.eb)
MRtrix provides a set of tools to perform diffusion-weighted MR
 white-matter tractography in a manner robust to crossing fibres, using
 constrained spherical deconvolution (CSD) and probabilistic streamlines.
 - [MUMPS/5.7.2-foss-2024a-metis](https://mumps-solver.org)
A parallel sparse direct solver
 - [MUMmer/4.0.0rc1-GCCcore-12.3.0](https://mummer.sourceforge.net/)
MUMmer is a system for rapidly aligning entire genomes,
 whether in complete or draft form. AMOS makes use of it.

 - [MUSCLE/5.1.0-GCCcore-12.3.0](https://drive5.com/muscle/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/MUSCLE/MUSCLE-5.1.0-GCCcore-12.3.0.eb)
MUSCLE is one of the best-performing multiple alignment programs
 according to published benchmark tests, with accuracy and speed that are consistently
 better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users
 learn everything they need to know about MUSCLE in a few minutes-only a handful of
 command-line options are needed to perform common alignment tasks.
 - [Mako/1.3.10-GCCcore-14.2.0](https://www.makotemplates.org)
A super-fast templating language that borrows the best ideas from the existing templating languages
 - [Mamba/23.11.0-0](https://mamba.readthedocs.io/)
Mamba is a fast, robust, and cross-platform package manager. It runs on Windows, OS X and Linux
(ARM64 and PPC64LE included) and is fully compatible with conda packages and supports most of conda's commands.

 - [MariaDB/11.7.0-GCC-13.3.0](https://mariadb.org/)
MariaDB is an enhanced, drop-in replacement for MySQL.
Included engines: myISAM, Aria, InnoDB, RocksDB, TokuDB, OQGraph, Mroonga.
 - [Markdown/3.7-GCCcore-13.3.0](https://python-markdown.github.io/)
This is a Python implementation of John Gruber's Markdown.
It is almost completely compliant with the reference implementation, though there are a few known issues.
Additional features are supported by the Available Extensions.

 - [Mesa/25.1.3-GCCcore-14.2.0](https://www.mesa3d.org/)
Mesa is an open-source implementation of the OpenGL specification -
 a system for rendering interactive 3D graphics.
 - [Meson/1.6.1-GCCcore-14.2.0](https://mesonbuild.com)
Meson is a cross-platform build system designed to be both as fast and as user friendly as possible.
 - [Miniforge3/24.1.2-0](https://github.com/conda-forge/miniforge)
Miniforge is a free minimal installer for conda and Mamba specific
 to conda-forge.
 - [Monocle3/1.4.26-foss-2024a](https://cole-trapnell-lab.github.io/monocle3)
Single-cell transcriptome sequencing (sc-RNA-seq) experiments allow us to discover new cell types and help us understand how they arise in development. The Monocle 3 package provides a toolkit for analyzing single-cell gene expression experiments.
 - [MultiQC/1.21-foss-2023a](https://multiqc.info)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/MultiQC/MultiQC-1.21-foss-2023a.eb)
Aggregate results from bioinformatics analyses across many samples into a single 
 report.

 MultiQC searches a given directory for analysis logs and compiles a HTML report. It's a general
 use tool, perfect for summarising the output from numerous bioinformatics tools.
 - [NASM/2.16.03-GCCcore-13.3.0](https://www.nasm.us/)
NASM: General-purpose x86 assembler
 - [NCCL/2.23.4-GCCcore-13.3.0-CUDA-12.6.2](https://developer.nvidia.com/nccl)
The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective
communication primitives that are performance optimized for NVIDIA GPUs.
 - [NLTK/3.8.1-foss-2023a](https://www.nltk.org/)
NLTK is a leading platform for building Python programs to work with human language data.
 - [NLopt/2.7.1-GCCcore-12.3.0](http://ab-initio.mit.edu/wiki/index.php/NLopt)
NLopt is a free/open-source library for nonlinear optimization,
 providing a common interface for a number of different free optimization routines
 available online as well as original implementations of various other algorithms. 
 - [NSPR/4.35-GCCcore-13.2.0](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR)
Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level
 and libc-like functions.
 - [NSS/3.104-GCCcore-13.3.0](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS)
Network Security Services (NSS) is a set of libraries designed to support cross-platform development
 of security-enabled client and server applications.
 - [NextPolish/1.4.1-GCC-13.3.0-2025-01-08](https://github.com/Nextomics/NextPolish)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/n/NextPolish/NextPolish-1.4.1-GCC-13.3.0-2025-01-08.eb)
NextDenovo is a string graph-based de novo assembler for long reads.
 - [Nextflow/25.04.2](https://www.nextflow.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/n/Nextflow/Nextflow-25.04.2.eb)
Nextflow is a reactive workflow framework and a programming DSL
 that eases writing computational pipelines with complex data
 - [Ninja/1.12.1-GCCcore-13.3.0](https://ninja-build.org/)
Ninja is a small build system with a focus on speed.
 - [OpenBLAS/0.3.29-GCC-14.2.0](https://www.openblas.net/)
OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version.
 - [OpenCV/4.8.1-foss-2023a-contrib](https://opencv.org/)
OpenCV (Open Source Computer Vision Library) is an open source computer vision
 and machine learning software library. OpenCV was built to provide
 a common infrastructure for computer vision applications and to accelerate
 the use of machine perception in the commercial products.
 Includes extra modules for OpenCV from the contrib repository.
 - [OpenEXR/3.2.4-GCCcore-13.3.0](https://www.openexr.com/)
OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light & Magic
 for use in computer imaging applications
 - [OpenJPEG/2.5.2-GCCcore-13.3.0](https://www.openjpeg.org/)
OpenJPEG is an open-source JPEG 2000 codec written in
 C language. It has been developed in order to promote the use of JPEG 2000,
 a still-image compression standard from the Joint Photographic Experts Group
 (JPEG). Since may 2015, it is officially recognized by ISO/IEC and ITU-T as
 a JPEG 2000 Reference Software.
 - [OpenMM/8.0.0-foss-2023a-CUDA-12.1.1](https://openmm.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/o/OpenMM/OpenMM-8.0.0-foss-2023a-CUDA-12.1.1.eb)
OpenMM is a toolkit for molecular simulation.
 - [OpenMPI/5.0.7-GCC-14.2.0](https://www.open-mpi.org/)
The Open MPI Project is an open source MPI-3 implementation.
 - [OpenPGM/5.2.122-GCCcore-13.3.0](https://code.google.com/p/openpgm/)
OpenPGM is an open source implementation of the Pragmatic General Multicast
 (PGM) specification in RFC 3208 available at www.ietf.org. PGM is a reliable
 and scalable multicast protocol that enables receivers to detect loss, request
 retransmission of lost data, or notify an application of unrecoverable loss.
 PGM is a receiver-reliable protocol, which means the receiver is responsible
 for ensuring all data is received, absolving the sender of reception
 responsibility.

 - [OpenSSL/3](https://www.openssl.org/)
The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured,
 and Open Source toolchain implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1) 
 protocols as well as a full-strength general purpose cryptography library. 
 - [OpenSlide/4.0.0-GCCcore-13.2.0](https://openslide.org/)
OpenSlide is a C library that provides a simple interface to
read whole-slide images (also known as virtual slides).
 - [Optax/0.2.2-gfbf-2023a](https://github.com/deepmind/optax)
Optax is a gradient processing and optimization library for JAX.
 - [Osi/0.108.11-GCC-13.3.0](https://github.com/coin-or/Osi)
Osi (Open Solver Interface) provides an abstract base class to a generic linear
programming (LP) solver, along with derived classes for specific solvers. Many
applications may be able to use the Osi to insulate themselves from a specific
LP solver. That is, programs written to the OSI standard may be linked to any
solver with an OSI interface and should produce correct results. The OSI has
been significantly extended compared to its first incarnation. Currently, the
OSI supports linear programming solvers and has rudimentary support for integer
programming.
 - [PCRE/8.45-GCCcore-13.2.0](https://www.pcre.org/)
The PCRE library is a set of functions that implement regular expression
 pattern matching using the same syntax and semantics as Perl 5.

 - [PCRE2/10.45-GCCcore-14.2.0](https://www.pcre.org/)
The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax
 and semantics as Perl 5.

 - [PDM/2.18.2-GCCcore-13.3.0](https://pdm-project.org)
A modern Python package and dependency manager supporting the latest PEP standards.
 - [PETSc/3.20.3-foss-2023a](https://www.mcs.anl.gov/petsc)
PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the
 scalable (parallel) solution of scientific applications modeled by partial differential equations.
 - [PICRUSt2/2.5.2-foss-2023a](https://github.com/picrust/picrust2)
PICRUSt2 (Phylogenetic Investigation of Communities by
 Reconstruction of Unobserved States) is a software for predicting
 functional abundances based only on marker gene sequences.
 - [PLY/3.11-GCCcore-12.3.0](https://www.dabeaz.com/ply/)
PLY is yet another implementation of lex and yacc for Python.
 - [PMIx/5.0.6-GCCcore-14.2.0](https://pmix.org/)
Process Management for Exascale Environments
PMI Exascale (PMIx) represents an attempt to
provide an extended version of the PMI standard specifically designed
to support clusters up to and including exascale sizes. The overall
objective of the project is not to branch the existing pseudo-standard
definitions
 - [PROJ/9.4.1-GCCcore-13.3.0](https://proj.org)
Program proj is a standard Unix filter function which converts
geographic longitude and latitude coordinates into cartesian coordinates
 - [PRRTE/3.0.8-GCCcore-14.2.0](https://docs.prrte.org/)
PRRTE is the PMIx Reference RunTime Environment
 - [Pandoc/3.1.2](https://pandoc.org)
If you need to convert files from one markup format into another, pandoc is your swiss-army knife
 - [Pango/1.54.0-GCCcore-13.3.0](https://pango.gnome.org/)
Pango is a library for laying out and rendering of text, with an emphasis on internationalization.
Pango can be used anywhere that text layout is needed, though most of the work on Pango so far has been done in the
context of the GTK+ widget toolkit. Pango forms the core of text and font handling for GTK+-2.x.
 - [ParMETIS/4.0.3-gompi-2023a](https://karypis.github.io/glaros/projects/gp.html)
ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning
 unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the
 functionality provided by METIS and includes routines that are especially suited for parallel AMR computations and
 large scale numerical simulations. The algorithms implemented in ParMETIS are based on the parallel multilevel k-way
 graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning schemes.
 - [Parsl/2024.4.22-GCCcore-12.3.0](https://parsl-project.org/)
Parsl extends parallelism in Python beyond a single computer.
You can use Parsl just like Python's parallel executors but across multiple
cores and nodes. However, the real power of Parsl is in expressing multi-step
workflows of functions. Parsl lets you chain functions together and will launch
each function as inputs and computing resources are available.

 - [Perl/5.40.0-GCCcore-14.2.0](https://www.perl.org/)
Larry Wall's Practical Extraction and Report Language

Includes a small selection of extra CPAN packages for core functionality.

 - [Perl-bundle-CPAN/5.40.0-GCCcore-14.2.0](https://www.perl.org/)
A set of common packages from CPAN
 - [Pillow/11.0.0-GCCcore-13.3.0](https://pillow.readthedocs.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/p/Pillow/Pillow-11.0.0-GCCcore-13.3.0.eb)
Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
 PIL is the Python Imaging Library by Fredrik Lundh and Contributors.
 - [Pillow-SIMD/10.4.0-GCCcore-13.3.0](https://github.com/uploadcare/pillow-simd)
Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
 PIL is the Python Imaging Library by Fredrik Lundh and Contributors.
 - [Porechop/0.2.4-GCCcore-12.3.0](https://github.com/rrwick/Porechop)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/p/Porechop/Porechop-0.2.4-GCCcore-12.3.0.eb)
Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.
 Adapters on the ends of reads are trimmed off, and when a read has an adapter in its middle,
 it is treated as chimeric and chopped into separate reads. Porechop performs thorough alignments
 to effectively find adapters, even at low sequence identity
 - [PostgreSQL/16.4-GCCcore-13.3.0](https://www.postgresql.org/)
PostgreSQL is a powerful, open source object-relational database system.
 It is fully ACID compliant, has full support for foreign keys,
 joins, views, triggers, and stored procedures (in multiple languages).
 It includes most SQL:2008 data types, including INTEGER,
 NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP.
 It also supports storage of binary large objects, including pictures,
 sounds, or video. It has native programming interfaces for C/C++, Java,
 .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation.
 - [PuLP/2.8.0-foss-2023a](https://github.com/coin-or/pulp)
PuLP is an LP modeler written in Python. PuLP can generate MPS or LP files and
call GLPK, COIN-OR CLP/CBC, CPLEX, GUROBI, MOSEK, XPRESS, CHOCO, MIPCL, SCIP to
solve linear problems.

 - [PyCairo/1.25.0-GCCcore-12.3.0](https://pycairo.readthedocs.io/)
Python bindings for the cairo library
 - [PyGEOS/0.14-gfbf-2023a](https://pygeos.readthedocs.io)
PyGEOS is a C/Python library with vectorized geometry functions. The geometry operations are done in
 the open-source geometry library GEOS. PyGEOS wraps these operations in NumPy ufuncs providing a performance
 improvement when operating on arrays of geometries.
 - [PyGObject/3.46.0-GCCcore-12.3.0](https://pygobject.readthedocs.io/)
PyGObject is a Python package which provides bindings for GObject based
libraries such as GTK, GStreamer, WebKitGTK, GLib, GIO and many more.
 - [PyPy/7.3.18](https://www.pypy.org)
A fast, compliant alternative implementation of Python
 - [PyQt-builder/1.15.4-GCCcore-12.3.0](http://www.example.com)
PyQt-builder is the PEP 517 compliant build system for PyQt and projects that   
extend PyQt. It extends the SIP build system and uses Qt’s qmake to perform the 
actual compilation and installation of extension modules.

 - [PyQt5/5.15.10-GCCcore-12.3.0](https://www.riverbankcomputing.com/software/pyqt)
PyQt5 is a set of Python bindings for v5 of the Qt application framework from The Qt Company.
This bundle includes PyQtWebEngine, a set of Python bindings for The Qt Company’s Qt WebEngine framework.
 - [PyRosetta/4.387-gompi-2023a](https://www.pyrosetta.org/)
PyRosetta is an interactive Python-based interface to the powerful Rosetta molecular modeling
suite. It enables users to design their own custom molecular modeling algorithms using Rosetta
sampling methods and energy functions.

 - [PyTables/3.10.2-foss-2024a](https://www.pytables.org)
PyTables is a package for managing hierarchical datasets and designed to efficiently and easily cope
 with extremely large amounts of data. PyTables is built on top of the HDF5 library, using the Python language and the
 NumPy package. It features an object-oriented interface that, combined with C extensions for the performance-critical
 parts of the code (generated using Cython), makes it a fast, yet extremely easy to use tool for interactively browsing,
 processing and searching very large amounts of data. One important feature of PyTables is that it optimizes memory and
 disk resources so that data takes much less space (specially if on-flight compression is used) than other solutions
 such as relational or object oriented databases.
 - [PyTorch/2.6.0-foss-2024a](https://pytorch.org/)
Tensors and Dynamic neural networks in Python with strong GPU acceleration.
PyTorch is a deep learning framework that puts Python first.
 - [PyTorch-Lightning/2.2.1-foss-2023a](https://pytorchlightning.ai)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/p/PyTorch-Lightning/PyTorch-Lightning-2.2.1-foss-2023a.eb)
PyTorch Lightning is the lightweight PyTorch wrapper for ML researchers.
 - [PyTorch-bundle/2.1.2-foss-2023a-CUDA-12.1.1](https://pytorch.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/p/PyTorch-bundle/PyTorch-bundle-2.1.2-foss-2023a-CUDA-12.1.1.eb)
PyTorch with compatible versions of official Torch extensions.
 - [PyWavelets/1.8.0-foss-2024a](https://pywavelets.readthedocs.io)
PyWavelets is open source wavelet transform software for Python.
 - [PyYAML/6.0.2-GCCcore-14.2.0](https://github.com/yaml/pyyaml)
PyYAML is a YAML parser and emitter for the Python programming language.
 - [PyZMQ/26.2.0-GCCcore-13.3.0](https://www.zeromq.org/bindings:python)
Python bindings for ZeroMQ
 - [PycURL/7.45.2-GCCcore-12.3.0](http://pycurl.io/)
PycURL is a Python interface to libcurl. PycURL can be used to fetch objects identified by a URL
 from a Python program, similar to the urllib Python module. PycURL is mature, very fast, and supports a lot of
 features.
 - [Pygments/2.18.0-GCCcore-12.3.0](https://pygments.org/)
Generic syntax highlighter suitable for use in code hosting, forums, wikis or other applications
 that need to prettify source code.
 - [Pyomo/6.7.3-foss-2023a](https://www.pyomo.org/)
Pyomo is a Python-based open-source software package that supports a diverse set of optimization
 capabilities for formulating and analyzing optimization models. 
 - [Pysam/0.22.1-GCC-13.3.0](https://github.com/pysam-developers/pysam)
Pysam is a python module for reading and manipulating Samfiles.
 It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.
 - [Python/3.13.1-GCCcore-14.2.0](https://python.org/)
Python is a programming language that lets you work more quickly and integrate your systems
 more effectively.
 - [Python-bundle-PyPI/2025.04-GCCcore-14.2.0](https://python.org/)
Bundle of Python packages from PyPI
 - [QIIME2/2024.5.0-foss-2023a](https://qiime2.org)
QIIME 2 is a powerful, extensible, and decentralized microbiome bioinformatics platform that is free,
open source, and community developed.
 - [Qhull/2020.2-GCCcore-13.2.0](http://www.qhull.org)
Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram,
 halfspace intersection about a point, furthest-site Delaunay triangulation,
 and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and
 higher dimensions. Qhull implements the Quickhull algorithm for computing the
 convex hull.

 - [Qt5/5.15.13-GCCcore-13.2.0](https://qt.io/)
Qt is a comprehensive cross-platform C++ application framework.
 - [Qt6/6.7.2-GCCcore-13.3.0](https://qt.io/)
Qt is a comprehensive cross-platform C++ application framework.
 - [QtPy/2.4.1-GCCcore-12.3.0](https://github.com/spyder-ide/qtpy)
QtPy is a small abstraction layer that lets you write applications using a single API call to
 either PyQt or PySide. It provides support for PyQt5, PyQt4, PySide2 and PySide.
 - [Qtconsole/5.5.1-GCCcore-12.3.0](https://jupyter.org/)
A rich Qt-based console for working with Jupyter kernels, supporting rich media 
output, session export, and more.
The Qtconsole is a very lightweight application that largely feels like a       
terminal, but provides a number of enhancements only possible in a GUI, such as 
inline figures, proper multiline editing with syntax highlighting, graphical    
calltips, and more.
 - [R/4.5.1-gfbf-2024a](https://www.r-project.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/r/R/R-4.5.1-gfbf-2024a.eb)
R is a free software environment for statistical computing
 and graphics.
 - [R-Tidyverse/4.4.0-gfbf-2023b](http://www.r-project.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/r/R-Tidyverse/R-Tidyverse-4.4.0-gfbf-2023b.eb)
R is a free software environment for statistical computing and graphics.
 - [R-Torch/2.7.1-foss-2024a-R-4.4.2](https://cran.r-project.org/web/packages/torch/index.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/r/R-Torch/R-Torch-2.7.1-foss-2024a-R-4.4.2.eb)
Provides functionality to define and train neural networks similar to 'PyTorch'.
 Includes Bioconductor 3.20.
 - [R-bundle-Bioconductor/3.21-foss-2024a-R-4.5.1](https://bioconductor.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.21-foss-2024a-R-4.5.1.eb)
Bioconductor provides tools for the analysis and coprehension
 of high-throughput genomic data.
 - [R-bundle-CRAN/2025.06-foss-2024a](https://www.r-project.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2025.06-foss-2024a.eb)
Bundle of R packages from CRAN
 - [RAPIDS/24.4-foss-2023a-CUDA-12.1.1](https://rapids.ai/)
RAPIDS provides unmatched speed with familiar APIs that match the most popular
PyData libraries. Built on state-of-the-art foundations like NVIDIA CUDA and
Apache Arrow, it unlocks the speed of GPUs with code you already know.
 - [RDFlib/7.0.0-GCCcore-12.3.0](https://github.com/RDFLib/rdflib)
RDFLib is a Python library for working with RDF, a simple yet powerful language
 for representing information.
 - [RDKit/2025.03.3-foss-2024a](https://www.rdkit.org)
RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python.
 - [RE2/2024-07-02-GCCcore-13.3.0](https://github.com/google/re2)
RE2 is a fast, safe, thread-friendly alternative to backtracking regular
expression engines like those used in PCRE, Perl, and Python. It is a C++
library. 
 - [RStudio-Server/2024.12.0+467-foss-2024a-Java-17-R-4.4.2](https://www.rstudio.com/)
This is the RStudio Server version.
RStudio is a set of integrated tools designed to help you be more productive with R.

The server can be started with:
  rserver --server-daemonize=0 --www-port=8787

If you need a database config one can be created with:
  MYTMP=`mktemp -d` && echo -e "provider=sqlite\ndirectory=${MYTMP}/sqlite" > "${MYTMP}/db.conf"
and then used with:
  rserver ... --database-config-file="${MYTMP}/db.conf"

 - [RapidJSON/1.1.0-20240409-GCCcore-13.2.0](https://rapidjson.org)
A fast JSON parser/generator for C++ with both SAX/DOM style API
 - [Ray-project/2.9.1-foss-2023a](https://docs.ray.io/en/latest/)
Ray is a fast and simple framework for building and running distributed applications.
 - [Redis/7.2.3-GCCcore-12.3.0](https://redis.io)
Redis is an open source (BSD licensed), in-memory data structure store, used as
a database, cache, and message broker. Redis provides data structures such as
strings, hashes, lists, sets, sorted sets with range queries, bitmaps,
hyperloglogs, geospatial indexes, and streams. Redis has built-in replication,
Lua scripting, LRU eviction, transactions, and different levels of on-disk
persistence, and provides high availability via Redis Sentinel and automatic
partitioning with Redis Cluster.
 - [Ruby/3.3.0-GCCcore-12.3.0](https://www.ruby-lang.org)
Ruby is a dynamic, open source programming language with
 a focus on simplicity and productivity. It has an elegant syntax that is
 natural to read and easy to write.
 - [Rust/1.86.0-GCCcore-13.3.0](https://www.rust-lang.org)
Rust is a systems programming language that runs blazingly fast, prevents segfaults,
 and guarantees thread safety.
 - [SAMtools/1.21-GCC-13.3.0](https://www.htslib.org/)
SAM Tools provide various utilities for manipulating alignments in the SAM format,
 including sorting, merging, indexing and generating alignments in a per-position format.
 - [SCOTCH/7.0.6-gompi-2024a](https://www.labri.fr/perso/pelegrin/scotch/)
Software package and libraries for sequential and parallel graph partitioning,
static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning.
 - [SCimilarity/0.3.0-foss-2023a](https://github.com/genentech/scimilarity)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/SCimilarity/SCimilarity-0.3.0-foss-2023a.eb)
SCimilarity is a unifying representation of single cell expression profiles that 
quantifies similarity between expression states and generalizes to represent new studies without additional 
training.
 - [SDL2/2.30.6-GCCcore-13.3.0](https://www.libsdl.org/)
SDL: Simple DirectMedia Layer, a cross-platform multimedia library
 - [SIP/6.8.1-GCCcore-12.3.0](http://www.riverbankcomputing.com/software/sip/)
SIP is a tool that makes it very easy to create Python bindings for C and C++ libraries.
 - [SOCI/4.0.3-GCC-13.2.0-Boost-1.83.0](http://soci.sourceforge.net/)
SOCI is a database access library for C++ that makes the illusion of embedding SQL queries in the
 regular C++ code, staying entirely within the Standard C++.
 - [SQLAlchemy/2.0.36-GCCcore-13.3.0](https://www.sqlalchemy.org/)
SQLAlchemy is the Python SQL toolkit and Object Relational Mapper that gives
application developers the full power and flexibility of SQL. SQLAlchemy
provides a full suite of well known enterprise-level persistence patterns,
designed for efficient and high-performing database access, adapted into a
simple and Pythonic domain language.
 - [SQLite/3.47.2-GCCcore-14.2.0](https://www.sqlite.org/)
SQLite: SQL Database Engine in a C Library
 - [SRA-Toolkit/3.2.0-gompi-2024a](https://github.com/ncbi/sra-tools)
The SRA Toolkit, and the source-code SRA System Development
 Kit (SDK), will allow you to programmatically access data housed within SRA
 and convert it from the SRA format
 - [STAR/2.7.11b-GCC-13.2.0](https://github.com/alexdobin/STAR)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/STAR/STAR-2.7.11b-GCC-13.2.0.eb)
STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.
 - [STAR-Fusion/1.12.0-foss-2022b](https://github.com/STAR-Fusion/STAR-Fusion)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/STAR-Fusion/STAR-Fusion-1.12.0-foss-2022b.eb)
STAR-Fusion uses the STAR aligner to identify candidate fusion transcripts
 supported by Illumina reads. STAR-Fusion further processes the output generated by the STAR aligner
 to map junction reads and spanning reads to a reference annotation set.
 - [SVIM/2.0.0-foss-2024a](https://github.com/eldariont/svim)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/SVIM/SVIM-2.0.0-foss-2024a.eb)
SVIM (pronounced swim) is a structural variant caller for third-generation sequencing reads.
 It is able to detect and classify the following six classes of structural variation: deletions, insertions,
 inversions, tandem duplications, interspersed duplications and translocations.
 - [SVclone/1.1.2-foss-2022b](https://github.com/mcmero/SVclone)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/SVclone/SVclone-1.1.2-foss-2022b.eb)
Cluster structural variants of similar cancer cell fraction (CCF).
 - [SWIG/4.2.1-GCCcore-13.3.0](http://www.swig.org/)
SWIG is a software development tool that connects programs written in C and C++ with
 a variety of high-level programming languages.
 - [SYMPHONY/5.7.2-foss-2023b](https://github.com/coin-or/SYMPHONY/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/SYMPHONY/SYMPHONY-5.7.2-foss-2023b.eb)
SYMPHONY is an open-source solver for mixed-integer linear programs (MILPs) written in C.
 - [Sambamba/1.0.1-GCC-13.2.0](https://lomereiter.github.io/sambamba/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/Sambamba/Sambamba-1.0.1-GCC-13.2.0.eb)
Sambamba is a high performance modern robust and fast tool
 (and library), written in the D programming language, for working with SAM
 and BAM files. Current functionality is an important subset of samtools
 functionality, including view, index, sort, markdup, and depth.
 - [ScaLAPACK/2.2.2-gompi-2025a-fb](https://www.netlib.org/scalapack/)
The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines
 redesigned for distributed memory MIMD parallel computers.
 - [Scalene/1.5.26-GCCcore-12.3.0](https://github.com/plasma-umass/scalene)
Scalene is a high-performance CPU, GPU and memory profiler for Python that does a number of things
that other Python profilers do not and cannot do. It runs orders of magnitude faster than other profilers while
delivering far more detailed information.
 - [SciPy-bundle/2025.07-gfbf-2024a](https://python.org/)
Bundle of Python packages for scientific software
 - [Seaborn/0.13.2-gfbf-2023b](https://seaborn.pydata.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/Seaborn/Seaborn-0.13.2-gfbf-2023b.eb)
Seaborn is a Python visualization library based on matplotlib.
 It provides a high-level interface for drawing attractive statistical graphics. 
 - [SentencePiece/0.2.0-GCC-12.3.0](https://github.com/google/sentencepiece)
Unsupervised text tokenizer for Neural Network-based text generation.
 - [SeqKit/2.10.1](https://bioinf.shenwei.me/seqkit/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/SeqKit/SeqKit-2.10.1.eb)
SeqKit
 - [Seurat/5.1.0-foss-2023a-R-4.3.2](https://satijalab.org/seurat)
Seurat is an R package designed for QC, analysis, and exploration of single cell RNA-seq data.
 - [ShapeMapper2/2.3-GCC-13.3.0](https://github.com/Weeks-UNC/shapemapper2)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/ShapeMapper2/ShapeMapper2-2.3-GCC-13.3.0.eb)
ShapeMapper automates the calculation of RNA chemical probing reactivities
 from mutational profiling (MaP) experiments, in which chemical adducts on RNA
 are detected as internal mutations in cDNA through reverse transcription and
 read out by massively parallel sequencing.

 - [Shapely/2.0.1-gfbf-2023a](https://github.com/Toblerity/Shapely)
Shapely is a BSD-licensed Python package for manipulation and analysis of planar geometric objects.
It is based on the widely deployed GEOS (the engine of PostGIS) and JTS (from which GEOS is ported) libraries.
 - [SigProfiler/202507-foss-2024a-numpy-2](https://cancer.sanger.ac.uk/signatures/tools/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/SigProfiler/SigProfiler-202507-foss-2024a-numpy-2.eb)
SigProfiler provides a comprehensive and integrated suite of bioinformatic
 tools for performing mutational signature analysis.

 - [Sniffles/2.6.2-gfbf-2024a](https://github.com/fritzsedlazeck/Sniffles)
A fast structural variant caller for long-read sequencing,
 Sniffles2 accurately detect SVs on germline, somatic and population-level for PacBio and Oxford Nanopore read data.
 - [SoX/14.4.2-GCCcore-12.3.0](http://sox.sourceforge.net/)
Sound eXchange, the Swiss Army knife of audio manipulation
 - [Sphinx-RTD-Theme/2.0.0-GCCcore-12.3.0](https://sphinx-rtd-theme.readthedocs.io)
Sphinx theme designed to provide a great reader experience
 for documentation users on both desktop and mobile devices.
 - [Squidpy/1.4.1-foss-2023a](https://squidpy.readthedocs.io)
Squidpy is a tool for the analysis and visualization of spatial molecular data.
 - [SuiteSparse/7.1.0-foss-2023a](https://faculty.cse.tamu.edu/davis/suitesparse.html)
SuiteSparse is a collection of libraries to manipulate sparse matrices.
 - [SuperLU_DIST/8.1.2-foss-2023a](https://crd-legacy.lbl.gov/~xiaoye/SuperLU/)
SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems
 of linear equations on high performance machines.
 - [Szip/2.1.1-GCCcore-12.2.0](https://support.hdfgroup.org/doc_resource/SZIP/)
Szip compression software, providing lossless compression of scientific data

 - [TGS-GapCloser/1.2.1-GCCcore-13.3.0](https://github.com/BGI-Qingdao/TGS-GapCloser)
A gap-closing software tool that uses error-prone long reads generated by
 third-generation-sequence techniques (Pacbio, Oxford Nanopore, etc.) or preassembled contigs
 to fill N-gap in the genome assembly.
 - [Tcl/8.6.16-GCCcore-14.2.0](https://www.tcl.tk/)
Tcl (Tool Command Language) is a very powerful but easy to learn dynamic
 programming language, suitable for a very wide range of uses, including web
 and desktop applications, networking, administration, testing and many more.

 - [Telescope/1.0.3-gfbf-2022b](https://github.com/mlbendall/telescope)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/t/Telescope/Telescope-1.0.3-gfbf-2022b.eb)
Single locus resolution of Transposable ELEment expression using next-generation sequencing.
 - [TensorFlow/2.18.1-foss-2024a](https://www.tensorflow.org/)
An open-source software library for Machine Intelligence
 - [Tk/8.6.16-GCCcore-14.2.0](https://www.tcl.tk/)
Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for
 building a graphical user interface (GUI) in many different programming languages.
 - [Tkinter/3.12.3-GCCcore-13.3.0](https://python.org/)
Tkinter module, built with the Python buildsystem
 - [UCC/1.3.0-GCCcore-13.3.0](https://www.openucx.org/)
UCC (Unified Collective Communication) is a collective
communication operations API and library that is flexible, complete, and 
feature-rich for current and emerging programming models and runtimes.

 - [UCX/1.18.0-GCCcore-14.2.0](https://www.openucx.org/)
Unified Communication X
An open-source production grade communication framework for data centric
and high-performance applications

 - [UCX-CUDA/1.16.0-GCCcore-13.3.0-CUDA-12.6.0](http://www.openucx.org/)
Unified Communication X
An open-source production grade communication framework for data centric
and high-performance applications

This module adds the UCX CUDA support.

 - [UDUNITS/2.2.28-GCCcore-13.3.0](https://www.unidata.ucar.edu/software/udunits/)
UDUNITS supports conversion of unit specifications between formatted and binary forms,
 arithmetic manipulation of units, and conversion of values between compatible scales of measurement.
 - [UMI-tools/1.1.4-foss-2023b](https://umi-tools.readthedocs.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/u/UMI-tools/UMI-tools-1.1.4-foss-2023b.eb)
Tools for handling Unique Molecular Identifiers in NGS data sets
 - [UnZip/6.0-GCCcore-12.3.0](http://www.info-zip.org/UnZip.html)
UnZip is an extraction utility for archives compressed
in .zip format (also called "zipfiles"). Although highly compatible both
with PKWARE's PKZIP and PKUNZIP utilities for MS-DOS and with Info-ZIP's
own Zip program, our primary objectives have been portability and
non-MSDOS functionality.
 - [Uni-Core/0.0.3-foss-2023a](https://github.com/dptech-corp/Uni-Core)
An efficient distributed PyTorch framework
 - [UniFrac/1.4-foss-2023a](https://github.com/biocore/unifrac-binaries)
UniFrac is the de facto repository for high-performance phylogenetic diversity
calculations. The methods in this repository are based on an implementation of
the Strided State UniFrac algorithm which is faster, and uses less memory than
Fast UniFrac. Strided State UniFrac supports Unweighted UniFrac, Weighted
UniFrac, Generalized UniFrac, Variance Adjusted UniFrac and meta UniFrac, in
both double and single precision (fp32). This repository also includes Stacked
Faith (manuscript in preparation), a method for calculating Faith's PD that is
faster and uses less memory than the Fast UniFrac-based reference
implementation.
 - [VEP/113.3-GCC-13.3.0](https://www.ensembl.org/info/docs/tools/vep)
Variant Effect Predictor (VEP) determines the effect of your
 variants (SNPs, insertions, deletions, CNVs or structural variants) on genes,
 transcripts, and protein sequence, as well as regulatory regions.
 Includes EnsEMBL-XS, which provides pre-compiled replacements for frequently
 used routines in VEP.
 - [VSEARCH/2.25.0-GCC-12.3.0](https://github.com/torognes/vsearch)
VSEARCH supports de novo and reference based chimera detection,
 clustering, full-length and prefix dereplication, rereplication,
 reverse complementation, masking, all-vs-all pairwise global alignment,
 exact and global alignment searching, shuffling, subsampling and sorting.
 It also supports FASTQ file analysis, filtering,
 conversion and merging of paired-end reads.
 - [VTK/9.3.0-foss-2023b](https://www.vtk.org)
The Visualization Toolkit (VTK) is an open-source, freely available software system for
 3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several
 interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization
 algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques
 such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation.
 - [Wayland/1.23.92-GCCcore-14.2.0](https://wayland.freedesktop.org/)
Wayland is a project to define a protocol for a compositor to talk to
 its clients as well as a library implementation of the protocol.  The
 compositor can be a standalone display server running on Linux kernel
 modesetting and evdev input devices, an X application, or a wayland
 client itself.  The clients can be traditional applications, X servers
 (rootless or fullscreen) or other display servers.

 - [WhatsHap/2.2-foss-2023a](https://whatshap.readthedocs.io)
WhatsHap is a software for phasing genomic variants using DNA
sequencing reads, also called read-based phasing or haplotype assembly. It is
especially suitable for long reads, but works also well with short reads.
 - [X11/20250521-GCCcore-14.2.0](https://www.x.org)
The X Window System (X11) is a windowing system for bitmap displays
 - [XML-LibXML/2.0210-GCCcore-13.3.0](https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod)
Perl binding for libxml2
 - [XZ/5.6.3-GCCcore-14.2.0](https://tukaani.org/xz/)
xz: XZ utilities
 - [Xerces-C++/3.2.5-GCCcore-13.2.0](https://xerces.apache.org/xerces-c/)
Xerces-C++ is a validating XML parser written in a portable
subset of C++. Xerces-C++ makes it easy to give your application the ability to
read and write XML data. A shared library is provided for parsing, generating,
manipulating, and validating XML documents using the DOM, SAX, and SAX2
APIs.
 - [XlsxWriter/3.2.0-GCCcore-13.3.0](https://xlsxwriter.readthedocs.io/)
A Python module for creating Excel XLSX files
 - [Xvfb/21.1.14-GCCcore-13.3.0](https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml)
Xvfb is an X server that can run on machines with no display hardware and no physical input devices.
 It emulates a dumb framebuffer using virtual memory.
 - [Yasm/1.3.0-GCCcore-12.2.0](https://www.tortall.net/projects/yasm/)
Yasm: Complete rewrite of the NASM assembler with BSD license
 - [Z3/4.13.4-GCCcore-14.2.0](https://github.com/Z3Prover/z3)
Z3 is a theorem prover from Microsoft Research with support for bitvectors,
booleans, arrays, floating point numbers, strings, and other data types. This
module includes z3-solver, the Python interface of Z3.

 - [ZeroMQ/4.3.5-GCCcore-13.3.0](https://www.zeromq.org/)
ZeroMQ looks like an embeddable networking library but acts like a concurrency framework.
 It gives you sockets that carry atomic messages across various transports like in-process,
 inter-process, TCP, and multicast. You can connect sockets N-to-N with patterns like fanout,
 pub-sub, task distribution, and request-reply. It's fast enough to be the fabric for clustered
 products. Its asynchronous I/O model gives you scalable multicore applications, built as asynchronous
 message-processing tasks. It has a score of language APIs and runs on most operating systems.
 - [Zip/3.0-GCCcore-13.3.0](http://www.info-zip.org/Zip.html)
Zip is a compression and file packaging/archive utility.
Although highly compatible both with PKWARE's PKZIP and PKUNZIP
utilities for MS-DOS and with Info-ZIP's own UnZip, our primary objectives
have been portability and other-than-MSDOS functionality
 - [absl-py/2.1.0-GCCcore-13.3.0](https://github.com/abseil/abseil-py)
absl-py is a collection of Python library code for building Python
applications. The code is collected from Google's own Python code base, and has
been extensively tested and used in production.
 - [adjustText/0.7.3-foss-2023a](https://github.com/Phlya/adjustText)
A small library for automatically adjustment of text position in matplotlib plots to minimize overlaps.
 - [aiohttp/3.10.10-GCCcore-13.3.0](https://github.com/aio-libs/aiohttp)
Asynchronous HTTP client/server framework for asyncio and Python.
 - [alsa-lib/1.2.9-GCCcore-12.3.0](https://www.alsa-project.org)
The Advanced Linux Sound Architecture (ALSA) provides audio and MIDI functionality
 to the Linux operating system.
 - [anndata/0.11.3-foss-2024a](https://github.com/scverse/anndata)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/a/anndata/anndata-0.11.3-foss-2024a.eb)
anndata is a Python package for handling annotated data matrices in memory and on disk,
 positioned between pandas and xarray
 - [ant/1.10.14-Java-11](https://ant.apache.org/)
Apache Ant is a Java library and command-line tool whose mission is to drive processes described in
 build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of
 Java applications.
 - [arpack-ng/3.9.1-foss-2024a](https://github.com/opencollab/arpack-ng)
ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems.
 - [arrow-R/17.0.0.1-foss-2024a-R-4.5.1](https://cran.r-project.org/web/packages/arrow)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/a/arrow-R/arrow-R-17.0.0.1-foss-2024a-R-4.5.1.eb)
R interface to the Apache Arrow C++ library
 - [assimp/5.4.3-GCCcore-13.3.0](https://github.com/assimp/assimp)
Open Asset Import Library (assimp) is a library to import and export various
 3d-model-formats including scene-post-processing to generate missing render data.

 - [at-spi2-atk/2.38.0-GCCcore-13.2.0](https://wiki.gnome.org/Accessibility)
AT-SPI 2 toolkit bridge
 - [at-spi2-core/2.50.0-GCCcore-13.2.0](https://wiki.gnome.org/Accessibility)
Assistive Technology Service Provider Interface.

 - [attr/2.5.1-GCCcore-12.3.0](https://savannah.nongnu.org/projects/attr)
Commands for Manipulating Filesystem Extended Attributes
 - [awscli/2.17.54-GCCcore-12.3.0](https://pypi.python.org/pypi/awscli)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/a/awscli/awscli-2.17.54-GCCcore-12.3.0.eb)
Universal Command Line Environment for AWS
 - [bam-readcount/1.0.1-GCC-12.2.0](https://github.com/genome/bam-readcount)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/bam-readcount/bam-readcount-1.0.1-GCC-12.2.0.eb)
Count DNA sequence reads in BAM files
 - [bcrypt/4.1.3-GCCcore-13.2.0](https://github.com/pyca/bcrypt/)
Acceptable password hashing for your software and your servers (but you should
really use argon2id or scrypt)

 - [beagle-lib/4.0.1-GCC-12.3.0-CUDA-12.1.1](https://github.com/beagle-dev/beagle-lib)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/beagle-lib/beagle-lib-4.0.1-GCC-12.3.0-CUDA-12.1.1.eb)
beagle-lib is a high-performance library that can perform the core calculations at the heart of most
 Bayesian and Maximum Likelihood phylogenetics packages.
 - [binutils/2.42](https://directory.fsf.org/project/binutils/)
binutils: GNU binary utilities
 - [biom-format/2.1.15-foss-2023a](https://biom-format.org)
The BIOM file format (canonically pronounced biome) is designed to be
 a general-use format for representing biological sample by observation
 contingency tables. BIOM is a recognized standard for the Earth Microbiome
 Project and is a Genomics Standards Consortium supported project.

 - [bokeh/3.6.0-gfbf-2024a](https://github.com/bokeh/bokeh)
Statistical and novel interactive HTML plots for Python
 - [bwidget/1.9.16-GCCcore-12.3.0](https://core.tcl-lang.org/bwidget/home)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/bwidget/bwidget-1.9.16-GCCcore-12.3.0.eb)
The BWidget Toolkit is a high-level Widget Set for Tcl/Tk built using native Tcl/Tk 8.x namespaces.
 - [bx-python/0.13.0-foss-2023b](https://github.com/bxlab/bx-python)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/bx-python/bx-python-0.13.0-foss-2023b.eb)
The bx-python project is a Python library and associated set of scripts to allow for rapid
 implementation of genome scale analyses.
 - [bzip2/1.0.8-GCCcore-12.2.0](https://sourceware.org/bzip2)
bzip2 is a freely available, patent free, high-quality data compressor. It
 typically compresses files to within 10% to 15% of the best available
 techniques (the PPM family of statistical compressors), whilst being around
 twice as fast at compression and six times faster at decompression.

 - [cURL/8.11.1-GCCcore-14.2.0](https://curl.haxx.se)
libcurl is a free and easy-to-use client-side URL transfer library,
 supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP,
 LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.
 libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP
 form based upload, proxies, cookies, user+password authentication (Basic,
 Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling
 and more.

 - [cairo/1.18.4-GCCcore-14.2.0](https://cairographics.org)
Cairo is a 2D graphics library with support for multiple output devices.
 Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers,
 PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB
 - [castor/1.8.2-foss-2023a](https://CRAN.R-project.org/package=castor)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/castor/castor-1.8.2-foss-2023a.eb)
Efficient phylogenetic analyses on massive phylogenies comprising up to millions of tips.
Functions include pruning, rerooting, calculation of most-recent common ancestors, calculating
distances from the tree root and calculating pairwise distances. Calculation of phylogenetic
signal and mean trait depth (trait conservatism), ancestral state reconstruction and hidden
character prediction of discrete characters, simulating and fitting models of trait evolution,
fitting and simulating diversification models, dating trees, comparing trees, and
reading/writing trees in Newick format. 

 - [cellsnp-lite/1.2.3-GCC-13.2.0](https://cellsnp-lite.readthedocs.io/en/latest)
Cellsnp-lite is a C/C++ tool for efficient genotyping
bi-allelic SNPs on single cells. You can use cellsnp-lite after read
alignment to obtain the snp x cell pileup UMI or read count matrices for
each allele of given or detected SNPs.
 - [cffi/1.17.1-GCCcore-14.2.0](https://cffi.readthedocs.io/en/latest/)
C Foreign Function Interface for Python. Interact with almost any C code from
Python, based on C-like declarations that you can often copy-paste from header
files or documentation.

 - [cirro/1.6.0-foss-2024a](https://github.com/FredHutch/Cirro-client)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/cirro/cirro-1.6.0-foss-2024a.eb)
A Python 3.8+ library for the Cirro platform.
 - [cmake/3.24.3-GCCcore-12.2.0](https://cmake.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/cmake/cmake-3.24.3-GCCcore-12.2.0.eb)
Python cmake module
 CMake, the cross-platform, open-source build system.  CMake is a family of
 tools designed to build, test and package software.

 - [code-server/4.97.2](https://github.com/coder/code-server)
Run VS Code on any machine anywhere and access it in the browser.
 - [configurable-http-proxy/4.5.6-GCCcore-12.3.0](https://github.com/jupyterhub/configurable-http-proxy)
HTTP proxy for node.js including a REST API for updating the routing table.
 Developed as a part of the Jupyter Hub multi-user server.
 - [cooler/0.10.2-foss-2023a](https://open2c.github.io/cooler)
Cooler is a support library for a storage format, also called cooler, used to store
 genomic interaction data of any size, such as Hi-C contact matrices.
 - [cooltools/0.7.1-foss-2024a](https://cooltools.readthedocs.io/en/latest/index.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/cooltools/cooltools-0.7.1-foss-2024a.eb)
cooltools provides a suite of computational tools with a paired python API and
 command line access, which facilitates workflows either on high-performance
 computing clusters or via custom analysis notebooks. As part of the Open2C
 ecosystem, cooltools also provides detailed introductions to key concepts in
 Hi-C-data analysis with interactive notebook documentation.
 - [cpio/2.15-GCCcore-13.3.0](https://savannah.gnu.org/projects/cpio/)
The cpio package contains tools for archiving.
 - [cppy/1.2.1-GCCcore-12.3.0](https://github.com/nucleic/cppy)
A small C++ header library which makes it easier to write
Python extension modules. The primary feature is a PyObject smart pointer
which automatically handles reference counting and provides convenience
methods for performing common object operations.
 - [cryptography/44.0.2-GCCcore-14.2.0](https://github.com/pyca/cryptography)
cryptography is a package designed to expose cryptographic primitives and recipes to Python developers.
 - [cuDNN/9.5.1.17-CUDA-12.6.2](https://developer.nvidia.com/cudnn)
The NVIDIA CUDA Deep Neural Network library (cuDNN) is
a GPU-accelerated library of primitives for deep neural networks.
 - [cutadapt/4.9-GCCcore-12.3.0](https://opensource.scilifelab.se/projects/cutadapt/)
Cutadapt finds and removes adapter sequences, primers, poly-A tails and
 other types of unwanted sequence from your high-throughput sequencing reads.
 - [cuteSV/2.1.2-foss-2024a](https://github.com/tjiangHIT/cuteSV)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/cuteSV/cuteSV-2.1.2-foss-2024a.eb)
cuteSV uses tailored methods to collect the signatures of various types of SVs
 and employs a clustering-and-refinement method to analyze the signatures to implement sensitive SV detection.
 - [dask/2024.9.1-gfbf-2024a](https://dask.org/)
Dask natively scales Python. Dask provides advanced parallelism for analytics,
enabling performance at scale for the tools you love.
 - [dcm2niix/1.0.20241211-GCCcore-13.3.0](https://github.com/rordenlab/dcm2niix)
dcm2niix is designed to convert neuroimaging data from the DICOM format to the NIfTI format.
 - [deepdiff/6.7.1-GCCcore-12.3.0](https://zepworks.com/deepdiff/current/)
DeepDiff: Deep Difference of dictionaries, iterables and almost any other object recursively.
 - [dill/0.3.9-GCCcore-13.3.0](https://pypi.org/project/dill/)
dill extends python's pickle module for serializing and de-serializing python objects to the majority
 of the built-in python types. Serialization is the process of converting an object to a byte stream, and the inverse
 of which is converting a byte stream back to on python object hierarchy.
 - [dm-haiku/0.0.13-foss-2023a](https://github.com/deepmind/dm-haiku)
Haiku is a simple neural network library for JAX developed by some of the authors of Sonnet, a neural
network library for TensorFlow.
 - [dm-tree/0.1.8-GCCcore-12.3.0](https://github.com/deepmind/tree)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/d/dm-tree/dm-tree-0.1.8-GCCcore-12.3.0.eb)
dm-tree provides tree, a library for working with nested data structures. In a way,
tree generalizes the builtin map function which only supports flat sequences, and
allows to apply a function to each "leaf" preserving the overall structure.
 - [dorado/1.0.2-foss-2024a-CUDA-12.6.0](https://github.com/nanoporetech/dorado)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/d/dorado/dorado-1.0.2-foss-2024a-CUDA-12.6.0.eb)
Dorado is a high-performance, easy-to-use, open source basecaller for Oxford Nanopore reads.
 - [double-conversion/3.3.0-GCCcore-13.3.0](https://github.com/google/double-conversion)
Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles.
 - [dssp/4.4.10-GCC-13.3.0](https://github.com/PDB-REDO/dssp)
This is a rewrite of DSSP, now offering full mmCIF support. The difference with
previous releases of DSSP is that it now writes out an annotated mmCIF file by
default, storing the secondary structure information in the _struct_conf
category.

Another new feature in this version of DSSP is that it now defines Poly-Proline
helices as well.

The DSSP program was designed by Wolfgang Kabsch and Chris Sander to standardize
secondary structure assignment. DSSP is a database of secondary structure
assignments (and much more) for all protein entries in the Protein Data Bank
(PDB). DSSP is also the program that calculates DSSP entries from PDB entries.

DSSP does not predict secondary structure.
 - [e3nn/0.5.0-foss-2022b-PyTorch-1.13.1-CUDA-11.8.0](https://e3nn.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/e/e3nn/e3nn-0.5.0-foss-2022b-PyTorch-1.13.1-CUDA-11.8.0.eb)
Euclidean neural networks (e3nn) is a python library based on pytorch to create equivariant
neural networks for the group O(3).

 - [eSpeak-NG/1.51-gfbf-2023a](https://github.com/espeak-ng/espeak-ng)
The eSpeak NG is a compact open source software text-to-speech synthesizer
for Linux, Windows, Android and other operating systems.
It supports more than 100 languages and accents.
It is based on the eSpeak engine created by Jonathan Duddington.

 - [easel/0.48-GCC-12.2.0](https://bioeasel.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/e/easel/easel-0.48-GCC-12.2.0.eb)
Easel supports computational analysis of biological sequences using probabilistic models.
 - [edlib/1.3.9.post1-GCC-13.3.0](https://martinsos.github.io/edlib)
Lightweight, super fast library for sequence alignment using edit (Levenshtein) distance.
 - [einops/0.7.0-GCCcore-12.3.0](https://einops.rocks/)
Flexible and powerful tensor operations for readable and reliable code.
Supports numpy, pytorch, tensorflow, jax, and others.
 - [elfutils/0.190-GCCcore-13.2.0](https://elfutils.org/)
The elfutils project provides libraries and tools for ELF files
 and DWARF data.

 - [epiScanpy/0.4.0-foss-2023a](https://github.com/colomemaria/episcanpy)
EpiScanpy is a toolkit to analyse single-cell open chromatin
(scATAC-seq) and single-cell DNA methylation (for example scBS-seq)
data. EpiScanpy is the epigenomic extension of the very popular scRNA-seq
analysis tool Scanpy (Genome Biology, 2018) [Wolf18].
 - [expat/2.6.4-GCCcore-14.2.0](https://libexpat.github.io)
Expat is an XML parser library written in C. It is a stream-oriented parser
in which an application registers handlers for things the parser might find
in the XML document (like start tags).
 - [expecttest/0.2.1-GCCcore-13.3.0](https://github.com/ezyang/expecttest)
This library implements expect tests (also known as "golden" tests). Expect tests are a method of
 writing tests where instead of hard-coding the expected output of a test, you run the test to get the output, and
 the test framework automatically populates the expected output. If the output of the test changes, you can rerun
 the test with the environment variable EXPECTTEST_ACCEPT=1 to automatically update the expected output.
 - [fastp/0.23.4-GCC-13.2.0](https://github.com/OpenGene/fastp)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/f/fastp/fastp-0.23.4-GCC-13.2.0.eb)
A tool designed to provide fast all-in-one preprocessing for FastQ files.
 This tool is developed in C++ with multithreading supported to afford high performance.
 - [festival/2.5.0-GCCcore-12.3.0](['http://festvox.org/festival/'])
University of Edinburgh's Festival Speech Synthesis Systems is a free software
multi-lingual speech synthesis workbench that runs on multiple-platforms
offering black box text to speech, as well as an open architecture for
research in speech synthesis.
It designed as a component of large speech technology systems.

 - [ffnvcodec/12.2.72.0](https://git.videolan.org/?p=ffmpeg/nv-codec-headers.git)
FFmpeg nvidia headers. Adds support for nvenc and nvdec. Requires Nvidia GPU and drivers to be present
(picked up dynamically).
 - [fhPython/3.12.3-foss-2024a](http://python.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/f/fhPython/fhPython-3.12.3-foss-2024a.eb)
Fred Hutch Python
 - [fhR/4.4.2-foss-2024a](http://www.r-project.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/f/fhR/fhR-4.4.2-foss-2024a.eb)
R is a free software environment for statistical computing and graphics.
 - [fibertools-rs/0.6.4]()
fibertools-rs a CLI tool for creating and interacting with Fiber-seq BAM files.
 - [file/5.46-GCCcore-13.3.0](https://www.darwinsys.com/file/)
The file command is 'a file type guesser', that is, a command-line tool
 that tells you in words what kind of data a file contains.
 - [flash-attention/2.6.3-foss-2023a-CUDA-12.1.1](https://github.com/Dao-AILab/flash-attention)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/f/flash-attention/flash-attention-2.6.3-foss-2023a-CUDA-12.1.1.eb)
Fast and memory-efficient exact attention.
 - [flatbuffers/24.3.25-GCCcore-13.3.0](https://github.com/google/flatbuffers/)
FlatBuffers: Memory Efficient Serialization Library
 - [flatbuffers-python/24.3.25-GCCcore-13.3.0](https://github.com/google/flatbuffers/)
Python Flatbuffers runtime library.
 - [flex/2.6.4-GCCcore-13.2.0](https://github.com/westes/flex)
Flex (Fast Lexical Analyzer) is a tool for generating scanners. A scanner,
 sometimes called a tokenizer, is a program which recognizes lexical patterns
 in text.

 - [flit/3.10.1-GCCcore-14.2.0](https://github.com/pypa/flit)
A simple packaging tool for simple packages.
 - [fmt/9.1.0-GCCcore-12.2.0](http://fmtlib.net/)
fmt (formerly cppformat) is an open-source formatting library.
 - [fontconfig/2.16.2-GCCcore-14.2.0](https://www.freedesktop.org/wiki/Software/fontconfig/)
Fontconfig is a library designed to provide system-wide font configuration,
 customization and application access.

 - [fonttools/4.58.4-GCCcore-14.2.0](https://python-markdown.github.io/)
fontTools is a library for manipulating fonts, written in Python.
The project includes the TTX tool, that can convert TrueType and OpenType fonts to and from an XML text format,
which is also called TTX.
It supports TrueType, OpenType, AFM and to an extent Type 1 and some Mac-specific formats.
 - [foss/2025a](https://easybuild.readthedocs.io/en/master/Common-toolchains.html#foss-toolchain)
GNU Compiler Collection (GCC) based compiler toolchain, including
 OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.
 - [freetype/2.13.3-GCCcore-14.2.0](https://www.freetype.org)
FreeType 2 is a software font engine that is designed to be small, efficient,
 highly customizable, and portable while capable of producing high-quality
 output (glyph images). It can be used in graphics libraries, display servers,
 font conversion tools, text image generation tools, and many other products
 as well.

 - [gappa/0.8.5-GCC-12.3.0](https://github.com/lczech/gappa)
gappa is a collection of commands for working with phylogenetic data. Its main
focus are evolutionary placements of short environmental sequences on a
reference phylogenetic tree. Such data is typically produced by tools like
EPA-ng, RAxML-EPA or pplacer and usually stored in jplace files.

 - [gawk/5.3.0-GCC-12.3.0](https://www.gnu.org/software/gawk)
The awk utility interprets a special-purpose programming language that makes it possible to handle
simple data-reformatting jobs with just a few lines of code.
 - [gensim/4.3.2-foss-2023a](https://radimrehurek.com/gensim)
Gensim is a Python library for topic modelling, document indexing and similarity retrieval with
 large corpora.
 - [geopandas/0.14.2-foss-2023a](https://geopandas.org)
GeoPandas is a project to add support for geographic data to pandas objects.
It currently implements GeoSeries and GeoDataFrame types which are subclasses of pandas.Series
and pandas.DataFrame respectively. GeoPandas objects can act on shapely geometry objects and
perform geometric operations.
 - [gettext/0.24-GCCcore-14.2.0](https://www.gnu.org/software/gettext/)
GNU 'gettext' is an important step for the GNU Translation Project, as it is an asset on which we may
build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools
and documentation
 - [gfbf/2025a]((none))
GNU Compiler Collection (GCC) based compiler toolchain, including
 FlexiBLAS (BLAS and LAPACK support) and (serial) FFTW.
 - [gffread/0.12.7-GCCcore-12.2.0](https://ccb.jhu.edu/software/stringtie/gff.shtml#gffread)
GFF/GTF parsing utility providing format conversions,
region filtering, FASTA sequence extraction and more.
 - [gh/2.52.0](https://github.com/cli/)
GitHub’s official command line tool
 - [giflib/5.2.1-GCCcore-12.2.0](http://giflib.sourceforge.net/)
giflib is a library for reading and writing gif images.
It is API and ABI compatible with libungif which was in wide use while
the LZW compression algorithm was patented.
 - [git/2.49.0-GCCcore-14.2.0](https://git-scm.com)
Git is a free and open source distributed version control system designed
to handle everything from small to very large projects with speed and efficiency.
 - [glslang-SPIRV/15.3.0-GCCcore-14.2.0](https://www.khronos.org/opengles/sdk/tools/Reference-Compiler/)
Glslang is the official reference compiler front end for the OpenGL ES and OpenGL shading languages.
It implements a strict interpretation of the specifications for these languages. It is open and free for anyone to use,
either from a command line or programmatically. The OpenGL and OpenGL ES working groups are committed to maintaining
consistency between the reference compiler and the corresponding shading language specifications.
 - [gmpy2/2.2.0-GCCcore-13.3.0](https://github.com/aleaxit/gmpy)
GMP/MPIR, MPFR, and MPC interface to Python 2.6+ and 3.x
 - [gnupg-bundle/20250519-GCCcore-13.3.0](https://www.gnupg.org/software/index.html)
GnuPG — The Universal Crypto Engine
 - [gnuplot/5.4.8-GCCcore-12.3.0](http://gnuplot.sourceforge.net)
Portable interactive, function plotting utility
 - [gompi/2025a]((none))
GNU Compiler Collection (GCC) based compiler toolchain,
 including OpenMPI for MPI support.
 - [googletest/1.15.2-GCCcore-13.3.0](https://github.com/google/googletest)
Google's framework for writing C++ tests on a variety of platforms
 - [gperf/3.3-GCCcore-14.2.0](https://www.gnu.org/software/gperf/)
GNU gperf is a perfect hash function generator. For a given list of strings,
 it produces a hash function and hash table, in form of C or C++ code, for
 looking up a value depending on the input string. The hash function is
 perfect, which means that the hash table has no collisions, and the hash
 table lookup needs a single string comparison only.

 - [gperftools/2.12-GCCcore-12.3.0](https://github.com/gperftools/gperftools)
gperftools is a collection of a high-performance multi-threaded malloc()
implementation, plus some pretty nifty performance analysis tools.
Includes TCMalloc, heap-checker, heap-profiler and cpu-profiler.

 - [graphite2/1.3.14-GCCcore-12.2.0](https://scripts.sil.org/cms/scripts/page.php?site_id=projects&item_id=graphite_home)
Graphite is a "smart font" system developed specifically to
 handle the complexities of lesser-known languages of the world.
 - [groff/1.23.0-GCCcore-13.3.0](https://www.gnu.org/software/groff)
Groff (GNU troff) is a typesetting system that reads plain text mixed with formatting commands
 and produces formatted output.
 - [grpcio/1.70.0-GCCcore-13.3.0](https://grpc.io/)
gRPC is a modern, open source, high-performance remote procedure call (RPC)
framework that can run anywhere. gRPC enables client and server applications to
communicate transparently, and simplifies the building of connected systems.
 - [gzip/1.13-GCCcore-13.2.0](https://www.gnu.org/software/gzip/)
gzip (GNU zip) is a popular data compression program as a replacement for compress
 - [h5netcdf/1.2.0-foss-2023a](https://h5netcdf.org/)
A Python interface for the netCDF4 file-format that reads and writes local or
remote HDF5 files directly via h5py or h5pyd, without relying on the Unidata
netCDF library.
 - [h5py/3.12.1-foss-2024a](https://www.h5py.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/h/h5py/h5py-3.12.1-foss-2024a.eb)
HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,
 version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous
 amounts of data.
 - [hatch-jupyter-builder/0.9.1-GCCcore-13.2.0](https://hatch-jupyter-builder.readthedocs.io)
Hatch Jupyter Builder is a plugin for the hatchling Python build backend. It is
primarily targeted for package authors who are providing JavaScript as part of
their Python packages.
Typical use cases are Jupyter Lab Extensions and Jupyter Widgets.
 - [hatchling/1.27.0-GCCcore-14.2.0](https://hatch.pypa.io)
Extensible, standards compliant build backend used by Hatch,
a modern, extensible Python project manager.
 - [help2man/1.49.3-GCCcore-13.2.0](https://www.gnu.org/software/help2man/)
help2man produces simple manual pages from the '--help' and '--version' output of other commands.
 - [hermes/0.1.0-foss-2022b-PyTorch-1.13.1-CUDA-11.8.0](https://github.com/StatPhysBio/hermes)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/h/hermes/hermes-0.1.0-foss-2022b-PyTorch-1.13.1-CUDA-11.8.0.eb)
HERMES: Holographic Equivariant neuRal network model for
 Mutational Effect and Stability prediction
 - [hmmlearn/0.3.2-gfbf-2023a](https://github.com/hmmlearn/hmmlearn)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/h/hmmlearn/hmmlearn-0.3.2-gfbf-2023a.eb)
hmmlearn is a set of algorithms for unsupervised learning and inference of Hidden Markov Models
 - [hwloc/2.11.2-GCCcore-14.2.0](https://www.open-mpi.org/projects/hwloc/)
The Portable Hardware Locality (hwloc) software package provides a portable
 abstraction (across OS, versions, architectures, ...) of the hierarchical
 topology of modern architectures, including NUMA memory nodes, sockets, shared
 caches, cores and simultaneous multithreading. It also gathers various system
 attributes such as cache and memory information as well as the locality of I/O
 devices such as network interfaces, InfiniBand HCAs or GPUs. It primarily
 aims at helping applications with gathering information about modern computing
 hardware so as to exploit it accordingly and efficiently.

 - [hypothesis/6.133.2-GCCcore-14.2.0](https://github.com/HypothesisWorks/hypothesis)
Hypothesis is an advanced testing library for Python. It lets you write tests which are parametrized
 by a source of examples, and then generates simple and comprehensible examples that make your tests fail. This lets
 you find more bugs in your code with less work.
 - [igraph/0.10.16-foss-2024a](https://igraph.org)
igraph is a collection of network analysis tools with the emphasis on
efficiency, portability and ease of use. igraph is open source and free. igraph can be
programmed in R, Python and C/C++.
 - [imageio/2.36.1-gfbf-2024a](https://imageio.github.io)
Imageio is a Python library that provides an easy interface to read and write a wide range of
 image data, including animated images, video, volumetric data, and scientific formats.
 - [infercnvpy/0.4.3-foss-2023a](https://github.com/icbi-lab/infercnvpy)
Infer copy number variation (CNV) from scRNA-seq data. Plays nicely with Scanpy.
 - [intltool/0.51.0-GCCcore-12.2.0](https://freedesktop.org/wiki/Software/intltool/)
intltool is a set of tools to centralize translation of
 many different file formats using GNU gettext-compatible PO files.
 - [ipympl/0.9.3-gfbf-2023a](https://matplotlib.org/ipympl)
Leveraging the Jupyter interactive widgets framework, ipympl enables the
interactive features of matplotlib in the Jupyter notebook and in JupyterLab.
Besides, the figure canvas element is a proper Jupyter interactive widget which
can be positioned in interactive widget layouts.

 - [jax/0.4.25-gfbf-2023a](https://jax.readthedocs.io/)
Composable transformations of Python+NumPy programs:
differentiate, vectorize, JIT to GPU/TPU, and more
 - [jbigkit/2.1-GCCcore-12.2.0](https://www.cl.cam.ac.uk/~mgk25/jbigkit/)
JBIG-KIT is a software implementation of the JBIG1 data
 compression standard (ITU-T T.82), which was designed for bi-level image
 data, such as scanned documents.
 - [jedi/0.19.1-GCCcore-13.2.0](https://github.com/davidhalter/jedi)
Jedi
 - [jemalloc/5.3.0-GCCcore-12.3.0](http://jemalloc.net)
jemalloc is a general purpose malloc(3) implementation that emphasizes fragmentation avoidance and
 scalable concurrency support.
 - [json-c/0.17-GCCcore-13.3.0](https://github.com/json-c/json-c)
JSON-C implements a reference counting object model that allows you to easily construct JSON objects
 in C, output them as JSON formatted strings and parse JSON formatted strings back into the C representation of JSON
objects.
 - [jupyter-contrib-nbextensions/0.7.0-GCCcore-12.3.0](https://github.com/ipython-contrib/jupyter_contrib_nbextensions)
A collection of various notebook extensions for Jupyter
 - [jupyter-resource-usage/1.0.0-GCCcore-12.3.0](https://github.com/jupyter-server/jupyter-resource-usage)
Jupyter Notebook Extension for monitoring your own Resource Usage (memory and/or CPU)
 - [jupyter-server/2.14.2-GCCcore-13.3.0](https://jupyter.org/)
The Jupyter Server provides the backend (i.e. the core services, APIs, and REST
endpoints) for Jupyter web applications like Jupyter notebook, JupyterLab, and
Voila.
 - [jupyter-server-proxy/4.0.0-GCCcore-12.3.0](https://github.com/jupyterhub/jupyter-server-proxy)
Jupyter Server Proxy lets you run arbitrary external processes
(such as RStudio, Shiny Server, Syncthing, PostgreSQL, Code Server, etc)
alongside your notebook server and provide authenticated web access to them    
using a path like /rstudio next to others like /lab. Alongside the python
package that provides the main functionality, the JupyterLab extension
(@jupyterlab/server-proxy) provides buttons in the JupyterLab launcher window
to get to RStudio for example.
 - [jupyterlmod/4.0.3-GCCcore-12.3.0](https://github.com/cmd-ntrf/jupyter-lmod)
Jupyter interactive notebook server extension that allows users to interact with
environment modules before launching kernels. The extension uses Lmod's Python
interface to accomplish module-related tasks like loading, unloading, saving
collections, etc.
 - [kallisto/0.50.1-foss-2022b](https://pachterlab.github.io/kallisto/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/k/kallisto/kallisto-0.50.1-foss-2022b.eb)
kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally
 of target sequences using high-throughput sequencing reads.
 - [kineto/20250715-GCC-13.3.0](https://github.com/pytorch/kineto)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/k/kineto/kineto-20250715-GCC-13.3.0.eb)
A CPU+GPU Profiling library that provides access to timeline traces and hardware performance counters
 - [leidenalg/0.10.2-foss-2023a](https://github.com/vtraag/leidenalg)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/l/leidenalg/leidenalg-0.10.2-foss-2023a.eb)
Implementation of the Leiden algorithm for various quality
functions to be used with igraph in Python.
 - [libGLU/9.0.3-GCCcore-14.2.0](https://gitlab.freedesktop.org/mesa/glu)
The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL.
 - [libaec/1.1.3-GCCcore-13.3.0](https://gitlab.dkrz.de/k202009/libaec)
Libaec provides fast lossless compression of 1 up to 32 bit wide signed or unsigned integers
(samples). The library achieves best results for low entropy data as often encountered in space imaging
instrument data or numerical model output from weather or climate simulations. While floating point representations
are not directly supported, they can also be efficiently coded by grouping exponents and mantissa.
 - [libaio/0.3.113-GCCcore-12.3.0](https://pagure.io/libaio)
Asynchronous input/output library that uses the kernels native interface.
 - [libarchive/3.7.7-GCCcore-14.2.0](https://www.libarchive.org/)
Multi-format archive and compression library

 - [libcerf/2.3-GCCcore-12.2.0](https://jugit.fz-juelich.de/mlz/libcerf)
libcerf is a self-contained numeric library that provides an efficient and
 accurate implementation of complex error functions, along with Dawson,
 Faddeeva, and Voigt functions.

 - [libcifpp/7.0.8-GCC-12.2.0](https://github.com/PDB-REDO/libcifpp)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/l/libcifpp/libcifpp-7.0.8-GCC-12.2.0.eb)
This library contains code to work with mmCIF and PDB files
 - [libdeflate/1.24-GCCcore-14.2.0](https://github.com/ebiggers/libdeflate)
Heavily optimized library for DEFLATE/zlib/gzip compression and decompression.
 - [libdrm/2.4.125-GCCcore-14.2.0](https://dri.freedesktop.org)
Direct Rendering Manager runtime library.
 - [libepoxy/1.5.10-GCCcore-13.2.0](https://github.com/anholt/libepoxy)
Epoxy is a library for handling OpenGL function pointer management for you
 - [libevent/2.1.12-GCCcore-13.2.0](https://libevent.org/)
The libevent API provides a mechanism to execute a callback function when
 a specific event occurs on a file descriptor or after a timeout has been
 reached.  Furthermore, libevent also support callbacks due to signals or
 regular timeouts.

 - [libfabric/2.0.0-GCCcore-14.2.0](https://ofiwg.github.io/libfabric/)
Libfabric is a core component of OFI. It is the library that defines and exports
the user-space API of OFI, and is typically the only software that applications
deal with directly. It works in conjunction with provider libraries, which are
often integrated directly into libfabric.

 - [libffi/3.4.5-GCCcore-14.2.0](https://sourceware.org/libffi/)
The libffi library provides a portable, high level programming interface to
 various calling conventions. This allows a programmer to call any function
 specified by a call interface description at run-time.
 - [libgd/2.3.3-GCCcore-13.2.0](https://libgd.github.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/l/libgd/libgd-2.3.3-GCCcore-13.2.0.eb)
GD is an open source code library for the dynamic creation of images by programmers.
 - [libgeotiff/1.7.3-GCCcore-13.3.0](https://directory.fsf.org/wiki/Libgeotiff)
Library for reading and writing coordinate system information from/to GeoTIFF files
 - [libgit2/1.9.1-GCCcore-14.2.0](https://libgit2.org/)
libgit2 is a portable, pure C implementation of the Git core methods provided as a re-entrant
linkable library with a solid API, allowing you to write native speed custom Git applications in any language
which supports C bindings.
 - [libglvnd/1.7.0-GCCcore-14.2.0](https://gitlab.freedesktop.org/glvnd/libglvnd)
libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API calls between multiple vendors.
 - [libharu/2.3.0-GCCcore-13.2.0](https://github.com/libharu/libharu/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/l/libharu/libharu-2.3.0-GCCcore-13.2.0.eb)
libHaru is a free, cross platform, open source library for generating PDF files.
 - [libiconv/1.18-GCCcore-14.2.0](https://www.gnu.org/software/libiconv)
Libiconv converts from one character encoding to another through Unicode conversion
 - [libidn2/2.3.8-GCCcore-13.3.0](http://www.gnu.org/software/libidn2)
Libidn2 implements the revised algorithm for internationalized domain names called IDNA2008/TR46.
 - [libjpeg-turbo/3.1.0-GCCcore-14.2.0](https://libjpeg-turbo.org/)
libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to
 accelerate baseline JPEG compression and decompression. libjpeg is a library
 that implements JPEG image encoding, decoding and transcoding.

 - [libleidenalg/0.11.1-foss-2023b](https://github.com/vtraag/libleidenalg)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/l/libleidenalg/libleidenalg-0.11.1-foss-2023b.eb)
Implements the Leiden algorithm in C++
 - [libmad/0.15.1b-GCCcore-12.2.0](https://www.underbit.com/products/mad/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/l/libmad/libmad-0.15.1b-GCCcore-12.2.0.eb)
MAD is a high-quality MPEG audio decoder.
 - [libmcfp/1.3.4-GCCcore-13.3.0](https://github.com/mhekkel/libmcfp)
A library for parsing command line arguments and configuration files and making
them available throughout a program.
 - [libogg/1.3.5-GCCcore-12.2.0](https://xiph.org/ogg/)
Ogg is a multimedia container format, and the native file and stream format for the Xiph.org
multimedia codecs.
 - [libopus/1.5.2-GCCcore-13.3.0](https://www.opus-codec.org/)
Opus is a totally open, royalty-free, highly versatile audio codec. Opus is unmatched for interactive
 speech and music transmission over the Internet, but is also intended for storage and streaming applications. It is
 standardized by the Internet Engineering Task Force (IETF) as RFC 6716 which incorporated technology from Skype’s
 SILK codec and Xiph.Org’s CELT codec.
 - [libpciaccess/0.18.1-GCCcore-13.3.0](https://cgit.freedesktop.org/xorg/lib/libpciaccess/)
Generic PCI access library.
 - [libpng/1.6.48-GCCcore-14.2.0](https://www.libpng.org/pub/png/libpng.html)
libpng is the official PNG reference library
 - [libpsl/0.21.5-GCCcore-14.2.0](https://rockdaboot.github.io/libpsl)
C library for the Public Suffix List
 - [libreadline/8.2-GCCcore-13.3.0](https://tiswww.case.edu/php/chet/readline/rltop.html)
The GNU Readline library provides a set of functions for use by applications
 that allow users to edit command lines as they are typed in. Both Emacs and
 vi editing modes are available. The Readline library includes additional
 functions to maintain a list of previously-entered command lines, to recall
 and perhaps reedit those lines, and perform csh-like history expansion on
 previous commands.

 - [librosa/0.10.1-foss-2023a](https://librosa.org/)
Audio and music processing in Python
 - [librsvg/2.58.0-GCCcore-13.2.0](https://wiki.gnome.org/Projects/LibRsvg)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/l/librsvg/librsvg-2.58.0-GCCcore-13.2.0.eb)
Librsvg is a library to render SVG files using cairo.
 - [libsigc++/3.6.0-GCCcore-12.3.0](https://libsigcplusplus.github.io/libsigcplusplus/)
The libsigc++ package implements a typesafe callback system
for standard C++.
 - [libsndfile/1.2.2-GCCcore-13.3.0](http://www.mega-nerd.com/libsndfile)
Libsndfile is a C library for reading and writing files containing sampled sound
 (such as MS Windows WAV and the Apple/SGI AIFF format) through one standard library interface.
 - [libsodium/1.0.20-GCCcore-13.3.0](https://doc.libsodium.org/)
Sodium is a modern, easy-to-use software library for encryption, decryption,
 signatures, password hashing and more.

 - [libtirpc/1.3.5-GCCcore-13.3.0](https://sourceforge.net/projects/libtirpc/)
Libtirpc is a port of Suns Transport-Independent RPC library to Linux.
 - [libtool/2.5.4-GCCcore-14.2.0](https://www.gnu.org/software/libtool)
GNU libtool is a generic library support script. Libtool hides the complexity
 of using shared libraries behind a consistent, portable interface.

 - [libunistring/1.3-GCCcore-14.2.0](https://www.gnu.org/software/libunistring/)
This library provides functions for manipulating Unicode strings and for
 manipulating C strings according to the Unicode standard.
 - [libunwind/1.8.1-GCCcore-14.2.0](https://www.nongnu.org/libunwind/)
The primary goal of libunwind is to define a portable and efficient C programming interface
 (API) to determine the call-chain of a program. The API additionally provides the means to manipulate the
 preserved (callee-saved) state of each call-frame and to resume execution at any point in the call-chain
 (non-local goto). The API supports both local (same-process) and remote (across-process) operation.
 As such, the API is useful in a number of applications
 - [libvips/8.16.1-GCC-13.2.0](https://github.com/libvips/libvips)
libvips is a demand-driven, horizontally threaded image processing library.
 - [libvorbis/1.3.7-GCCcore-12.3.0](https://xiph.org/vorbis/)
Ogg Vorbis is a fully open, non-proprietary, patent-and-royalty-free, general-purpose compressed
audio format
 - [libwebp/1.4.0-GCCcore-13.3.0](https://developers.google.com/speed/webp/)
WebP is a modern image format that provides superior
lossless and lossy compression for images on the web. Using WebP,
webmasters and web developers can create smaller, richer images that
make the web faster.
 - [libxml2/2.13.4-GCCcore-14.2.0](https://gitlab.gnome.org/GNOME/libxml2/-/wikis)
Libxml2 is the XML C parser and toolchain developed for the Gnome project
 (but usable outside of the Gnome platform).

 - [libxslt/1.1.42-GCCcore-13.3.0](http://xmlsoft.org/)
Libxslt is the XSLT C library developed for the GNOME project
 (but usable outside of the Gnome platform).
 - [libyaml/0.2.5-GCCcore-12.3.0](https://pyyaml.org/wiki/LibYAML)
LibYAML is a YAML parser and emitter written in C.
 - [lit/18.1.8-GCCcore-13.3.0](https://llvm.org/docs/CommandGuide/lit.html)
lit is a portable tool for executing LLVM and Clang style test suites, summarizing their results, and
providing indication of failures.
 - [loompy/3.0.8-foss-2024a](https://loompy.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/l/loompy/loompy-3.0.8-foss-2024a.eb)
Python implementation of the Loom file format, an efficient file format for large omics datasets
 - [lxml/5.3.0-GCCcore-13.3.0](https://lxml.de/)
The lxml XML toolkit is a Pythonic binding for the C libraries libxml2 and libxslt.
 - [lz4/1.10.0-GCCcore-14.2.0](https://lz4.github.io/lz4/)
LZ4 is lossless compression algorithm, providing compression speed at 400 MB/s per core.
 It features an extremely fast decoder, with speed in multiple GB/s per core.
 - [magma/2.9.0-foss-2024a-CUDA-12.6.0](https://icl.cs.utk.edu/magma/)
The MAGMA project aims to develop a dense linear algebra library similar to
 LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems.
 - [make/4.4.1-GCCcore-13.2.0](https://www.gnu.org/software/make/make.html)
GNU version of make utility
 - [makeinfo/7.1-GCCcore-13.3.0](https://www.gnu.org/software/texinfo/)
makeinfo is part of the Texinfo project, the official documentation format of the GNU project.
 - [matplotlib/3.9.2-gfbf-2024a-numpy-2](https://matplotlib.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/matplotlib/matplotlib-3.9.2-gfbf-2024a-numpy-2.eb)
matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
 hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
 and ipython shell, web application servers, and six graphical user interface toolkits.
 - [maturin/1.8.3-GCCcore-14.2.0](https://github.com/pyo3/maturin)
This project is meant as a zero configuration
replacement for setuptools-rust and milksnake. It supports building
wheels for python 3.5+ on windows, linux, mac and freebsd, can upload
them to pypi and has basic pypy and graalpy support.
 - [meson-python/0.18.0-GCCcore-14.2.0](https://github.com/mesonbuild/meson-python)
Python build backend (PEP 517) for Meson projects
 - [minimap2/2.29-GCCcore-13.3.0](https://github.com/lh3/minimap2)
Minimap2 is a fast sequence mapping and alignment
program that can find overlaps between long noisy reads, or map long
reads or their assemblies to a reference genome optionally with detailed
alignment (i.e. CIGAR). At present, it works efficiently with query
sequences from a few kilobases to ~100 megabases in length at an error
rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited
test data sets, minimap2 is over 20 times faster than most other
long-read aligners. It will replace BWA-MEM for long reads and contig
alignment.
 - [ml-collections/0.1.1-foss-2023a](https://github.com/google/ml_collections)
ML Collections is a library of Python Collections designed for ML use cases.

 - [ml_dtypes/0.5.0-gfbf-2024a](https://github.com/jax-ml/ml_dtypes)
ml_dtypes is a stand-alone implementation of several NumPy dtype extensions used
in machine learning libraries, including:

bfloat16: an alternative to the standard float16 format
float8_*: several experimental 8-bit floating point representations including:
float8_e4m3b11fnuz
float8_e4m3fn
float8_e4m3fnuz
float8_e5m2
float8_e5m2fnuz

 - [mm-common/1.0.6-GCCcore-12.3.0](https://gitlab.gnome.org/GNOME/mm-common)
The mm-common module provides the build infrastructure and
utilities shared among the GNOME C++ binding libraries.
 - [mpath/1.1.3-GCCcore-12.3.0](https://pypi.org/project/mpath/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/mpath/mpath-1.1.3-GCCcore-12.3.0.eb)
For now it's quit simple and get_path_info()
method returns information about given path. It can be either
a directory or a file path.
 - [mpi4py/4.0.1-gompi-2024a](https://github.com/mpi4py/mpi4py)
MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for
 the Python programming language, allowing any Python program to exploit multiple processors.
 - [mpmath/1.3.0-GCCcore-12.3.0](https://mpmath.org/)
mpmath can be used as an arbitrary-precision substitute for Python's float/complex
 types and math/cmath modules, but also does much more advanced mathematics. Almost any calculation
 can be performed just as well at 10-digit or 1000-digit precision, with either real or complex
 numbers, and in many cases mpmath implements efficient algorithms that scale well for extremely
 high precision work.
 - [mrc/1.3.13-GCCcore-13.3.0](https://github.com/mhekkel/mrc)
Maartens Resource Compiler to store resources in the static section of an ELF binary.
 - [mrcfile/1.4.3-gfbf-2022b](https://github.com/ccpem/mrcfile)
mrcfile is a Python implementation of the MRC2014 file format, which is used in
structural biology to store image and volume data.

It allows MRC files to be created and opened easily using a very simple API,
which exposes the file’s header and data as numpy arrays. The code runs in
Python 2 and 3 and is fully unit-tested.

This library aims to allow users and developers to read and write standard-
compliant MRC files in Python as easily as possible, and with no dependencies on
any compiled libraries except numpy. You can use it interactively to inspect
files, correct headers and so on, or in scripts and larger software packages to
provide basic MRC file I/O functions. 
 - [multiprocess/0.70.17-gfbf-2024a](https://github.com/uqfoundation/multiprocess)
better multiprocessing and multithreading in python
 - [nbclassic/1.0.0-GCCcore-12.3.0](https://jupyter.org/)
NbClassic provides a backwards compatible Jupyter Notebook interface
 that you can install side-by-side with the latest versions: That way, you can fearlessly
 upgrade without worrying about your classic extensions and customizations breaking.
 - [ncbi-vdb/3.2.0-gompi-2024a](https://github.com/ncbi/ncbi-vdb)
The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for
 using data in the INSDC Sequence Read Archives.
 - [ncurses/6.5-GCCcore-14.2.0](https://www.gnu.org/software/ncurses/)
The Ncurses (new curses) library is a free software emulation of curses in
 System V Release 4.0, and more. It uses Terminfo format, supports pads and
 color and multiple highlights and forms characters and function-key mapping,
 and has all the other SYSV-curses enhancements over BSD Curses.

 - [netCDF/4.9.2-gompi-2023a](https://www.unidata.ucar.edu/software/netcdf/)
NetCDF (network Common Data Form) is a set of software libraries
 and machine-independent data formats that support the creation, access, and sharing of array-oriented
 scientific data.
 - [netMHCpan/4.1b](https://www.cbs.dtu.dk/services/NetMHCpan/)
The NetMHCpan software predicts binding of peptides to any known MHC
 molecule using artificial neural networks (ANNs).
 - [netcdf4-python/1.6.4-foss-2023a](https://unidata.github.io/netcdf4-python/)
Python/numpy interface to netCDF.
 - [nettle/3.10-GCCcore-13.3.0](https://www.lysator.liu.se/~nisse/nettle/)
Nettle is a cryptographic library that is designed to fit easily
 in more or less any context: In crypto toolkits for object-oriented
 languages (C++, Python, Pike, ...), in applications like LSH or GNUPG,
 or even in kernel space.
 - [networkx/3.4.2-gfbf-2024a](https://pypi.python.org/pypi/networkx)
NetworkX is a Python package for the creation, manipulation,
and study of the structure, dynamics, and functions of complex networks.
 - [nf-core/3.3.1-foss-2024a](https://github.com/nf-core/tools)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/n/nf-core/nf-core-3.3.1-foss-2024a.eb)
Python package with helper tools for the nf-core community.
 - [nlohmann_json/3.11.3-GCCcore-13.3.0](https://github.com/nlohmann/json)
JSON for Modern C++
 - [nodejs/20.13.1-GCCcore-13.3.0](https://nodejs.org)
Node.js is a platform built on Chrome's JavaScript runtime
 for easily building fast, scalable network applications. Node.js uses an
 event-driven, non-blocking I/O model that makes it lightweight and efficient,
 perfect for data-intensive real-time applications that run across distributed devices.
 - [nsync/1.29.2-GCCcore-13.3.0](https://github.com/google/nsync)
nsync is a C library that exports various synchronization primitives, such as mutexes
 - [numactl/2.0.19-GCCcore-14.2.0](https://github.com/numactl/numactl)
The numactl program allows you to run your application program on specific
 cpu's and memory nodes. It does this by supplying a NUMA memory policy to
 the operating system before running your program. The libnuma library provides
 convenient ways for you to add NUMA memory policies into your own program.

 - [numba/0.60.0-foss-2024a](https://numba.pydata.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/n/numba/numba-0.60.0-foss-2024a.eb)
Numba is an Open Source NumPy-aware optimizing compiler for
Python sponsored by Continuum Analytics, Inc. It uses the remarkable LLVM
compiler infrastructure to compile Python syntax to machine code.
 - [numexpr/2.9.0-foss-2023a](https://numexpr.readthedocs.io/en/latest/)
The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.
 It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into
 code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a
 compiler at runtime.
 - [olmocr/0.3.9-foss-2024a](http://python.org/)
Fast, efficient, and high quality OCR powered by open visual language models
 - [openpyxl/3.1.2-GCCcore-12.3.0](https://openpyxl.readthedocs.io)
A Python library to read/write Excel 2010 xlsx/xlsm files
 - [optree/0.14.1-GCCcore-13.3.0](https://optree.readthedocs.io/en/latest/)
Optimized PyTree Utilities
 - [parallel/20230722-GCCcore-12.3.0](https://savannah.gnu.org/projects/parallel/)
parallel: Build and execute shell commands in parallel
 - [parameterized/0.9.0-GCCcore-12.3.0](https://github.com/wolever/parameterized)
Parameterized testing with any Python test framework 
 - [paramiko/3.5.1-GCCcore-13.2.0](https://paramiko.org)
Paramiko is a pure-Python (3.6+) implementation of the SSHv2 protocol,
providing both client and server functionality. It provides the foundation
for the high-level SSH library Fabric, which is what we recommend you use for
common client use-cases such as running remote shell commands or transferring
files.

 - [parasail/2.6.2-GCC-12.3.0](https://github.com/jeffdaily/parasail)
parasail is a SIMD C (C99) library containing implementations
 of the Smith-Waterman (local), Needleman-Wunsch (global), and semi-global
 pairwise sequence alignment algorithms. 
 - [patchelf/0.18.0-GCCcore-13.2.0](https://github.com/NixOS/patchelf)
PatchELF is a small utility to modify the dynamic linker and RPATH of ELF executables.
 - [phonemizer/3.2.1-gfbf-2023a](https://github.com/bootphon/phonemizer)
The phonemizer allows simple phonemization of words and texts in many languages.
Provides both the phonemize command-line tool and the Python function phonemizer.phonemize.
It is using four backends: espeak, espeak-mbrola, festival and segments.

 - [pigz/2.8-GCCcore-13.3.0](https://zlib.net/pigz/)
pigz, which stands for parallel implementation of gzip, is a fully
 functional replacement for gzip that exploits multiple processors and multiple
 cores to the hilt when compressing data. pigz was written by Mark Adler, and
 uses the zlib and pthread libraries.

 - [pixman/0.46.2-GCCcore-14.2.0](http://www.pixman.org/)
Pixman is a low-level software library for pixel manipulation, providing
 features such as image compositing and trapezoid rasterization. Important
 users of pixman are the cairo graphics library and the X server.

 - [pkgconf/2.3.0-GCCcore-14.2.0](https://github.com/pkgconf/pkgconf)
pkgconf is a program which helps to configure compiler and linker flags for development libraries.
 It is similar to pkg-config from freedesktop.org.
 - [pkgconfig/1.5.5-GCCcore-12.3.0-python](https://github.com/matze/pkgconfig)
pkgconfig is a Python module to interface with the pkg-config command line tool
 - [plotly.py/5.24.1-GCCcore-13.3.0](https://plot.ly/python)
An open-source, interactive graphing library for Python
 - [poetry/2.1.2-GCCcore-14.2.0](https://python-poetry.org)
Python packaging and dependency management made easy. Poetry helps you declare, manage and install
 dependencies of Python projects, ensuring you have the right stack everywhere.
 - [polars/0.20.2-gfbf-2023a](https://pola.rs/)
Polars is a blazingly fast DataFrame library for manipulating
structured data. The core is written in Rust and this module provides its
interface for Python.
 - [poppler/25.07.0-GCC-13.3.0](https://poppler.freedesktop.org)
Poppler is a PDF rendering library
 - [prodigal/2.6.3-GCCcore-13.3.0](https://github.com/hyattpd/Prodigal/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/p/prodigal/prodigal-2.6.3-GCCcore-13.3.0.eb)
Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)
   is a microbial (bacterial and archaeal) gene finding program developed
   at Oak Ridge National Laboratory and the University of Tennessee.
 - [prompt-toolkit/3.0.36-GCCcore-13.3.0](https://github.com/jonathanslenders/python-prompt-toolkit)
prompt_toolkit is a Python library for building powerful interactive command lines and
 terminal applications.
 - [protobuf/28.0-GCCcore-13.3.0](https://github.com/protocolbuffers/protobuf)
Protocol Buffers (a.k.a., protobuf) are Google's
language-neutral, platform-neutral, extensible mechanism for
serializing structured data.
 - [protobuf-python/5.28.0-GCCcore-13.3.0](https://github.com/google/protobuf/)
Python Protocol Buffers runtime library.
 - [psutil/7.0.0-GCCcore-14.2.0](https://github.com/giampaolo/psutil)
A cross-platform process and system utilities module for Python
 - [psycopg/3.2.3-GCCcore-13.3.0](https://psycopg.org/)
Psycopg is the most popular PostgreSQL adapter for the Python programming language.
 - [psycopg2/2.9.9-GCCcore-12.3.0](https://psycopg.org/)
Psycopg is the most popular PostgreSQL adapter for the Python programming language.
 - [py-cpuinfo/9.0.0-GCCcore-14.2.0](https://github.com/workhorsy/py-cpuinfo)
py-cpuinfo gets CPU info with pure Python.
 - [py3Dmol/2.1.0-GCCcore-12.3.0](https://github.com/3dmol/3Dmol.js/tree/master/py3Dmol)
A simple IPython/Jupyter widget to embed an interactive 3Dmol.js viewer in a notebook.
 - [pyBigWig/0.3.23-gfbf-2023b](https://github.com/deeptools/pyBigWig)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/p/pyBigWig/pyBigWig-0.3.23-gfbf-2023b.eb)
A python extension, written in C, for quick access to bigBed
files and access to and creation of bigWig files.
 - [pyEGA3/5.0.2-GCCcore-12.3.0](https://github.com/EGA-archive/ega-download-client)
A basic Python-based EGA download client 
 - [pyGAM/0.9.1-gfbf-2023a](https://pygam.readthedocs.io)
pyGAM is a package for building Generalized Additive Models in Python, with an emphasis on
modularity and performance. The API will be immediately familiar to anyone with experience of
scikit-learn or scipy.

 - [pySCENIC/0.12.1-20240311-foss-2023a](https://github.com/aertslab/pySCENIC)
pySCENIC is a lightning-fast python implementation of the SCENIC
pipeline (Single-Cell rEgulatory Network Inference and Clustering) which enables
biologists to infer transcription factors, gene regulatory networks and
cell types from single-cell RNA-seq data.
 - [pybedtools/0.12.0-foss-2024a](https://daler.github.io/pybedtools)
pybedtools wraps and extends BEDTools and offers feature-level manipulations from within Python.
 - [pybigtools/0.1.2-gfbf-2023a](https://github.com/jackh726/bigtools)
A high-performance BigWig and BigBed library in Rust.
 - [pybind11/2.13.6-GCC-14.2.0](https://pybind11.readthedocs.io)
pybind11 is a lightweight header-only library that exposes C++ types in Python and vice versa,
 mainly to create Python bindings of existing C++ code.
 - [pydantic/2.9.1-GCCcore-13.3.0](https://github.com/samuelcolvin/pydantic)
Data validation and settings management using Python type hinting.
 - [pydot/2.0.0-GCCcore-12.3.0](https://github.com/pydot/pydot)
Python interface to Graphviz's Dot language.
 - [pyfaidx/0.8.1.2-GCCcore-13.3.0](https://pypi.python.org/pypi/pyfaidx)
pyfaidx: efficient pythonic random access to fasta subsequences
 - [pyfasta/0.5.2-foss-2023a](https://pypi.org/project/pyfasta/)
fast, memory-efficient, pythonic (and command-line) access to fasta sequence files
 - [pyproj/3.6.0-GCCcore-12.3.0](https://pyproj4.github.io/pyproj)
Python interface to PROJ4 library for cartographic transformations
 - [pyro-ppl/1.9.0-foss-2023a-CUDA-12.1.1](https://github.com/pyro-ppl/pyro)
Pyro is a flexible, scalable deep probabilistic programming library built on PyTorch.
 - [pyspoa/0.2.1-GCC-13.3.0](https://github.com/nanoporetech/pyspoa)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/p/pyspoa/pyspoa-0.2.1-GCC-13.3.0.eb)
Python bindings to spoa.
 - [pytest/8.3.3-GCCcore-13.3.0](https://docs.pytest.org/en/latest/)
The pytest framework makes it easy to write small,
readable tests, and can scale to support complex functional testing for
applications and libraries.
 - [pytest-flakefinder/1.1.0-GCCcore-12.3.0](https://github.com/dropbox/pytest-flakefinder)
Runs tests multiple times to expose flakiness.
 - [pytest-rerunfailures/15.0-GCCcore-13.3.0](https://github.com/pytest-dev/pytest-rerunfailures)
pytest plugin to re-run tests to eliminate flaky failures.
 - [pytest-shard/0.1.2-GCCcore-13.3.0](https://github.com/AdamGleave/pytest-shard)
pytest plugin to support parallelism across multiple machines.

Shards tests based on a hash of their test name enabling easy parallelism across machines,
suitable for a wide variety of continuous integration services.
Tests are split at the finest level of granularity, individual test cases,
enabling parallelism even if all of your tests are in a single file
(or even single parameterized test method).

 - [pytest-subtests/0.13.1-GCCcore-13.3.0](https://github.com/pytest-dev/pytest-subtests)
unittest subTest() support and subtests fixture.
 - [pytest-workflow/2.1.0-GCCcore-13.3.0](https://github.com/LUMC/pytest-workflow)
Configure workflow/pipeline tests using yaml files.

pytest-workflow is a workflow-system agnostic testing framework that aims to make pipeline/workflow testing easy by
using YAML files for the test configuration.
Whether you write your pipelines in WDL, snakemake, nextflow, bash or any other workflow framework,
pytest-workflow makes testing easy.
pytest-workflow is build on top of the pytest test framework.
 - [pytest-xdist/3.6.1-GCCcore-13.3.0](https://github.com/pytest-dev/pytest-xdist)
xdist: pytest distributed testing plugin

The pytest-xdist plugin extends pytest with some unique test execution modes:

    * test run parallelization: if you have multiple CPUs or hosts you
      can use those for a combined test run. This allows to speed up
      development or to use special resources of remote machines.

    * --looponfail: run your tests repeatedly in a subprocess. After
        each run pytest waits until a file in your project changes and
        then re-runs the previously failing tests. This is repeated
        until all tests pass after which again a full run is
        performed.

    * Multi-Platform coverage: you can specify different Python
      interpreters or different platforms and run tests in parallel on
      all of them.

Before running tests remotely, pytest efficiently “rsyncs” your
program source code to the remote place. All test results are reported
back and displayed to your local terminal. You may specify different
Python versions and interpreters.
 - [python-igraph/0.11.4-foss-2023a](https://igraph.org/python)
Python interface to the igraph high performance graph library, primarily aimed at complex network
 research and analysis.
 - [python-isal/1.7.0-GCCcore-13.3.0](https://github.com/pycompression/python-isal)
Faster zlib and gzip compatible compression and decompression
 by providing python bindings for the isa-l library.

 - [python-parasail/1.3.4-foss-2023a](https://github.com/jeffdaily/parasail-python)
Python Bindings for the Parasail C Library
 - [quarto/1.5.57-x64](https://quarto.org/)
An open-source scientific and technical publishing system.

 - [re2c/3.1-GCCcore-13.2.0](https://re2c.org)
re2c is a free and open-source lexer generator for C and C++. Its main goal is generating
fast lexers: at least as fast as their reasonably optimized hand-coded counterparts. Instead of using
traditional table-driven approach, re2c encodes the generated finite state automata directly in the form
of conditional jumps and comparisons.
 - [redis-py/5.0.1-GCCcore-12.3.0](https://github.com/redis/redis-py)
The Python interface to the Redis key-value store.
 - [rpy2/3.5.17-foss-2024a](https://rpy2.github.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/r/rpy2/rpy2-3.5.17-foss-2024a.eb)
rpy2 is an interface to R running embedded in a Python process.
 - [ruamel.yaml/0.18.6-GCCcore-13.3.0](https://sourceforge.net/projects/ruamel-yaml)
ruamel.yaml is a YAML 1.2 loader/dumper package for Python.
 - [samblaster/0.1.26-GCC-13.2.0](https://github.com/GregoryFaust/samblaster)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/samblaster/samblaster-0.1.26-GCC-13.2.0.eb)
samblaster is a fast and flexible program for marking
 duplicates in read-id grouped1 paired-end SAM files.
 - [scArches/0.6.1-foss-2023a](https://github.com/theislab/scarches)
Single-cell architecture surgery (scArches) is a package for reference-based analysis of single-cell
 data.
 - [scCODA/0.1.9-foss-2023a](https://github.com/theislab/scCODA)
scCODA allows for identification of compositional changes in high-throughput sequencing count data,
especially cell compositions from scRNA-seq.
 - [scGPT/0.2.1-foss-2023a-CUDA-12.1.1](https://github.com/bowang-lab/scGPT/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/scGPT/scGPT-0.2.1-foss-2023a-CUDA-12.1.1.eb)
scGPT: Towards Building a Foundation Model for Single-Cell Multi-omics Using Generative AI.
 - [scVelo/0.3.1-foss-2023a](https://scvelo.org)
scVelo is a scalable toolkit for estimating and analyzing RNA velocities in single cells using
 dynamical modeling.
 - [scanpy/1.10.4-foss-2024a](https://scanpy.readthedocs.io/en/stable/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/scanpy/scanpy-1.10.4-foss-2024a.eb)
Scanpy is a scalable toolkit for analyzing single-cell gene expression data built
 jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference
 and differential expression testing. The Python-based implementation efficiently deals with
 datasets of more than one million cells.

 - [scatac_fragment_tools/0.1.0-foss-2023a](https://github.com/aertslab/scatac_fragment_tools)
Tools for working with scATAC-seq fragment files.
 - [scib/1.1.4-foss-2023a](https://github.com/theislab/scib)
Benchmarking atlas-level data integration in single-cell genomics.
 - [scib-metrics/0.5.1-foss-2023a](https://scib-metrics.readthedocs.io)
Accelerated and Python-only metrics for benchmarking single-cell integration outputs
 - [scikit-bio/0.6.0-foss-2023a](http://scikit-bio.org)
scikit-bio is an open-source, BSD-licensed Python 3 package providing data structures, algorithms
and educational resources for bioinformatics.
 - [scikit-build/0.18.1-GCCcore-14.2.0](https://scikit-build.readthedocs.io/en/latest)
Scikit-Build, or skbuild, is an improved build system generator
for CPython C/C++/Fortran/Cython extensions.
 - [scikit-build-core/0.11.1-GCCcore-14.2.0](https://scikit-build.readthedocs.io/en/latest/)
Scikit-build-core is a complete ground-up rewrite of scikit-build on top of
modern packaging APIs. It provides a bridge between CMake and the Python build
system, allowing you to make Python modules with CMake.
 - [scikit-image/0.25.0-foss-2024a](https://scikit-image.org/)
scikit-image is a collection of algorithms for image processing.
 - [scikit-learn/1.6.1-gfbf-2024a-numpy-2](https://scikit-learn.org/stable/index.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/scikit-learn/scikit-learn-1.6.1-gfbf-2024a-numpy-2.eb)
Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world,
building upon numpy, scipy, and matplotlib. As a machine-learning module,
it provides versatile tools for data mining and analysis in any field of science and engineering.
It strives to be simple and efficient, accessible to everybody, and reusable in various contexts.
 - [scikit-misc/0.3.1-foss-2023a](https://github.com/has2k1/scikit-misc)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/scikit-misc/scikit-misc-0.3.1-foss-2023a.eb)
Miscellaneous tools for data analysis and scientific computing
 - [scrublet/0.2.3-foss-2023a](https://github.com/swolock/scrublet)
Single-Cell Remover of Doublets
 - [scvi-tools/1.1.2-foss-2023a-CUDA-12.1.1](https://github.com/scverse/scvi-tools)
scvi-tools (single-cell variational inference tools) is a package for probabilistic modeling and
analysis of single-cell omics data, built on top of PyTorch and AnnData.
 - [setuptools/80.9.0-GCCcore-14.2.0](https://pypi.org/project/setuptools)
Easily download, build, install, upgrade, and uninstall Python packages
 - [setuptools-rust/1.11.0-GCCcore-14.2.0](https://github.com/PyO3/setuptools-rust)
setuptools-rust is a plugin for setuptools to build Rust Python extensions
implemented with PyO3 or rust-cpython.
 - [skani/0.2.2-GCCcore-12.3.0](https://github.com/bluenote-1577/skani)
skani
 - [smithlab/1.2-GCC-13.3.0](https://www.htslib.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/smithlab/smithlab-1.2-GCC-13.3.0.eb)
A C library for reading/writing high-throughput sequencing data.
 This package includes the utilities bgzip and tabix
 - [snakemake/8.27.0-foss-2024a](https://snakemake.readthedocs.io)
The Snakemake workflow management system is a tool to create reproducible and scalable data analyses.
 - [snappy/1.2.1-GCCcore-13.3.0](https://github.com/google/snappy)
Snappy is a compression/decompression library. It does not aim
for maximum compression, or compatibility with any other compression library;
instead, it aims for very high speeds and reasonable compression.
 - [sonic/20180202-gfbf-2023a](https://github.com/espeak-ng/sonic)
Sonic is a simple algorithm for speeding up or slowing down speech.  However,
it's optimized for speed ups of over 2X, unlike previous algorithms for changing
speech rate.  The Sonic library is a very simple ANSI C library that is designed
to easily be integrated into streaming voice applications, like TTS back ends.

 - [spams/2.6.5.4-foss-2023b](https://thoth.inrialpes.fr/people/mairal/spams/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/spams/spams-2.6.5.4-foss-2023b.eb)
SPAMS (SPArse Modeling Software) is an optimization toolbox for solving various
 sparse estimation problems.
 - [sparsehash/2.0.4-GCCcore-12.3.0](https://github.com/sparsehash/sparsehash)
An extremely memory-efficient hash_map implementation. 2 bits/entry overhead!
 The SparseHash library contains several hash-map implementations, including
 implementations that optimize for space or speed.

 - [speech_tools/2.5.0-GCCcore-12.3.0](['http://festvox.org/festival/'])
The Edinburgh Speech Tools Library is a collection of C++ class,
functions and related programs for manipulating the sorts of objects
used in speech processing. It includes support for reading and writing
waveforms, parameter files (LPC, Ceptra, F0) in various formats
and converting between them.
It also includes support for linguistic type objects and support
for various label files and ngrams (with smoothing).

 - [spektral/1.3.1-foss-2023a-CUDA-12.1.1](https://graphneural.network)
Spektral is a Python library for graph deep learning, based on the Keras API and TensorFlow 2
 - [spin/0.14-GCCcore-14.2.0](https://github.com/scientific-python/spin)
Developer tool for scientific Python libraries
 - [splitpipe/1.2.1-foss-2023b](https://support.parsebiosciences.com/hc/en-us)
splitpipe tool from Parse Biosciences. The pipeline takes FASTQ files and delivers
 processed data in the form of a cell-gene count matrix, which serves as the input for various open
 sources tools such as scanpy and seuratProcess sequencing results with our pipeline.

 A Parse Bioscience login ID is required to download
 - [spoa/4.1.0-GCC-13.3.0](https://github.com/rvaser/spoa)
Spoa (SIMD POA) is a c++ implementation of the partial order alignment (POA) algorithm
 which is used to generate consensus sequences
 - [sprocket/0.16.0]()
Sprocket is a bioinformatics workflow orchestration engine.
 - [starcode/1.4-GCC-13.2.0](https://github.com/gui11aume/starcode)
Starcode is a DNA sequence clustering software. Starcode clustering is based on all pairs
search within a specified Levenshtein distance (allowing insertions and deletions), followed by a clustering
algorithm: Message Passing, Spheres or Connected Components.
 - [statsmodels/0.14.4-gfbf-2024a-numpy-2](https://www.statsmodels.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/statsmodels/statsmodels-0.14.4-gfbf-2024a-numpy-2.eb)
Statsmodels is a Python module that allows users to explore data, estimate statistical models,
and perform statistical tests.
 - [sympy/1.13.3-gfbf-2024a-numpy-2](https://sympy.org/)
SymPy is a Python library for symbolic mathematics. It aims to
 become a full-featured computer algebra system (CAS) while keeping the code as
 simple as possible in order to be comprehensible and easily extensible. SymPy
 is written entirely in Python and does not require any external libraries.
 - [tbb/2021.11.0-GCCcore-12.3.0](https://github.com/oneapi-src/oneTBB)
Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that
 take full advantage of multicore performance, that are portable, composable and have future-proof scalability.
 - [tensorboard/2.18.0-gfbf-2024a](https://github.com/tensorflow/tensorboard)
TensorBoard is a suite of web applications for inspecting and
understanding your TensorFlow runs and graphs.
 - [tensorboardX/2.6.2.2-foss-2023a](https://github.com/lanpa/tensorboardX)
Tensorboard for PyTorch.
 - [tensorflow-probability/0.20.0-foss-2023a](https://www.tensorflow.org/probability)
TensorFlow Probability (TFP) is a library for probabilistic reasoning and statistical analysis.
 - [tensorstore/0.1.65-foss-2023a](https://github.com/google/tensorstore)
TensorStore is an open-source C++ and Python software library designed for
storage and manipulation of large multi-dimensional arrays.
 - [tlparse/0.3.37-GCCcore-13.3.0](https://github.com/ezyang/tlparse)
Parse structured PT2 (PyTorch 2) logs
 - [tmux/3.4-GCCcore-13.3.0](https://github.com/tmux/tmux/)
tmux is a terminal multiplexer: it enables a number of
terminals to be created, accessed, and controlled from a single screen. tmux
may be detached from a screen and continue running in the background, then
later reattached.
 - [tokenizers/0.15.2-GCCcore-12.3.0](https://github.com/huggingface/tokenizers)
Fast State-of-the-Art Tokenizers optimized for Research and Production
 - [torchaudio/0.12.0-foss-2022b-PyTorch-1.13.1-CUDA-11.8.0](https://github.com/pytorch/audio)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/t/torchaudio/torchaudio-0.12.0-foss-2022b-PyTorch-1.13.1-CUDA-11.8.0.eb)
Data manipulation and transformation for audio signal
processing, powered by PyTorch 
 - [torchvision/0.14.1-foss-2022b-CUDA-11.8.0](https://github.com/pytorch/vision)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/t/torchvision/torchvision-0.14.1-foss-2022b-CUDA-11.8.0.eb)
Datasets, Transforms and Models specific to Computer Vision
 - [tornado/6.4.1-GCCcore-13.3.0](https://github.com/tornadoweb/tornado)
Tornado is a Python web framework and asynchronous networking library.
 - [tqdm/4.66.5-GCCcore-13.3.0](https://github.com/tqdm/tqdm)
A fast, extensible progress bar for Python and CLI
 - [typing-extensions/4.11.0-GCCcore-13.3.0](https://github.com/python/typing_extensions)
Typing Extensions
 - [umap-learn/0.5.7-foss-2024a](https://umap-learn.readthedocs.io/en/latest/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/u/umap-learn/umap-learn-0.5.7-foss-2024a.eb)
Uniform Manifold Approximation and Projection (UMAP) is a dimension
 reduction technique that can be used for visualisation similarly to t-SNE, but also
 for general non-linear dimension reduction.

 - [unittest-xml-reporting/3.1.0-GCCcore-13.3.0](http://github.com/xmlrunner/unittest-xml-reporting)
A unittest test runner that can save test results to XML files in xUnit format.
The files can be consumed by a wide range of tools, such as build systems, IDEs and continuous integration servers.
 - [unixODBC/2.3.12-GCC-13.2.0](https://www.unixodbc.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/u/unixODBC/unixODBC-2.3.12-GCC-13.2.0.eb)
unixODBC provides a uniform interface between application and database driver
 - [utf8proc/2.9.0-GCCcore-13.2.0](https://github.com/JuliaStrings/utf8proc)
utf8proc is a small, clean C library that provides Unicode normalization, case-folding, 
and other operations for data in the UTF-8 encoding.
 - [util-linux/2.41-GCCcore-14.2.0](https://www.kernel.org/pub/linux/utils/util-linux)
Set of Linux utilities
 - [uv/0.6.4](https://docs.astral.sh/uv/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/u/uv/uv-0.6.4.eb)
A single tool to replace pip, pip-tools, pipx, poetry, pyenv, twine, virtualenv, and more.

 - [versioningit/3.1.2-GCCcore-13.3.0](https://github.com/jwodder/versioningit)
versioningit is yet another Python packaging plugin for automatically determining your
package’s version based on your version control repository’s tags.
Unlike others, it allows easy customization of the version format and even lets you easily override
the separate functions used for version extraction & calculation.
 - [virtualenv/20.29.2-GCCcore-14.2.0](https://github.com/pypa/virtualenv)
A tool for creating isolated virtual python environments.
 - [wandb/0.16.1-GCC-12.3.0](https://wandb.ai)
CLI and Python API for Weights and Biases (wandb), a tool for visualizing and
tracking your machine learning experiments.
 - [wget/1.25.0-GCCcore-13.3.0](https://www.gnu.org/software/wget)
GNU Wget is a free software package for retrieving files using HTTP, HTTPS and FTP,
 the most widely-used Internet protocols. It is a non-interactive commandline tool,
 so it may easily be called from scripts, cron jobs, terminals without X-Windows support, etc.
 - [wrapt/1.17.2-GCCcore-14.2.0](https://pypi.org/project/wrapt/)
The aim of the wrapt module is to provide a transparent object
proxy for Python, which can be used as the basis for the construction of
function wrappers and decorator functions.
 - [x264/20240513-GCCcore-13.3.0](https://www.videolan.org/developers/x264.html)
x264 is a free software library and application for encoding video streams
 into the H.264/MPEG-4 AVC compression format, and is released under the
 terms of the GNU GPL.

 - [x265/3.6-GCCcore-13.3.0](https://x265.org/)
x265 is a free software library and application for encoding video streams
 into the H.265 AVC compression format, and is released under the terms of
 the GNU GPL.

 - [xarray/2023.9.0-gfbf-2023a](https://github.com/pydata/xarray)
xarray (formerly xray) is an open source project and Python package that aims to bring
 the labeled data power of pandas to the physical sciences, by providing N-dimensional variants of the
 core pandas data structures.
 - [xorg-macros/1.20.2-GCCcore-14.2.0](https://gitlab.freedesktop.org/xorg/util/macros)
X.org macros utilities.
 - [xxd/9.1.0307-GCCcore-13.2.0](https://www.vim.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/x/xxd/xxd-9.1.0307-GCCcore-13.2.0.eb)
xxd is part of the VIM package and this will only install xxd, not vim!
xxd converts to/from hexdumps of binary files.
 - [yaml-cpp/0.8.0-GCCcore-13.2.0](https://github.com/jbeder/yaml-cpp)
yaml-cpp is a YAML parser and emitter in C++ matching the YAML 1.2 spec
 - [zarr/3.0.10-foss-2025a](https://zarr.readthedocs.io/en/stable/)
Zarr is a Python package providing an implementation of compressed,
chunked, N-dimensional arrays, designed for use in parallel computing.
 - [zlib/1.3.1-GCCcore-13.3.0](https://www.zlib.net/)
zlib is designed to be a free, general-purpose, legally unencumbered -- that is,
 not covered by any patents -- lossless data-compression library for use on virtually any
 computer hardware and operating system.
 - [zlib-ng/2.2.1-GCCcore-13.3.0](https://github.com/zlib-ng/zlib-ng)
zlib data compression library for the next generation systems
 - [zstd/1.5.6-GCCcore-13.3.0](https://facebook.github.io/zstd)
Zstandard is a real-time compression algorithm, providing high compression ratios.
 It offers a very wide range of compression/speed trade-off, while being backed by a very fast decoder.
 It also offers a special mode for small data, called dictionary compression, and can create dictionaries
 from any sample set.
