---
title: chorus Bio Modules 24.04
layout: single
permalink: /sw_inventory/chorus-bio-modules-24.04/
created: 2025-09-28
toc: true
toc_label: "On This Page"
sidebar:
  nav: "docs"
---

 - [AGAT/1.4.0-GCC-12.3.0](AGAT: Another GTF/GFF Analysis Toolkit. Suite of tools to handle gene annotations
 in any GTF/GFF format.)
AGAT: Another GTF/GFF Analysis Toolkit. Suite of tools to handle gene annotations
 in any GTF/GFF format.
 - [AlphaFold/2.3.2-foss-2023a-CUDA-12.1.1](AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known)
AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known
 - [AlphaPulldown/2.0.3-foss-2023a](AlphaPulldown is a Python package that streamlines protein-protein
interaction screens and high-throughput modelling of higher-order oligomers using AlphaFold-Multimer)
AlphaPulldown is a Python package that streamlines protein-protein
interaction screens and high-throughput modelling of higher-order oligomers using AlphaFold-Multimer
 - [Arriba/2.4.0-GCC-12.2.0](Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data.
 It was developed for the use in a clinical research setting. Therefore, short runtimes and high
 sensitivity were important design criteria.)
Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data.
 It was developed for the use in a clinical research setting. Therefore, short runtimes and high
 sensitivity were important design criteria.
 - [ArviZ/0.16.1-foss-2023a](Exploratory analysis of Bayesian models with Python)
Exploratory analysis of Bayesian models with Python
 - [BAli-Phy/4.0-beta8-gfbf-2022b](BAli-Phy estimates multiple sequence alignments and evolutionary trees from DNA, amino acid, 
or codon sequences.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/BAli-Phy/BAli-Phy-4.0-beta8-gfbf-2022b.eb)
BAli-Phy estimates multiple sequence alignments and evolutionary trees from DNA, amino acid, 
or codon sequences.
 - [BCFtools/1.19-GCC-13.2.0](Samtools is a suite of programs for interacting with high-throughput sequencing data.
 BCFtools)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/BCFtools/BCFtools-1.19-GCC-13.2.0.eb)
Samtools is a suite of programs for interacting with high-throughput sequencing data.
 BCFtools
 - [BEDTools/2.31.1-GCC-13.3.0](BEDTools: a powerful toolset for genome arithmetic.
The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and
computing coverage.
The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM.)
BEDTools: a powerful toolset for genome arithmetic.
The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and
computing coverage.
The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM.
 - [BLAST+/2.16.0-gompi-2024a](Basic Local Alignment Search Tool, or BLAST, is an algorithm
 for comparing primary biological sequence information, such as the amino-acid
 sequences of different proteins or the nucleotides of DNA sequences.)
Basic Local Alignment Search Tool, or BLAST, is an algorithm
 for comparing primary biological sequence information, such as the amino-acid
 sequences of different proteins or the nucleotides of DNA sequences.
 - [BPCells/0.3.0-foss-2024a](Single-cell transcriptome sequencing (sc-RNA-seq) experiments
 allow us to discover new cell types and help us understand how they arise in
 development. The Monocle 3 package provides a toolkit for analyzing
 single-cell gene expression experiments.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/BPCells/BPCells-0.3.0-foss-2024a.eb)
Single-cell transcriptome sequencing (sc-RNA-seq) experiments
 allow us to discover new cell types and help us understand how they arise in
 development. The Monocle 3 package provides a toolkit for analyzing
 single-cell gene expression experiments.
 - [BWA/0.7.18-GCCcore-13.3.0](Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively
 short nucleotide sequences against a long reference sequence such as the human
 genome.
)
Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively
 short nucleotide sequences against a long reference sequence such as the human
 genome.

 - [BamTools/2.5.2-GCC-13.3.0](BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.)
BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.
 - [Beast/10.5.0-beta3-GCC-12.3.0-beagle-lib-4.0.1-CUDA-12.1.1](BEAST is a cross-platform program for Bayesian analysis of molecular sequences
 using MCMC. It is entirely orientated towards rooted, time-measured phylogenies inferred using
 strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies
 but is also a framework for testing evolutionary hypotheses without conditioning on a single
 tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted
 proportional to its posterior probability. )
BEAST is a cross-platform program for Bayesian analysis of molecular sequences
 using MCMC. It is entirely orientated towards rooted, time-measured phylogenies inferred using
 strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies
 but is also a framework for testing evolutionary hypotheses without conditioning on a single
 tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted
 proportional to its posterior probability. 
 - [Beast2/2.7.7-GCC-12.3.0-beagle-lib-4.0.1-CUDA-12.1.1](BEAST is a cross-platform program for Bayesian MCMC analysis of molecular 
 sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using 
 strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies 
 but is also a framework for testing evolutionary hypotheses without conditioning on a single 
 tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted 
 proportional to its posterior probability. )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/Beast2/Beast2-2.7.7-GCC-12.3.0-beagle-lib-4.0.1-CUDA-12.1.1.eb)
BEAST is a cross-platform program for Bayesian MCMC analysis of molecular 
 sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using 
 strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies 
 but is also a framework for testing evolutionary hypotheses without conditioning on a single 
 tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted 
 proportional to its posterior probability. 
 - [BindCraft/20250520-foss-2023a-CUDA-12.1.1](Simple binder design pipeline using AlphaFold2 backpropagation, MPNN, and PyRosetta.
 Select your target and let the script do the rest of the work and finish once you have enough designs to order!)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/BindCraft/BindCraft-20250520-foss-2023a-CUDA-12.1.1.eb)
Simple binder design pipeline using AlphaFold2 backpropagation, MPNN, and PyRosetta.
 Select your target and let the script do the rest of the work and finish once you have enough designs to order!
 - [Bio-DB-HTS/3.01-GCC-13.3.0](Read files using HTSlib including BAM/CRAM, Tabix and BCF database files)
Read files using HTSlib including BAM/CRAM, Tabix and BCF database files
 - [BioPerl/1.7.8-GCCcore-13.3.0](Bioperl is the product of a community effort to produce Perl code which is useful in biology.
 Examples include Sequence objects, Alignment objects and database searching objects.)
Bioperl is the product of a community effort to produce Perl code which is useful in biology.
 Examples include Sequence objects, Alignment objects and database searching objects.
 - [Biopython/1.84-foss-2023b](Biopython is a set of freely available tools for biological
 computation written in Python by an international team of developers. It is
 a distributed collaborative effort to develop Python libraries and
 applications which address the needs of current and future work in
 bioinformatics. )
Biopython is a set of freely available tools for biological
 computation written in Python by an international team of developers. It is
 a distributed collaborative effort to develop Python libraries and
 applications which address the needs of current and future work in
 bioinformatics. 
 - [Bismark/0.24.1-GCC-12.2.0](A tool to map bisulfite converted sequence reads and determine cytosine methylation states)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/Bismark/Bismark-0.24.1-GCC-12.2.0.eb)
A tool to map bisulfite converted sequence reads and determine cytosine methylation states
 - [Bowtie2/2.5.4-GCC-13.2.0](Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads
 to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s
 of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.
 Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,
 its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/Bowtie2/Bowtie2-2.5.4-GCC-13.2.0.eb)
Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads
 to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s
 of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.
 Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,
 its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.
 - [CITE-seq-Count/1.4.4-foss-2023b-Python-3.11.5](A python package that allows to count antibody TAGS from a CITE-seq and/or cell hashing experiment.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/CITE-seq-Count/CITE-seq-Count-1.4.4-foss-2023b-Python-3.11.5.eb)
A python package that allows to count antibody TAGS from a CITE-seq and/or cell hashing experiment.
 - [CRISPResso2/2.3.1-foss-2023b](CRISPResso2 is a software pipeline designed to enable rapid and
intuitive interpretation of genome editing experiments.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/CRISPResso2/CRISPResso2-2.3.1-foss-2023b.eb)
CRISPResso2 is a software pipeline designed to enable rapid and
intuitive interpretation of genome editing experiments.
 - [Cassiopeia/2.0.0-foss-2023a](A Package for Cas9-Enabled Single Cell Lineage Tracing Tree Reconstruction.)
A Package for Cas9-Enabled Single Cell Lineage Tracing Tree Reconstruction.
 - [Cbc/2.10.12-foss-2024a](Cbc (Coin-or branch and cut) is an open-source mixed integer linear programming
solver written in C++. It can be used as a callable library or using a
stand-alone executable.)
Cbc (Coin-or branch and cut) is an open-source mixed integer linear programming
solver written in C++. It can be used as a callable library or using a
stand-alone executable.
 - [CellBender/0.3.0-foss-2023a](CellBender is a software package for eliminating technical artifacts from
high-throughput single-cell RNA sequencing (scRNA-seq) data.
)
CellBender is a software package for eliminating technical artifacts from
high-throughput single-cell RNA sequencing (scRNA-seq) data.

 - [CellRanger/8.0.0](Cell Ranger is a set of analysis pipelines that process Chromium
 single-cell RNA-seq output to align reads, generate gene-cell matrices and perform
 clustering and gene expression analysis.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/CellRanger/CellRanger-8.0.0.eb)
Cell Ranger is a set of analysis pipelines that process Chromium
 single-cell RNA-seq output to align reads, generate gene-cell matrices and perform
 clustering and gene expression analysis.
 - [CellRanger-ATAC/2.1.0](Cell Ranger ATAC is a set of analysis pipelines that process 
Chromium Single Cell ATAC data.)
Cell Ranger ATAC is a set of analysis pipelines that process 
Chromium Single Cell ATAC data.
 - [CellRank/2.0.2-foss-2023a-CUDA-12.1.1](CellRank is a toolkit to uncover cellular dynamics based on
 Markov state modeling of single-cell data.  It contains two main modules:
kernels compute cell-cell transition probabilities and estimators generate
hypothesis based on these. )
CellRank is a toolkit to uncover cellular dynamics based on
 Markov state modeling of single-cell data.  It contains two main modules:
kernels compute cell-cell transition probabilities and estimators generate
hypothesis based on these. 
 - [CellTypist/1.6.2-foss-2023a](A tool for semi-automatic cell type annotation)
A tool for semi-automatic cell type annotation
 - [Cgl/0.60.8-foss-2023a](The COIN-OR Cut Generation Library (Cgl) is a collection of cut generators that
can be used with other COIN-OR packages that make use of cuts, such as, among
others, the linear solver Clp or the mixed integer linear programming solvers
Cbc or BCP. Cgl uses the abstract class OsiSolverInterface (see Osi) to use or
communicate with a solver. It does not directly call a solver.)
The COIN-OR Cut Generation Library (Cgl) is a collection of cut generators that
can be used with other COIN-OR packages that make use of cuts, such as, among
others, the linear solver Clp or the mixed integer linear programming solvers
Cbc or BCP. Cgl uses the abstract class OsiSolverInterface (see Osi) to use or
communicate with a solver. It does not directly call a solver.
 - [CheckM2/1.1.0-foss-2024a](Assessing the quality of metagenome-derived genome bins using machine learning)
Assessing the quality of metagenome-derived genome bins using machine learning
 - [Clp/1.17.10-foss-2024a](Clp (Coin-or linear programming) is an open-source linear programming solver.
It is primarily meant to be used as a callable library, but a basic,
stand-alone executable version is also available.)
Clp (Coin-or linear programming) is an open-source linear programming solver.
It is primarily meant to be used as a callable library, but a basic,
stand-alone executable version is also available.
 - [Cogent_NGS_Immune_Profiler/v2.0-foss-2024a](Cogent NGS Immune Profiler (CogentIP) is software designed to analyze sequence
 data stored in FASTQ files generated by Illumina sequencers from libraries prepared using certain
 Takara Bio immune profiling kits. )
Cogent NGS Immune Profiler (CogentIP) is software designed to analyze sequence
 data stored in FASTQ files generated by Illumina sequencers from libraries prepared using certain
 Takara Bio immune profiling kits. 
 - [CoinUtils/2.11.12-GCC-13.3.0](CoinUtils (Coin-OR Utilities) is an open-source collection of classes and
functions that are generally useful to more than one COIN-OR project.)
CoinUtils (Coin-OR Utilities) is an open-source collection of classes and
functions that are generally useful to more than one COIN-OR project.
 - [CrossMap/0.7.3-foss-2023b](CrossMap is a program for genome coordinates conversion
 between different assemblies (such as hg18 (NCBI36) <=> hg19 (GRCh37)).
 It supports commonly used file formats including BAM, CRAM, SAM, Wiggle,
 BigWig, BED, GFF, GTF and VCF.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/CrossMap/CrossMap-0.7.3-foss-2023b.eb)
CrossMap is a program for genome coordinates conversion
 between different assemblies (such as hg18 (NCBI36) <=> hg19 (GRCh37)).
 It supports commonly used file formats including BAM, CRAM, SAM, Wiggle,
 BigWig, BED, GFF, GTF and VCF.
 - [DIAMOND/2.1.11-GCC-13.3.0](Accelerated BLAST compatible local sequence aligner)
Accelerated BLAST compatible local sequence aligner
 - [DendroPy/4.6.1-GCCcore-12.3.0](A Python library for phylogenetics and phylogenetic computing: 
reading, writing, simulation, processing and manipulation of phylogenetic trees 
(phylogenies) and characters.)
A Python library for phylogenetics and phylogenetic computing: 
reading, writing, simulation, processing and manipulation of phylogenetic trees 
(phylogenies) and characters.
 - [EPA-ng/0.3.8-GCC-12.3.0](EPA-ng)
EPA-ng
 - [ESM-2/2.0.0-foss-2023a-CUDA-12.1.1](ESM-2 outperforms all tested single-sequence protein language models
 across a range of structure prediction tasks. ESMFold harnesses the ESM-2 language model to generate
 accurate structure predictions end to end directly from the sequence of a protein.)
ESM-2 outperforms all tested single-sequence protein language models
 across a range of structure prediction tasks. ESMFold harnesses the ESM-2 language model to generate
 accurate structure predictions end to end directly from the sequence of a protein.
 - [Eigen/3.4.0-GCCcore-13.2.0](Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers,
 and related algorithms.)
Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers,
 and related algorithms.
 - [FASTA/36.3.8i-GCC-12.2.0](The FASTA programs find regions of local or global (new) similarity between 
protein or DNA sequences, either by searching Protein or DNA databases, or by identifying 
local duplications within a sequence.)
The FASTA programs find regions of local or global (new) similarity between 
protein or DNA sequences, either by searching Protein or DNA databases, or by identifying 
local duplications within a sequence.
 - [FLAIR/2.0-foss-2023a](FLAIR (Full-Length Alternative Isoform analysis of RNA)
for the correction, isoform definition, and alternative splicing analysis of noisy reads.
FLAIR has primarily been used for nanopore cDNA, native RNA, and PacBio sequencing reads.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/f/FLAIR/FLAIR-2.0-foss-2023a.eb)
FLAIR (Full-Length Alternative Isoform analysis of RNA)
for the correction, isoform definition, and alternative splicing analysis of noisy reads.
FLAIR has primarily been used for nanopore cDNA, native RNA, and PacBio sequencing reads.
 - [FSL/6.0.7.17](FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/f/FSL/FSL-6.0.7.17.eb)
FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data.
 - [FastQC/0.12.1-Java-11](FastQC is a quality control application for high throughput
sequence data. It reads in sequence data in a variety of formats and can either
provide an interactive application to review the results of several different
QC checks, or create an HTML based report which can be integrated into a
pipeline.)
FastQC is a quality control application for high throughput
sequence data. It reads in sequence data in a variety of formats and can either
provide an interactive application to review the results of several different
QC checks, or create an HTML based report which can be integrated into a
pipeline.
 - [FastTree/2.1.11-GCCcore-12.3.0](FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide
 or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of
 time and memory. )
FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide
 or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of
 time and memory. 
 - [Flax/0.8.4-gfbf-2023a](Flax is a high-performance neural network library and ecosystem for JAX that is
designed for flexibility: Try new forms of training by forking an example and
by modifying the training loop, not by adding features to a framework.)
Flax is a high-performance neural network library and ecosystem for JAX that is
designed for flexibility: Try new forms of training by forking an example and
by modifying the training loop, not by adding features to a framework.
 - [GATK/4.4.0.0-GCCcore-12.2.0-Java-17](The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute
 to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,
 with a primary focus on variant discovery and genotyping as well as strong emphasis on
 data quality assurance. Its robust architecture, powerful processing engine and
 high-performance computing features make it capable of taking on projects of any size.)
The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute
 to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,
 with a primary focus on variant discovery and genotyping as well as strong emphasis on
 data quality assurance. Its robust architecture, powerful processing engine and
 high-performance computing features make it capable of taking on projects of any size.
 - [GEOS/3.12.2-GCC-13.3.0](GEOS (Geometry Engine)
GEOS (Geometry Engine
 - [GMP/6.3.0-GCCcore-13.2.0](GMP is a free library for arbitrary precision arithmetic, operating on signed
 integers, rational numbers, and floating point numbers.
)
GMP is a free library for arbitrary precision arithmetic, operating on signed
 integers, rational numbers, and floating point numbers.

 - [GROMACS/2024.4-foss-2023b-CUDA-12.4.0](GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the
Newtonian equations of motion for systems with hundreds to millions of
particles.

This is a GPU enabled build, containing both MPI and threadMPI binaries.

It also contains the gmxapi extension for the single precision MPI build.
)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/g/GROMACS/GROMACS-2024.4-foss-2023b-CUDA-12.4.0.eb)
GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the
Newtonian equations of motion for systems with hundreds to millions of
particles.

This is a GPU enabled build, containing both MPI and threadMPI binaries.

It also contains the gmxapi extension for the single precision MPI build.

 - [HH-suite/3.3.0-gompi-2023a](The HH-suite is an open-source software package
 for sensitive protein sequence searching based on the pairwise
 alignment of hidden Markov models (HMMs).)
The HH-suite is an open-source software package
 for sensitive protein sequence searching based on the pairwise
 alignment of hidden Markov models (HMMs).
 - [HMMER/3.4-gompi-2023a](HMMER is used for searching sequence databases for homologs
 of protein sequences, and for making protein sequence alignments. It
 implements methods using probabilistic models called profile hidden Markov
 models (profile HMMs).  Compared to BLAST, FASTA, and other sequence
 alignment and database search tools based on older scoring methodology,
 HMMER aims to be significantly more accurate and more able to detect remote
 homologs because of the strength of its underlying mathematical models. In the
 past, this strength came at significant computational expense, but in the new
 HMMER3 project, HMMER is now essentially as fast as BLAST.)
HMMER is used for searching sequence databases for homologs
 of protein sequences, and for making protein sequence alignments. It
 implements methods using probabilistic models called profile hidden Markov
 models (profile HMMs).  Compared to BLAST, FASTA, and other sequence
 alignment and database search tools based on older scoring methodology,
 HMMER aims to be significantly more accurate and more able to detect remote
 homologs because of the strength of its underlying mathematical models. In the
 past, this strength came at significant computational expense, but in the new
 HMMER3 project, HMMER is now essentially as fast as BLAST.
 - [HTSlib/1.21-GCC-13.3.0](A C library for reading/writing high-throughput sequencing data.
 This package includes the utilities bgzip and tabix)
A C library for reading/writing high-throughput sequencing data.
 This package includes the utilities bgzip and tabix
 - [IgBLAST/1.22.0-x64-linux](IgBLAST faclilitates the analysis of immunoglobulin and T cell receptor variable domain sequences.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/i/IgBLAST/IgBLAST-1.22.0-x64-linux.eb)
IgBLAST faclilitates the analysis of immunoglobulin and T cell receptor variable domain sequences.
 - [InChI/1.07.1-GCC-13.3.0](The IUPAC International Chemical Identifier (InChI TM) is a non-proprietary
identifier for chemical substances that can be used in printed and electronic
data sources thus enabling easier linking of diverse data compilations.)
The IUPAC International Chemical Identifier (InChI TM) is a non-proprietary
identifier for chemical substances that can be used in printed and electronic
data sources thus enabling easier linking of diverse data compilations.
 - [Infernal/1.1.4-foss-2022b](Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases
 for RNA structure and sequence similarities.)
Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases
 for RNA structure and sequence similarities.
 - [JAGS/4.3.2-foss-2024a](JAGS is Just Another Gibbs Sampler.  It is a program for analysis
 of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  )
JAGS is Just Another Gibbs Sampler.  It is a program for analysis
 of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  
 - [Kalign/3.4.0-GCCcore-12.3.0](Kalign is a fast multiple sequence alignment program for biological sequences.)
Kalign is a fast multiple sequence alignment program for biological sequences.
 - [Kent_tools/479-GCC-13.3.0](Kent utilities: collection of tools used by the UCSC genome browser.)
Kent utilities: collection of tools used by the UCSC genome browser.
 - [Kraken2/2.1.3-gompi-2022b](Kraken is a system for assigning taxonomic labels to short DNA sequences,
 usually obtained through metagenomic studies. Previous attempts by other
 bioinformatics software to accomplish this task have often used sequence
 alignment or machine learning techniques that were quite slow, leading to
 the development of less sensitive but much faster abundance estimation
 programs. Kraken aims to achieve high sensitivity and high speed by
 utilizing exact alignments of k-mers and a novel classification algorithm.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/k/Kraken2/Kraken2-2.1.3-gompi-2022b.eb)
Kraken is a system for assigning taxonomic labels to short DNA sequences,
 usually obtained through metagenomic studies. Previous attempts by other
 bioinformatics software to accomplish this task have often used sequence
 alignment or machine learning techniques that were quite slow, leading to
 the development of less sensitive but much faster abundance estimation
 programs. Kraken aims to achieve high sensitivity and high speed by
 utilizing exact alignments of k-mers and a novel classification algorithm.
 - [LightGBM/4.6.0-foss-2024a](A fast, distributed, high performance gradient boosting (GBT, GBDT, GBRT, GBM
or MART) framework based on decision tree algorithms, used for ranking,
classification and many other machine learning tasks.)
A fast, distributed, high performance gradient boosting (GBT, GBDT, GBRT, GBM
or MART) framework based on decision tree algorithms, used for ranking,
classification and many other machine learning tasks.
 - [Longshot/1.0.0-GCCcore-13.2.0](Longshot is a variant calling tool for diploid genomes using long error prone reads such as Pacific
 Biosciences (PacBio) SMRT and Oxford Nanopore Technologies (ONT). It takes as input an aligned BAM file and outputs
 a phased VCF file with variants and haplotype information. It can also output haplotype-separated BAM files that can
 be used for downstream analysis. Currently, it only calls single nucleotide variants (SNVs).)
Longshot is a variant calling tool for diploid genomes using long error prone reads such as Pacific
 Biosciences (PacBio) SMRT and Oxford Nanopore Technologies (ONT). It takes as input an aligned BAM file and outputs
 a phased VCF file with variants and haplotype information. It can also output haplotype-separated BAM files that can
 be used for downstream analysis. Currently, it only calls single nucleotide variants (SNVs).
 - [MACS2/2.2.9.1-foss-2023a](Model Based Analysis for ChIP-Seq data)
Model Based Analysis for ChIP-Seq data
 - [MACS3/3.0.1-gfbf-2023a](Model Based Analysis for ChIP-Seq data)
Model Based Analysis for ChIP-Seq data
 - [MAFFT/7.520-GCC-12.3.0-with-extensions](MAFFT is a multiple sequence alignment program for unix-like operating systems.
It offers a range of multiple alignment methods, L-INS-i (accurate; for alignment
of <∼200 sequences), FFT-NS-2 (fast; for alignment of <∼30,000 sequences), etc.)
MAFFT is a multiple sequence alignment program for unix-like operating systems.
It offers a range of multiple alignment methods, L-INS-i (accurate; for alignment
of <∼200 sequences), FFT-NS-2 (fast; for alignment of <∼30,000 sequences), etc.
 - [MCR/R2023b.9](The MATLAB Runtime is a standalone set of shared libraries
 that enables the execution of compiled MATLAB applications
 or components on computers that do not have MATLAB installed.)
The MATLAB Runtime is a standalone set of shared libraries
 that enables the execution of compiled MATLAB applications
 or components on computers that do not have MATLAB installed.
 - [METIS/5.1.0-GCCcore-12.3.0](METIS is a set of serial programs for partitioning graphs, partitioning
 finite element meshes, and producing fill reducing orderings for sparse
 matrices. The algorithms implemented in METIS are based on the multilevel
 recursive-bisection, multilevel k-way, and multi-constraint partitioning
 schemes.
)
METIS is a set of serial programs for partitioning graphs, partitioning
 finite element meshes, and producing fill reducing orderings for sparse
 matrices. The algorithms implemented in METIS are based on the multilevel
 recursive-bisection, multilevel k-way, and multi-constraint partitioning
 schemes.

 - [MPC/1.3.1-GCCcore-13.2.0](Gnu Mpc is a C library for the arithmetic of
 complex numbers with arbitrarily high precision and correct
 rounding of the result. It extends the principles of the IEEE-754
 standard for fixed precision real floating point numbers to
 complex numbers, providing well-defined semantics for every
 operation. At the same time, speed of operation at high precision
 is a major design goal.)
Gnu Mpc is a C library for the arithmetic of
 complex numbers with arbitrarily high precision and correct
 rounding of the result. It extends the principles of the IEEE-754
 standard for fixed precision real floating point numbers to
 complex numbers, providing well-defined semantics for every
 operation. At the same time, speed of operation at high precision
 is a major design goal.
 - [MPFR/4.2.1-GCCcore-13.2.0](The MPFR library is a C library for multiple-precision floating-point
 computations with correct rounding.
)
The MPFR library is a C library for multiple-precision floating-point
 computations with correct rounding.

 - [MRtrix/3.0.5-foss-2023b](MRtrix provides a set of tools to perform diffusion-weighted MR
 white-matter tractography in a manner robust to crossing fibres, using
 constrained spherical deconvolution (CSD) and probabilistic streamlines.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/MRtrix/MRtrix-3.0.5-foss-2023b.eb)
MRtrix provides a set of tools to perform diffusion-weighted MR
 white-matter tractography in a manner robust to crossing fibres, using
 constrained spherical deconvolution (CSD) and probabilistic streamlines.
 - [MUMPS/5.7.2-foss-2024a-metis](A parallel sparse direct solver)
A parallel sparse direct solver
 - [MUMmer/4.0.0rc1-GCCcore-12.3.0](MUMmer is a system for rapidly aligning entire genomes,
 whether in complete or draft form. AMOS makes use of it.
)
MUMmer is a system for rapidly aligning entire genomes,
 whether in complete or draft form. AMOS makes use of it.

 - [MUSCLE/5.1.0-GCCcore-12.3.0](MUSCLE is one of the best-performing multiple alignment programs
 according to published benchmark tests, with accuracy and speed that are consistently
 better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users
 learn everything they need to know about MUSCLE in a few minutes-only a handful of
 command-line options are needed to perform common alignment tasks.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/MUSCLE/MUSCLE-5.1.0-GCCcore-12.3.0.eb)
MUSCLE is one of the best-performing multiple alignment programs
 according to published benchmark tests, with accuracy and speed that are consistently
 better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users
 learn everything they need to know about MUSCLE in a few minutes-only a handful of
 command-line options are needed to perform common alignment tasks.
 - [Monocle3/1.4.26-foss-2024a](Single-cell transcriptome sequencing (sc-RNA-seq) experiments allow us to discover new cell types and help us understand how they arise in development. The Monocle 3 package provides a toolkit for analyzing single-cell gene expression experiments.)
Single-cell transcriptome sequencing (sc-RNA-seq) experiments allow us to discover new cell types and help us understand how they arise in development. The Monocle 3 package provides a toolkit for analyzing single-cell gene expression experiments.
 - [MultiQC/1.21-foss-2023a](Aggregate results from bioinformatics analyses across many samples into a single 
 report.

 MultiQC searches a given directory for analysis logs and compiles a HTML report. It's a general
 use tool, perfect for summarising the output from numerous bioinformatics tools.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/MultiQC/MultiQC-1.21-foss-2023a.eb)
Aggregate results from bioinformatics analyses across many samples into a single 
 report.

 MultiQC searches a given directory for analysis logs and compiles a HTML report. It's a general
 use tool, perfect for summarising the output from numerous bioinformatics tools.
 - [NextPolish/1.4.1-GCC-13.3.0-2025-01-08](NextDenovo is a string graph-based de novo assembler for long reads.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/n/NextPolish/NextPolish-1.4.1-GCC-13.3.0-2025-01-08.eb)
NextDenovo is a string graph-based de novo assembler for long reads.
 - [OpenMM/8.0.0-foss-2022b-CUDA-11.8.0](OpenMM is a toolkit for molecular simulation.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/o/OpenMM/OpenMM-8.0.0-foss-2022b-CUDA-11.8.0.eb)
OpenMM is a toolkit for molecular simulation.
 - [Osi/0.108.11-GCC-13.3.0](Osi (Open Solver Interface) provides an abstract base class to a generic linear
programming (LP) solver, along with derived classes for specific solvers. Many
applications may be able to use the Osi to insulate themselves from a specific
LP solver. That is, programs written to the OSI standard may be linked to any
solver with an OSI interface and should produce correct results. The OSI has
been significantly extended compared to its first incarnation. Currently, the
OSI supports linear programming solvers and has rudimentary support for integer
programming.)
Osi (Open Solver Interface) provides an abstract base class to a generic linear
programming (LP) solver, along with derived classes for specific solvers. Many
applications may be able to use the Osi to insulate themselves from a specific
LP solver. That is, programs written to the OSI standard may be linked to any
solver with an OSI interface and should produce correct results. The OSI has
been significantly extended compared to its first incarnation. Currently, the
OSI supports linear programming solvers and has rudimentary support for integer
programming.
 - [PICRUSt2/2.5.2-foss-2023a](PICRUSt2 (Phylogenetic Investigation of Communities by
 Reconstruction of Unobserved States) is a software for predicting
 functional abundances based only on marker gene sequences.)
PICRUSt2 (Phylogenetic Investigation of Communities by
 Reconstruction of Unobserved States) is a software for predicting
 functional abundances based only on marker gene sequences.
 - [ParMETIS/4.0.3-gompi-2023a](ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning
 unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the
 functionality provided by METIS and includes routines that are especially suited for parallel AMR computations and
 large scale numerical simulations. The algorithms implemented in ParMETIS are based on the parallel multilevel k-way
 graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning schemes.)
ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning
 unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the
 functionality provided by METIS and includes routines that are especially suited for parallel AMR computations and
 large scale numerical simulations. The algorithms implemented in ParMETIS are based on the parallel multilevel k-way
 graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning schemes.
 - [Porechop/0.2.4-GCCcore-12.3.0](Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.
 Adapters on the ends of reads are trimmed off, and when a read has an adapter in its middle,
 it is treated as chimeric and chopped into separate reads. Porechop performs thorough alignments
 to effectively find adapters, even at low sequence identity)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/p/Porechop/Porechop-0.2.4-GCCcore-12.3.0.eb)
Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.
 Adapters on the ends of reads are trimmed off, and when a read has an adapter in its middle,
 it is treated as chimeric and chopped into separate reads. Porechop performs thorough alignments
 to effectively find adapters, even at low sequence identity
 - [PyRosetta/4.387-gompi-2023a](PyRosetta is an interactive Python-based interface to the powerful Rosetta molecular modeling
suite. It enables users to design their own custom molecular modeling algorithms using Rosetta
sampling methods and energy functions.
)
PyRosetta is an interactive Python-based interface to the powerful Rosetta molecular modeling
suite. It enables users to design their own custom molecular modeling algorithms using Rosetta
sampling methods and energy functions.

 - [PyTorch/2.6.0-foss-2024a-CUDA-12.6.0](Tensors and Dynamic neural networks in Python with strong GPU acceleration.
PyTorch is a deep learning framework that puts Python first.)
Tensors and Dynamic neural networks in Python with strong GPU acceleration.
PyTorch is a deep learning framework that puts Python first.
 - [PyTorch-bundle/2.1.2-foss-2023a-CUDA-12.1.1](PyTorch with compatible versions of official Torch extensions.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/p/PyTorch-bundle/PyTorch-bundle-2.1.2-foss-2023a-CUDA-12.1.1.eb)
PyTorch with compatible versions of official Torch extensions.
 - [Pyomo/6.7.3-foss-2023a](Pyomo is a Python-based open-source software package that supports a diverse set of optimization
 capabilities for formulating and analyzing optimization models. )
Pyomo is a Python-based open-source software package that supports a diverse set of optimization
 capabilities for formulating and analyzing optimization models. 
 - [Pysam/0.22.1-GCC-13.3.0](Pysam is a python module for reading and manipulating Samfiles.
 It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.)
Pysam is a python module for reading and manipulating Samfiles.
 It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.
 - [QIIME2/2024.5.0-foss-2023a](QIIME 2 is a powerful, extensible, and decentralized microbiome bioinformatics platform that is free,
open source, and community developed.)
QIIME 2 is a powerful, extensible, and decentralized microbiome bioinformatics platform that is free,
open source, and community developed.
 - [Qhull/2020.2-GCCcore-12.2.0](Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram,
 halfspace intersection about a point, furthest-site Delaunay triangulation,
 and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and
 higher dimensions. Qhull implements the Quickhull algorithm for computing the
 convex hull.
)
Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram,
 halfspace intersection about a point, furthest-site Delaunay triangulation,
 and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and
 higher dimensions. Qhull implements the Quickhull algorithm for computing the
 convex hull.

 - [R-Torch/2.7.1-foss-2024a-R-4.4.2](Provides functionality to define and train neural networks similar to 'PyTorch'.
 Includes Bioconductor 3.20.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/r/R-Torch/R-Torch-2.7.1-foss-2024a-R-4.4.2.eb)
Provides functionality to define and train neural networks similar to 'PyTorch'.
 Includes Bioconductor 3.20.
 - [R-bundle-Bioconductor/3.21-foss-2024a-R-4.5.1](Bioconductor provides tools for the analysis and coprehension
 of high-throughput genomic data.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.21-foss-2024a-R-4.5.1.eb)
Bioconductor provides tools for the analysis and coprehension
 of high-throughput genomic data.
 - [RAPIDS/24.4-foss-2023a-CUDA-12.1.1](RAPIDS provides unmatched speed with familiar APIs that match the most popular
PyData libraries. Built on state-of-the-art foundations like NVIDIA CUDA and
Apache Arrow, it unlocks the speed of GPUs with code you already know.)
RAPIDS provides unmatched speed with familiar APIs that match the most popular
PyData libraries. Built on state-of-the-art foundations like NVIDIA CUDA and
Apache Arrow, it unlocks the speed of GPUs with code you already know.
 - [RDKit/2025.03.3-foss-2024a](RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python.)
RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python.
 - [SAMtools/1.21-GCC-13.3.0](SAM Tools provide various utilities for manipulating alignments in the SAM format,
 including sorting, merging, indexing and generating alignments in a per-position format.)
SAM Tools provide various utilities for manipulating alignments in the SAM format,
 including sorting, merging, indexing and generating alignments in a per-position format.
 - [SCOTCH/7.0.6-gompi-2024a](Software package and libraries for sequential and parallel graph partitioning,
static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning.)
Software package and libraries for sequential and parallel graph partitioning,
static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning.
 - [SCimilarity/0.3.0-foss-2023a](SCimilarity is a unifying representation of single cell expression profiles that 
quantifies similarity between expression states and generalizes to represent new studies without additional 
training.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/SCimilarity/SCimilarity-0.3.0-foss-2023a.eb)
SCimilarity is a unifying representation of single cell expression profiles that 
quantifies similarity between expression states and generalizes to represent new studies without additional 
training.
 - [SRA-Toolkit/3.2.0-gompi-2024a](The SRA Toolkit, and the source-code SRA System Development
 Kit (SDK), will allow you to programmatically access data housed within SRA
 and convert it from the SRA format)
The SRA Toolkit, and the source-code SRA System Development
 Kit (SDK), will allow you to programmatically access data housed within SRA
 and convert it from the SRA format
 - [STAR/2.7.11b-GCC-13.2.0](STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/STAR/STAR-2.7.11b-GCC-13.2.0.eb)
STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.
 - [STAR-Fusion/1.12.0-foss-2022b](STAR-Fusion uses the STAR aligner to identify candidate fusion transcripts
 supported by Illumina reads. STAR-Fusion further processes the output generated by the STAR aligner
 to map junction reads and spanning reads to a reference annotation set.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/STAR-Fusion/STAR-Fusion-1.12.0-foss-2022b.eb)
STAR-Fusion uses the STAR aligner to identify candidate fusion transcripts
 supported by Illumina reads. STAR-Fusion further processes the output generated by the STAR aligner
 to map junction reads and spanning reads to a reference annotation set.
 - [SVclone/1.1.2-foss-2022b](Cluster structural variants of similar cancer cell fraction (CCF).)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/SVclone/SVclone-1.1.2-foss-2022b.eb)
Cluster structural variants of similar cancer cell fraction (CCF).
 - [SYMPHONY/5.7.2-foss-2023b](SYMPHONY is an open-source solver for mixed-integer linear programs (MILPs) written in C.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/SYMPHONY/SYMPHONY-5.7.2-foss-2023b.eb)
SYMPHONY is an open-source solver for mixed-integer linear programs (MILPs) written in C.
 - [Sambamba/1.0.1-GCC-13.2.0](Sambamba is a high performance modern robust and fast tool
 (and library), written in the D programming language, for working with SAM
 and BAM files. Current functionality is an important subset of samtools
 functionality, including view, index, sort, markdup, and depth.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/Sambamba/Sambamba-1.0.1-GCC-13.2.0.eb)
Sambamba is a high performance modern robust and fast tool
 (and library), written in the D programming language, for working with SAM
 and BAM files. Current functionality is an important subset of samtools
 functionality, including view, index, sort, markdup, and depth.
 - [Seaborn/0.13.2-gfbf-2023a](Seaborn is a Python visualization library based on matplotlib.
 It provides a high-level interface for drawing attractive statistical graphics. )
Seaborn is a Python visualization library based on matplotlib.
 It provides a high-level interface for drawing attractive statistical graphics. 
 - [SeqKit/2.10.1](SeqKit)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/SeqKit/SeqKit-2.10.1.eb)
SeqKit
 - [Seurat/5.1.0-foss-2023a-R-4.3.2](Seurat is an R package designed for QC, analysis, and exploration of single cell RNA-seq data.)
Seurat is an R package designed for QC, analysis, and exploration of single cell RNA-seq data.
 - [ShapeMapper2/2.3-GCC-13.3.0](ShapeMapper automates the calculation of RNA chemical probing reactivities
 from mutational profiling (MaP) experiments, in which chemical adducts on RNA
 are detected as internal mutations in cDNA through reverse transcription and
 read out by massively parallel sequencing.
)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/ShapeMapper2/ShapeMapper2-2.3-GCC-13.3.0.eb)
ShapeMapper automates the calculation of RNA chemical probing reactivities
 from mutational profiling (MaP) experiments, in which chemical adducts on RNA
 are detected as internal mutations in cDNA through reverse transcription and
 read out by massively parallel sequencing.

 - [Shapely/2.0.1-gfbf-2023a](Shapely is a BSD-licensed Python package for manipulation and analysis of planar geometric objects.
It is based on the widely deployed GEOS (the engine of PostGIS) and JTS (from which GEOS is ported) libraries.)
Shapely is a BSD-licensed Python package for manipulation and analysis of planar geometric objects.
It is based on the widely deployed GEOS (the engine of PostGIS) and JTS (from which GEOS is ported) libraries.
 - [SigProfiler/202507-foss-2024a-numpy-2](SigProfiler provides a comprehensive and integrated suite of bioinformatic
 tools for performing mutational signature analysis.
)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/SigProfiler/SigProfiler-202507-foss-2024a-numpy-2.eb)
SigProfiler provides a comprehensive and integrated suite of bioinformatic
 tools for performing mutational signature analysis.

 - [Sniffles/2.6.2-gfbf-2024a](A fast structural variant caller for long-read sequencing,
 Sniffles2 accurately detect SVs on germline, somatic and population-level for PacBio and Oxford Nanopore read data.)
A fast structural variant caller for long-read sequencing,
 Sniffles2 accurately detect SVs on germline, somatic and population-level for PacBio and Oxford Nanopore read data.
 - [Squidpy/1.4.1-foss-2023a](Squidpy is a tool for the analysis and visualization of spatial molecular data.)
Squidpy is a tool for the analysis and visualization of spatial molecular data.
 - [TGS-GapCloser/1.2.1-GCCcore-13.3.0](A gap-closing software tool that uses error-prone long reads generated by
 third-generation-sequence techniques (Pacbio, Oxford Nanopore, etc.) or preassembled contigs
 to fill N-gap in the genome assembly.)
A gap-closing software tool that uses error-prone long reads generated by
 third-generation-sequence techniques (Pacbio, Oxford Nanopore, etc.) or preassembled contigs
 to fill N-gap in the genome assembly.
 - [Telescope/1.0.3-gfbf-2022b](Single locus resolution of Transposable ELEment expression using next-generation sequencing.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/t/Telescope/Telescope-1.0.3-gfbf-2022b.eb)
Single locus resolution of Transposable ELEment expression using next-generation sequencing.
 - [UMI-tools/1.1.4-foss-2023b](Tools for handling Unique Molecular Identifiers in NGS data sets)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/u/UMI-tools/UMI-tools-1.1.4-foss-2023b.eb)
Tools for handling Unique Molecular Identifiers in NGS data sets
 - [Uni-Core/0.0.3-foss-2023a](An efficient distributed PyTorch framework)
An efficient distributed PyTorch framework
 - [UniFrac/1.4-foss-2023a](UniFrac is the de facto repository for high-performance phylogenetic diversity
calculations. The methods in this repository are based on an implementation of
the Strided State UniFrac algorithm which is faster, and uses less memory than
Fast UniFrac. Strided State UniFrac supports Unweighted UniFrac, Weighted
UniFrac, Generalized UniFrac, Variance Adjusted UniFrac and meta UniFrac, in
both double and single precision (fp32). This repository also includes Stacked
Faith (manuscript in preparation), a method for calculating Faith's PD that is
faster and uses less memory than the Fast UniFrac-based reference
implementation.)
UniFrac is the de facto repository for high-performance phylogenetic diversity
calculations. The methods in this repository are based on an implementation of
the Strided State UniFrac algorithm which is faster, and uses less memory than
Fast UniFrac. Strided State UniFrac supports Unweighted UniFrac, Weighted
UniFrac, Generalized UniFrac, Variance Adjusted UniFrac and meta UniFrac, in
both double and single precision (fp32). This repository also includes Stacked
Faith (manuscript in preparation), a method for calculating Faith's PD that is
faster and uses less memory than the Fast UniFrac-based reference
implementation.
 - [VEP/113.3-GCC-13.3.0](Variant Effect Predictor (VEP) determines the effect of your
 variants (SNPs, insertions, deletions, CNVs or structural variants) on genes,
 transcripts, and protein sequence, as well as regulatory regions.
 Includes EnsEMBL-XS, which provides pre-compiled replacements for frequently
 used routines in VEP.)
Variant Effect Predictor (VEP) determines the effect of your
 variants (SNPs, insertions, deletions, CNVs or structural variants) on genes,
 transcripts, and protein sequence, as well as regulatory regions.
 Includes EnsEMBL-XS, which provides pre-compiled replacements for frequently
 used routines in VEP.
 - [VSEARCH/2.25.0-GCC-12.3.0](VSEARCH supports de novo and reference based chimera detection,
 clustering, full-length and prefix dereplication, rereplication,
 reverse complementation, masking, all-vs-all pairwise global alignment,
 exact and global alignment searching, shuffling, subsampling and sorting.
 It also supports FASTQ file analysis, filtering,
 conversion and merging of paired-end reads.)
VSEARCH supports de novo and reference based chimera detection,
 clustering, full-length and prefix dereplication, rereplication,
 reverse complementation, masking, all-vs-all pairwise global alignment,
 exact and global alignment searching, shuffling, subsampling and sorting.
 It also supports FASTQ file analysis, filtering,
 conversion and merging of paired-end reads.
 - [WhatsHap/2.2-foss-2023a](WhatsHap is a software for phasing genomic variants using DNA
sequencing reads, also called read-based phasing or haplotype assembly. It is
especially suitable for long reads, but works also well with short reads.)
WhatsHap is a software for phasing genomic variants using DNA
sequencing reads, also called read-based phasing or haplotype assembly. It is
especially suitable for long reads, but works also well with short reads.
 - [anndata/0.11.3-foss-2024a](anndata is a Python package for handling annotated data matrices in memory and on disk,
 positioned between pandas and xarray)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/a/anndata/anndata-0.11.3-foss-2024a.eb)
anndata is a Python package for handling annotated data matrices in memory and on disk,
 positioned between pandas and xarray
 - [bam-readcount/1.0.1-GCC-12.2.0](Count DNA sequence reads in BAM files)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/bam-readcount/bam-readcount-1.0.1-GCC-12.2.0.eb)
Count DNA sequence reads in BAM files
 - [biom-format/2.1.15-foss-2023a](The BIOM file format (canonically pronounced biome) is designed to be
 a general-use format for representing biological sample by observation
 contingency tables. BIOM is a recognized standard for the Earth Microbiome
 Project and is a Genomics Standards Consortium supported project.
)
The BIOM file format (canonically pronounced biome) is designed to be
 a general-use format for representing biological sample by observation
 contingency tables. BIOM is a recognized standard for the Earth Microbiome
 Project and is a Genomics Standards Consortium supported project.

 - [bx-python/0.13.0-foss-2023b](The bx-python project is a Python library and associated set of scripts to allow for rapid
 implementation of genome scale analyses.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/bx-python/bx-python-0.13.0-foss-2023b.eb)
The bx-python project is a Python library and associated set of scripts to allow for rapid
 implementation of genome scale analyses.
 - [castor/1.8.2-foss-2023a](Efficient phylogenetic analyses on massive phylogenies comprising up to millions of tips.
Functions include pruning, rerooting, calculation of most-recent common ancestors, calculating
distances from the tree root and calculating pairwise distances. Calculation of phylogenetic
signal and mean trait depth (trait conservatism), ancestral state reconstruction and hidden
character prediction of discrete characters, simulating and fitting models of trait evolution,
fitting and simulating diversification models, dating trees, comparing trees, and
reading/writing trees in Newick format. 
)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/castor/castor-1.8.2-foss-2023a.eb)
Efficient phylogenetic analyses on massive phylogenies comprising up to millions of tips.
Functions include pruning, rerooting, calculation of most-recent common ancestors, calculating
distances from the tree root and calculating pairwise distances. Calculation of phylogenetic
signal and mean trait depth (trait conservatism), ancestral state reconstruction and hidden
character prediction of discrete characters, simulating and fitting models of trait evolution,
fitting and simulating diversification models, dating trees, comparing trees, and
reading/writing trees in Newick format. 

 - [cellsnp-lite/1.2.3-GCC-13.2.0](Cellsnp-lite is a C/C++ tool for efficient genotyping
bi-allelic SNPs on single cells. You can use cellsnp-lite after read
alignment to obtain the snp x cell pileup UMI or read count matrices for
each allele of given or detected SNPs.)
Cellsnp-lite is a C/C++ tool for efficient genotyping
bi-allelic SNPs on single cells. You can use cellsnp-lite after read
alignment to obtain the snp x cell pileup UMI or read count matrices for
each allele of given or detected SNPs.
 - [cooler/0.10.2-foss-2023a](Cooler is a support library for a storage format, also called cooler, used to store
 genomic interaction data of any size, such as Hi-C contact matrices.)
Cooler is a support library for a storage format, also called cooler, used to store
 genomic interaction data of any size, such as Hi-C contact matrices.
 - [cooltools/0.7.1-foss-2024a](cooltools provides a suite of computational tools with a paired python API and
 command line access, which facilitates workflows either on high-performance
 computing clusters or via custom analysis notebooks. As part of the Open2C
 ecosystem, cooltools also provides detailed introductions to key concepts in
 Hi-C-data analysis with interactive notebook documentation.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/cooltools/cooltools-0.7.1-foss-2024a.eb)
cooltools provides a suite of computational tools with a paired python API and
 command line access, which facilitates workflows either on high-performance
 computing clusters or via custom analysis notebooks. As part of the Open2C
 ecosystem, cooltools also provides detailed introductions to key concepts in
 Hi-C-data analysis with interactive notebook documentation.
 - [cutadapt/4.9-GCCcore-12.3.0](Cutadapt finds and removes adapter sequences, primers, poly-A tails and
 other types of unwanted sequence from your high-throughput sequencing reads.)
Cutadapt finds and removes adapter sequences, primers, poly-A tails and
 other types of unwanted sequence from your high-throughput sequencing reads.
 - [cuteSV/2.1.2-foss-2024a](cuteSV uses tailored methods to collect the signatures of various types of SVs
 and employs a clustering-and-refinement method to analyze the signatures to implement sensitive SV detection.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/cuteSV/cuteSV-2.1.2-foss-2024a.eb)
cuteSV uses tailored methods to collect the signatures of various types of SVs
 and employs a clustering-and-refinement method to analyze the signatures to implement sensitive SV detection.
 - [dcm2niix/1.0.20241211-GCCcore-13.3.0](dcm2niix is designed to convert neuroimaging data from the DICOM format to the NIfTI format.)
dcm2niix is designed to convert neuroimaging data from the DICOM format to the NIfTI format.
 - [dorado/1.0.2-foss-2024a-CUDA-12.6.0](Dorado is a high-performance, easy-to-use, open source basecaller for Oxford Nanopore reads.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/d/dorado/dorado-1.0.2-foss-2024a-CUDA-12.6.0.eb)
Dorado is a high-performance, easy-to-use, open source basecaller for Oxford Nanopore reads.
 - [dssp/4.4.10-GCC-13.3.0](This is a rewrite of DSSP, now offering full mmCIF support. The difference with
previous releases of DSSP is that it now writes out an annotated mmCIF file by
default, storing the secondary structure information in the _struct_conf
category.

Another new feature in this version of DSSP is that it now defines Poly-Proline
helices as well.

The DSSP program was designed by Wolfgang Kabsch and Chris Sander to standardize
secondary structure assignment. DSSP is a database of secondary structure
assignments (and much more) for all protein entries in the Protein Data Bank
(PDB). DSSP is also the program that calculates DSSP entries from PDB entries.

DSSP does not predict secondary structure.)
This is a rewrite of DSSP, now offering full mmCIF support. The difference with
previous releases of DSSP is that it now writes out an annotated mmCIF file by
default, storing the secondary structure information in the _struct_conf
category.

Another new feature in this version of DSSP is that it now defines Poly-Proline
helices as well.

The DSSP program was designed by Wolfgang Kabsch and Chris Sander to standardize
secondary structure assignment. DSSP is a database of secondary structure
assignments (and much more) for all protein entries in the Protein Data Bank
(PDB). DSSP is also the program that calculates DSSP entries from PDB entries.

DSSP does not predict secondary structure.
 - [e3nn/0.5.0-foss-2022b-PyTorch-1.13.1-CUDA-11.8.0](Euclidean neural networks (e3nn) is a python library based on pytorch to create equivariant
neural networks for the group O(3).
)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/e/e3nn/e3nn-0.5.0-foss-2022b-PyTorch-1.13.1-CUDA-11.8.0.eb)
Euclidean neural networks (e3nn) is a python library based on pytorch to create equivariant
neural networks for the group O(3).

 - [easel/0.48-GCC-12.2.0](Easel supports computational analysis of biological sequences using probabilistic models.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/e/easel/easel-0.48-GCC-12.2.0.eb)
Easel supports computational analysis of biological sequences using probabilistic models.
 - [edlib/1.3.9.post1-GCC-13.3.0](Lightweight, super fast library for sequence alignment using edit (Levenshtein) distance.)
Lightweight, super fast library for sequence alignment using edit (Levenshtein) distance.
 - [einops/0.7.0-GCCcore-12.3.0](Flexible and powerful tensor operations for readable and reliable code.
Supports numpy, pytorch, tensorflow, jax, and others.)
Flexible and powerful tensor operations for readable and reliable code.
Supports numpy, pytorch, tensorflow, jax, and others.
 - [epiScanpy/0.4.0-foss-2023a](EpiScanpy is a toolkit to analyse single-cell open chromatin
(scATAC-seq) and single-cell DNA methylation (for example scBS-seq)
data. EpiScanpy is the epigenomic extension of the very popular scRNA-seq
analysis tool Scanpy (Genome Biology, 2018) [Wolf18].)
EpiScanpy is a toolkit to analyse single-cell open chromatin
(scATAC-seq) and single-cell DNA methylation (for example scBS-seq)
data. EpiScanpy is the epigenomic extension of the very popular scRNA-seq
analysis tool Scanpy (Genome Biology, 2018) [Wolf18].
 - [fastp/0.23.4-GCC-13.2.0](A tool designed to provide fast all-in-one preprocessing for FastQ files.
 This tool is developed in C++ with multithreading supported to afford high performance.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/f/fastp/fastp-0.23.4-GCC-13.2.0.eb)
A tool designed to provide fast all-in-one preprocessing for FastQ files.
 This tool is developed in C++ with multithreading supported to afford high performance.
 - [fhR/4.4.0-foss-2023b](R is a free software environment for statistical computing and graphics.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/f/fhR/fhR-4.4.0-foss-2023b.eb)
R is a free software environment for statistical computing and graphics.
 - [gappa/0.8.5-GCC-12.3.0](gappa is a collection of commands for working with phylogenetic data. Its main
focus are evolutionary placements of short environmental sequences on a
reference phylogenetic tree. Such data is typically produced by tools like
EPA-ng, RAxML-EPA or pplacer and usually stored in jplace files.
)
gappa is a collection of commands for working with phylogenetic data. Its main
focus are evolutionary placements of short environmental sequences on a
reference phylogenetic tree. Such data is typically produced by tools like
EPA-ng, RAxML-EPA or pplacer and usually stored in jplace files.

 - [gffread/0.12.7-GCCcore-12.2.0](GFF/GTF parsing utility providing format conversions,
region filtering, FASTA sequence extraction and more.)
GFF/GTF parsing utility providing format conversions,
region filtering, FASTA sequence extraction and more.
 - [gmpy2/2.2.0-GCCcore-13.3.0](GMP/MPIR, MPFR, and MPC interface to Python 2.6+ and 3.x)
GMP/MPIR, MPFR, and MPC interface to Python 2.6+ and 3.x
 - [hermes/0.1.0-foss-2022b-PyTorch-1.13.1-CUDA-11.8.0](HERMES: Holographic Equivariant neuRal network model for
 Mutational Effect and Stability prediction)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/h/hermes/hermes-0.1.0-foss-2022b-PyTorch-1.13.1-CUDA-11.8.0.eb)
HERMES: Holographic Equivariant neuRal network model for
 Mutational Effect and Stability prediction
 - [infercnvpy/0.4.3-foss-2023a](Infer copy number variation (CNV) from scRNA-seq data. Plays nicely with Scanpy.)
Infer copy number variation (CNV) from scRNA-seq data. Plays nicely with Scanpy.
 - [jax/0.4.25-gfbf-2023a](Composable transformations of Python+NumPy programs:
differentiate, vectorize, JIT to GPU/TPU, and more)
Composable transformations of Python+NumPy programs:
differentiate, vectorize, JIT to GPU/TPU, and more
 - [kallisto/0.50.1-foss-2022b](kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally
 of target sequences using high-throughput sequencing reads.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/k/kallisto/kallisto-0.50.1-foss-2022b.eb)
kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally
 of target sequences using high-throughput sequencing reads.
 - [libcerf/2.3-GCCcore-12.3.0](libcerf is a self-contained numeric library that provides an efficient and
 accurate implementation of complex error functions, along with Dawson,
 Faddeeva, and Voigt functions.
)
libcerf is a self-contained numeric library that provides an efficient and
 accurate implementation of complex error functions, along with Dawson,
 Faddeeva, and Voigt functions.

 - [libcifpp/7.0.8-GCC-12.2.0](This library contains code to work with mmCIF and PDB files)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/l/libcifpp/libcifpp-7.0.8-GCC-12.2.0.eb)
This library contains code to work with mmCIF and PDB files
 - [libmcfp/1.3.4-GCCcore-13.3.0](A library for parsing command line arguments and configuration files and making
them available throughout a program.)
A library for parsing command line arguments and configuration files and making
them available throughout a program.
 - [loompy/3.0.8-foss-2024a](Python implementation of the Loom file format, an efficient file format for large omics datasets)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/l/loompy/loompy-3.0.8-foss-2024a.eb)
Python implementation of the Loom file format, an efficient file format for large omics datasets
 - [magma/2.9.0-foss-2024a-CUDA-12.6.0](The MAGMA project aims to develop a dense linear algebra library similar to
 LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems.)
The MAGMA project aims to develop a dense linear algebra library similar to
 LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems.
 - [minimap2/2.29-GCCcore-13.3.0](Minimap2 is a fast sequence mapping and alignment
program that can find overlaps between long noisy reads, or map long
reads or their assemblies to a reference genome optionally with detailed
alignment (i.e. CIGAR). At present, it works efficiently with query
sequences from a few kilobases to ~100 megabases in length at an error
rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited
test data sets, minimap2 is over 20 times faster than most other
long-read aligners. It will replace BWA-MEM for long reads and contig
alignment.)
Minimap2 is a fast sequence mapping and alignment
program that can find overlaps between long noisy reads, or map long
reads or their assemblies to a reference genome optionally with detailed
alignment (i.e. CIGAR). At present, it works efficiently with query
sequences from a few kilobases to ~100 megabases in length at an error
rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited
test data sets, minimap2 is over 20 times faster than most other
long-read aligners. It will replace BWA-MEM for long reads and contig
alignment.
 - [mpath/1.1.3-GCCcore-12.3.0](For now it's quit simple and get_path_info()
method returns information about given path. It can be either
a directory or a file path.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/mpath/mpath-1.1.3-GCCcore-12.3.0.eb)
For now it's quit simple and get_path_info()
method returns information about given path. It can be either
a directory or a file path.
 - [mpmath/1.3.0-GCCcore-12.3.0](mpmath can be used as an arbitrary-precision substitute for Python's float/complex
 types and math/cmath modules, but also does much more advanced mathematics. Almost any calculation
 can be performed just as well at 10-digit or 1000-digit precision, with either real or complex
 numbers, and in many cases mpmath implements efficient algorithms that scale well for extremely
 high precision work.)
mpmath can be used as an arbitrary-precision substitute for Python's float/complex
 types and math/cmath modules, but also does much more advanced mathematics. Almost any calculation
 can be performed just as well at 10-digit or 1000-digit precision, with either real or complex
 numbers, and in many cases mpmath implements efficient algorithms that scale well for extremely
 high precision work.
 - [mrcfile/1.4.3-gfbf-2022b](mrcfile is a Python implementation of the MRC2014 file format, which is used in
structural biology to store image and volume data.

It allows MRC files to be created and opened easily using a very simple API,
which exposes the file’s header and data as numpy arrays. The code runs in
Python 2 and 3 and is fully unit-tested.

This library aims to allow users and developers to read and write standard-
compliant MRC files in Python as easily as possible, and with no dependencies on
any compiled libraries except numpy. You can use it interactively to inspect
files, correct headers and so on, or in scripts and larger software packages to
provide basic MRC file I/O functions. )
mrcfile is a Python implementation of the MRC2014 file format, which is used in
structural biology to store image and volume data.

It allows MRC files to be created and opened easily using a very simple API,
which exposes the file’s header and data as numpy arrays. The code runs in
Python 2 and 3 and is fully unit-tested.

This library aims to allow users and developers to read and write standard-
compliant MRC files in Python as easily as possible, and with no dependencies on
any compiled libraries except numpy. You can use it interactively to inspect
files, correct headers and so on, or in scripts and larger software packages to
provide basic MRC file I/O functions. 
 - [ncbi-vdb/3.2.0-gompi-2024a](The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for
 using data in the INSDC Sequence Read Archives.)
The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for
 using data in the INSDC Sequence Read Archives.
 - [netMHCpan/4.1b](The NetMHCpan software predicts binding of peptides to any known MHC
 molecule using artificial neural networks (ANNs).)
The NetMHCpan software predicts binding of peptides to any known MHC
 molecule using artificial neural networks (ANNs).
 - [nf-core/3.3.1-foss-2024a](Python package with helper tools for the nf-core community.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/n/nf-core/nf-core-3.3.1-foss-2024a.eb)
Python package with helper tools for the nf-core community.
 - [numexpr/2.9.0-foss-2023a](The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.
 It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into
 code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a
 compiler at runtime.)
The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.
 It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into
 code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a
 compiler at runtime.
 - [parasail/2.6.2-GCC-12.3.0](parasail is a SIMD C (C99) library containing implementations
 of the Smith-Waterman (local), Needleman-Wunsch (global), and semi-global
 pairwise sequence alignment algorithms. )
parasail is a SIMD C (C99) library containing implementations
 of the Smith-Waterman (local), Needleman-Wunsch (global), and semi-global
 pairwise sequence alignment algorithms. 
 - [prodigal/2.6.3-GCCcore-13.3.0](Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)
   is a microbial (bacterial and archaeal) gene finding program developed
   at Oak Ridge National Laboratory and the University of Tennessee.)
Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)
   is a microbial (bacterial and archaeal) gene finding program developed
   at Oak Ridge National Laboratory and the University of Tennessee.
 - [pyBigWig/0.3.23-gfbf-2023b](A python extension, written in C, for quick access to bigBed
files and access to and creation of bigWig files.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/p/pyBigWig/pyBigWig-0.3.23-gfbf-2023b.eb)
A python extension, written in C, for quick access to bigBed
files and access to and creation of bigWig files.
 - [pyEGA3/5.0.2-GCCcore-12.3.0](A basic Python-based EGA download client )
A basic Python-based EGA download client 
 - [pySCENIC/0.12.1-20240311-foss-2023a](pySCENIC is a lightning-fast python implementation of the SCENIC
pipeline (Single-Cell rEgulatory Network Inference and Clustering) which enables
biologists to infer transcription factors, gene regulatory networks and
cell types from single-cell RNA-seq data.)
pySCENIC is a lightning-fast python implementation of the SCENIC
pipeline (Single-Cell rEgulatory Network Inference and Clustering) which enables
biologists to infer transcription factors, gene regulatory networks and
cell types from single-cell RNA-seq data.
 - [pybedtools/0.12.0-foss-2024a](pybedtools wraps and extends BEDTools and offers feature-level manipulations from within Python.)
pybedtools wraps and extends BEDTools and offers feature-level manipulations from within Python.
 - [pyfaidx/0.8.1.2-GCCcore-13.3.0](pyfaidx: efficient pythonic random access to fasta subsequences)
pyfaidx: efficient pythonic random access to fasta subsequences
 - [python-parasail/1.3.4-foss-2023a](Python Bindings for the Parasail C Library)
Python Bindings for the Parasail C Library
 - [samblaster/0.1.26-GCC-13.2.0](samblaster is a fast and flexible program for marking
 duplicates in read-id grouped1 paired-end SAM files.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/samblaster/samblaster-0.1.26-GCC-13.2.0.eb)
samblaster is a fast and flexible program for marking
 duplicates in read-id grouped1 paired-end SAM files.
 - [scArches/0.6.1-foss-2023a](Single-cell architecture surgery (scArches) is a package for reference-based analysis of single-cell
 data.)
Single-cell architecture surgery (scArches) is a package for reference-based analysis of single-cell
 data.
 - [scCODA/0.1.9-foss-2023a](scCODA allows for identification of compositional changes in high-throughput sequencing count data,
especially cell compositions from scRNA-seq.)
scCODA allows for identification of compositional changes in high-throughput sequencing count data,
especially cell compositions from scRNA-seq.
 - [scGPT/0.2.1-foss-2023a-CUDA-12.1.1](scGPT: Towards Building a Foundation Model for Single-Cell Multi-omics Using Generative AI.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/scGPT/scGPT-0.2.1-foss-2023a-CUDA-12.1.1.eb)
scGPT: Towards Building a Foundation Model for Single-Cell Multi-omics Using Generative AI.
 - [scVelo/0.3.1-foss-2023a](scVelo is a scalable toolkit for estimating and analyzing RNA velocities in single cells using
 dynamical modeling.)
scVelo is a scalable toolkit for estimating and analyzing RNA velocities in single cells using
 dynamical modeling.
 - [scanpy/1.10.4-foss-2024a](Scanpy is a scalable toolkit for analyzing single-cell gene expression data built
 jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference
 and differential expression testing. The Python-based implementation efficiently deals with
 datasets of more than one million cells.
)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/scanpy/scanpy-1.10.4-foss-2024a.eb)
Scanpy is a scalable toolkit for analyzing single-cell gene expression data built
 jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference
 and differential expression testing. The Python-based implementation efficiently deals with
 datasets of more than one million cells.

 - [scib-metrics/0.5.1-foss-2023a](Accelerated and Python-only metrics for benchmarking single-cell integration outputs)
Accelerated and Python-only metrics for benchmarking single-cell integration outputs
 - [scikit-bio/0.6.0-foss-2023a](scikit-bio is an open-source, BSD-licensed Python 3 package providing data structures, algorithms
and educational resources for bioinformatics.)
scikit-bio is an open-source, BSD-licensed Python 3 package providing data structures, algorithms
and educational resources for bioinformatics.
 - [scrublet/0.2.3-foss-2023a](Single-Cell Remover of Doublets)
Single-Cell Remover of Doublets
 - [scvi-tools/1.1.2-foss-2023a-CUDA-12.1.1](scvi-tools (single-cell variational inference tools) is a package for probabilistic modeling and
analysis of single-cell omics data, built on top of PyTorch and AnnData.)
scvi-tools (single-cell variational inference tools) is a package for probabilistic modeling and
analysis of single-cell omics data, built on top of PyTorch and AnnData.
 - [skani/0.2.2-GCCcore-12.3.0](skani)
skani
 - [smithlab/1.2-GCC-13.3.0](A C library for reading/writing high-throughput sequencing data.
 This package includes the utilities bgzip and tabix)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/smithlab/smithlab-1.2-GCC-13.3.0.eb)
A C library for reading/writing high-throughput sequencing data.
 This package includes the utilities bgzip and tabix
 - [spams/2.6.5.4-foss-2023b](SPAMS (SPArse Modeling Software) is an optimization toolbox for solving various
 sparse estimation problems.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/spams/spams-2.6.5.4-foss-2023b.eb)
SPAMS (SPArse Modeling Software) is an optimization toolbox for solving various
 sparse estimation problems.
 - [spektral/1.3.1-foss-2023a-CUDA-12.1.1](Spektral is a Python library for graph deep learning, based on the Keras API and TensorFlow 2)
Spektral is a Python library for graph deep learning, based on the Keras API and TensorFlow 2
 - [splitpipe/1.2.1-foss-2023b](splitpipe tool from Parse Biosciences. The pipeline takes FASTQ files and delivers
 processed data in the form of a cell-gene count matrix, which serves as the input for various open
 sources tools such as scanpy and seuratProcess sequencing results with our pipeline.

 A Parse Bioscience login ID is required to download)
splitpipe tool from Parse Biosciences. The pipeline takes FASTQ files and delivers
 processed data in the form of a cell-gene count matrix, which serves as the input for various open
 sources tools such as scanpy and seuratProcess sequencing results with our pipeline.

 A Parse Bioscience login ID is required to download
 - [starcode/1.4-GCC-13.2.0](Starcode is a DNA sequence clustering software. Starcode clustering is based on all pairs
search within a specified Levenshtein distance (allowing insertions and deletions), followed by a clustering
algorithm: Message Passing, Spheres or Connected Components.)
Starcode is a DNA sequence clustering software. Starcode clustering is based on all pairs
search within a specified Levenshtein distance (allowing insertions and deletions), followed by a clustering
algorithm: Message Passing, Spheres or Connected Components.
 - [statsmodels/0.14.4-gfbf-2024a-numpy-2](Statsmodels is a Python module that allows users to explore data, estimate statistical models,
and perform statistical tests.)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/statsmodels/statsmodels-0.14.4-gfbf-2024a-numpy-2.eb)
Statsmodels is a Python module that allows users to explore data, estimate statistical models,
and perform statistical tests.
 - [sympy/1.13.3-gfbf-2024a-numpy-2](SymPy is a Python library for symbolic mathematics. It aims to
 become a full-featured computer algebra system (CAS) while keeping the code as
 simple as possible in order to be comprehensible and easily extensible. SymPy
 is written entirely in Python and does not require any external libraries.)
SymPy is a Python library for symbolic mathematics. It aims to
 become a full-featured computer algebra system (CAS) while keeping the code as
 simple as possible in order to be comprehensible and easily extensible. SymPy
 is written entirely in Python and does not require any external libraries.
 - [wandb/0.16.1-GCC-12.3.0](CLI and Python API for Weights and Biases (wandb), a tool for visualizing and
tracking your machine learning experiments.)
CLI and Python API for Weights and Biases (wandb), a tool for visualizing and
tracking your machine learning experiments.
