---
title: ermine Bio Modules 18.04
layout: single
permalink: /sw_inventory/ermine-bio-modules-18.04/
created: 2025-05-15
toc: true
toc_label: "On This Page"
sidebar:
  nav: "docs"
---

 - [AlphaFold/2.2.2-foss-2021a-CUDA-11.3.1](https://deepmind.com/research/case-studies/alphafold)
AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known
 - [Beast/1.10.5pre_thorney_v0.1.2-GCC-11.2.0](https://beast.community)
BEAST is a cross-platform program for Bayesian MCMC analysis of molecular
 sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using
 strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies
 but is also a framework for testing evolutionary hypotheses without conditioning on a single
 tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted
 proportional to its posterior probability. 
 - [Biopython/1.79-foss-2021b](https://www.biopython.org)
Biopython is a set of freely available tools for biological
 computation written in Python by an international team of developers. It is
 a distributed collaborative effort to develop Python libraries and
 applications which address the needs of current and future work in
 bioinformatics. 
 - [Eigen/3.3.9-GCCcore-10.3.0](https://eigen.tuxfamily.org)
Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers,
 and related algorithms.
 - [GEOS/3.9.1-GCC-11.2.0](https://trac.osgeo.org/geos)
GEOS (Geometry Engine
 - [GMP/6.2.1-GCCcore-10.3.0](https://gmplib.org/)
GMP is a free library for arbitrary precision arithmetic, operating on signed
 integers, rational numbers, and floating point numbers.

 - [HH-suite/3.3.0-gompi-2021b](https://github.com/soedinglab/hh-suite)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/h/HH-suite/HH-suite-3.3.0-gompi-2021b.eb)
The HH-suite is an open-source software package 
for sensitive protein sequence searching based on the pairwise 
alignment of hidden Markov models (HMMs).
 - [HMMER/3.3.2-gompi-2021b](http://hmmer.org/)
HMMER is used for searching sequence databases for homologs
 of protein sequences, and for making protein sequence alignments. It
 implements methods using probabilistic models called profile hidden Markov
 models (profile HMMs).  Compared to BLAST, FASTA, and other sequence
 alignment and database search tools based on older scoring methodology,
 HMMER aims to be significantly more accurate and more able to detect remote
 homologs because of the strength of its underlying mathematical models. In the
 past, this strength came at significant computational expense, but in the new
 HMMER3 project, HMMER is now essentially as fast as BLAST.
 - [Kalign/3.3.2-GCCcore-11.2.0](https://github.com/TimoLassmann/kalign)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/k/Kalign/Kalign-3.3.2-GCCcore-11.2.0.eb)
Kalign is a fast multiple sequence alignment program for biological sequences.
 - [MPFR/4.1.0-GCCcore-11.2.0](https://www.mpfr.org)
The MPFR library is a C library for multiple-precision floating-point
 computations with correct rounding.

 - [MUMmer/4.0.0beta2-GCCcore-11.2.0](http://mummer.sourceforge.net/)
MUMmer is a system for rapidly aligning entire genomes,
 whether in complete or draft form. AMOS makes use of it.

 - [MrBayes/3.2.7a-foss-2021b](http://mrbayes.csit.fsu.edu)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/MrBayes/MrBayes-3.2.7a-foss-2021b.eb)
MrBayes is a program for the Bayesian estimation of phylogeny.
 - [Qhull/2020.2-GCCcore-11.2.0](http://www.qhull.org)
Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram,
 halfspace intersection about a point, furthest-site Delaunay triangulation,
 and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and
 higher dimensions. Qhull implements the Quickhull algorithm for computing the
 convex hull.

 - [einops/0.7.0-GCCcore-12.3.0](https://einops.rocks/)
Flexible and powerful tensor operations for readable and reliable code.
Supports numpy, pytorch, tensorflow, jax, and others.
