diff -ruN salmon-1.2.0.orig/CMakeLists.txt salmon-1.2.0/CMakeLists.txt
--- salmon-1.2.0.orig/CMakeLists.txt	2020-04-10 13:51:05.000000000 -0700
+++ salmon-1.2.0/CMakeLists.txt	2020-05-08 11:01:40.095319000 -0700
@@ -70,8 +70,8 @@
 set(REMOVE_WARNING_FLAGS "-Wno-unused-function;-Wno-unused-local-typedefs")
 set(TGT_COMPILE_FLAGS "-ftree-vectorize;-funroll-loops;-fPIC;-fomit-frame-pointer;-O3;-DNDEBUG;-DSTX_NO_STD_STRING_VIEW")
 set(TGT_WARN_FLAGS "-Wall;-Wno-unknown-pragmas;-Wno-reorder;-Wno-unused-variable;-Wreturn-type;-Werror=return-type;${REMOVE_WARNING_FLAGS}")
-#set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=address")
-#set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
 
 ###
 # Sanitizers BEGIN
