--- run_gistic_example.orig	2017-02-24 14:50:55.603886000 -0800
+++ run_gistic_example	2017-03-03 14:22:46.782747000 -0800
@@ -1,12 +1,13 @@
-#!/bin/csh
+#!/bin/csh -f
 
 set thisdir = `pwd`
+set base = $EBROOTGISTIC
 
 # --- SET UP ENVIRONMENT VARIABLES ---
 echo --- setting up environment variables ---
 # presumed location of MATLAB Component Runtime (MCR) v7.14
 # if the MCR is in a different location, edit the line below
-set mcr_root = $thisdir/MATLAB_Component_Runtime
+set mcr_root = /app/MATLAB/7.14/MATLAB_Compiler_Runtime 
 setenv LD_LIBRARY_PATH $mcr_root/v714/runtime/glnxa64:$LD_LIBRARY_PATH
 setenv LD_LIBRARY_PATH $mcr_root/v714/sys/os/glnxa64:$LD_LIBRARY_PATH
 setenv LD_LIBRARY_PATH $mcr_root/v714/sys/java/jre/glnxa64/jre/lib/amd64/native_threads:$LD_LIBRARY_PATH
@@ -21,11 +22,11 @@
 mkdir -p $basedir 
 
 echo --- running GISTIC ---
-set segfile = $thisdir/examplefiles/segmentationfile.txt
-set markersfile = $thisdir/examplefiles/markersfile.txt
-set refgenefile = $thisdir/refgenefiles/hg16.mat
-set alf = $thisdir/examplefiles/arraylistfile.txt
-set cnvfile = $thisdir/examplefiles/cnvfile.txt
+set segfile = $base/examplefiles/segmentationfile.txt
+set markersfile = $base/examplefiles/markersfile.txt
+set refgenefile = $base/refgenefiles/hg16.mat
+set alf = $base/examplefiles/arraylistfile.txt
+set cnvfile = $base/examplefiles/cnvfile.txt
 
-./gp_gistic2_from_seg -b $basedir -seg $segfile -mk $markersfile -refgene $refgenefile -alf $alf -cnv $cnvfile -genegistic 1 -smallmem 1 -broad 1 -brlen 0.5 -conf 0.90 -armpeel 1 -savegene 1 -gcm extreme
+$base/gp_gistic2_from_seg -b $basedir -seg $segfile -mk $markersfile -refgene $refgenefile -alf $alf -cnv $cnvfile -genegistic 1 -smallmem 1 -broad 1 -brlen 0.5 -conf 0.90 -armpeel 1 -savegene 1 -gcm extreme
 
