---
title: All Modules 18.04
permalink: /all-modules-18.04/
layout: single
toc: true
toc_label: "On This Page"
sidebar:
  nav: "docs"
---

 - [AGAT/0.9.2-GCC-11.2.0](https://agat.readthedocs.io/en/latest/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/a/AGAT/AGAT-0.9.2-GCC-11.2.0.eb)
AGAT: Another GTF/GFF Analysis Toolkit. Suite of tools to handle gene annotations
 in any GTF/GFF format.
 - [ARGoS/3.0.0-beta59-GCC-11.2.0](http://www.argos-sim.info)
A parallel, multi-engine simulator for heterogeneous swarm robotics
 - [ASE/3.22.0-foss-2020b](https://wiki.fysik.dtu.dk/ase)
ASE is a python package providing an open source Atomic Simulation Environment
 in the Python scripting language.

From version 3.20.1 we also include the ase-ext package, it contains optional reimplementations
in C of functions in ASE.  ASE uses it automatically when installed.
 - [ATK/2.38.0-GCCcore-12.2.0](https://developer.gnome.org/atk/)
ATK provides the set of accessibility interfaces that are implemented by other
 toolkits and applications. Using the ATK interfaces, accessibility tools have
 full access to view and control running applications.

 - [AUGUSTUS/3.4.0-foss-2020b](https://bioinf.uni-greifswald.de/augustus/)
AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences
 - [Abseil/20230125.2-GCCcore-12.2.0](https://abseil.io/)
Abseil is an open-source collection of C++ library code designed to augment the
C++ standard library. The Abseil library code is collected from Google's own
C++ code base, has been extensively tested and used in production, and is the
same code we depend on in our daily coding lives.
 - [AdapterRemoval/2.2.2-foss-2018b](https://github.com/MikkelSchubert/adapterremoval)
AdapterRemoval searches for and removes remnant adapter sequences
 from High-Throughput Sequencing (HTS) data and (optionally) trims low quality bases
 from the 3' end of reads following adapter removal.
 - [AlphaFold/2.1.1-fosscuda-2020b](https://deepmind.com/research/case-studies/alphafold)
AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known
 - [Anaconda3/2022.05](https://www.anaconda.com)
Built to complement the rich, open source Python community,
the Anaconda platform provides an enterprise-ready data analytics platform 
that empowers companies to adopt a modern open data science analytics architecture.

 - [Apptainer/1.1.6](https://apptainer.org/docs/user/1.0/index.html)
Apptainer is a portable application stack packaging and runtime utility.
 - [Archive-Zip/1.68-GCCcore-10.2.0](https://metacpan.org/pod/Archive::Zip)
Provide an interface to ZIP archive files.
 - [Armadillo/11.4.3-foss-2022b](https://arma.sourceforge.net/)
Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards
 a good balance between speed and ease of use. Integer, floating point and complex numbers are supported,
 as well as a subset of trigonometric and statistics functions.
 - [ArrayFire/3.8.1-foss-2019b-CUDA-10.2.89](https://arrayfire.com/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/a/ArrayFire/ArrayFire-3.8.1-foss-2019b-CUDA-10.2.89.eb)
ArrayFire is a general-purpose library that simplifies the process of
 developing software that targets parallel and massively-parallel architectures
 including CPUs, GPUs, and other hardware acceleration devices.

 - [Arriba/2.1.0-GCC-10.2.0](https://github.com/suhrig/arriba)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/a/Arriba/Arriba-2.1.0-GCC-10.2.0.eb)
Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data.
 It was developed for the use in a clinical research setting. Therefore, short runtimes and high
 sensitivity were important design criteria.
 - [Arrow/11.0.0-gfbf-2022b](https://arrow.apache.org)
Apache Arrow (incl. PyArrow Python bindings), a cross-language development platform
 for in-memory data.
 - [Aspera-CLI/3.9.6.1467.159c5b1](https://asperasoft.com)
IBM Aspera Command-Line Interface (the Aspera CLI) is
a collection of Aspera tools for performing high-speed, secure data
transfers from the command line. The Aspera CLI is for users and
organizations who want to automate their transfer workflows.
 - [Aspera-Connect/3.9.6](http://downloads.asperasoft.com/connect2/)
Connect is an install-on-demand Web browser plug-in that facilitates high-speed uploads and
 downloads with an Aspera transfer server.
 - [Autoconf/2.71-GCCcore-12.2.0](https://www.gnu.org/software/autoconf/)
Autoconf is an extensible package of M4 macros that produce shell scripts
 to automatically configure software source code packages. These scripts can
 adapt the packages to many kinds of UNIX-like systems without manual user
 intervention. Autoconf creates a configuration script for a package from a
 template file that lists the operating system features that the package can
 use, in the form of M4 macro calls.

 - [Automake/1.16.5-GCCcore-12.2.0](https://www.gnu.org/software/automake/automake.html)
Automake: GNU Standards-compliant Makefile generator
 - [Autotools/20220317-GCCcore-11.3.0](https://autotools.io)
This bundle collect the standard GNU build tools: Autoconf, Automake
 and libtool

 - [BAli-Phy/3.6.0](http://www.bali-phy.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/BAli-Phy/BAli-Phy-3.6.0.eb)
BAli-Phy estimates multiple sequence alignments and evolutionary trees from DNA, amino acid, 
or codon sequences.
 - [BBMap/38.97-GCC-10.2.0](https://sourceforge.net/projects/bbmap/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/BBMap/BBMap-38.97-GCC-10.2.0.eb)
BBMap short read aligner, and other bioinformatic tools.
 - [BCFtools/1.14-GCC-11.2.0](https://www.htslib.org/)
Samtools is a suite of programs for interacting with high-throughput sequencing data.
 BCFtools
 - [BEDOPS/2.4.41-foss-2021b](http://bedops.readthedocs.io/en/latest/index.html)
BEDOPS is an open-source command-line toolkit that performs highly efficient and
 scalable Boolean and other set operations, statistical calculations, archiving, conversion and
 other management of genomic data of arbitrary scale. Tasks can be easily split by chromosome for
 distributing whole-genome analyses across a computational cluster.
 - [BEDTools/2.30.0-GCC-12.2.0](https://bedtools.readthedocs.io/)
BEDTools: a powerful toolset for genome arithmetic.
The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and
computing coverage.
The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM.
 - [BGEN/1.1.7](https://enkre.net/cgi-bin/code/bgen/dir?ci=trunk)
This repository contains a reference implementation 
of the BGEN format, written in C++. The library can be used as the 
basis for BGEN support in other software, or as a reference for 
developers writing their own implementations of the BGEN format.
Please cite:
Band, G. and Marchini, J., "BGEN: a binary file format for imputed genotype and haplotype data", 
bioArxiv 308296; doi: https://doi.org/10.1101/308296

 - [BGEN-enkre/1.1.7-GCC-10.2.0](https://enkre.net/cgi-bin/code/bgen/dir?ci=trunk)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/BGEN-enkre/BGEN-enkre-1.1.7-GCC-10.2.0.eb)
This repository contains a reference implementation
 of the BGEN format, written in C++. The library can be used as the
 basis for BGEN support in other software, or as a reference for
 developers writing their own implementations of the BGEN format.
 Please cite:
 Band, G. and Marchini, J., "BGEN: a binary file format for imputed genotype and haplotype data", 
 bioArxiv 308296; doi: https://doi.org/10.1101/308296

 - [BLAST+/2.14.0-gompi-2022b](https://blast.ncbi.nlm.nih.gov/)
Basic Local Alignment Search Tool, or BLAST, is an algorithm
 for comparing primary biological sequence information, such as the amino-acid
 sequences of different proteins or the nucleotides of DNA sequences.
 - [BLAT/3.5-foss-2018b](http://genome.ucsc.edu/FAQ/FAQblat.html)
BLAT on DNA is designed to quickly find sequences of 95% and greater similarity of length 25 bases or
 more.
 - [BLIS/0.9.0-GCC-12.2.0](https://github.com/flame/blis/)
BLIS is a portable software framework for instantiating high-performance
BLAS-like dense linear algebra libraries.
 - [BUStools/0.40.0-foss-2019b](https://github.com/BUStools/bustools)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/BUStools/BUStools-0.40.0-foss-2019b.eb)
bustools is a program for manipulating BUS files for single cell RNA-Seq datasets.
 It can be used to error correct barcodes, collapse  UMIs, produce gene count or transcript compatibility
 count matrices, and is useful for many other tasks. See the kallisto | bustools  website for examples
 and instructions on how to use bustools as part of a single-cell RNA-seq workflow.
 - [BWA/0.7.17-GCCcore-11.2.0](http://bio-bwa.sourceforge.net/)
Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively
 short nucleotide sequences against a long reference sequence such as the human
 genome.

 - [BamTools/2.5.2-GCC-12.2.0](https://github.com/pezmaster31/bamtools)
BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.
 - [BaseSpaceCLI/1.5.1](https://developer.basespace.illumina.com/docs/content/documentation/cli/cli-overview)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/BaseSpaceCLI/BaseSpaceCLI-1.5.1.eb)
BaseSpace is a powerful website where biologists and informaticians can easily store,
 analyze, and share genetic data. BaseSpace is a commerical product from Illumina.
 - [Bazel/3.7.2-GCCcore-11.2.0](https://bazel.io/)
Bazel is a build tool that builds code quickly and reliably.
It is used to build the majority of Google's software.
 - [Beagle/5.2.1](https://faculty.washington.edu/browning/beagle/beagle.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/Beagle/Beagle-5.2.1.eb)
Beagle is a software package for phasing genotypes and for imputing ungenotyped markers.
 - [Beast/2.6.4-GCC-10.2.0](http://beast2.org)
BEAST is a cross-platform program for Bayesian MCMC analysis of molecular 
 sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using 
 strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies 
 but is also a framework for testing evolutionary hypotheses without conditioning on a single 
 tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted 
 proportional to its posterior probability. 
 - [BeautifulSoup/4.11.1-GCCcore-12.2.0](https://www.crummy.com/software/BeautifulSoup)
Beautiful Soup is a Python library designed for quick turnaround projects like screen-scraping.
 - [Bio-DB-HTS/3.01-GCC-10.2.0](https://metacpan.org/release/Bio-DB-HTS)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-10.2.0.eb)
Read files using HTSlib including BAM/CRAM, Tabix and BCF database files
 - [Bio-SearchIO-hmmer/1.7.3-GCC-10.2.0](https://metacpan.org/pod/Bio::SearchIO::hmmer3)
Code to parse output from hmmsearch, hmmscan, phmmer and nhmmer, compatible
with both version 2 and version 3 of the HMMER package from http://hmmer.org.
 - [BioPerl/1.7.8-GCCcore-11.2.0](https://bioperl.org/)
Bioperl is the product of a community effort to produce Perl code which is useful in biology.
 Examples include Sequence objects, Alignment objects and database searching objects.
 - [Biopython/1.81-foss-2022b](https://www.biopython.org)
Biopython is a set of freely available tools for biological
 computation written in Python by an international team of developers. It is
 a distributed collaborative effort to develop Python libraries and
 applications which address the needs of current and future work in
 bioinformatics. 
 - [Bison/3.8.2-GCCcore-12.2.0](https://www.gnu.org/software/bison)
Bison is a general-purpose parser generator that converts an annotated context-free grammar
 into a deterministic LR or generalized LR (GLR) parser employing LALR(1) parser tables.
 - [Blosc/1.21.3-GCCcore-12.2.0](https://www.blosc.org/)
Blosc, an extremely fast, multi-threaded, meta-compressor library
 - [Blosc2/2.8.0-GCCcore-12.2.0](https://www.blosc.org/)
Blosc, an extremely fast, multi-threaded, meta-compressor library
 - [Boost/1.81.0-GCC-12.2.0](https://www.boost.org/)
Boost provides free peer-reviewed portable C++ source libraries.
 - [Boost.MPI/1.74.0-gompi-2020b](https://www.boost.org/)
Boost provides free peer-reviewed portable C++ source libraries.
 - [Boost.Python/1.67.0-foss-2018b-Python-2.7.15](http://boostorg.github.io/python)
Boost.Python is a C++ library which enables seamless interoperability between C++
 and the Python programming language.
 - [Bowtie/1.3.0-GCC-10.2.0](http://bowtie-bio.sourceforge.net/index.shtml)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/Bowtie/Bowtie-1.3.0-GCC-10.2.0.eb)
Bowtie is an ultrafast, memory-efficient short read aligner.
 It aligns short DNA sequences (reads) to the human genome.
 - [Bowtie2/2.4.4-GCC-11.2.0](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml)
Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads
 to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s
 of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.
 Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,
 its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.
 - [Brotli/1.0.9-GCCcore-10.3.0](https://github.com/google/brotli)
Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination
 of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio
 comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate
 but offers more dense compression.
The specification of the Brotli Compressed Data Format is defined in RFC 7932.
 - [Brunsli/0.1-GCCcore-12.2.0](https://github.com/google/brunsli/)
Brunsli is a lossless JPEG repacking library.
 - [CARBayes/5.1.1-foss-2018b-R-3.5.2](http://github.com/duncanplee/CARBayes)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CARBayes/CARBayes-5.1.1-foss-2018b-R-3.5.2.eb)
Implements a class of univariate and multivariate spatial generalised linear mixed models for areal unit data, with inference in a Bayesian setting using Markov chain Monte Carlo (MCMC) simulation.
 - [CD-HIT/4.8.1-foss-2019b](http://weizhong-lab.ucsd.edu/cd-hit/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CD-HIT/CD-HIT-4.8.1-foss-2019b.eb)
CD-HIT is a very widely used program for clustering and 
 comparing protein or nucleotide sequences.
 - [CFITSIO/4.2.0-GCCcore-12.2.0](https://heasarc.gsfc.nasa.gov/fitsio/)
CFITSIO is a library of C and Fortran subroutines for reading and writing data files in
FITS (Flexible Image Transport System) data format.
 - [CITE-seq-Count/1.4.2-foss-2018b-Python-3.6.6](https://hoohm.github.io/CITE-seq-Count)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CITE-seq-Count/CITE-seq-Count-1.4.2-foss-2018b-Python-3.6.6.eb)
A python package that allows to count antibody TAGS from a CITE-seq and/or
 cell hashing experiment.
 - [CMake/3.24.3-GCCcore-12.2.0](https://www.cmake.org)
CMake, the cross-platform, open-source build system.  CMake is a family of
 tools designed to build, test and package software.

 - [CNVkit/0.9.9-foss-2021b-R-4.2.0](https://github.com/etal/cnvkit)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CNVkit/CNVkit-0.9.9-foss-2021b-R-4.2.0.eb)
A command-line toolkit and Python library for detecting copy
 number variants and alterations genome-wide from high-throughput sequencing.
 - [CRISPRCasTyper/1.2.1-foss-2020a-Python-3.8.2](https://github.com/Russel88/CRISPRCasTyper)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CRISPRCasTyper/CRISPRCasTyper-1.2.1-foss-2020a-Python-3.8.2.eb)
Detect CRISPR-Cas genes and arrays, and predict the subtype based on both Cas genes and CRISPR repeat sequence.
 - [CRIU/3.13-foss-2019b-Python-3.7.4](https://github.com/checkpoint-restore/criu)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CRIU/CRIU-3.13-foss-2019b-Python-3.7.4.eb)
Checkpoint/Restore In Userspace (CRIU) is a Linux software which can freeze a running container
(or an individual application) and checkpoint its state to disk. The data saved can be used to restore the 
application and run it exactly as it was during the time of the freeze. Using this functionality, application 
or container live migration, snapshots, remote debugging, and many other things are now possible.
 - [CUDA/12.0.0](https://developer.nvidia.com/cuda-toolkit)
CUDA (formerly Compute Unified Device Architecture) is a parallel
 computing platform and programming model created by NVIDIA and implemented by the
 graphics processing units (GPUs) that they produce. CUDA gives developers access
 to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.
 - [CUDAcore/11.1.1](https://developer.nvidia.com/cuda-toolkit)
CUDA (formerly Compute Unified Device Architecture) is a parallel
 computing platform and programming model created by NVIDIA and implemented by the
 graphics processing units (GPUs) that they produce. CUDA gives developers access
 to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.
 - [CUnit/2.1-3-GCCcore-11.2.0](https://sourceforge.net/projects/cunit/)
Automated testing framework for C.
 - [Catch2/2.13.4](https://github.com/catchorg/Catch2)
A modern, C++-native, header-only,
 test framework for unit-tests, TDD and BDD

 - [CellBender/0.2.2-foss-2021b-CUDA-11.4.1](https://cellbender.readthedocs.io/)
CellBender is a software package for eliminating technical artifacts from high-throughput
 single-cell RNA sequencing (scRNA-seq) data.
 - [CellProfiler/4.2.4-foss-2021b](http://cellprofiler.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CellProfiler/CellProfiler-4.2.4-foss-2021b.eb)
CellProfiler is a free open-source software designed to enable biologists without training in computer vision or programming to quantitatively measure phenotypes from thousands of images automatically.
 - [CellRanger/7.1.0](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CellRanger/CellRanger-7.1.0.eb)
Cell Ranger is a set of analysis pipelines that process Chromium
 single-cell RNA-seq output to align reads, generate gene-cell matrices and perform
 clustering and gene expression analysis.
 - [CellRanger-ARC/2.0.0](https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/pipelines/latest/what-is-cell-ranger-arc)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CellRanger-ARC/CellRanger-ARC-2.0.0.eb)
Cell Ranger ARC is a set of analysis pipelines that process
 Chromium Single Cell Multiome ATAC + Gene Expression sequencing data to generate a
 variety of analyses pertaining to gene expression, chromatin accessibility and
 their linkage. Furthermore, since the ATAC and gene expression measurements are on
 the very same cell, we are able to perform analyses that link chromatin
 accessibility and gene expression.
 - [CellRanger-ATAC/2.0.0](https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac)
Cell Ranger ATAC is a set of analysis pipelines that process 
Chromium Single Cell ATAC data.
 - [CellRank/1.5.1-foss-2020b](https://cellrank.readthedocs.io/en/stable/)
CellRank is a toolkit to uncover cellular dynamics based on
 Markov state modeling of single-cell data.  It contains two main modules:
kernels compute cell-cell transition probabilities and estimators generate
hypothesis based on these. 
 - [Cellpose/0.6.5-fosscuda-2020b](http://cellpose.org/)
Cellpose is an anatomical segmentation algorithm written in Python 3 by Carsen 
Stringer and Marius Pachitariu.
 - [Cereal/1.3.2-GCCcore-12.2.0](https://uscilab.github.io/cereal/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/Cereal/Cereal-1.3.2-GCCcore-12.2.0.eb)
cereal is a header-only C++11 serialization library. cereal takes arbitrary data types and reversibly 
turns them into different representations, such as compact binary encodings, XML, or JSON. cereal was designed to be
fast, light-weight, and easy to extend
 - [Cgl/0.60.7-foss-2022b](https://github.com/coin-or/Cgl)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/Cgl/Cgl-0.60.7-foss-2022b.eb)
The COIN-OR Cut Generation Library (Cgl) is a collection of cut generators that
can be used with other COIN-OR packages that make use of cuts, such as, among
others, the linear solver Clp or the mixed integer linear programming solvers
Cbc or BCP. Cgl uses the abstract class OsiSolverInterface (see Osi) to use or
communicate with a solver. It does not directly call a solver.
 - [Check/0.15.2-GCCcore-10.2.0](https://libcheck.github.io/check/)
Check is a unit testing framework for C. It features a simple interface for
defining unit tests, putting little in the way of the developer. Tests are
run in a separate address space, so both assertion failures and code errors
that cause segmentation faults or other signals can be caught. Test results
are reportable in the following: Subunit, TAP, XML, and a generic logging
format.
 - [Circos/0.69-6-GCCcore-7.3.0-Perl-5.28.0](http://www.circos.ca/)
Circos is a software package for visualizing data and information.
 It visualizes data in a circular layout
 - [Cirro/0.6.11-foss-2021b](https://github.com/FredHutch/Cirro-client)
A Python 3.8+ library for the Cirro platform.
 - [Clamp/1.6.4](http://www.clamp.uth.edu/)

 - [Clang/16.0.4-GCCcore-12.2.0](https://clang.llvm.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/Clang/Clang-16.0.4-GCCcore-12.2.0.eb)
C, C++, Objective-C compiler, based on LLVM.  Does not
 include C++ standard library -- use libstdc++ from GCC.
 - [Clp/1.17.8-foss-2022b](https://github.com/coin-or/Clp)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/Clp/Clp-1.17.8-foss-2022b.eb)
Clp (Coin-or linear programming) is an open-source linear programming solver.
It is primarily meant to be used as a callable library, but a basic,
stand-alone executable version is also available.
 - [Clustal-Omega/1.2.4-GCC-8.3.0](http://www.clustal.org/omega/)
Clustal Omega is a multiple sequence alignment
 program for proteins. It produces biologically meaningful multiple
 sequence alignments of divergent sequences. Evolutionary relationships
 can be seen via viewing Cladograms or Phylograms 
 - [ClustalW2/2.1-foss-2019b](https://www.ebi.ac.uk/Tools/msa/clustalw2/)
ClustalW2 is a general purpose multiple sequence alignment program for DNA or proteins.
 - [Cluster-Buster/0.0-GCC-12.2.0](https://github.com/weng-lab/cluster-buster)
Cluster-Buster is a program for finding interesting functional regions,
 such as transcriptional enhancers, in DNA sequences.
 - [CoinUtils/2.11.9-GCC-12.2.0](https://github.com/coin-or/CoinUtils)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CoinUtils/CoinUtils-2.11.9-GCC-12.2.0.eb)
CoinUtils (Coin-OR Utilities) is an open-source collection of classes and
functions that are generally useful to more than one COIN-OR project.
 - [Control-FREEC/11.5-GCC-8.3.0](https://github.com/BoevaLab/FREEC)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/Control-FREEC/Control-FREEC-11.5-GCC-8.3.0.eb)
Copy number and genotype annotation from whole 
genome and whole exome sequencing data.
 - [Coreutils/8.32-GCCcore-8.3.0](http://www.gnu.org/software/coreutils/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/Coreutils/Coreutils-8.32-GCCcore-8.3.0.eb)
The GNU Core Utilities are the basic file, shell and text manipulation utilities of the
 GNU operating system.  These are the core utilities which are expected to exist on every operating system.

 - [CppUnit/1.15.1-GCCcore-10.2.0](https://freedesktop.org/wiki/Software/cppunit/)
CppUnit is the C++ port of the famous JUnit framework for unit testing.

 - [Cufflinks/2.2.1-foss-2018b](http://cole-trapnell-lab.github.io/cufflinks/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/Cufflinks/Cufflinks-2.2.1-foss-2018b.eb)
Transcript assembly, differential expression, and differential regulation for RNA-Seq
 - [Cython/0.29.22-GCCcore-10.2.0](https://cython.org/)
Cython is an optimising static compiler for both the Python programming
language and the extended Cython programming language (based on Pyrex).

 - [DB/18.1.40-GCCcore-11.3.0](https://www.oracle.com/technetwork/products/berkeleydb)
Berkeley DB enables the development of custom data management
 solutions, without the overhead traditionally associated with such custom
 projects.
 - [DBD-mysql/4.050-GCC-10.2.0](https://metacpan.org/pod/distribution/DBD-mysql/lib/DBD/mysql.pm)
Perl binding for MySQL
 - [DB_File/1.857-GCCcore-11.2.0](https://perldoc.perl.org/DB_File.html)
Perl5 access to Berkeley DB version 1.x.
 - [DBus/1.15.2-GCCcore-12.2.0](https://dbus.freedesktop.org/)
D-Bus is a message bus system, a simple way for applications to talk
 to one another.  In addition to interprocess communication, D-Bus helps
 coordinate process lifecycle; it makes it simple and reliable to code
 a "single instance" application or daemon, and to launch applications
 and daemons on demand when their services are needed.

 - [DIAMOND/2.0.13-GCC-11.2.0](https://github.com/bbuchfink/diamond)
Accelerated BLAST compatible local sequence aligner
 - [DeepCell/0.11.1-foss-2021b-CUDA-11.4.1](https://deepcell.readthedocs.io/en/master/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/d/DeepCell/DeepCell-0.11.1-foss-2021b-CUDA-11.4.1.eb)
deepcell-tf is a deep learning library for single-cell analysis of biological
 images.This library allows users to apply pre-existing models to imaging data as well as to
 develop new deep learning models for single-cell analysis.
 - [DeepTCR/2.1.27-foss-2021b-CUDA-11.4.1](https://sidhomj.github.io/DeepTCR/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/d/DeepTCR/DeepTCR-2.1.27-foss-2021b-CUDA-11.4.1.eb)
DeepTCR is a python package that has a collection of unsupervised and supervised deep
 learning methods to parse TCRSeq data.
 - [Delly/0.9.1-gompi-2020b](https://github.com/dellytools/delly/)
Delly is an integrated structural variant (SV) prediction
method that can discover, genotype and visualize deletions, tandem duplications,
inversions and translocations at single-nucleotide resolution in short-read
massively parallel sequencing data. It uses paired-ends, split-reads and
read-depth to sensitively and accurately delineate genomic rearrangements
throughout the genome.
 - [DiMSum/1.2.9-foss-2021b-R-4.2.0](https://github.com/lehner-lab/DiMSum)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/d/DiMSum/DiMSum-1.2.9-foss-2021b-R-4.2.0.eb)
An error model and pipeline for analyzing deep mutational scanning (DMS) data and diagnosing common experimental pathologies.
 - [Doxygen/1.9.5-GCCcore-12.2.0](https://www.doxygen.org)
Doxygen is a documentation system for C++, C, Java, Objective-C, Python,
 IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some
 extent D.

 - [EIGENSOFT/7.2.1-foss-2019b](https://www.hsph.harvard.edu/alkes-price/software/)
The EIGENSOFT package combines functionality from our population genetics methods (Patterson et al. 
2006)  and our EIGENSTRAT stratification correction method (Price et al. 2006). The EIGENSTRAT method uses principal 
components  analysis to explicitly model ancestry differences between cases and controls along continuous axes of 
variation;  the resulting correction is specific to a candidate marker’s variation in frequency across ancestral 
populations,  minimizing spurious associations while maximizing power to detect true associations. The EIGENSOFT 
package has a built-in plotting script and supports multiple file formats and quantitative phenotypes.
 - [EMAN2/2.3-foss-2019b-Python-2.7.16](https://blake.bcm.edu/emanwiki/EMAN2)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/e/EMAN2/EMAN2-2.3-foss-2019b-Python-2.7.16.eb)
EMAN2 is the successor to EMAN1. It is a broadly based greyscale scientific image processing suite
 with a primary focus on processing data from transmission electron microscopes. 
 - [EMBOSS/6.6.0-foss-2018b](http://emboss.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/e/EMBOSS/EMBOSS-6.6.0-foss-2018b.eb)
EMBOSS is 'The European Molecular Biology Open Software Suite'
 EMBOSS is a free Open Source software analysis package specially developed for
 the needs of the molecular biology (e.g. EMBnet) user community.
 - [EPACTS/3.3.2-foss-2020b](https://github.com/statgen/EPACTS)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/e/EPACTS/EPACTS-3.3.2-foss-2020b.eb)
EPACTS is a versatile software pipeline to perform various statistical tests for identifying
 genome-wide association from sequence data through a user-friendly interface, both to scientific analysts and
 to method developers.
 - [ESS/18.10.2](http://ess.r-project.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/e/ESS/ESS-18.10.2.eb)
Emacs Speaks Statistics (ESS) is an add-on package for emacs
 text editors such as GNU Emacs and XEmacs. It is designed to support editing
 of scripts and interaction with various statistical analysis programs such as
 R, S-Plus, SAS, Stata and OpenBUGS/JAGS.
 - [EasyBuild/4.8.1](https://easybuilders.github.io/easybuild)
EasyBuild is a software build and installation framework
 written in Python that allows you to install software in a structured,
 repeatable and robust way.
 - [Eigen/3.4.0-GCCcore-11.3.0](https://eigen.tuxfamily.org)
Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers,
 and related algorithms.
 - [Emacs/27.1-GCCcore-10.2.0](https://www.gnu.org/software/emacs/)
GNU Emacs is an extensible, customizable text editor--and more.
 At its core is an interpreter for Emacs Lisp, a dialect of the Lisp programming
 language with extensions to support text editing.
 - [Emacs/27.1-GCCcore-10.2.0](https://www.gnu.org/software/emacs/)
GNU Emacs is an extensible, customizable text editor--and more.
 At its core is an interpreter for Emacs Lisp, a dialect of the Lisp programming
 language with extensions to support text editing.
 - [FASTX-Toolkit/0.0.14-foss-2018b](http://hannonlab.cshl.edu/fastx_toolkit/)
The FASTX-Toolkit is a collection of command line tools for 
 Short-Reads FASTA/FASTQ files preprocessing.
 - [FFTW/3.3.10-GCC-11.3.0](https://www.fftw.org)
FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)
in one or more dimensions, of arbitrary input size, and of both real and complex data.
 - [FFTW.MPI/3.3.10-gompi-2022b](https://www.fftw.org)
FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)
in one or more dimensions, of arbitrary input size, and of both real and complex data.
 - [FFmpeg/5.1.2-GCCcore-12.2.0](https://www.ffmpeg.org/)
A complete, cross-platform solution to record, convert and stream audio and video.
 - [FIt-SNE/1.1.0-gompi-2018b](https://github.com/KlugerLab/FIt-SNE)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/FIt-SNE/FIt-SNE-1.1.0-gompi-2018b.eb)
t-distributed stochastic neighbor embedding (t-SNE) is widely used for
 visualizing single-cell RNA-sequencing (scRNA-seq) data, but it scales poorly to large
 datasets. We dramatically accelerate t-SNE, obviating the need for data downsampling, and
 hence allowing visualization of rare cell populations. Furthermore, we implement a
 heatmap-style visualization for scRNA-seq based on one-dimensional t-SNE for simultaneously
 visualizing the expression patterns of thousands of genes.
 - [FLAC/1.4.2-GCCcore-12.2.0](https://xiph.org/flac/)
FLAC stands for Free Lossless Audio Codec, an audio format similar to MP3, but lossless, meaning
that audio is compressed in FLAC without any loss in quality.
 - [FLASH/2.2.00-foss-2018b](https://ccb.jhu.edu/software/FLASH/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/FLASH/FLASH-2.2.00-foss-2018b.eb)
FLASH (Fast Length Adjustment of SHort reads) is a very fast
and accurate software tool to merge paired-end reads from next-generation
sequencing experiments. FLASH is designed to merge pairs of reads when the
original DNA fragments are shorter than twice the length of reads. The
resulting longer reads can significantly improve genome assemblies. They can
also improve transcriptome assembly when FLASH is used to merge RNA-seq data.

 - [FLTK/1.3.5-GCCcore-10.2.0](https://www.fltk.org)
FLTK is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows,
 and MacOS X. FLTK provides modern GUI functionality without the bloat and supports 3D graphics via OpenGL
 and its built-in GLUT emulation.
 - [FSL/5.0.11-foss-2018b](http://www.fmrib.ox.ac.uk/fsl/)
FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data.
 - [FTGL/2.3-GCCcore-10.2.0](http://ftgl.sourceforge.net/docs/html/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/FTGL/FTGL-2.3-GCCcore-10.2.0.eb)
FTGL is a free open source library to enable developers to use arbitrary
fonts in their OpenGL (www.opengl.org) applications. 
 - [FastANI/1.1-foss-2018b](http://www.iodbc.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/FastANI/FastANI-1.1-foss-2018b.eb)
FastANI is developed for fast alignment-free computation of
 whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean
 nucleotide identity of orthologous gene pairs shared between two microbial
 genomes. FastANI supports pairwise comparison of both complete and draft
 genome assemblies.
 - [FastQC/0.11.9-Java-11](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
FastQC is a quality control application for high throughput
sequence data. It reads in sequence data in a variety of formats and can either
provide an interactive application to review the results of several different
QC checks, or create an HTML based report which can be integrated into a
pipeline.
 - [FastTree/2.1.11-GCCcore-11.2.0](http://www.microbesonline.org/fasttree/)
FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide
 or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of
 time and memory. 
 - [Fiji/2.9.0-Java-1.8](https://fiji.sc)
Fiji is an image processing package—a 'batteries-included' distribution of
 ImageJ, bundling a lot of plugins which facilitate scientific image analysis.
This release is based on ImageJ-2.1.0 and Fiji-2.1.1
 - [File-ReadBackwards/1.05-GCCcore-7.3.0-Perl-5.28.0](https://metacpan.org/pod/File::ReadBackwards)
This Perl module reads a file backwards line by line.
 - [FlashPCA2/2.0-GCC-10.2.0](https://github.com/gabraham/flashpca)
FlashPCA performs fast principal component analysis (PCA) of single nucleotide
 polymorphism (SNP) data.
 - [Flask/1.1.2-GCCcore-10.2.0](https://www.palletsprojects.com/p/flask/)
Flask is a lightweight WSGI web application framework. It is designed to make
getting started quick and easy, with the ability to scale up to complex
applications.
This module includes the Flask extensions: Flask-Cors
 - [FlexiBLAS/3.2.1-GCC-12.2.0](https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release)
FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation
used by a program without recompiling or relinking it.
 - [FreeImage/3.18.0-GCCcore-11.2.0](http://freeimage.sourceforge.net)
FreeImage is an Open Source library project for developers who would like to support popular graphics
image formats like PNG, BMP, JPEG, TIFF and others as needed by today's multimedia applications. FreeImage is easy to
use, fast, multithreading safe.
 - [FreeTDS/1.3.6-GCCcore-11.2.0](https://www.freetds.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/FreeTDS/FreeTDS-1.3.6-GCCcore-11.2.0.eb)
FreeTDS is a set of libraries for Unix and Linux that allows your programs to
 natively talk to Microsoft SQL Server and Sybase databases.
 - [FriBidi/1.0.12-GCCcore-11.3.0](https://github.com/fribidi/fribidi)
The Free Implementation of the Unicode Bidirectional Algorithm.

 - [GATK/4.2.6.1-GCCcore-11.2.0](https://www.broadinstitute.org/gatk/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GATK/GATK-4.2.6.1-GCCcore-11.2.0.eb)
The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute
 to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,
 with a primary focus on variant discovery and genotyping as well as strong emphasis on
 data quality assurance. Its robust architecture, powerful processing engine and
 high-performance computing features make it capable of taking on projects of any size.
 - [GCC/12.2.0](https://gcc.gnu.org/)
The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
 as well as libraries for these languages (libstdc++, libgcj,...).
 - [GCCcore/12.2.0](https://gcc.gnu.org/)
The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
 as well as libraries for these languages (libstdc++, libgcj,...).
 - [GCTA/1.92.2beta](https://cnsgenomics.com/software/gcta)
GCTA (Genome-wide Complex Trait Analysis) was originally designed to
 estimate the proportion of phenotypic variance explained by all genome-wide SNPs for
 complex traits (the GREML method), and has subsequently extended for many other analyses
 to better understand the genetic architecture of complex traits.
 - [GConf/3.2.6-GCCcore-11.2.0](https://gitlab.gnome.org/iainl/gconf)
GConf is a system for storing application preferences.
 It is intended for user preferences; not configuration
 of something like Apache, or arbitrary data storage.
 - [GD/2.69-GCCcore-7.3.0-Perl-5.28.0](https://github.com/lstein/Perl-GD)
GD.pm
 - [GDAL/3.6.2-foss-2022b](https://www.gdal.org)
GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
 Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
 to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
 data translation and processing.
 - [GDB/11.1-GCCcore-11.2.0](https://www.gnu.org/software/gdb/gdb.html)
The GNU Project Debugger
 - [GDRCopy/2.3-GCCcore-12.2.0](https://github.com/NVIDIA/gdrcopy)
A low-latency GPU memory copy library based on NVIDIA GPUDirect RDMA technology.
 - [GEOS/3.11.1-GCC-12.2.0](https://trac.osgeo.org/geos)
GEOS (Geometry Engine
 - [GISTIC/2.0.23-GCCcore-8.3.0](www.broadinstitute.org/cancer/cga/gistc)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GISTIC/GISTIC-2.0.23-GCCcore-8.3.0.eb)
GISTIC is a tool to identify genes targeted by somatic copy-number alterations (SCNAs) that drive cancer growth. By separating SCNA profiles into underlying arm-level and focal alterations, GISTIC estimates the background rates for each category as well as defines the boundaries of SCNA regions.
 - [GLFW/3.3.4-GCCcore-11.2.0](https://www.glfw.org)
GLFW is an Open Source, multi-platform library for OpenGL,
OpenGL ES and Vulkan development on the desktop
 - [GLIPH2/0.1](http://50.255.35.37:8080  )
GLIPH 2 clusters TCRs that are predicted to bind the same MHC-restricted peptide antigen.
 - [GLPK/5.0-GCCcore-12.2.0](https://www.gnu.org/software/glpk/)
The GLPK (GNU Linear Programming Kit) package is intended for
 solving large-scale linear programming (LP),
 mixed integer programming (MIP), and other related problems.
 It is a set of routines written in ANSI C
 and organized in the form of a callable library.
 - [GLib/2.75.0-GCCcore-12.2.0](https://www.gtk.org/)
GLib is one of the base libraries of the GTK+ project
 - [GLibmm/2.49.7-GCCcore-8.3.0](https://www.gtk.org/)
C++ bindings for Glib
 - [GMAP-GSNAP/2018-07-04-foss-2018b](http://research-pub.gene.com/gmap/)
GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences
 GSNAP: Genomic Short-read Nucleotide Alignment Program
 - [GMP/6.2.1-GCCcore-12.2.0](https://gmplib.org/)
GMP is a free library for arbitrary precision arithmetic, operating on signed
 integers, rational numbers, and floating point numbers.

 - [GMime/3.2.7-GCCcore-8.3.0](http://spruce.sourceforge.net/gmime/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GMime/GMime-3.2.7-GCCcore-8.3.0.eb)
The GMime package contains a set of utilities for parsing and
 creating messages using the Multipurpose Internet Mail Extension (MIME) as
 defined by the applicable RFCs.
 - [GObject-Introspection/1.74.0-GCCcore-12.2.0](https://gi.readthedocs.io/en/latest/)
GObject introspection is a middleware layer between C libraries
 (using GObject) and language bindings. The C library can be scanned at
 compile time and generate a metadata file, in addition to the actual
 native C library. Then at runtime, language bindings can read this
 metadata and automatically provide bindings to call into the C library.
 - [GRIDSS/2.13.2-foss-2021b](https://github.com/PapenfussLab/gridss/wiki/GRIDSS-Documentation)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GRIDSS/GRIDSS-2.13.2-foss-2021b.eb)
GRIDSS is a module software suite containing tools useful for
the detection of genomic rearrangements. GRIDSS includes a genome-wide
break-end assembler, as well as a structural variation caller for Illumina
sequencing data. GRIDSS calls variants based on alignment-guided positional de
Bruijn graph genome-wide break-end assembly, split read, and read pair
evidence.
 - [GROMACS/2021-foss-2020b](https://www.gromacs.org)
GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the
Newtonian equations of motion for systems with hundreds to millions of
particles.

This is a CPU only build, containing both MPI and threadMPI builds
for both single and double precision.

It also contains the gmxapi extension for the single precision MPI build.

 - [GSEA/4.3.2](https://www.gsea-msigdb.org/gsea/index.jsp)
Gene Set Enrichment Analysis (GSEA) is a computational method that 
determines whether an a priori defined set of genes shows statistically
significant, concordant differences between two biological states
(e.g. phenotypes). 
 - [GSL/2.7-GCCcore-12.2.0](https://www.gnu.org/software/gsl/)
The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.
 The library provides a wide range of mathematical routines such as random number generators, special functions
 and least-squares fitting.
 - [GST-plugins-base/1.22.1-GCCcore-12.2.0](https://gstreamer.freedesktop.org/)
GStreamer is a library for constructing graphs of media-handling
 components. The applications it supports range from simple
 Ogg/Vorbis playback, audio/video streaming to complex audio
 (mixing) and video (non-linear editing) processing.
 - [GStreamer/1.22.1-GCCcore-12.2.0](https://gstreamer.freedesktop.org/)
GStreamer is a library for constructing graphs of media-handling
 components. The applications it supports range from simple
 Ogg/Vorbis playback, audio/video streaming to complex audio
 (mixing) and video (non-linear editing) processing.
 - [GTK+/3.24.23-GCCcore-10.2.0](https://developer.gnome.org/gtk3/stable/)
GTK+ is the primary library used to construct user interfaces in GNOME. It
 provides all the user interface controls, or widgets, used in a common
 graphical application. Its object-oriented API allows you to construct
 user interfaces without dealing with the low-level details of drawing and
 device interaction.

 - [GTK3/3.24.35-GCCcore-12.2.0](https://developer.gnome.org/gtk3/stable/)
GTK+ is the primary library used to construct user interfaces in GNOME. It
 provides all the user interface controls, or widgets, used in a common
 graphical application. Its object-oriented API allows you to construct
 user interfaces without dealing with the low-level details of drawing and
 device interaction.

 - [GTS/0.7.6-foss-2018b](http://gts.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GTS/GTS-0.7.6-foss-2018b.eb)
GTS stands for the GNU Triangulated Surface Library.
 It is an Open Source Free Software Library intended to provide a set of useful
 functions to deal with 3D surfaces meshed with interconnected triangles.
 - [Garnett/20220903-foss-2021b-R-4.2.2](https://cole-trapnell-lab.github.io/garnett/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/Garnett/Garnett-20220903-foss-2021b-R-4.2.2.eb)
Garnett is a software package that faciliates automated cell type classification from single-cell expression data.
 - [Gdk-Pixbuf/2.42.10-GCCcore-12.2.0](https://docs.gtk.org/gdk-pixbuf/)
The Gdk Pixbuf is a toolkit for image loading and pixel buffer manipulation.
 It is used by GTK+ 2 and GTK+ 3 to load and manipulate images. In the past it
 was distributed as part of GTK+ 2 but it was split off into a separate package
 in preparation for the change to GTK+ 3.

 - [Geekbench/5.4.1](https://Geekbench.com)
Geekbench 5 is a cross-platform benchmark that measures your system's performance 
with the press of a button.
 - [GenomeSTRiP/2.00.1958-GCCcore-8.3.0-Java-11](https://www.broadinstitute.org/software/genomestrip)
Genome STRiP (Genome STRucture In Populations) is a suite of
tools for discovery and genotyping of structural variation using whole-genome
sequencing data. The methods used in Genome STRiP are designed to find shared
variation using data from multiple individuals. Genome STRiP looks both across
and within a set of sequenced genomes to detect variation.
 - [Ghostscript/10.0.0-GCCcore-12.2.0](https://ghostscript.com)
Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
 different targets. It used to be part of the cups printing stack, but is no longer used for that.
 - [Giotto/1.0.0-foss-2019b-R-4.0.2](https://rubd.github.io/Giotto_site/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/Giotto/Giotto-1.0.0-foss-2019b-R-4.0.2.eb)
The Giotto package consists of two modules, Giotto Analyzer and Viewer, which provide tools
 to process, analyze and visualize single-cell spatial expression data.
 - [GitPython/3.1.24-GCCcore-11.2.0](https://gitpython.readthedocs.org)
GitPython is a python library used to interact with Git repositories 
 - [Globus-CLI/3.6.0-GCCcore-11.2.0](https://docs.globus.org/cli/)
A Command Line Wrapper over the Globus SDK for Python, which provides an interface to Globus services
 from the shell, and is suited to both interactive and simple scripting use cases.
 - [GlobusConnectPersonal/3.2.0-GCCcore-11.2.0](https://www.globus.org/globus-connect-personal)
Globus Connect Personal turns your laptop or other personal computer into a 
Globus endpoint with a just a few clicks. With Globus Connect Personal you can 
share and transfer files to/from a local machine—campus server, desktop computer
or laptop—even if it's behind a firewall and you don't have administrator 
privileges.

 - [GnuTLS/3.7.3-GCCcore-11.2.0](https://www.gnutls.org)
GnuTLS is a secure communications library implementing the SSL, TLS
 and DTLS protocols and technologies around them. It provides a simple
 C language application programming interface (API) to access the secure
 communications protocols as well as APIs to parse and write X.509, PKCS #12,
 OpenPGP and other required structures. It is aimed to be portable
 and efficient with focus on security and interoperability.
 - [Go/1.18.1](https://www.golang.org)
Go is an open source programming language that makes it easy to build
 simple, reliable, and efficient software.
 - [GoPeaks/1.0.0](https://github.com/maxsonBraunLab/gopeaks)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GoPeaks/GoPeaks-1.0.0.eb)
GoPeaks is a peak caller designed for CUT&TAG/CUT&RUN sequencing data.
 - [Graphene/1.10.8-GCCcore-12.2.0](https://ebassi.github.io/graphene/)
Graphene is a thin layer of types for graphic libraries
 - [GraphicsMagick/1.3.31-foss-2018b](http://www.graphicsmagick.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GraphicsMagick/GraphicsMagick-1.3.31-foss-2018b.eb)
GraphicsMagick is the swiss army knife of image processing.
 - [Graphviz/8.1.0-GCCcore-12.2.0](https://www.graphviz.org/)
Graphviz is open source graph visualization software. Graph visualization
 is a way of representing structural information as diagrams of
 abstract graphs and networks. It has important applications in networking,
 bioinformatics,  software engineering, database and web design, machine learning,
 and in visual interfaces for other technical domains.
 - [Guile/3.0.7-GCCcore-11.2.0](https://www.gnu.org/software/guile/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/Guile/Guile-3.0.7-GCCcore-11.2.0.eb)
Guile is a programming language, designed to help programmers create flexible
 applications that can be extended by users or other programmers with plug-ins,
 modules, or scripts.

 - [Gurobi/10.0.2](https://gurobi.com)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/Gurobi/Gurobi-10.0.2.eb)
The Gurobi Optimizer allows users to state their toughest
 business problems as mathematical models, and then automatically considers
 billions or even trillions of possible solutions to find the best one. Our
 solver can be used as both a decision-making assistant, to help guide the
 choices of a skilled expert, or as a fully automated tool to make decisions
 with no human intervention.
 - [HDF/4.2.15-GCCcore-11.2.0](https://www.hdfgroup.org/products/hdf4/)
HDF (also known as HDF4) is a library and multi-object file format for
 storing and managing data between machines.

 - [HDF5/1.14.0-gompi-2022b](https://portal.hdfgroup.org/display/support)
HDF5 is a data model, library, and file format for storing and managing data.
 It supports an unlimited variety of datatypes, and is designed for flexible
 and efficient I/O and for high volume and complex data.
 - [HH-suite/3.3.0-gompi-2020b](https://github.com/soedinglab/hh-suite)
The HH-suite is an open-source software package 
for sensitive protein sequence searching based on the pairwise 
alignment of hidden Markov models (HMMs).
 - [HISAT2/2.2.1-gompi-2021b](https://daehwankimlab.github.io/hisat2)
HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads
 (both DNA and RNA) against the general human population (as well as against a single reference genome).
 - [HLA-HD/1.6.1-GCC-11.2.0](https://www.genome.med.kyoto-u.ac.jp/HLA-HD/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/h/HLA-HD/HLA-HD-1.6.1-GCC-11.2.0.eb)
HLA-HD (HLA typing from High-quality Dictionary) can accurately
 determine HLA alleles with 6-digit precision from NGS data (fastq format).
 RNA-Seq data can also be applied.
 - [HMMER/3.3.2-gompi-2021b](http://hmmer.org/)
HMMER is used for searching sequence databases for homologs
 of protein sequences, and for making protein sequence alignments. It
 implements methods using probabilistic models called profile hidden Markov
 models (profile HMMs).  Compared to BLAST, FASTA, and other sequence
 alignment and database search tools based on older scoring methodology,
 HMMER aims to be significantly more accurate and more able to detect remote
 homologs because of the strength of its underlying mathematical models. In the
 past, this strength came at significant computational expense, but in the new
 HMMER3 project, HMMER is now essentially as fast as BLAST.
 - [HOME/1.0.0-foss-2019b-Python-3.7.4](https://github.com/ListerLab/HOME)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/h/HOME/HOME-1.0.0-foss-2019b-Python-3.7.4.eb)
HOME (histogram of methylation) is a python package for differential methylation region (DMR)
 identification. The method uses histogram of methylation features and the linear Support Vector Machine (SVM)
 to identify DMRs from whole genome bisulfite sequencing (WGBS) data.
 - [HTSeq/0.11.0-foss-2018b-Python-2.7.15](http://www-huber.embl.de/users/anders/HTSeq/)
A framework to process and analyze data from high-throughput sequencing (HTS) assays
 - [HTSlib/1.17-GCC-12.2.0](https://www.htslib.org/)
A C library for reading/writing high-throughput sequencing data.
 This package includes the utilities bgzip and tabix
 - [Hail/0.2.64-foss-2020b](https://hail.is/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/h/Hail/Hail-0.2.64-foss-2020b.eb)
Hail is an open-source, general-purpose, Python-based data analysis tool with
 additional data types and methods for working with genomic data.
 - [HarfBuzz/5.3.1-GCCcore-12.2.0](https://www.freedesktop.org/wiki/Software/HarfBuzz)
HarfBuzz is an OpenType text shaping engine.
 - [HiC-Pro/3.1.0-foss-2021b](http://nservant.github.io/HiC-Pro)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/h/HiC-Pro/HiC-Pro-3.1.0-foss-2021b.eb)
HiC-Pro was designed to process Hi-C data, from raw fastq files (paired-end Illumina data) 
 to the normalized contact maps.
 - [Highway/1.0.3-GCCcore-12.2.0](https://github.com/google/highway)
Highway is a C++ library for SIMD (Single Instruction, Multiple Data), i.e. applying the same
operation to 'lanes'.
 - [Homer/4.11-Perl-5.30.0](http://homer.ucsd.edu/homer/)
HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and ChIP-Seq analysis. It is a collection of command line programs for unix-style operating systems written in mostly perl and c++. Homer was primarily written as a de novo motif discovery algorithm that is well suited for finding 8-12 bp motifs in large scale genomics data.
 - [Hypre/2.21.0-foss-2021a](https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods)
Hypre is a library for solving large, sparse linear systems of equations on massively
 parallel computers. The problems of interest arise in the simulation codes being developed at LLNL
 and elsewhere to study physical phenomena in the defense, environmental, energy, and biological sciences.
 - [ICU/72.1-GCCcore-12.2.0](https://icu.unicode.org)
ICU is a mature, widely used set of C/C++ and Java libraries providing Unicode and Globalization
 support for software applications.
 - [IDBA-UD/1.1.3-foss-2018b](http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/)
IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads
 Sequencing data with Highly Uneven Sequencing Depth. It is an extension of IDBA algorithm.
 IDBA-UD also iterates from small k to a large k. In each iteration, short and low-depth
 contigs are removed iteratively with cutoff threshold from low to high to reduce the errors
 in low-depth and high-depth regions. Paired-end reads are aligned to contigs and assembled
 locally to generate some missing k-mers in low-depth regions. With these technologies, IDBA-UD
 can iterate k value of de Bruijn graph to a very large value with less gaps and less branches
 to form long contigs in both low-depth and high-depth regions.
 - [IGV/2.15.4-Java-11](https://www.broadinstitute.org/software/igv/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/IGV/IGV-2.15.4-Java-11.eb)
This package contains command line utilities for
 preprocessing, computing feature count density (coverage),  sorting, and
 indexing data files.
 - [IGVTools/2.4.16-Java-1.8](http://www.broadinstitute.org/software/igv/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/IGVTools/IGVTools-2.4.16-Java-1.8.eb)
This package contains command line utilities for preprocessing, 
 computing feature count density (coverage),  sorting, and indexing data files.
 See also http://www.broadinstitute.org/software/igv/igvtools_commandline. 
 - [IPython/8.14.0-GCCcore-12.2.0](https://ipython.org/index.html)
IPython provides a rich architecture for interactive computing with:
 Powerful interactive shells (terminal and Qt-based).
 A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media.
 Support for interactive data visualization and use of GUI toolkits.
 Flexible, embeddable interpreters to load into your own projects.
 Easy to use, high performance tools for parallel computing.
 - [IRIS/2.0.1](https://github.com/Xinglab/IRIS )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/IRIS/IRIS-2.0.1.eb)
IRIS 
Isoform peptides from RNA splicing for Immunotherapy target Screening 
 - [ISA-L/2.30.0-GCCcore-11.2.0](https://github.com/intel/isa-l)
Intelligent Storage Acceleration Library
 - [ISL/0.24-GCCcore-11.2.0](http://isl.gforge.inria.fr/)
isl is a library for manipulating sets and relations of integer points bounded by linear constraints.
 - [IgBLAST/1.22.0-x64-linux](https://ncbi.github.io/igblast)
IgBLAST faclilitates the analysis of immunoglobulin and T cell receptor variable domain sequences.
 - [ImageMagick/7.1.0-53-GCCcore-12.2.0](https://www.imagemagick.org/)
ImageMagick is a software suite to create, edit, compose, or convert bitmap images
 - [Imath/3.1.6-GCCcore-12.2.0](https://imath.readthedocs.io/en/latest/)
Imath is a C++ and python library of 2D and 3D vector, matrix, and math operations for computer graphics

 - [Infernal/1.1.4-foss-2022b](http://eddylab.org/infernal/)
Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases
 for RNA structure and sequence similarities.
 - [JAGS/4.3.2-foss-2022b](http://mcmc-jags.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/j/JAGS/JAGS-4.3.2-foss-2022b.eb)
JAGS is Just Another Gibbs Sampler.  It is a program for analysis
 of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  
 - [JUnit/4.12-Java-1.8](http://sourceforge.net/projects/junit)
A programmer-oriented testing framework for Java.
 - [Jansson/2.13.1-GCC-11.2.0](https://www.digip.org/jansson/)
Jansson is a C library for encoding, decoding and manipulating JSON data.
 Its main features and design principles are:
 * Simple and intuitive API and data model
 * Comprehensive documentation
 * No dependencies on other libraries
 * Full Unicode support (UTF-8)
 * Extensive test suite
 - [JasPer/4.0.0-GCCcore-12.2.0](https://www.ece.uvic.ca/~frodo/jasper/)
The JasPer Project is an open-source initiative to provide a free
 software-based reference implementation of the codec specified in
 the JPEG-2000 Part-1 standard.

 - [Java/15.0.1](http://openjdk.java.net)
Java Platform, Standard Edition (Java SE) lets you develop and deploy
 Java applications on desktops and servers.
 - [Jellyfish/2.3.0-GCC-10.2.0](http://www.genome.umd.edu/jellyfish.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/j/Jellyfish/Jellyfish-2.3.0-GCC-10.2.0.eb)
Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA.
 - [JsonCpp/1.9.4-GCCcore-10.2.0](https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html)
JsonCpp is a C++ library that allows manipulating JSON values,
 including serialization and deserialization to and from strings. It can also preserve existing comment in
 unserialization/serialization steps, making it a convenient format to store user input files. 
 - [Judy/1.0.5-GCCcore-8.3.0](http://judy.sourceforge.net/)
A C library that implements a dynamic array.
 - [Julia/1.7.3-linux-x86_64](https://julialang.org)
Julia is a high-level, high-performance dynamic programming language for numerical computing
 - [JupyterHub/4.0.1-GCCcore-12.2.0](https://jupyter.org)
JupyterHub is a multiuser version of the Jupyter (IPython) notebook designed
 for centralized deployments in companies, university classrooms and research labs.
 - [JupyterLab/4.0.3-GCCcore-12.2.0](https://jupyter.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/j/JupyterLab/JupyterLab-4.0.3-GCCcore-12.2.0.eb)
JupyterLab is the next-generation user interface for Project Jupyter offering all the familiar
 building blocks of the classic Jupyter Notebook (notebook, terminal, text editor, file browser, rich outputs,
 etc.) in a flexible and powerful user interface. JupyterLab will eventually replace the classic Jupyter
 Notebook.
 - [Kalign/3.3.2-GCCcore-11.2.0](https://github.com/TimoLassmann/kalign)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/k/Kalign/Kalign-3.3.2-GCCcore-11.2.0.eb)
Kalign is a fast multiple sequence alignment program for biological sequences.
 - [Kent_tools/20201201-linux.x86_64](http://genome.cse.ucsc.edu/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/k/Kent_tools/Kent_tools-20201201-linux.x86_64.eb)
Jim Kent's tools: collection of tools used by the UCSC genome browser.
 - [Keras/2.4.3-foss-2020b](https://keras.io/)
Keras is a deep learning API written in Python, running on top of the machine learning platform TensorFlow.

 - [Kraken2/2.0.9-beta-gompi-2020a-Perl-5.30.2](http://www.ccb.jhu.edu/software/kraken2/)
Kraken is a system for assigning taxonomic labels to short DNA sequences,
 usually obtained through metagenomic studies. Previous attempts by other
 bioinformatics software to accomplish this task have often used sequence
 alignment or machine learning techniques that were quite slow, leading to
 the development of less sensitive but much faster abundance estimation
 programs. Kraken aims to achieve high sensitivity and high speed by
 utilizing exact alignments of k-mers and a novel classification algorithm.
 - [Krona/2.7.1-GCCcore-9.3.0-Perl-5.30.2](https://github.com/marbl/Krona/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/k/Krona/Krona-2.7.1-GCCcore-9.3.0-Perl-5.30.2.eb)
Krona allows hierarchical data to be explored with zooming, multi-layered pie charts. Krona charts can be created using an Excel template or KronaTools, which includes support for several bioinformatics tools and raw data formats.
 - [LAME/3.100-GCCcore-11.2.0](http://lame.sourceforge.net/)
LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL.
 - [LAPACK/3.10.1-GCC-11.2.0](https://www.netlib.org/lapack/)
LAPACK is written in Fortran90 and provides routines for solving systems of
 simultaneous linear equations, least-squares solutions of linear systems of equations, eigenvalue
 problems, and singular value problems.
 - [LAST/963-foss-2018b](http://last.cbrc.jp/)
LAST finds similar regions between sequences.
 - [LERC/4.0.0-GCCcore-12.2.0](https://github.com/Esri/lerc)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/LERC/LERC-4.0.0-GCCcore-12.2.0.eb)
LERC is an open-source image or raster format which supports rapid encoding and decoding
for any pixel type (not just RGB or Byte). Users set the maximum compression error per pixel while encoding,
so the precision of the original input image is preserved (within user defined error bounds).
 - [LLVM/16.0.6-GCCcore-12.2.0](https://llvm.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/LLVM/LLVM-16.0.6-GCCcore-12.2.0.eb)
The LLVM Core libraries provide a modern source- and target-independent
 optimizer, along with code generation support for many popular CPUs
 (as well as some less common ones!) These libraries are built around a well
 specified code representation known as the LLVM intermediate representation
 ("LLVM IR"). The LLVM Core libraries are well documented, and it is
 particularly easy to invent your own language (or port an existing compiler)
 to use LLVM as an optimizer and code generator.
 - [LMDB/0.9.29-GCCcore-11.2.0](https://symas.com/lmdb)
LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance
 of a pure in-memory database while retaining the persistence of standard disk-based databases.
 - [LZO/2.10-GCCcore-8.3.0](https://www.oberhumer.com/opensource/lzo/)
Portable lossless data compression library
 - [Leptonica/1.83.0-GCCcore-11.2.0](http://www.leptonica.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/Leptonica/Leptonica-1.83.0-GCCcore-11.2.0.eb)
Leptonica is a collection of pedagogically-oriented open source software
 that is broadly useful for image processing and image analysis applications.
 - [LibSoup/3.0.7-GCC-11.2.0](https://wiki.gnome.org/Projects/libsoup)
libsoup is an HTTP client/server library for GNOME. It
uses GObjects and the glib main loop, to integrate well with GNOME
applications, and also has a synchronous API, for use in threaded
applications.
 - [LibTIFF/4.4.0-GCCcore-12.2.0](https://libtiff.gitlab.io/libtiff/)
tiff: Library and tools for reading and writing TIFF data files
 - [LightGBM/2.2.3-foss-2018b](https://github.com/microsoft/LightGBM)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/LightGBM/LightGBM-2.2.3-foss-2018b.eb)
A fast, distributed, high performance gradient boosting (GBT, GBDT, GBRT,
 GBM or MART) framework based on decision tree algorithms, used for ranking, classification
 and many other machine learning tasks.
 - [LittleCMS/2.14-GCCcore-12.2.0](https://www.littlecms.com/)
Little CMS intends to be an OPEN SOURCE small-footprint color management engine,
 with special focus on accuracy and performance. 
 - [LoFreq/2.1.3.1-foss-2018b-Python-2.7.15](http://csb5.github.io/lofreq)
Fast and sensitive variant calling from next-gen sequencing data
 - [LoomXpy/0.4.2-foss-2022b](https://loomxpy.readthedocs.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/LoomXpy/LoomXpy-0.4.2-foss-2022b.eb)
Python package (compatible with SCope) to create .loom files and extend them with
 other data e.g.: SCENIC regulons
 - [Lua/5.4.4-GCCcore-12.2.0](https://www.lua.org/)
Lua is a powerful, fast, lightweight, embeddable scripting language.
 Lua combines simple procedural syntax with powerful data description constructs based
 on associative arrays and extensible semantics. Lua is dynamically typed,
 runs by interpreting bytecode for a register-based virtual machine,
 and has automatic memory management with incremental garbage collection,
 making it ideal for configuration, scripting, and rapid prototyping.
 - [M4/1.4.19-GCCcore-11.3.0](https://www.gnu.org/software/m4/m4.html)
GNU M4 is an implementation of the traditional Unix macro processor. It is mostly SVR4 compatible
  although it has some extensions (for example, handling more than 9 positional parameters to macros).
 GNU M4 also has built-in functions for including files, running shell commands, doing arithmetic, etc.
 - [MACS2/2.2.9.1-foss-2022b](https://github.com/taoliu/MACS)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MACS2/MACS2-2.2.9.1-foss-2022b.eb)
Model Based Analysis for ChIP-Seq data
 - [MACS3/3.0.0b2-foss-2022b](https://macs3-project.github.io/MACS/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MACS3/MACS3-3.0.0b2-foss-2022b.eb)
Model Based Analysis for ChIP-Seq data
 - [MAESTRO/1.2.1-foss-2019b-Python-3.7.4](https://github.com/liulab-dfci/MAESTRO)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MAESTRO/MAESTRO-1.2.1-foss-2019b-Python-3.7.4.eb)
MAESTRO(Model-based AnalysEs of Single-cell Transcriptome and RegulOme) is a comprehensive
 single-cell RNA-seq and ATAC-seq analysis suit built using snakemake. MAESTRO combines several dozen tools and
 packages to create an integrative pipeline, which enables scRNA-seq and scATAC-seq analysis from raw sequencing
 data (fastq files) all the way through alignment, quality control, cell filtering, normalization, unsupervised
 clustering, differential expression and peak calling, celltype annotation and transcription regulation analysis.
 - [MAFFT/7.490-GCC-11.2.0-with-extensions](https://mafft.cbrc.jp/alignment/software/source.html)
MAFFT is a multiple sequence alignment program for unix-like operating systems.
It offers a range of multiple alignment methods, L-INS-i (accurate; for alignment
of <∼200 sequences), FFT-NS-2 (fast; for alignment of <∼30,000 sequences), etc.
 - [MAGeCK-VISPR/0.5.5-Python-3.7.4](https://bitbucket.org/liulab/mageck-vispr/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MAGeCK-VISPR/MAGeCK-VISPR-0.5.5-Python-3.7.4.eb)
MAGeCK-VISPR is a comprehensive quality control, analysis and visualization workflow for CRISPR/Cas9
  screens The workflow combines the MAGeCK algorithm to identify essential genes from CRISPR/Cas9 screens considering
  multiple conditions with VISPR to interactively explore results and quality control in a web-based frontend.
 - [MAGeCK-VISPR/0.5.5-Python-3.7.4](https://bitbucket.org/liulab/mageck-vispr/)
MAGeCK-VISPR is a comprehensive quality control, analysis and visualization workflow for CRISPR/Cas9
  screens The workflow combines the MAGeCK algorithm to identify essential genes from CRISPR/Cas9 screens considering
  multiple conditions with VISPR to interactively explore results and quality control in a web-based frontend.
 - [MAGeCK-VISPR/0.5.5-Python-3.7.4](https://bitbucket.org/liulab/mageck-vispr/)
MAGeCK-VISPR is a comprehensive quality control, analysis and visualization workflow for CRISPR/Cas9
  screens The workflow combines the MAGeCK algorithm to identify essential genes from CRISPR/Cas9 screens considering
  multiple conditions with VISPR to interactively explore results and quality control in a web-based frontend.
 - [MEME/5.5.1-gompi-2021b](https://meme-suite.org/meme/index.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MEME/MEME-5.5.1-gompi-2021b.eb)
The MEME Suite allows you to: * discover motifs using MEME, DREME (DNA only) or
 GLAM2 on groups of related DNA or protein sequences, * search sequence databases with motifs using
 MAST, FIMO, MCAST or GLAM2SCAN, * compare a motif to all motifs in a database of motifs, * associate
 motifs with Gene Ontology terms via their putative target genes, and * analyse motif enrichment
 using SpaMo or CentriMo.
 - [METIS/5.1.0-GCCcore-10.3.0](http://glaros.dtc.umn.edu/gkhome/metis/metis/overview)
METIS is a set of serial programs for partitioning graphs, partitioning
 finite element meshes, and producing fill reducing orderings for sparse
 matrices. The algorithms implemented in METIS are based on the multilevel
 recursive-bisection, multilevel k-way, and multi-constraint partitioning
 schemes.

 - [MMseqs2/13-45111-gompi-2021b](https://mmseqs.com)
MMseqs2: ultra fast and sensitive search and clustering suite
 - [MPC/1.3.1-GCCcore-12.2.0](http://www.multiprecision.org/)
Gnu Mpc is a C library for the arithmetic of
 complex numbers with arbitrarily high precision and correct
 rounding of the result. It extends the principles of the IEEE-754
 standard for fixed precision real floating point numbers to
 complex numbers, providing well-defined semantics for every
 operation. At the same time, speed of operation at high precision
 is a major design goal.
 - [MPFR/4.2.0-GCCcore-12.2.0](https://www.mpfr.org)
The MPFR library is a C library for multiple-precision floating-point
 computations with correct rounding.

 - [MUMPS/5.4.1-foss-2021b-metis](https://graal.ens-lyon.fr/MUMPS/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MUMPS/MUMPS-5.4.1-foss-2021b-metis.eb)
A parallel sparse direct solver
 - [MUMmer/4.0.0beta2-GCCcore-11.2.0](http://mummer.sourceforge.net/)
MUMmer is a system for rapidly aligning entire genomes,
 whether in complete or draft form. AMOS makes use of it.

 - [MUSCLE/3.8.31-foss-2018b](http://drive5.com/muscle/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MUSCLE/MUSCLE-3.8.31-foss-2018b.eb)
MUSCLE is one of the best-performing multiple alignment programs 
 according to published benchmark tests, with accuracy and speed that are consistently 
 better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users 
 learn everything they need to know about MUSCLE in a few minutes—only a handful of 
 command-line options are needed to perform common alignment tasks.
 - [Magic-BLAST/1.5.0-Linux_x86_64](https://ncbi.github.io/magicblast/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/Magic-BLAST/Magic-BLAST-1.5.0-Linux_x86_64.eb)
Magic-BLAST is a tool for mapping large next-generation RNA or DNA sequencing runs
 against a whole genome or transcriptome.
 - [Mako/1.2.4-GCCcore-12.2.0](https://www.makotemplates.org)
A super-fast templating language that borrows the best ideas from the existing templating languages
 - [MariaDB/10.11.2-GCC-12.2.0](https://mariadb.org/)
MariaDB is an enhanced, drop-in replacement for MySQL.
Included engines: myISAM, Aria, InnoDB, RocksDB, TokuDB, OQGraph, Mroonga.
 - [MariaDB-connector-c/2.3.7-GCCcore-8.3.0](https://downloads.mariadb.org/connector-c/)
MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases.
 - [Maven/3.6.3](https://maven.apache.org/index.html)
Binary maven install, Apache Maven is a software project management and comprehension tool. Based on
the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a
central piece of information.

 - [MaxQuant/2.4.9.0-foss-2022b](https://www.maxquant.org/)
MaxQuant is a quantitative proteomics software package designed for analyzing large
 mass-spectrometric data sets. It is specifically aimed at high-resolution MS data. Several labeling
 techniques as well as label-free quantification are supported.
 - [Mesa/22.2.4-GCCcore-12.2.0](https://www.mesa3d.org/)
Mesa is an open-source implementation of the OpenGL specification -
 a system for rendering interactive 3D graphics.
 - [Meson/0.64.0-GCCcore-12.2.0](https://mesonbuild.com)
Meson is a cross-platform build system designed to be both as fast and as user friendly as possible.
 - [Metal/2020-05-05-GCC-10.2.0](http://csg.sph.umich.edu/abecasis/Metal/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/Metal/Metal-2020-05-05-GCC-10.2.0.eb)
Metal
 - [MethGo/24c9319-foss-2018b-Python-2.7.15](https://methgo.readthedocs.io/en/latest/index.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MethGo/MethGo-24c9319-foss-2018b-Python-2.7.15.eb)
DNA methylation is a major epigenetic modification regulating several biological
 processes. A standard approach in the study of DNA methylation is bisulfite sequencing (BS-Seq).
 MethGo is a simple and effective tool designed for the analysis of data from whole genome bisulfite
 sequencing (WGBS) and reduced representation bisulfite sequencing (RRBS).
 - [MiXCR/3.0.3-Java-1.8](https://milaboratory.com/software/mixcr/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MiXCR/MiXCR-3.0.3-Java-1.8.eb)
MiXCR processes big immunome data from raw sequences 
to quantitated clonotypes 
 - [MinCED/0.4.2-GCCcore-9.3.0-Java-11](https://github.com/ctSkennerton/minced)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MinCED/MinCED-0.4.2-GCCcore-9.3.0-Java-11.eb)
Mining CRISPRs in Environmental Datasets
 - [Miniconda3/4.9.2](https://docs.conda.io/en/latest/miniconda.html)
Miniconda is a free minimal installer for conda. It is a small,
 bootstrap version of Anaconda that includes only conda, Python, the packages they
 depend on, and a small number of other useful packages.
 - [Mono/6.12.0.199-GCCcore-12.2.0](https://www.mono-project.com/)
An open source, cross-platform, implementation of C# and the CLR that is
 binary compatible with Microsoft.NET.
 - [Monocle3/1.3.1-foss-2021b-R-4.2.2](https://cole-trapnell-lab.github.io/monocle3)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/Monocle3/Monocle3-1.3.1-foss-2021b-R-4.2.2.eb)
Single-cell transcriptome sequencing (sc-RNA-seq) experiments allow us to discover new cell types and help us understand how they arise in development. The Monocle 3 package provides a toolkit for analyzing single-cell gene expression experiments.
 - [MoreRONN/4.9-foss-2019b](https://github.com/varun-ramraj/MoreRONN)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MoreRONN/MoreRONN-4.9-foss-2019b.eb)
MoreRONN is the spiritual successor of RONN and is useful for surveying disorder in proteins as well as designing expressible constructs for X-ray crystallography.
 - [Mothur/1.41.0-foss-2018b-Python-2.7.15](http://www.mothur.org/)
Mothur is a single piece of open-source, expandable software
 to fill the bioinformatics needs of the microbial ecology community.
 - [MotionCor2/1.4.2-gcccuda-2020b](https://msg.ucsf.edu/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MotionCor2/MotionCor2-1.4.2-gcccuda-2020b.eb)
MotionCor2 correct anisotropic image motion at the
single pixel level across the whole frame, suitable for both single
particle and tomographic images. Iterative, patch-based motion detection
is combined with spatial and temporal constraints and dose weighting.

Cite publication: Shawn Q. Zheng, Eugene Palovcak, Jean-Paul Armache,
Yifan Cheng and David A. Agard (2016) Anisotropic Correction of
Beam-induced Motion for Improved Single-particle Electron
Cryo-microscopy, Nature Methods, submitted.
BioArxiv: https://biorxiv.org/content/early/2016/07/04/061960

 - [MrBayes/3.2.7a-gompi-2021b](https://nbisweden.github.io/MrBayes/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MrBayes/MrBayes-3.2.7a-gompi-2021b.eb)
MrBayes is a program for Bayesian inference and model choice across
 a wide range of phylogenetic and evolutionary models.
 - [MuSE/2.0.1-GCC-11.2.0](https://github.com/wwylab/MuSE)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MuSE/MuSE-2.0.1-GCC-11.2.0.eb)
An accurate and ultra-fast somatic mutation calling tool for whole-genome sequencing (WGS)
 and whole-exome sequencing (WES) data from heterogeneous tumor samples.
 - [MultiQC/1.9-foss-2019b-Python-3.7.4](https://multiqc.info)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MultiQC/MultiQC-1.9-foss-2019b-Python-3.7.4.eb)
Aggregate results from bioinformatics analyses across many samples into a single 
 report.

 MultiQC searches a given directory for analysis logs and compiles a HTML report. It's a general
 use tool, perfect for summarising the output from numerous bioinformatics tools.
 - [MutSig/2](http://software.broadinstitute.org/cancer/cga/mutsig)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MutSig/MutSig-2.eb)
MutSig stands for "Mutation Significance".  MutSig analyzes lists of mutations
 discovered in DNA sequencing, to identify genes that were mutated more often than expected by
 chance given background mutation processes.
 - [MutSig2CV/3.11](https://github.com/getzlab/MutSig2CV)
MutSig2CV analyzes somatic point mutations discovered in DNA sequencing,
identifying genes mutated more often than expected by chance given inferred background mutation processes. 

 - [MutSigCV/1.4.1](https://www.genepattern.org/modules/docs/MutSigCV)
MutSigCV accepts whole genome or whole exome sequencing data
from multiple samples, with information about point mutations, small insertions/deletions,
and coverage, and identifies genes that are mutated more often than one would expect by chance.

 - [NAMD/2.14-foss-2020a-mpi](https://www.ks.uiuc.edu/Research/namd/)
NAMD is a parallel molecular dynamics code designed for high-performance simulation of
 large biomolecular systems.
 - [NASM/2.15.05-GCCcore-11.3.0](https://www.nasm.us/)
NASM: General-purpose x86 assembler
 - [NCCL/2.12.12-GCCcore-11.3.0-CUDA-11.7.0](https://developer.nvidia.com/nccl)
The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective
communication primitives that are performance optimized for NVIDIA GPUs.
 - [NGS/2.11.2-GCCcore-11.2.0](https://github.com/ncbi/ngs)
NGS is a new, domain-specific API for accessing reads, alignments and pileups
produced from Next Generation Sequencing.
 - [NLopt/2.7.1-GCCcore-12.2.0](http://ab-initio.mit.edu/wiki/index.php/NLopt)
NLopt is a free/open-source library for nonlinear optimization,
 providing a common interface for a number of different free optimization routines
 available online as well as original implementations of various other algorithms. 
 - [NSPR/4.35-GCCcore-12.2.0](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR)
Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level
 and libc-like functions.
 - [NSS/3.85-GCCcore-12.2.0](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS)
Network Security Services (NSS) is a set of libraries designed to support cross-platform development
 of security-enabled client and server applications.
 - [NextGenMap/0.5.5-GCC-11.2.0](http://cibiv.github.io/NextGenMap/)
NextGenMap is a flexible highly sensitive short read mapping tool that
 handles much higher mismatch rates than comparable algorithms while still outperforming
 them in terms of runtime.
 - [Nextflow/23.04.2](https://www.nextflow.io/)
Nextflow is a reactive workflow framework and a programming DSL
 that eases writing computational pipelines with complex data
 - [Nim/0.19.2-GCCcore-7.3.0](https://nim-lang.org/)
Nim is a systems and applications programming language.
 - [Ninja/1.11.1-GCCcore-12.2.0](https://ninja-build.org/)
Ninja is a small build system with a focus on speed.
 - [OpenBLAS/0.3.21-GCC-12.2.0](http://www.openblas.net/)
OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version.
 - [OpenCV/4.5.5-foss-2021b-contrib](https://opencv.org/)
OpenCV (Open Source Computer Vision Library) is an open source computer vision
 and machine learning software library. OpenCV was built to provide
 a common infrastructure for computer vision applications and to accelerate
 the use of machine perception in the commercial products.
 Includes extra modules for OpenCV from the contrib repository.
 - [OpenEXR/3.1.5-GCCcore-12.2.0](https://www.openexr.com/)
OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light & Magic
 for use in computer imaging applications
 - [OpenJPEG/2.5.0-GCCcore-12.2.0](https://www.openjpeg.org/)
OpenJPEG is an open-source JPEG 2000 codec written in
 C language. It has been developed in order to promote the use of JPEG 2000,
 a still-image compression standard from the Joint Photographic Experts Group
 (JPEG). Since may 2015, it is officially recognized by ISO/IEC and ITU-T as
 a JPEG 2000 Reference Software.
 - [OpenMM/7.5.1-foss-2020b](https://openmm.org)
OpenMM is a toolkit for molecular simulation.
 - [OpenMPI/4.1.4-GCC-12.2.0](https://www.open-mpi.org/)
The Open MPI Project is an open source MPI-3 implementation.
 - [OpenPGM/5.2.122-GCCcore-12.2.0](https://code.google.com/p/openpgm/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/o/OpenPGM/OpenPGM-5.2.122-GCCcore-12.2.0.eb)
OpenPGM is an open source implementation of the Pragmatic General Multicast
 (PGM) specification in RFC 3208 available at www.ietf.org. PGM is a reliable
 and scalable multicast protocol that enables receivers to detect loss, request
 retransmission of lost data, or notify an application of unrecoverable loss.
 PGM is a receiver-reliable protocol, which means the receiver is responsible
 for ensuring all data is received, absolving the sender of reception
 responsibility.

 - [OpenSSL/1.1](https://www.openssl.org/)
The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured,
 and Open Source toolchain implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1) 
 protocols as well as a full-strength general purpose cryptography library. 
 - [OpenSlide/3.4.1-GCCcore-11.2.0](https://openslide.org/)
OpenSlide is a C library that provides a simple interface to
read whole-slide images (also known as virtual slides).
 - [OptiType/1.3.5-foss-2019b-Python-3.7.4](https://github.com/FRED-2/OptiType)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/o/OptiType/OptiType-1.3.5-foss-2019b-Python-3.7.4.eb)
OptiType is a novel HLA genotyping algorithm based on integer linear programming,
 capable of producing accurate 4-digit HLA genotyping predictions from NGS data
 by simultaneously selecting all major and minor HLA Class I alleles. 
 - [Osi/0.108.8-GCC-12.2.0](https://github.com/coin-or/Osi)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/o/Osi/Osi-0.108.8-GCC-12.2.0.eb)
Osi (Open Solver Interface) provides an abstract base class to a generic linear
programming (LP) solver, along with derived classes for specific solvers. Many
applications may be able to use the Osi to insulate themselves from a specific
LP solver. That is, programs written to the OSI standard may be linked to any
solver with an OSI interface and should produce correct results. The OSI has
been significantly extended compared to its first incarnation. Currently, the
OSI supports linear programming solvers and has rudimentary support for integer
programming.
 - [PANDAseq/2.11-foss-2018b](https://github.com/neufeld/pandaseq)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PANDAseq/PANDAseq-2.11-foss-2018b.eb)
PANDASEQ is a program to align Illumina reads, optionally
 with PCR primers embedded in the sequence, and reconstruct an overlapping
 sequence.
 - [PATRIC-cli/1.035-GCC-10.2.0](https://docs.patricbrc.org/cli_tutorial/index.html )
Read files using HTSlib including BAM/CRAM, Tabix and BCF database files
 - [PCRE/8.45-GCCcore-11.2.0](https://www.pcre.org/)
The PCRE library is a set of functions that implement regular expression
 pattern matching using the same syntax and semantics as Perl 5.

 - [PCRE2/10.40-GCCcore-12.2.0](https://www.pcre.org/)
The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax
 and semantics as Perl 5.

 - [PDBFixer/1.7-foss-2020b](https://github.com/openmm/pdbfixer)
PDBFixer is an easy to use application for fixing problems in Protein Data Bank files in preparation
for simulating them.
 - [PDFCrop/0.4b](http://pdfcrop.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PDFCrop/PDFCrop-0.4b.eb)
PDFCrop is a Perl script that crops the white margins of PDF pages and rescales them
 to fit a standard size sheet of paper. It makes the printed pages far more attractive to read!
 - [PEAR/0.9.11-foss-2018b](http://sco.h-its.org/exelixis/web/software/pear/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PEAR/PEAR-0.9.11-foss-2018b.eb)
PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. 
 It is fully parallelized and can run with as low as just a few kilobytes of memory.
 - [PETSc/3.15.1-foss-2021a](https://www.mcs.anl.gov/petsc)
PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the
 scalable (parallel) solution of scientific applications modeled by partial differential equations.
 - [PHASE/2.1.2-GCCcore-8.3.0](http://stephenslab.uchicago.edu/phase/download.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PHASE/PHASE-2.1.2-GCCcore-8.3.0.eb)
PHASE is a program implementing the method for reconstructing haplotypes
 from population data
 - [PLINK/2.00-alpha1-x86_64](https://www.cog-genomics.org/plink/1.9/)
Whole-genome association analysis toolset
 - [PLINK2/20210826-linux_x86_64](https://www.cog-genomics.org/plink/2.0)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PLINK2/PLINK2-20210826-linux_x86_64.eb)
PLINK is a free, open-source whole genome association analysis toolset, designed to
 perform a range of basic, large-scale analyses in a computationally efficient manner.
 The focus of PLINK is purely on analysis of genotype/phenotype data.
 - [PLUMED/2.7.2-foss-2020b](https://www.plumed.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PLUMED/PLUMED-2.7.2-foss-2020b.eb)
PLUMED is an open source library for free energy calculations in molecular systems which
 works together with some of the most popular molecular dynamics engines. Free energy calculations can be
 performed as a function of many order parameters with a particular  focus on biological problems, using
 state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD.
 The software, written in C++, can be easily interfaced with both fortran and C/C++ codes.

 - [PMIx/4.2.2-GCCcore-12.2.0](https://pmix.org/)
Process Management for Exascale Environments
PMI Exascale (PMIx) represents an attempt to
provide an extended version of the PMI standard specifically designed
to support clusters up to and including exascale sizes. The overall
objective of the project is not to branch the existing pseudo-standard
definitions
 - [PROJ/9.1.1-GCCcore-12.2.0](https://proj.org)
Program proj is a standard Unix filter function which converts
geographic longitude and latitude coordinates into cartesian coordinates
 - [Pandoc/2.13](https://pandoc.org)
If you need to convert files from one markup format into another, pandoc is your swiss-army knife
 - [Pango/1.50.12-GCCcore-12.2.0](https://www.pango.org/)
Pango is a library for laying out and rendering of text, with an emphasis on internationalization.
Pango can be used anywhere that text layout is needed, though most of the work on Pango so far has been done in the
context of the GTK+ widget toolkit. Pango forms the core of text and font handling for GTK+-2.x.
 - [Pango/1.50.12-GCCcore-12.2.0](https://www.pango.org/)
Pango is a library for laying out and rendering of text, with an emphasis on internationalization.
Pango can be used anywhere that text layout is needed, though most of the work on Pango so far has been done in the
context of the GTK+ widget toolkit. Pango forms the core of text and font handling for GTK+-2.x.
 - [Perl/5.36.0-GCCcore-12.2.0-minimal](https://www.perl.org/)
Larry Wall's Practical Extraction and Report Language

This is a minimal build without any modules. Should only be used for build dependencies.

 - [Pillow/9.4.0-GCCcore-12.2.0](https://pillow.readthedocs.org/)
Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
 PIL is the Python Imaging Library by Fredrik Lundh and Contributors.
 - [Pillow-SIMD/9.1.0-GCCcore-11.2.0](https://github.com/uploadcare/pillow-simd)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/Pillow-SIMD/Pillow-SIMD-9.1.0-GCCcore-11.2.0.eb)
Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
 PIL is the Python Imaging Library by Fredrik Lundh and Contributors.
 - [Pindel/0.2.5b9-20170508-foss-2018b](http://gmt.genome.wustl.edu/packages/pindel/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/Pindel/Pindel-0.2.5b9-20170508-foss-2018b.eb)
Pindel can detect breakpoints of large deletions, medium sized
 insertions, inversions, tandem duplications and other structural variants at single-based
 resolution from next-gen sequence data. It uses a pattern growth approach to identify the
 breakpoints of these variants from paired-end short reads. 
 - [Porechop/0.2.4-GCCcore-11.2.0](https://github.com/rrwick/Porechop)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/Porechop/Porechop-0.2.4-GCCcore-11.2.0.eb)
Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.
 Adapters on the ends of reads are trimmed off, and when a read has an adapter in its middle,
 it is treated as chimeric and chopped into separate reads. Porechop performs thorough alignments
 to effectively find adapters, even at low sequence identity
 - [PostgreSQL/15.2-GCCcore-12.2.0](https://www.postgresql.org/)
PostgreSQL is a powerful, open source object-relational database system.
 It is fully ACID compliant, has full support for foreign keys,
 joins, views, triggers, and stored procedures (in multiple languages).
 It includes most SQL:2008 data types, including INTEGER,
 NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP.
 It also supports storage of binary large objects, including pictures,
 sounds, or video. It has native programming interfaces for C/C++, Java,
 .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation.
 - [PubWeb/0.4.0-foss-2021b](https://github.com/FredHutch/PubWeb-client)
A Python 3.8+ library for the PubWeb platform.
 - [PyCairo/1.24.0-GCCcore-12.2.0](https://pycairo.readthedocs.io/)
Python bindings for the cairo library
 - [PyClone/2020.9b2-GCCcore-10.2.0](https://gitlab.com/ltgiv/pyclone)
PyClone is a Python package that wraps rclone and provides a threaded interface for
 an installation at the host or container level.
 - [PyOpenGL/3.1.5-GCCcore-8.3.0](http://pyopengl.sourceforge.net)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PyOpenGL/PyOpenGL-3.1.5-GCCcore-8.3.0.eb)
PyOpenGL is the most common cross platform Python binding to OpenGL and related APIs.
 - [PyQt5/5.15.1-GCCcore-10.2.0](https://www.riverbankcomputing.com/software/pyqt)
PyQt5 is a set of Python bindings for v5 of the Qt application framework from The Qt Company.
This bundle includes PyQtWebEngine, a set of Python bindings for The Qt Company’s Qt WebEngine framework.
 - [PySide6/6.5.0-GCCcore-12.2.0](https://pypi.python.org/pypi/PySide6)
PySide6 is the official Python module from the Qt for Python project,
    which provides access to the complete Qt 6.5+ framework.
 - [PyStan/3.5.0-foss-2021b](https://github.com/stan-dev/pystan)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PyStan/PyStan-3.5.0-foss-2021b.eb)
Python interface to Stan, a package for Bayesian inference
 using the No-U-Turn sampler, a variant of Hamiltonian Monte Carlo.
 - [PyTables/3.8.0-foss-2022b](https://www.pytables.org)
PyTables is a package for managing hierarchical datasets and designed to efficiently and easily cope
 with extremely large amounts of data. PyTables is built on top of the HDF5 library, using the Python language and the
 NumPy package. It features an object-oriented interface that, combined with C extensions for the performance-critical
 parts of the code (generated using Cython), makes it a fast, yet extremely easy to use tool for interactively browsing,
 processing and searching very large amounts of data. One important feature of PyTables is that it optimizes memory and 
 disk resources so that data takes much less space (specially if on-flight compression is used) than other solutions 
 such as relational or object oriented databases.
 - [PyTorch/1.12.1-foss-2022a-CUDA-11.7.0](https://pytorch.org/)
Tensors and Dynamic neural networks in Python with strong GPU acceleration.
PyTorch is a deep learning framework that puts Python first.
 - [PyTorch-Geometric/2.0.3-fosscuda-2020b](https://github.com/rusty1s/pytorch_geometric)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PyTorch-Geometric/PyTorch-Geometric-2.0.3-fosscuda-2020b.eb)
PyTorch Geometric (PyG) is a geometric deep learning extension library for PyTorch.
 - [PyTorch-Lightning/1.5.9-foss-2021a-CUDA-11.3.1](https://pytorchlightning.ai)
PyTorch Lightning is the lightweight PyTorch wrapper for ML researchers.
 - [PyYAML/6.0-GCCcore-11.3.0](https://github.com/yaml/pyyaml)
PyYAML is a YAML parser and emitter for the Python programming language.
 - [PycURL/7.45.2-GCCcore-12.2.0](http://pycurl.io/)
PycURL is a Python interface to libcurl. PycURL can be used to fetch objects identified by a URL
 from a Python program, similar to the urllib Python module. PycURL is mature, very fast, and supports a lot of
 features.
 - [Pyomo/6.4.4-foss-2021b](https://www.pyomo.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/Pyomo/Pyomo-6.4.4-foss-2021b.eb)
Pyomo is a Python-based open-source software package that supports a diverse set of optimization
 capabilities for formulating and analyzing optimization models. 
 - [Pysam/0.21.0-GCC-12.2.0](https://github.com/pysam-developers/pysam)
Pysam is a python module for reading and manipulating Samfiles.
 It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.
 - [Python/3.10.8-GCCcore-12.2.0](https://python.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/Python/Python-3.10.8-GCCcore-12.2.0.eb)
Python is a programming language that lets you work more quickly and integrate your systems
 more effectively.
 - [QIIME2/2020.11](http://qiime2.org/)
QIIME is an open-source bioinformatics pipeline for performing microbiome analysis
 from raw DNA sequencing data.
 - [QUAST/5.1.0rc1-foss-2020b](https://github.com/ablab/quast)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/q/QUAST/QUAST-5.1.0rc1-foss-2020b.eb)
QUAST evaluates genome assemblies by computing various metrics.
It works both with and without reference genomes. The tool accepts multiple
assemblies, thus is suitable for comparison.
 - [Qhull/2020.2-GCCcore-10.3.0](http://www.qhull.org)
Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram,
 halfspace intersection about a point, furthest-site Delaunay triangulation,
 and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and
 higher dimensions. Qhull implements the Quickhull algorithm for computing the
 convex hull.

 - [Qt/4.8.7-foss-2018b](http://qt.io/)
Qt is a comprehensive cross-platform C++ application framework.
 - [Qt5/5.15.7-GCCcore-12.2.0](https://qt.io/)
Qt is a comprehensive cross-platform C++ application framework.
 - [Qt6/6.5.0-GCCcore-12.2.0](https://qt.io/)
Qt is a comprehensive cross-platform C++ application framework.
 - [R/4.3.1-gfbf-2022b](https://www.r-project.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/R/R-4.3.1-gfbf-2022b.eb)
R is a free software environment for statistical computing
 and graphics.
 - [R-Tidyverse/4.3.1-gfbf-2022b](http://www.r-project.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/R-Tidyverse/R-Tidyverse-4.3.1-gfbf-2022b.eb)
R is a free software environment for statistical computing and graphics.
 - [R-bundle-Bioconductor/3.10-foss-2019b-R-3.6.2](https://bioconductor.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.10-foss-2019b-R-3.6.2.eb)
Bioconductor provides tools for the analysis and coprehension
 of high-throughput genomic data.
 - [R-keras/2.2.5.0-foss-2019b-Python-3.7.4-R-3.6.2](https://cran.r-project.org/web/packages/keras)
Interface to 'Keras' <https://keras.io>, a high-level neural networks 'API'. 
 - [RAxML/8.2.12-GCC-10.2.0-pthreads-avx2](https://github.com/stamatak/standard-RAxML)
RAxML search algorithm for maximum likelihood based inference of phylogenetic trees.
 - [RAxML-NG/1.0.3-GCC-10.2.0](https://github.com/amkozlov/raxml-ng)
RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML)
 optimality criterion. Its search heuristic is based on iteratively performing a series of Subtree
 Pruning and Regrafting (SPR) moves, which allows to quickly navigate to the best-known ML tree.
 - [RDFlib/5.0.0-GCCcore-10.2.0](https://github.com/RDFLib/rdflib)
RDFLib is a Python library for working with RDF, a simple yet powerful language
 for representing information.
 - [RE2/2023-03-01-GCCcore-12.2.0](https://github.com/google/re2)
RE2 is a fast, safe, thread-friendly alternative to backtracking regular
expression engines like those used in PCRE, Perl, and Python. It is a C++
library. 
 - [RELION/3.1.2-fosscuda-2019b](https://github.com/3dem/relion)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/RELION/RELION-3.1.2-fosscuda-2019b.eb)
RELION (for REgularised LIkelihood OptimisatioN) is a stand-alone computer
 program for Maximum A Posteriori refinement of (multiple) 3D reconstructions or 2D class
 averages in cryo-electron microscopy.
 - [RNA-SeQC/2.4.2-foss-2021b](https://github.com/getzlab/rnaseqc)
Fast, efficient RNA-Seq metrics for quality control and process optimization
 - [ROSE/1-GCCcore-8.3.0-Python-2.7.16](http://younglab.wi.mit.edu/super_enhancer_code.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/ROSE/ROSE-1-GCCcore-8.3.0-Python-2.7.16.eb)
To create stitched enhancers, and to separate super-enhancers from typical enhancers using sequencing data (.bam) given a file of previously identified constituent enhancers (.gff)
 - [RSEM/1.3.3-foss-2019b](http://deweylab.github.io/RSEM/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/RSEM/RSEM-1.3.3-foss-2019b.eb)
RNA-Seq by Expectation-Maximization
 - [RSeQC/5.0.1-foss-2021b](http://rseqc.sourceforge.net/)
RSeQC provides a number of useful modules that can
 comprehensively evaluate high throughput sequence data especially RNA-seq
 data. Some basic modules quickly inspect sequence quality, nucleotide
 composition bias, PCR bias and GC bias, while RNA-seq specific modules
 evaluate sequencing saturation, mapped reads distribution, coverage
 uniformity, strand specificity, transcript level RNA integrity etc.
 - [RStudio-Server/2022.07.2+576-foss-2022b-Java-11-R-4.3.1](https://www.rstudio.com/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/RStudio-Server/RStudio-Server-2022.07.2+576-foss-2022b-Java-11-R-4.3.1.eb)
This is the RStudio Server version.
RStudio is a set of integrated tools designed to help you be more productive with R.

The server can be started with:
  rserver --server-daemonize=0 --www-port=8787

If you need a database config one can be created with:
  MYTMP=`mktemp -d` && echo -e "provider=sqlite\ndirectory=${MYTMP}/sqlite" > "${MYTMP}/db.conf"
and then used with:
  rserver ... --database-config-file="${MYTMP}/db.conf"

 - [Racon/1.5.0-GCCcore-11.2.0](https://github.com/lbcb-sci/racon)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/Racon/Racon-1.5.0-GCCcore-11.2.0.eb)
Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads.
 - [RapidJSON/1.1.0-GCCcore-11.2.0](https://rapidjson.org)
A fast JSON parser/generator for C++ with both SAX/DOM style API
 - [Ray-project/2.6.2-foss-2022b](https://docs.ray.io/en/latest/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/Ray-project/Ray-project-2.6.2-foss-2022b.eb)
Ray is a fast and simple framework for building and running distributed applications.
 - [Redis/7.0.12-GCC-12.2.0](https://redis.io)
Redis is an open source (BSD licensed), in-memory data structure store, used as
a database, cache, and message broker. Redis provides data structures such as
strings, hashes, lists, sets, sorted sets with range queries, bitmaps,
hyperloglogs, geospatial indexes, and streams. Redis has built-in replication,
Lua scripting, LRU eviction, transactions, and different levels of on-disk
persistence, and provides high availability via Redis Sentinel and automatic
partitioning with Redis Cluster.
 - [Regenie/3.1.2-GCC-11.2.0](https://rgcgithub.github.io/regenie)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/Regenie/Regenie-3.1.2-GCC-11.2.0.eb)
Regenie is a C++ program for whole genome regression modelling of large genome-wide 
association studies.  It is developed and supported by a team of scientists at the Regeneron Genetics Center.
 - [RepeatMasker/4.0.8-foss-2018b-Perl-5.28.0-HMMER](http://www.repeatmasker.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/RepeatMasker/RepeatMasker-4.0.8-foss-2018b-Perl-5.28.0-HMMER.eb)
RepeatMasker is a program that screens DNA sequences for interspersed repeats
 and low complexity DNA sequences.
 - [RevBayes/1.1.1-GCC-10.2.0](https://revbayes.github.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/RevBayes/RevBayes-1.1.1-GCC-10.2.0.eb)
RevBayes provides an interactive environment for statistical computation in
 phylogenetics. It is primarily intended for modeling, simulation, and Bayesian inference in
 evolutionary biology, particularly phylogenetics.
 - [Ruby/3.0.1-GCCcore-11.2.0](https://www.ruby-lang.org)
Ruby is a dynamic, open source programming language with
 a focus on simplicity and productivity. It has an elegant syntax that is
 natural to read and easy to write.
 - [Rust/1.65.0-GCCcore-12.2.0](https://www.rust-lang.org)
Rust is a systems programming language that runs blazingly fast, prevents segfaults,
 and guarantees thread safety.
 - [SAMtools/1.17-GCC-12.2.0](https://www.htslib.org/)
SAM Tools provide various utilities for manipulating alignments in the SAM format, 
 including sorting, merging, indexing and generating alignments in a per-position format.
 - [SCOTCH/7.0.1-gompi-2021b](https://gforge.inria.fr/projects/scotch/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SCOTCH/SCOTCH-7.0.1-gompi-2021b.eb)
Software package and libraries for sequential and parallel graph partitioning,
static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning.
 - [SCons/4.0.1-GCCcore-10.2.0](https://www.scons.org/)
SCons is a software construction tool.
 - [SDL2/2.26.3-GCCcore-12.2.0](https://www.libsdl.org/)
SDL: Simple DirectMedia Layer, a cross-platform multimedia library
 - [SHAPEIT4/4.2.2-foss-2020b](https://odelaneau.github.io/shapeit4/)
SHAPEIT4 is a fast and accurate method for estimation of haplotypes
(aka phasing) for SNP array and high coverage sequencing data.

 - [SKESA/2.3.0-foss-2018b](https://github.com/ncbi/SKESA)
SKESA is a de-novo sequence read assembler for cultured single isolate genomes based on DeBruijn graphs.
 - [SLEPc/3.15.1-foss-2021a](https://www.grycap.upv.es/slepc/)
SLEPc (Scalable Library for Eigenvalue Problem Computations) is a software library for the solution
 of large scale sparse eigenvalue problems on parallel computers. It is an extension of PETSc and can be used for
 either standard or generalized eigenproblems, with real or complex arithmetic. It can also be used for computing a
 partial SVD of a large, sparse, rectangular matrix, and to solve quadratic eigenvalue problems.
 - [SOCI/4.0.3-GCC-11.2.0](http://soci.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SOCI/SOCI-4.0.3-GCC-11.2.0.eb)
SOCI is a database access library for C++ that makes the illusion of embedding SQL queries in the
 regular C++ code, staying entirely within the Standard C++.
 - [SPAdes/3.13.0-foss-2018b](http://cab.spbu.ru/software/spades/)
Genome assembler for single-cell and isolates data sets
 - [SPRING/1.6-foss-2018b-Python-2.7.15](http://https://github.com/AllonKleinLab/SPRING_dev)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SPRING/SPRING-1.6-foss-2018b-Python-2.7.15.eb)
SPRING is a collection of pre-processing scripts and a web
browser-based tool for visualizing and interacting with high dimensional data.
View an example dataset here. SPRING was developed for single cell RNA-Seq data
but can be applied more generally. The minimal input is a matrix of high
dimensional data points (cells) and a list of dimension names (genes).
 - [SQANTI3/1.0-foss-2019b-Python-3.7.4](https://github.com/ConesaLab/SQANTI3)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SQANTI3/SQANTI3-1.0-foss-2019b-Python-3.7.4.eb)
SQANTI3 is the first module of the Functional IsoTranscriptomics (FIT) framework,
 that also includes IsoAnnot and tappAS. Used for new long read-defined transcriptome.
 - [SQLite/3.39.4-GCCcore-12.2.0](https://www.sqlite.org/)
SQLite: SQL Database Engine in a C Library
 - [SRA-Toolkit/3.0.0-ubuntu64](https://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=software)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SRA-Toolkit/SRA-Toolkit-3.0.0-ubuntu64.eb)
The SRA Toolkit, and the source-code SRA System Development
 Kit (SDK), will allow you to programmatically access data housed within SRA
 and convert it from the SRA format
 - [SSW/1.1-GCCcore-11.2.0](https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library)
SSW is a fast implementation of the Smith-Waterman algorithm, which uses the
 Single-Instruction Multiple-Data (SIMD) instructions to parallelize the algorithm at the
 instruction level. SSW library provides an API that can be flexibly used by programs written in
 C, C++ and other languages. We also provide a software that can do protein and genome alignment
 directly. Current version of our implementation is ~50 times faster than an ordinary
 Smith-Waterman. It can return the Smith-Waterman score, alignment location and traceback path
 (cigar) of the optimal alignment accurately; and return the sub-optimal alignment score and
 location heuristically.

 - [STAR/2.7.10b-GCC-12.2.0](https://github.com/alexdobin/STAR)
STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.
 - [STAR-Fusion/1.12.0-foss-2022b](https://github.com/STAR-Fusion/STAR-Fusion)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/STAR-Fusion/STAR-Fusion-1.12.0-foss-2022b.eb)
STAR-Fusion uses the STAR aligner to identify candidate fusion transcripts
 supported by Illumina reads. STAR-Fusion further processes the output generated by the STAR aligner
 to map junction reads and spanning reads to a reference annotation set.
 - [SUM/2.5.2](https://www.supermicro.com/en/solutions/management-software/supermicro-update-manager )

 - [SVG/2.84-foss-2019b-Perl-5.30.0](https://metacpan.org/pod/SVG)
Perl binding for SVG
 - [SVclone/1.1.2-foss-2022b](https://github.com/mcmero/SVclone)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SVclone/SVclone-1.1.2-foss-2022b.eb)
Cluster structural variants of similar cancer cell fraction (CCF).
 - [SWIG/4.1.1-GCCcore-12.2.0](http://www.swig.org/)
SWIG is a software development tool that connects programs written in C and C++ with
 a variety of high-level programming languages.
 - [SYMPHONY/5.7.1-foss-2022b](https://github.com/coin-or/SYMPHONY/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SYMPHONY/SYMPHONY-5.7.1-foss-2022b.eb)
SYMPHONY is an open-source solver for mixed-integer linear programs (MILPs) written in C.
 - [Salmon/1.10.1-GCC-12.2.0](https://github.com/COMBINE-lab/salmon)
Salmon is a wicked-fast program to produce a highly-accurate,
 transcript-level quantification estimates from RNA-seq data.
 - [ScaLAPACK/2.2.0-gompi-2022a-fb](https://www.netlib.org/scalapack/)
The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines
 redesigned for distributed memory MIMD parallel computers.
 - [SciPy-bundle/2023.02-gfbf-2022b](https://python.org/)
Bundle of Python packages for scientific software
 - [Seaborn/0.12.2-foss-2022b](https://seaborn.pydata.org/)
Seaborn is a Python visualization library based on matplotlib.
 It provides a high-level interface for drawing attractive statistical graphics. 
 - [SeqAn/2.4.0-GCCcore-8.3.0](https://github.com/seqan/seqan)
SeqAn is an open source C++ library of efficient algorithms and data structures
 for the analysis of sequences with the focus on biological data.

 - [SeqLib/1.2.0-GCC-10.2.0](https://github.com/walaj/SeqLib)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SeqLib/SeqLib-1.2.0-GCC-10.2.0.eb)
C++ interface to HTSlib, BWA-MEM and Fermi
 - [SeqPrep/1.3.2-GCCcore-8.3.0](https://github.com/jstjohn/SeqPrep)
Tool for stripping adaptors and/or merging paired reads with overlap into single reads.
 - [Seqmagick/0.6.2-foss-2018b-Python-2.7.15](https://fhcrc.github.io/seqmagick/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/Seqmagick/Seqmagick-0.6.2-foss-2018b-Python-2.7.15.eb)
We often have to convert between sequence formats and do
 little tasks on them, and it's not worth writing scripts for that. Seqmagick
 is a kickass little utility built in the spirit of imagemagick to expose the
 file format conversion in Biopython in a convenient way. Instead of having a
 big mess of scripts, there is one that takes arguments.
 - [Seurat/4.1.1-foss-2021b-R-4.2.0](https://satijalab.org/seurat)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/Seurat/Seurat-4.1.1-foss-2021b-R-4.2.0.eb)
Seurat is an R package designed for QC, analysis, and exploration of single cell RNA-seq data.
 - [Singularity/3.5.3](https://singularity-tutorial.github.io)
Singularity is a portable application stack packaging and runtime utility.
 - [SlamDunk/0.4.3-foss-2021b](https://t-neumann.github.io/slamdunk)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SlamDunk/SlamDunk-0.4.3-foss-2021b.eb)
SlamDunk is a novel, fully automated software tool for automated, robust, scalable and reproducible
SLAMseq data analysis.
 - [Sniffles/1.0.8-foss-2018b](https://github.com/fritzsedlazeck/Sniffles)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/Sniffles/Sniffles-1.0.8-foss-2018b.eb)
Sniffles is a structural variation caller using third
 generation sequencing (PacBio or Oxford Nanopore). It detects all types of
 SVs (10bp+) using evidence from split-read alignments, high-mismatch regions,
 and coverage analysis.
 - [SpaceRanger/2.0.0-GCC-11.2.0](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger)
Space Ranger is a set of analysis pipelines that process Visium spatial RNA-seq output
and brightfield microscope images in order to detect tissue, align reads, generate feature-spot matrices,
perform clustering and gene expression analysis, and place spots in spatial context on the slide image. 
 - [SparK/2.6.2-GCCcore-10.2.0](https://github.com/harbourlab/SparK)
SparK
 - [Spark/3.1.1-foss-2020b](https://spark.apache.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/Spark/Spark-3.1.1-foss-2020b.eb)
Spark is Hadoop MapReduce done in memory
 - [SpectrA/1.0.1-GCC-11.2.0](https://spectralib.org/)
Spectra stands for Sparse Eigenvalue Computation Toolkit as a Redesigned ARPACK. It is a C++
 library for large scale eigenvalue problems, built on top of Eigen, an open source linear algebra library.
 - [Sphinx/1.8.1-foss-2018b-Python-3.6.6](http://sphinx.pocoo.org/)
Sphinx is a tool that makes it easy to create intelligent and beautiful documentation.
 It was originally created for the new Python documentation, and it has excellent facilities for the 
 documentation of Python projects, but C/C++ is already supported as well, and it is planned to add 
 special support for other languages as well.
 - [Stack/2.3.3-x86_64](https://docs.haskellstack.org)
Stack is a cross-platform program for developing Haskell projects.
It is intended for Haskellers both new and experienced.
 - [Stacks/2.53-foss-2019b](https://catchenlab.life.illinois.edu/stacks/)
Stacks is a software pipeline for building loci from short-read sequences, such as those generated on
 the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq,
 for the purpose of building genetic maps and conducting population genomics and phylogeography.

 - [StringTie/2.1.4-GCC-8.3.0](https://ccb.jhu.edu/software/stringtie/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/StringTie/StringTie-2.1.4-GCC-8.3.0.eb)
StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts
 - [Subread/2.0.3-GCC-11.2.0](http://subread.sourceforge.net/)
High performance read alignment, quantification and mutation discovery
 - [SuiteSparse/5.10.1-foss-2021a-METIS-5.1.0](https://faculty.cse.tamu.edu/davis/suitesparse.html)
SuiteSparse is a collection of libraries manipulate sparse matrices.
 - [SuperLU/5.3.0-foss-2020b](https://crd-legacy.lbl.gov/~xiaoye/SuperLU/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SuperLU/SuperLU-5.3.0-foss-2020b.eb)
SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems
 of linear equations on high performance machines.
 - [Szip/2.1.1-GCCcore-8.3.0](https://www.hdfgroup.org/doc_resource/SZIP/)
Szip compression software, providing lossless compression of scientific data

 - [TOPAZ/0.2.4-foss-2020b](http://cb.csail.mit.edu/cb/topaz/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/TOPAZ/TOPAZ-0.2.4-foss-2020b.eb)
Topaz is a pipeline for particle picking in cryo-electron micrographs using neural networks and positive-unlabeled learning
 - [TRF/4.09-linux64](https://tandem.bu.edu/trf/trf.html)
Tandem repeats finder: a program to analyze DNA sequences. Legacy version.
 - [TRUST4/1.0.7-GCC-11.2.0](https://github.com/liulab-dfci/TRUST4)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/TRUST4/TRUST4-1.0.7-GCC-11.2.0.eb)
Tcr Receptor Utilities for Solid Tissue (TRUST) is a
computational tool to analyze TCR and BCR sequences using unselected RNA
sequencing data, profiled from solid tissues, including tumors. TRUST4 performs
de novo assembly on V, J, C genes including the hypervariable
complementarity-determining region 3 (CDR3) and reports consensus of BCR/TCR
sequences. TRUST4 then realigns the contigs to IMGT reference gene sequences to
report the corresponding information. TRUST4 supports both single-end and
paired-end sequencing data with any read length.
 - [TagDust/2.33-GCCcore-10.2.0](http://tagdust.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/TagDust/TagDust-2.33-GCCcore-10.2.0.eb)
Raw sequences produced by next generation sequencing (NGS) machines may contain adapter,
 linker, barcode and fingerprint sequences. TagDust2 is a program to extract and correctly label the
 sequences to be mapped in downstream pipelines.
 - [Tcl/8.6.12-GCCcore-12.2.0](https://www.tcl.tk/)
Tcl (Tool Command Language) is a very powerful but easy to learn dynamic
 programming language, suitable for a very wide range of uses, including web
 and desktop applications, networking, administration, testing and many more.

 - [Telescope/1.0.3-gfbf-2022b](https://github.com/mlbendall/telescope)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/Telescope/Telescope-1.0.3-gfbf-2022b.eb)
Single locus resolution of Transposable ELEment expression using next-generation sequencing.
 - [TensorFlow/2.7.1-foss-2021b-CUDA-11.4.1](https://www.tensorflow.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/TensorFlow/TensorFlow-2.7.1-foss-2021b-CUDA-11.4.1.eb)
An open-source software library for Machine Intelligence
 - [Theano/1.1.2-foss-2020b-PyMC](https://deeplearning.net/software/theano)
Theano is a Python library that allows you to define, optimize,
and evaluate mathematical expressions involving multi-dimensional arrays efficiently.
 - [Tk/8.6.12-GCCcore-12.2.0](https://www.tcl.tk/)
Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for
 building a graphical user interface (GUI) in many different programming languages.
 - [Tkinter/3.10.8-GCCcore-12.2.0](https://python.org/)
Tkinter module, built with the Python buildsystem
 - [TopHat/2.1.2-foss-2018b](http://ccb.jhu.edu/software/tophat/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/TopHat/TopHat-2.1.2-foss-2018b.eb)
TopHat is a fast splice junction mapper for RNA-Seq reads.
 - [Tracer/1.7.1](http://beast.community/tracer)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/Tracer/Tracer-1.7.1.eb)
Tracer is a program for analysing the trace files generated
 by Bayesian MCMC runs (that is, the continuous parameter values sampled from
 the chain). It can be used to analyse runs of BEAST, MrBayes, LAMARC and
 possibly other MCMC programs.
 - [Trim_Galore/0.6.7-GCCcore-11.2.0](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/Trim_Galore/Trim_Galore-0.6.7-GCCcore-11.2.0.eb)
Trim Galore! is a wrapper script to automate quality and adapter
trimming as well as quality control, with some added functionality to remove biased
methylation positions for RRBS sequence files (for directional, non-directional 
(or paired-end) sequencing).
 - [Trimmomatic/0.39-Java-11](http://www.usadellab.org/cms/?page=trimmomatic)
Trimmomatic performs a variety of useful trimming tasks for illumina 
 paired-end and single ended data.The selection of trimming steps and their associated 
 parameters are supplied on the command line. 
 - [Trinity/2.12.0-foss-2020b](https://trinityrnaseq.github.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/Trinity/Trinity-2.12.0-foss-2020b.eb)
Trinity represents a novel method for the efficient and robust de novo reconstruction
 of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,
 Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads.
 - [UCC/1.1.0-GCCcore-12.2.0](https://www.openucx.org/)
UCC (Unified Collective Communication) is a collective
communication operations API and library that is flexible, complete, and 
feature-rich for current and emerging programming models and runtimes.

 - [UCX/1.13.1-GCCcore-12.2.0](https://www.openucx.org/)
Unified Communication X
An open-source production grade communication framework for data centric
and high-performance applications

 - [UCX-CUDA/1.13.1-GCCcore-12.2.0-CUDA-12.0.0](http://www.openucx.org/)
Unified Communication X
An open-source production grade communication framework for data centric
and high-performance applications

This module adds the UCX CUDA support.

 - [UDUNITS/2.2.28-GCCcore-11.2.0](https://www.unidata.ucar.edu/software/udunits/)
UDUNITS supports conversion of unit specifications between formatted and binary forms,
 arithmetic manipulation of units, and conversion of values between compatible scales of measurement.
 - [UMI-tools/1.0.1-foss-2019b-Python-3.7.4](https://umi-tools.readthedocs.io)
Tools for handling Unique Molecular Identifiers in NGS data sets
 - [UnZip/6.0-GCCcore-11.3.0](http://www.info-zip.org/UnZip.html)
UnZip is an extraction utility for archives compressed
in .zip format (also called "zipfiles"). Although highly compatible both
with PKWARE's PKZIP and PKUNZIP utilities for MS-DOS and with Info-ZIP's
own Zip program, our primary objectives have been portability and
non-MSDOS functionality.
 - [V8/3.4.0-foss-2019b-R-4.0.2](https://cran.r-project.org/web/packages/V8/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/v/V8/V8-3.4.0-foss-2019b-R-4.0.2.eb)
R interface to Google's open source JavaScript engine
 - [VCFtools/0.1.16-GCC-11.2.0](https://vcftools.github.io)
The aim of VCFtools is to provide 
 easily accessible methods for working with complex 
 genetic variation data in the form of VCF files.
 - [VEP/103.1-GCC-10.2.0](https://www.ensembl.org/info/docs/tools/vep)
Variant Effect Predictor (VEP) determines the effect of your
 variants (SNPs, insertions, deletions, CNVs or structural variants) on genes,
 transcripts, and protein sequence, as well as regulatory regions.
 Includes EnsEMBL-XS, which provides pre-compiled replacements for frequently
 used routines in VEP.
 - [VSEARCH/2.21.1-GCC-11.2.0](https://github.com/torognes/vsearch)
VSEARCH supports de novo and reference based chimera detection,
 clustering, full-length and prefix dereplication, rereplication,
 reverse complementation, masking, all-vs-all pairwise global alignment,
 exact and global alignment searching, shuffling, subsampling and sorting.
 It also supports FASTQ file analysis, filtering,
 conversion and merging of paired-end reads.
 - [VTK/8.2.0-foss-2019b-Python-3.7.4](https://www.vtk.org)
The Visualization Toolkit (VTK) is an open-source, freely available software system for
 3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several
 interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization
 algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques
 such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation.
 - [Valgrind/3.16.1-gompi-2020b](https://valgrind.org)
Valgrind: Debugging and profiling tools
 - [VarScan/2.4.4-Java-11](https://github.com/dkoboldt/varscan)
Variant calling and somatic mutation/CNV detection for next-generation sequencing data
 - [ViennaRNA/2.5.1-foss-2021b](https://www.tbi.univie.ac.at/RNA/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/v/ViennaRNA/ViennaRNA-2.5.1-foss-2021b.eb)
The Vienna RNA Package consists of a C code library and several
stand-alone programs for the prediction and comparison of RNA secondary structures.
 - [Vim/9.0.0950-GCCcore-11.2.0](http://www.vim.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/v/Vim/Vim-9.0.0950-GCCcore-11.2.0.eb)
Vim is an advanced text editor that seeks to provide the power 
 of the de-facto Unix editor 'Vi', with a more complete feature set. 
 - [Wayland/1.21.0-GCCcore-11.2.0](https://wayland.freedesktop.org/)
Wayland is a project to define a protocol for a compositor to talk to
 its clients as well as a library implementation of the protocol.  The
 compositor can be a standalone display server running on Linux kernel
 modesetting and evdev input devices, an X application, or a wayland
 client itself.  The clients can be traditional applications, X servers
 (rootless or fullscreen) or other display servers.

 - [Waylandpp/1.0.0-GCCcore-11.2.0](https://github.com/NilsBrause/waylandpp)
Wayland is an object oriented display protocol, which 
features request and events. Requests can be seen as method calls on 
certain objects, whereas events can be seen as signals of an object. 
This makes the Wayland protocol a perfect candidate for a C++ binding.

The goal of this library is to create such a C++ binding for Wayland 
using the most modern C++ technology currently available, providing 
an easy to use C++ API to Wayland.
 - [WebKitGTK+/2.37.1-GCC-11.2.0](https://webkitgtk.org/)
WebKitGTK+ is a full-featured port of the WebKit
rendering engine, suitable for projects requiring any kind of web
integration, from hybrid HTML/CSS applications to full-fledged web
browsers. It offers WebKit's full functionality and is useful in a wide
range of systems from desktop computers to embedded systems like phones,
tablets, and televisions.
 - [WiggleTools/1.2.4-GCC-8.3.0]()
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/w/WiggleTools/WiggleTools-1.2.4-GCC-8.3.0.eb)
The WiggleTools package allows genomewide data files to be manipulated as numerical functions, equipped with all the standard functional analysis operators (sum, product, product by a scalar, comparators), and derived statistics (mean, median, variance, stddev, t-test, Wilcoxon's rank sum test, etc).
 - [X11/20221110-GCCcore-12.2.0](https://www.x.org)
The X Window System (X11) is a windowing system for bitmap displays
 - [XGBoost/0.90-foss-2019b-Python-3.7.4](https://github.com/dmlc/xgboost)
XGBoost is an optimized distributed gradient boosting library designed to be highly efficient,
 flexible and portable.
 - [XML-Compile/1.63-GCCcore-11.2.0](https://metacpan.org/pod/XML::Compile)
Perl module for compilation based XML processing
 - [XML-LibXML/2.0208-GCCcore-12.2.0](https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/x/XML-LibXML/XML-LibXML-2.0208-GCCcore-12.2.0.eb)
Perl binding for libxml2
 - [XML-Parser/2.44_01-GCCcore-7.3.0-Perl-5.28.0](http://search.cpan.org/~toddr/XML-Parser-2.41/)
This is a Perl extension interface to James Clark's XML parser, expat.
 - [XZ/5.2.7-GCCcore-12.2.0](https://tukaani.org/xz/)
xz: XZ utilities
 - [XeniumRanger/1.6.0](https://www.10xgenomics.com/support/software/xenium-ranger)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/x/XeniumRanger/XeniumRanger-1.6.0.eb)
The Xenium In Situ software suite is a set of software
 applications for analyzing and visualizing in situ gene expression data
 produced by the Xenium Analyzer.
 - [Xerces-C++/3.2.4-GCCcore-12.2.0](https://xerces.apache.org/xerces-c/)
Xerces-C++ is a validating XML parser written in a portable
subset of C++. Xerces-C++ makes it easy to give your application the ability to
read and write XML data. A shared library is provided for parsing, generating,
manipulating, and validating XML documents using the DOM, SAX, and SAX2
APIs.
 - [XlsxWriter/3.0.2-GCCcore-11.2.0](https://xlsxwriter.readthedocs.io/)
A Python module for creating Excel XLSX files
 - [Xvfb/21.1.6-GCCcore-12.2.0](https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml)
Xvfb is an X server that can run on machines with no display hardware and no physical input devices.
 It emulates a dumb framebuffer using virtual memory.
 - [YACS/0.1.8-GCCcore-10.2.0](https://github.com/rbgirshick/yacs)
YACS was created as a lightweight library to define and
manage system configurations, such as those commonly found in software
designed for scientific experimentation. These "configurations"
typically cover concepts like hyperparameters used in training a machine
learning model or configurable model hyperparameters, such as the depth
of a convolutional neural network.
 - [Yasm/1.3.0-GCCcore-12.2.0](https://www.tortall.net/projects/yasm/)
Yasm: Complete rewrite of the NASM assembler with BSD license
 - [Z3/4.12.2-GCCcore-12.2.0](https://github.com/Z3Prover/z3)
Z3 is a theorem prover from Microsoft Research.

 - [ZeroMQ/4.3.4-GCCcore-12.2.0](https://www.zeromq.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/z/ZeroMQ/ZeroMQ-4.3.4-GCCcore-12.2.0.eb)
ZeroMQ looks like an embeddable networking library but acts like a concurrency framework.
 It gives you sockets that carry atomic messages across various transports like in-process,
 inter-process, TCP, and multicast. You can connect sockets N-to-N with patterns like fanout,
 pub-sub, task distribution, and request-reply. It's fast enough to be the fabric for clustered
 products. Its asynchronous I/O model gives you scalable multicore applications, built as asynchronous
 message-processing tasks. It has a score of language APIs and runs on most operating systems.
 - [Zip/3.0-GCCcore-8.3.0](http://www.info-zip.org/Zip.html)
Zip is a compression and file packaging/archive utility.
Although highly compatible both with PKWARE's PKZIP and PKUNZIP
utilities for MS-DOS and with Info-ZIP's own UnZip, our primary objectives
have been portability and other-than-MSDOS functionality
 - [agrep/2.04-GCCcore-8.3.0](https://github.com/Wikinaut/agrep)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/a/agrep/agrep-2.04-GCCcore-8.3.0.eb)
AGREP
 - [aiohttp/3.8.5-GCCcore-12.2.0](https://github.com/aio-libs/aiohttp)
Asynchronous HTTP client/server framework for asyncio and Python.
 - [alleleCount/4.2.1-GCC-11.2.0](http://cancerit.github.io/alleleCount/)
The alleleCount package primarily exists to prevent code duplication
between some other projects, specifically AscatNGS and Battenberg. As of v4 the perl 
code wraps the C implementation of allele counting code for BAM/CRAM processing. 
 - [almosthere/2.0.2-GCCcore-10.2.0](https://github.com/horta/almosthere)
Progress indicator C library.
ATHR is a simple yet powerful progress indicator library that works on Windows, 
Linux, and macOS. It is non-blocking as the progress update is done via a 
dedicated, lightweight thread, as to not impair the performance of the calling program.
 - [ancestry/1.0.0-GCCcore-8.3.0-Python-2.7.16](https://github.com/vibansal)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/a/ancestry/ancestry-1.0.0-GCCcore-8.3.0-Python-2.7.16.eb)
Fast individual ancestry inference from DNA sequence data leveraging allele frequencies
 from multiple populations. iAdmix Using population allele frequencies for computing individual admixture
 estimates
 - [angsd/0.933-GCC-8.3.0](http://www.popgen.dk/angsd)
Program for analysing NGS data.
 - [annovar/20200607-GCCcore-11.2.0-Perl-5.34.0](http://annovar.openbioinformatics.org/en/latest/)
ANNOVAR is an efficient software tool to utilize update-to-date information 
to functionally annotate genetic variants detected from diverse genomes (including human 
genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others).
 - [ant/1.10.11-Java-11](https://ant.apache.org/)
Apache Ant is a Java library and command-line tool whose mission is to drive processes described in
 build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of
 Java applications.
 - [arcasHLA/0.2.0-foss-2019b-Python-3.7.4](https://github.com/RabadanLab/arcasHLA)
arcasHLA performs high resolution genotyping for HLA class I and class II
genes from RNA sequencing, supporting both paired and single-end samples.
 - [argtable/2.13-GCCcore-8.3.0](http://argtable.sourceforge.net/)
Argtable is an ANSI C library for parsing GNU style
 command line options with a minimum of fuss. 
 - [aria2/1.35.0-GCCcore-10.2.0](https://aria2.github.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/a/aria2/aria2-1.35.0-GCCcore-10.2.0.eb)
aria2 is a lightweight multi-protocol and multi-source command-line download utility.
 - [arpack-ng/3.8.0-foss-2022b](https://github.com/opencollab/arpack-ng)
ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems.
 - [arrow-R/6.0.0.2-foss-2021b-R-4.2.0](https://cran.r-project.org/web/packages/arrow)
R interface to the Apache Arrow C++ library
 - [at-spi2-atk/2.38.0-GCCcore-12.2.0](https://wiki.gnome.org/Accessibility)
AT-SPI 2 toolkit bridge
 - [at-spi2-core/2.46.0-GCCcore-12.2.0](https://wiki.gnome.org/Accessibility)
Assistive Technology Service Provider Interface.

 - [awscli/2.11.21-GCCcore-11.2.0](https://pypi.python.org/pypi/awscli)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/a/awscli/awscli-2.11.21-GCCcore-11.2.0.eb)
Universal Command Line Environment for AWS
 - [bam-readcount/1.0.1-GCC-12.2.0](https://github.com/genome/bam-readcount)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/bam-readcount/bam-readcount-1.0.1-GCC-12.2.0.eb)
Count DNA sequence reads in BAM files
 - [bam2fastx/1.3.0](https://github.com/pacificbiosciences/bam2fastx)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/bam2fastx/bam2fastx-1.3.0.eb)
Conversion of PacBio BAM files into gzipped fasta and fastq files, including splitting of barcoded data
 - [bam2wig/1.5](https://github.com/MikeAxtell/bam2wig)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/bam2wig/bam2wig-1.5.eb)
Conversion of a BAM alignment to wiggle and bigwig coverage files, with flexible reporting options.
 - [basicfiltering/1.0.7-foss-2020a-Python-3.8.2](https://github.com/rhshah/basicfiltering)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/basicfiltering/basicfiltering-1.0.7-foss-2020a-Python-3.8.2.eb)
Basic Filtering for; Variant Allele Frequency, Variat Reads, tumor-Normal Variant Allele Frequencey Ratio.
 - [bcl-convert/4.0.3-2el7.x86_64](https://support.illumina.com/sequencing/sequencing_software/bcl-convert.html)
The Illumina BCL Convert v4.0 is a standalone local software app that converts the
 Binary Base Call (BCL) files produced by Illumina sequencing systems to FASTQ files.
 - [bcl2fastq/1.8.4](http://www.iodbc.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/bcl2fastq/bcl2fastq-1.8.4.eb)

 - [bcl2fastq2/2.20.0-GCC-11.2.0](https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html)
bcl2fastq Conversion Software both demultiplexes data and converts BCL files generated by
 Illumina sequencing systems to standard FASTQ file formats for downstream analysis.
 - [beagle-lib/3.1.2-GCCcore-8.3.0](https://github.com/beagle-dev/beagle-lib)
beagle-lib is a high-performance library that can perform the core calculations at the heart of most
 Bayesian and Maximum Likelihood phylogenetics packages.
 - [bgen/4.1.3-GCCcore-10.2.0](https://github.com/limix/bgen)
A BGEN file format reader. It fully supports the BGEN format specifications 1.2 and 1.3.
 - [binutils/2.39-GCCcore-12.2.0](https://directory.fsf.org/project/binutils/)
binutils: GNU binary utilities
 - [bioawk/1.0-foss-2018b](https://github.com/lh3/bioawk)
Bioawk is an extension to Brian Kernighan's awk,
 adding the support of several common biological data formats,
 including optionally gzip'ed BED, GFF, SAM, VCF, FASTA/Q and TAB-delimited formats with column names. 
 - [biobambam2/2.0.95-foss-2018b](https://github.com/gt1/biobambam2)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/biobambam2/biobambam2-2.0.95-foss-2018b.eb)
Tools for processing BAM files; 
 bamsormadup, bamcollate2, bammarkduplicates, bammaskflags, bamrecompress,
 bamsort, bamtofastq
 - [bokeh/3.2.1-foss-2022b](https://github.com/bokeh/bokeh)
Statistical and novel interactive HTML plots for Python
 - [boto3/1.20.13-GCCcore-11.2.0](https://github.com/boto/boto3)
Boto3 is the Amazon Web Services (AWS) Software Development Kit
(SDK) for Python, which allows Python developers to write software that makes
use of services like Amazon S3 and Amazon EC2.
 - [brotli/1.0.9-GCC-8.3.0](https://github.com/google/brotli)
Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate but offers more dense compression.

The specification of the Brotli Compressed Data Format is defined in RFC 7932.
 - [bsddb3/6.2.9-GCCcore-10.2.0](https://pypi.org/project/bsddb3/)
bsddb3 is a nearly complete Python binding of the
Oracle/Sleepycat C API for the Database Environment, Database, Cursor,
Log Cursor, Sequence and Transaction objects.
 - [bx-python/0.8.13-foss-2021b](https://github.com/bxlab/bx-python)
The bx-python project is a Python library and associated set of scripts to allow for rapid
 implementation of genome scale analyses.
 - [bzip2/1.0.8-GCCcore-10.2.0](https://sourceware.org/bzip2)
bzip2 is a freely available, patent free, high-quality data compressor. It
 typically compresses files to within 10% to 15% of the best available
 techniques (the PPM family of statistical compressors), whilst being around
 twice as fast at compression and six times faster at decompression.

 - [c-ares/1.18.1-GCCcore-11.2.0](https://c-ares.org/)
c-ares is a C library for asynchronous DNS requests (including name resolves)
 - [cDNA_Cupcake/12.4.0-foss-2019b-Python-3.7.4](https://github.com/Magdoll/cDNA_Cupcake)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/cDNA_Cupcake/cDNA_Cupcake-12.4.0-foss-2019b-Python-3.7.4.eb)
cDNA_Cupcake is a miscellaneous collection of Python and R scripts used for analyzing sequencing data.
 - [cURL/7.86.0-GCCcore-12.2.0](https://curl.haxx.se)
libcurl is a free and easy-to-use client-side URL transfer library,
 supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP,
 LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.
 libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP
 form based upload, proxies, cookies, user+password authentication (Basic,
 Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling
 and more.

 - [cairo/1.17.4-GCCcore-12.2.0](https://cairographics.org)
Cairo is a 2D graphics library with support for multiple output devices.
 Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers,
 PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB
 - [canu/1.8-foss-2018b-Perl-5.28.0](http://canu.readthedocs.io)
Canu is a fork of the Celera Assembler designed for high-noise single-molecule sequencing
 - [cas-offinder/2.4-foss-2018b]()
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/cas-offinder/cas-offinder-2.4-foss-2018b.eb)
Cas-OFFinder is OpenCL based, ultrafast and versatile program that searches
 for potential off-target sites of CRISPR/Cas-derived RNA-guided endonucleases (RGEN).
 - [ccache/4.6.1-GCCcore-11.2.0](https://ccache.dev/)
Ccache (or “ccache”) is a compiler cache. It speeds up recompilation by
caching previous compilations and detecting when the same compilation is being done again
 - [cellranger/3.1.0](https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome)
Chromium Single Cell Software Suite is a set of software applications for analyzing and visualizing single cell 3’ RNA-seq data produced by the 10x Genomics Chromium Platform.
 - [cellranger-atac/1.1.0-foss-2018b](https://support.10xgenomics.com/single-cell-atac/software/overview/welcome)
The Chromium Single Cell ATAC Software Suite is a complete package for analyzing and visualizing single cell chromatin accessibility data produced by the Chromium Single Cell ATAC Solution on the 10x Chromium Platform.
 - [cget/0.2.0-GCCcore-10.2.0](https://cget.readthedocs.io/en/latest/index.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/cget/cget-0.2.0-GCCcore-10.2.0.eb)
Cmake package retrieval. This can be used to download and install cmake packages
 - [cirro/0.8.0-foss-2022b](https://github.com/FredHutch/Cirro-client)
A Python 3.8+ library for the Cirro platform.
 - [cisTEM/1.0.0-beta](https://cistem.org/)
cisTEM is user-friendly software to process cryo-EM images of macromolecular complexes
 and obtain high-resolution 3D reconstructions from them. 
 - [clusTCR/1.0.2-foss-2019b-Python-3.7.4](https://svalkiers.github.io/clusTCR/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/clusTCR/clusTCR-1.0.2-foss-2019b-Python-3.7.4.eb)
Python interface for rapid clustering of large sets of CDR3 sequences with unknown
 antigen specificity.
 - [cnpy/master-GCC-11.2.0](https://github.com/rogersce/cnpy)
cnpy lets you read and write NumPy arrays saved as npy file formats in C++.
 - [configurable-http-proxy/4.5.5-GCCcore-12.2.0](https://github.com/jupyterhub/configurable-http-proxy)
HTTP proxy for node.js including a REST API for updating the routing table.
 Developed as a part of the Jupyter Hub multi-user server.
 - [cppy/1.2.1-GCCcore-12.2.0](https://github.com/nucleic/cppy)
A small C++ header library which makes it easier to write
Python extension modules. The primary feature is a PyObject smart pointer
which automatically handles reference counting and provides convenience
methods for performing common object operations.
 - [cromwell/84](https://github.com/broadinstitute/cromwell)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/cromwell/cromwell-84.eb)
Scientific workflow engine designed for simplicity & scalability.
 - [ctffind/4.1.14-fosscuda-2019b](https://grigoriefflab.umassmed.edu/ctffind4)
Program for finding CTFs of electron micrographs.
 - [cuDNN/8.8.0.121-CUDA-12.0.0](https://developer.nvidia.com/cudnn)
The NVIDIA CUDA Deep Neural Network library (cuDNN) is
a GPU-accelerated library of primitives for deep neural networks.
 - [cupcake/0.0.4-foss-2019b-Python-3.7.4](https://github.com/thejohnfreeman/cupcake)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/cupcake/cupcake-0.0.4-foss-2019b-Python-3.7.4.eb)
Cupcake is a thin layer over CMake and Conan that tries to offer a better user experience in the style of Yarn or Poetry.
 - [cutadapt/4.1-GCCcore-11.2.0](https://opensource.scilifelab.se/projects/cutadapt/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/cutadapt/cutadapt-4.1-GCCcore-11.2.0.eb)
Cutadapt finds and removes adapter sequences, primers, poly-A tails and
 other types of unwanted sequence from your high-throughput sequencing reads.
 - [dask/2023.7.1-foss-2022b](https://dask.org/)
Dask natively scales Python. Dask provides advanced parallelism for analytics, enabling performance
 at scale for the tools you love.
 - [datamash/1.8-GCCcore-11.2.0](https://www.gnu.org/software/datamash/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/d/datamash/datamash-1.8-GCCcore-11.2.0.eb)
GNU datamash performs basic numeric, textual and statistical operations on input data files
 - [dbus-glib/0.112-GCCcore-11.2.0](https://dbus.freedesktop.org/doc/dbus-glib)
D-Bus is a message bus system, a simple way for applications to talk to one another.
 - [deepTools/3.5.1-foss-2021b](https://deeptools.readthedocs.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/d/deepTools/deepTools-3.5.1-foss-2021b.eb)
deepTools is a suite of python tools particularly developed for the efficient analysis of
 high-throughput sequencing data, such as ChIP-seq, RNA-seq or MNase-seq.
 - [delly/0.8.3](https://github.com/dellytools/delly)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/d/delly/delly-0.8.3.eb)
DELLY2: Structural variant discovery by integrated paired-end and split-read analysis
 - [dill/0.3.7-GCCcore-12.2.0](https://pypi.org/project/dill/)
dill extends python's pickle module for serializing and de-serializing python objects to the majority
 of the built-in python types. Serialization is the process of converting an object to a byte stream, and the inverse
 of which is converting a byte stream back to on python object hierarchy.
 - [dm-tree/0.1.6-GCCcore-11.2.0](https://github.com/deepmind/tree)
dm-tree provides tree, a library for working with nested data structures. In a way,
tree generalizes the builtin map function which only supports flat sequences, and
allows to apply a function to each "leaf" preserving the overall structure.
 - [dms_tools2/2.6.11-foss-2020b](https://jbloomlab.github.io/dms_tools2/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/d/dms_tools2/dms_tools2-2.6.11-foss-2020b.eb)
dms_tools2 is a software package for analyzing deep mutational scanning data.
 It is tailored to analyze libraries created using comprehensive codon mutagenesis of protein-coding
 of genes.
 - [dorado/0.3.4-foss-2022a-CUDA-11.7.0](https://github.com/nanoporetech/dorado)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/d/dorado/dorado-0.3.4-foss-2022a-CUDA-11.7.0.eb)
Dorado is a high-performance, easy-to-use, open source basecaller for Oxford Nanopore reads.
 - [dotNET-SDK/3.1.300-linux-x64](https://www.microsoft.com/net/)
.NET is a free, cross-platform, open source developer platform for building many different types
 of applications.
 - [double-conversion/3.2.1-GCCcore-12.2.0](https://github.com/google/double-conversion)
Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles.
 - [easel/0.48-GCC-12.2.0](https://www.wxwidgets.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/e/easel/easel-0.48-GCC-12.2.0.eb)
Easel supports computational analysis of biological sequences using probabilistic models.
 - [edlib/1.3.8.post1-GCC-8.3.0-Python-3.7.4](https://martinsos.github.io/edlib)
Lightweight, super fast library for sequence alignment using edit (Levenshtein) distance.
 - [eggnog-mapper/2.1.7-foss-2021b](https://github.com/eggnogdb/eggnog-mapper)
EggNOG-mapper is a tool for fast functional annotation of novel
sequences. It uses precomputed orthologous groups and phylogenies from the
eggNOG database (http://eggnog5.embl.de) to transfer functional information from
fine-grained orthologs only. Common uses of eggNOG-mapper include the annotation
of novel genomes, transcriptomes or even metagenomic gene catalogs.
 - [elfutils/0.189-GCCcore-12.2.0](https://elfutils.org/)
The elfutils project provides libraries and tools for ELF files
 and DWARF data.

 - [enchant-2/2.3.3-GCCcore-11.2.0](https://github.com/AbiWord/enchant)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/e/enchant-2/enchant-2-2.3.3-GCCcore-11.2.0.eb)
Enchant aims to provide a simple but comprehensive abstraction for dealing
with different spell checking libraries in a consistent way. A client, such
as a text editor or word processor, need not know anything about a specific
spell-checker, and since all back-ends are plugins, new spell-checkers can
be added without needing any change to the program using Enchant.
 - [expat/2.4.9-GCCcore-12.2.0](https://libexpat.github.io)
Expat is an XML parser library written in C. It is a stream-oriented parser
in which an application registers handlers for things the parser might find
in the XML document (like start tags).
 - [expect/5.45.4-GCCcore-9.3.0](https://core.tcl.tk/expect/index)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/e/expect/expect-5.45.4-GCCcore-9.3.0.eb)
Expect is a tool for automating interactive applications 
  such as telnet, ftp, passwd, fsck, rlogin, tip, etc. 
  Expect really makes this stuff trivial. 
  Expect is also useful for testing these same applications.
 - [expecttest/0.1.3-GCCcore-10.2.0](https://github.com/ezyang/expecttest)
This library implements expect tests (also known as "golden" tests). Expect tests are a method of
 writing tests where instead of hard-coding the expected output of a test, you run the test to get the output, and
 the test framework automatically populates the expected output. If the output of the test changes, you can rerun
 the test with the environment variable EXPECTTEST_ACCEPT=1 to automatically update the expected output.
 - [factera/1.4.4-foss-2019b-Perl-5.30.0](https://factera.stanford.edu/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/factera/factera-1.4.4-foss-2019b-Perl-5.30.0.eb)
(Fusion And Chromosomal Translocation Enumeration and Recovery Algorithm) is a tool for detection of genomic fusions in paired-end targeted (or genome-wide) sequencing data.
 - [faiss/1.7.3-foss-2021b-CUDA-11.4.1](https://ai.facebook.com/tools/faiss/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/faiss/faiss-1.7.3-foss-2021b-CUDA-11.4.1.eb)
FAISS (Facebook AI Similarity Search) is a library that allows developers to
 quickly search for embeddings of multimedia documents that are similar to each other.
 - [fast5/0.6.5](http://simpsonlab.github.io/2017/02/27/packing_fast5/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/fast5/fast5-0.6.5.eb)
A lightweight C++ library for accessing Oxford Nanopore
 Technologies sequencing data.
 - [fastNGSadmix/dda93a4-GCC-10.2.0](http://www.popgen.dk/software/index.php/FastNGSadmix)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/fastNGSadmix/fastNGSadmix-dda93a4-GCC-10.2.0.eb)
Program for infering admixture proportions and doing PCA with a single NGS
 sample. Inferences based on reference panel.
 - [fastai/2.7.10-foss-2021b-CUDA-11.4.1](https://github.com/fastai/fastai)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/fastai/fastai-2.7.10-foss-2021b-CUDA-11.4.1.eb)
fastai is a deep learning library which provides practitioners with high-level components.
 - [fastp/0.20.0-GCC-8.3.0](https://github.com/OpenGene/fastp)
A tool designed to provide fast all-in-one preprocessing for FastQ files.
 This tool is developed in C++ with multithreading supported to afford high performance.
 - [fastq-tools/0.8.3-GCC-11.2.0](https://homes.cs.washington.edu/~dcjones/fastq-tools/)
This package provides a number of small and efficient programs to perform
 common tasks with high throughput sequencing data in the FASTQ format. All of the programs
 work with typical FASTQ files as well as gzipped FASTQ files.
 - [fermi-lite/20190320-GCCcore-11.2.0](https://github.com/lh3/fermi-lite)
Standalone C library for assembling Illumina short reads in small regions.
 - [ffnvcodec/11.1.5.2](https://git.videolan.org/?p=ffmpeg/nv-codec-headers.git)
FFmpeg nvidia headers. Adds support for nvenc and nvdec. Requires Nvidia GPU and drivers to be present
(picked up dynamically).
 - [fgbio/2.0.2](https://github.com/fulcrumgenomics/fgbio)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/fgbio/fgbio-2.0.2.eb)
A set of tools to analyze genomic data with a focus on Next Generation Sequencing.
 - [fhCellRanger/7.1.0](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger)
Cell Ranger is a set of analysis pipelines that process Chromium
 single-cell RNA-seq output to align reads, generate gene-cell matrices and perform
 clustering and gene expression analysis.
 - [fhCellpose/0.6.5-fosscuda-2020b](http://cellpose.org/)
Cellpose is an anatomical segmentation algorithm written in Python 3 by Carsen 
Stringer and Marius Pachitariu.
 - [fhDev/GCCcore-8.3.0](https://fredhutch.github.io/easybuild-life-sciences/)
fhDev Fred Hutch Development environment is a collection
 of development tools that will work with LMOD modules for a given environment.
 
 - [fhPerl/5.34.0-GCCcore-11.2.0](https://www.perl.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/fhPerl/fhPerl-5.34.0-GCCcore-11.2.0.eb)
Larry Wall's Practical Extraction and Report Language
 - [fhPython/3.9.6-foss-2021b](http://python.org/)
Fred Hutch Python
 - [fhR/4.3.1-foss-2022b](http://www.r-project.org/)
R is a free software environment for statistical computing and graphics.
 - [fhSeurat/4.1.1-foss-2021b-R-4.2.0](https://satijalab.org/seurat)
Seurat is an R package designed for QC, analysis, and exploration of single cell RNA-seq data.
fhSeurat module has additional Bioconductor packages for single-cell analysis.
 - [fhStack/2.3.3-x86_64](https://docs.haskellstack.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/fhStack/fhStack-2.3.3-x86_64.eb)
Stack is a cross-platform program for developing Haskell projects.
It is intended for Haskellers both new and experienced.
 - [fig2dev/3.2.6a-foss-2019b](http://www.xfig.org/)
Xfig is an interactive drawing tool which runs under X Window System.
 - [file/5.41-GCCcore-11.2.0](https://www.darwinsys.com/file/)
The file command is 'a file type guesser', that is, a command-line tool
 that tells you in words what kind of data a file contains.
 - [filezilla/3.62.2-GCC-11.2.0](https://filezilla-project.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/filezilla/filezilla-3.62.2-GCC-11.2.0.eb)
FileZilla Client is a fast and reliable cross-platform FTP, FTPS and SFTP client with lots of useful features and an intuitive graphical user interface.
 - [flatbuffers/2.0.0-GCCcore-11.2.0](https://github.com/google/flatbuffers/)
FlatBuffers: Memory Efficient Serialization Library
 - [flatbuffers-python/2.0-GCCcore-11.2.0](https://github.com/google/flatbuffers/)
Python Flatbuffers runtime library.
 - [flex/2.6.4-GCCcore-11.2.0](http://flex.sourceforge.net/)
Flex (Fast Lexical Analyzer) is a tool for generating scanners. A scanner,
 sometimes called a tokenizer, is a program which recognizes lexical patterns
 in text.

 - [fmt/7.1.1-GCCcore-11.2.0](http://fmtlib.net/)
fmt (formerly cppformat) is an open-source formatting library.
 - [fontconfig/2.14.1-GCCcore-12.2.0](https://www.freedesktop.org/wiki/Software/fontconfig/)
Fontconfig is a library designed to provide system-wide font configuration,
 customization and application access.

 - [foss/2022b](https://easybuild.readthedocs.io/en/master/Common-toolchains.html#foss-toolchain)
GNU Compiler Collection (GCC) based compiler toolchain, including
 OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.
 - [fosscuda/2020b]((none))
GCC based compiler toolchain __with CUDA support__, and including
 OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.
 - [freebayes/1.3.2-GCCcore-8.3.0](https://github.com/ekg/freebayes)
Bayesian haplotype-based polymorphism discovery and genotyping.
 - [freeglut/3.2.1-GCCcore-8.3.0](http://freeglut.sourceforge.net/)
freeglut is a completely OpenSourced alternative to the OpenGL Utility Toolkit (GLUT) library.
 - [freetds/1.2-GCCcore-9.3.0](http://www.freedesktop.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/freetds/freetds-1.2-GCCcore-9.3.0.eb)
FreeTDS is a set of libraries for Unix and Linux that allows your programs to
 natively talk to Microsoft SQL Server and Sybase databases.
 - [freetype/2.12.1-GCCcore-11.3.0](https://www.freetype.org)
FreeType 2 is a software font engine that is designed to be small, efficient,
 highly customizable, and portable while capable of producing high-quality
 output (glyph images). It can be used in graphics libraries, display servers,
 font conversion tools, text image generation tools, and many other products
 as well.

 - [future/0.18.3-foss-2021b](http://python-future.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/future/future-0.18.3-foss-2021b.eb)
python-future is the missing compatibility layer between Python 2 and Python 3.
 - [g2lib/3.1.0-foss-2018b](https://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/)
Library contains GRIB2 encoder/decoder and search/indexing routines.
 - [gc/8.2.0-GCCcore-11.2.0](https://hboehm.info/gc/)
The Boehm-Demers-Weiser conservative garbage collector can be used as a
garbage collecting replacement for C malloc or C++ new.

 - [gcccuda/2020b]((none))
GNU Compiler Collection (GCC) based compiler toolchain, along with CUDA toolkit.
 - [gcloud/439.0.0](https://cloud.google.com/sdk)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/gcloud/gcloud-439.0.0.eb)
Libraries and tools for interacting with Google Cloud products and services.
 - [gdc-client/1.6.1-GCCcore-10.2.0](https://gdc.nci.nih.gov/access-data/gdc-data-transfer-tool)
The gdc-client provides several convenience functions over
 the GDC API which provides general download/upload via HTTPS.
 - [gettext/0.21.1-GCCcore-12.2.0](https://www.gnu.org/software/gettext/)
GNU 'gettext' is an important step for the GNU Translation Project, as it is an asset on which we may
build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools
and documentation
 - [gfbf/2022b]((none))
GNU Compiler Collection (GCC) based compiler toolchain, including
 FlexiBLAS (BLAS and LAPACK support) and (serial) FFTW.
 - [gffread/0.11.6-GCCcore-8.3.0](https://github.com/gpertea/gffread)
GFF/GTF parsing utility providing format conversions,
region filtering, FASTA sequence extraction and more.
 - [gflags/2.2.2-GCCcore-8.3.0](https://github.com/gflags/gflags)
The gflags package contains a C++ library that implements commandline flags
processing.  It includes built-in support for standard types such as string
and the ability to define flags in the source file in which they are used.

 - [ggVennDiagram/3484e8-foss-2019b-R-4.0.2](https://cran.r-project.org/package=VennDiagram)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/ggVennDiagram/ggVennDiagram-3484e8-foss-2019b-R-4.0.2.eb)
A set of functions to generate high-resolution Venn and Euler plots. Includes handling for several special cases, including two-case scaling, and extensive customization of plot shape and structure.
 - [gh/2.14.7](https://github.com/cli/cli)
gh is GitHub on the command line.
 - [giflib/5.2.1-GCCcore-10.2.0](http://giflib.sourceforge.net/)
giflib is a library for reading and writing gif images.
It is API and ABI compatible with libungif which was in wide use while
the LZW compression algorithm was patented.
 - [giggle/master-foss-2020b](https://github.com/ryanlayer/giggle)
GIGGLE is a genomics search engine that identifies and ranks the significance of shared genomic loci between query features and thousands of genome interval files.
 - [git/2.38.1-GCCcore-12.2.0-nodocs](https://git-scm.com)
Git is a free and open source distributed version control system designed
to handle everything from small to very large projects with speed and efficiency.
 - [git-lfs/2.11.0](https://git-lfs.github.com)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/git-lfs/git-lfs-2.11.0.eb)
Git Large File Storage (LFS) replaces large files such as audio
 samples, videos, datasets, and graphics with text pointers inside Git, while 
 storing the file contents on a remote server like GitHub.com
 - [glew/2.2.0-GCCcore-11.2.0-egl](https://github.com/nigels-com/glew)
The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source
C/C++ extension loading library. GLEW provides efficient run-time mechanisms
for determining which OpenGL extensions are supported on the target platform.
 - [glib-networking/2.72.1-GCCcore-11.2.0](https://gitlab.gnome.org/GNOME/glib-networking)
Network extensions for GLib
 - [glog/0.4.0-GCCcore-8.3.0](https://github.com/google/glog)
A C++ implementation of the Google logging module.
 - [gmpy2/2.1.5-GCC-12.2.0](https://github.com/aleaxit/gmpy)
GMP/MPIR, MPFR, and MPC interface to Python 2.6+ and 3.x
 - [gnuplot/5.4.6-GCCcore-12.2.0](http://gnuplot.sourceforge.net)
Portable interactive, function plotting utility
 - [gompi/2022b]((none))
GNU Compiler Collection (GCC) based compiler toolchain,
 including OpenMPI for MPI support.
 - [gompic/2020b]((none))
GNU Compiler Collection (GCC) based compiler toolchain along with CUDA toolkit,
 including OpenMPI for MPI support with CUDA features enabled.
 - [googletest/1.12.1-GCCcore-12.2.0](https://github.com/google/googletest)
Google's framework for writing C++ tests on a variety of platforms
 - [gperf/3.1-GCCcore-9.3.0](https://www.gnu.org/software/gperf/)
GNU gperf is a perfect hash function generator. For a given list of strings,
 it produces a hash function and hash table, in form of C or C++ code, for
 looking up a value depending on the input string. The hash function is
 perfect, which means that the hash table has no collisions, and the hash
 table lookup needs a single string comparison only.

 - [gperftools/2.9.1-GCCcore-11.2.0](https://github.com/gperftools/gperftools)
gperftools is a collection of a high-performance multi-threaded malloc()
implementation, plus some pretty nifty performance analysis tools.
Includes TCMalloc, heap-checker, heap-profiler and cpu-profiler.

 - [graphite2/1.3.14-GCCcore-11.2.0](https://scripts.sil.org/cms/scripts/page.php?site_id=projects&item_id=graphite_home)
Graphite is a "smart font" system developed specifically to
 handle the complexities of lesser-known languages of the world.
 - [groff/1.22.4-GCCcore-10.3.0](https://www.gnu.org/software/groff)
Groff (GNU troff) is a typesetting system that reads plain text mixed with formatting commands
 and produces formatted output.
 - [gsutil/5.10-GCCcore-11.2.0](https://cloud.google.com/storage/docs/gsutil)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/gsutil/gsutil-5.10-GCCcore-11.2.0.eb)
gsutil is a Python application that lets you access Cloud Storage from the command line.
 - [gtest/1.8.1-GCCcore-8.3.0](https://github.com/google/googletest)
Google's framework for writing C++ tests on a variety of platforms
 - [guidescan/1.2-foss-2018b-Python-2.7.15](http://www.guidescan.com)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/guidescan/guidescan-1.2-foss-2018b-Python-2.7.15.eb)
A generalized CRISPR guideRNA design tool.
 - [gzip/1.12-GCCcore-12.2.0](https://www.gnu.org/software/gzip/)
gzip (GNU zip) is a popular data compression program as a replacement for compress
 - [h5py/3.8.0-foss-2022b](https://www.h5py.org/)
HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,
 version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous
 amounts of data.
 - [hatchling/1.11.1-GCCcore-11.2.0](https://hatch.pypa.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/h/hatchling/hatchling-1.11.1-GCCcore-11.2.0.eb)
Extensible, standards compliant build backend used by Hatch,
a modern, extensible Python project manager.
 - [help2man/1.49.2-GCCcore-11.3.0](https://www.gnu.org/software/help2man/)
help2man produces simple manual pages from the '--help' and '--version' output of other commands.
 - [hiredis/1.0.2-GCCcore-11.2.0](https://github.com/redis/hiredis)
Hiredis is a minimalistic C client library for the Redis database.

It is minimalistic because it just adds minimal support for the protocol, 
but at the same time it uses a high level printf-alike API in order to 
make it much higher level than otherwise suggested by its minimal code base 
and the lack of explicit bindings for every Redis command.
 - [hivmmer/0.1.2-foss-2018b-Python-3.6.6](https://github.com/kantorlab/hivmmer)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/h/hivmmer/hivmmer-0.1.2-foss-2018b-Python-3.6.6.eb)
An alignment and variant-calling pipeline for Illumina deep
 sequencing of HIV-1, based on the probabilistic aligner HMMER
 - [hunspell/1.7.1-GCCcore-11.2.0](http://hunspell.github.io/)
Hunspell is a spell checker and morphological analyzer 
library and program designed for languageswith rich morphology and 
complex word compounding or character encoding.
 - [hwloc/2.8.0-GCCcore-12.2.0](https://www.open-mpi.org/projects/hwloc/)
The Portable Hardware Locality (hwloc) software package provides a portable
 abstraction (across OS, versions, architectures, ...) of the hierarchical
 topology of modern architectures, including NUMA memory nodes, sockets, shared
 caches, cores and simultaneous multithreading. It also gathers various system
 attributes such as cache and memory information as well as the locality of I/O
 devices such as network interfaces, InfiniBand HCAs or GPUs. It primarily
 aims at helping applications with gathering information about modern computing
 hardware so as to exploit it accordingly and efficiently.

 - [hyperfreq/1.2.0-foss-2020b](https://python.org/)
Hypermutation analysis software using BetaRat distribution for Bayesian
 analysis of the relative probability ratio (RPR) of observing mutations in two contexts. Includes
 Alnclst, for clustering pre-aligned nucleotide sequences.
 - [hypothesis/6.68.2-GCCcore-12.2.0](https://github.com/HypothesisWorks/hypothesis)
Hypothesis is an advanced testing library for Python. It lets you write tests which are parametrized
 by a source of examples, and then generates simple and comprehensible examples that make your tests fail. This lets
 you find more bugs in your code with less work.
 - [iCount/20180820-foss-2018b-Python-3.6.6](https://github.com/tomazc/iCount)
iCount: protein-RNA interaction analysis
 is a Python module and associated command-line interface (CLI),
 which provides all the commands needed to process iCLIP data on protein-RNA interactions.
 - [iVar/1.3.2-GCC-11.2.0](https://github.com/andersen-lab/ivar)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/iVar/iVar-1.3.2-GCC-11.2.0.eb)
iVar is a computational package that contains functions broadly useful for viral amplicon-based sequencing.

 - [iced/0.5.6-foss-2021b](https://pypi.org/project/iced/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/iced/iced-0.5.6-foss-2021b.eb)
ICE normalization
 - [igraph/0.10.6-foss-2022b](https://igraph.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/igraph/igraph-0.10.6-foss-2022b.eb)
igraph is a collection of network analysis tools with the emphasis on
efficiency, portability and ease of use. igraph is open source and free. igraph can be
programmed in R, Python and C/C++.
 - [imageio/2.31.1-foss-2022b](https://imageio.github.io)
Imageio is a Python library that provides an easy interface to read and write a wide range of
 image data, including animated images, video, volumetric data, and scientific formats.
 - [imkl/2022.2.0](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html)
Intel oneAPI Math Kernel Library
 - [index-hopping-filter/1.0.1](https://support.10xgenomics.com/docs/index-hopping-filter)
index-hopping-filter is a tool that filters index hopped reads from a set of demultiplexed samples.
The tool detects and removes likely index hopped reads from demultiplexed FASTQs, and in turn emits new, filtered,
FASTQs with similar file and directory layout as the inputs, suitable for use with cellranger count and cellranger vdj. 
 - [interop/1.1.10-foss-2019b-Python-3.7.4](https://github.com/Illumina/interop)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/interop/interop-1.1.10-foss-2019b-Python-3.7.4.eb)
The Illumina InterOp libraries are a set of common routines used for reading InterOp metric files produced by Illumina sequencers including NextSeq 1k/2k. These libraries are backwards compatible and capable of supporting prior releases of the software, with one exception: GA systems have been excluded.
 - [intervene/0.6.4-foss-2019b-Python-3.7.4](https://intervene.readthedocs.io/en/latest/index.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/intervene/intervene-0.6.4-foss-2019b-Python-3.7.4.eb)
Intervene a tool for intersection and visualization of multiple genomic region sets
 - [intltool/0.51.0-GCCcore-9.3.0](https://freedesktop.org/wiki/Software/intltool/)
intltool is a set of tools to centralize translation of
 many different file formats using GNU gettext-compatible PO files.
 - [itpp/4.3.1-foss-2019b](https://sourceforge.net/projects/itpp/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/itpp/itpp-4.3.1-foss-2019b.eb)
IT++ is a C++ library of mathematical, signal processing and communication
 classes and functions. Its main use is in simulation of communication systems and for
 performing research in the area of communications.
 - [jax/0.3.23-foss-2021b-CUDA-11.4.1](https://pypi.python.org/pypi/jax)
Composable transformations of Python+NumPy programs:
differentiate, vectorize, JIT to GPU/TPU, and more
 - [jbigkit/2.1-GCCcore-10.3.0](https://www.cl.cam.ac.uk/~mgk25/jbigkit/)
JBIG-KIT is a software implementation of the JBIG1 data
 compression standard (ITU-T T.82), which was designed for bi-level image
 data, such as scanned documents.
 - [jemalloc/5.3.0-GCCcore-12.2.0](http://jemalloc.net)
jemalloc is a general purpose malloc(3) implementation that emphasizes fragmentation avoidance and
 scalable concurrency support.
 - [json-c/0.16-GCCcore-12.2.0](https://github.com/json-c/json-c)
JSON-C implements a reference counting object model that allows you to easily construct JSON objects
 in C, output them as JSON formatted strings and parse JSON formatted strings back into the C representation of JSON
objects.
 - [jupyter-server/2.7.0-GCCcore-12.2.0](https://jupyter.org/)
The Jupyter Server provides the backend (i.e. the core services, APIs, and REST
endpoints) for Jupyter web applications like Jupyter notebook, JupyterLab, and
Voila.
 - [jupyter-server-proxy/3.2.2-GCCcore-12.2.0](https://github.com/jupyterhub/jupyter-server-proxy)
Jupyter Server Proxy lets you run arbitrary external processes
(such as RStudio, Shiny Server, Syncthing, PostgreSQL, Code Server, etc)
alongside your notebook server and provide authenticated web access to them    
using a path like /rstudio next to others like /lab. Alongside the python
package that provides the main functionality, the JupyterLab extension
(@jupyterlab/server-proxy) provides buttons in the JupyterLab launcher window
to get to RStudio for example.
 - [kallisto/0.48.0-foss-2020b](https://pachterlab.github.io/kallisto/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/k/kallisto/kallisto-0.48.0-foss-2020b.eb)
kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally
 of target sequences using high-throughput sequencing reads.
 - [kineto/0.4.0-GCC-11.3.0](https://github.com/pytorch/kineto)
A CPU+GPU Profiling library that provides access to timeline traces and hardware performance counters
 - [king/2.2.5](https://people.virginia.edu/~wc9c/KING/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/k/king/king-2.2.5.eb)
KING is a toolset that makes use of high-throughput 
SNP data typically seen in a genome-wide association study (GWAS) or 
a sequencing project. Applications of KING include family relationship 
inference and pedigree error checking, quality control, population 
substructure identification, forensics, gene mapping, etc.
 - [leidenalg/0.10.1-foss-2022b](https://github.com/vtraag/leidenalg)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/leidenalg/leidenalg-0.10.1-foss-2022b.eb)
Implementation of the Leiden algorithm for various quality
functions to be used with igraph in Python.
 - [lftp/4.9.1-GCCcore-8.3.0](http://lftp.yar.ru)
LFTP is a sophisticated ftp/http client, and a file transfer program supporting
  a number of network protocols. Like BASH, it has job control and uses the readline library for
  input. It has bookmarks, a built-in mirror command, and can transfer several files in parallel.
  It was designed with reliability in mind.
 - [libBigWig/0.4.6-GCCcore-11.2.0](https://github.com/dpryan79/libBigWig)
A C library for handling bigWig files
 - [libGLU/9.0.2-GCCcore-12.2.0](https://mesa.freedesktop.org/archive/glu/)
The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. 
 - [libXaw3d/1.6.3-GCCcore-8.3.0](http://www.freedesktop.org/wiki/Software/xlibs)
X11 client-side library
 - [libaec/1.0.6-GCCcore-11.3.0](https://gitlab.dkrz.de/k202009/libaec)
Libaec provides fast lossless compression of 1 up to 32 bit wide signed or unsigned integers
(samples). The library achieves best results for low entropy data as often encountered in space imaging
instrument data or numerical model output from weather or climate simulations. While floating point representations
are not directly supported, they can also be efficiently coded by grouping exponents and mantissa.
 - [libaio/0.3.113-GCCcore-12.2.0](https://pagure.io/libaio)
Asynchronous input/output library that uses the kernels native interface.
 - [libarchive/3.6.1-GCCcore-12.2.0](https://www.libarchive.org/)
Multi-format archive and compression library

 - [libasound/1.2.2-GCCcore-8.3.0]()
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libasound/libasound-1.2.2-GCCcore-8.3.0.eb)
The libnl suite is a collection of libraries providing APIs
 to netlink protocol based Linux kernel interfaces.
 - [libcerf/2.3-GCCcore-12.2.0](https://jugit.fz-juelich.de/mlz/libcerf)
libcerf is a self-contained numeric library that provides an efficient and
 accurate implementation of complex error functions, along with Dawson,
 Faddeeva, and Voigt functions.

 - [libcroco/0.6.13-GCCcore-8.3.0](https://github.com/GNOME/libcroco)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libcroco/libcroco-0.6.13-GCCcore-8.3.0.eb)
Libcroco is a standalone css2 parsing and manipulation library.
 - [libdeflate/1.15-GCCcore-12.2.0](https://github.com/ebiggers/libdeflate)
Heavily optimized library for DEFLATE/zlib/gzip compression and decompression.
 - [libdrm/2.4.114-GCCcore-12.2.0](https://dri.freedesktop.org)
Direct Rendering Manager runtime library.
 - [libedit/20191231-GCC-8.3.0](http://thrysoee.dk/editline/)
This BSD-style licensed command line editor library provides generic line editing,
history, and tokenization functions, similar to those found in GNU Readline.
 - [libepoxy/1.5.10-GCCcore-12.2.0](https://github.com/anholt/libepoxy)
Epoxy is a library for handling OpenGL function pointer management for you
 - [libev/4.33-GCC-11.2.0](http://software.schmorp.de/pkg/libev.html)
A full-featured and high-performance (see benchmark) 
event loop that is loosely modelled after libevent, but without its 
limitations and bugs. It is used in GNU Virtual Private Ethernet, 
rxvt-unicode, auditd, the Deliantra MORPG Server and Client, and many 
other programs.
 - [libevent/2.1.12-GCCcore-11.3.0](https://libevent.org/)
The libevent API provides a mechanism to execute a callback function when
 a specific event occurs on a file descriptor or after a timeout has been
 reached.  Furthermore, libevent also support callbacks due to signals or
 regular timeouts.

 - [libfabric/1.16.1-GCCcore-12.2.0](https://ofiwg.github.io/libfabric/)
Libfabric is a core component of OFI. It is the library that defines and exports
the user-space API of OFI, and is typically the only software that applications
deal with directly. It works in conjunction with provider libraries, which are
often integrated directly into libfabric.

 - [libffi/3.4.4-GCCcore-12.2.0](https://sourceware.org/libffi/)
The libffi library provides a portable, high level programming interface to
 various calling conventions. This allows a programmer to call any function
 specified by a call interface description at run-time.
 - [libfilezilla/0.40.0-GCC-11.2.0](https://lib.filezilla-project.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libfilezilla/libfilezilla-0.40.0-GCC-11.2.0.eb)
libfilezilla is a small and modern C++ library, offering some basic functionality to build high-performing, platform-independent programs.
 - [libgcrypt/1.9.3-GCCcore-11.2.0](https://gnupg.org/related_software/libgcrypt/index.html)
Libgcrypt is a general purpose cryptographic library originally based on code from GnuPG
 - [libgd/2.3.3-GCCcore-11.2.0](https://libgd.github.io)
GD is an open source code library for the dynamic creation of images by programmers.
 - [libgeotiff/1.7.1-GCCcore-12.2.0](https://directory.fsf.org/wiki/Libgeotiff)
Library for reading and writing coordinate system information from/to GeoTIFF files
 - [libgit2/1.5.0-GCCcore-12.2.0](https://libgit2.org/)
libgit2 is a portable, pure C implementation of the Git core methods provided as a re-entrant
linkable library with a solid API, allowing you to write native speed custom Git applications in any language
which supports C bindings.
 - [libglvnd/1.6.0-GCCcore-12.2.0](https://gitlab.freedesktop.org/glvnd/libglvnd)
libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API calls between multiple vendors.
 - [libgpg-error/1.42-GCCcore-11.2.0](https://gnupg.org/related_software/libgpg-error/index.html)
Libgpg-error is a small library that defines common error values for all GnuPG components.
 - [libgtextutils/0.7-GCCcore-7.3.0](http://hannonlab.cshl.edu/fastx_toolkit/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libgtextutils/libgtextutils-0.7-GCCcore-7.3.0.eb)
ligtextutils is a dependency of fastx-toolkit and is provided
 via the same upstream
 - [libharu/2.3.0-GCCcore-7.3.0](http://libharu.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libharu/libharu-2.3.0-GCCcore-7.3.0.eb)
libHaru is a free, cross platform, open source library for generating PDF files.
 - [libiconv/1.17-GCCcore-12.2.0](https://www.gnu.org/software/libiconv)
Libiconv converts from one character encoding to another through Unicode conversion
 - [libidn/1.38-GCCcore-11.2.0](http://www.gnu.org/software/libidn)
GNU Libidn is a fully documented implementation of the Stringprep, Punycode and IDNA specifications.
Libidn's purpose is to encode and decode internationalized domain names.
 - [libidn2/2.3.2-GCCcore-11.2.0](http://www.gnu.org/software/libidn2)
Libidn2 implements the revised algorithm for internationalized domain names called IDNA2008/TR46.
 - [libjpeg-turbo/2.1.4-GCCcore-12.2.0](https://sourceforge.net/projects/libjpeg-turbo/)
libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to
 accelerate baseline JPEG compression and decompression. libjpeg is a library
 that implements JPEG image encoding, decoding and transcoding.

 - [libleidenalg/0.11.1-foss-2022b](https://github.com/vtraag/libleidenalg)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libleidenalg/libleidenalg-0.11.1-foss-2022b.eb)
Implements the Leiden algorithm in C++
 - [libmaus2/2.0.611-foss-2018b](https://github.com/gt1/libmaus2)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libmaus2/libmaus2-2.0.611-foss-2018b.eb)
libmaus2 is a collection of data structures and algorithms.
 - [libmicrohttpd/0.9.73-GCCcore-10.2.0](https://www.gnu.org/software/libmicrohttpd/)
GNU libmicrohttpd is a small C library that is supposed to make it easy to run
 an HTTP server as part of another application.

 - [libogg/1.3.5-GCCcore-12.2.0](https://xiph.org/ogg/)
Ogg is a multimedia container format, and the native file and stream format for the Xiph.org
multimedia codecs.
 - [libopus/1.3.1-GCCcore-12.2.0](https://www.opus-codec.org/)
Opus is a totally open, royalty-free, highly versatile audio codec. Opus is unmatched for interactive
 speech and music transmission over the Internet, but is also intended for storage and streaming applications. It is
 standardized by the Internet Engineering Task Force (IETF) as RFC 6716 which incorporated technology from Skype’s
 SILK codec and Xiph.Org’s CELT codec.
 - [libpciaccess/0.17-GCCcore-12.2.0](https://cgit.freedesktop.org/xorg/lib/libpciaccess/)
Generic PCI access library.
 - [libpll/0.3.2-GCCcore-7.3.0](https://github.com/xflouris/libpll/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libpll/libpll-0.3.2-GCCcore-7.3.0.eb)
libpll is a versatile high-performance software library for phylogenetic analysis.
 - [libpng/1.6.38-GCCcore-12.2.0](http://www.libpng.org/pub/png/libpng.html)
libpng is the official PNG reference library
 - [libpsl/0.21.1-GCCcore-11.2.0](https://rockdaboot.github.io/libpsl)
C library for the Public Suffix List
 - [libpthread-stubs/0.4-GCCcore-8.3.0](http://xcb.freedesktop.org/)
The X protocol C-language Binding (XCB) is a replacement for Xlib featuring
 a small footprint, latency hiding, direct access to the protocol, improved
 threading support, and extensibility.

 - [libreadline/8.2-GCCcore-12.2.0](https://tiswww.case.edu/php/chet/readline/rltop.html)
The GNU Readline library provides a set of functions for use by applications
 that allow users to edit command lines as they are typed in. Both Emacs and
 vi editing modes are available. The Readline library includes additional
 functions to maintain a list of previously-entered command lines, to recall
 and perhaps reedit those lines, and perform csh-like history expansion on
 previous commands.

 - [librsvg/2.56.0-GCCcore-12.2.0](https://wiki.gnome.org/Projects/LibRsvg)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/librsvg/librsvg-2.56.0-GCCcore-12.2.0.eb)
Librsvg is a library to render SVG files using cairo.
 - [libsigc++/2.10.2-GCCcore-8.3.0](https://libsigcplusplus.github.io/libsigcplusplus/)
The libsigc++ package implements a typesafe callback system for standard C++.
 - [libsndfile/1.2.0-GCCcore-12.2.0](http://www.mega-nerd.com/libsndfile)
Libsndfile is a C library for reading and writing files containing sampled sound
 (such as MS Windows WAV and the Apple/SGI AIFF format) through one standard library interface.
 - [libsodium/1.0.18-GCCcore-11.2.0](https://doc.libsodium.org/)
Sodium is a modern, easy-to-use software library for encryption, decryption,
 signatures, password hashing and more.

 - [libtasn1/4.18.0-GCCcore-11.2.0](https://www.gnu.org/software/libtasn1/)
Libtasn1 is the ASN.1 library used by GnuTLS, GNU Shishi and
 some other packages. It was written by Fabio Fiorina, and has been shipped as
 part of GnuTLS for some time but is now a proper GNU package.
 - [libtirpc/1.3.3-GCCcore-12.2.0](https://sourceforge.net/projects/libtirpc/)
Libtirpc is a port of Suns Transport-Independent RPC library to Linux.
 - [libtool/2.4.7](https://www.gnu.org/software/libtool)
GNU libtool is a generic library support script. Libtool hides the complexity
 of using shared libraries behind a consistent, portable interface.

 - [libunistring/1.0-GCCcore-11.2.0](https://www.gnu.org/software/libunistring/)
This library provides functions for manipulating Unicode strings and for
 manipulating C strings according to the Unicode standard.
 - [libunwind/1.6.2-GCCcore-12.2.0](https://www.nongnu.org/libunwind/)
The primary goal of libunwind is to define a portable and efficient C programming interface
 (API) to determine the call-chain of a program. The API additionally provides the means to manipulate the
 preserved (callee-saved) state of each call-frame and to resume execution at any point in the call-chain
 (non-local goto). The API supports both local (same-process) and remote (across-process) operation.
 As such, the API is useful in a number of applications
 - [libuv/1.41.0-GCCcore-10.2.0](https://libuv.org)
libuv is a multi-platform support library with a focus on asynchronous I/O.
 - [libvorbis/1.3.7-GCCcore-11.2.0](https://xiph.org/vorbis/)
Ogg Vorbis is a fully open, non-proprietary, patent-and-royalty-free, general-purpose compressed
audio format
 - [libwebp/1.3.0-GCCcore-12.2.0](https://developers.google.com/speed/webp/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libwebp/libwebp-1.3.0-GCCcore-12.2.0.eb)
WebP is a modern image format that provides superior
lossless and lossy compression for images on the web. Using WebP,
webmasters and web developers can create smaller, richer images that
make the web faster.
 - [libwpe/1.13.3-GCCcore-11.2.0](https://webkit.org/wpe)
WPE is the reference WebKit port for embedded and 
low-consumption computer devices. It has been designed from the 
ground-up with performance, small footprint, accelerated content 
rendering, and simplicity of deployment in mind, bringing the 
excellence of the WebKit engine to countless platforms and target devices.
 - [libxml++/2.40.1-GCCcore-8.3.0](http://libxmlplusplus.sourceforge.net)
libxml++ is a C++ wrapper for the libxml XML parser library.
 - [libxml2/2.10.3-GCCcore-12.2.0](http://xmlsoft.org/)
Libxml2 is the XML C parser and toolchain developed for the Gnome project
 (but usable outside of the Gnome platform).

 - [libxslt/1.1.37-GCCcore-12.2.0](http://xmlsoft.org/)
Libxslt is the XSLT C library developed for the GNOME project
 (but usable outside of the Gnome platform).
 - [libyaml/0.2.5-GCCcore-10.2.0](https://pyyaml.org/wiki/LibYAML)
LibYAML is a YAML parser and emitter written in C.
 - [loompy/3.0.7-foss-2022b](https://loompy.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/loompy/loompy-3.0.7-foss-2022b.eb)
Python implementation of the Loom file format, an efficient file format for large omics datasets
 - [lpsolve/5.5.2.11-GCC-10.2.0](https://sourceforge.net/projects/lpsolve/)
Mixed Integer Linear Programming (MILP) solver
 - [lumpy/0.2.13-foss-2018b](https://github.com/pezmaster31/bamtools)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/lumpy/lumpy-0.2.13-foss-2018b.eb)
A probabilistic framework for structural variant discovery.
 - [lxml/4.9.2-GCCcore-12.2.0](https://lxml.de/)
The lxml XML toolkit is a Pythonic binding for the C libraries libxml2 and libxslt.
 - [lz4/1.9.4-GCCcore-12.2.0](https://lz4.github.io/lz4/)
LZ4 is lossless compression algorithm, providing compression speed at 400 MB/s per core.
 It features an extremely fast decoder, with speed in multiple GB/s per core.
 - [magma/2.7.1-foss-2022b-CUDA-12.0.0](https://icl.cs.utk.edu/magma/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/magma/magma-2.7.1-foss-2022b-CUDA-12.0.0.eb)
The MAGMA project aims to develop a dense linear algebra library similar to
 LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems.
 - [make/4.3-GCCcore-11.3.0](https://www.gnu.org/software/make/make.html)
GNU version of make utility
 - [makeinfo/6.8-GCCcore-11.2.0](https://www.gnu.org/software/texinfo/)
makeinfo is part of the Texinfo project, the official documentation format of the GNU project.
 - [manta/1.6.0](https://github.com/Illumina/manta)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/manta/manta-1.6.0.eb)
Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads.
It is optimized for analysis of germline variation in small sets of individuals and 
somatic variation in tumor/normal sample pairs. Manta discovers, assembles and 
scores large-scale SVs, medium-sized indels and large insertions within a 
single efficient workflow. 

 - [matplotlib/3.7.0-gfbf-2022b](https://matplotlib.org)
matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
 hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
 and ipython shell, web application servers, and six graphical user interface toolkits.
 - [maturin/1.1.0-GCCcore-12.2.0](https://github.com/pyo3/maturin)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/maturin/maturin-1.1.0-GCCcore-12.2.0.eb)
This project is meant as a zero configuration
replacement for setuptools-rust and milksnake. It supports building
wheels for python 3.5+ on windows, linux, mac and freebsd, can upload
them to pypi and has basic pypy and graalpy support.
 - [medaka/1.2.3-foss-2019b-Python-3.7.4](https://github.com/nanoporetech/medaka)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/medaka/medaka-1.2.3-foss-2019b-Python-3.7.4.eb)
medaka is a tool to create a consensus sequence from nanopore sequencing data.
 - [meson-python/0.11.0-GCCcore-12.2.0](https://github.com/mesonbuild/meson-python)
Python build backend (PEP 517) for Meson projects
 - [minimap2/2.26-GCCcore-12.2.0](https://github.com/lh3/minimap2)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/minimap2/minimap2-2.26-GCCcore-12.2.0.eb)
Minimap2 is a fast sequence mapping and alignment
program that can find overlaps between long noisy reads, or map long
reads or their assemblies to a reference genome optionally with detailed
alignment (i.e. CIGAR). At present, it works efficiently with query
sequences from a few kilobases to ~100 megabases in length at an error
rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited
test data sets, minimap2 is over 20 times faster than most other
long-read aligners. It will replace BWA-MEM for long reads and contig
alignment.
 - [monocle3/0.2.2-foss-2019b-R-4.0.2](https://cole-trapnell-lab.github.io/monocle3)
Single-cell transcriptome sequencing (sc-RNA-seq) experiments allow us to discover new cell types and help us understand how they arise in development. The Monocle 3 package provides a toolkit for analyzing single-cell gene expression experiments.
 - [monolix/2019R2](http://lixoft.com/products/monolix)
Monolix performs non-linear mixed effects modeling (NLME) for pharmacometrics.
 - [motif/2.3.8-GCCcore-11.2.0](https://motif.ics.com/)
Motif refers to both a graphical user interface (GUI) specification and the widget toolkit for building
 applications that follow that specification under the X Window System on Unix and other POSIX-compliant systems.
 It was the standard toolkit for the Common Desktop Environment and thus for Unix.
 - [mpi4py/3.1.4-gompi-2022b](https://github.com/mpi4py/mpi4py)
MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for
 the Python programming language, allowing any Python program to exploit multiple processors.
 - [nanofilt/2.5.0-foss-2018b-Python-3.6.6](https://github.com/wdecoster/nanofilt)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/nanofilt/nanofilt-2.5.0-foss-2018b-Python-3.6.6.eb)
Filtering and trimming of long read sequencing data.
 - [nanopolish/0.11.1-foss-2018b-Python-2.7.15](https://github.com/jts/nanopolish)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/nanopolish/nanopolish-0.11.1-foss-2018b-Python-2.7.15.eb)
Software package for signal-level analysis of Oxford Nanopore sequencing data.
 - [ncbi-vdb/3.0.0-gompi-2021b](https://github.com/ncbi/ncbi-vdb)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/ncbi-vdb/ncbi-vdb-3.0.0-gompi-2021b.eb)
The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for
 using data in the INSDC Sequence Read Archives.
 - [ncdf4/1.17-foss-2019b](https://cran.r-project.org/web/packages/ncdf4)
ncdf4: Interface to Unidata netCDF (version 4 or earlier) format data files
 - [ncdu/1.15.1-GCCcore-8.3.0](https://dev.yorhel.nl/ncdu)
Ncdu is a disk usage analyzer with an ncurses interface. It is designed to find space hogs on a remote server where you don't have an entire graphical setup available, but it is a useful tool even on regular desktop systems.
 - [ncurses/6.3-GCCcore-11.3.0](https://www.gnu.org/software/ncurses/)
The Ncurses (new curses) library is a free software emulation of curses in
 System V Release 4.0, and more. It uses Terminfo format, supports pads and
 color and multiple highlights and forms characters and function-key mapping,
 and has all the other SYSV-curses enhancements over BSD Curses.

 - [netCDF/4.9.0-gompi-2022b](https://www.unidata.ucar.edu/software/netcdf/)
NetCDF (network Common Data Form) is a set of software libraries
 and machine-independent data formats that support the creation, access, and sharing of array-oriented
 scientific data.
 - [nettle/3.8.1-GCCcore-12.2.0](https://www.lysator.liu.se/~nisse/nettle/)
Nettle is a cryptographic library that is designed to fit easily
 in more or less any context: In crypto toolkits for object-oriented
 languages (C++, Python, Pike, ...), in applications like LSH or GNUPG,
 or even in kernel space.
 - [networkx/3.0-gfbf-2022b](https://pypi.python.org/pypi/networkx)
NetworkX is a Python package for the creation, manipulation,
and study of the structure, dynamics, and functions of complex networks.
 - [nextflow/23.04.0](https://github.com/nextflow-io/nextflow)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/nextflow/nextflow-23.04.0.eb)
Nextflow is a bioinformatics workflow manager that enables the development
 of portable and reproducible workflows. It supports deploying workflows on a variety of
 execution platforms including local, HPC schedulers, AWS Batch, Google Genomics Pipelines,
 and Kubernetes.
 - [nghttp2/1.48.0-GCC-11.2.0](https://github.com/nghttp2/nghttp2)
This is an implementation of the Hypertext Transfer Protocol version 2 in C.

The framing layer of HTTP/2 is implemented as a reusable C library. 
On top of that, we have implemented an HTTP/2 client, server and proxy. 
We have also developed load test and benchmarking tools for HTTP/2.

An HPACK encoder and decoder are available as a public API.
 - [nghttp3/0.6.0-GCCcore-11.2.0](https://github.com/ngtcp2/nghttp3)
nghttp3 is an implementation of RFC 9114 HTTP/3 
mapping over QUIC and RFC 9204 QPACK in C.
It does not depend on any particular QUIC transport implementation.
 - [ngtcp2/0.7.0-GCC-11.2.0](https://github.com/ngtcp2/ngtcp2)
'Call it TCP/2. One More Time.'

ngtcp2 project is an effort to implement RFC9000 QUIC protocol.
 - [nlohmann_json/3.11.2-GCCcore-12.2.0](https://github.com/nlohmann/json)
JSON for Modern C++
 - [nodejs/18.12.1-GCCcore-12.2.0](https://nodejs.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/nodejs/nodejs-18.12.1-GCCcore-12.2.0.eb)
Node.js is a platform built on Chrome's JavaScript runtime
 for easily building fast, scalable network applications. Node.js uses an
 event-driven, non-blocking I/O model that makes it lightweight and efficient,
 perfect for data-intensive real-time applications that run across distributed devices.
 - [nsync/1.24.0-GCCcore-10.2.0](https://github.com/google/nsync)
nsync is a C library that exports various synchronization primitives, such as mutexes
 - [nullarbor/2.0.20191013](https://github.com/tseemann/nullarbor)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/nullarbor/nullarbor-2.0.20191013.eb)
Pipeline to generate complete public health microbiology reports from sequenced isolates

 - [numactl/2.0.16-GCCcore-12.2.0](https://github.com/numactl/numactl)
The numactl program allows you to run your application program on specific
 cpu's and memory nodes. It does this by supplying a NUMA memory policy to
 the operating system before running your program. The libnuma library provides
 convenient ways for you to add NUMA memory policies into your own program.

 - [numba/0.57.0rc1-foss-2022b](https://numba.pydata.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/numba/numba-0.57.0rc1-foss-2022b.eb)
Numba is an Open Source NumPy-aware optimizing compiler for
Python sponsored by Continuum Analytics, Inc. It uses the remarkable LLVM
compiler infrastructure to compile Python syntax to machine code.
 - [numexpr/2.7.1-foss-2019b-Python-2.7.16](https://numexpr.readthedocs.io/en/latest/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/numexpr/numexpr-2.7.1-foss-2019b-Python-2.7.16.eb)
The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.
 It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into
 code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a
 compiler at runtime.
 - [oncosnpseq/2.01](https://sites.google.com/site/oncosnpseq)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/o/oncosnpseq/oncosnpseq-2.01.eb)
OncoSNP-SEQ is an analytical tool for characterising copy number alterations
 and loss-of-heterozygosity (LOH) events in cancer samples from whole genome sequencing data.
 - [ont-guppy-cpu/2.3.7](https://nanoporetech.com/products/minit)
Guppy is a production basecaller provided by Oxford Nanopore,
 and uses a command-line interface.
 - [openpyxl/3.0.9-GCCcore-11.2.0](https://openpyxl.readthedocs.io)
A Python library to read/write Excel 2010 xlsx/xlsm files
 - [p11-kit/0.24.1-GCCcore-11.2.0](https://p11-glue.freedesktop.org/p11-kit.html)
Provides a way to load and enumerate PKCS#11 modules.
 Provides a standard configuration setup for installing
 PKCS#11 modules in such a way that they're discoverable.
 Also solves problems with coordinating the use of PKCS#11
 by different components or libraries living in the same process.
 - [packmol/20.2.2-GCC-10.2.0](http://m3g.iqm.unicamp.br/packmol)
Packing Optimization for Molecular Dynamics Simulations
 - [parallel/20210322-GCCcore-10.2.0](https://savannah.gnu.org/projects/parallel/)
parallel: Build and execute shell commands in parallel
 - [parallel-fastq-dump/0.6.7-GCCcore-11.2.0](https://github.com/rvalieris/parallel-fastq-dump)
parallel fastq-dump wrapper
 - [parasail/2.4-foss-2018b](https://github.com/jeffdaily/parasail)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/parasail/parasail-2.4-foss-2018b.eb)
parasail is a SIMD C (C99) library containing implementations
 of the Smith-Waterman (local), Needleman-Wunsch (global), and semi-global
 pairwise sequence alignment algorithms. 
 - [patchelf/0.18.0-GCCcore-12.2.0](https://github.com/NixOS/patchelf)
PatchELF is a small utility to modify the dynamic linker and RPATH of ELF executables.
 - [pbcopper/1.3.0-foss-2019b](https://github.com/PacificBiosciences/pbcopper)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pbcopper/pbcopper-1.3.0-foss-2019b.eb)
The pbcopper library provides a suite of data structures, algorithms,
 and utilities for C++ applications.
 - [pblat/2.5.1-GCC-11.2.0](http://icebert.github.io/pblat/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pblat/pblat-2.5.1-GCC-11.2.0.eb)
Parallel blat based on Jim Kent's blat
 - [petsc4py/3.15.0-foss-2021a](https://bitbucket.org/petsc/petsc4py)
petsc4py are Python bindings for PETSc, the Portable, Extensible Toolchain for Scientific Computation.
 - [philosopher/3.3.11](https://github.com/Nesvilab/philosopher/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/philosopher/philosopher-3.3.11.eb)
Philosopher provides easy access to third-party tools and custom algorithms
 allowing users to develop proteomics analysis, from Peptide Spectrum Matching to annotated
 protein reports. Philosopher is also tuned for Open Search analysis, providing a modified
 version of the prophets for peptide validation and protein inference. To this date, Philosopher
 is the only proteomics toolkit that allows you to process and analyze close and
 open search results.
 - [picard/2.25.1-Java-11](https://broadinstitute.github.io/picard/)
A set of tools (in Java) for working with next generation sequencing data in the BAM format.
 - [pigz/2.6-GCCcore-10.2.0](https://zlib.net/pigz/)
pigz, which stands for parallel implementation of gzip, is a fully
 functional replacement for gzip that exploits multiple processors and multiple
 cores to the hilt when compressing data. pigz was written by Mark Adler, and
 uses the zlib and pthread libraries.

 - [pipdeptree/0.13.2-foss-2019b-Python-3.7.4](https://github.com/naiquevin/pipdeptree)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pipdeptree/pipdeptree-0.13.2-foss-2019b-Python-3.7.4.eb)
pipdeptree is a command line utility for displaying the installed python packages in form of a dependency tree. It works for packages installed globally on a machine as well as in a virtualenv.
 - [pixman/0.42.2-GCCcore-12.2.0](http://www.pixman.org/)
Pixman is a low-level software library for pixel manipulation, providing
 features such as image compositing and trapezoid rasterization. Important
 users of pixman are the cairo graphics library and the X server.

 - [pkg-config/0.29.2-GCCcore-7.3.0](http://www.freedesktop.org/wiki/Software/pkg-config/)
pkg-config is a helper tool used when compiling applications and libraries.
 It helps you insert the correct compiler options on the command line so an
 application can use gcc -o test test.c `pkg-config --libs --cflags glib-2.0`
 for instance, rather than hard-coding values on where to find glib (or other
 libraries).

 - [pkgconf/1.9.3-GCCcore-12.2.0](https://github.com/pkgconf/pkgconf)
pkgconf is a program which helps to configure compiler and linker flags for development libraries.
 It is similar to pkg-config from freedesktop.org.
 - [pkgconfig/1.5.5-GCCcore-11.2.0-python](https://github.com/matze/pkgconfig)
pkgconfig is a Python module to interface with the pkg-config command line tool
 - [plink/1.9-20200616](https://www.cog-genomics.org/plink/1.9)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/plink/plink-1.9-20200616.eb)
Whole-genome association analysis toolset
 - [plinkliftover/0.3.0-foss-2022b](https://github.com/milescsmith/plinkliftover)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/plinkliftover/plinkliftover-0.3.0-foss-2022b.eb)
PLINKLiftOver is a utility enabling liftOver to work on genomics files from PLINK,
 allowing one to update the coordinates from one genome reference version to another.
 - [plotly.py/5.13.1-GCCcore-12.2.0](https://plot.ly/python)
An open-source, interactive graphing library for Python
 - [pocl/4.0-GCC-12.2.0](http://portablecl.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pocl/pocl-4.0-GCC-12.2.0.eb)
PoCL is a portable open source (MIT-licensed) implementation 
of the OpenCL standard (1.2 with some 2.0 features supported).
 - [poetry/1.0.9-GCCcore-9.3.0-Python-3.8.2](https://python-poetry.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/poetry/poetry-1.0.9-GCCcore-9.3.0-Python-3.8.2.eb)
Python packaging and dependency management made easy
 - [poppler/23.01.0-GCCcore-12.2.0](https://poppler.freedesktop.org)
Poppler is a PDF rendering library
 - [popscle/0.1-beta-foss-2019b](https://github.com/statgen/popscle)
A suite of
population scale analysis tools for single-cell genomics data including
implementation of Demuxlet / Freemuxlet methods and auxilary tools 
 - [pplacer/1.1.alpha19-foss-2018b](['https://matsen.fhcrc.org/pplacer/'])
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pplacer/pplacer-1.1.alpha19-foss-2018b.eb)
Pplacer places reads on a phylogenetic tree. guppy (Grand
 Unified Phylogenetic Placement Yanalyzer) yanalyzes them. rppr is a helpful
 tool for working with reference packages.
 - [prodigal/2.6.3-GCCcore-7.3.0](http://prodigal.ornl.gov/)
Prodigal (Prokaryotic Dynamic Programming Genefinding
   Algorithm) is a microbial (bacterial and archaeal) gene finding program
   developed at Oak Ridge National Laboratory and the University of
   Tennessee.
 - [prokka/1.14.5-gompi-2020b](https://www.vicbioinformatics.com/software.prokka.shtml)
Prokka is a software tool for the rapid annotation of prokaryotic genomes.
 - [protobuf/23.0-GCCcore-12.2.0](https://github.com/protocolbuffers/protobuf)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/protobuf/protobuf-23.0-GCCcore-12.2.0.eb)
Protocol Buffers (a.k.a., protobuf) are Google's 
language-neutral, platform-neutral, extensible mechanism for 
serializing structured data.
 - [protobuf-c/1.3.3-GCCcore-8.3.0](https://github.com/protobuf-c/protobuf-c)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/protobuf-c/protobuf-c-1.3.3-GCCcore-8.3.0.eb)
This is protobuf-c, a C implementation of the Google Protocol Buffers data serialization format
 - [protobuf-python/4.23.0-GCCcore-12.2.0](https://github.com/google/protobuf/)
Python Protocol Buffers runtime library.
 - [psipred/4.02-foss-2018b](http://bioinf.cs.ucl.ac.uk/psipred/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/psipred/psipred-4.02-foss-2018b.eb)
The PSIPRED Protein Sequence Analysis Workbench aggregates
 several UCL structure prediction methods into one location.
 - [pugixml/1.12.1-GCCcore-11.2.0](https://pugixml.org/)
pugixml is a light-weight C++ XML processing library
 - [py-cpuinfo/9.0.0-GCCcore-12.2.0](https://github.com/workhorsy/py-cpuinfo)
py-cpuinfo gets CPU info with pure Python.
 - [pyBigWig/0.3.22-foss-2022b](https://github.com/deeptools/pyBigWig)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pyBigWig/pyBigWig-0.3.22-foss-2022b.eb)
A python extension, written in C, for quick access to bigBed
 files and access to and creation of bigWig files.
 - [pyGenomeTracks/3.5-foss-2019b-Python-3.7.4](https://github.com/deeptools/pyGenomeTracks)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pyGenomeTracks/pyGenomeTracks-3.5-foss-2019b-Python-3.7.4.eb)
pyGenomeTracks aims to produce high-quality genome browser tracks that are highly customizable.
 - [pySCENIC/0.12.1-foss-2022b](https://github.com/aertslab/pySCENIC)
pySCENIC is a lightning-fast python implementation of the SCENIC
pipeline (Single-Cell rEgulatory Network Inference and Clustering) which enables
biologists to infer transcription factors, gene regulatory networks and
cell types from single-cell RNA-seq data.
 - [pyaml/21.10.1-GCC-11.2.0](https://github.com/mk-fg/pretty-yaml)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pyaml/pyaml-21.10.1-GCC-11.2.0.eb)
PyYAML-based python module to produce pretty and readable YAML-serialized data.
This module is for serialization only, see ruamel.yaml module for literate YAML
parsing (keeping track of comments, spacing, line/column numbers of values, etc).
 - [pybedtools/0.9.0-GCC-12.2.0](https://daler.github.io/pybedtools)
pybedtools wraps and extends BEDTools and offers feature-level manipulations from within Python.
 - [pybind11/2.10.3-GCCcore-12.2.0](https://pybind11.readthedocs.io)
pybind11 is a lightweight header-only library that exposes C++ types in Python and vice versa,
 mainly to create Python bindings of existing C++ code.
 - [pycisTarget/1.0.2-foss-2022b](https://pycistarget.readthedocs.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pycisTarget/pycisTarget-1.0.2-foss-2022b.eb)
pycistarget is a python module to perform motif enrichment analysis in sets of
 regions with different tools and identify high confidence TF cistromes.
 - [pyclone/0.13.1-foss-2019b-Python-2.7.16](https://github.com/Roth-Lab/pyclone)
PyClone is a Bayesian clustering method for grouping sets of deeply sequenced
 somatic mutations into putative clonal clusters while estimating their cellular prevalences and
 accounting for allelic imbalances introduced by segmental copy-number changes and normal-cell
 contamination.
 - [pyclone-vi/0.1.0-foss-2019b-Python-3.7.4](https://github.com/Roth-Lab/pyclone-vi)
PyClone is a Bayesian clustering method for grouping sets of deeply sequenced
 somatic mutations into putative clonal clusters while estimating their cellular prevalences and
 accounting for allelic imbalances introduced by segmental copy-number changes and normal-cell
 contamination.
 - [pydantic/1.10.2-GCCcore-11.2.0](https://github.com/samuelcolvin/pydantic)
Data validation and settings management using Python type hinting.
 - [pyfaidx/0.7.0-GCCcore-11.2.0](https://pypi.python.org/pypi/pyfaidx)
pyfaidx: efficient pythonic random access to fasta subsequences
 - [pyspoa/0.0.4-GCC-8.3.0-Python-3.7.4](https://github.com/nanoporetech/pyspoa)
Python bindings to spoa.
 - [pytest/7.1.3-GCCcore-11.2.0](https://pytest.org)
pytest: simple powerful testing with Python
 - [pytest-benchmark/3.4.1-GCCcore-10.2.0](https://github.com/ionelmc/pytest-benchmark)
A pytest fixture for benchmarking code.
 - [pytest-rerunfailures/12.0-GCCcore-12.2.0](https://github.com/pytest-dev/pytest-rerunfailures)
pytest plugin to re-run tests to eliminate flaky failures.
 - [pytest-shard/0.1.2-GCCcore-12.2.0](https://github.com/AdamGleave/pytest-shard)
pytest plugin to support parallelism across multiple machines.

Shards tests based on a hash of their test name enabling easy parallelism across machines,
suitable for a wide variety of continuous integration services.
Tests are split at the finest level of granularity, individual test cases,
enabling parallelism even if all of your tests are in a single file
(or even single parameterized test method).

 - [pytest-xdist/2.5.0-GCCcore-11.2.0](https://github.com/pytest-dev/pytest-xdist)
xdist: pytest distributed testing plugin

The pytest-xdist plugin extends pytest with some unique test execution modes:

    * test run parallelization: if you have multiple CPUs or hosts you
      can use those for a combined test run. This allows to speed up
      development or to use special resources of remote machines.

    * --looponfail: run your tests repeatedly in a subprocess. After
        each run pytest waits until a file in your project changes and
        then re-runs the previously failing tests. This is repeated
        until all tests pass after which again a full run is
        performed.

    * Multi-Platform coverage: you can specify different Python
      interpreters or different platforms and run tests in parallel on
      all of them.

Before running tests remotely, pytest efficiently “rsyncs” your
program source code to the remote place. All test results are reported
back and displayed to your local terminal. You may specify different
Python versions and interpreters.
 - [python-igraph/0.10.6-foss-2022b](https://igraph.org/python)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/python-igraph/python-igraph-0.10.6-foss-2022b.eb)
Python interface to the igraph high performance graph library, primarily aimed at complex network
 research and analysis.
 - [python-isal/0.11.1-GCCcore-11.2.0](https://github.com/pycompression/python-isal)
Faster zlib and gzip compatible compression and decompression
 by providing python bindings for the isa-l library.

 - [python-parasail/1.1.16-foss-2018b-Python-3.6.6](https://pypi.org/project/parasail/)
This package contains Python bindings for parasail. 
 - [qcat/1.0.7-foss-2018b-Python-3.6.6](https://github.com/nanoporetech/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/q/qcat/qcat-1.0.7-foss-2018b-Python-3.6.6.eb)
Qcat is Python command-line tool for demultiplexing Oxford
 Nanopore reads from FASTQ files.
 - [rMATS-turbo/4.1.2-foss-2021b](https://github.com/Xinglab/rmats-turbo)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/rMATS-turbo/rMATS-turbo-4.1.2-foss-2021b.eb)
rMATS turbo is the C/Cython version of rMATS (refer to http://rnaseq-mats.sourceforge.net).
 - [re2c/3.0-GCCcore-12.2.0](https://re2c.org)
re2c is a free and open-source lexer generator for C and C++. Its main goal is generating
fast lexers: at least as fast as their reasonably optimized hand-coded counterparts. Instead of using
traditional table-driven approach, re2c encodes the generated finite state automata directly in the form
of conditional jumps and comparisons.
 - [redis-py/4.6.0-foss-2022b](https://github.com/redis/redis-py)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/redis-py/redis-py-4.6.0-foss-2022b.eb)
The Python interface to the Redis key-value store.
 - [revbayes/1.0.11-foss-2018b](https://revbayes.github.io/)
RevBayes provides an interactive environment for statistical
 computation in phylogenetics. It is primarily intended for modeling,
 simulation, and Bayesian inference in evolutionary biology, particularly
 phylogenetics.
 - [rgdal/1.4-8-foss-2019b-R-4.0.2](http://rgdal.r-forge.r-project.org/)
Provides bindings to the 'Geospatial' Data Abstraction Library ('GDAL') (>= 1.11.4 and <= 2.5.0) and
 access to projection/transformation operations from the 'PROJ.4' library.
 - [rpy2/3.4.5-foss-2021b](https://rpy2.bitbucket.io/)
rpy2 is an interface to R running embedded in a Python process.
 - [rstudio/1.3.1093-foss-2019b-Java-11-R-4.0.2](https://www.rstudio.com/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/rstudio/rstudio-1.3.1093-foss-2019b-Java-11-R-4.0.2.eb)
This RStudio Server version.
RStudio is a set of integrated tools designed to help you be more productive with R.

 - [rstudio-server/1.2.5033-foss-2019b](https://github.com/rstudio/rstudio)
RStudio is an integrated development environment 
(IDE) for the R programming language.
 - [ruamel.yaml/0.17.21-GCCcore-11.2.0](https://sourceforge.net/projects/ruamel-yaml)
ruamel.yaml is a YAML 1.2 loader/dumper package for Python.
 - [samblaster/0.1.24-foss-2018b](https://github.com/GregoryFaust/samblaster)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/samblaster/samblaster-0.1.24-foss-2018b.eb)
samblaster is a fast and flexible program for marking
 duplicates in read-id grouped1 paired-end SAM files.
 - [savvy/2.0.1-GCC-10.2.0](https://github.com/statgen/savvy)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/savvy/savvy-2.0.1-GCC-10.2.0.eb)
Interface to various variant calling formats.
 - [sbt/1.7.2-Java-1.8](http://www.scala-sbt.org/)
sbt is a build tool for Scala, Java, and more.
 - [scVelo/0.2.4-foss-2020b](https://scvelo.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/scVelo/scVelo-0.2.4-foss-2020b.eb)
scVelo is a scalable toolkit for estimating and analyzing RNA velocities in single cells using
 dynamical modeling.
 - [scanpy/1.9.3-foss-2022b](https://scanpy.readthedocs.io/en/stable/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/scanpy/scanpy-1.9.3-foss-2022b.eb)
Scanpy is a scalable toolkit for analyzing single-cell gene expression data built
 jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference
 and differential expression testing. The Python-based implementation efficiently deals with
 datasets of more than one million cells.

 - [scenicplus/1.0.0-foss-2022b](https://scenicplus.readthedocs.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/scenicplus/scenicplus-1.0.0-foss-2022b.eb)
SCENIC+ is a python package to build enhancer driven gene regulatory networks
 (GRNs) using combined or separate single-cell gene expression (scRNA-seq) and single-cell
 chromatin accessibility (scATAC-seq) data.
 - [scikit-bio/0.5.7-foss-2021b](http://scikit-bio.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/scikit-bio/scikit-bio-0.5.7-foss-2021b.eb)
scikit-bio is an open-source, BSD-licensed Python 3 package providing data structures, algorithms
and educational resources for bioinformatics.
 - [scikit-build/0.17.2-GCCcore-12.2.0](https://scikit-build.readthedocs.io/en/latest)
Scikit-Build, or skbuild, is an improved build system generator
for CPython C/C++/Fortran/Cython extensions.
 - [scikit-image/0.21.0-foss-2022b](https://scikit-image.org/)
scikit-image is a collection of algorithms for image processing.
 - [scikit-learn/1.2.1-gfbf-2022b](https://scikit-learn.org/stable/index.html)
Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world,
building upon numpy, scipy, and matplotlib. As a machine-learning module,
it provides versatile tools for data mining and analysis in any field of science and engineering.
It strives to be simple and efficient, accessible to everybody, and reusable in various contexts.
 - [scikit-misc/0.1.4-foss-2021a](https://github.com/has2k1/scikit-misc)
Miscellaneous tools for data analysis and scientific computing
 - [scikit-optimize/0.9.0-foss-2021b](https://scikit-optimize.github.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/scikit-optimize/scikit-optimize-0.9.0-foss-2021b.eb)
Scikit-Optimize, or skopt, is a simple and efficient library to minimize (very) expensive
 and noisy black-box functions.
 - [scipy/1.4.1-foss-2019b-Python-3.7.4](https://www.scipy.org)
SciPy is a collection of mathematical algorithms and convenience
 functions built on the Numpy extension for Python.
 - [scrublet/0.2.3-foss-2022b](https://github.com/AllonKleinLab/scrublet)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/scrublet/scrublet-0.2.3-foss-2022b.eb)
Python code for identifying doublets in single-cell RNA-seq data
 - [scvi-tools/0.16.4-foss-2021a-CUDA-11.3.1](https://github.com/scverse/scvi-tools)
scvi-tools (single-cell variational inference tools) is a package for probabilistic modeling and
analysis of single-cell omics data, built on top of PyTorch and AnnData.
 - [seq2HLA/2.3-foss-2019b-Python-2.7.16](https://github.com/TRON-Bioinformatics/seq2HLA)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/seq2HLA/seq2HLA-2.3-foss-2019b-Python-2.7.16.eb)
In-silico method written in Python and R to determine HLA genotypes of a sample.
 seq2HLA takes standard RNA-Seq sequence reads in fastq format as input,
 uses a bowtie index comprising all HLA alleles and outputs the most likely
 HLA class I and class II genotypes (in 4 digit resolution),
 a p-value for each call, and the expression of each class. 
 - [seqtk/1.3-GCC-10.2.0](https://github.com/lh3/seqtk/)
Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format.
 It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip.
 - [seqtools/4.44.1-foss-2019b](http://www.sanger.ac.uk/science/tools/seqtools)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/seqtools/seqtools-4.44.1-foss-2019b.eb)
The SeqTools package contains three tools for visualising sequence alignments: Blixem, Dotter and Belvu.
 - [sequenza-utils/3.0.0-GCCcore-8.3.0-Python-3.7.4](https://sequenza-utils.readthedocs.io/)
Sequenza is a software for the estimation and quantification of purity/ploidy and copy number alteration in sequencing experiments of tumor samples. Sequenza-utils provide command lines programs to transform common NGS file format
 - [setuptools/64.0.3-GCCcore-12.2.0](https://pypi.org/project/setuptools)
Easily download, build, install, upgrade, and uninstall Python packages
 - [shrinkwrap/1.1.0-GCCcore-10.2.0](https://github.com/jonathonl/shrinkwrap)
A std::streambuf wrapper for compression formats.
 - [slepc4py/3.15.1-foss-2021a](https://bitbucket.org/slepc/slepc4py)
Python bindings for SLEPc, the Scalable Library for Eigenvalue Problem Computations.
 - [smallgenomeutilities/0.2.1-foss-2018b-Python-3.6.6](https://github.com/cbg-ethz/smallgenomeutilities)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/smallgenomeutilities/smallgenomeutilities-0.2.1-foss-2018b-Python-3.6.6.eb)
The smallgenomeutilities are a collection of scripts that is
 useful for dealing and manipulating NGS data of small viral genomes. They
 are written in Python 3 with a small number of dependencies.
 - [snakemake/7.18.2-foss-2021b](https://snakemake.readthedocs.io)
The Snakemake workflow management system is a tool to create reproducible and scalable data analyses.
 - [snappy/1.1.9-GCCcore-12.2.0](https://github.com/google/snappy)
Snappy is a compression/decompression library. It does not aim
for maximum compression, or compatibility with any other compression library;
instead, it aims for very high speeds and reasonable compression.
 - [snippy/4.6.0-foss-2019b-Perl-5.30.0](https://github.com/tseemann/snippy)
Snippy finds SNPs between a haploid reference genome and your NGS sequence
 reads. It will find both substitutions (snps) and insertions/deletions (indels).
Rapid haploid variant calling and core genome alignment.
 - [spams/2.6.5.4-foss-2021b](https://thoth.inrialpes.fr/people/mairal/spams/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/spams/spams-2.6.5.4-foss-2021b.eb)
SPAMS (SPArse Modeling Software) is an optimization toolbox for solving various
 sparse estimation problems.
 - [span-lite/0.10.3-GCC-11.2.0](http:///)
span lite is a single-file header-only library to provide a bounds-safe view for sequences
 of objects. The library provides a C++20-like span for use with C++98 and later. 
 - [spdlog/1.10.0-GCCcore-11.2.0](https://github.com/gabime/spdlog)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/spdlog/spdlog-1.10.0-GCCcore-11.2.0.eb)
Very fast, header-only/compiled, C++ logging library.
 - [spektral/1.1.0-foss-2021b-CUDA-11.4.1](https://github.com/danielegrattarola/spektral)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/spektral/spektral-1.1.0-foss-2021b-CUDA-11.4.1.eb)
Spektral is a Python library for graph deep learning. The main goal of this
 project is to provide a simple but flexible framework for creating graph neural networks (GNNs).
 - [spglib-python/1.16.0-foss-2020b](https://pypi.python.org/pypi/spglib)
Spglib for Python. Spglib is a library for finding and handling crystal symmetries written in C.
 - [spglib-python/1.16.0-foss-2020b](https://pypi.python.org/pypi/spglib)
Spglib for Python. Spglib is a library for finding and handling crystal symmetries written in C.
 - [spoa/4.0.0-GCC-8.3.0](https://github.com/rvaser/spoa)
Spoa (SIMD POA) is a c++ implementation of the partial order alignment (POA) algorithm
 which is used to generate consensus sequences
 - [stack/2.3.1]()

 - [starcode/1.4-GCC-11.2.0](https://github.com/gui11aume/starcode)
Starcode is a DNA sequence clustering software. Starcode clustering is based on all pairs
search within a specified Levenshtein distance (allowing insertions and deletions), followed by a clustering
algorithm: Message Passing, Spheres or Connected Components.
 - [stardist/0.8.3-foss-2021b-CUDA-11.4.1](https://github.com/stardist/stardist)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/stardist/stardist-0.8.3-foss-2021b-CUDA-11.4.1.eb)
Object Detection with Star-convex Shapes.
 - [statsmodels/0.14.0-gfbf-2022b](https://www.statsmodels.org/)
Statsmodels is a Python module that allows users to explore data, estimate statistical models,
and perform statistical tests.
 - [strelka/2.9.9-foss-2018b](https://github.com/Illumina/strelka)
Strelka2 is a fast and accurate small variant caller
 optimized for analysis of germline variation in small cohorts and
 somatic variation in tumor/normal sample pairs.
 - [svaba/1.1.0-GCC-10.2.0](https://github.com/walaj/svaba)
SvABA
 - [sympy/1.12-gfbf-2022b](https://sympy.org/)
SymPy is a Python library for symbolic mathematics. It aims to
 become a full-featured computer algebra system (CAS) while keeping the code as
 simple as possible in order to be comprehensible and easily extensible. SymPy
 is written entirely in Python and does not require any external libraries.
 - [sysbench/1.0.20-GCC-12.2.0](https://github.com/akopytov/sysbench)
sysbench is a scriptable multi-threaded benchmark tool based on LuaJIT.
 It is most frequently used for database benchmarks, but can also be used to create arbitrarily complex
 workloads that do not involve a database server.
 - [tabix/0.2.6-GCCcore-7.3.0](http://samtools.sourceforge.net)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/tabix/tabix-0.2.6-GCCcore-7.3.0.eb)
Generic indexer for TAB-delimited genome position files 
 - [tbb/2021.10.0-GCCcore-12.2.0](https://github.com/oneapi-src/oneTBB)
Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that
 take full advantage of multicore performance, that are portable, composable and have future-proof scalability.
 - [tbl2asn/20220427-linux64](https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/)
Tbl2asn is a command-line program that automates the creation of
 sequence records for submission to GenBank
 - [tcsh/6.20.00-GCCcore-7.3.0](http://www.tcsh.org)
Tcsh is an enhanced, but completely compatible version of the Berkeley UNIX C shell (csh). 
 It is a command language interpreter usable both as an interactive login shell and a shell script command 
 processor. It includes a command-line editor, programmable word completion, spelling correction, a history 
 mechanism, job control and a C-like syntax.
 - [tensorboard/2.8.0-foss-2021a](https://github.com/tensorflow/tensorboard)
TensorBoard is a suite of web applications for inspecting and understanding your TensorFlow runs
and graphs.
 - [terminator/1.91-GCCcore-8.3.0-Python-2.7.16](http://freedesktop.org/wiki/Software/intltool/)
Multiple terminals in one window. The goal of this project is to produce a useful tool for arranging terminals.
 - [terraphast/master-foss-2018b](https://github.com/xflouris/libpll/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/terraphast/terraphast-master-foss-2018b.eb)
libpll is a versatile high-performance software library for phylogenetic analysis.
 - [tesseract/5.3.0-GCCcore-11.2.0](https://github.com/tesseract-ocr/tesseract)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/tesseract/tesseract-5.3.0-GCCcore-11.2.0.eb)
Tesseract is an optical character recognition engine
 - [texinfo/6.7-GCCcore-8.3.0](https://www.gnu.org/software/texinfo/)
Texinfo is the official documentation format of the GNU project.
 - [texlive/20210324-GCC-11.2.0](https://tug.org)
TeX is a typesetting language. Instead of visually formatting your text, you enter your manuscript
 text intertwined with TeX commands in a plain text file. You then run TeX to produce formatted output, such as a
 PDF file. Thus, in contrast to standard word processors, your document is a separate file that does not pretend to
 be a representation of the final typeset output, and so can be easily edited and manipulated.
 - [thrift/0.13.0-foss-2018b](https://arrow.apache.org)
Thrift is a lightweight, language-independent software stack for point-to-point
 RPC implementation. Thrift provides clean abstractions and implementations for data transport, data
 serialization, and application level processing.
 - [tmux/3.2a-GCCcore-11.2.0](https://github.com/tmux/tmux/)
tmux is a terminal multiplexer: it enables a number of
terminals to be created, accessed, and controlled from a single screen. tmux
may be detached from a screen and continue running in the background, then
later reattached.
 - [torchvision/0.13.0-foss-2021b-CUDA-11.4.1](https://github.com/pytorch/vision)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/torchvision/torchvision-0.13.0-foss-2021b-CUDA-11.4.1.eb)
Datasets, Transforms and Models specific to Computer Vision
 - [tqdm/4.64.1-GCCcore-12.2.0](https://github.com/tqdm/tqdm)
A fast, extensible progress bar for Python and CLI
 - [typing-extensions/4.2.0-GCCcore-11.2.0](https://github.com/python/typing/blob/master/typing_extensions/README.rst)
Typing Extensions – Backported and Experimental Type Hints for Python
 - [umap-learn/0.5.3-foss-2022b](https://umap-learn.readthedocs.io/en/latest/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/u/umap-learn/umap-learn-0.5.3-foss-2022b.eb)
Uniform Manifold Approximation and Projection (UMAP) is a dimension reduction technique 
that can be used for visualisation similarly to t-SNE, but also for general non-linear 
dimension reduction.

 - [unixODBC/2.3.11-GCC-12.2.0](https://www.unixodbc.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/u/unixODBC/unixODBC-2.3.11-GCC-12.2.0.eb)
unixODBC provides a uniform interface between application and database driver
 - [utf8proc/2.8.0-GCCcore-12.2.0](https://github.com/JuliaStrings/utf8proc)
utf8proc is a small, clean C library that provides Unicode normalization, case-folding, 
and other operations for data in the UTF-8 encoding.
 - [util-linux/2.38.1-GCCcore-12.2.0](https://www.kernel.org/pub/linux/utils/util-linux)
Set of Linux utilities
 - [vcf2maf/1.6.19](https://github.com/mskcc/vcf2maf )
vcflib provides methods to manipulate and interpret sequence variation as it can be
 described by VCF. The Variant Call Format (VCF) is a flat-file, tab-delimited textual format intended
 to concisely describe reference-indexed genetic variations between individuals.
 - [vcflib/1.0.1-GCCcore-8.3.0](https://github.com/vcflib/vcflib)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/v/vcflib/vcflib-1.0.1-GCCcore-8.3.0.eb)
vcflib provides methods to manipulate and interpret sequence variation as it can be
 described by VCF. The Variant Call Format (VCF) is a flat-file, tab-delimited textual format intended
 to concisely describe reference-indexed genetic variations between individuals.
 - [velocyto.R/0.6-foss-2019b-R-4.0.2](http://velocyto.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/v/velocyto.R/velocyto.R-0.6-foss-2019b-R-4.0.2.eb)
velocyto (velox + κύτος, quick cell) is a package for the analysis of expression dynamics in single cell RNA seq data. In particular, it enables estimations of RNA velocities of single cells by distinguishing unspliced and spliced mRNAs in standard single-cell RNA sequencing protocols (see pre-print below for more information).
 - [vt/0.57721-foss-2018b](http://genome.sph.umich.edu/wiki/Vt)
A tool set for short variant discovery in genetic sequence data.
 - [wget/1.21.2-GCCcore-11.2.0](https://www.gnu.org/software/wget)
GNU Wget is a free software package for retrieving files using HTTP, HTTPS and FTP,
 the most widely-used Internet protocols. It is a non-interactive commandline tool,
 so it may easily be called from scripts, cron jobs, terminals without X-Windows support, etc.
 - [wheel/0.31.1-fosscuda-2018b-Python-3.6.6](https://pypi.python.org/pypi/wheel)
A built-package format for Python.
 - [wot/1.0.8-foss-2018b-Python-3.6.6]()
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/w/wot/wot-1.0.8-foss-2018b-Python-3.6.6.eb)
Single-cell RNA sequencing is a powerful technology that can reveal a lot about what happens in a group of cells as they develop. But because the technology destroys a cell, it can only provide snapshots of the cells in a group at one point in time. To really understand how cells develop over time, snapshots aren't good enough: scientists want to fill in the gaps between snapshots and string everything together into a movie.
 - [wpebackend-fdo/1.13.1-GCCcore-11.2.0](https://wpewebkit.org/)
WPE WebKit allows embedders to create simple and performant 
systems based on Web platform technologies. It is a WebKit port designed with 
flexibility and hardware acceleration in mind, leveraging common 3D graphics 
APIs for best performance.
 - [wxPython/4.2.0-foss-2021b](https://www.wxpython.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/w/wxPython/wxPython-4.2.0-foss-2021b.eb)
Wraps the wxWidgets C++ toolkit and provides access to the user interface portions of the wxWidgets 
API, enabling Python applications to have a native GUI on Windows, Macs or Unix systems, with a native look and feel 
and requiring very little (if any) platform specific code.
 - [wxWidgets/3.2.2.1-GCCcore-12.2.0](https://www.wxwidgets.org)
wxWidgets is a C++ library that lets developers create
applications for Windows, Mac OS X, Linux and other platforms with a
single code base. It has popular language bindings for Python, Perl,
Ruby and many other languages, and unlike other cross-platform toolkits,
wxWidgets gives applications a truly native look and feel because it
uses the platform's native API rather than emulating the GUI.
 - [x264/20230226-GCCcore-12.2.0](https://www.videolan.org/developers/x264.html)
x264 is a free software library and application for encoding video streams
 into the H.264/MPEG-4 AVC compression format, and is released under the
 terms of the GNU GPL.

 - [x265/3.5-GCCcore-10.3.0](https://x265.org/)
x265 is a free software library and application for encoding video streams
 into the H.265 AVC compression format, and is released under the terms of
 the GNU GPL.

 - [xbitmaps/1.1.2](http://www.freedesktop.org/wiki/Software/xlibs)
provides bitmaps for x
 - [xfig/3.2.6a-foss-2019b](http://www.xfig.org/)
Xfig is an interactive drawing tool which runs under X Window System.
 - [xmlto/0.0.28-GCC-11.2.0](https://www.pagure.io/xmlto)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/x/xmlto/xmlto-0.0.28-GCC-11.2.0.eb)
A tool for converting XML files to various formats
 - [xorg-macros/1.19.3-GCCcore-10.3.0](https://cgit.freedesktop.org/xorg/util/macros)
X.org macros utilities.
 - [xprop/1.2.5-GCCcore-10.2.0](https://www.x.org/wiki/)
The xprop utility is for displaying window and font properties in an X server.
 One window or font is selected using the command line arguments or possibly
 in the case of a window, by clicking on the desired window. A list of
 properties is then given, possibly with formatting information.
 - [xproto/7.0.31-GCCcore-7.3.0](http://www.freedesktop.org/wiki/Software/xlibs)
X protocol and ancillary headers
 - [yaml-cpp/0.7.0-GCCcore-12.2.0](https://github.com/jbeder/yaml-cpp)
yaml-cpp is a YAML parser and emitter in C++ matching the YAML 1.2 spec
 - [zlib/1.2.13](https://www.zlib.net/)
zlib is designed to be a free, general-purpose, legally unencumbered -- that
 is, not covered by any patents -- lossless data-compression library for use
 on virtually any computer hardware and operating system.

 - [zstd/1.5.2-GCCcore-11.3.0](https://facebook.github.io/zstd)
Zstandard is a real-time compression algorithm, providing high compression ratios.
 It offers a very wide range of compression/speed trade-off, while being backed by a very fast decoder.
 It also offers a special mode for small data, called dictionary compression, and can create dictionaries
 from any sample set.
