---
title: All Modules 14.04
permalink: /all-modules-14.04/
layout: single
toc: true
toc_label: "On This Page"
sidebar:
  nav: "docs"
---
Archive list of modules built for Trusty. End of support May 22, 2020.
 
 - [AMOS/3.1.0-foss-2016b](http://sourceforge.net/apps/mediawiki/amos/index.php?title=AMOS)
The AMOS consortium is committed to the development of open-source whole genome assembly software
 - [ATK/2.28.1-foss-2018b](https://developer.gnome.org/ATK/stable/)
ATK provides the set of accessibility interfaces that are implemented by other
 toolkits and applications. Using the ATK interfaces, accessibility tools have
 full access to view and control running applications.

 - [Anaconda3/2019.10](https://www.anaconda.com)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/a/Anaconda3/Anaconda3-2019.10.eb)
Built to complement the rich, open source Python community,
the Anaconda platform provides an enterprise-ready data analytics platform 
that empowers companies to adopt a modern open data science analytics architecture.

 - [Autoconf/2.69-GCCcore-8.3.0](http://www.gnu.org/software/autoconf/)
Autoconf is an extensible package of M4 macros that produce shell scripts
 to automatically configure software source code packages. These scripts can
 adapt the packages to many kinds of UNIX-like systems without manual user
 intervention. Autoconf creates a configuration script for a package from a
 template file that lists the operating system features that the package can
 use, in the form of M4 macro calls.

 - [Automake/1.16.1-GCCcore-7.3.0](http://www.gnu.org/software/automake/automake.html)
Automake: GNU Standards-compliant Makefile generator
 - [Autotools/20180311-GCCcore-8.3.0](http://autotools.io)
This bundle collect the standard GNU build tools: Autoconf, Automake
 and libtool

 - [BBMap/38.44-foss-2016b](https://sourceforge.net/projects/bbmap/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/BBMap/BBMap-38.44-foss-2016b.eb)
BBMap short read aligner, and other bioinformatic tools.
 - [BCFtools/1.9-foss-2016b](http://www.htslib.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/BCFtools/BCFtools-1.9-foss-2016b.eb)
Samtools is a suite of programs for interacting with high-throughput sequencing data.
 BCFtools
 - [BEAST/1.8.2]()

 - [BEDOPS/2.4.35-foss-2016b](http://bedops.readthedocs.io/en/latest/index.html)
BEDOPS is an open-source command-line toolkit that performs highly efficient and
 scalable Boolean and other set operations, statistical calculations, archiving, conversion and
 other management of genomic data of arbitrary scale. Tasks can be easily split by chromosome for
 distributing whole-genome analyses across a computational cluster.
 - [BEDTools/2.29.1-foss-2016b](https://github.com/arq5x/bedtools2)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/BEDTools/BEDTools-2.29.1-foss-2016b.eb)
The BEDTools utilities allow one to address common genomics
 tasks such as finding feature overlaps and computing coverage. The utilities
 are largely based on four widely-used file formats: BED, GFF/GTF, VCF,
 and SAM/BAM.
 - [BLAST/2.2.26-Linux_x86_64](http://blast.ncbi.nlm.nih.gov/ )

 - [BLAST+/2.7.1-foss-2016b](http://blast.ncbi.nlm.nih.gov/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/BLAST+/BLAST+-2.7.1-foss-2016b.eb)
Basic Local Alignment Search Tool, or BLAST, is an algorithm
 for comparing primary biological sequence information, such as the amino-acid
 sequences of different proteins or the nucleotides of DNA sequences.
 - [BWA/0.7.17-foss-2016b](http://bio-bwa.sourceforge.net)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/BWA/BWA-0.7.17-foss-2016b.eb)
BWA is a software package for mapping DNA sequences against a large reference genome,
 such as the human genome. It consists of three algorithms: BWA-backtrack, BWA-SW and BWA-MEM. The
 first algorithm is designed for Illumina sequence reads up to 100bp, while the rest two for longer
 sequences ranged from 70bp to a few megabases. BWA-MEM and BWA-SW share similar features such as the
 support of long reads and chimeric alignment, but BWA-MEM, which is the latest, is generally
 recommended as it is faster and more accurate. BWA-MEM also has better performance than BWA-backtrack
 for 70-100bp Illumina reads.
 - [BamTools/2.4.1-foss-2016b](https://github.com/pezmaster31/bamtools)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/BamTools/BamTools-2.4.1-foss-2016b.eb)
BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.
 - [Bazel/0.24.1-GCCcore-5.4.0](http://bazel.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/Bazel/Bazel-0.24.1-GCCcore-5.4.0.eb)
Bazel is a build tool that builds code quickly and reliably. 
It is used to build the majority of Google's software.
 - [Beast/2.6.2-GCCcore-5.4.0](http://beast2.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/Beast/Beast-2.6.2-GCCcore-5.4.0.eb)
BEAST is a cross-platform program for Bayesian MCMC analysis of molecular
 sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using
 strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies
 but is also a framework for testing evolutionary hypotheses without conditioning on a single
 tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted
 proportional to its posterior probability. 
 - [Bio-DB-HTS/2.11-foss-2016b-Perl-5.24.0](https://metacpan.org/release/Bio-DB-HTS)
Read files using HTSlib including BAM/CRAM, Tabix and BCF database files
 - [BioPerl/1.7.2-foss-2016b-Perl-5.28.0](http://www.bioperl.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/BioPerl/BioPerl-1.7.2-foss-2016b-Perl-5.28.0.eb)
Bioperl is the product of a community effort to produce Perl code which is useful in biology.
 Examples include Sequence objects, Alignment objects and database searching objects.
 - [Biopython/1.74-foss-2016b-Python-3.7.4](http://www.biopython.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/Biopython/Biopython-1.74-foss-2016b-Python-3.7.4.eb)
Biopython is a set of freely available tools for biological
 computation written in Python by an international team of developers. It is
 a distributed collaborative effort to develop Python libraries and
 applications which address the needs of current and future work in
 bioinformatics. 
 - [Bismark/0.22.3-foss-2016b](https://www.bioinformatics.babraham.ac.uk/projects/bismark/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/Bismark/Bismark-0.22.3-foss-2016b.eb)
A tool to map bisulfite converted sequence reads and determine cytosine methylation states
 - [Bison/3.3.2-GCCcore-8.3.0](http://www.gnu.org/software/bison)
Bison is a general-purpose parser generator that converts an annotated context-free grammar
 into a deterministic LR or generalized LR (GLR) parser employing LALR(1) parser tables.
 - [Blosc/1.14.2-foss-2016b](http://www.blosc.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/Blosc/Blosc-1.14.2-foss-2016b.eb)
Blosc, an extremely fast, multi-threaded, meta-compressor library
 - [Boost/1.69.0-foss-2016b](http://www.boost.org/)
Boost provides free peer-reviewed portable C++ source libraries.
 - [Bowtie/1.2.1.1-foss-2016b](http://bowtie-bio.sourceforge.net/index.shtml)
Bowtie is an ultrafast, memory-efficient short read aligner.
 It aligns short DNA sequences (reads) to the human genome.
 - [Bowtie2/2.3.4.3-foss-2016b](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/Bowtie2/Bowtie2-2.3.4.3-foss-2016b.eb)
Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads
 to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s
 of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.
 Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,
 its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.
 - [CAP3/20150211](http://seq.cs.iastate.edu/)
CAP3 assembly program 
 - [CARBayes/5.1.1-foss-2016b-R-3.5.2](http://github.com/duncanplee/CARBayes)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CARBayes/CARBayes-5.1.1-foss-2016b-R-3.5.2.eb)
Implements a class of univariate and multivariate spatial generalised linear mixed models for areal unit data, with inference in a Bayesian setting using Markov chain Monte Carlo (MCMC) simulation.
 - [CITE-seq-Count/1.4.2-foss-2016b-Python-3.7.4](https://hoohm.github.io/CITE-seq-Count)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CITE-seq-Count/CITE-seq-Count-1.4.2-foss-2016b-Python-3.7.4.eb)
A python package that allows to count antibody TAGS from a CITE-seq and/or
 cell hashing experiment.
 - [CLISP/2.49-GCCcore-6.4.0](http://www.clisp.org/)
Common Lisp is a high-level, general-purpose, object-oriented, dynamic,
 functional programming language.

 - [CMake/3.12.1-GCCcore-5.4.0](https://www.cmake.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CMake/CMake-3.12.1-GCCcore-5.4.0.eb)
CMake, the cross-platform, open-source build system.  CMake is a family of
 tools designed to build, test and package software.

 - [CNVnator/0.4.1-foss-2016b](https://github.com/abyzovlab/CNVnator)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CNVnator/CNVnator-0.4.1-foss-2016b.eb)
A tool for CNV discovery and genotyping from depth-of-coverage by mapped reads

 - [CUDA/10.1.105-GCC-5.4.0-2.26](https://developer.nvidia.com/cuda-toolkit)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CUDA/CUDA-10.1.105-GCC-5.4.0-2.26.eb)
CUDA (formerly Compute Unified Device Architecture) is a parallel
 computing platform and programming model created by NVIDIA and implemented by the
 graphics processing units (GPUs) that they produce. CUDA gives developers access
 to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.
 - [Cgl/0.60.2-foss-2016b](https://github.com/coin-or/Clg)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/Cgl/Cgl-0.60.2-foss-2016b.eb)
The COIN-OR Cut Generation Library (Cgl) is a collection of cut generators that can be used with other COIN-OR packages that make use of cuts, such as, among others, the linear solver Clp or the mixed integer linear programming solvers Cbc or BCP. Cgl uses the abstract class OsiSolverInterface (see Osi) to use or communicate with a solver. It does not directly call a solver.
 - [Check/0.12.0-foss-2016b](https://github.com/libcheck/check)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/Check/Check-0.12.0-foss-2016b.eb)
A unit testing framework for C 
 - [Circos/0.69-6-GCCcore-7.3.0-Perl-5.28.0](http://www.circos.ca/)
Circos is a software package for visualizing data and information.
 It visualizes data in a circular layout
 - [Clairvoyante/1.02-foss-2016b-Python-3.6.6](https://github.com/aquaskyline/Clairvoyante)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/Clairvoyante/Clairvoyante-1.02-foss-2016b-Python-3.6.6.eb)
Clairvoyante, a multi-task five-layer convolutional neural network model
 for predicting variant type, zygosity, alternative allele and Indel length.
 - [Clang/6.0.1-GCC-5.4.0-2.26](http://clang.llvm.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/Clang/Clang-6.0.1-GCC-5.4.0-2.26.eb)
C, C++, Objective-C compiler, based on LLVM.  Does not
 include C++ standard library -- use libstdc++ from GCC.
 - [Clp/1.17.3-foss-2016b](https://github.com/coin-or/Clp)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/Clp/Clp-1.17.3-foss-2016b.eb)
Clp (Coin-or linear programming) is an open-source linear programming solver
 - [CoinUtils/2.11.2-foss-2016b](https://www.coin-or.org/Doxygen/CoinUtils/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/CoinUtils/CoinUtils-2.11.2-foss-2016b.eb)
CoinUtils (Coin-OR Utilities) is an open-source collection of classes
 and functions that are generally useful to more than one COIN-OR project. A collection
 of routines for manipulating sparse matrices and other matrix operations
 - [Coreutils/8.27-GCCcore-5.4.0](http://www.gnu.org/software/coreutils/)
The GNU Core Utilities are the basic file, shell and text manipulation utilities of the
 GNU operating system.  These are the core utilities which are expected to exist on every operating system.

 - [DANPOS2/2.2.2-foss-2016b-Python-2.7.12](https://sites.google.com/site/danposdoc)
A toolkit for Dynamic Analysis of Nucleosome and Protein Occupancy by Sequencing, version 2
 - [DB/6.2.32-GCCcore-6.4.0](http://www.oracle.com/technetwork/products/berkeleydb)
Berkeley DB enables the development of custom data management solutions,
 without the overhead traditionally associated with such custom projects.

 - [DBD-mysql/4.033-foss-2016b-Perl-5.24.0](http://search.cpan.org/~capttofu/DBD-mysql-4.033/)
Perl binding for MySQL
 - [DIAMOND/0.9.22-foss-2016b](https://github.com/bbuchfink/diamond)
Accelerated BLAST compatible local sequence aligner
 - [DMTCP/2.5.2-foss-2016b](https://github.com/dmtcp/dmtcp)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/d/DMTCP/DMTCP-2.5.2-foss-2016b.eb)
DMTCP is a tool to transparently checkpoint the state of multiple simultaneous applications, including multi-threaded and distributed applications. It operates directly on the user binary executable, without any Linux kernel modules or other kernel modifications.
 - [Doxygen/1.8.13-foss-2016b](http://www.doxygen.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/d/Doxygen/Doxygen-1.8.13-foss-2016b.eb)
Doxygen is a documentation system for C++, C, Java, Objective-C, Python, 
 IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some extent D.
 - [EMBOSS/6.6.0-foss-2016b](http://emboss.sourceforge.net/)
EMBOSS is 'The European Molecular Biology Open Software Suite'.
 EMBOSS is a free Open Source software analysis package specially developed for
 the needs of the molecular biology (e.g. EMBnet) user community.
 - [EPACTS/a5209db-foss-2016b-R-3.4.1-fh1](http://github.com/statgen/EPACTS)
EPACTS is a versatile software pipeline to perform various statistical tests for identifying genome-wide association from sequence data through a user-friendly interface, both to scientific analysts and to method developer.s
 - [ESS/16.10](http://ess.r-project.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/e/ESS/ESS-16.10.eb)
Emacs Speaks Statistics (ESS) is an add-on package for emacs
 text editors such as GNU Emacs and XEmacs. It is designed to support editing
 of scripts and interaction with various statistical analysis programs such as
 R, S-Plus, SAS, Stata and OpenBUGS/JAGS.
 - [EasyBuild/4.2.0](https://easybuilders.github.io/easybuild)
EasyBuild is a software build and installation framework
 written in Python that allows you to install software in a structured,
 repeatable and robust way.
 - [Eigen/3.3.5](http://eigen.tuxfamily.org/index.php?title=Main_Page)
Eigen is a C++ template library for linear algebra:
 matrices, vectors, numerical solvers, and related algorithms.

 - [Emacs/26.3-foss-2016b](http://www.gnu.org/software/emacs/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/e/Emacs/Emacs-26.3-foss-2016b.eb)
GNU Emacs is an extensible, customizable text editor—and more.
 At its core is an interpreter for Emacs Lisp, a dialect of the Lisp programming language
 with extensions to support text editing.
 - [ExtUtils-CppGuess/0.19-foss-2016b-Perl-5.24.0](https://metacpan.org/pod/ExtUtils::CppGuess)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/e/ExtUtils-CppGuess/ExtUtils-CppGuess-0.19-foss-2016b-Perl-5.24.0.eb)
guess C++ compiler and flags
 - [FASTX-Toolkit/0.0.14-foss-2016b](http://hannonlab.cshl.edu/fastx_toolkit/)
The FASTX-Toolkit is a collection of command line tools for 
 Short-Reads FASTA/FASTQ files preprocessing.
 - [FFTW/3.3.8-gompic-2016b](http://www.fftw.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/FFTW/FFTW-3.3.8-gompic-2016b.eb)
FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)
 in one or more dimensions, of arbitrary input size, and of both real and complex data.
 - [FFmpeg/3.3.1-foss-2016b](https://www.ffmpeg.org/)
A complete, cross-platform solution to record, convert and stream audio and video.
 - [FIt-SNE/1.1.0-GCCcore-7.3.0](https://github.com/KlugerLab/FIt-SNE)
t-distributed stochastic neighbor embedding (t-SNE) is widely used for
 visualizing single-cell RNA-sequencing (scRNA-seq) data, but it scales poorly to large
 datasets. We dramatically accelerate t-SNE, obviating the need for data downsampling, and
 hence allowing visualization of rare cell populations. Furthermore, we implement a
 heatmap-style visualization for scRNA-seq based on one-dimensional t-SNE for simultaneously
 visualizing the expression patterns of thousands of genes.
 - [FLASH/2.2.00-foss-2016b](https://ccb.jhu.edu/software/FLASH/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/FLASH/FLASH-2.2.00-foss-2016b.eb)
FLASH (Fast Length Adjustment of SHort reads) is a very fast and accurate software
  tool to merge paired-end reads from next-generation sequencing experiments. FLASH is designed to
  merge pairs of reads when the original DNA fragments are shorter than twice the length of reads.
  The resulting longer reads can significantly improve genome assemblies. They can also improve
  transcriptome assembly when FLASH is used to merge RNA-seq data.
 - [FLTK/1.3.4-foss-2018b](http://www.fltk.org)
FLTK is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows,
 and MacOS X. FLTK provides modern GUI functionality without the bloat and supports 3D graphics via OpenGL
 and its built-in GLUT emulation.
 - [FastANI/1.1-foss-2016b](http://www.iodbc.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/FastANI/FastANI-1.1-foss-2016b.eb)
FastANI is developed for fast alignment-free computation of whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean nucleotide identity of orthologous gene pairs shared between two microbial genomes. FastANI supports pairwise comparison of both complete and draft genome assemblies.
 - [FastQC/0.11.8-Java-1.8](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/FastQC/FastQC-0.11.8-Java-1.8.eb)
FastQC is a quality control application for high throughput
sequence data. It reads in sequence data in a variety of formats and can either
provide an interactive application to review the results of several different
QC checks, or create an HTML based report which can be integrated into a
pipeline.
 - [FastTree/2.1.9-foss-2016b](http://www.microbesonline.org/fasttree/ )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/FastTree/FastTree-2.1.9-foss-2016b.eb)
FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide  or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of  time and memory. 
 - [File-ReadBackwards/1.05-GCCcore-7.3.0-Perl-5.28.0](https://metacpan.org/pod/File::ReadBackwards)
This Perl module reads a file backwards line by line.
 - [Flt-SNE/1.1.0-GCCcore-7.3.0](https://github.com/KlugerLab/FIt-SNE)
t-distributed stochastic neighbor embedding (t-SNE) is widely used for
 visualizing single-cell RNA-sequencing (scRNA-seq) data, but it scales poorly to large
 datasets. We dramatically accelerate t-SNE, obviating the need for data downsampling, and
 hence allowing visualization of rare cell populations. Furthermore, we implement a
 heatmap-style visualization for scRNA-seq based on one-dimensional t-SNE for simultaneously
 visualizing the expression patterns of thousands of genes.
 - [FriBidi/1.0.5-GCCcore-7.3.0](https://github.com/fribidi/fribidi)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/FriBidi/FriBidi-1.0.5-GCCcore-7.3.0.eb)
The Free Implementation of the Unicode Bidirectional Algorithm.

 - [GATK/4.1.4.1-foss-2016b-Python-3.7.4](http://www.broadinstitute.org/gatk/)
The Genome Analysis Toolkit or GATK is a software package
developed at the Broad Institute to analyse next-generation resequencing
data. The toolkit offers a wide variety of tools, with a primary focus on
variant discovery and genotyping as well as strong emphasis on data quality
assurance. Its robust architecture, powerful processing engine and
high-performance computing features make it capable of taking on projects
of any size.
 - [GCC/8.3.0](http://gcc.gnu.org/)
The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
 as well as libraries for these languages (libstdc++, libgcj,...).
 - [GCCcore/8.3.0](http://gcc.gnu.org/)
The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
 as well as libraries for these languages (libstdc++, libgcj,...).
 - [GCTA/1.92.3beta3](https://cnsgenomics.com/software/gcta)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GCTA/GCTA-1.92.3beta3.eb)
GCTA (Genome-wide Complex Trait Analysis) was originally designed to
 estimate the proportion of phenotypic variance explained by all genome-wide SNPs for
 complex traits (the GREML method), and has subsequently extended for many other analyses
 to better understand the genetic architecture of complex traits.
 - [GD/2.69-GCCcore-7.3.0-Perl-5.28.0](https://github.com/lstein/Perl-GD)
GD.pm
 - [GDAL/3.0.0-foss-2016b-Python-3.7.4](http://www.gdal.org/)
GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
 Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
 to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
 data translation and processing.
 - [GEOS/3.7.2-foss-2016b-Python-3.7.4](http://trac.osgeo.org/geos)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GEOS/GEOS-3.7.2-foss-2016b-Python-3.7.4.eb)
GEOS (Geometry Engine
 - [GISTIC/2.0.23](www.broadinstitute.org/cancer/cga/gistc)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GISTIC/GISTIC-2.0.23.eb)
GISTIC is a tool to identify genes targeted by somatic copy-number alterations (SCNAs) that drive cancer growth. By separating SCNA profiles into underlying arm-level and focal alterations, GISTIC estimates the background rates for each category as well as defines the boundaries of SCNA regions.
 - [GLPK/4.61-foss-2016b](https://www.gnu.org/software/glpk/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GLPK/GLPK-4.61-foss-2016b.eb)
The GLPK (GNU Linear Programming Kit) package is intended for
 solving large-scale linear programming (LP),
 mixed integer programming (MIP), and other related problems.
 It is a set of routines written in ANSI C 
 and organized in the form of a callable library.
 - [GLib/2.54.3-GCCcore-7.3.0](http://www.gtk.org/)
GLib is one of the base libraries of the GTK+ project
 - [GMAP-GSNAP/2017-02-25-foss-2016b](http://research-pub.gene.com/gmap/)
GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences
 GSNAP: Genomic Short-read Nucleotide Alignment Program
 - [GMP/6.1.2-GCCcore-7.3.0](http://gmplib.org/)
GMP is a free library for arbitrary precision arithmetic, operating on signed
 integers, rational numbers, and floating point numbers.

 - [GMime/2.6.23-GCCcore-5.4.0](http://spruce.sourceforge.net/gmime/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GMime/GMime-2.6.23-GCCcore-5.4.0.eb)
The GMime package contains a set of utilities for parsing and
 creating messages using the Multipurpose Internet Mail Extension (MIME) as
 defined by the applicable RFCs.
 - [GNU/4.9.3-2.25](http://www.gnu.org/software/ )
Compiler-only toolchain with GCC and binutils.
 - [GObject-Introspection/1.54.1-foss-2018b-Python-2.7.15](https://wiki.gnome.org/GObjectIntrospection/)
GObject introspection is a middleware layer between C
 libraries (using GObject) and language bindings. The C library can be scanned
 at compile time and generate a metadata file, in addition to the actual
 native C library. Then at runtime, language bindings can read this
 metadata and automatically provide bindings to call into the C library.
 - [GSL/2.5-GCCcore-5.4.0](http://www.gnu.org/software/gsl/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GSL/GSL-2.5-GCCcore-5.4.0.eb)
The GNU Scientific Library (GSL) is a numerical library for C and C++
 programmers.  The library provides a wide range of mathematical routines
 such as random number generators, special functions and least-squares fitting.

 - [GTDBTk/0.1.3-foss-2016b-Python-2.7.15](https://github.com/Ecogenomics/GTDBTk)
GTDB-Tk is a software toolkit for assigning objective
 taxonomic classifications to bacterial and archaeal genomes. It is
 computationally efficient and designed to work with recent advances that
 allow hundreds or thousands of metagenome-assembled genomes (MAGs) to be
 obtained directly from environmental samples. It can also be applied to
 isolate and single-cell genomes.
 - [GTK+/2.24.32-foss-2018b](https://developer.gnome.org/gtk+/stable/)
The GTK+ 2 package contains libraries used for creating graphical user interfaces for applications.

 - [GTS/0.7.6-foss-2016b](http://gts.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GTS/GTS-0.7.6-foss-2016b.eb)
GTS stands for the GNU Triangulated Surface Library.
 It is an Open Source Free Software Library intended to provide a set of useful
 functions to deal with 3D surfaces meshed with interconnected triangles.
 - [Gctf/1.18_b2-sm70-cu9.2](https://www.mrc-lmb.cam.ac.uk/kzhang/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/Gctf/Gctf-1.18_b2-sm70-cu9.2.eb)
Real-time CTF determination and correction.
 - [Gdk-Pixbuf/2.36.12-foss-2018b](https://developer.gnome.org/gdk-pixbuf/stable/)
The Gdk Pixbuf is a toolkit for image loading and pixel buffer manipulation.
 It is used by GTK+ 2 and GTK+ 3 to load and manipulate images. In the past it
 was distributed as part of GTK+ 2 but it was split off into a separate package
 in preparation for the change to GTK+ 3.

 - [Ghostscript/9.20-foss-2016b](http://ghostscript.com)
Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
 different targets. It used to be part of the cups printing stack, but is no longer used for that.
 - [GitPython/2.1.11-foss-2016b-Python-3.7.4](https://github.com/gitpython-developers/GitPython)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GitPython/GitPython-2.1.11-foss-2016b-Python-3.7.4.eb)
GitPython is a python library used to interact with Git repositories 
 - [Go/1.14](http://www.golang.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/Go/Go-1.14.eb)
Go is an open source programming language that makes it easy to build
 simple, reliable, and efficient software.
 - [Guile/2.0.11-foss-2016b](https://www.gnu.org/software/guile/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/Guile/Guile-2.0.11-foss-2016b.eb)
Guile is a programming language, designed to help programmers create flexible applications that
 can be extended by users or other programmers with plug-ins, modules, or scripts.
 - [HDF5/1.10.2-foss-2018b](https://portal.hdfgroup.org/display/support)
HDF5 is a data model, library, and file format for storing and managing data.
 It supports an unlimited variety of datatypes, and is designed for flexible
 and efficient I/O and for high volume and complex data.
 - [HISAT2/2.0.4-foss-2016b](https://ccb.jhu.edu/software/hisat2/index.shtml)
HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads
 (both DNA and RNA) against the general human population (as well as against a single reference genome).
 - [HMMER/3.2.1-foss-2016b](http://hmmer.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/h/HMMER/HMMER-3.2.1-foss-2016b.eb)
HMMER is used for searching sequence databases for homologs
 of protein sequences, and for making protein sequence alignments. It
 implements methods using probabilistic models called profile hidden Markov
 models (profile HMMs).  Compared to BLAST, FASTA, and other sequence
 alignment and database search tools based on older scoring methodology,
 HMMER aims to be significantly more accurate and more able to detect remote
 homologs because of the strength of its underlying mathematical models. In the
 past, this strength came at significant computational expense, but in the new
 HMMER3 project, HMMER is now essentially as fast as BLAST.
 - [HTSlib/1.10-foss-2016b](http://www.htslib.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/h/HTSlib/HTSlib-1.10-foss-2016b.eb)
A C library for reading/writing high-throughput sequencing data.
 HTSlib also provides the bgzip, htsfile, and tabix utilities
 - [HarfBuzz/2.2.0-foss-2018b](http://www.freedesktop.org/wiki/Software/HarfBuzz)
HarfBuzz is an OpenType text shaping engine.
 - [Homer/v4.9-foss-2016b](http://homer.ucsd.edu/homer/download.html)
HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and ChIP-Seq analysis. It is a collection of command line programs for unix-style operating systems written in mostly perl and c++. Homer was primarily written as a de novo motif discovery algorithm that is well suited for finding 8-12 bp motifs in large scale genomics data.
 - [ICU/61.1-foss-2016b](http://site.icu-project.org/home)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/ICU/ICU-61.1-foss-2016b.eb)
ICU is a mature, widely used set of C/C++ and Java libraries providing Unicode and Globalization
 support for software applications.
 - [IGV/2.5.0-Java-11](http://www.broadinstitute.org/software/igv/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/IGV/IGV-2.5.0-Java-11.eb)
This package contains command line utilities for
 preprocessing, computing feature count density (coverage),  sorting, and
 indexing data files.
 - [IGVTools/2.4.16-Java-1.8](http://www.broadinstitute.org/software/igv/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/IGVTools/IGVTools-2.4.16-Java-1.8.eb)
This package contains command line utilities for
 preprocessing, computing feature count density (coverage),  sorting, and
 indexing data files. 
 - [IOR/3.2.1-foss-2016b](https://github.com/chaos/ior)
The IOR software is used for benchmarking parallel file systems using POSIX, MPIIO,
 or HDF5 interfaces. 
 - [IPython/7.13.0-foss-2016b-Python-3.7.4](http://ipython.org/index.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/IPython/IPython-7.13.0-foss-2016b-Python-3.7.4.eb)
IPython provides a rich architecture for interactive computing with:
 Powerful interactive shells (terminal and Qt-based).
 A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media.
 Support for interactive data visualization and use of GUI toolkits.
 Flexible, embeddable interpreters to load into your own projects.
 Easy to use, high performance tools for parallel computing.
 - [IQ-TREE/1.6.12-GCCcore-5.4.0](http://www.iqtree.org/)
A fast and effective stochastic algorithm to infer phylogenetic
 trees by maximum likelihood.
 - [IgBLAST/1.15.0-x64-linux](https://ncbi.github.io/igblast)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/IgBLAST/IgBLAST-1.15.0-x64-linux.eb)
IgBLAST faclilitates the analysis of immunoglobulin and T cell receptor variable domain sequences.
 - [ImageMagick/7.0.5-10-foss-2016b](http://www.imagemagick.org/)
ImageMagick is a software suite to create, edit, compose, or convert bitmap images
 - [ImmunediveRsity/1.0.9-foss-2016b](https://bitbucket.org/ImmunediveRsity/immunediversity)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/ImmunediveRsity/ImmunediveRsity-1.0.9-foss-2016b.eb)
Evaluation of the antibody repertoire by analyzing HTS data. Manipulation and processing of HTS reads to identify VDJ usage and clonal origin to gain insight of the antibody repertoire of a given organism.
 - [JAGS/4.2.0-foss-2016b](http://mcmc-jags.sourceforge.net/ )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/j/JAGS/JAGS-4.2.0-foss-2016b.eb)
JAGS is Just Another Gibbs Sampler.  It is a program for analysis   of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  
 - [JUnit/4.12-Java-1.8.0_92](http://sourceforge.net/projects/junit)
A programmer-oriented testing framework for Java.
 - [JasPer/2.0.12-foss-2016b](http://www.ece.uvic.ca/~frodo/jasper/)
The JasPer Project is an open-source initiative to provide a free
 software-based reference implementation of the codec specified in the JPEG-2000 Part-1 standard.
 - [Java/12+32](https://jdk.java.net/)
The official Reference Implementation for Java SE 11 (JSR 384)
 is based solely upon open-source code available from the JDK 11 Project in
 the OpenJDK Community.
 - [Kent_tools/20181218-linux.x86_64](http://genome.cse.ucsc.edu/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/k/Kent_tools/Kent_tools-20181218-linux.x86_64.eb)
Kent tools: collection of tools used by the UCSC genome browser.
 - [Keras/2.3.0-foss-2016b-Python-3.7.4](https://keras.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/k/Keras/Keras-2.3.0-foss-2016b-Python-3.7.4.eb)
Keras is a minimalist, highly modular neural networks library, written in Python and
capable of running on top of either TensorFlow or Theano.
 - [Kraken2/2.0.7-beta-foss-2016b-Perl-5.28.0](http://www.ccb.jhu.edu/software/kraken2/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/k/Kraken2/Kraken2-2.0.7-beta-foss-2016b-Perl-5.28.0.eb)
Kraken is a system for assigning taxonomic labels to short DNA sequences,
 usually obtained through metagenomic studies. Previous attempts by other
 bioinformatics software to accomplish this task have often used sequence
 alignment or machine learning techniques that were quite slow, leading to
 the development of less sensitive but much faster abundance estimation
 programs. Kraken aims to achieve high sensitivity and high speed by
 utilizing exact alignments of k-mers and a novel classification algorithm.
 - [LAME/3.99.5-foss-2016b](http://lame.sourceforge.net/)
LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL.
 - [LAST/926-foss-2016b](http://last.cbrc.jp/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/LAST/LAST-926-foss-2016b.eb)
LAST finds similar regions between sequences.
 - [LEMON/1.3.1-foss-2016b](http://lemon.cs.elte.hu/trac/lemon)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/LEMON/LEMON-1.3.1-foss-2016b.eb)
LEMON stands for Library for Efficient Modeling and Optimization in Networks. It is a C++ template library providing efficient implementations of common data structures and algorithms with focus on combinatorial optimization tasks connected mainly with graphs and networks.
 - [LLVM/7.1.0-GCCcore-5.4.0](http://llvm.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/LLVM/LLVM-7.1.0-GCCcore-5.4.0.eb)
The LLVM Core libraries provide a modern source- and
 target-independent optimizer, along with code generation support for many
 popular CPUs (as well as some less common ones!) These libraries are built
 around a well specified code representation known as the LLVM intermediate
 representation ("LLVM IR"). The LLVM Core libraries are well documented, and
 it is particularly easy to invent your own language (or port an existing
 compiler) to use LLVM as an optimizer and code generator.
 - [LMDB/0.9.22-GCCcore-7.3.0](https://symas.com/lmdb)
LMDB is a fast, memory-efficient database. With memory-mapped
 files, it has the read performance of a pure in-memory database while
 retaining the persistence of standard disk-based databases.
 - [LZO/2.10-foss-2016b](http://www.oberhumer.com/opensource/lzo/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/LZO/LZO-2.10-foss-2016b.eb)
Portable lossless data compression library
 - [LibTIFF/4.0.10-GCCcore-5.4.0](https://libtiff.gitlab.io/libtiff/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/LibTIFF/LibTIFF-4.0.10-GCCcore-5.4.0.eb)
tiff: Library and tools for reading and writing TIFF data files
 - [LibUUID/1.0.3-GCCcore-6.4.0](http://sourceforge.net/projects/libuuid/)
Portable uuid C library
 - [LightGBM/2.2.3-foss-2016b](https://github.com/microsoft/LightGBM)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/LightGBM/LightGBM-2.2.3-foss-2016b.eb)
A fast, distributed, high performance gradient boosting (GBT, GBDT, GBRT,
 GBM or MART) framework based on decision tree algorithms, used for ranking, classification
 and many other machine learning tasks.
 - [LittleCMS/2.8-foss-2016b](http://www.littlecms.com/)
Little CMS intends to be an OPEN SOURCE small-footprint color management engine,
 with special focus on accuracy and performance. 
 - [Lmod/5.9-GCC-4.8.4](http://sourceforge.net/projects/lmod/ )
Lmod is a Lua based module system. Modules allow for dynamic modification  of a user's environment under Unix systems. See www.tacc.utexas.edu/tacc-projects/lmod  for a complete description. Lmod is a new implementation that easily handles the MODULEPATH  Hierarchical problem. It is drop-in replacement for TCL/C modules and reads TCL modulefiles directly.
 - [Lua/5.1.4-8-GCC-4.8.4](http://www.lua.org/ )
Lua is a powerful, fast, lightweight, embeddable scripting language.  Lua combines simple procedural syntax with powerful data description constructs based  on associative arrays and extensible semantics. Lua is dynamically typed,  runs by interpreting bytecode for a register-based virtual machine,  and has automatic memory management with incremental garbage collection,  making it ideal for configuration, scripting, and rapid prototyping.
 - [M4/1.4.18-GCCcore-6.4.0](http://www.gnu.org/software/m4/m4.html)
GNU M4 is an implementation of the traditional Unix macro processor. It is
 mostly SVR4 compatible although it has some extensions (for example, handling
 more than 9 positional parameters to macros). GNU M4 also has built-in
 functions for including files, running shell commands, doing arithmetic, etc.

 - [MACS2/2.2.6-foss-2016b-Python-3.7.4](http://liulab.dfci.harvard.edu/MACS)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MACS2/MACS2-2.2.6-foss-2016b-Python-3.7.4.eb)
Model Based Analysis for ChIP-Seq data
 - [MAFFT/7.305-foss-2016b-with-extensions](http://mafft.cbrc.jp/alignment/software/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MAFFT/MAFFT-7.305-foss-2016b-with-extensions.eb)
MAFFT is a multiple sequence alignment program 
 for unix-like operating systems.  It offers a range of multiple
 alignment methods, L-INS-i (accurate; for alignment of <∼200 sequences), 
 FFT-NS-2 (fast; for alignment of <∼10,000 sequences), etc.
 - [MAFFT/7.305-foss-2016b-with-extensions](http://mafft.cbrc.jp/alignment/software/)
MAFFT is a multiple sequence alignment program 
 for unix-like operating systems.  It offers a range of multiple
 alignment methods, L-INS-i (accurate; for alignment of <∼200 sequences), 
 FFT-NS-2 (fast; for alignment of <∼10,000 sequences), etc.
 - [MAFFT/7.305-foss-2016b-with-extensions](http://mafft.cbrc.jp/alignment/software/)
MAFFT is a multiple sequence alignment program 
 for unix-like operating systems.  It offers a range of multiple
 alignment methods, L-INS-i (accurate; for alignment of <∼200 sequences), 
 FFT-NS-2 (fast; for alignment of <∼10,000 sequences), etc.
 - [MOFA/1.0.0-foss-2016b-R-3.6.1](https://github.com/bioFAM/MOFA)
MOFA is a factor analysis model that provides a general framework for the integration
 of multi-omic data sets in a completely unsupervised fashion.
 - [MPFR/3.1.4-foss-2016b](http://www.mpfr.org)
The MPFR library is a C library for multiple-precision 
 floating-point computations with correct rounding.
 - [MUMmer/3.23-foss-2016b](http://mummer.sourceforge.net/)
MUMmer is a system for rapidly aligning entire genomes,
 whether in complete or draft form. AMOS makes use of it.
 - [Mako/1.0.7-foss-2018b-Python-2.7.15](http://www.makotemplates.org)
A super-fast templating language that borrows the best ideas from the existing templating languages
 - [MariaDB/10.3.10-foss-2018b](https://mariadb.org/)
MariaDB An enhanced, drop-in replacement for MySQL.
 - [MaxQuant/1.6.10.43-foss-2018b](https://maxquant.org/maxquant/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MaxQuant/MaxQuant-1.6.10.43-foss-2018b.eb)
MaxQuant is a quantitative proteomics software package designed for analyzing large
 mass-spectrometric data sets. It is specifically aimed at high-resolution MS data. Several labeling
 techniques as well as label-free quantification are supported.
 - [Mesa/18.1.1-foss-2018b](http://www.mesa3d.org/)
Mesa is an open-source implementation of the OpenGL specification -
 a system for rendering interactive 3D graphics.
 - [MiXCR/3.0.3-Java-1.8](https://milaboratory.com/software/mixcr/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MiXCR/MiXCR-3.0.3-Java-1.8.eb)
MiXCR processes big immunome data from raw sequences 
to quantitated clonotypes 
 - [Miniconda2/4.5.11](https://www.continuum.io/anaconda-overview)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/Miniconda2/Miniconda2-4.5.11.eb)
Built to complement the rich, open source Python community,
the Anaconda platform provides an enterprise-ready data analytics platform 
that empowers companies to adopt a modern open data science analytics architecture.

 - [Miniconda3/4.7.10](https://www.continuum.io/anaconda-overview)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/Miniconda3/Miniconda3-4.7.10.eb)
Built to complement the rich, open source Python community,
the Anaconda platform provides an enterprise-ready data analytics platform 
that empowers companies to adopt a modern open data science analytics architecture.

 - [Mono/6.4.0.198-foss-2018b](https://www.mono-project.com/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/Mono/Mono-6.4.0.198-foss-2018b.eb)
An open source, cross-platform, implementation of C# and the CLR that is
 binary compatible with Microsoft.NET.
 - [MoreRONN/4.9-foss-2016b](https://github.com/varun-ramraj/MoreRONN/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MoreRONN/MoreRONN-4.9-foss-2016b.eb)
The successor to RONN, prediction of protein disorder.
 - [MotionCor2/1.2.1-Cuda92](http://msg.ucsf.edu/em/software/motioncor2.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MotionCor2/MotionCor2-1.2.1-Cuda92.eb)
Based on observations that the electron beam induces doming of the thin vitreous ice layer, we developed an algorithm to correct anisotropic image motion at the single pixel level across the whole frame, suitable for both single particle and tomographic images.
 - [MultiQC/1.8-foss-2016b-Python-3.7.4](http://multiqc.info)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MultiQC/MultiQC-1.8-foss-2016b-Python-3.7.4.eb)
Aggregate results from bioinformatics analyses across many samples into a single 
 report.

 MultiQC searches a given directory for analysis logs and compiles a HTML report. It's a general
 use tool, perfect for summarising the output from numerous bioinformatics tools.
 - [MutSig/2](http://software.broadinstitute.org/cancer/cga/mutsig)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/MutSig/MutSig-2.eb)
MutSig stands for "Mutation Significance".  MutSig analyzes lists of mutations
 discovered in DNA sequencing, to identify genes that were mutated more often than expected by
 chance given background mutation processes.
 - [NASM/2.14.02-GCCcore-5.4.0](http://www.nasm.us/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/NASM/NASM-2.14.02-GCCcore-5.4.0.eb)
NASM: General-purpose x86 assembler
 - [NGS/1.2.5-foss-2016a](https://github.com/ncbi/ngs )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/NGS/NGS-1.2.5-foss-2016a.eb)
NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from  Next Generation Sequencing.
 - [NLopt/2.6.1-GCCcore-7.3.0](http://ab-initio.mit.edu/wiki/index.php/NLopt)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/NLopt/NLopt-2.6.1-GCCcore-7.3.0.eb)
NLopt is a free/open-source library for nonlinear optimization, 
 providing a common interface for a number of different free optimization routines 
 available online as well as original implementations of various other algorithms. 
 - [NanoOk/1.26-foss-2016b](http://nanook.readthedocs.io/en/latest/)
NanoOK (pronounced na-nook) is a tool for extraction, alignment and analysis of Nanopore reads. NanoOK will extract reads as FASTA or FASTQ files, align them (with a choice of alignment tools), then generate a comprehensive multi-page PDF report containing yield, accuracy and quality analysis. Along the way, it generates plain text files which can be used for further analysis, as well as graphs suitable for inclusion in presentations and papers.
 - [NanoSV/1.1.2-foss-2016b-Python-3.6.4](http://pmw.sourceforge.net)
Pmw is a toolkit for building high-level compound widgets in Python using the Tkinter module. 
 - [NiftyNet/0.2.2-foss-2016b-Python-3.6.5-fh1](https://http://www.niftynet.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/NiftyNet/NiftyNet-0.2.2-foss-2016b-Python-3.6.5-fh1.eb)
NiftyNet is a TensorFlow-based open-source convolutional neural networks (CNNs) platform for research in medical image analysis and image-guided therapy.
 - [Nim/1.0.0-GCCcore-7.3.0](https://nim-lang.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/Nim/Nim-1.0.0-GCCcore-7.3.0.eb)
Nim is a systems and applications programming language.
 - [Ninja/1.8.2-foss-2016b](https://ninja-build.org/)
Ninja is a small build system with a focus on speed.
 - [OCaml/4.02.3-foss-2016b](http://ocaml.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/o/OCaml/OCaml-4.02.3-foss-2016b.eb)
OCaml is a general purpose industrial-strength programming
 language with an emphasis on expressiveness and safety. Developed for more
 than 20 years at Inria it benefits from one of the most advanced type systems
 and supports functional, imperative and object-oriented styles of
 programming.
 - [OpenBLAS/0.3.1-GCC-7.3.0-2.30](http://xianyi.github.com/OpenBLAS/)
OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version.
 - [OpenCV/3.1.0-foss-2016b](http://opencv.org/)
OpenCV (Open Source Computer Vision Library) is an open source computer vision
 and machine learning software library. OpenCV was built to provide
 a common infrastructure for computer vision applications and to accelerate
 the use of machine perception in the commercial products.
 - [OpenJDK/11.0.2](https://jdk.java.net/)
The official Reference Implementation for Java SE 11 (JSR 384)
 is based solely upon open-source code available from the JDK 11 Project in
 the OpenJDK Community.
 - [OpenJPEG/2.3.0-GCCcore-5.4.0](http://www.openjpeg.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-5.4.0.eb)
OpenJPEG is an open-source JPEG 2000 codec written in
 C language. It has been developed in order to promote the use of JPEG 2000,
 a still-image compression standard from the Joint Photographic Experts Group
 (JPEG). Since may 2015, it is officially recognized by ISO/IEC and ITU-T as
 a JPEG 2000 Reference Software.
 - [OpenMPI/3.1.4-GCC-8.3.0](https://www.open-mpi.org/)
The Open MPI Project is an open source MPI-3 implementation.
 - [OpenPGM/5.2.122-foss-2016b](http://code.google.com/p/openpgm/)
OpenPGM is an open source implementation of the Pragmatic General Multicast (PGM)
 specification in RFC 3208 available at www.ietf.org. PGM is a reliable and scalable multicast protocol
 that enables receivers to detect loss, request retransmission of lost data, or notify an application
 of unrecoverable loss. PGM is a receiver-reliable protocol, which means the receiver is responsible
 for ensuring all data is received, absolving the sender of reception responsibility.
 - [OpenSSL/1.1.0c-GCC-5.4.0-2.26](http://www.openssl.org/)
The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured,
 and Open Source toolchain implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1) 
 protocols as well as a full-strength general purpose cryptography library. 
 - [Osi/0.108.4-foss-2016b](https://github.com/coin-or/Osi)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/o/Osi/Osi-0.108.4-foss-2016b.eb)
Osi (Open Solver Interface) provides an abstract base class to a generic linear programming (LP) solver, along with derived classes for specific solvers. Many applications may be able to use the Osi to insulate themselves from a specific LP solver.
 - [PAINTOR/3.1-GCCcore-7.3.0](https://github.com/gkichaev/PAINTOR_V3.0)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PAINTOR/PAINTOR-3.1-GCCcore-7.3.0.eb)
PAINTOR Probabilistic Annotation INtegraTOR. PAINTOR is a statistical
 fine-mapping method that integrates functional genomic data with association strength
 from potentially multiple populations (or traits) to prioritize variants for follow-up
 analysis
 - [PANDAseq/2.11-foss-2016b](https://github.com/neufeld/pandaseq)
PANDAseq assembles Illumina Solexa overlapping pair-end reads.
 - [PCRE/8.41-GCCcore-7.3.0](http://www.pcre.org/)
The PCRE library is a set of functions that implement regular expression
 pattern matching using the same syntax and semantics as Perl 5.

 - [PCRE2/10.30-foss-2016b](http://www.pcre.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PCRE2/PCRE2-10.30-foss-2016b.eb)
The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax
 and semantics as Perl 5.

 - [PEAR/0.9.11-foss-2016b](http://sco.h-its.org/exelixis/web/software/pear/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PEAR/PEAR-0.9.11-foss-2016b.eb)
PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. 
 It is fully parallelized and can run with as low as just a few kilobytes of memory.
 - [PHASE/2.1.1](http://stephenslab.uchicago.edu/phase/download.html)
The program PHASE implements a Bayesian statistical 
method for reconstructing haplotypes from population genotype data.
Documentation: http://stephenslab.uchicago.edu/assets/software/phase/instruct2.1.pdf
 - [PICRUSt2/2.3.0-b-foss-2016b-Python-3.7.4](https://github.com/picrust/picrust2/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PICRUSt2/PICRUSt2-2.3.0-b-foss-2016b-Python-3.7.4.eb)
PICRUSt2 (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a software for predicting functional abundances based only on marker gene sequences. Check out the pre-print here.
 - [PRINSEQ/0.20.4-foss-2016b-Perl-5.28.0](http://prinseq.sourceforge.net)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PRINSEQ/PRINSEQ-0.20.4-foss-2016b-Perl-5.28.0.eb)
A bioinformatics tool to PRe-process and show INformation of SEQuence data.
 - [PROJ/6.0.0-GCCcore-5.4.0](http://trac.osgeo.org/proj/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PROJ/PROJ-6.0.0-GCCcore-5.4.0.eb)
Program proj is a standard Unix filter function which converts 
geographic longitude and latitude coordinates into cartesian coordinates
 - [Pandoc/2.7.3](http://pandoc.org)
If you need to convert files from one markup format into another, pandoc is your swiss-army knife
 - [Pango/1.42.4-foss-2018b](http://www.pango.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/Pango/Pango-1.42.4-foss-2018b.eb)
Pango is a library for laying out and rendering of text, with an emphasis on internationalization.
Pango can be used anywhere that text layout is needed, though most of the work on Pango so far has been done in the
context of the GTK+ widget toolkit. Pango forms the core of text and font handling for GTK+-2.x.
 - [ParMETIS/4.0.3-foss-2016b](http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/ParMETIS/ParMETIS-4.0.3-foss-2016b.eb)
ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning
 unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the
 functionality provided by METIS and includes routines that are especially suited for parallel AMR computations and
 large scale numerical simulations. The algorithms implemented in ParMETIS are based on the parallel multilevel k-way
 graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning schemes.
 - [Perl/5.28.0-GCCcore-7.3.0-fh1](http://www.perl.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/Perl/Perl-5.28.0-GCCcore-7.3.0-fh1.eb)
Fred Hutch Custom Perl
 - [Pillow/6.0.0-foss-2016b-Python-3.7.4](http://pillow.readthedocs.org/)
Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
 PIL is the Python Imaging Library by Fredrik Lundh and Contributors.
 - [Pindel/0.2.5b8-foss-2016b](http://gmt.genome.wustl.edu/packages/pindel/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/Pindel/Pindel-0.2.5b8-foss-2016b.eb)
Pindel can detect breakpoints of large deletions, medium sized 
 insertions, inversions, tandem duplications and other structural variants at single-based 
 resolution from next-gen sequence data. It uses a pattern growth approach to identify the 
 breakpoints of these variants from paired-end short reads. 
 - [PostgreSQL/10.1-foss-2016b](http://www.mysql.com/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PostgreSQL/PostgreSQL-10.1-foss-2016b.eb)
PostgreSQL is a powerful, open source object-relational database system.
 It is fully ACID compliant, has full support for foreign keys,
 joins, views, triggers, and stored procedures (in multiple languages).
 It includes most SQL:2008 data types, including INTEGER,
 NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP.
 It also supports storage of binary large objects, including pictures,
 sounds, or video. It has native programming interfaces for C/C++, Java,
 .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation.
 - [PyCairo/1.18.0-foss-2016b-Python-3.7.4](http://cairographics.org/pycairo/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PyCairo/PyCairo-1.18.0-foss-2016b-Python-3.7.4.eb)
Python bindings for the cairo library
 - [PyTables/3.6.1-foss-2016b-Python-3.7.4](http://www.pytables.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PyTables/PyTables-3.6.1-foss-2016b-Python-3.7.4.eb)
PyTables is a package for managing hierarchical datasets and designed to efficiently and easily cope
 with extremely large amounts of data. PyTables is built on top of the HDF5 library, using the Python language and the
 NumPy package. It features an object-oriented interface that, combined with C extensions for the performance-critical
 parts of the code (generated using Cython), makes it a fast, yet extremely easy to use tool for interactively browse,
 process and search very large amounts of data. One important feature of PyTables is that it optimizes memory and disk
 resources so that data takes much less space (specially if on-flight compression is used) than other solutions such as
 relational or object oriented databases.
 - [PyYAML/5.1.2-foss-2016b-Python-3.7.4](https://pypi.python.org/pypi/PyYAML/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PyYAML/PyYAML-5.1.2-foss-2016b-Python-3.7.4.eb)
PyYAML is a YAML parser and emitter for the Python programming language.
 - [Pysam/0.15.4-foss-2016b-Python-3.7.4](https://github.com/pysam-developers/pysam)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/Pysam/Pysam-0.15.4-foss-2016b-Python-3.7.4.eb)
Pysam is a python module for reading and manipulating Samfiles. 
 It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.
 - [Pysamstats/1.1.2-foss-2016b-Python-3.7.4](https://github.com/alimanfoo/pysamstats)
A Python utility for calculating statistics against genome positions based on
 sequence alignments from a SAM or BAM file.
 - [Python/3.7.4-foss-2016b](http://python.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/Python/Python-3.7.4-foss-2016b.eb)
Python is a programming language that lets you work more quickly and integrate your systems
 more effectively.
 - [Qt/4.8.7-foss-2016b](http://qt.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/q/Qt/Qt-4.8.7-foss-2016b.eb)
Qt is a comprehensive cross-platform C++ application framework.
 - [Qt5/5.9.1-foss-2016b](http://qt.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/q/Qt5/Qt5-5.9.1-foss-2016b.eb)
Qt is a comprehensive cross-platform C++ application framework.
 - [Queue/3.5-Java-1.8.0_66](http://www.broadinstitute.org/gatk/ )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/q/Queue/Queue-3.5-Java-1.8.0_66.eb)
The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute   to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,  with a primary focus on variant discovery and genotyping as well as strong emphasis on   data quality assurance. Its robust architecture, powerful processing engine and   high-performance computing features make it capable of taking on projects of any size.
 - [R/3.6.2-foss-2016b-fh1](http://www.r-project.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/R/R-3.6.2-foss-2016b-fh1.eb)
R is a free software environment for statistical computing and graphics.
 - [R-bundle-Bioconductor/3.5-foss-2016b-R-3.4.0-fh1](http://www.r-project.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.5-foss-2016b-R-3.4.0-fh1.eb)
R is a free software environment for statistical computing and graphics.
 - [RAxML/8.2.11-foss-2016b-hybrid-avx2](https://github.com/stamatak/standard-RAxML)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/RAxML/RAxML-8.2.11-foss-2016b-hybrid-avx2.eb)
RAxML search algorithm for maximum likelihood based inference of phylogenetic trees.
 - [RAxML-NG/0.9.0-foss-2016b](https://github.com/amkozlov/raxml-ng)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/RAxML-NG/RAxML-NG-0.9.0-foss-2016b.eb)
RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML) optimality criterion. Its search heuristic is based on iteratively performing a series of Subtree Pruning and Regrafting (SPR) moves, which allows to quickly navigate to the best-known ML tree. RAxML-NG is a successor of RAxML (Stamatakis 2014) and leverages the highly optimized likelihood computation implemented in libpll (Flouri et al. 2014).
 - [RELION/3.0_beta-foss-2018b](http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/RELION/RELION-3.0_beta-foss-2018b.eb)
RELION (for REgularised LIkelihood OptimisatioN, pronounce
rely-on) is a stand-alone computer program that employs an empirical Bayesian
approach to refinement of (multiple) 3D reconstructions or 2D class averages in
electron cryo-microscopy (cryo-EM).
 - [RNA-SeQC/2.3.4-foss-2016b](https://software.broadinstitute.org/cancer/cga/rna-seqc)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/RNA-SeQC/RNA-SeQC-2.3.4-foss-2016b.eb)
RNA-SeQC is a java program which computes a series of quality control metrics for
 RNA-seq data. The input can be one or more BAM files. The output consists of HTML reports and tab
 delimited files of metrics data. This program can be valuable for comparing sequencing quality
 across different samples or experiments to evaluate different experimental parameters. It can
 also be run on individual samples as a means of quality control before continuing with downstream
 analysis.
 - [RSeQC/3.0.0-foss-2016b-Python-3.6.6](http://rseqc.sourceforge.net/)
RSeQC provides a number of useful modules that can 
 comprehensively evaluate high throughput sequence data especially RNA-seq 
 data. Some basic modules quickly inspect sequence quality, nucleotide 
 composition bias, PCR bias and GC bias, while RNA-seq specific modules 
 evaluate sequencing saturation, mapped reads distribution, coverage 
 uniformity, strand specificity, transcript level RNA integrity etc.
 - [RepeatMasker/4-0-7-foss-2016b](www.repeatmasker.org)
RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences. The output of the program is a detailed annotation of the repeats that are present in the query sequence as well as a modified version of the query sequence in which all the annotated repeats have been masked
 - [ResMap/1.1.4-linux64](http://resmap.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/ResMap/ResMap-1.1.4-linux64.eb)
ResMap (Resolution Map) is a Python (NumPy/SciPy) application
with a Tkinter GUI. It is an easy to use software package for computing the
local resolution of 3D density maps studied in structural biology, primarily
electron cryo-microscopy (cryo-EM).
 - [RevBayes/1.0.11-foss-2016b](https://revbayes.github.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/RevBayes/RevBayes-1.0.11-foss-2016b.eb)
RevBayes provides an interactive environment for statistical
 computation in phylogenetics. It is primarily intended for modeling,
 simulation, and Bayesian inference in evolutionary biology, particularly
 phylogenetics.
 - [Rkeras/3.6.0-foss-2016b](http://www.r-project.org/)
R is a free software environment for statistical computing and graphics.
 - [Ruby/2.3.1](https://www.ruby-lang.org )
Ruby is a dynamic, open source programming language with   a focus on simplicity and productivity. It has an elegant syntax that is   natural to read and easy to write.
 - [SAMtools/1.10-foss-2016b](http://www.htslib.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SAMtools/SAMtools-1.10-foss-2016b.eb)
SAM Tools provide various utilities for manipulating alignments in the SAM format, 
 including sorting, merging, indexing and generating alignments in a per-position format.
 - [SCons/3.0.4-foss-2016b-Python-3.6.6](http://www.scons.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SCons/SCons-3.0.4-foss-2016b-Python-3.6.6.eb)
SCons is a software construction tool.
 - [SPRING_dev/1.6-foss-2016b-Python-2.7.15](http://https://github.com/AllonKleinLab/SPRING_dev)
SPRING is a collection of pre-processing scripts and a web
browser-based tool for visualizing and interacting with high dimensional data.
View an example dataset here. SPRING was developed for single cell RNA-Seq data
but can be applied more generally. The minimal input is a matrix of high
dimensional data points (cells) and a list of dimension names (genes).
 - [SQLite/3.24.0-GCCcore-7.3.0](http://www.sqlite.org/)
SQLite: SQL Database Engine in a C Library
 - [SRA-Toolkit/2.10.4-ubuntu64](http://trace.ncbi.nlm.nih.gov/Traces/sra)
The NCBI SRA Toolkit enables reading (dumping) of sequencing files 
 from the SRA database and writing (loading) files into the .sra format 
 - [STAR/2.7.3a-foss-2016b](https://github.com/alexdobin/STAR)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/STAR/STAR-2.7.3a-foss-2016b.eb)
STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.
 - [STAR-Fusion/1.5.0-foss-2016b-Perl-5.28.0](https://github.com/STAR-Fusion/STAR-Fusion)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/STAR-Fusion/STAR-Fusion-1.5.0-foss-2016b-Perl-5.28.0.eb)
STAR-Fusion uses the STAR aligner to identify candidate fusion transcripts 
 supported by Illumina reads. STAR-Fusion further processes the output generated by the STAR aligner 
 to map junction reads and spanning reads to a reference annotation set.
 - [SUGEN/8.3-foss-2016b](https://github.com/dragontaoran/SUGEN/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SUGEN/SUGEN-8.3-foss-2016b.eb)
SUGEN is a command-line software program written in C++ to implement the weighted and unweighted approaches described by Lin et al. (2014) for various types of association analysis under complex survey sampling. The current version of the program can accommodate continuous, binary, and right-censored time-to-event traits.
 - [SWIG/4.0.0-foss-2016b-Python-3.7.4](http://www.swig.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SWIG/SWIG-4.0.0-foss-2016b-Python-3.7.4.eb)
SWIG is a software development tool that connects programs written in C and C++ with
 a variety of high-level programming languages.
 - [SYMPHONY/5.6.17-foss-2016b](https://github.com/coin-or/SYMPHONY/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SYMPHONY/SYMPHONY-5.6.17-foss-2016b.eb)
SYMPHONY is an open-source solver for mixed-integer linear programs (MILPs) written in C. It can be used in four different main modes.
 - [Sablotron/1.0.3-goolf-1.5.14](http://sablotron.sourceforge.net/ )

 - [Salmon/1.2.0-foss-2016b](https://salmon.readthedocs.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/Salmon/Salmon-1.2.0-foss-2016b.eb)
Salmon is a wicked-fast program to produce a highly-accurate,
transcript-level quantification estimates from RNA-seq data. Salmon achieves
its accuracy and speed via a number of different innovations, including the use
of selective-alignment, and massively-parallel stochastic collapsed variational
inference.
 - [ScaLAPACK/2.0.2-gompi-2016a-OpenBLAS-0.2.15-LAPACK-3.6.0](http://www.netlib.org/scalapack/ )
The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines  redesigned for distributed memory MIMD parallel computers.
 - [SciPy-bundle/2020.04-foss-2016b-Python-3.7.4](https://python.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SciPy-bundle/SciPy-bundle-2020.04-foss-2016b-Python-3.7.4.eb)
Bundle of Python packages for scientific software
 - [Set-IntervalTree/0.12-foss-2016b-Perl-5.24.0](https://metacpan.org/pod/Set::IntervalTree)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/Set-IntervalTree/Set-IntervalTree-0.12-foss-2016b-Perl-5.24.0.eb)
Perform range-based lookups on sets of ranges
 - [Singularity/3.5.3](http://gmkurtzer.github.io/singularity)
Singularity is a portable application stack packaging and runtime utility.
 - [Sniffles/1.0.8-foss-2016b](https://github.com/fritzsedlazeck/Sniffles)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/Sniffles/Sniffles-1.0.8-foss-2016b.eb)
Sniffles is a structural variation caller using third generation sequencing (PacBio or Oxford Nanopore). It detects all types of SVs (10bp+) using evidence from split-read alignments, high-mismatch regions, and coverage analysis.
 - [Spark/2.2.0-Hadoop-2.6-Java-1.8.0_92](http://spark.apache.org)
Spark is Hadoop MapReduce done in memory
 - [Sphinx/1.3.3-foss-2016a-Python-2.7.12](http://sphinx.pocoo.org/ )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/Sphinx/Sphinx-1.3.3-foss-2016a-Python-2.7.12.eb)
Sphinx is a tool that makes it easy to create intelligent and beautiful documentation.  It was originally created for the new Python documentation, and it has excellent facilities for the   documentation of Python projects, but C/C++ is already supported as well, and it is planned to add   special support for other languages as well.
 - [Statistics-R/0.34-foss-2016b-Perl-5.28.0](https://metacpan.org/pod/Statistics::R)
Statistics::R is a module that lets you start R, pass commands to it and retrieve their output. A shared mode allows several instances of Statistics::R to talk to the same R process.
 - [SuiteSparse/5.1.0-foss-2016b-ParMETIS-4.0.3](http://faculty.cse.tamu.edu/davis/suitesparse.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SuiteSparse/SuiteSparse-5.1.0-foss-2016b-ParMETIS-4.0.3.eb)
SuiteSparse is a collection of libraries manipulate sparse matrices.
 - [Szip/2.1.1-GCCcore-7.3.0](http://www.hdfgroup.org/doc_resource/SZIP/)
Szip compression software, providing lossless compression of scientific data

 - [TRF/4.09-linux64-fh](https://tandem.bu.edu/trf/trf.html)
Tandem repeats finder: a program to analyze DNA sequences. Legacy version.
 - [Tcl/8.6.8-GCCcore-7.3.0](http://www.tcl.tk/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/Tcl/Tcl-8.6.8-GCCcore-7.3.0.eb)
Tcl (Tool Command Language) is a very powerful but easy to learn dynamic
 programming language, suitable for a very wide range of uses, including web
 and desktop applications, networking, administration, testing and many more.

 - [TensorFlow/2.1.0-foss-2016b-Python-3.7.4](https://www.tensorflow.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/TensorFlow/TensorFlow-2.1.0-foss-2016b-Python-3.7.4.eb)
An open-source software library for Machine Intelligence
 - [TensorFlowCUDA/1.13.1-foss-2016b-Python-3.6.6](https://www.tensorflow.org/)
An open-source software library for Machine Intelligence
 - [Theano/1.0.4-foss-2016b-Python-3.7.4](http://deeplearning.net/software/theano)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/Theano/Theano-1.0.4-foss-2016b-Python-3.7.4.eb)
Theano is a Python library that allows you to define, optimize,
and evaluate mathematical expressions involving multi-dimensional arrays efficiently.
 - [Tk/8.6.8-GCCcore-7.3.0](http://www.tcl.tk/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/Tk/Tk-8.6.8-GCCcore-7.3.0.eb)
Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for
 building a graphical user interface (GUI) in many different programming languages.
 - [Tkinter/3.7.4-foss-2016b-Python-3.7.4](http://python.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/Tkinter/Tkinter-3.7.4-foss-2016b-Python-3.7.4.eb)
Tkinter module, built with the Python buildsystem
 - [TopHat/2.1.2-foss-2016b](http://ccb.jhu.edu/software/tophat/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/TopHat/TopHat-2.1.2-foss-2016b.eb)
TopHat is a fast splice junction mapper for RNA-Seq reads.
 - [TraCeR/0.6.0-foss-2016b-Python-3.7.4](https://github.com/Teichlab/tracer)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/TraCeR/TraCeR-0.6.0-foss-2016b-Python-3.7.4.eb)
TraCeR reconstructs the sequences of rearranged and expressed T cell receptor
 genes from single-cell RNA-seq data. It then uses the TCR sequences to identify cells that
 have the same receptor sequences and so derive from the same original clonally-expanded cell.
 - [Tracer/v1.6](http://beast.bio.ed.ac.uk/tracer )

 - [Trimmomatic/0.39-Java-1.8](http://www.usadellab.org/cms/?page=trimmomatic)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/Trimmomatic/Trimmomatic-0.39-Java-1.8.eb)
Trimmomatic performs a variety of useful trimming tasks for illumina 
 paired-end and single ended data.The selection of trimming steps and their associated 
 parameters are supplied on the command line. 
 - [Trinity/2.5.1-foss-2016b](http://trinityrnaseq.github.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/Trinity/Trinity-2.5.1-foss-2016b.eb)
Trinity represents a novel method for the efficient and robust de novo reconstruction
 of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,
 Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads.
 - [UDUNITS/2.2.25-foss-2016b](https://www.unidata.ucar.edu/downloads/udunits/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/u/UDUNITS/UDUNITS-2.2.25-foss-2016b.eb)
The UDUNITS package supports units of physical quantities. Its C library provides for arithmetic manipulation of units and for conversion of numeric values between compatible units.
 - [UMI-tools/1.0.0-foss-2016b-Python-3.7.4](https://umi-tools.readthedocs.io)
Tools for handling Unique Molecular Identifiers in NGS data sets
 - [Unicycler/0.4.8-foss-2016b-Python-3.7.4](https://github.com/rrwick/Unicycler)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/u/Unicycler/Unicycler-0.4.8-foss-2016b-Python-3.7.4.eb)
Unicycler is an assembly pipeline for bacterial genomes. It can assemble Illumina-only read sets where it functions as a SPAdes-optimiser. It can also assembly long-read-only sets (PacBio or Nanopore) where it runs a miniasm+Racon pipeline. For the best possible assemblies, give it both Illumina reads and long reads, and it will conduct a hybrid assembly.
 - [VCFtools/0.1.15-foss-2016b-Perl-5.24.1](https://vcftools.github.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/v/VCFtools/VCFtools-0.1.15-foss-2016b-Perl-5.24.1.eb)
The aim of VCFtools is to provide 
 easily accessible methods for working with complex 
 genetic variation data in the form of VCF files.
 - [VEP/96.0-foss-2016b-Perl-5.24.0](https://www.ensembl.org/info/docs/tools/vep)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/v/VEP/VEP-96.0-foss-2016b-Perl-5.24.0.eb)
Variant Effect Predictor (VEP) determines the effect of your variants (SNPs, insertions, deletions,
 CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions.
 - [VariationHunter/0.4-foss-2016b](http://variationhunter.sourceforge.net/Home )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/v/VariationHunter/VariationHunter-0.4-foss-2016b.eb)
Variation Hunter
 - [ViennaRNA/2.4.13-foss-2016b-Python-3.6.6](http://www.tbi.univie.ac.at/RNA/)
The Vienna RNA Package consists of a C code library and several
stand-alone programs for the prediction and comparison of RNA secondary structures.
 - [Wandy/Feb2_2015-foss-2016b-Java-1.8.0_92](http://bioinformaticstools.mayo.edu/research/wandy/ )
Wandy is designed for Copy Number Variation (CNV) and Aneuploidy detection from large genomes such as human. It takes a sorted BAM file as input and report predicted chromosome regions that have amplifications or deletions using LOG2 ratio, generate graphic reports.
 - [X11/20180604-GCCcore-7.3.0](https://www.x.org)
The X Window System (X11) is a windowing system for bitmap displays
 - [XKeyboardConfig/2.17-foss-2016b](http://www.freedesktop.org/wiki/Software/XKeyboardConfig/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/x/XKeyboardConfig/XKeyboardConfig-2.17-foss-2016b.eb)
The non-arch keyboard configuration database for X Window.
 The goal is to provide the consistent, well-structured,
 frequently released open source of X keyboard configuration data
 for X Window System implementations (free, open source and commercial).
 The project is targeted to XKB-based systems.
 - [XML-LibXML/2.0132-GCCcore-7.3.0-Perl-5.28.0](http://search.cpan.org/src/SHLOMIF/XML-LibXML-2.0018/)
Perl binding for libxml2
 - [XML-Parser/2.44-foss-2016b-Perl-5.24.1](http://search.cpan.org/~toddr/XML-Parser-2.41/)
This is a Perl extension interface to James Clark's XML parser, expat.
 - [XZ/5.2.4-GCCcore-7.3.0](http://tukaani.org/xz/)
xz: XZ utilities
 - [Xerces-C++/3.2.0-foss-2016b](http://xerces.apache.org/xerces-c/)
Xerces-C++ is a validating XML parser written in a portable
subset of C++. Xerces-C++ makes it easy to give your application the ability to
read and write XML data. A shared library is provided for parsing, generating,
manipulating, and validating XML documents using the DOM, SAX, and SAX2
APIs.
 - [ZeroMQ/4.1.4-foss-2016b](http://www.zeromq.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/z/ZeroMQ/ZeroMQ-4.1.4-foss-2016b.eb)
ZeroMQ looks like an embeddable networking library but acts like a concurrency framework.
 It gives you sockets that carry atomic messages across various transports like in-process,
 inter-process, TCP, and multicast. You can connect sockets N-to-N with patterns like fanout,
 pub-sub, task distribution, and request-reply. It's fast enough to be the fabric for clustered
 products. Its asynchronous I/O model gives you scalable multicore applications, built as asynchronous
 message-processing tasks. It has a score of language APIs and runs on most operating systems.
 - [ack/2.22](http://beyondgrep.com/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/a/ack/ack-2.22.eb)
ack is a tool like grep, optimized for programmers
 - [ack/2.22](http://beyondgrep.com/)
ack is a tool like grep, optimized for programmers
 - [ant/1.10.1-Java-1.8.0_121](http://ant.apache.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/a/ant/ant-1.10.1-Java-1.8.0_121.eb)
Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files
 as targets and extension points dependent upon each other. The main known usage of Ant is the build of Java applications.
 - [anvio/6.1-foss-2016b-Python-3.7.4](https://merenlab.org/software/anvio/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/a/anvio/anvio-6.1-foss-2016b-Python-3.7.4.eb)
An analysis and visualization platform for 'omics data.
 - [argtable/2.13-foss-2016b](http://argtable.sourceforge.net/)
Argtable is an ANSI C library for parsing GNU style
 command line options with a minimum of fuss. 
 - [arpack-ng/3.4.0-foss-2016b](http://forge.scilab.org/index.php/p/arpack-ng/)
ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems.
 - [assembly-stats/1.0.0-foss-2016b](https://github.com/sanger-pathogens/assembly-stats)
Get assembly statistics from FASTA and FASTQ files
 - [ataqv/0.9.5-foss-2016b](https://github.com/ParkerLab/ataqv)
A toolkit for measuring and comparing ATAC-seq results, made in the
`Parker lab`_ at the University of Michigan. We wrote it to help us
understand how well our ATAC-seq assays had worked, and to make it
easier to spot differences that might be caused by library prep or
sequencing.
 - [autokeras/1.0.1-foss-2016b-Python-3.7.4](https://autokeras.com/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/a/autokeras/autokeras-1.0.1-foss-2016b-Python-3.7.4.eb)
AutoKeras: An AutoML system based on Keras. It is developed by DATA Lab at Texas A&M University. The goal of AutoKeras is to make machine learning accessible for everyone.
 - [awscli/1.18.23-foss-2016b-Python-3.7.4](https://pypi.python.org/pypi/awscli)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/a/awscli/awscli-1.18.23-foss-2016b-Python-3.7.4.eb)
Universal Command Line Environment for AWS. Includes package
awscli-plugin-endpiont
 - [azure-cli/2.0.20-foss-2016b-Python-2.7.14](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
Azures new command-line experience for managing Azure resources
 - [bam2fastx/1.3.0](https://github.com/pacificbiosciences/bam2fastx)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/bam2fastx/bam2fastx-1.3.0.eb)
Conversion of PacBio BAM files into gzipped fasta and fastq files, including splitting of barcoded data
 - [bam2wig/1.4](https://github.com/MikeAxtell/bam2wig)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/bam2wig/bam2wig-1.4.eb)
Conversion of a BAM alignment to wiggle and bigwig coverage files, with flexible reporting options.
 - [bamUtil/1.0.14-foss-2016a](http://genome.sph.umich.edu/wiki/BamUtil )
BamUtil is a repository that contains several programs   that perform operations on SAM/BAM files. All of these programs   are built into a single executable, bam.
 - [bcl2fastq2/2.20.0-foss-2016b](https://support.illumina.com/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/bcl2fastq2/bcl2fastq2-2.20.0-foss-2016b.eb)
Illumina conversion Software bcl2fastq can be used to
both demultiplex data and convert BCL files to FASTQ file formats for
downstream analysis.
 - [beagle-lib/3.1.2-GCCcore-5.4.0](https://github.com/beagle-dev/beagle-lib)
beagle-lib is a high-performance library that can perform the core calculations at the heart of most
 Bayesian and Maximum Likelihood phylogenetics packages.
 - [binutils/2.32-GCCcore-8.3.0](https://directory.fsf.org/project/binutils/)
binutils: GNU binary utilities
 - [biobambam2/2.0.95-foss-2016b](https://github.com/gt1/biobambam2)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/biobambam2/biobambam2-2.0.95-foss-2016b.eb)
Tools for processing BAM files; 
 bamsormadup, bamcollate2, bammarkduplicates, bammaskflags, bamrecompress,
 bamsort, bamtofastq
 - [bokeh/1.4.0-foss-2016b-Python-3.7.4](https://github.com/bokeh/bokeh)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/bokeh/bokeh-1.4.0-foss-2016b-Python-3.7.4.eb)
Statistical and novel interactive HTML plots for Python
 - [breakdancer/1.4.5-foss-2014b](http://gmt.genome.wustl.edu/packages/breakdancer/index.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/breakdancer/breakdancer-1.4.5-foss-2014b.eb)
BreakDancerMax predicts five types of structural variants: insertions, deletions, inversions, inter- and intra-chromosomal translocations from next-generation short paired-end sequencing reads using read pairs that are mapped with unexpected separation distances or orientation.
 - [bubblewrap/0.3.1-foss-2016b](https://github.com/projectatomic/bubblewrap/)
The goal of bubblewrap is to run an application in a sandbox,
 where it has restricted access to parts of the operating system or user data
 such as the home directory.
 - [buildenv/default-foss-2016b](None)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/buildenv/buildenv-default-foss-2016b.eb)
This module sets a group of environment variables for compilers, linkers, maths libraries, etc., that
 you can use to easily transition between toolchains when building your software. To query the variables being set
 please use: module show <this module name>
 - [bustools/0.39.2-foss-2016b](http://bustools.github.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/bustools/bustools-0.39.2-foss-2016b.eb)
BUS format is a file format for single-cell RNA-seq data designed to facilitate the development of modular workflows for data processing.
 - [bx-python/0.8.2-foss-2016b-Python-3.6.6](https://github.com/bxlab/bx-python)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/bx-python/bx-python-0.8.2-foss-2016b-Python-3.6.6.eb)
The bx-python project is a Python library and associated set of scripts to allow for rapid
 implementation of genome scale analyses.
 - [bzip2/1.0.6-GCC-5.4.0-2.26](https://sourceware.org/bzip2)
bzip2 is a freely available, patent free, high-quality data compressor. It typically
 compresses files to within 10% to 15% of the best available techniques (the PPM family of statistical
 compressors), whilst being around twice as fast at compression and six times faster at decompression.
 - [cURL/7.66.0-GCCcore-5.4.0](https://curl.haxx.se)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/cURL/cURL-7.66.0-GCCcore-5.4.0.eb)
libcurl is a free and easy-to-use client-side URL transfer library,
 supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP,
 LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.
 libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP
 form based upload, proxies, cookies, user+password authentication (Basic,
 Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling
 and more.

 - [cairo/1.14.12-GCCcore-7.3.0](http://cairographics.org)
Cairo is a 2D graphics library with support for multiple output devices.
 Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers,
 PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB
 - [canu/1.8-foss-2016b](http://canu.readthedocs.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/canu/canu-1.8-foss-2016b.eb)
Canu is a fork of the Celera Assembler designed for
 high-noise single-molecule sequencing
 - [cas-offinder/2.4-foss-2016b]()
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/cas-offinder/cas-offinder-2.4-foss-2016b.eb)
Cas-OFFinder is OpenCL based, ultrafast and versatile program that searches for potential off-target sites of CRISPR/Cas-derived RNA-guided endonucleases (RGEN).
 - [cellranger/3.1.0](https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/cellranger/cellranger-3.1.0.eb)
Chromium Single Cell Software Suite is a set of software applications for analyzing and visualizing single cell 3’ RNA-seq data produced by the 10x Genomics Chromium Platform.
 - [cellranger-atac/1.1.0-foss-2016b](https://support.10xgenomics.com/single-cell-atac/software/overview/welcome)
The Chromium Single Cell ATAC Software Suite is a complete package for analyzing and visualizing single cell chromatin accessibility data produced by the Chromium Single Cell ATAC Solution on the 10x Chromium Platform.
 - [cisTEM/1.0.0-beta-bin-foss-2016b](https://cistem.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/cisTEM/cisTEM-1.0.0-beta-bin-foss-2016b.eb)
cisTEM is user-friendly software to process cryo-EM images of
macromolecular complexes and obtain high-resolution 3D reconstructions from
them.
 - [clustal-omega/1.2.4-foss-2016b](http://www.clustal.org/omega)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/clustal-omega/clustal-omega-1.2.4-foss-2016b.eb)
Clustal Omega is a multiple sequence alignment
 program for proteins. It produces biologically meaningful multiple
 sequence alignments of divergent sequences. Evolutionary relationships
 can be seen via viewing Cladograms or Phylograms 
 - [cromwell/49-Java-1.8](https://github.com/broadinstitute/cromwell)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/cromwell/cromwell-49-Java-1.8.eb)
Scientific workflow engine designed for simplicity & scalability.
 - [ctffind/4.1.10-foss-2018b](http://grigoriefflab.janelia.org/ctf)
CTFFIND is a programing for finding CTFs of electron
micrographs
 - [cutadapt/2.9-foss-2016b-Python-3.7.4](http://opensource.scilifelab.se/projects/cutadapt/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/c/cutadapt/cutadapt-2.9-foss-2016b-Python-3.7.4.eb)
Cutadapt finds and removes adapter sequences, primers, poly-A tails and
 other types of unwanted sequence from your high-throughput sequencing reads.
 - [cyrus-sasl/2.1.26-foss-2016b](https://www.cyrusimap.org/sasl/)
Simple Authentication and Security Layer (SASL) is a specification that describes how authentication mechanisms can be plugged into an application protocol on the wire. Cyrus SASL is an implementation of SASL that makes it easy for application developers to integrate authentication mechanisms into their application in a generic way.
 - [damageproto/1.2.1-foss-2016a](http://www.freedesktop.org/wiki/Software/xlibs )
X damage extension headers
 - [dask/2.8.0-foss-2016b-Python-3.7.4](https://dask.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/d/dask/dask-2.8.0-foss-2016b-Python-3.7.4.eb)
Dask natively scales Python. Dask provides advanced parallelism for analytics, enabling performance
 at scale for the tools you love.
 - [deepTools/3.3.1-foss-2016b-Python-3.7.4](https://deeptools.readthedocs.io/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/d/deepTools/deepTools-3.3.1-foss-2016b-Python-3.7.4.eb)
deepTools is a suite of python tools particularly developed for the efficient analysis of
 high-throughput sequencing data, such as ChIP-seq, RNA-seq or MNase-seq.
 - [delly/0.7.6-foss-2016b](https://github.com/tobiasrausch/delly)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/d/delly/delly-0.7.6-foss-2016b.eb)
DELLY2: Structural variant discovery by integrated paired-end and split-read analysis
 - [expat/2.2.5-GCCcore-7.3.0](http://expat.sourceforge.net/)
Expat is an XML parser library written in C. It is a stream-oriented parser
 in which an application registers handlers for things the parser might find
 in the XML document (like start tags)

 - [factera/1.4.4-foss-2016b-Perl-5.24.1](https://factera.stanford.edu/)
(Fusion And Chromosomal Translocation Enumeration and Recovery Algorithm) is a tool for detection of genomic fusions in paired-end targeted (or genome-wide) sequencing data.
 - [fast5/0.6.5](http://simpsonlab.github.io/2017/02/27/packing_fast5/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/fast5/fast5-0.6.5.eb)
A lightweight C++ library for accessing Oxford Nanopore
 Technologies sequencing data.
 - [fastqc/0.11.3](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/ )

 - [fermi-lite/0.1-foss-2016b](https://github.com/lh3/fermi-lite)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/fermi-lite/fermi-lite-0.1-foss-2016b.eb)
Fermi-lite is a standalone C library as well as a command-line tool
 for assembling Illumina short reads in regions from 100bp to 10 million bp in size. It
 is largely a light-weight in-memory version of fermikit without generating any intermediate
 files.
 - [fgbio/1.1.0-foss-2016b](https://github.com/fulcrumgenomics/fgbio)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/fgbio/fgbio-1.1.0-foss-2016b.eb)
A set of tools to analyze genomic data with a focus on Next Generation Sequencing.
 - [fiji/1.0.0-Java-1.8.0_66](http://imagej.net/Fiji )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/fiji/fiji-1.0.0-Java-1.8.0_66.eb)
Fiji is a distribution of ImageJ which includes many useful plugins contributed by the community.
 - [file/5.28-foss-2016a](http://www.darwinsys.com/file/ )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/file/file-5.28-foss-2016a.eb)
The file command is 'a file type guesser', that is, a command-line tool  that tells you in words what kind of data a file contains.
 - [fio/3.16-foss-2016b](https://fio.readthedocs.io/en/latest/fio_doc.html)
Fio spawns a number of threads or processes doing a
particular type of I/O action as specified by the user.
 - [fixesproto/5.0-goolf-1.7.20](http://www.freedesktop.org/wiki/Software/xlibs )

 - [flex/2.6.4-GCCcore-6.4.0](http://flex.sourceforge.net/)
Flex (Fast Lexical Analyzer) is a tool for generating scanners. A scanner, 
 sometimes called a tokenizer, is a program which recognizes lexical patterns
 in text.

 - [fontconfig/2.13.0-GCCcore-7.3.0](http://www.freedesktop.org/software/fontconfig)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/fontconfig/fontconfig-2.13.0-GCCcore-7.3.0.eb)
Fontconfig is a library designed to provide system-wide font configuration,
 customization and application access.

 - [fontsproto/2.1.3-foss-2016a](http://www.freedesktop.org/wiki/Software/xlibs )
X11 font extension wire protocol
 - [foss/2018b](https://easybuild.readthedocs.io/en/master/Common-toolchains.html#foss-toolchain)
GNU Compiler Collection (GCC) based compiler toolchain, including
 OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.
 - [fosscuda/2016b]((none))
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/fosscuda/fosscuda-2016b.eb)
GCC based compiler toolchain __with CUDA support__, and including
 OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.
 - [freeglut/3.0.0-foss-2016b](http://freeglut.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/freeglut/freeglut-3.0.0-foss-2016b.eb)
freeglut is a completely OpenSourced alternative to the OpenGL Utility Toolkit (GLUT) library.
 - [freetds/1.00.9-foss-2016b](http://www.freedesktop.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/freetds/freetds-1.00.9-foss-2016b.eb)
FreeTDS is a set of libraries for Unix and Linux that allows your programs to natively talk to Microsoft SQL Server and Sybase databases.
 - [freetype/2.9.1-GCCcore-7.3.0](http://freetype.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/freetype/freetype-2.9.1-GCCcore-7.3.0.eb)
FreeType 2 is a software font engine that is designed to be small, efficient,
 highly customizable, and portable while capable of producing high-quality
 output (glyph images). It can be used in graphics libraries, display servers,
 font conversion tools, text image generation tools, and many other products
 as well.

 - [gbench/2.12.0-foss-2016b](https://www.ncbi.nlm.nih.gov/tools/gbench/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/gbench/gbench-2.12.0-foss-2016b.eb)
NCBI Genome Workbench is an integrated application for viewing and analyzing sequence data. With Genome Workbench, you can view data in publically available sequence databases at NCBI, and mix this data with your own private data.
 - [gc/7.4.4-foss-2016b](http://hboehm.info/gc/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/gc/gc-7.4.4-foss-2016b.eb)
The Boehm-Demers-Weiser conservative garbage collector can be used as a garbage collecting 
 replacement for C malloc or C++ new.
 - [gcccuda/2016b]((none))
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/gcccuda/gcccuda-2016b.eb)
GNU Compiler Collection (GCC) based compiler toolchain, along with CUDA toolkit.
 - [gdc-client/1.5.0-foss-2016b-Python-3.7.4](https://gdc.nci.nih.gov/access-data/gdc-data-transfer-tool)
The gdc-client provides several convenience functions over the GDC API which provides general
 download/upload via HTTPS.
 - [generic-metal/2011-03-25-foss-2016b](http://csg.sph.umich.edu/abecasis/Metal/download/)
The METAL software is designed to facilitate meta-analysis of large datasets (such as several whole genome scans) in a convenient, rapid and memory efficient manner. 
 - [gettext/0.19.8.1](http://www.gnu.org/software/gettext/)
GNU 'gettext' is an important step for the GNU Translation Project, as it is an asset on which we may
build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools
and documentation
 - [ghostscript/9.21-foss-2016b](http://ghostscript.com)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/ghostscript/ghostscript-9.21-foss-2016b.eb)
Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
 different targets. It used to be part of the cups printing stack, but is no longer used for that.
 - [giflib/5.1.4-foss-2016b](http://libungif.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/giflib/giflib-5.1.4-foss-2016b.eb)
libungif: Tools and library routines for working with GIF images
 - [git/2.24.1-foss-2016b](http://git-scm.com/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/git/git-2.24.1-foss-2016b.eb)
Git is a free and open source distributed version control system designed
to handle everything from small to very large projects with speed and efficiency.
 - [glew/1.13.0-foss-2016b](http://glew.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/glew/glew-1.13.0-foss-2016b.eb)
GLEW provides efficient run-time mechanisms for determining which OpenGL extensions are supported on the target platform.
 - [glibc/2.26-GCCcore-5.4.0](https://www.gnu.org/software/libc/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/glibc/glibc-2.26-GCCcore-5.4.0.eb)
The GNU C Library project provides the core libraries for the GNU system and GNU/Linux systems,
 as well as many other systems that use Linux as the kernel.
 - [gnuplot/5.0.6-foss-2016b](http://gnuplot.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/gnuplot/gnuplot-5.0.6-foss-2016b.eb)
Portable interactive, function plotting utility
 - [gnutls/3.5.13-foss-2016b](http://www.gnutls.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/gnutls/gnutls-3.5.13-foss-2016b.eb)
gnutls-3.0.22: GNU Transport Layer Security library
 - [gompi/2019b]((none))
GNU Compiler Collection (GCC) based compiler toolchain,
 including OpenMPI for MPI support.
 - [gompic/2016b]((none))
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/gompic/gompic-2016b.eb)
GNU Compiler Collection (GCC) based compiler toolchain along with CUDA toolkit,
 including OpenMPI for MPI support with CUDA features enabled.
 - [goolf/1.7.20]((none) )
GNU Compiler Collection (GCC) based compiler toolchain, including  OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.
 - [gperf/3.1-GCCcore-7.3.0](http://www.gnu.org/software/gperf/)
GNU gperf is a perfect hash function generator. For a given list of strings,
 it produces a hash function and hash table, in form of C or C++ code, for
 looking up a value depending on the input string. The hash function is
 perfect, which means that the hash table has no collisions, and the hash
 table lookup needs a single string comparison only.

 - [graphite2/1.3.10-foss-2016b](http://graphite.sil.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/graphite2/graphite2-1.3.10-foss-2016b.eb)
Graphite is a "smart font" system developed specifically to handle the complexities of lesser-known languages of the world.
 - [gsutil/4.50-foss-2016b-Python-3.7.4](https://cloud.google.com/storage/docs/gsutil)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/gsutil/gsutil-4.50-foss-2016b-Python-3.7.4.eb)
gsutil is a Python application that lets you access Cloud Storage from the command line.
 - [gtest/1.8.1-GCCcore-5.4.0](https://github.com/google/googletest)
Google's framework for writing C++ tests on a variety of platforms
 - [guile/1.8.8-foss-2016a](http://www.gnu.org/software/guile )
Guile is the GNU Ubiquitous Intelligent Language for Extensions,  the official extension language for the GNU operating system.
 - [gurobi/8.1.0](https://gurobi.com)
The Gurobi Optimizer allows users to state their toughest
 business problems as mathematical models, and then automatically considers
 billions or even trillions of possible solutions to find the best one. Our
 solver can be used as both a decision-making assistant, to help guide the
 choices of a skilled expert, or as a fully automated tool to make decisions
 with no human intervention.
 - [gzip/1.8-GCCcore-5.4.0](http://www.gnu.org/software/gzip/)
gzip (GNU zip) is a popular data compression program as a replacement for compress
 - [h5py/2.10.0-foss-2016b-Python-3.7.4](http://www.h5py.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/h/h5py/h5py-2.10.0-foss-2016b-Python-3.7.4.eb)
HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,
 version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous
 amounts of data.
 - [help2man/1.47.8-GCCcore-8.3.0](https://www.gnu.org/software/help2man/)
help2man produces simple manual pages from the '--help' and '--version' output of other commands.
 - [hivmmer/0.1.2-foss-2016b-Python-3.6.6](https://github.com/kantorlab/hivmmer)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/h/hivmmer/hivmmer-0.1.2-foss-2016b-Python-3.6.6.eb)
An alignment and variant-calling pipeline for Illumina deep
 sequencing of HIV-1, based on the probabilistic aligner HMMER
 - [hmmcopy/master-foss-2016b](https://github.com/shahcompbio/hmmcopy_utils/tree/master)
Tools for extracting read counts and gc and mappability statistics in preparation for running HMMCopy.
 - [hwloc/1.11.12-GCCcore-8.3.0](https://www.open-mpi.org/projects/hwloc/)
The Portable Hardware Locality (hwloc) software package provides a portable
 abstraction (across OS, versions, architectures, ...) of the hierarchical
 topology of modern architectures, including NUMA memory nodes, sockets, shared
 caches, cores and simultaneous multithreading. It also gathers various system
 attributes such as cache and memory information as well as the locality of I/O
 devices such as network interfaces, InfiniBand HCAs or GPUs. It primarily
 aims at helping applications with gathering information about modern computing
 hardware so as to exploit it accordingly and efficiently.

 - [icc/2016.1.150-GCC-4.9.3-2.25](http://software.intel.com/en-us/intel-compilers/ )
C and C++ compiler from Intel
 - [iccifort/2016.1.150-GCC-4.9.3-2.25](http://software.intel.com/en-us/intel-cluster-toolkit-compiler/ )
Intel Cluster Toolkit Compiler Edition provides Intel C,C++ and fortran compilers, Intel MPI and Intel MKL
 - [ifort/2016.1.150-GCC-4.9.3-2.25](http://software.intel.com/en-us/intel-compilers/ )
C and C++ compiler from Intel
 - [igraph/1.2.4-foss-2016b-R-3.5.2](https://igraph.org/r/)
igraph is a collection of network analysis tools with the emphasis on 
efficiency, portability and ease of use. igraph is open source and free. igraph can be 
programmed in R, Python and C/C++.
 - [igrec/3.1.1-linux](https://github.com/yana-safonova/ig_repertoire_constructor)

 - [iimpi/8.1.5-GCC-4.9.3-2.25](http://software.intel.com/en-us/intel-cluster-toolkit-compiler/ )
Intel C/C++ and Fortran compilers, alongside Intel MPI.
 - [imake/1.0.5-goolf-1.5.14](http://www.x.org/ )

 - [imkl/11.3.1.150-iimpi-8.1.5-GCC-4.9.3-2.25](http://software.intel.com/en-us/intel-mkl/ )
Intel Math Kernel Library is a library of highly optimized,  extensively threaded math routines for science, engineering, and financial  applications that require maximum performance. Core math functions include  BLAS, LAPACK, ScaLAPACK, Sparse Solvers, Fast Fourier Transforms, Vector Math, and more.
 - [impi/5.1.2.150-iccifort-2016.1.150-GCC-4.9.3-2.25](http://software.intel.com/en-us/intel-mpi-library/ )
The Intel(R) MPI Library for Linux* OS is a multi-fabric message  passing library based on ANL MPICH2 and OSU MVAPICH2. The Intel MPI Library for  Linux OS implements the Message Passing Interface, version 2 (MPI-2) specification.
 - [imrep/0.8-foss-2016b](https://github.com/mandricigor/imrep/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/imrep/imrep-0.8-foss-2016b.eb)
ImReP is a method for rapid and accurate profiling of the adaptive immune repertoires from regular RNA-Seq data
 - [inputproto/2.3.1-foss-2016a](http://www.freedesktop.org/wiki/Software/xlibs )
X.org InputProto protocol headers.
 - [intel/2016a](http://software.intel.com/en-us/intel-cluster-toolkit-compiler/ )
Intel Cluster Toolkit Compiler Edition provides Intel C/C++ and Fortran compilers, Intel MPI & Intel MKL.
 - [interpret/0.1.20-foss-2016b-Python-3.7.4](https://github.com/interpretml/interpret)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/i/interpret/interpret-0.1.20-foss-2016b-Python-3.7.4.eb)
nterpretML is an open-source python package for training interpretable machine learning models and explaining blackbox systems.
 - [intltool/0.51.0-GCCcore-7.3.0-Perl-5.28.0](http://freedesktop.org/wiki/Software/intltool/)
intltool is a set of tools to centralize translation of
 many different file formats using GNU gettext-compatible PO files.
 - [intro-bio-bundle/2.0-foss-2016b-fh1](http://scicomp.fhcrc.org )
Bundle used for CompBio teaching at FredHutch.
 - [jbigkit/2.1-GCCcore-5.4.0]()
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/j/jbigkit/jbigkit-2.1-GCCcore-5.4.0.eb)
JBIG-KIT is a software implementation of the JBIG1 data
 compression standard (ITU-T T.82), which was designed for bi-level image
 data, such as scanned documents.
 - [jemalloc/5.2.0-GCCcore-5.4.0](http://www.canonware.com/jemalloc)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/j/jemalloc/jemalloc-5.2.0-GCCcore-5.4.0.eb)
jemalloc is a general purpose malloc(3) implementation that
 emphasizes fragmentation avoidance and scalable concurrency support.
 - [julia/1.0.3-foss-2016b](http://www.julialang.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/j/julia/julia-1.0.3-foss-2016b.eb)
Julia is a high-level, high-performance dynamic programming
language for technical computing, with syntax that is familiar to users of
other technical computing environments.
 - [jupyter/1.0.0-foss-2016b-Python-3.7.4](http://jupyter.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/j/jupyter/jupyter-1.0.0-foss-2016b-Python-3.7.4.eb)
JupyterLab is a web-based interactive development environment for Jupyter
 notebooks, code, and data.
 - [kallisto/0.46.0-foss-2016b](http://pachterlab.github.io/kallisto/)
kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally
 of target sequences using high-throughput sequencing reads.
 - [kbproto/1.0.7-intel-2016a](http://www.freedesktop.org/wiki/Software/xlibs )
X.org KBProto protocol headers.
 - [kentUtils/302.1.0-foss-2016b](http://genome.cse.ucsc.edu/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/k/kentUtils/kentUtils-302.1.0-foss-2016b.eb)
Jim Kent tools: collection of tools used by the UCSC genome browser.
 - [klib/master](https://github.com/attractivechaos/klib)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/k/klib/klib-master.eb)
Klib strives for efficiency and a small memory footprint. Some components, such as hash table, B-tree, vector and sorting algorithms, are among the most efficient implementations of similar algorithms or data structures in all programming languages, in terms of both speed and memory use.
 - [lemon/1.3.1-foss-2016b](http://lemon.cs.elte.hu/trac/lemon)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/lemon/lemon-1.3.1-foss-2016b.eb)
LEMON stands for Library for Efficient Modeling and
 Optimization in Networks. It is a C++ template library providing efficient
 implementations of common data structures and algorithms with focus on
 combinatorial optimization tasks connected mainly with graphs and networks.
 - [leptonica/1.75.3-foss-2016b](https://www.leptonica.com/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/leptonica/leptonica-1.75.3-foss-2016b.eb)
Leptonica is useful for image processing and image analysis applications.
 - [lftp/4.6.4-foss-2016b](http://lftp.yar.ru)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/lftp/lftp-4.6.4-foss-2016b.eb)
LFTP is a sophisticated ftp/http client, and a file transfer program supporting
  a number of network protocols. Like BASH, it has job control and uses the readline library for
  input. It has bookmarks, a built-in mirror command, and can transfer several files in parallel.
  It was designed with reliability in mind.
 - [libGLU/9.0.0-foss-2016a-Mesa-11.1.4](ftp://ftp.freedesktop.org/pub/mesa/glu/ )
The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. 
 - [libICE/1.0.9-foss-2016b](http://www.freedesktop.org/wiki/Software/xlibs)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libICE/libICE-1.0.9-foss-2016b.eb)
X Inter-Client Exchange library for freedesktop.org
 - [libSM/1.2.2-foss-2016b](http://www.freedesktop.org/wiki/Software/xlibs)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libSM/libSM-1.2.2-foss-2016b.eb)
X11 Session Management library, which allows for applications to both manage sessions,
 and make use of session managers to save and restore their state for later use.
 - [libXau/1.0.8-intel-2016a](http://www.freedesktop.org/wiki/Software/xlibs )
The libXau package contains a library implementing the X11 Authorization Protocol. This is useful for restricting client access to the display.
 - [libXaw/1.0.12-goolf-1.7.20](http://www.freedesktop.org/wiki/Software/xlibs )

 - [libXcursor/1.1.14-foss-2016b](http://www.freedesktop.org/wiki/Software/xlibs)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libXcursor/libXcursor-1.1.14-foss-2016b.eb)
X Cursor management library
 - [libXdmcp/1.1.2-foss-2016a](http://www.freedesktop.org/wiki/Software/xlibs )
The libXdmcp package contains a library implementing the X Display Manager Control Protocol. This is useful for allowing clients to interact with the X Display Manager. 
 - [libXfixes/5.0.1-foss-2016b](http://www.freedesktop.org/wiki/Software/xlibs)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libXfixes/libXfixes-5.0.1-foss-2016b.eb)
X Fixes extension library
 - [libXfont/1.5.1-foss-2016a](http://www.freedesktop.org/wiki/Software/xlibs )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libXfont/libXfont-1.5.1-foss-2016a.eb)
X font libary
 - [libXft/2.3.2-foss-2016a](http://www.freedesktop.org/wiki/Software/xlibs )
X11 client-side library
 - [libXi/1.7.6-foss-2016b](http://www.freedesktop.org/wiki/Software/xlibs)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libXi/libXi-1.7.6-foss-2016b.eb)
LibXi provides an X Window System client interface to the XINPUT extension to the X protocol.
 - [libXinerama/1.1.3-foss-2016b](http://www.freedesktop.org/wiki/Software/xlibs)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libXinerama/libXinerama-1.1.3-foss-2016b.eb)
Xinerama multiple monitor library
 - [libXmu/1.1.2-goolf-1.5.14](http://www.freedesktop.org/wiki/Software/xlibs )

 - [libXp/1.0.3-foss-2016b](http://www.freedesktop.org/wiki/Software/xlibs)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libXp/libXp-1.0.3-foss-2016b.eb)
libXp provides the X print library.
 - [libXpm/3.5.11-goolf-1.5.14](http://www.freedesktop.org/wiki/Software/xlibs )

 - [libXrandr/1.5.0-foss-2016b](http://www.freedesktop.org/wiki/Software/xlibs)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libXrandr/libXrandr-1.5.0-foss-2016b.eb)
X Resize, Rotate and Reflection extension library
 - [libXrender/0.9.9-foss-2016b](http://www.freedesktop.org/wiki/Software/xlibs)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libXrender/libXrender-0.9.9-foss-2016b.eb)
X11 client-side library
 - [libXt/1.1.5-foss-2016b](http://www.freedesktop.org/wiki/Software/xlibs)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libXt/libXt-1.1.5-foss-2016b.eb)
libXt provides the X Toolkit Intrinsics, an abstract widget library upon which other toolkits are
 based. Xt is the basis for many toolkits, including the Athena widgets (Xaw), and LessTif (a Motif implementation).
 - [libarchive/3.3.2-GCC-5.4.0-2.26](https://www.libarchive.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libarchive/libarchive-3.3.2-GCC-5.4.0-2.26.eb)
Multi-format archive and compression library.
 - [libbsd/0.8.6-foss-2016b](https://libbsd.freedesktop.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libbsd/libbsd-0.8.6-foss-2016b.eb)
This library provides useful functions commonly found on BSD systems, and lacking on others like GNU systems, thus making it easier to port projects with strong BSD origins, without needing to embed the same code over and over again on each project. Part of freedesktop.org
 - [libcap/2.26-foss-2016b](https://libbsd.freedesktop.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libcap/libcap-2.26-foss-2016b.eb)
This library provides useful functions commonly found on BSD systems, and lacking on others like GNU systems, thus making it easier to port projects with strong BSD origins, without needing to embed the same code over and over again on each project. Part of freedesktop.org
 - [libcerf/1.5-foss-2016b](http://gnuplot.sourceforge.net/)
libcerf is a self-contained numeric library that provides an efficient and accurate
 implementation of complex error functions, along with Dawson, Faddeeva, and Voigt functions.
 - [libcroco/0.6.11-foss-2016b](http://www.gnome.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libcroco/libcroco-0.6.11-foss-2016b.eb)
The libcroco package contains a standalone CSS2 parsing and manipulation library
 - [libdrm/2.4.92-GCCcore-7.3.0](http://dri.freedesktop.org)
Direct Rendering Manager runtime library.
 - [libedit/20160903-foss-2016b](http://thrysoee.dk/editline/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libedit/libedit-20160903-foss-2016b.eb)
This BSD-style licensed command line editor library provides generic line editing,
history, and tokenization functions, similar to those found in GNU Readline.

 - [libevent/2.1.8-GCCcore-5.4.0](http://libevent.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libevent/libevent-2.1.8-GCCcore-5.4.0.eb)
The libevent API provides a mechanism to execute a callback function when a specific
 event occurs on a file descriptor or after a timeout has been reached.
 Furthermore, libevent also support callbacks due to signals or regular timeouts.
 - [libffcall/1.13-GCCcore-6.4.0](https://www.gnu.org/software/libffcall/)
GNU Libffcall is a collection of four libraries which can be used to build
 foreign function call interfaces in embedded interpreters

 - [libffi/3.2.1-foss-2016a](http://sourceware.org/libffi/ )
The libffi library provides a portable, high level programming interface to various calling conventions. This allows a programmer to call any function specified by a call interface description at run-time.
 - [libgcrypt/1.8.1-foss-2016b](https://gnupg.org/related_software/libgcrypt/index.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libgcrypt/libgcrypt-1.8.1-foss-2016b.eb)
Libgpg-error is a small library that defines common error values for all GnuPG components.
 - [libgd/2.2.5-GCCcore-7.3.0](https://libgd.github.io/)
GD is an open source code library for the dynamic creation of images by programmers.
 - [libgeotiff/1.4.2-foss-2016b](https://trac.osgeo.org/geotiff)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libgeotiff/libgeotiff-1.4.2-foss-2016b.eb)
GeoTIFF represents an effort by over 160 different remote sensing, GIS, cartographic, and surveying related companies and organizations to establish a ​TIFF based interchange format for georeferenced raster imagery.
 - [libglvnd/1.2.0-GCCcore-5.4.0](https://github.com/NVIDIA/libglvnd)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libglvnd/libglvnd-1.2.0-GCCcore-5.4.0.eb)
libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API calls between multiple vendors.
 - [libgpg-error/1.27-GCCcore-5.4.0](https://gnupg.org/related_software/libgpg-error/index.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libgpg-error/libgpg-error-1.27-GCCcore-5.4.0.eb)
Libgpg-error is a small library that defines common error values for all GnuPG components.
 - [libgtextutils/0.7-GCCcore-5.4.0](http://hannonlab.cshl.edu/fastx_toolkit/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libgtextutils/libgtextutils-0.7-GCCcore-5.4.0.eb)
ligtextutils is a dependency of fastx-toolkit and is provided
 via the same upstream
 - [libharu/2.3.0-foss-2016b](http://libharu.org/)
libHaru is a free, cross platform, open source library for generating PDF files.
 - [libiconv/1.15-GCCcore-5.4.0](https://www.gnu.org/software/libiconv)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libiconv/libiconv-1.15-GCCcore-5.4.0.eb)
Libiconv converts from one character encoding to another through Unicode conversion
 - [libidn/1.32-intel-2016a](http://www.gnu.org/software/libidn )
GNU Libidn is a fully documented implementation of the Stringprep, Punycode and IDNA specifications. Libidn's purpose is to encode and decode internationalized domain names.
 - [libjpeg-turbo/2.0.3-GCCcore-5.4.0](http://sourceforge.net/projects/libjpeg-turbo/)
libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to accelerate baseline JPEG
compression and decompression. libjpeg is a library that implements JPEG image encoding, decoding and transcoding.

 - [libmaus2/2.0.611-foss-2016b](https://github.com/gt1/libmaus2)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libmaus2/libmaus2-2.0.611-foss-2016b.eb)
libmaus2 is a collection of data structures and algorithms.
 - [libnl/3.4.0-GCCcore-5.4.0]()
The libnl suite is a collection of libraries providing APIs
 to netlink protocol based Linux kernel interfaces.
 - [libpciaccess/0.14-GCCcore-8.3.0](http://cgit.freedesktop.org/xorg/lib/libpciaccess/)
Generic PCI access library.
 - [libpipeline/1.5.0-foss-2016b](https://savannah.nongnu.org/projects/libpipeline)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libpipeline/libpipeline-1.5.0-foss-2016b.eb)
libpipeline is a C library for manipulating pipelines of subprocesses in a flexible and convenient way 
 - [libpll/0.3.2-GCCcore-5.4.0](https://github.com/xflouris/libpll/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libpll/libpll-0.3.2-GCCcore-5.4.0.eb)
libpll is a versatile high-performance software library for phylogenetic analysis.
 - [libpll-2/0.3.2-GCCcore-5.4.0](https://github.com/xflouris/libpll/wiki)
libpll is a versatile high-performance software library for phylogenetic analysis.
 - [libpng/1.6.34-GCCcore-7.3.0](http://www.libpng.org/pub/png/libpng.html)
libpng is the official PNG reference library
 - [libpqxx/5.0.1-foss-2016b](http://pqxx.org/development/libpqxx/)
libpqxx is the official C++ client API for PostgreSQL,
 the enterprise-strength open-source relational database.
 - [libreadline/7.0-GCCcore-6.4.0](http://cnswww.cns.cwru.edu/php/chet/readline/rltop.html)
The GNU Readline library provides a set of functions for use by applications
 that allow users to edit command lines as they are typed in. Both Emacs and
 vi editing modes are available. The Readline library includes additional
 functions to maintain a list of previously-entered command lines, to recall
 and perhaps reedit those lines, and perform csh-like history expansion on
 previous commands.

 - [librsvg/2.40.18-foss-2016b](http://www.libpng.org/pub/png/libpng.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/librsvg/librsvg-2.40.18-foss-2016b.eb)
The librsvg package contains a library and tools used to manipulate, convert and view Scalable Vector Graphic (SVG) images.
 - [libseccomp/2.4.2-foss-2016b](https://github.com/seccomp/libseccomp)
The libseccomp library provides an easy to use, platform independent, interface to the Linux Kernel's syscall filtering mechanism. The libseccomp API is designed to abstract away the underlying BPF based syscall filter language and present a more conventional function-call based filtering interface that should be familiar to, and easily adopted by, application developers.
 - [libsigsegv/2.11-GCCcore-6.4.0](https://www.gnu.org/software/libsigsegv/)
GNU libsigsegv is a library for handling page faults in user mode.
 - [libsndfile/1.0.28-foss-2016b](http://www.mega-nerd.com/libsndfile)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libsndfile/libsndfile-1.0.28-foss-2016b.eb)
Libsndfile is a C library for reading and writing files containing sampled sound
 (such as MS Windows WAV and the Apple/SGI AIFF format) through one standard library interface.
 - [libsodium/1.0.11-foss-2016b](http://doc.libsodium.org/)
Sodium is a modern, easy-to-use software library for encryption, decryption, signatures,
 password hashing and more.
 - [libssh/0.9.0-GCCcore-5.4.0](https://www.libssh.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libssh/libssh-0.9.0-GCCcore-5.4.0.eb)
Multiplatform C library implementing the SSHv2 protocol on client and server side
 - [libtasn1/4.9-foss-2016b](https://www.gnu.org/software/libtasn1/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libtasn1/libtasn1-4.9-foss-2016b.eb)
Libtasn1 is the ASN.1 library used by GnuTLS, GNU Shishi and some other packages.
 It was written by Fabio Fiorina, and has been shipped as part of GnuTLS
 for some time but is now a proper GNU package.
 - [libtool/2.4.6-foss-2016a](http://www.gnu.org/software/libtool )
GNU libtool is a generic library support script. Libtool hides the complexity of using shared libraries  behind a consistent, portable interface.
 - [libunistring/0.9.3-foss-2016a](http://www.gnu.org/software/libunistring/ )
This library provides functions for manipulating Unicode strings and for manipulating C strings  according to the Unicode standard.
 - [libwebp/1.0.0-foss-2016b](https://developers.google.com/speed/webp)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libwebp/libwebp-1.0.0-foss-2016b.eb)
WebP is a modern image format that provides superior
 lossless and lossy compression for images on the web. Using WebP, webmasters
 and web developers can create smaller, richer images that make the web
 faster.
 - [libxkbcommon/0.6.1-foss-2016b](http://xkbcommon.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libxkbcommon/libxkbcommon-0.6.1-foss-2016b.eb)
xkbcommon is a library to handle keyboard descriptions,
 including loading them from disk, parsing them and handling their state.
 It's mainly meant for client toolkits, window systems,
 and other system applications.
 - [libxml2/2.9.9-GCCcore-8.3.0](http://xmlsoft.org/)
Libxml2 is the XML C parser and toolchain developed for the Gnome project
 (but usable outside of the Gnome platform).

 - [libxslt/1.1.29-foss-2016b](http://xmlsoft.org/ )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/libxslt/libxslt-1.1.29-foss-2016b.eb)
Libxslt is the XSLT C library developed for the GNOME project  (but usable outside of the Gnome platform).
 - [libyaml/0.2.1-GCCcore-7.3.0](http://pyyaml.org/wiki/LibYAML)
LibYAML is a YAML parser and emitter written in C.
 - [liquidprompt/1.11](https://github.com/nojhan/liquidprompt)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/liquidprompt/liquidprompt-1.11.eb)
Liquid Prompt gives you a nicely displayed prompt with useful information
 when you need it. It shows you what you need when you need it. You will notice what changes
 when it changes, saving time and frustration.
 - [lumpy/0.2.13-foss-2016b](https://github.com/pezmaster31/bamtools)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/lumpy/lumpy-0.2.13-foss-2016b.eb)
A probabilistic framework for structural variant discovery.
 - [lynx/2.8.8-foss-2016b](http://lynx.browser.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/l/lynx/lynx-2.8.8-foss-2016b.eb)
lynx is an alphanumeric display oriented World-Wide Web Client
 - [lz4/1.8.2-GCCcore-5.4.0](https://lz4.github.io/lz4/)
LZ4 is lossless compression algorithm, providing compression speed at 400 MB/s per core. 
 It features an extremely fast decoder, with speed in multiple GB/s per core.
 - [magicblast/1.0.0-foss-2016b](http://blast.ncbi.nlm.nih.gov/ )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/magicblast/magicblast-1.0.0-foss-2016b.eb)
Magic-BLAST is a new tool for mapping large sets of next-generation RNA or DNA sequencing runs against a whole genome or transcriptome.
 - [manta/1.6.0](https://github.com/Illumina/manta)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/manta/manta-1.6.0.eb)
Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads.
It is optimized for analysis of germline variation in small sets of individuals and 
somatic variation in tumor/normal sample pairs. Manta discovers, assembles and 
scores large-scale SVs, medium-sized indels and large insertions within a 
single efficient workflow. 

 - [matplotlib/3.1.1-foss-2016b-Python-3.7.4](http://matplotlib.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/matplotlib/matplotlib-3.1.1-foss-2016b-Python-3.7.4.eb)
matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
 hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
 and ipython shell, web application servers, and six graphical user interface toolkits.
 - [medaka/0.10.0-foss-2016b-Python-3.6.6](https://nanoporetech.github.io/medaka)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/medaka/medaka-0.10.0-foss-2016b-Python-3.6.6.eb)
medaka is a tool to create a consensus sequence of nanopore sequencing data.
 - [megalodon/1.0.0-alpha.1-foss-2016b-Python-3.7.4](https://github.com/nanoporetech)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/megalodon/megalodon-1.0.0-alpha.1-foss-2016b-Python-3.7.4.eb)
Megalodon provides "basecalling augmentation" for raw nanopore
 sequencing reads, including direct, reference-guided SNP and modified base calling.
 - [minimap2/2.13-foss-2016b](https://lh3.github.io/minimap2)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/minimap2/minimap2-2.13-foss-2016b.eb)
A versatile pairwise aligner for genomic and spliced nucleotide sequences.
 - [misopy/0.5.4-foss-2016b-Python-2.7.15](https://www.genes.mit.edu/burgelab/miso/)
MISO (Mixture of Isoforms) is a probabilistic framework that quantitates the expression level of alternatively spliced genes from RNA-Seq data, and identifies differentially regulated isoforms or exons across samples. By modeling the generative process by which reads are produced from isoforms in RNA-Seq, the MISO model uses Bayesian inference to compute the probability that a read originated from a particular isoform.
 - [mixcr/2.1.5-foss-2016b](https://github.com/milaboratory/mixcr)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/mixcr/mixcr-2.1.5-foss-2016b.eb)
MiXCR is a universal software for fast and accurate analysis of T- and B- cell receptor repertoire sequencing data.
 - [mofapy/1.2-foss-2016b-Python-3.7.4](https://github.com/bioFAM/MOFA)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/mofapy/mofapy-1.2-foss-2016b-Python-3.7.4.eb)
Multi-Omics Factor Analysis, a statistical framework for the integration of multi-omics data
 - [monocle3/0.2.1-foss-2016b-R-3.6.2](https://cole-trapnell-lab.github.io/monocle3)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/monocle3/monocle3-0.2.1-foss-2016b-R-3.6.2.eb)
Single-cell transcriptome sequencing (sc-RNA-seq) experiments allow us to discover new cell types and help us understand how they arise in development. The Monocle 3 package provides a toolkit for analyzing single-cell gene expression experiments.
 - [motif/2.3.4-goolf-1.7.20](http://motif.ics.com/ )

 - [mrsFAST/3.3.11-foss-2016b](http://sfu-compbio.github.io/mrsfast/ )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/mrsFAST/mrsFAST-3.3.11-foss-2016b.eb)
mrsFAST is designed to map short reads to reference genome assemblies; in a fast and memory-efficient manner. mrsFAST is a cache-oblivous short read mapper that optimizes cache usage to get higher performance.
 - [multitail/6.4.2-foss-2016b](http://www.vanheusden.com/multitail/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/m/multitail/multitail-6.4.2-foss-2016b.eb)
MultiTail allows you to monitor logfiles and command
output in multiple windows in a terminal, colorize, filter and merge.
 - [nanofilt/2.6.0-foss-2016b-Python-3.7.4](https://github.com/wdecoster/nanofilt)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/nanofilt/nanofilt-2.6.0-foss-2016b-Python-3.7.4.eb)
Filtering and trimming of long read sequencing data.
 - [nanopolish/0.11.1-foss-2018b-Python-2.7.15](https://github.com/jts/nanopolish)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/nanopolish/nanopolish-0.11.1-foss-2018b-Python-2.7.15.eb)
Software package for signal-level analysis of Oxford Nanopore sequencing data.
 - [ncbi-blast/2.6.0-foss-2016b](http://blast.ncbi.nlm.nih.gov/)
Basic Local Alignment Search Tool, or BLAST, is an algorithm
 for comparing primary biological sequence information, such as the amino-acid
 sequences of different proteins or the nucleotides of DNA sequences.
 - [ncbi-vdb/2.7.0-foss-2016b](https://github.com/ncbi/ncbi-vdb)
The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for
 using data in the INSDC Sequence Read Archives.
 - [ncurses/6.1-GCCcore-7.3.0](http://www.gnu.org/software/ncurses/)
The Ncurses (new curses) library is a free software emulation of curses in
 System V Release 4.0, and more. It uses Terminfo format, supports pads and
 color and multiple highlights and forms characters and function-key mapping,
 and has all the other SYSV-curses enhancements over BSD Curses.

 - [netCDF/4.5.0-foss-2016b](http://www.unidata.ucar.edu/software/netcdf/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/netCDF/netCDF-4.5.0-foss-2016b.eb)
NetCDF (network Common Data Form) is a set of software libraries 
 and machine-independent data formats that support the creation, access, and sharing of array-oriented 
 scientific data.
 - [nettle/3.4-foss-2018b](http://www.lysator.liu.se/~nisse/nettle/)
Nettle is a cryptographic library that is designed to fit easily
 in more or less any context: In crypto toolkits for object-oriented
 languages (C++, Python, Pike, ...), in applications like LSH or GNUPG,
 or even in kernel space.
 - [networkx/2.4-foss-2016b-Python-3.7.4](https://pypi.python.org/pypi/networkx)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/networkx/networkx-2.4-foss-2016b-Python-3.7.4.eb)
NetworkX is a Python package for the creation, manipulation, and study of the structure, dynamics,
 and functions of complex networks.
 - [newick-utils/1.6-foss-2016b](http://cegg.unige.ch/newick_utils )
The Newick Utilities are a suite of Unix shell tools for processing phylogenetic trees. We distribute the package under the BSD License. Functions include re-rooting, extracting subtrees, trimming, pruning, condensing, drawing (ASCII graphics or SVG).
 - [nextflow/20.04.0-edge](https://github.com/nextflow-io/nextflow)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/nextflow/nextflow-20.04.0-edge.eb)
Nextflow is a bioinformatics workflow manager that enables the development
 of portable and reproducible workflows. It supports deploying workflows on a variety of
 execution platforms including local, HPC schedulers, AWS Batch, Google Genomics Pipelines,
 and Kubernetes.
 - [ngmlr/0.2.7-foss-2016b](https://github.com/philres/ngmlr)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/ngmlr/ngmlr-0.2.7-foss-2016b.eb)
Ngmlr is a long-read mapper designed to align PacBilo or Oxford Nanopore to a
 reference genome with a focus on reads that span structural variations.
 - [nodejs/9.8.0-foss-2016b](http://nodejs.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/nodejs/nodejs-9.8.0-foss-2016b.eb)
Node.js is a platform built on Chrome's JavaScript runtime 
 for easily building fast, scalable network applications. Node.js uses an 
 event-driven, non-blocking I/O model that makes it lightweight and efficient, 
 perfect for data-intensive real-time applications that run across distributed devices.
 - [nodejs-bundle/9.8.0-foss-2016b](http:/scicomp.fhcrc.org)
nodejs with Canvas and Vega-lite packages
 - [nullarbor/2.0.20191013](https://github.com/tseemann/nullarbor)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/nullarbor/nullarbor-2.0.20191013.eb)
Pipeline to generate complete public health microbiology reports from sequenced isolates

 - [numactl/2.0.12-GCCcore-8.3.0](http://oss.sgi.com/projects/libnuma/)
The numactl program allows you to run your application program on specific
 cpu's and memory nodes. It does this by supplying a NUMA memory policy to
 the operating system before running your program. The libnuma library provides
 convenient ways for you to add NUMA memory policies into your own program.

 - [numba/0.46.0-foss-2016b-Python-3.7.4](http://numba.pydata.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/numba/numba-0.46.0-foss-2016b-Python-3.7.4.eb)
Numba is an Open Source NumPy-aware optimizing compiler for Python sponsored by Continuum Analytics,
 Inc. It uses the remarkable LLVM compiler infrastructure to compile Python syntax to machine code.
 - [numexpr/2.7.0-foss-2016b](https://numexpr.readthedocs.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/n/numexpr/numexpr-2.7.0-foss-2016b.eb)
The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.
 It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into
 code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a
 compiler at runtime.
 - [oncotator/1.9.3.0-foss-2016b-Python-2.7.14-bare](http://portals.broadinstitute.org/oncotator/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/o/oncotator/oncotator-1.9.3.0-foss-2016b-Python-2.7.14-bare.eb)
Oncotator is a web application for annotating human genomic point mutations and indels with data relevant to cancer researchers.
 - [oncotator_v1_ds/April052016](http://portals.broadinstitute.org/oncotator/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/o/oncotator_v1_ds/oncotator_v1_ds-April052016.eb)
Oncotator is a web application for annotating human genomic point mutations and indels with data relevant to cancer researchers.
 - [ont-guppy-cpu/2.3.7](https://nanoporetech.com/products/minit)
Guppy software supports MinIT and MinION instruments from
 Nanopore Technologies
 - [p11-kit/0.23.2-foss-2016b](http://p11-glue.freedesktop.org/p11-kit.html)
Provides a way to load and enumerate PKCS#11 modules.
 Provides a standard configuration setup for installing
 PKCS#11 modules in such a way that they're discoverable.
 Also solves problems with coordinating the use of PKCS#11
 by different components or libraries living in the same process.
 - [pandoc/2.1.3](https://pandoc.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pandoc/pandoc-2.1.3.eb)
If you need to convert files from one markup format into another, pandoc is your swiss-army knife.
 - [parallel/20170222-foss-2016b](http://savannah.gnu.org/projects/parallel/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/parallel/parallel-20170222-foss-2016b.eb)
parallel: Build and execute shell commands in parallel
 - [parasail/2.4-foss-2016b](https://github.com/jeffdaily/parasail)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/parasail/parasail-2.4-foss-2016b.eb)
parasail is a SIMD C (C99) library containing implementations
 of the Smith-Waterman (local), Needleman-Wunsch (global), and semi-global
 pairwise sequence alignment algorithms. 
 - [patchelf/0.9-foss-2016b](http://nixos.org/patchelf.html)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/patchelf/patchelf-0.9-foss-2016b.eb)
PatchELF is a small utility to modify the dynamic linker and RPATH of ELF executables.
 - [pdfgrep/2.0.1-foss-2016b](https://pdfgrep.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pdfgrep/pdfgrep-2.0.1-foss-2016b.eb)
a commandline utility to search text in PDF files
 - [peer-master/1.0.0-foss-2016b](https://www.sanger.ac.uk/science/tools/peer)
PEER is a collection of Bayesian approaches to infer hidden
 determinants and their effects from gene expression profiles using factor
 analysis methods. 
 - [pgadmin3/1.22.1-foss-2016a](https://www.pgadmin.org/ )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pgadmin3/pgadmin3-1.22.1-foss-2016a.eb)
pgAdmin is the most popular and feature rich Open Source administration and development platform for PostgreSQL
 - [philosopher/3.2.3](https://github.com/Nesvilab/philosopher/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/philosopher/philosopher-3.2.3.eb)
Philosopher provides easy access to third-party tools and custom algorithms
 allowing users to develop proteomics analysis, from Peptide Spectrum Matching to annotated
 protein reports. Philosopher is also tuned for Open Search analysis, providing a modified
 version of the prophets for peptide validation and protein inference. To this date, Philosopher
 is the only proteomics toolkit that allows you to process and analyze close and
 open search results.
 - [picard/2.18.29-Java](http://sourceforge.net/projects/picard)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/picard/picard-2.18.29-Java.eb)
A set of tools (in Java) for working with next generation sequencing data in the BAM format.
 - [pipdeptree/0.13.2-foss-2016b-Python-3.7.4](https://github.com/naiquevin/pipdeptree)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pipdeptree/pipdeptree-0.13.2-foss-2016b-Python-3.7.4.eb)
pipdeptree is a command line utility for displaying the installed python packages in form of a dependency tree. It works for packages installed globally on a machine as well as in a virtualenv.
 - [pkg-config/0.29.2-GCCcore-7.3.0](http://www.freedesktop.org/wiki/Software/pkg-config/)
pkg-config is a helper tool used when compiling applications and libraries.
 It helps you insert the correct compiler options on the command line so an
 application can use gcc -o test test.c `pkg-config --libs --cflags glib-2.0`
 for instance, rather than hard-coding values on where to find glib (or other
 libraries).

 - [pkgconfig/1.3.1-foss-2016b-Python-3.7.4](http://github.com/matze/pkgconfig)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pkgconfig/pkgconfig-1.3.1-foss-2016b-Python-3.7.4.eb)
pkgconfig is a Python module to interface with the pkg-config command line tool
 - [plink/1.9b_4.6-x86_64](https://www.cog-genomics.org/plink/1.9)
plink-1.9-x86_64: Whole-genome association analysis toolset
 - [plink2/20190910](https://www.cog-genomics.org/plink/2.0)
plink2 Whole-genome association analysis toolset
 - [pocl/1.3-GCC-5.4.0-2.26](http://portablecl.org)
Pocl is a portable open source (MIT-licensed) implementation of the OpenCL standard
 - [poppler/0.54.0-foss-2016b](https://poppler.freedesktop.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/poppler/poppler-0.54.0-foss-2016b.eb)
Poppler is a PDF rendering library based on the xpdf-3.0 code base.
 - [pplacer/1.1.alpha19-foss-2016b](['https://matsen.fhcrc.org/pplacer/'])
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pplacer/pplacer-1.1.alpha19-foss-2016b.eb)
Pplacer places reads on a phylogenetic tree. guppy (Grand
 Unified Phylogenetic Placement Yanalyzer) yanalyzes them. rppr is a helpful
 tool for working with reference packages.
 - [printproto/1.0.5-foss-2016b](http://xorg.freedesktop.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/printproto/printproto-1.0.5-foss-2016b.eb)
X.org PrintProto protocol headers.
 - [prodigal/2.6.3-GCC-5.4.0-2.26](http://prodigal.ornl.gov/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/prodigal/prodigal-2.6.3-GCC-5.4.0-2.26.eb)
Prodigal (Prokaryotic Dynamic Programming Genefinding
   Algorithm) is a microbial (bacterial and archaeal) gene finding program
   developed at Oak Ridge National Laboratory and the University of
   Tennessee.
 - [prokka/1.13-foss-2016b-BioPerl-1.7.0](http://www.vicbioinformatics.com/software.prokka.shtml)
Prokka is a software tool for the rapid annotation of prokaryotic genomes.
 - [protobuf/3.6.1-GCCcore-5.4.0](https://github.com/google/protobuf/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/protobuf/protobuf-3.6.1-GCCcore-5.4.0.eb)
Google Protocol Buffers
 - [psipred/4.01-foss-2016b](http://bioinf.cs.ucl.ac.uk/psipred/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/psipred/psipred-4.01-foss-2016b.eb)
The PSIPRED Protein Sequence Analysis Workbench aggregates several UCL structure prediction methods into one location.
 - [pyBigWig/0.3.13-foss-2016b-Python-3.6.6](https://github.com/deeptools/pyBigWig)
A python extension, written in C, for quick access to bigBed files and access to and creation of
 bigWig files.
 - [pyEGA3/3.0.44-foss-2016b-Python-3.7.4](https://github.com/EGA-archive/ega-download-client)
A basic Python-based EGA download client 
 - [pybedtools/0.8.0-foss-2016b-Python-3.7.4](https://daler.github.io/pybedtools)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pybedtools/pybedtools-0.8.0-foss-2016b-Python-3.7.4.eb)
pybedtools wraps and extends BEDTools and offers feature-level manipulations from
 within Python.
 - [pybind11/2.3.0-foss-2016b-Python-3.7.4](https://pybind11.readthedocs.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pybind11/pybind11-2.3.0-foss-2016b-Python-3.7.4.eb)
pybind11 is a lightweight header-only library that exposes C++ types in Python and vice versa,
 mainly to create Python bindings of existing C++ code.
 - [pypy/7.1.1-foss-2016b-Python-3.6.6](pypy.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pypy/pypy-7.1.1-foss-2016b-Python-3.6.6.eb)
PyPy is a replacement for CPython. It is built using the RPython language that was co-developed with it. The main reason to use it instead of CPython is speed: it runs generally faster
 - [pytest/5.0.1-foss-2016b-Python-3.7.4](http://pytest.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/pytest/pytest-5.0.1-foss-2016b-Python-3.7.4.eb)
pytest: simple powerful testing with Python
 - [python-Levenshtein/0.12.0-foss-2016b-Python-3.6.6](https://github.com/ztane/python-Levenshtein)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/python-Levenshtein/python-Levenshtein-0.12.0-foss-2016b-Python-3.6.6.eb)
The Levenshtein Python C extension module contains functions for fast computation of;
 Levenshtein (edit) distance, and edit operations
 string similarity
 approximate median strings, and generally string averaging
 string sequence and set similarity.
 - [python-fire/0.2.1-foss-2016b-Python-3.7.4](https://github.com/google/python-fire)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/python-fire/python-fire-0.2.1-foss-2016b-Python-3.7.4.eb)
Python Fire is a library for automatically generating command line interfaces
 (CLIs) from absolutely any Python object.
 - [python-igraph/0.7.1.post6-foss-2016b-Python-3.7.4](http://igraph.org/python)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/python-igraph/python-igraph-0.7.1.post6-foss-2016b-Python-3.7.4.eb)
Python interface to the igraph high performance graph library, primarily aimed at complex network
 research and analysis.
 - [python-partis/0.15.0-foss-2016b-Python-3.6.6](https://github.com/psathyrella/partis)
Partis is an HMM-based framework for B- and T-cell receptor sequence annotation,
 simulation, clonal family, and germline inference. It is built on top of the ham HMM compiler
 and ig-sw set of Smith-Waterman annotation tools.
 - [qcat/1.0.5-foss-2016b-Python-3.6.6](https://github.com/nanoporetech/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/q/qcat/qcat-1.0.5-foss-2016b-Python-3.6.6.eb)
Qcat is Python command-line tool for demultiplexing Oxford
 Nanopore reads from FASTQ files.
 - [ragel/6.9-foss-2016b](http://www.colm.net/open-source/ragel/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/ragel/ragel-6.9-foss-2016b.eb)
Ragel compiles executable finite state machines from regular
languages. Ragel targets C, C++ and ASM. Ragel state machines can not only
recognize byte sequences as regular expression machines do, but can also
execute code at arbitrary points in the recognition of a regular language.
Code embedding is done using inline operators that do not disrupt the regular
language syntax.
 - [rclonebrowser/1.2-foss-2016b](https://mmozeiko.github.io/RcloneBrowser/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/rclonebrowser/rclonebrowser-1.2-foss-2016b.eb)
Simple cross platfrom GUI for rclone command line tool. Supports Windows, macOS and GNU/Linux.
 - [renderproto/0.11-foss-2016a](http://www.freedesktop.org/wiki/Software/xlibs )
Xrender protocol and ancillary headers
 - [rgeos/0.3-26-foss-2016b-R-3.5.0](https://cran.r-project.org/web/packages/rgeos/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/rgeos/rgeos-0.3-26-foss-2016b-R-3.5.0.eb)
R interface to Geometry Engine
 - [rosetta/2017.39.59729-foss-2016b](https://www.rosettacommons.org )
Rosetta is the premier software suite for modeling macromolecular structures. As a flexible, multi-purpose application, it includes tools for structure prediction, design, and remodeling of proteins and nucleic acids.
 - [rpy2/2.7.9-foss-2016b-Python-2.7.12](http://rpy.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/rpy2/rpy2-2.7.9-foss-2016b-Python-2.7.12.eb)
rpy2 is a redesign and rewrite of rpy. It is providing a low-level 
interface to R from Python, a proposed high-level interface, including wrappers to 
graphical libraries, as well as R-like structures and functions. 
 - [rstudio/1.2.5033-foss-2016b](https://github.com/rstudio/rstudio)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/rstudio/rstudio-1.2.5033-foss-2016b.eb)
RStudio is an integrated development environment (IDE) for the R programming language.
 - [rstudio-server/1.2.5033-foss-2016b](https://github.com/rstudio/rstudio)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/r/rstudio-server/rstudio-server-1.2.5033-foss-2016b.eb)
RStudio is an integrated development environment 
(IDE) for the R programming language.
 - [samblaster/0.1.22-foss-2016a](https://github.com/GregoryFaust/samblaster )
samblaster is a fast and flexible program for marking duplicates in read-id grouped1 paired-end SAM files.
 - [sbcl/1.4.6](https://sourceforge.net/projects/sbcl/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/sbcl/sbcl-1.4.6.eb)
A high performance Common Lisp compiler. In addition to standard ANSI Common Lisp, it provides an interactive environment including an a debugger, a statistical profiler, a code coverage tool, and many other extensions.
 - [sbt/1.0.4-Java-1.8.0_121](http://www.scala-sbt.org/)
sbt is a build tool for Scala, Java, and more.
 - [scanpy/1.4.5.1-foss-2016b-Python-3.7.4](https://scvelo.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/scanpy/scanpy-1.4.5.1-foss-2016b-Python-3.7.4.eb)
scVelo is a scalable toolkit for estimating and analyzing RNA velocities in single cells using
 dynamical modeling.
 - [scikit-image/0.16.2-foss-2016b-Python-3.7.4](https://scikit-image.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/scikit-image/scikit-image-0.16.2-foss-2016b-Python-3.7.4.eb)
scikit-image is a collection of algorithms for image processing.
 - [scikit-learn/0.21.2-foss-2016b-Python-3.6.6](http://scikit-learn.org/stable/index.html)
Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world,
building upon numpy, scipy, and matplotlib. As a machine-learning module,
it provides versatile tools for data mining and analysis in any field of science and engineering.
It strives to be simple and efficient, accessible to everybody, and reusable in various contexts.
 - [scons/3.0.0-foss-2016b-Python-2.7.12](https://www.scons.org/)
Open Source next-generation build tool.
 - [screen/4.4.0-foss-2016b](http://www.gnu.org/software/screen/ )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/screen/screen-4.4.0-foss-2016b.eb)
Screen is a full-screen window manager that multiplexes a physical terminal between several processes, typically interactive shells.
 - [seqtk/1.3-foss-2016b](https://bitbucket.org/statgen/xhmm/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/seqtk/seqtk-1.3-foss-2016b.eb)
Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format. It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip.
 - [seqtools/4.44.1-foss-2016b](http://www.sanger.ac.uk/science/tools/seqtools)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/seqtools/seqtools-4.44.1-foss-2016b.eb)
The SeqTools package contains three tools for visualising sequence alignments: Blixem, Dotter and Belvu.
 - [smallgenomeutilities/1.0-foss-2016b-Python-3.6.6](https://github.com/cbg-ethz/smallgenomeutilities)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/smallgenomeutilities/smallgenomeutilities-1.0-foss-2016b-Python-3.6.6.eb)
The smallgenomeutilities are a collection of scripts that is
 useful for dealing and manipulating NGS data of small viral genomes. They
 are written in Python 3 with a small number of dependencies.
 - [smithwaterman/1.1-foss-2016b](https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/smithwaterman/smithwaterman-1.1-foss-2016b.eb)
Overview SSW is a fast implementation of the Smith-Waterman
 algorithm, which uses the Single-Instruction Multiple-Data (SIMD)
 instructions to parallelize the algorithm at the instruction level.
 - [snakemake/5.8.1-foss-2016b-Python-3.7.4](https://snakemake.readthedocs.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/snakemake/snakemake-5.8.1-foss-2016b-Python-3.7.4.eb)
The Snakemake workflow management system is a tool to create reproducible and scalable data analyses.
 - [snappy/1.1.6-foss-2016b](https://github.com/google/snappy)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/snappy/snappy-1.1.6-foss-2016b.eb)
Snappy is a compression/decompression library. It does not aim
for maximum compression, or compatibility with any other compression library;
instead, it aims for very high speeds and reasonable compression.
 - [snippy/4.4.1-foss-2016b-Perl-5.28.0](https://github.com/tseemann/snippy)
Snippy finds SNPs between a haploid reference genome and your NGS sequence
 reads. It will find both substitutions (snps) and insertions/deletions (indels).
Rapid haploid variant calling and core genome alignment.
 - [spacy/2.0.12-foss-2016b-Python-3.6.5-fh3](https://spacy.io/)
Industrial-strength natural language processing in Python
 - [squashfs-tools/4.3-GCC-5.4.0-2.26](http://squashfs.sourceforge.net)
Squashfs is a compressed read-only filesystem for Linux.
 - [strelka/2.9.9-foss-2016b](https://github.com/Illumina/strelka)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/strelka/strelka-2.9.9-foss-2016b.eb)
Strelka2 is a fast and accurate small variant caller
 optimized for analysis of germline variation in small cohorts and
 somatic variation in tumor/normal sample pairs.
 - [summovie/1.0.2-binary](http://grigoriefflab.janelia.org/unblur)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/summovie/summovie-1.0.2-binary.eb)
Unblur is used to align the frames of movies recorded on an
 electron microscope to reduce image blurring due to beam-induced motion. Movie
frame sums can also be calculated using Summovie, which uses the alignment
results from a prior run of Unblur. 
 - [svtoolkit/2.00.1678-foss-2016a](http://www.broadinstitute.org/software/genomestrip/ )
Genome STRiP (Genome STRucture In Populations) is a suite of tools for discovering and genotyping structural variations using sequencing data. The methods are designed to detect shared variation using data from multiple individuals.
 - [svtyper/0.0.4-foss-2016b-Python-2.7.12-fh1](https://github.com/hall-lab/svtyper )
Bayesian genotyper for structural variants
 - [tabix/0.2.6-foss-2016b](http://samtools.sourceforge.net )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/tabix/tabix-0.2.6-foss-2016b.eb)
Generic indexer for TAB-delimited genome position files 
 - [tandem/17-01-01-2-foss-2016b](http://www.thegpm.org/TANDEM/instructions.html )
X! Tandem open source is software that can match tandem mass spectra with peptide sequences, in a process that has come to be known as protein identification.
 - [tandem/17-01-01-2-foss-2016b](http://www.thegpm.org/TANDEM/instructions.html )
X! Tandem open source is software that can match tandem mass spectra with peptide sequences, in a process that has come to be known as protein identification.
 - [tcr-dist/0.0.2-foss-2016b-Python-2.7.15](https://github.com/phbradley/tcr-dist)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/tcr-dist/tcr-dist-0.0.2-foss-2016b-Python-2.7.15.eb)
Quantifiable predictive features define epitope-specific T cell receptor repertoires
 - [terraphast/master-foss-2016b](https://github.com/xflouris/libpll/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/terraphast/terraphast-master-foss-2016b.eb)
libpll is a versatile high-performance software library for phylogenetic analysis.
 - [tesseract/4.0.0-beta.1-foss-2016b](https://github.com/tesseract-ocr/tesseract/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/tesseract/tesseract-4.0.0-beta.1-foss-2016b.eb)
This package contains an OCR engine
 - [texlive/20180531-foss-2016b](http://tug.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/texlive/texlive-20180531-foss-2016b.eb)
TeX is a typesetting language. Instead of visually
formatting your text, you enter your manuscript text intertwined with TeX
commands in a plain text file. You then run TeX to produce formatted output,
such as a PDF file. Thus, in contrast to standard word processors, your
document is a separate file that does not pretend to be a representation
of the final typeset output, and so can be easily edited and manipulated.
 - [tmux/3.0-GCCcore-5.4.0](http://tmux.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/t/tmux/tmux-3.0-GCCcore-5.4.0.eb)
tmux is a terminal multiplexer. It lets you switch easily 
between several programs in one terminal, detach them (they keep running in the background) and reattach them to a different terminal.
 - [udunits/2.2.25-foss-2016b](https://www.unidata.ucar.edu/downloads/udunits/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/u/udunits/udunits-2.2.25-foss-2016b.eb)
The UDUNITS package supports units of physical quantities. Its C library provides for arithmetic manipulation of units and for conversion of numeric values between compatible units.
 - [unblur/1.0.2-binary](http://grigoriefflab.janelia.org/unblur)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/u/unblur/unblur-1.0.2-binary.eb)
Unblur is used to align the frames of movies recorded on an
 electron microscope to reduce image blurring due to beam-induced motion.
 - [unixODBC/2.3.4-foss-2016b](http://www.unixodbc.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/u/unixODBC/unixODBC-2.3.4-foss-2016b.eb)
What is the unixODBC Project?

The unixODBC Project goals are to develop and promote unixODBC to be the definitive standard for ODBC on non MS Windows platforms. This is to include GUI support for both KDE and GNOME.

What is ODBC?

ODBC is an open specification for providing application developers with a predictable API with which to access Data Sources. Data Sources include SQL Servers and any Data Source with an ODBC Driver.
 - [util-linux/2.32-GCCcore-7.3.0](http://www.kernel.org/pub/linux/utils/util-linux)
Set of Linux utilities
 - [v-pipe/1.0-foss-2016b](https://github.com/cbg-ethz/V-pipe)
V-pipe is a workflow designed for analysis of next generation
 sequencing (NGS) data from viral pathogens. It produces a number of results
 in a curated format.
 - [vdjpuzzle2/3771ca670a3a-foss-2016b](https://bitbucket.org/kirbyvisp/vdjpuzzle2)
TCR and BCR reconstruction from scRNA-seq data
 - [vdjtools/1.1.5-foss-2016b](https://github.com/milaboratory/mixcr)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/v/vdjtools/vdjtools-1.1.5-foss-2016b.eb)
VDJtools is an open-source Java/Groovy-based framework
designed to facilitate analysis of immune repertoire sequencing (RepSeq)
data. VDJtools computes a wide set of statistics and is able to perform
various forms of cross-sample analysis.
 - [vim/8.0-foss-2016b](http://www.vim.org )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/v/vim/vim-8.0-foss-2016b.eb)
Vim is an advanced text editor that seeks to provide the power   of the de-facto Unix editor 'Vi', with a more complete feature set. 
 - [vt/0.57721-foss-2016b](http://genome.sph.umich.edu/wiki/Vt)
A tool set for short variant discovery in genetic sequence data.
 - [wget/1.17.1-foss-2016b](https://www.gnu.org/software/wget/)
GNU Wget is a free software package for retrieving files using HTTP, HTTPS and FTP,
 the most widely-used Internet protocols. It is a non-interactive commandline tool,
 so it may easily be called from scripts, cron jobs, terminals without X-Windows support, etc.
 - [wham/1.8.0-foss-2016a](https://github.com/zeeev/wham )
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/w/wham/wham-1.8.0-foss-2016a.eb)
The wham suite consists of two programs, wham and whamg. wham, the original tool, is a very sensitive method with a high false discovery rate. The second program, whamg, is more accurate and better suited for general structural variant (SV) discovery.
 - [wheel/0.33.4-foss-2016b-Python-3.7.4](https://pypi.python.org/pypi/wheel)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/w/wheel/wheel-0.33.4-foss-2016b-Python-3.7.4.eb)
A built-package format for Python.
 - [womtool/36-Java-1.8](https://github.com/broadinstitute/cromwell)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/w/womtool/womtool-36-Java-1.8.eb)
Scientific workflow engine designed for simplicity &
scalability.
 - [wxGTK/2.8.12-foss-2016a](https://www.wxwidgets.org/ )
wxWidgets is a C++ library that lets developers create applications for Windows, Mac OS X, Linux and other platforms with a single code base.
 - [wxWidgets/3.1.2-foss-2018b](https://www.wxwidgets.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/w/wxWidgets/wxWidgets-3.1.2-foss-2018b.eb)
wxWidgets is a C++ library that lets developers create
applications for Windows, Mac OS X, Linux and other platforms with a single
code base. It has popular language bindings for Python, Perl, Ruby and many
other languages, and unlike other cross-platform toolkits, wxWidgets gives
applications a truly native look and feel because it uses the platform's native
API rather than emulating the GUI. 
 - [x264/20160614-foss-2016b](http://www.videolan.org/developers/x264.html)
x264 is a free software library and application for encoding video streams into the H.264/MPEG-4
 AVC compression format, and is released under the terms of the GNU GPL.
 - [x265/2.4-foss-2016b](http://x265.org/)
x265 is a free software library and application for encoding video streams into the H.265
 AVC compression format, and is released under the terms of the GNU GPL.
 - [xbitmaps/1.1.1-goolf-1.7.20](http://www.freedesktop.org/wiki/Software/xlibs )

 - [xcb-proto/1.11](http://xcb.freedesktop.org/ )
The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint, latency hiding, direct access to the protocol, improved threading support, and extensibility.
 - [xcb-util/0.4.0-foss-2016b](http://xcb.freedesktop.org/)
The xcb-util package provides additional extensions to the XCB library,
 many that were previously found in Xlib, but are not part of core X protocol
 - [xcb-util-image/0.4.0-foss-2016b](http://xcb.freedesktop.org/)
The xcb-util-image package provides additional extensions to the XCB library.
 - [xcb-util-keysyms/0.4.0-foss-2016b](http://xcb.freedesktop.org/)
The xcb-util-keysyms package contains a library for
 handling standard X key constants and conversion to/from keycodes.
 - [xcb-util-renderutil/0.3.9-foss-2016b](http://xcb.freedesktop.org/)
The xcb-util-renderutil package provides additional extensions to the XCB library.
 - [xcb-util-wm/0.4.1-foss-2016a](http://xcb.freedesktop.org/)
The xcb-util-wm package contains libraries which provide client and window-manager helpers
 for EWMH and ICCCM.
 - [xineramaproto/1.2.1-foss-2016b](http://www.freedesktop.org/wiki/Software/xlibs)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/x/xineramaproto/xineramaproto-1.2.1-foss-2016b.eb)
X protocol and ancillary headers for xinerama
 - [xorg-macros/1.19.2-GCCcore-8.3.0](https://cgit.freedesktop.org/xorg/util/macros)
X.org macros utilities.
 - [xprop/1.2.3-GCCcore-7.3.0](http://www.x.org/wiki/)
The xprop utility is for displaying window and font properties in an X server.
 One window or font is selected using the command line arguments or possibly
 in the case of a window, by clicking on the desired window. A list of
 properties is then given, possibly with formatting information.
 - [xproto/7.0.28-foss-2016b](http://www.freedesktop.org/wiki/Software/xlibs)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/x/xproto/xproto-7.0.28-foss-2016b.eb)
X protocol and ancillary headers
 - [xtrans/1.3.5-foss-2016a](http://www.freedesktop.org/wiki/Software/xlibs )
xtrans includes a number of routines to make X implementations transport-independent;  at time of writing, it includes support for UNIX sockets, IPv4, IPv6, and DECnet. 
 - [zlib/1.2.11-GCCcore-5.4.0](http://www.zlib.net/)
zlib is designed to be a free, general-purpose, legally unencumbered -- that is,
 not covered by any patents -- lossless data-compression library for use on virtually any
 computer hardware and operating system.
 - [zstd/1.3.4-foss-2016b](https://facebook.github.io/zstd)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/z/zstd/zstd-1.3.4-foss-2016b.eb)
Zstandard is a real-time compression algorithm, providing high compression ratios. 
 It offers a very wide range of compression/speed trade-off, while being backed by a very fast decoder.
 It also offers a special mode for small data, called dictionary compression, and can create dictionaries
 from any sample set.
