---
title: All Modules  18.04
layout: single
sidebar:
  nav: "docs"
date: 2019-02-21
---

 - [AdapterRemoval/2.2.2-foss-2018b](https://github.com/MikkelSchubert/adapterremoval)  
AdapterRemoval searches for and removes remnant adapter sequences
 from High-Throughput Sequencing (HTS) data and (optionally) trims low quality bases
 from the 3' end of reads following adapter removal.
 - [Autoconf/2.69-GCCcore-7.3.0](http://www.gnu.org/software/autoconf/)  
Autoconf is an extensible package of M4 macros that produce shell scripts
 to automatically configure software source code packages. These scripts can
 adapt the packages to many kinds of UNIX-like systems without manual user
 intervention. Autoconf creates a configuration script for a package from a
 template file that lists the operating system features that the package can
 use, in the form of M4 macro calls.

 - [Automake/1.16.1-GCCcore-7.3.0](http://www.gnu.org/software/automake/automake.html)  
Automake: GNU Standards-compliant Makefile generator
 - [Autotools/20180311-GCCcore-7.3.0](http://autotools.io)  
This bundle collect the standard GNU build tools: Autoconf, Automake
 and libtool

 - [BBMap/38.26-foss-2018b](https://sourceforge.net/projects/bbmap/)  
BBMap short read aligner, and other bioinformatic tools.
 - [BCFtools/1.9-foss-2018b](http://www.htslib.org/)  
Samtools is a suite of programs for interacting with high-throughput sequencing data.
 BCFtools
 - [BEDOPS/2.4.35-foss-2018b](http://bedops.readthedocs.io/en/latest/index.html)  
BEDOPS is an open-source command-line toolkit that performs highly efficient and
 scalable Boolean and other set operations, statistical calculations, archiving, conversion and
 other management of genomic data of arbitrary scale. Tasks can be easily split by chromosome for
 distributing whole-genome analyses across a computational cluster.
 - [BEDTools/2.27.1-foss-2018b](https://github.com/arq5x/bedtools2)  
The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps
 and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF,
 and SAM/BAM.
 - [BLAST+/2.8.1-foss-2018b](http://blast.ncbi.nlm.nih.gov/)  
Basic Local Alignment Search Tool, or BLAST, is an algorithm
 for comparing primary biological sequence information, such as the amino-acid
 sequences of different proteins or the nucleotides of DNA sequences.
 - [BLAT/3.5-foss-2018b](http://genome.ucsc.edu/FAQ/FAQblat.html)  
BLAT on DNA is designed to quickly find sequences of 95% and greater similarity of length 25 bases or
 more.
 - [BWA/0.7.17-foss-2018b](http://bio-bwa.sourceforge.net/)  
Burrows-Wheeler Aligner (BWA) is an efficient program that aligns
 relatively short nucleotide sequences against a long reference sequence such as the human genome.
 - [BamTools/2.5.1-foss-2018b](https://github.com/pezmaster31/bamtools)  
BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.
 - [Beast/2.5.1-foss-2018b](http://beast2.org/)  
BEAST is a cross-platform program for Bayesian MCMC analysis of molecular 
 sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using 
 strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies 
 but is also a framework for testing evolutionary hypotheses without conditioning on a single 
 tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted 
 proportional to its posterior probability. 
 - [BioPerl/1.7.2-foss-2018b-Perl-5.28.0](http://www.bioperl.org/)  
Bioperl is the product of a community effort to produce Perl code which is useful in biology.
 Examples include Sequence objects, Alignment objects and database searching objects.
 - [Biopython/1.72-foss-2018b-Python-2.7.15](http://www.biopython.org)  
Biopython is a set of freely available tools for biological
 computation written in Python by an international team of developers. It is
 a distributed collaborative effort to develop Python libraries and
 applications which address the needs of current and future work in
 bioinformatics. 
 - [Bison/3.0.5-GCCcore-7.3.0](http://www.gnu.org/software/bison)  
Bison is a general-purpose parser generator that converts an annotated context-free grammar
 into a deterministic LR or generalized LR (GLR) parser employing LALR(1) parser tables.
 - [Blosc/1.14.4-GCCcore-7.3.0](http://www.blosc.org/)  
Blosc, an extremely fast, multi-threaded, meta-compressor library
 - [Boost/1.68.0-foss-2018b-Python-2.7.15](http://www.boost.org/)  
Boost provides free peer-reviewed portable C++ source libraries.
 - [Boost.Python/1.67.0-foss-2018b-Python-2.7.15](http://boostorg.github.io/python)  
Boost.Python is a C++ library which enables seamless interoperability between C++
 and the Python programming language.
 - [Bowtie/1.2.2-foss-2018b](http://bowtie-bio.sourceforge.net/index.shtml)  
Bowtie is an ultrafast, memory-efficient short read aligner.
 It aligns short DNA sequences (reads) to the human genome.
 - [Bowtie2/2.3.4.2-foss-2018b](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml)  
Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads
 to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s
 of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.
 Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,
 its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.
 - [CMake/3.12.1-GCCcore-7.3.0](http://www.cmake.org)  
CMake, the cross-platform, open-source build system.  CMake is a family of
 tools designed to build, test and package software.

 - [DBus/1.13.6-GCCcore-7.3.0](http://dbus.freedesktop.org/)  
D-Bus is a message bus system, a simple way for applications to talk
 to one another.  In addition to interprocess communication, D-Bus helps
 coordinate process lifecycle; it makes it simple and reliable to code
 a "single instance" application or daemon, and to launch applications
 and daemons on demand when their services are needed.

 - [DIAMOND/0.9.22-foss-2018b](https://github.com/bbuchfink/diamond)  
Accelerated BLAST compatible local sequence aligner
 - [Doxygen/1.8.14-GCCcore-7.3.0](http://www.doxygen.org)  
Doxygen is a documentation system for C++, C, Java, Objective-C, Python,
 IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some
 extent D.

 - [EMBOSS/6.6.0-foss-2018b](http://emboss.sourceforge.net/)  
EMBOSS is 'The European Molecular Biology Open Software Suite'.
 EMBOSS is a free Open Source software analysis package specially developed for
 the needs of the molecular biology (e.g. EMBnet) user community.
 - [EasyBuild/3.8.0](https://easybuilders.github.io/easybuild)  
EasyBuild is a software build and installation framework
 written in Python that allows you to install software in a structured,
 repeatable and robust way.
 - [Eigen/3.3.5](http://eigen.tuxfamily.org/index.php?title=Main_Page)  
Eigen is a C++ template library for linear algebra:
 matrices, vectors, numerical solvers, and related algorithms.

 - [FFTW/3.3.8-gompi-2018b](http://www.fftw.org)  
FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)
 in one or more dimensions, of arbitrary input size, and of both real and complex data.
 - [FLASH/2.2.00-foss-2018b](https://ccb.jhu.edu/software/FLASH/)  
FLASH (Fast Length Adjustment of SHort reads) is a very fast
and accurate software tool to merge paired-end reads from next-generation
sequencing experiments. FLASH is designed to merge pairs of reads when the
original DNA fragments are shorter than twice the length of reads. The
resulting longer reads can significantly improve genome assemblies. They can
also improve transcriptome assembly when FLASH is used to merge RNA-seq data.

 - [FSL/5.0.11-foss-2018b](http://www.fmrib.ox.ac.uk/fsl/)  
FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data.
 - [FastANI/1.1-foss-2018b](http://www.iodbc.org/)  
FastANI is developed for fast alignment-free computation of
 whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean
 nucleotide identity of orthologous gene pairs shared between two microbial
 genomes. FastANI supports pairwise comparison of both complete and draft
 genome assemblies.
 - [FastQC/0.11.8-Java-1.8](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/)  
FastQC is a quality control application for high throughput
sequence data. It reads in sequence data in a variety of formats and can either
provide an interactive application to review the results of several different
QC checks, or create an HTML based report which can be integrated into a
pipeline.
 - [FastTree/2.1.10-foss-2018b](http://www.microbesonline.org/fasttree/)  
FastTree infers approximately-maximum-likelihood phylogenetic
 trees from alignments of nucleotide or protein sequences. FastTree can handle
 alignments with up to a million of sequences in a reasonable amount of
 time and memory. 
 - [FriBidi/1.0.5-GCCcore-7.3.0](https://github.com/fribidi/fribidi)  
The Free Implementation of the Unicode Bidirectional Algorithm.

 - [GATK/4.0.8.1-foss-2018b-Python-3.6.6](http://www.broadinstitute.org/gatk/)  
The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute
 to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,
 with a primary focus on variant discovery and genotyping as well as strong emphasis on
 data quality assurance. Its robust architecture, powerful processing engine and
 high-performance computing features make it capable of taking on projects of any size.
 - [GCC/7.3.0-2.30](http://gcc.gnu.org/)  
The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
 as well as libraries for these languages (libstdc++, libgcj,...).
 - [GCCcore/7.3.0](http://gcc.gnu.org/)  
The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
 as well as libraries for these languages (libstdc++, libgcj,...).
 - [GDAL/2.2.3-foss-2018b-Python-3.6.6](http://www.gdal.org/)  
GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
 Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
 to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
 data translation and processing.
 - [GEOS/3.6.2-foss-2018b-Python-3.6.6](http://trac.osgeo.org/geos)  
GEOS (Geometry Engine
 - [GLPK/4.65-GCCcore-7.3.0](https://www.gnu.org/software/glpk/)  
The GLPK (GNU Linear Programming Kit) package is intended for
 solving large-scale linear programming (LP),
 mixed integer programming (MIP), and other related problems.
 It is a set of routines written in ANSI C 
 and organized in the form of a callable library.
 - [GLib/2.54.3-GCCcore-7.3.0](http://www.gtk.org/)  
GLib is one of the base libraries of the GTK+ project
 - [GMAP-GSNAP/2018-07-04-foss-2018b](http://research-pub.gene.com/gmap/)  
GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences
 GSNAP: Genomic Short-read Nucleotide Alignment Program
 - [GMP/6.1.2-GCCcore-7.3.0](http://gmplib.org/)  
GMP is a free library for arbitrary precision arithmetic, operating on signed
 integers, rational numbers, and floating point numbers.

 - [GObject-Introspection/1.54.1-foss-2018b-Python-2.7.15](https://wiki.gnome.org/GObjectIntrospection/)  
GObject introspection is a middleware layer between C
 libraries (using GObject) and language bindings. The C library can be scanned
 at compile time and generate a metadata file, in addition to the actual
 native C library. Then at runtime, language bindings can read this
 metadata and automatically provide bindings to call into the C library.
 - [GROMACS/2018.3-foss-2018b](http://www.gromacs.org)  
GROMACS is a versatile package to perform molecular dynamics,
 i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.

This is a CPU only build, containing both MPI and threadMPI builds.

 - [GSL/2.5-GCC-7.3.0-2.30](http://www.gnu.org/software/gsl/)  
The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.
 The library provides a wide range of mathematical routines such as random number generators, special functions
 and least-squares fitting.
 - [GTS/0.7.6-GCCcore-7.3.0](http://gts.sourceforge.net/)  
GTS stands for the GNU Triangulated Surface Library.
 It is an Open Source Free Software Library intended to provide a set of useful
 functions to deal with 3D surfaces meshed with interconnected triangles.
 - [Ghostscript/9.23-GCCcore-7.3.0](http://ghostscript.com)  
Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
 different targets. It used to be part of the cups printing stack, but is no longer used for that.
 - [GraphicsMagick/1.3.31-foss-2018b](http://www.graphicsmagick.org/)  
GraphicsMagick is the swiss army knife of image processing.
 - [HDF5/1.10.2-foss-2018b](https://portal.hdfgroup.org/display/support)  
HDF5 is a data model, library, and file format for storing and managing data.
 It supports an unlimited variety of datatypes, and is designed for flexible
 and efficient I/O and for high volume and complex data.
 - [HMMER/3.2.1-foss-2018b](http://hmmer.org/)  
HMMER is used for searching sequence databases for homologs
 of protein sequences, and for making protein sequence alignments. It
 implements methods using probabilistic models called profile hidden Markov
 models (profile HMMs).  Compared to BLAST, FASTA, and other sequence
 alignment and database search tools based on older scoring methodology,
 HMMER aims to be significantly more accurate and more able to detect remote
 homologs because of the strength of its underlying mathematical models. In the
 past, this strength came at significant computational expense, but in the new
 HMMER3 project, HMMER is now essentially as fast as BLAST.
 - [HTSeq/0.11.0-foss-2018b-Python-2.7.15](http://www-huber.embl.de/users/anders/HTSeq/)  
A framework to process and analyze data from high-throughput sequencing (HTS) assays
 - [HTSlib/1.9-foss-2018b](http://www.htslib.org/)  
A C library for reading/writing high-throughput sequencing data.
 This package includes the utilities bgzip and tabix
 - [HarfBuzz/2.2.0-foss-2018b](http://www.freedesktop.org/wiki/Software/HarfBuzz)  
HarfBuzz is an OpenType text shaping engine.
 - [ICU/61.1-GCCcore-7.3.0](http://site.icu-project.org/home)  
ICU is a mature, widely used set of C/C++ and Java libraries providing Unicode and Globalization
 support for software applications.
 - [IDBA-UD/1.1.3-foss-2018b](http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/)  
IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads
 Sequencing data with Highly Uneven Sequencing Depth. It is an extension of IDBA algorithm.
 IDBA-UD also iterates from small k to a large k. In each iteration, short and low-depth
 contigs are removed iteratively with cutoff threshold from low to high to reduce the errors
 in low-depth and high-depth regions. Paired-end reads are aligned to contigs and assembled
 locally to generate some missing k-mers in low-depth regions. With these technologies, IDBA-UD
 can iterate k value of de Bruijn graph to a very large value with less gaps and less branches
 to form long contigs in both low-depth and high-depth regions.
 - [IGV/2.4.16-Java-1.8](http://www.broadinstitute.org/software/igv/)  
This package contains command line utilities for preprocessing, 
 computing feature count density (coverage),  sorting, and indexing data files.
 See also http://www.broadinstitute.org/software/igv/igvtools_commandline. 
 - [ImageMagick/7.0.8-22-GCCcore-7.3.0](http://www.imagemagick.org/)  
ImageMagick is a software suite to create, edit, compose, or convert bitmap images
 - [JasPer/2.0.14-GCCcore-7.3.0](http://www.ece.uvic.ca/~frodo/jasper/)  
The JasPer Project is an open-source initiative to provide a free
 software-based reference implementation of the codec specified in
 the JPEG-2000 Part-1 standard.

 - [Java/1.8.0_181](http://java.com/)  
Java Platform, Standard Edition (Java SE) lets you develop and deploy
 Java applications on desktops and servers.
 - [Jellyfish/2.2.10-foss-2018b](http://www.genome.umd.edu/jellyfish.html)  
Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA.
 - [Kent_tools/20181218-linux.x86_64](http://genome.cse.ucsc.edu/)  
Kent tools: collection of tools used by the UCSC genome browser.
 - [LAST/963-foss-2018b](http://last.cbrc.jp/)  
LAST finds similar regions between sequences.
 - [LLVM/6.0.0-GCCcore-7.3.0](http://llvm.org/)  
The LLVM Core libraries provide a modern source- and target-independent
 optimizer, along with code generation support for many popular CPUs
 (as well as some less common ones!) These libraries are built around a well
 specified code representation known as the LLVM intermediate representation
 ("LLVM IR"). The LLVM Core libraries are well documented, and it is
 particularly easy to invent your own language (or port an existing compiler)
 to use LLVM as an optimizer and code generator.
 - [LMDB/0.9.22-GCCcore-7.3.0](https://symas.com/lmdb)  
LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance
 of a pure in-memory database while retaining the persistence of standard disk-based databases.
 - [LZO/2.10-foss-2018b](http://www.oberhumer.com/opensource/lzo/)  
Portable lossless data compression library
 - [LibTIFF/4.0.9-GCCcore-7.3.0](http://www.remotesensing.org/libtiff/)  
tiff: Library and tools for reading and writing TIFF data files
 - [LittleCMS/2.9-foss-2018b](http://www.littlecms.com/)  
Little CMS intends to be an OPEN SOURCE small-footprint
 color management engine, with special focus on accuracy and performance. 
 - [M4/1.4.18](http://www.gnu.org/software/m4/m4.html)  
GNU M4 is an implementation of the traditional Unix macro processor. It is
 mostly SVR4 compatible although it has some extensions (for example, handling
 more than 9 positional parameters to macros). GNU M4 also has built-in
 functions for including files, running shell commands, doing arithmetic, etc.

 - [MACS2/2.1.2.1-foss-2018b-Python-2.7.15](http://liulab.dfci.harvard.edu/MACS)  
Model Based Analysis for ChIP-Seq data
 - [MUMmer/4.0.0beta2-foss-2018b](http://mummer.sourceforge.net/)  
MUMmer is a system for rapidly aligning entire genomes,
 whether in complete or draft form. AMOS makes use of it.
 - [MUSCLE/3.8.31-foss-2018b](http://drive5.com/muscle/)  
MUSCLE is one of the best-performing multiple alignment programs 
 according to published benchmark tests, with accuracy and speed that are consistently 
 better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users 
 learn everything they need to know about MUSCLE in a few minutesâ€”only a handful of 
 command-line options are needed to perform common alignment tasks.
 - [Mako/1.0.7-foss-2018b-Python-2.7.15](http://www.makotemplates.org)  
A super-fast templating language that borrows the best ideas from the existing templating languages
 - [Mesa/18.1.1-foss-2018b](http://www.mesa3d.org/)  
Mesa is an open-source implementation of the OpenGL specification -
 a system for rendering interactive 3D graphics.
 - [MiXCR/3.0.3-Java-1.8](https://milaboratory.com/software/mixcr/)  
MiXCR processes big immunome data from raw sequences 
to quantitated clonotypes 
 - [Mothur/1.41.0-foss-2018b-Python-2.7.15](http://www.mothur.org/)  
Mothur is a single piece of open-source, expandable software
 to fill the bioinformatics needs of the microbial ecology community.
 - [NASM/2.13.03-GCCcore-7.3.0](http://www.nasm.us/)  
NASM: General-purpose x86 assembler
 - [NGS/2.9.3-foss-2018b-Java-1.8](https://github.com/ncbi/ngs)  
NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from
 Next Generation Sequencing.
 - [NLopt/2.4.2-GCCcore-7.3.0](http://ab-initio.mit.edu/wiki/index.php/NLopt)  
NLopt is a free/open-source library for nonlinear optimization, 
 providing a common interface for a number of different free optimization routines 
 available online as well as original implementations of various other algorithms. 
 - [NSPR/4.20-GCCcore-7.3.0](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR)  
Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level
 and libc-like functions.
 - [NSS/3.39-GCCcore-7.3.0](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS)  
Network Security Services (NSS) is a set of libraries designed to support cross-platform development
 of security-enabled client and server applications.
 - [OpenBLAS/0.3.1-GCC-7.3.0-2.30](http://xianyi.github.com/OpenBLAS/)  
OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version.
 - [OpenMPI/3.1.1-GCC-7.3.0-2.30](http://www.open-mpi.org/)  
The Open MPI Project is an open source MPI-3 implementation.
 - [OptiType/1.3.2-foss-2018b-Python-2.7.15](https://github.com/FRED-2/OptiType)  
OptiType is a novel HLA genotyping algorithm based on integer linear programming,
 capable of producing accurate 4-digit HLA genotyping predictions from NGS data
 by simultaneously selecting all major and minor HLA Class I alleles. 
 - [PANDAseq/2.11-foss-2018b](https://github.com/neufeld/pandaseq)  
PANDASEQ is a program to align Illumina reads, optionally
 with PCR primers embedded in the sequence, and reconstruct an overlapping
 sequence.
 - [PCRE/8.41-GCCcore-7.3.0](http://www.pcre.org/)  
The PCRE library is a set of functions that implement regular expression
 pattern matching using the same syntax and semantics as Perl 5.

 - [PEAR/0.9.11-foss-2018b](http://sco.h-its.org/exelixis/web/software/pear/)  
PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. 
 It is fully parallelized and can run with as low as just a few kilobytes of memory.
 - [PLINK/2.00-alpha1-x86_64](https://www.cog-genomics.org/plink/1.9/)  
Whole-genome association analysis toolset
 - [PROJ/5.0.0-foss-2018b](http://trac.osgeo.org/proj/)  
Program proj is a standard Unix filter function which converts 
geographic longitude and latitude coordinates into cartesian coordinates
 - [Pango/1.42.4-foss-2018b](http://www.pango.org/)  
Pango is a library for laying out and rendering of text, with an emphasis on internationalization.
Pango can be used anywhere that text layout is needed, though most of the work on Pango so far has been done in the
context of the GTK+ widget toolkit. Pango forms the core of text and font handling for GTK+-2.x.
 - [Perl/5.28.0-GCCcore-7.3.0](https://www.perl.org/)  
Larry Wall's Practical Extraction and Report Language
 - [PyQt5/5.11.3-foss-2018b-Python-3.6.6](http://www.riverbankcomputing.co.uk/software/pyqt)  
PyQt5 is a set of Python bindings for v5 of the Qt application framework from The Qt Company.
 - [PyTables/3.4.4-foss-2018b-Python-2.7.15](http://www.pytables.org)  
PyTables is a package for managing hierarchical datasets and designed to efficiently and easily cope
 with extremely large amounts of data. PyTables is built on top of the HDF5 library, using the Python language and the
 NumPy package. It features an object-oriented interface that, combined with C extensions for the performance-critical
 parts of the code (generated using Cython), makes it a fast, yet extremely easy to use tool for interactively browse,
 process and search very large amounts of data. One important feature of PyTables is that it optimizes memory and disk
 resources so that data takes much less space (specially if on-flight compression is used) than other solutions such as
 relational or object oriented databases.
 - [Pyomo/5.5.0-foss-2018b-Python-2.7.15](http://www.pyomo.org/)  
Pyomo is a Python-based open-source software package that supports a diverse set of optimization
 capabilities for formulating and analyzing optimization models. 
 - [Pysam/0.15.1-foss-2018b-Python-3.6.6](https://github.com/pysam-developers/pysam)  
Pysam is a python module for reading and manipulating Samfiles. 
 It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.
 - [Python/3.6.6-foss-2018b](http://python.org/)  
Python is a programming language that lets you work more quickly and integrate your systems
 more effectively.
 - [Qt/4.8.7-foss-2018b](http://qt.io/)  
Qt is a comprehensive cross-platform C++ application framework.
 - [Qt5/5.10.1-foss-2018b](http://qt.io/)  
Qt is a comprehensive cross-platform C++ application framework.
 - [R/3.5.2-foss-2018b](http://www.r-project.org/)  
R is a free software environment for statistical computing and graphics.
 - [SAMtools/1.9-foss-2018b](http://www.htslib.org/)  
SAM Tools provide various utilities for manipulating alignments in the SAM format, 
 including sorting, merging, indexing and generating alignments in a per-position format.
 - [SKESA/2.3.0-foss-2018b](https://github.com/ncbi/SKESA)  
SKESA is a de-novo sequence read assembler for cultured single isolate genomes based on DeBruijn graphs.
 - [SPAdes/3.13.0-foss-2018b](http://cab.spbu.ru/software/spades/)  
Genome assembler for single-cell and isolates data sets
 - [SPRING/1.6-foss-2018b-Python-2.7.15](http://https://github.com/AllonKleinLab/SPRING_dev)  
SPRING is a collection of pre-processing scripts and a web
browser-based tool for visualizing and interacting with high dimensional data.
View an example dataset here. SPRING was developed for single cell RNA-Seq data
but can be applied more generally. The minimal input is a matrix of high
dimensional data points (cells) and a list of dimension names (genes).
 - [SQLite/3.24.0-GCCcore-7.3.0](http://www.sqlite.org/)  
SQLite: SQL Database Engine in a C Library
 - [SRA-Toolkit/2.9.2-ubuntu64](http://trace.ncbi.nlm.nih.gov/Traces/sra)  
The NCBI SRA Toolkit enables reading (dumping) of sequencing files 
 from the SRA database and writing (loading) files into the .sra format 
 - [STAR/2.6.1c-foss-2018b](https://github.com/alexdobin/STAR)  
STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.
 - [SWIG/3.0.12-foss-2018b-Python-3.6.6](http://www.swig.org/)  
SWIG is a software development tool that connects programs written in C and C++ with
 a variety of high-level programming languages.
 - [ScaLAPACK/2.0.2-gompi-2018b-OpenBLAS-0.3.1](http://www.netlib.org/scalapack/)  
The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines
 redesigned for distributed memory MIMD parallel computers.
 - [Seaborn/0.9.0-foss-2018b-Python-3.6.6](http://stanford.edu/~mwaskom/software/seaborn/)  
Seaborn is a Python visualization library based on matplotlib. 
 It provides a high-level interface for drawing attractive statistical graphics. 
 - [SeqAn/2.4.0-foss-2018b](https://www.seqan.de/)  
SeqAn is an open source C++ library of efficient algorithms and data structures
 for the analysis of sequences with the focus on biological data
 - [Seqmagick/0.6.2-foss-2018b-Python-2.7.15](https://fhcrc.github.io/seqmagick/)  
We often have to convert between sequence formats and do
 little tasks on them, and it's not worth writing scripts for that. Seqmagick
 is a kickass little utility built in the spirit of imagemagick to expose the
 file format conversion in Biopython in a convenient way. Instead of having a
 big mess of scripts, there is one that takes arguments.
 - [Sniffles/1.0.8-foss-2018b](https://github.com/fritzsedlazeck/Sniffles)  
Sniffles is a structural variation caller using third
 generation sequencing (PacBio or Oxford Nanopore). It detects all types of
 SVs (10bp+) using evidence from split-read alignments, high-mismatch regions,
 and coverage analysis.
 - [Sphinx/1.8.1-foss-2018b-Python-3.6.6](http://sphinx.pocoo.org/)  
Sphinx is a tool that makes it easy to create intelligent and beautiful documentation.
 It was originally created for the new Python documentation, and it has excellent facilities for the 
 documentation of Python projects, but C/C++ is already supported as well, and it is planned to add 
 special support for other languages as well.
 - [Szip/2.1.1-GCCcore-7.3.0](http://www.hdfgroup.org/doc_resource/SZIP/)  
Szip compression software, providing lossless compression of scientific data

 - [Tcl/8.6.8-GCCcore-7.3.0](http://www.tcl.tk/)  
Tcl (Tool Command Language) is a very powerful but easy to learn dynamic
 programming language, suitable for a very wide range of uses, including web
 and desktop applications, networking, administration, testing and many more.

 - [Tk/8.6.8-GCCcore-7.3.0](http://www.tcl.tk/)  
Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for
 building a graphical user interface (GUI) in many different programming languages.
 - [Tkinter/3.6.6-foss-2018b-Python-3.6.6](http://python.org/)  
Tkinter module, built with the Python buildsystem
 - [TopHat/2.1.1-foss-2018b](http://ccb.jhu.edu/software/tophat/)  
TopHat is a fast splice junction mapper for RNA-Seq reads.
 - [UDUNITS/2.2.26-foss-2018b](http://www.unidata.ucar.edu/software/udunits/)  
UDUNITS supports conversion of unit specifications between formatted and binary forms,
 arithmetic manipulation of units, and conversion of values between compatible scales of measurement.
 - [VCFtools/0.1.16-foss-2018b-Perl-5.28.0](https://vcftools.github.io)  
The aim of VCFtools is to provide
 easily accessible methods for working with complex
 genetic variation data in the form of VCF files.
 - [VSEARCH/2.9.1-foss-2018b](https://github.com/torognes/vsearch)  
VSEARCH supports de novo and reference based chimera detection,
 clustering, full-length and prefix dereplication, rereplication,
 reverse complementation, masking, all-vs-all pairwise global alignment,
 exact and global alignment searching, shuffling, subsampling and sorting.
 It also supports FASTQ file analysis, filtering,
 conversion and merging of paired-end reads.
 - [VTK/8.1.1-foss-2018b-Python-3.6.6](http://www.vtk.org)  
The Visualization Toolkit (VTK) is an open-source, freely available software system for
 3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several
 interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization
 algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques
 such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation.
 - [ViennaRNA/2.4.10-foss-2018b-Python-2.7.15](http://www.tbi.univie.ac.at/RNA/)  
The Vienna RNA Package consists of a C code library and several
stand-alone programs for the prediction and comparison of RNA secondary structures.
 - [X11/20180604-GCCcore-7.3.0](https://www.x.org)  
The X Window System (X11) is a windowing system for bitmap displays
 - [XML-LibXML/2.0132-GCCcore-7.3.0-Perl-5.28.0](http://search.cpan.org/src/SHLOMIF/XML-LibXML-2.0018/)  
Perl binding for libxml2
 - [XML-Parser/2.44_01-GCCcore-7.3.0-Perl-5.28.0](http://search.cpan.org/~toddr/XML-Parser-2.41/)  
This is a Perl extension interface to James Clark's XML parser, expat.
 - [XZ/5.2.4-GCCcore-7.3.0](http://tukaani.org/xz/)  
xz: XZ utilities
 - [annovar/2018Dec04](http://annovar.openbioinformatics.org/en/latest/)  
ANNOVAR is an efficient software tool to utilize update-to-date
 information to functionally annotate genetic variants detected from diverse
 genomes (including human genome hg18, hg19, hg38, as well as mouse, worm, fly,
 yeast and many others).
 - [bam2wig/1.5](https://github.com/MikeAxtell/bam2wig)  
Conversion of a BAM alignment to wiggle and bigwig coverage files, with flexible reporting options.
 - [beagle-lib/3.0.2-foss-2018b](https://github.com/beagle-dev/beagle-lib)  
beagle-lib is a high-performance library that can perform the core calculations at the heart of most
 Bayesian and Maximum Likelihood phylogenetics packages.
 - [binutils/2.30](http://directory.fsf.org/project/binutils/)  
binutils: GNU binary utilities
 - [bioawk/1.0-foss-2018b](https://github.com/lh3/bioawk)  
Bioawk is an extension to Brian Kernighan's awk,
 adding the support of several common biological data formats,
 including optionally gzip'ed BED, GFF, SAM, VCF, FASTA/Q and TAB-delimited formats with column names. 
 - [bzip2/1.0.6-GCCcore-7.3.0](https://sourceware.org/bzip2)  
bzip2 is a freely available, patent free, high-quality data compressor. It
 typically compresses files to within 10% to 15% of the best available
 techniques (the PPM family of statistical compressors), whilst being around
 twice as fast at compression and six times faster at decompression.

 - [cURL/7.60.0-GCCcore-7.3.0](http://curl.haxx.se)  
libcurl is a free and easy-to-use client-side URL transfer library,
 supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP,
 LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.
 libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP
 form based upload, proxies, cookies, user+password authentication (Basic,
 Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling
 and more.

 - [cairo/1.14.12-GCCcore-7.3.0](http://cairographics.org)  
Cairo is a 2D graphics library with support for multiple output devices.
 Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers,
 PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB
 - [canu/1.8-foss-2018b](http://canu.readthedocs.io)  
Canu is a fork of the Celera Assembler designed for
 high-noise single-molecule sequencing
 - [cellranger/2.1.1](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest)  
Single Cell Analysis Pipelines
 - [cutadapt/1.18-foss-2018b-Python-3.6.6](http://opensource.scilifelab.se/projects/cutadapt/)  
Cutadapt finds and removes adapter sequences, primers, poly-A tails and
 other types of unwanted sequence from your high-throughput sequencing reads.
 - [expat/2.2.5-GCCcore-7.3.0](http://expat.sourceforge.net/)  
Expat is an XML parser library written in C. It is a stream-oriented parser
 in which an application registers handlers for things the parser might find
 in the XML document (like start tags)

 - [factera/1.4.4-foss-2018b-Perl-5.28.0](https://factera.stanford.edu/)  
(Fusion And Chromosomal Translocation Enumeration and Recovery Algorithm) is a tool for detection of genomic fusions in paired-end targeted (or genome-wide) sequencing data.
 - [fast5/0.6.5](http://simpsonlab.github.io/2017/02/27/packing_fast5/)  
A lightweight C++ library for accessing Oxford Nanopore
 Technologies sequencing data.
 - [fastq-tools/0.8-foss-2018b](https://homes.cs.washington.edu/~dcjones/fastq-tools/)  
This package provides a number of small and efficient programs to perform
 common tasks with high throughput sequencing data in the FASTQ format. All of the programs
 work with typical FASTQ files as well as gzipped FASTQ files.
 - [file/5.35-GCCcore-7.3.0](http://www.darwinsys.com/file/)  
The file command is 'a file type guesser', that is, a command-line tool
 that tells you in words what kind of data a file contains.
 - [flex/2.6.4](http://flex.sourceforge.net/)  
Flex (Fast Lexical Analyzer) is a tool for generating scanners. A scanner,
 sometimes called a tokenizer, is a program which recognizes lexical patterns
 in text.

 - [fontconfig/2.13.0-GCCcore-7.3.0](http://www.freedesktop.org/software/fontconfig)  
Fontconfig is a library designed to provide system-wide font configuration,
 customization and application access.

 - [foss/2018b]((none))  
GNU Compiler Collection (GCC) based compiler toolchain, including
 OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.
 - [freeglut/3.0.0-foss-2018b](http://freeglut.sourceforge.net/)  
freeglut is a completely OpenSourced alternative to the OpenGL Utility Toolkit (GLUT) library.
 - [freetype/2.9.1-GCCcore-7.3.0](http://freetype.org)  
FreeType 2 is a software font engine that is designed to be small, efficient,
 highly customizable, and portable while capable of producing high-quality
 output (glyph images). It can be used in graphics libraries, display servers,
 font conversion tools, text image generation tools, and many other products
 as well.

 - [future/0.16.0-foss-2018b-Python-2.7.15](http://python-future.org/)  
python-future is the missing compatibility layer between Python 2 and Python 3.
 - [gc/7.6.4-GCCcore-7.3.0](http://hboehm.info/gc/)  
The Boehm-Demers-Weiser conservative garbage collector can be used as a
 garbage collecting replacement for C malloc or C++ new.

 - [gettext/0.19.8.1](http://www.gnu.org/software/gettext/)  
GNU 'gettext' is an important step for the GNU Translation Project, as it is an asset on which we may
build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools
and documentation
 - [gnuplot/5.2.5-foss-2018b](http://gnuplot.sourceforge.net/)  
Portable interactive, function plotting utility
 - [gompi/2018b]((none))  
GNU Compiler Collection (GCC) based compiler toolchain,
 including OpenMPI for MPI support.
 - [gperf/3.1-GCCcore-7.3.0](http://www.gnu.org/software/gperf/)  
GNU gperf is a perfect hash function generator. For a given list of strings,
 it produces a hash function and hash table, in form of C or C++ code, for
 looking up a value depending on the input string. The hash function is
 perfect, which means that the hash table has no collisions, and the hash
 table lookup needs a single string comparison only.

 - [help2man/1.47.4](https://www.gnu.org/software/help2man/)  
help2man produces simple manual pages from the '--help' and '--version'
 output of other commands.

 - [hwloc/1.11.10-GCCcore-7.3.0](http://www.open-mpi.org/projects/hwloc/)  
The Portable Hardware Locality (hwloc) software package provides a portable
 abstraction (across OS, versions, architectures, ...) of the hierarchical
 topology of modern architectures, including NUMA memory nodes, sockets, shared
 caches, cores and simultaneous multithreading. It also gathers various system
 attributes such as cache and memory information as well as the locality of I/O
 devices such as network interfaces, InfiniBand HCAs or GPUs. It primarily
 aims at helping applications with gathering information about modern computing
 hardware so as to exploit it accordingly and efficiently.

 - [iCount/20180820-foss-2018b-Python-3.6.6](https://github.com/tomazc/iCount)  
iCount: protein-RNA interaction analysis
 is a Python module and associated command-line interface (CLI),
 which provides all the commands needed to process iCLIP data on protein-RNA interactions.
 - [intltool/0.51.0-GCCcore-7.3.0-Perl-5.28.0](http://freedesktop.org/wiki/Software/intltool/)  
intltool is a set of tools to centralize translation of
 many different file formats using GNU gettext-compatible PO files.
 - [kallisto/0.45.0-foss-2018b](http://pachterlab.github.io/kallisto/)  
kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally
 of target sequences using high-throughput sequencing reads.
 - [libGLU/9.0.0-foss-2018b](ftp://ftp.freedesktop.org/pub/mesa/glu/)  
The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. 
 - [libcerf/1.7-GCCcore-7.3.0](http://apps.jcns.fz-juelich.de/doku/sc/libcerf)  
libcerf is a self-contained numeric library that provides an efficient and
 accurate implementation of complex error functions, along with Dawson,
 Faddeeva, and Voigt functions.

 - [libdrm/2.4.92-GCCcore-7.3.0](http://dri.freedesktop.org)  
Direct Rendering Manager runtime library.
 - [libevent/2.1.8-GCCcore-7.3.0](http://libevent.org/)  
The libevent API provides a mechanism to execute a callback function when
 a specific event occurs on a file descriptor or after a timeout has been
 reached.  Furthermore, libevent also support callbacks due to signals or
 regular timeouts.

 - [libffi/3.2.1-GCCcore-7.3.0](http://sourceware.org/libffi/)  
The libffi library provides a portable, high level programming interface to
 various calling conventions. This allows a programmer to call any function
 specified by a call interface description at run-time.

 - [libgd/2.2.5-GCCcore-7.3.0](https://libgd.github.io/)  
GD is an open source code library for the dynamic creation of images by programmers.
 - [libgeotiff/1.4.2-foss-2018b](https://directory.fsf.org/wiki/Libgeotiff)  
Library for reading and writing coordinate system information from/to GeoTIFF files
 - [libgtextutils/0.7-GCCcore-7.3.0](http://hannonlab.cshl.edu/fastx_toolkit/)  
ligtextutils is a dependency of fastx-toolkit and is provided
 via the same upstream
 - [libharu/2.3.0-GCCcore-7.3.0](http://libharu.org/)  
libHaru is a free, cross platform, open source library for generating PDF files.
 - [libjpeg-turbo/2.0.0-GCCcore-7.3.0](http://sourceforge.net/projects/libjpeg-turbo/)  
libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to
 accelerate baseline JPEG compression and decompression. libjpeg is a library
 that implements JPEG image encoding, decoding and transcoding.

 - [libpciaccess/0.14-GCCcore-7.3.0](http://cgit.freedesktop.org/xorg/lib/libpciaccess/)  
Generic PCI access library.
 - [libpng/1.6.34-GCCcore-7.3.0](http://www.libpng.org/pub/png/libpng.html)  
libpng is the official PNG reference library
 - [libpthread-stubs/0.4-GCCcore-7.3.0](http://xcb.freedesktop.org/)  
The X protocol C-language Binding (XCB) is a replacement for Xlib featuring
 a small footprint, latency hiding, direct access to the protocol, improved
 threading support, and extensibility.

 - [libreadline/7.0-GCCcore-7.3.0](http://cnswww.cns.cwru.edu/php/chet/readline/rltop.html)  
The GNU Readline library provides a set of functions for use by applications
 that allow users to edit command lines as they are typed in. Both Emacs and
 vi editing modes are available. The Readline library includes additional
 functions to maintain a list of previously-entered command lines, to recall
 and perhaps reedit those lines, and perform csh-like history expansion on
 previous commands.

 - [libsndfile/1.0.28-GCCcore-7.3.0](http://www.mega-nerd.com/libsndfile)  
Libsndfile is a C library for reading and writing files containing sampled sound
 (such as MS Windows WAV and the Apple/SGI AIFF format) through one standard library interface.
 - [libtool/2.4.6-GCCcore-7.3.0](http://www.gnu.org/software/libtool)  
GNU libtool is a generic library support script. Libtool hides the complexity
 of using shared libraries behind a consistent, portable interface.

 - [libunistring/0.9.10-GCCcore-7.3.0](http://www.gnu.org/software/libunistring/)  
This library provides functions for manipulating Unicode strings and for
 manipulating C strings according to the Unicode standard.

 - [libxml2/2.9.8-GCCcore-7.3.0](http://xmlsoft.org/)  
Libxml2 is the XML C parser and toolchain developed for the Gnome project
 (but usable outside of the Gnome platform).

 - [libxslt/1.1.32-GCCcore-7.3.0](http://xmlsoft.org/)  
Libxslt is the XSLT C library developed for the GNOME project
 (but usable outside of the Gnome platform).
 - [lumpy/0.2.13-foss-2018b](https://github.com/pezmaster31/bamtools)  
A probabilistic framework for structural variant discovery.
 - [matplotlib/3.0.0-foss-2018b-Python-3.6.6](http://matplotlib.org)  
matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
 hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
 and ipython shell, web application servers, and six graphical user interface toolkits.
 - [minimap2/2.13-foss-2018b](https://github.com/lh3/minimap2)  
Minimap2 is a fast sequence mapping and alignment
program that can find overlaps between long noisy reads, or map long
reads or their assemblies to a reference genome optionally with detailed
alignment (i.e. CIGAR). At present, it works efficiently with query
sequences from a few kilobases to ~100 megabases in length at an error
rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited
test data sets, minimap2 is over 20 times faster than most other
long-read aligners. It will replace BWA-MEM for long reads and contig
alignment.
 - [nanopolish/0.10.2-foss-2018b](http://simpsonlab.github.io/2015/10/07/nanopolish-v0.4.0/)  
Our computational work focuses on the development of
algorithms for handling very large sequencing data sets, from indexing
and text compression, to probabilistic modeling and inference. We make
our methods freely available as open source software.
 - [ncbi-vdb/2.9.3-foss-2018b](https://github.com/ncbi/ncbi-vdb)  
The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for
 using data in the INSDC Sequence Read Archives.
 - [ncurses/6.1-GCCcore-7.3.0](http://www.gnu.org/software/ncurses/)  
The Ncurses (new curses) library is a free software emulation of curses in
 System V Release 4.0, and more. It uses Terminfo format, supports pads and
 color and multiple highlights and forms characters and function-key mapping,
 and has all the other SYSV-curses enhancements over BSD Curses.

 - [netCDF/4.6.1-foss-2018b](http://www.unidata.ucar.edu/software/netcdf/)  
NetCDF (network Common Data Form) is a set of software libraries 
 and machine-independent data formats that support the creation, access, and sharing of array-oriented 
 scientific data.
 - [nettle/3.4-foss-2018b](http://www.lysator.liu.se/~nisse/nettle/)  
Nettle is a cryptographic library that is designed to fit easily
 in more or less any context: In crypto toolkits for object-oriented
 languages (C++, Python, Pike, ...), in applications like LSH or GNUPG,
 or even in kernel space.
 - [numactl/2.0.11-GCCcore-7.3.0](http://oss.sgi.com/projects/libnuma/)  
The numactl program allows you to run your application program on specific
 cpu's and memory nodes. It does this by supplying a NUMA memory policy to
 the operating system before running your program. The libnuma library provides
 convenient ways for you to add NUMA memory policies into your own program.

 - [numexpr/2.6.5-foss-2018b-Python-2.7.15](http://code.google.com/p/numexpr/)  
The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.
 It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into
 code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a
 compiler at runtime.
 - [ont-guppy-cpu/2.1.3](https://nanoporetech.com/products/minit)  
Guppy software supports MinIT and MinION instruments from
 Nanopore Technologies
 - [picard/2.18.17-Java-1.8](http://sourceforge.net/projects/picard)  
A set of tools (in Java) for working with next generation sequencing data in the BAM format.
 - [pixman/0.34.0-GCCcore-7.3.0](http://www.pixman.org/)  
Pixman is a low-level software library for pixel manipulation, providing
 features such as image compositing and trapezoid rasterization. Important
 users of pixman are the cairo graphics library and the X server.

 - [pkg-config/0.29.2-GCCcore-7.3.0](http://www.freedesktop.org/wiki/Software/pkg-config/)  
pkg-config is a helper tool used when compiling applications and libraries.
 It helps you insert the correct compiler options on the command line so an
 application can use gcc -o test test.c `pkg-config --libs --cflags glib-2.0`
 for instance, rather than hard-coding values on where to find glib (or other
 libraries).

 - [prokka/1.13-foss-2018b-BioPerl-1.7.2](http://www.vicbioinformatics.com/software.prokka.shtml)  
Prokka is a software tool for the rapid annotation of prokaryotic genomes.
 - [psipred/4.02-foss-2018b](http://bioinf.cs.ucl.ac.uk/psipred/)  
The PSIPRED Protein Sequence Analysis Workbench aggregates
 several UCL structure prediction methods into one location.
 - [pybedtools/0.7.10-foss-2018b-Python-3.6.6](https://daler.github.io/pybedtools)  
pybedtools wraps and extends BEDTools and offers feature-level manipulations from within Python.
 - [samblaster/0.1.24-foss-2018b](https://github.com/GregoryFaust/samblaster)  
samblaster is a fast and flexible program for marking
 duplicates in read-id grouped1 paired-end SAM files.
 - [strelka/2.9.9-foss-2018b](https://github.com/Illumina/strelka)  
Strelka2 is a fast and accurate small variant caller
 optimized for analysis of germline variation in small cohorts and
 somatic variation in tumor/normal sample pairs.
 - [tabix/0.2.6-GCCcore-7.3.0](http://samtools.sourceforge.net)  
Generic indexer for TAB-delimited genome position files 
 - [tbb/2018_U5-GCCcore-7.3.0](https://01.org/tbb/)  
Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that
 take full advantage of multicore performance, that are portable, composable and have future-proof scalability.
 - [tcsh/6.20.00-GCCcore-7.3.0](http://www.tcsh.org)  
Tcsh is an enhanced, but completely compatible version of the Berkeley UNIX C shell (csh). 
 It is a command language interpreter usable both as an interactive login shell and a shell script command 
 processor. It includes a command-line editor, programmable word completion, spelling correction, a history 
 mechanism, job control and a C-like syntax.
 - [tmux/2.8-GCC-7.3.0-2.30](http://tmux.sourceforge.net/)  
tmux is a terminal multiplexer. It lets you switch easily 
between several programs in one terminal, detach them (they keep running in the background) and reattach them to a different terminal.
 - [util-linux/2.32-GCCcore-7.3.0](http://www.kernel.org/pub/linux/utils/util-linux)  
Set of Linux utilities
 - [xbitmaps/1.1.2](http://www.freedesktop.org/wiki/Software/xlibs)  
provides bitmaps for x
 - [xorg-macros/1.19.2-GCCcore-7.3.0](http://cgit.freedesktop.org/xorg/util/macros)  
X.org macros utilities.
 - [zlib/1.2.11](http://www.zlib.net/)  
zlib is designed to be a free, general-purpose, legally unencumbered -- that
 is, not covered by any patents -- lossless data-compression library for use
 on virtually any computer hardware and operating system.

