---
layout: post
title: Bio Modules  16.04
date: 2018-11-02
---

 - [BBMap/38.26-foss-2018b](https://sourceforge.net/projects/bbmap/)  
BBMap short read aligner, and other bioinformatic tools.
 - [BCFtools/1.9-foss-2018b](http://www.htslib.org/)  
Samtools is a suite of programs for interacting with high-throughput sequencing data.
 BCFtools
 - [BEDOPS/2.4.35-foss-2018b](http://bedops.readthedocs.io/en/latest/index.html)  
BEDOPS is an open-source command-line toolkit that performs highly efficient and
 scalable Boolean and other set operations, statistical calculations, archiving, conversion and
 other management of genomic data of arbitrary scale. Tasks can be easily split by chromosome for
 distributing whole-genome analyses across a computational cluster.
 - [BEDTools/2.27.1-foss-2018b](https://github.com/arq5x/bedtools2)  
The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps
 and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF,
 and SAM/BAM.
 - [BWA/0.7.17-foss-2018b](http://bio-bwa.sourceforge.net/)  
Burrows-Wheeler Aligner (BWA) is an efficient program that aligns
 relatively short nucleotide sequences against a long reference sequence such as the human genome.
 - [BamTools/2.5.1-foss-2018b](https://github.com/pezmaster31/bamtools)  
BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.
 - [Beast/2.5.1-foss-2018b](http://beast2.org/)  
BEAST is a cross-platform program for Bayesian MCMC analysis of molecular 
 sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using 
 strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies 
 but is also a framework for testing evolutionary hypotheses without conditioning on a single 
 tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted 
 proportional to its posterior probability. 
 - [Biopython/1.72-foss-2018b-Python-2.7.15](http://www.biopython.org)  
Biopython is a set of freely available tools for biological
 computation written in Python by an international team of developers. It is
 a distributed collaborative effort to develop Python libraries and
 applications which address the needs of current and future work in
 bioinformatics. 
 - [Bowtie2/2.3.4.2-foss-2018b](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml)  
Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads
 to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s
 of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.
 Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,
 its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.
 - [DIAMOND/0.9.22-foss-2018b](https://github.com/bbuchfink/diamond)  
Accelerated BLAST compatible local sequence aligner
 - [FLASH/2.2.00-foss-2018b](https://ccb.jhu.edu/software/FLASH/)  
FLASH (Fast Length Adjustment of SHort reads) is a very fast
and accurate software tool to merge paired-end reads from next-generation
sequencing experiments. FLASH is designed to merge pairs of reads when the
original DNA fragments are shorter than twice the length of reads. The
resulting longer reads can significantly improve genome assemblies. They can
also improve transcriptome assembly when FLASH is used to merge RNA-seq data.

 - [FastQC/0.11.8-Java-1.8.0_181](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/)  
FastQC is a quality control application for high throughput
sequence data. It reads in sequence data in a variety of formats and can either
provide an interactive application to review the results of several different
QC checks, or create an HTML based report which can be integrated into a
pipeline.
 - [FastTree/2.1.10-foss-2018b](http://www.microbesonline.org/fasttree/)  
FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide
 or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of
 time and memory. 
 - [GATK/4.0.10.0-foss-2018b-Python-3.6.6](http://www.broadinstitute.org/gatk/)  
The Genome Analysis Toolkit or GATK is a software package
developed at the Broad Institute to analyse next-generation resequencing
data. The toolkit offers a wide variety of tools, with a primary focus on
variant discovery and genotyping as well as strong emphasis on data quality
assurance. Its robust architecture, powerful processing engine and
high-performance computing features make it capable of taking on projects
of any size.
 - [GEOS/3.6.2-foss-2018b-Python-3.7.0](http://trac.osgeo.org/geos)  
GEOS (Geometry Engine
 - [GMP/6.1.2-GCCcore-7.3.0](http://gmplib.org/)  
GMP is a free library for arbitrary precision arithmetic, operating on signed
 integers, rational numbers, and floating point numbers.

 - [HMMER/3.2.1-foss-2018b](http://hmmer.org/)  
HMMER is used for searching sequence databases for homologs
 of protein sequences, and for making protein sequence alignments. It
 implements methods using probabilistic models called profile hidden Markov
 models (profile HMMs).  Compared to BLAST, FASTA, and other sequence
 alignment and database search tools based on older scoring methodology,
 HMMER aims to be significantly more accurate and more able to detect remote
 homologs because of the strength of its underlying mathematical models. In the
 past, this strength came at significant computational expense, but in the new
 HMMER3 project, HMMER is now essentially as fast as BLAST.
 - [HTSlib/1.9-foss-2018b](http://www.htslib.org/)  
A C library for reading/writing high-throughput sequencing data.
 This package includes the utilities bgzip and tabix
 - [JAGS/4.2.0-foss-2016b](http://mcmc-jags.sourceforge.net/)  
JAGS is Just Another Gibbs Sampler.  It is a program for analysis 
 of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  
 - [MUMmer/4.0.0beta2-foss-2018b](http://mummer.sourceforge.net/)  
MUMmer is a system for rapidly aligning entire genomes,
 whether in complete or draft form. AMOS makes use of it.
 - [PEAR/0.9.11-foss-2018b](http://sco.h-its.org/exelixis/web/software/pear/)  
PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. 
 It is fully parallelized and can run with as low as just a few kilobytes of memory.
 - [PHASE/2.1.1](http://stephenslab.uchicago.edu/phase/download.html)  
The program PHASE implements a Bayesian statistical 
method for reconstructing haplotypes from population genotype data.
Documentation: http://stephenslab.uchicago.edu/assets/software/phase/instruct2.1.pdf
 - [SAMtools/1.9-foss-2018b](http://www.htslib.org/)  
SAM Tools provide various utilities for manipulating alignments in the SAM format, 
 including sorting, merging, indexing and generating alignments in a per-position format.
 - [Seqmagick/0.6.2-foss-2018b-Python-2.7.15](https://fhcrc.github.io/seqmagick/)  
We often have to convert between sequence formats and do little tasks on them, and it's not worth 
 writing scripts for that. Seqmagick is a kickass little utility built in the spirit of imagemagick to expose the 
 file format conversion in Biopython in a convenient way. Instead of having a big mess of scripts, there is one that 
 takes arguments.

 - [strelka/2.9.9-foss-2018b](https://github.com/Illumina/strelka)  
Strelka2 is a fast and accurate small variant caller
 optimized for analysis of germline variation in small cohorts and
 somatic variation in tumor/normal sample pairs.
