---
title: Bio Modules 24.04
layout: single
permalink: /bio-modules-24.04/
created: 2024-11-22
toc: true
toc_label: "On This Page"
sidebar:
  nav: "docs"
---

 - [AGAT/1.4.0-GCC-12.3.0](https://agat.readthedocs.io/en/latest/)
AGAT: Another GTF/GFF Analysis Toolkit. Suite of tools to handle gene annotations
 in any GTF/GFF format.
 - [AlphaFold/2.3.2-foss-2023a-CUDA-12.1.1](https://deepmind.com/research/case-studies/alphafold)
AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known
 - [Arriba/2.4.0-GCC-12.2.0](https://github.com/suhrig/arriba)
Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data.
 It was developed for the use in a clinical research setting. Therefore, short runtimes and high
 sensitivity were important design criteria.
 - [BAli-Phy/4.0-beta8-gfbf-2022b](http://www.bali-phy.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/BAli-Phy/BAli-Phy-4.0-beta8-gfbf-2022b.eb)
BAli-Phy estimates multiple sequence alignments and evolutionary trees from DNA, amino acid, 
or codon sequences.
 - [BCFtools/1.19-GCC-13.2.0](https://www.htslib.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/BCFtools/BCFtools-1.19-GCC-13.2.0.eb)
Samtools is a suite of programs for interacting with high-throughput sequencing data.
 BCFtools
 - [BEDTools/2.31.0-GCC-12.3.0](https://bedtools.readthedocs.io/)
BEDTools: a powerful toolset for genome arithmetic.
The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and
computing coverage.
The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM.
 - [BLAST+/2.14.0-gompi-2022b](https://blast.ncbi.nlm.nih.gov/)
Basic Local Alignment Search Tool, or BLAST, is an algorithm
 for comparing primary biological sequence information, such as the amino-acid
 sequences of different proteins or the nucleotides of DNA sequences.
 - [BWA/0.7.17-GCCcore-12.2.0](http://bio-bwa.sourceforge.net/)
Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively
 short nucleotide sequences against a long reference sequence such as the human
 genome.

 - [BamTools/2.5.2-GCC-12.2.0](https://github.com/pezmaster31/bamtools)
BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.
 - [Beast/10.5.0-beta3-GCC-12.3.0-beagle-lib-4.0.1-CUDA-12.1.1](https://beast.community)
BEAST is a cross-platform program for Bayesian analysis of molecular sequences
 using MCMC. It is entirely orientated towards rooted, time-measured phylogenies inferred using
 strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies
 but is also a framework for testing evolutionary hypotheses without conditioning on a single
 tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted
 proportional to its posterior probability. 
 - [Beast2/2.7.7-GCC-12.3.0-beagle-lib-4.0.1-CUDA-12.1.1](https://beast2.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/Beast2/Beast2-2.7.7-GCC-12.3.0-beagle-lib-4.0.1-CUDA-12.1.1.eb)
BEAST is a cross-platform program for Bayesian MCMC analysis of molecular 
 sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using 
 strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies 
 but is also a framework for testing evolutionary hypotheses without conditioning on a single 
 tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted 
 proportional to its posterior probability. 
 - [Bio-DB-HTS/3.01-GCC-12.2.0](https://metacpan.org/release/Bio-DB-HTS)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-12.2.0.eb)
Read files using HTSlib including BAM/CRAM, Tabix and BCF database files
 - [BioPerl/1.7.8-GCCcore-12.2.0](http://www.bioperl.org/)
Bioperl is the product of a community effort to produce Perl code which is useful in biology.
 Examples include Sequence objects, Alignment objects and database searching objects.
 - [Biopython/1.84-foss-2023b](https://www.biopython.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/Biopython/Biopython-1.84-foss-2023b.eb)
Biopython is a set of freely available tools for biological
 computation written in Python by an international team of developers. It is
 a distributed collaborative effort to develop Python libraries and
 applications which address the needs of current and future work in
 bioinformatics. 
 - [Bismark/0.24.1-GCC-12.2.0](https://www.bioinformatics.babraham.ac.uk/projects/bismark/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/Bismark/Bismark-0.24.1-GCC-12.2.0.eb)
A tool to map bisulfite converted sequence reads and determine cytosine methylation states
 - [Bowtie2/2.5.4-GCC-13.2.0](https://github.com/BenLangmead/bowtie2)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/Bowtie2/Bowtie2-2.5.4-GCC-13.2.0.eb)
Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads
 to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s
 of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.
 Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,
 its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.
 - [CITE-seq-Count/1.4.4-foss-2023b-Python-3.11.5](https://github.com/Hoohm/CITE-seq-Count)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/CITE-seq-Count/CITE-seq-Count-1.4.4-foss-2023b-Python-3.11.5.eb)
A python package that allows to count antibody TAGS from a CITE-seq and/or cell hashing experiment.
 - [CRISPResso2/2.3.1-foss-2023b](https://github.com/pinellolab/CRISPResso2/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/CRISPResso2/CRISPResso2-2.3.1-foss-2023b.eb)
CRISPResso2 is a software pipeline designed to enable rapid and
intuitive interpretation of genome editing experiments.
 - [Cassiopeia/2.0.0-foss-2023a](https://github.com/YosefLab/Cassiopeia)
A Package for Cas9-Enabled Single Cell Lineage Tracing Tree Reconstruction.
 - [Cbc/2.10.11-foss-2023a](https://github.com/coin-or/Cbc)
Cbc (Coin-or branch and cut) is an open-source mixed integer linear programming
solver written in C++. It can be used as a callable library or using a
stand-alone executable.
 - [CellBender/0.3.0-foss-2023a](http://github.com/broadinstitute/CellBender)
CellBender is a software package for eliminating technical artifacts from
high-throughput single-cell RNA sequencing (scRNA-seq) data.

 - [CellRank/2.0.2-foss-2023a-CUDA-12.1.1](https://cellrank.readthedocs.io/en/stable/)
CellRank is a toolkit to uncover cellular dynamics based on
 Markov state modeling of single-cell data.  It contains two main modules:
kernels compute cell-cell transition probabilities and estimators generate
hypothesis based on these. 
 - [Cgl/0.60.8-foss-2023a](https://github.com/coin-or/Cgl)
The COIN-OR Cut Generation Library (Cgl) is a collection of cut generators that
can be used with other COIN-OR packages that make use of cuts, such as, among
others, the linear solver Clp or the mixed integer linear programming solvers
Cbc or BCP. Cgl uses the abstract class OsiSolverInterface (see Osi) to use or
communicate with a solver. It does not directly call a solver.
 - [Clp/1.17.9-foss-2023b](https://github.com/coin-or/Clp)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/Clp/Clp-1.17.9-foss-2023b.eb)
Clp (Coin-or linear programming) is an open-source linear programming solver.
It is primarily meant to be used as a callable library, but a basic,
stand-alone executable version is also available.
 - [CoinUtils/2.11.10-GCC-13.2.0](https://github.com/coin-or/CoinUtils)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/c/CoinUtils/CoinUtils-2.11.10-GCC-13.2.0.eb)
CoinUtils (Coin-OR Utilities) is an open-source collection of classes and
functions that are generally useful to more than one COIN-OR project.
 - [Eigen/3.4.0-GCCcore-13.3.0](https://eigen.tuxfamily.org)
Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers,
 and related algorithms.
 - [FASTA/36.3.8i-GCC-12.2.0](https://fasta.bioch.virginia.edu/fasta_www2/fasta_list2.shtml)
The FASTA programs find regions of local or global (new) similarity between 
protein or DNA sequences, either by searching Protein or DNA databases, or by identifying 
local duplications within a sequence.
 - [FastQC/0.12.1-Java-11](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
FastQC is a quality control application for high throughput
sequence data. It reads in sequence data in a variety of formats and can either
provide an interactive application to review the results of several different
QC checks, or create an HTML based report which can be integrated into a
pipeline.
 - [Flax/0.8.4-gfbf-2023a-CUDA-12.1.1](https://flax.readthedocs.io)
Flax is a high-performance neural network library and ecosystem for JAX that is
designed for flexibility: Try new forms of training by forking an example and
by modifying the training loop, not by adding features to a framework.
 - [GATK/4.4.0.0-GCCcore-12.2.0-Java-17](https://www.broadinstitute.org/gatk/)
The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute
 to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,
 with a primary focus on variant discovery and genotyping as well as strong emphasis on
 data quality assurance. Its robust architecture, powerful processing engine and
 high-performance computing features make it capable of taking on projects of any size.
 - [GEOS/3.12.1-GCC-13.2.0](https://trac.osgeo.org/geos)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/g/GEOS/GEOS-3.12.1-GCC-13.2.0.eb)
GEOS (Geometry Engine
 - [GMP/6.3.0-GCCcore-13.2.0](https://gmplib.org/)
GMP is a free library for arbitrary precision arithmetic, operating on signed
 integers, rational numbers, and floating point numbers.

 - [HH-suite/3.3.0-gompi-2023a](https://github.com/soedinglab/hh-suite)
The HH-suite is an open-source software package
 for sensitive protein sequence searching based on the pairwise
 alignment of hidden Markov models (HMMs).
 - [HMMER/3.4-gompi-2024a](http://hmmer.org/)
HMMER is used for searching sequence databases for homologs
 of protein sequences, and for making protein sequence alignments. It
 implements methods using probabilistic models called profile hidden Markov
 models (profile HMMs).  Compared to BLAST, FASTA, and other sequence
 alignment and database search tools based on older scoring methodology,
 HMMER aims to be significantly more accurate and more able to detect remote
 homologs because of the strength of its underlying mathematical models. In the
 past, this strength came at significant computational expense, but in the new
 HMMER3 project, HMMER is now essentially as fast as BLAST.
 - [HTSlib/1.19.1-GCC-13.2.0](https://www.htslib.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/h/HTSlib/HTSlib-1.19.1-GCC-13.2.0.eb)
A C library for reading/writing high-throughput sequencing data.
 This package includes the utilities bgzip and tabix
 - [IgBLAST/1.22.0-x64-linux](https://ncbi.github.io/igblast)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/i/IgBLAST/IgBLAST-1.22.0-x64-linux.eb)
IgBLAST faclilitates the analysis of immunoglobulin and T cell receptor variable domain sequences.
 - [InChI/1.07.1-GCC-13.3.0](https://www.inchi-trust.org/)
The IUPAC International Chemical Identifier (InChI TM) is a non-proprietary
identifier for chemical substances that can be used in printed and electronic
data sources thus enabling easier linking of diverse data compilations.
 - [Infernal/1.1.4-foss-2022b](http://eddylab.org/infernal/)
Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases
 for RNA structure and sequence similarities.
 - [JAGS/4.3.2-foss-2023b](http://mcmc-jags.sourceforge.net/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/j/JAGS/JAGS-4.3.2-foss-2023b.eb)
JAGS is Just Another Gibbs Sampler.  It is a program for analysis
 of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  
 - [Kalign/3.4.0-GCCcore-12.3.0](https://github.com/TimoLassmann/kalign)
Kalign is a fast multiple sequence alignment program for biological sequences.
 - [Kent_tools/468-GCC-12.3.0](https://genome.cse.ucsc.edu/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/k/Kent_tools/Kent_tools-468-GCC-12.3.0.eb)
Kent utilities: collection of tools used by the UCSC genome browser.
 - [Kraken2/2.1.3-gompi-2022b](https://github.com/DerrickWood/kraken2/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/k/Kraken2/Kraken2-2.1.3-gompi-2022b.eb)
Kraken is a system for assigning taxonomic labels to short DNA sequences,
 usually obtained through metagenomic studies. Previous attempts by other
 bioinformatics software to accomplish this task have often used sequence
 alignment or machine learning techniques that were quite slow, leading to
 the development of less sensitive but much faster abundance estimation
 programs. Kraken aims to achieve high sensitivity and high speed by
 utilizing exact alignments of k-mers and a novel classification algorithm.
 - [MACS2/2.2.9.1-foss-2022b](https://github.com/taoliu/MACS)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/MACS2/MACS2-2.2.9.1-foss-2022b.eb)
Model Based Analysis for ChIP-Seq data
 - [MACS3/3.0.0-foss-2022b](https://macs3-project.github.io/MACS/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/MACS3/MACS3-3.0.0-foss-2022b.eb)
Model Based Analysis for ChIP-Seq data
 - [METIS/5.1.0-GCCcore-12.3.0](http://glaros.dtc.umn.edu/gkhome/metis/metis/overview)
METIS is a set of serial programs for partitioning graphs, partitioning
 finite element meshes, and producing fill reducing orderings for sparse
 matrices. The algorithms implemented in METIS are based on the multilevel
 recursive-bisection, multilevel k-way, and multi-constraint partitioning
 schemes.

 - [MPC/1.3.1-GCCcore-12.3.0](http://www.multiprecision.org/)
Gnu Mpc is a C library for the arithmetic of
 complex numbers with arbitrarily high precision and correct
 rounding of the result. It extends the principles of the IEEE-754
 standard for fixed precision real floating point numbers to
 complex numbers, providing well-defined semantics for every
 operation. At the same time, speed of operation at high precision
 is a major design goal.
 - [MPFR/4.2.1-GCCcore-13.2.0](https://www.mpfr.org)
The MPFR library is a C library for multiple-precision floating-point
 computations with correct rounding.

 - [MUMPS/5.6.1-foss-2023a-metis](https://graal.ens-lyon.fr/MUMPS/)
A parallel sparse direct solver
 - [MUMmer/4.0.0rc1-GCCcore-12.3.0](https://mummer.sourceforge.net/)
MUMmer is a system for rapidly aligning entire genomes,
 whether in complete or draft form. AMOS makes use of it.

 - [MUSCLE/5.1.0-GCCcore-12.3.0](https://drive5.com/muscle/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/MUSCLE/MUSCLE-5.1.0-GCCcore-12.3.0.eb)
MUSCLE is one of the best-performing multiple alignment programs
 according to published benchmark tests, with accuracy and speed that are consistently
 better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users
 learn everything they need to know about MUSCLE in a few minutes-only a handful of
 command-line options are needed to perform common alignment tasks.
 - [MultiQC/1.21-foss-2023a](https://multiqc.info)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/m/MultiQC/MultiQC-1.21-foss-2023a.eb)
Aggregate results from bioinformatics analyses across many samples into a single 
 report.

 MultiQC searches a given directory for analysis logs and compiles a HTML report. It's a general
 use tool, perfect for summarising the output from numerous bioinformatics tools.
 - [OpenMM/8.0.0-foss-2023a-CUDA-12.1.1](https://openmm.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/o/OpenMM/OpenMM-8.0.0-foss-2023a-CUDA-12.1.1.eb)
OpenMM is a toolkit for molecular simulation.
 - [Osi/0.108.9-GCC-13.2.0](https://github.com/coin-or/Osi)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/o/Osi/Osi-0.108.9-GCC-13.2.0.eb)
Osi (Open Solver Interface) provides an abstract base class to a generic linear
programming (LP) solver, along with derived classes for specific solvers. Many
applications may be able to use the Osi to insulate themselves from a specific
LP solver. That is, programs written to the OSI standard may be linked to any
solver with an OSI interface and should produce correct results. The OSI has
been significantly extended compared to its first incarnation. Currently, the
OSI supports linear programming solvers and has rudimentary support for integer
programming.
 - [Porechop/0.2.4-GCCcore-12.3.0](https://github.com/rrwick/Porechop)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/p/Porechop/Porechop-0.2.4-GCCcore-12.3.0.eb)
Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.
 Adapters on the ends of reads are trimmed off, and when a read has an adapter in its middle,
 it is treated as chimeric and chopped into separate reads. Porechop performs thorough alignments
 to effectively find adapters, even at low sequence identity
 - [PyTorch/2.1.2-foss-2023a-CUDA-12.1.1](https://pytorch.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/p/PyTorch/PyTorch-2.1.2-foss-2023a-CUDA-12.1.1.eb)
Tensors and Dynamic neural networks in Python with strong GPU acceleration.
PyTorch is a deep learning framework that puts Python first.
 - [PyTorch-bundle/2.1.2-foss-2023a-CUDA-12.1.1](https://pytorch.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/p/PyTorch-bundle/PyTorch-bundle-2.1.2-foss-2023a-CUDA-12.1.1.eb)
PyTorch with compatible versions of official Torch extensions.
 - [Pysam/0.22.0-GCC-12.3.0](https://github.com/pysam-developers/pysam)
Pysam is a python module for reading and manipulating Samfiles.
 It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.
 - [Qhull/2020.2-GCCcore-13.2.0](http://www.qhull.org)
Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram,
 halfspace intersection about a point, furthest-site Delaunay triangulation,
 and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and
 higher dimensions. Qhull implements the Quickhull algorithm for computing the
 convex hull.

 - [R-bundle-Bioconductor/3.18-foss-2023a-R-4.3.2](https://bioconductor.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.18-foss-2023a-R-4.3.2.eb)
Bioconductor provides tools for the analysis and coprehension
 of high-throughput genomic data.
 - [RAPIDS/24.4-foss-2023a-CUDA-12.1.1](https://rapids.ai/)
RAPIDS provides unmatched speed with familiar APIs that match the most popular
PyData libraries. Built on state-of-the-art foundations like NVIDIA CUDA and
Apache Arrow, it unlocks the speed of GPUs with code you already know.
 - [RDKit/2024.03.5-foss-2024a](https://www.rdkit.org)
RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python.
 - [SAMtools/1.19.2-GCC-13.2.0](https://www.htslib.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/SAMtools/SAMtools-1.19.2-GCC-13.2.0.eb)
SAM Tools provide various utilities for manipulating alignments in the SAM format, 
 including sorting, merging, indexing and generating alignments in a per-position format.
 - [SCOTCH/7.0.3-gompi-2022b](https://www.labri.fr/perso/pelegrin/scotch/)
Software package and libraries for sequential and parallel graph partitioning,
static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning.
 - [STAR/2.7.11b-GCC-13.2.0](https://github.com/alexdobin/STAR)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/STAR/STAR-2.7.11b-GCC-13.2.0.eb)
STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.
 - [STAR-Fusion/1.12.0-foss-2022b](https://github.com/STAR-Fusion/STAR-Fusion)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/STAR-Fusion/STAR-Fusion-1.12.0-foss-2022b.eb)
STAR-Fusion uses the STAR aligner to identify candidate fusion transcripts
 supported by Illumina reads. STAR-Fusion further processes the output generated by the STAR aligner
 to map junction reads and spanning reads to a reference annotation set.
 - [SVclone/1.1.2-foss-2022b](https://github.com/mcmero/SVclone)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/SVclone/SVclone-1.1.2-foss-2022b.eb)
Cluster structural variants of similar cancer cell fraction (CCF).
 - [SYMPHONY/5.7.2-foss-2023b](https://github.com/coin-or/SYMPHONY/wiki)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/SYMPHONY/SYMPHONY-5.7.2-foss-2023b.eb)
SYMPHONY is an open-source solver for mixed-integer linear programs (MILPs) written in C.
 - [Sambamba/1.0.1-GCC-13.2.0](https://lomereiter.github.io/sambamba/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/Sambamba/Sambamba-1.0.1-GCC-13.2.0.eb)
Sambamba is a high performance modern robust and fast tool
 (and library), written in the D programming language, for working with SAM
 and BAM files. Current functionality is an important subset of samtools
 functionality, including view, index, sort, markdup, and depth.
 - [Seaborn/0.13.2-gfbf-2023b](https://seaborn.pydata.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/Seaborn/Seaborn-0.13.2-gfbf-2023b.eb)
Seaborn is a Python visualization library based on matplotlib.
 It provides a high-level interface for drawing attractive statistical graphics. 
 - [Shapely/2.0.1-gfbf-2023a](https://github.com/Toblerity/Shapely)
Shapely is a BSD-licensed Python package for manipulation and analysis of planar geometric objects.
It is based on the widely deployed GEOS (the engine of PostGIS) and JTS (from which GEOS is ported) libraries.
 - [Telescope/1.0.3-20230222-gfbf-2022b](https://github.com/mlbendall/telescope)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/t/Telescope/Telescope-1.0.3-20230222-gfbf-2022b.eb)
Single locus resolution of Transposable ELEment expression using next-generation sequencing.
 - [UMI-tools/1.1.4-foss-2023b](https://umi-tools.readthedocs.io)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/u/UMI-tools/UMI-tools-1.1.4-foss-2023b.eb)
Tools for handling Unique Molecular Identifiers in NGS data sets
 - [anndata/0.10.7-foss-2023b](https://github.com/scverse/anndata)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/a/anndata/anndata-0.10.7-foss-2023b.eb)
anndata is a Python package for handling annotated data matrices in memory and on disk,
 positioned between pandas and xarray
 - [bam-readcount/1.0.1-GCC-12.2.0](https://github.com/genome/bam-readcount)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/b/bam-readcount/bam-readcount-1.0.1-GCC-12.2.0.eb)
Count DNA sequence reads in BAM files
 - [biom-format/2.1.15-foss-2023a](https://biom-format.org)
The BIOM file format (canonically pronounced biome) is designed to be
 a general-use format for representing biological sample by observation
 contingency tables. BIOM is a recognized standard for the Earth Microbiome
 Project and is a Genomics Standards Consortium supported project.

 - [cooler/0.10.2-foss-2023a](https://open2c.github.io/cooler)
Cooler is a support library for a storage format, also called cooler, used to store
 genomic interaction data of any size, such as Hi-C contact matrices.
 - [cutadapt/4.4-GCCcore-12.2.0](https://opensource.scilifelab.se/projects/cutadapt/)
Cutadapt finds and removes adapter sequences, primers, poly-A tails and
 other types of unwanted sequence from your high-throughput sequencing reads.
 - [easel/0.48-GCC-12.2.0](https://bioeasel.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/e/easel/easel-0.48-GCC-12.2.0.eb)
Easel supports computational analysis of biological sequences using probabilistic models.
 - [einops/0.7.0-GCCcore-12.3.0](https://einops.rocks/)
Flexible and powerful tensor operations for readable and reliable code.
Supports numpy, pytorch, tensorflow, jax, and others.
 - [fastp/0.23.4-GCC-13.2.0](https://github.com/OpenGene/fastp)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/f/fastp/fastp-0.23.4-GCC-13.2.0.eb)
A tool designed to provide fast all-in-one preprocessing for FastQ files.
 This tool is developed in C++ with multithreading supported to afford high performance.
 - [fhR/4.4.0-foss-2023b](http://www.r-project.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/f/fhR/fhR-4.4.0-foss-2023b.eb)
R is a free software environment for statistical computing and graphics.
 - [gffread/0.12.7-GCCcore-12.2.0](https://ccb.jhu.edu/software/stringtie/gff.shtml#gffread)
GFF/GTF parsing utility providing format conversions,
region filtering, FASTA sequence extraction and more.
 - [gmpy2/2.1.5-GCC-12.3.0](https://github.com/aleaxit/gmpy)
GMP/MPIR, MPFR, and MPC interface to Python 2.6+ and 3.x
 - [jax/0.4.25-gfbf-2023a-CUDA-12.1.1](https://jax.readthedocs.io/)
Composable transformations of Python+NumPy programs:
differentiate, vectorize, JIT to GPU/TPU, and more
 - [kallisto/0.50.1-foss-2022b](https://pachterlab.github.io/kallisto/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/k/kallisto/kallisto-0.50.1-foss-2022b.eb)
kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally
 of target sequences using high-throughput sequencing reads.
 - [libcerf/2.3-GCCcore-12.2.0](https://jugit.fz-juelich.de/mlz/libcerf)
libcerf is a self-contained numeric library that provides an efficient and
 accurate implementation of complex error functions, along with Dawson,
 Faddeeva, and Voigt functions.

 - [loompy/3.0.7-foss-2023a](https://loompy.org/)
Python implementation of the Loom file format, an efficient file format for large omics datasets
 - [magma/2.7.2-foss-2023a-CUDA-12.1.1](https://icl.cs.utk.edu/magma/)
The MAGMA project aims to develop a dense linear algebra library similar to
 LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems.
 - [minimap2/2.26-GCCcore-12.3.0](https://github.com/lh3/minimap2)
Minimap2 is a fast sequence mapping and alignment
program that can find overlaps between long noisy reads, or map long
reads or their assemblies to a reference genome optionally with detailed
alignment (i.e. CIGAR). At present, it works efficiently with query
sequences from a few kilobases to ~100 megabases in length at an error
rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited
test data sets, minimap2 is over 20 times faster than most other
long-read aligners. It will replace BWA-MEM for long reads and contig
alignment.
 - [ncbi-vdb/3.1.1-gompi-2023b](https://github.com/ncbi/ncbi-vdb)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/n/ncbi-vdb/ncbi-vdb-3.1.1-gompi-2023b.eb)
The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for
 using data in the INSDC Sequence Read Archives.
 - [pyEGA3/5.0.2-GCCcore-12.3.0](https://github.com/EGA-archive/ega-download-client)
A basic Python-based EGA download client 
 - [pybedtools/0.9.1-foss-2023a](https://daler.github.io/pybedtools)
pybedtools wraps and extends BEDTools and offers feature-level manipulations from within Python.
 - [pyfaidx/0.8.1.1-GCCcore-12.3.0](https://pypi.python.org/pypi/pyfaidx)
pyfaidx: efficient pythonic random access to fasta subsequences
 - [samblaster/0.1.26-GCC-13.2.0](https://github.com/GregoryFaust/samblaster)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/samblaster/samblaster-0.1.26-GCC-13.2.0.eb)
samblaster is a fast and flexible program for marking
 duplicates in read-id grouped1 paired-end SAM files.
 - [scGPT/0.2.1-foss-2023a-CUDA-12.1.1](https://github.com/bowang-lab/scGPT/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/scGPT/scGPT-0.2.1-foss-2023a-CUDA-12.1.1.eb)
scGPT: Towards Building a Foundation Model for Single-Cell Multi-omics Using Generative AI.
 - [scVelo/0.3.1-foss-2023a](https://scvelo.org)
scVelo is a scalable toolkit for estimating and analyzing RNA velocities in single cells using
 dynamical modeling.
 - [scanpy/1.10.1-foss-2023b](https://scanpy.readthedocs.io/en/stable/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/easyconfigs/s/scanpy/scanpy-1.10.1-foss-2023b.eb)
Scanpy is a scalable toolkit for analyzing single-cell gene expression data built
 jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference
 and differential expression testing. The Python-based implementation efficiently deals with
 datasets of more than one million cells.

 - [scikit-bio/0.6.0-foss-2023a](http://scikit-bio.org)
scikit-bio is an open-source, BSD-licensed Python 3 package providing data structures, algorithms
and educational resources for bioinformatics.
 - [scvi-tools/1.1.2-foss-2023a-CUDA-12.1.1](https://github.com/scverse/scvi-tools)
scvi-tools (single-cell variational inference tools) is a package for probabilistic modeling and
analysis of single-cell omics data, built on top of PyTorch and AnnData.
 - [splitpipe/1.2.1-foss-2023b](https://support.parsebiosciences.com/hc/en-us)
splitpipe tool from Parse Biosciences. The pipeline takes FASTQ files and delivers
 processed data in the form of a cell-gene count matrix, which serves as the input for various open
 sources tools such as scanpy and seuratProcess sequencing results with our pipeline.

 A Parse Bioscience login ID is required to download
 - [starcode/1.4-GCC-13.2.0](https://github.com/gui11aume/starcode)
Starcode is a DNA sequence clustering software. Starcode clustering is based on all pairs
search within a specified Levenshtein distance (allowing insertions and deletions), followed by a clustering
algorithm: Message Passing, Spheres or Connected Components.
 - [statsmodels/0.14.1-gfbf-2023a](https://www.statsmodels.org/)
Statsmodels is a Python module that allows users to explore data, estimate statistical models,
and perform statistical tests.
 - [sympy/1.12-gfbf-2023a](https://sympy.org/)
SymPy is a Python library for symbolic mathematics. It aims to
 become a full-featured computer algebra system (CAS) while keeping the code as
 simple as possible in order to be comprehensible and easily extensible. SymPy
 is written entirely in Python and does not require any external libraries.
 - [wandb/0.16.1-GCC-12.3.0](https://wandb.ai)
CLI and Python API for Weights and Biases (wandb), a tool for visualizing and
tracking your machine learning experiments.
