---
title: Bio Modules  18.04
date: 2019-01-10
layout: single
sidebar:
  nav: "docs"
---

 - [BCFtools/1.9-foss-2018b](http://www.htslib.org/)  
Samtools is a suite of programs for interacting with high-throughput sequencing data.
 BCFtools
 - [BEDOPS/2.4.35-foss-2018b](http://bedops.readthedocs.io/en/latest/index.html)  
BEDOPS is an open-source command-line toolkit that performs highly efficient and
 scalable Boolean and other set operations, statistical calculations, archiving, conversion and
 other management of genomic data of arbitrary scale. Tasks can be easily split by chromosome for
 distributing whole-genome analyses across a computational cluster.
 - [BEDTools/2.27.1-foss-2018b](https://github.com/arq5x/bedtools2)  
The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps
 and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF,
 and SAM/BAM.
 - [BLAST+/2.8.1-foss-2018b](http://blast.ncbi.nlm.nih.gov/)  
Basic Local Alignment Search Tool, or BLAST, is an algorithm
 for comparing primary biological sequence information, such as the amino-acid
 sequences of different proteins or the nucleotides of DNA sequences.
 - [BWA/0.7.17-foss-2018b](http://bio-bwa.sourceforge.net/)  
Burrows-Wheeler Aligner (BWA) is an efficient program that aligns
 relatively short nucleotide sequences against a long reference sequence such as the human genome.
 - [BamTools/2.5.1-foss-2018b](https://github.com/pezmaster31/bamtools)  
BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.
 - [Beast/2.5.1-foss-2018b](http://beast2.org/)  
BEAST is a cross-platform program for Bayesian MCMC analysis of molecular 
 sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using 
 strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies 
 but is also a framework for testing evolutionary hypotheses without conditioning on a single 
 tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted 
 proportional to its posterior probability. 
 - [BioPerl/1.7.2-foss-2018b-Perl-5.28.0](http://www.bioperl.org/)  
Bioperl is the product of a community effort to produce Perl code which is useful in biology.
 Examples include Sequence objects, Alignment objects and database searching objects.
 - [Bowtie2/2.3.4.2-foss-2018b](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml)  
Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads
 to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s
 of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.
 Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,
 its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.
 - [Eigen/3.3.5](http://eigen.tuxfamily.org/index.php?title=Main_Page)  
Eigen is a C++ template library for linear algebra:
 matrices, vectors, numerical solvers, and related algorithms.

 - [FastANI/1.1-foss-2018b](http://www.iodbc.org/)  
FastANI is developed for fast alignment-free computation of
 whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean
 nucleotide identity of orthologous gene pairs shared between two microbial
 genomes. FastANI supports pairwise comparison of both complete and draft
 genome assemblies.
 - [FastQC/0.11.8-Java-1.8](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/)  
FastQC is a quality control application for high throughput
sequence data. It reads in sequence data in a variety of formats and can either
provide an interactive application to review the results of several different
QC checks, or create an HTML based report which can be integrated into a
pipeline.
 - [GATK/4.0.8.1-foss-2018b-Python-3.6.6](http://www.broadinstitute.org/gatk/)  
The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute
 to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,
 with a primary focus on variant discovery and genotyping as well as strong emphasis on
 data quality assurance. Its robust architecture, powerful processing engine and
 high-performance computing features make it capable of taking on projects of any size.
 - [GEOS/3.6.2-foss-2018b-Python-3.6.6](http://trac.osgeo.org/geos)  
GEOS (Geometry Engine
 - [GMP/6.1.2-GCCcore-7.3.0](http://gmplib.org/)  
GMP is a free library for arbitrary precision arithmetic, operating on signed
 integers, rational numbers, and floating point numbers.

 - [HTSlib/1.9-foss-2018b](http://www.htslib.org/)  
A C library for reading/writing high-throughput sequencing data.
 This package includes the utilities bgzip and tabix
 - [IGV/2.4.16-Java-1.8](http://www.broadinstitute.org/software/igv/)  
This package contains command line utilities for preprocessing, 
 computing feature count density (coverage),  sorting, and indexing data files.
 See also http://www.broadinstitute.org/software/igv/igvtools_commandline. 
 - [MiXCR/3.0.3-Java-1.8](https://milaboratory.com/software/mixcr/)  
MiXCR processes big immunome data from raw sequences 
to quantitated clonotypes 
 - [SAMtools/1.9-foss-2018b](http://www.htslib.org/)  
SAM Tools provide various utilities for manipulating alignments in the SAM format, 
 including sorting, merging, indexing and generating alignments in a per-position format.
 - [SRA-Toolkit/2.9.2-ubuntu64](http://trace.ncbi.nlm.nih.gov/Traces/sra)  
The NCBI SRA Toolkit enables reading (dumping) of sequencing files 
 from the SRA database and writing (loading) files into the .sra format 
 - [STAR/2.6.1c-foss-2018b](https://github.com/alexdobin/STAR)  
STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.
 - [VCFtools/0.1.16-foss-2018b-Perl-5.28.0](https://vcftools.github.io)  
The aim of VCFtools is to provide
 easily accessible methods for working with complex
 genetic variation data in the form of VCF files.
 - [canu/1.8-foss-2018b](http://canu.readthedocs.io)  
Canu is a fork of the Celera Assembler designed for
 high-noise single-molecule sequencing
 - [nanopolish/0.10.2-foss-2018b](http://simpsonlab.github.io/2015/10/07/nanopolish-v0.4.0/)  
Our computational work focuses on the development of
algorithms for handling very large sequencing data sets, from indexing
and text compression, to probabilistic modeling and inference. We make
our methods freely available as open source software.
 - [picard/2.18.17-Java-1.8](http://sourceforge.net/projects/picard)  
A set of tools (in Java) for working with next generation sequencing data in the BAM format.
 - [samblaster/v.0.1.24-foss-2018b](https://github.com/GregoryFaust/samblaster)  
samblaster is a fast and flexible program for marking
 duplicates in read-id grouped1 paired-end SAM files.
 - [tabix/0.2.6-GCCcore-7.3.0](http://samtools.sourceforge.net)  
Generic indexer for TAB-delimited genome position files 
