---
title: Bio Modules 24.04
layout: single
permalink: /bio-modules-24.04/
created: 2024-09-26
toc: true
toc_label: "On This Page"
sidebar:
  nav: "docs"
---

 - [AGAT/1.4.0-GCC-12.3.0](https://agat.readthedocs.io/en/latest/)
AGAT: Another GTF/GFF Analysis Toolkit. Suite of tools to handle gene annotations
 in any GTF/GFF format.
 - [AlphaFold/2.3.2-foss-2023a-CUDA-12.1.1](https://deepmind.com/research/case-studies/alphafold)
AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known
 - [BCFtools/1.19-GCC-13.2.0](https://www.htslib.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/b/BCFtools/BCFtools-1.19-GCC-13.2.0.eb)
Samtools is a suite of programs for interacting with high-throughput sequencing data.
 BCFtools
 - [BEDTools/2.31.0-GCC-12.3.0](https://bedtools.readthedocs.io/)
BEDTools: a powerful toolset for genome arithmetic.
The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and
computing coverage.
The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM.
 - [BamTools/2.5.2-GCC-12.3.0](https://github.com/pezmaster31/bamtools)
BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.
 - [BioPerl/1.7.8-GCCcore-12.3.0](https://bioperl.org/)
Bioperl is the product of a community effort to produce Perl code which is useful in biology.
 Examples include Sequence objects, Alignment objects and database searching objects.
 - [Biopython/1.83-foss-2023a](https://www.biopython.org)
Biopython is a set of freely available tools for biological
 computation written in Python by an international team of developers. It is
 a distributed collaborative effort to develop Python libraries and
 applications which address the needs of current and future work in
 bioinformatics. 
 - [CellRank/2.0.2-foss-2023a-CUDA-12.1.1](https://cellrank.readthedocs.io/en/stable/)
CellRank is a toolkit to uncover cellular dynamics based on
 Markov state modeling of single-cell data.  It contains two main modules:
kernels compute cell-cell transition probabilities and estimators generate
hypothesis based on these. 
 - [Eigen/3.4.0-GCCcore-12.3.0](https://eigen.tuxfamily.org)
Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers,
 and related algorithms.
 - [GEOS/3.12.1-GCC-13.2.0](https://trac.osgeo.org/geos)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/g/GEOS/GEOS-3.12.1-GCC-13.2.0.eb)
GEOS (Geometry Engine
 - [GMP/6.3.0-GCCcore-13.2.0](https://gmplib.org/)
GMP is a free library for arbitrary precision arithmetic, operating on signed
 integers, rational numbers, and floating point numbers.

 - [HH-suite/3.3.0-gompi-2023a](https://github.com/soedinglab/hh-suite)
The HH-suite is an open-source software package
 for sensitive protein sequence searching based on the pairwise
 alignment of hidden Markov models (HMMs).
 - [HMMER/3.4-gompi-2023a](http://hmmer.org/)
HMMER is used for searching sequence databases for homologs
 of protein sequences, and for making protein sequence alignments. It
 implements methods using probabilistic models called profile hidden Markov
 models (profile HMMs).  Compared to BLAST, FASTA, and other sequence
 alignment and database search tools based on older scoring methodology,
 HMMER aims to be significantly more accurate and more able to detect remote
 homologs because of the strength of its underlying mathematical models. In the
 past, this strength came at significant computational expense, but in the new
 HMMER3 project, HMMER is now essentially as fast as BLAST.
 - [HTSlib/1.19.1-GCC-13.2.0](https://www.htslib.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/h/HTSlib/HTSlib-1.19.1-GCC-13.2.0.eb)
A C library for reading/writing high-throughput sequencing data.
 This package includes the utilities bgzip and tabix
 - [Kalign/3.4.0-GCCcore-12.3.0](https://github.com/TimoLassmann/kalign)
Kalign is a fast multiple sequence alignment program for biological sequences.
 - [MPC/1.3.1-GCCcore-12.3.0](http://www.multiprecision.org/)
Gnu Mpc is a C library for the arithmetic of
 complex numbers with arbitrarily high precision and correct
 rounding of the result. It extends the principles of the IEEE-754
 standard for fixed precision real floating point numbers to
 complex numbers, providing well-defined semantics for every
 operation. At the same time, speed of operation at high precision
 is a major design goal.
 - [MPFR/4.2.1-GCCcore-13.2.0](https://www.mpfr.org)
The MPFR library is a C library for multiple-precision floating-point
 computations with correct rounding.

 - [OpenMM/8.0.0-foss-2023a-CUDA-12.1.1](https://openmm.org)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/o/OpenMM/OpenMM-8.0.0-foss-2023a-CUDA-12.1.1.eb)
OpenMM is a toolkit for molecular simulation.
 - [PyTorch/2.1.2-foss-2023a-CUDA-12.1.1](https://pytorch.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/p/PyTorch/PyTorch-2.1.2-foss-2023a-CUDA-12.1.1.eb)
Tensors and Dynamic neural networks in Python with strong GPU acceleration.
PyTorch is a deep learning framework that puts Python first.
 - [Pysam/0.22.0-GCC-12.3.0](https://github.com/pysam-developers/pysam)
Pysam is a python module for reading and manipulating Samfiles.
 It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.
 - [Qhull/2020.2-GCCcore-12.3.0](http://www.qhull.org)
Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram,
 halfspace intersection about a point, furthest-site Delaunay triangulation,
 and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and
 higher dimensions. Qhull implements the Quickhull algorithm for computing the
 convex hull.

 - [RAPIDS/24.4-foss-2023a-CUDA-12.1.1](https://rapids.ai/)
RAPIDS provides unmatched speed with familiar APIs that match the most popular
PyData libraries. Built on state-of-the-art foundations like NVIDIA CUDA and
Apache Arrow, it unlocks the speed of GPUs with code you already know.
 - [SAMtools/1.19.2-GCC-13.2.0](https://www.htslib.org/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/SAMtools/SAMtools-1.19.2-GCC-13.2.0.eb)
SAM Tools provide various utilities for manipulating alignments in the SAM format, 
 including sorting, merging, indexing and generating alignments in a per-position format.
 - [STAR/2.7.11b-GCC-13.2.0](https://github.com/alexdobin/STAR)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/STAR/STAR-2.7.11b-GCC-13.2.0.eb)
STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.
 - [Seaborn/0.13.2-gfbf-2023a](https://seaborn.pydata.org/)
Seaborn is a Python visualization library based on matplotlib.
 It provides a high-level interface for drawing attractive statistical graphics. 
 - [Shapely/2.0.1-gfbf-2023a](https://github.com/Toblerity/Shapely)
Shapely is a BSD-licensed Python package for manipulation and analysis of planar geometric objects.
It is based on the widely deployed GEOS (the engine of PostGIS) and JTS (from which GEOS is ported) libraries.
 - [anndata/0.10.5.post1-foss-2023a](https://github.com/scverse/anndata)
anndata is a Python package for handling annotated data matrices in memory and on disk,
 positioned between pandas and xarray
 - [einops/0.7.0-GCCcore-12.3.0](https://einops.rocks/)
Flexible and powerful tensor operations for readable and reliable code.
Supports numpy, pytorch, tensorflow, jax, and others.
 - [fastp/0.23.4-GCC-13.2.0](https://github.com/OpenGene/fastp)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/f/fastp/fastp-0.23.4-GCC-13.2.0.eb)
A tool designed to provide fast all-in-one preprocessing for FastQ files.
 This tool is developed in C++ with multithreading supported to afford high performance.
 - [gmpy2/2.1.5-GCC-12.3.0](https://github.com/aleaxit/gmpy)
GMP/MPIR, MPFR, and MPC interface to Python 2.6+ and 3.x
 - [jax/0.4.25-gfbf-2023a-CUDA-12.1.1](https://jax.readthedocs.io/)
Composable transformations of Python+NumPy programs:
differentiate, vectorize, JIT to GPU/TPU, and more
 - [loompy/3.0.7-foss-2023a](https://loompy.org/)
Python implementation of the Loom file format, an efficient file format for large omics datasets
 - [magma/2.7.2-foss-2023a-CUDA-12.1.1](https://icl.cs.utk.edu/magma/)
The MAGMA project aims to develop a dense linear algebra library similar to
 LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems.
 - [pybedtools/0.9.1-foss-2023a](https://daler.github.io/pybedtools)
pybedtools wraps and extends BEDTools and offers feature-level manipulations from within Python.
 - [scVelo/0.3.1-foss-2023a](https://scvelo.org)
scVelo is a scalable toolkit for estimating and analyzing RNA velocities in single cells using
 dynamical modeling.
 - [scanpy/1.9.8-foss-2023a](https://scanpy.readthedocs.io/en/stable/)
[easyconfig](https://github.com/FredHutch/easybuild-life-sciences/blob/master/fh_easyconfigs/s/scanpy/scanpy-1.9.8-foss-2023a.eb)
Scanpy is a scalable toolkit for analyzing single-cell gene expression data built
 jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference
 and differential expression testing. The Python-based implementation efficiently deals with
 datasets of more than one million cells.

 - [starcode/1.4-GCC-13.2.0](https://github.com/gui11aume/starcode)
Starcode is a DNA sequence clustering software. Starcode clustering is based on all pairs
search within a specified Levenshtein distance (allowing insertions and deletions), followed by a clustering
algorithm: Message Passing, Spheres or Connected Components.
 - [statsmodels/0.14.1-gfbf-2023a](https://www.statsmodels.org/)
Statsmodels is a Python module that allows users to explore data, estimate statistical models,
and perform statistical tests.
 - [sympy/1.12-gfbf-2023a](https://sympy.org/)
SymPy is a Python library for symbolic mathematics. It aims to
 become a full-featured computer algebra system (CAS) while keeping the code as
 simple as possible in order to be comprehensible and easily extensible. SymPy
 is written entirely in Python and does not require any external libraries.
 - [wandb/0.16.1-GCC-12.3.0](https://wandb.ai)
CLI and Python API for Weights and Biases (wandb), a tool for visualizing and
tracking your machine learning experiments.
