diff -ruN revbayes-1.1.1.orig/src/CMakeLists.txt revbayes-1.1.1/src/CMakeLists.txt
--- revbayes-1.1.1.orig/src/CMakeLists.txt	2021-02-11 13:23:31.000000000 -0800
+++ revbayes-1.1.1/src/CMakeLists.txt	2021-03-06 10:39:20.763065000 -0800
@@ -91,15 +91,7 @@
 set(Boost_USE_MULTITHREADED ON)
 
 MESSAGE("Searching for BOOST:")
-find_package(Boost
-1.60.0
-COMPONENTS regex
-program_options
-thread
-system
-filesystem
-date_time
-serialization REQUIRED)
+find_package(Boost 1.60.0)
 MESSAGE("Boost configuration results:")
 MESSAGE("  Boost_INCLUDE_DIRS: ${Boost_INCLUDE_DIR}")
 MESSAGE("  Boost_LIBRARY_DIRS: ${Boost_LIBRARY_DIRS}")
